{
  "cells": [
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "view-in-github",
        "colab_type": "text"
      },
      "source": [
        "<a href=\"https://colab.research.google.com/github/maryamathar/2022S1/blob/main/A2.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "S9pWsPAt0dCh"
      },
      "source": [
        "# Assignment 2 - Find complex answers to medical questions\n",
        "\n",
        "**Submission deadline: Friday 22 April, 5pm.** \n",
        "\n",
        "Late submissions **will not be accepted** without an approved [Special Consideration](http://from.mq.edu.au/MT0X0E0FUrrU200rm0JB0U0) request.  Assessments submitted after the due date will receive a mark of **zero**.\n",
        "\n",
        "**Assessment marks: 20 marks (20% of the total unit assessment)**\n",
        "\n",
        "In this assignment we will work on a task of \"query-focused summarisation\" on medical questions where the goal is, given a medical question and a list of sentences extracted from relevant medical publications, to determine which of these sentences from the list can be used as part of the answer to the question.\n",
        "\n",
        "We will use data that has been derived from the **BioASQ challenge** (http://www.bioasq.org/), after some data manipulation to make it easier to process for this assignment. The BioASQ challenge organises several \"shared tasks\", including a task on biomedical semantic question answering which we are using here. The data are in the file `bioasq10_labelled.csv`, which is part of the zip file provided. Each row of the file has a question, a sentence text, and a label that indicates whether the sentence text is part of the answer to the question (1) or not (0).\n",
        "\n",
        "The following code uses pandas to store the file `bioasq10_labelled.csv` in a data frame and show the first rows of data. For this code to run, first you need to unzip the file `data.zip`:"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 1,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "omIGbpBfOC5a",
        "outputId": "76153c47-e73b-4828-e047-84e4d63a47ef"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "unzip:  cannot find or open data.zip, data.zip.zip or data.zip.ZIP.\n"
          ]
        }
      ],
      "source": [
        "!unzip data.zip"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 4,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 206
        },
        "id": "bgBeNNuwOC5b",
        "outputId": "3e4f8830-55f6-48e9-83fd-88789fed9550"
      },
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "   qid  sentid                                           question  \\\n",
              "0    0       0  Is Hirschsprung disease a mendelian or a multi...   \n",
              "1    0       1  Is Hirschsprung disease a mendelian or a multi...   \n",
              "2    0       2  Is Hirschsprung disease a mendelian or a multi...   \n",
              "3    0       3  Is Hirschsprung disease a mendelian or a multi...   \n",
              "4    0       4  Is Hirschsprung disease a mendelian or a multi...   \n",
              "\n",
              "                                       sentence text  label  \n",
              "0  Hirschsprung disease (HSCR) is a multifactoria...      0  \n",
              "1  In this study, we review the identification of...      1  \n",
              "2  The majority of the identified genes are relat...      1  \n",
              "3  The non-Mendelian inheritance of sporadic non-...      1  \n",
              "4                  Coding sequence mutations in e.g.      0  "
            ],
            "text/html": [
              "\n",
              "  <div id=\"df-6e376039-15fc-43bd-b2bf-9dd7d78c3331\">\n",
              "    <div class=\"colab-df-container\">\n",
              "      <div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>qid</th>\n",
              "      <th>sentid</th>\n",
              "      <th>question</th>\n",
              "      <th>sentence text</th>\n",
              "      <th>label</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>0</th>\n",
              "      <td>0</td>\n",
              "      <td>0</td>\n",
              "      <td>Is Hirschsprung disease a mendelian or a multi...</td>\n",
              "      <td>Hirschsprung disease (HSCR) is a multifactoria...</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>0</td>\n",
              "      <td>1</td>\n",
              "      <td>Is Hirschsprung disease a mendelian or a multi...</td>\n",
              "      <td>In this study, we review the identification of...</td>\n",
              "      <td>1</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2</th>\n",
              "      <td>0</td>\n",
              "      <td>2</td>\n",
              "      <td>Is Hirschsprung disease a mendelian or a multi...</td>\n",
              "      <td>The majority of the identified genes are relat...</td>\n",
              "      <td>1</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3</th>\n",
              "      <td>0</td>\n",
              "      <td>3</td>\n",
              "      <td>Is Hirschsprung disease a mendelian or a multi...</td>\n",
              "      <td>The non-Mendelian inheritance of sporadic non-...</td>\n",
              "      <td>1</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4</th>\n",
              "      <td>0</td>\n",
              "      <td>4</td>\n",
              "      <td>Is Hirschsprung disease a mendelian or a multi...</td>\n",
              "      <td>Coding sequence mutations in e.g.</td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "</div>\n",
              "      <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-6e376039-15fc-43bd-b2bf-9dd7d78c3331')\"\n",
              "              title=\"Convert this dataframe to an interactive table.\"\n",
              "              style=\"display:none;\">\n",
              "        \n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "       width=\"24px\">\n",
              "    <path d=\"M0 0h24v24H0V0z\" fill=\"none\"/>\n",
              "    <path d=\"M18.56 5.44l.94 2.06.94-2.06 2.06-.94-2.06-.94-.94-2.06-.94 2.06-2.06.94zm-11 1L8.5 8.5l.94-2.06 2.06-.94-2.06-.94L8.5 2.5l-.94 2.06-2.06.94zm10 10l.94 2.06.94-2.06 2.06-.94-2.06-.94-.94-2.06-.94 2.06-2.06.94z\"/><path d=\"M17.41 7.96l-1.37-1.37c-.4-.4-.92-.59-1.43-.59-.52 0-1.04.2-1.43.59L10.3 9.45l-7.72 7.72c-.78.78-.78 2.05 0 2.83L4 21.41c.39.39.9.59 1.41.59.51 0 1.02-.2 1.41-.59l7.78-7.78 2.81-2.81c.8-.78.8-2.07 0-2.86zM5.41 20L4 18.59l7.72-7.72 1.47 1.35L5.41 20z\"/>\n",
              "  </svg>\n",
              "      </button>\n",
              "      \n",
              "  <style>\n",
              "    .colab-df-container {\n",
              "      display:flex;\n",
              "      flex-wrap:wrap;\n",
              "      gap: 12px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert {\n",
              "      background-color: #E8F0FE;\n",
              "      border: none;\n",
              "      border-radius: 50%;\n",
              "      cursor: pointer;\n",
              "      display: none;\n",
              "      fill: #1967D2;\n",
              "      height: 32px;\n",
              "      padding: 0 0 0 0;\n",
              "      width: 32px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert:hover {\n",
              "      background-color: #E2EBFA;\n",
              "      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "      fill: #174EA6;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert {\n",
              "      background-color: #3B4455;\n",
              "      fill: #D2E3FC;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert:hover {\n",
              "      background-color: #434B5C;\n",
              "      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "      fill: #FFFFFF;\n",
              "    }\n",
              "  </style>\n",
              "\n",
              "      <script>\n",
              "        const buttonEl =\n",
              "          document.querySelector('#df-6e376039-15fc-43bd-b2bf-9dd7d78c3331 button.colab-df-convert');\n",
              "        buttonEl.style.display =\n",
              "          google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "        async function convertToInteractive(key) {\n",
              "          const element = document.querySelector('#df-6e376039-15fc-43bd-b2bf-9dd7d78c3331');\n",
              "          const dataTable =\n",
              "            await google.colab.kernel.invokeFunction('convertToInteractive',\n",
              "                                                     [key], {});\n",
              "          if (!dataTable) return;\n",
              "\n",
              "          const docLinkHtml = 'Like what you see? Visit the ' +\n",
              "            '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
              "            + ' to learn more about interactive tables.';\n",
              "          element.innerHTML = '';\n",
              "          dataTable['output_type'] = 'display_data';\n",
              "          await google.colab.output.renderOutput(dataTable, element);\n",
              "          const docLink = document.createElement('div');\n",
              "          docLink.innerHTML = docLinkHtml;\n",
              "          element.appendChild(docLink);\n",
              "        }\n",
              "      </script>\n",
              "    </div>\n",
              "  </div>\n",
              "  "
            ]
          },
          "metadata": {},
          "execution_count": 4
        }
      ],
      "source": [
        "import pandas as pd\n",
        "dataset = pd.read_csv(\"bioasq10b_labelled.csv\")\n",
        "dataset.head()"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 5,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "zV-AcQxfOC5d",
        "outputId": "4bcc1d4f-2bb3-4773-bc27-e4e4d9fc2a7a"
      },
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "64112"
            ]
          },
          "metadata": {},
          "execution_count": 5
        }
      ],
      "source": [
        "dataset.qid.size\n"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "fG9-nDD6OC5e"
      },
      "source": [
        "The columns of the CSV file are:\n",
        "\n",
        "* `qid`: an ID for a question. Several rows may have the same question ID, as we can see above.\n",
        "* `sentid`: an ID for a sentence.\n",
        "* `question`: The text of the question. In the above example, the first rows all have the same question: \"Is Hirschsprung disease a mendelian or a multifactorial disorder?\"\n",
        "* `sentence text`: The text of the sentence.\n",
        "* `label`: 1 if the sentence is a part of the answer, 0 if the sentence is not part of the answer."
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "m6xqxCmR0dCk"
      },
      "source": [
        "# Task 1 (5 marks): Data preparation\n",
        "\n",
        "Partition the data into the training, dev_test, and test sets using the proportions 6:2:2. That is, 60% of the questions must be in the training set, 20% must be in the dev_test set, and the remaining 20% in the test set. Make sure that you partition based on the questions, not on the rows. With this we mean that all the sentences related to a question must be in one file only. In other words, there must not be sentences from the same question in, say, the training and the test data.\n",
        "\n",
        "Also, make sure that you implement a random partition.\n",
        "\n",
        "Save the partitions as the files `training.csv`, `dev_test.csv`, and `test.csv`, so that they can be used by other people.\n",
        "\n",
        "The breakdown of marks is as follows:\n",
        "\n",
        "* **1 mark** if your explanation answers the following question correctly: Why do we want to split the partition on the questions, and not on the rows?\n",
        "* **1 mark** if the code partitions the data on the questions randomly and according to the split 6:2:2.\n",
        "* **1 mark** if your code generates partitions that have similar balance of labels and you demonstrate that they are similar.\n",
        "* **1 mark** if the partitions are saved as the CSV files `training.csv`, `dev_test.csv`, and `test.csv`.\n",
        "* **1 mark** for good coding and documentation in this task.\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 6,
      "metadata": {
        "id": "fhqUtqJL0dCm"
      },
      "outputs": [],
      "source": [
        "import random\n",
        "random.seed(1234)\n",
        "import numpy as np\n",
        "import pandas as pd\n",
        "from sklearn.model_selection import train_test_split"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 7,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "chGWAIzyOC5g",
        "outputId": "d449b9c9-e840-4cae-ed83-6a4f4b1850c7"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "4234\n"
          ]
        }
      ],
      "source": [
        "questions = list(set(dataset['question']))\n",
        "print(len(questions))"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 10,
      "metadata": {
        "id": "Su2UC0Ti0dCo"
      },
      "outputs": [],
      "source": [
        "random.shuffle(questions)"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 11,
      "metadata": {
        "id": "awhzZH6wOC5h"
      },
      "outputs": [],
      "source": [
        "threshold1 = int(len(questions)*.6)\n",
        "threshold2 = int(len(questions)*.8)\n",
        "q_train = questions[:threshold1]\n",
        "q_devtest = questions[threshold1:threshold2]\n",
        "q_test = questions[threshold2:]\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 12,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "f27Ew2s-OC5h",
        "outputId": "3566fbde-88f2-4b0f-97fd-26ea6a948edc"
      },
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "[(2651,\n",
              "  0,\n",
              "  'Where do mitochondrial DNA deletion breakpoints tend to occur?',\n",
              "  'n this work, we performed a computational analysis of the human mitochondrial genome using the \"Pattern Finder\" G-quadruplex (G4) predictor algorithm to assess whether G4-forming sequences reside in close proximity (within 20 base pairs) to known mitochondrial DNA deletion breakpoints.',\n",
              "  1),\n",
              " (2651,\n",
              "  1,\n",
              "  'Where do mitochondrial DNA deletion breakpoints tend to occur?',\n",
              "  'Circular dichroism and UV spectral analysis demonstrated that mitochondrial G-rich sequences near deletion breakpoints prevalent in human disease form G-quadruplex DNA structures.',\n",
              "  1),\n",
              " (2651,\n",
              "  2,\n",
              "  'Where do mitochondrial DNA deletion breakpoints tend to occur?',\n",
              "  'Direct repeat sequences are not required at the breakpoints of age-associated mitochondrial DNA deletions in rhesus monkeys.',\n",
              "  1),\n",
              " (2651,\n",
              "  3,\n",
              "  'Where do mitochondrial DNA deletion breakpoints tend to occur?',\n",
              "  'Most of the analyzed deletion breakpoints showed nucleotide repeats flanking the deletions.',\n",
              "  1),\n",
              " (2651,\n",
              "  4,\n",
              "  'Where do mitochondrial DNA deletion breakpoints tend to occur?',\n",
              "  'Currently, the multiple types of GQ sequences and their association with human mtDNA stability are unknown.<br><b>RESULTS</b>: Here, we show an association between human mtDNA deletion breakpoint locations (sites where DNA ends rejoin after deletion of a section) and sequences with G-quadruplex forming potential (QFP), and establish the ability of selected sequences to form GQ in vitro.',\n",
              "  0),\n",
              " (2651,\n",
              "  5,\n",
              "  'Where do mitochondrial DNA deletion breakpoints tend to occur?',\n",
              "  'A biochemical analysis of purified recombinant human Twinkle protein (gene product of c10orf2) showed that the mitochondrial replicative helicase inefficiently unwinds well characterized intermolecular and intramolecular G-quadruplex DNA substrates, as well as a unimolecular G4 substrate derived from a mitochondrial sequence that nests a deletion breakpoint described in human renal cell carcinoma.',\n",
              "  0),\n",
              " (2651,\n",
              "  6,\n",
              "  'Where do mitochondrial DNA deletion breakpoints tend to occur?',\n",
              "  'DNA sequences proximal to human mitochondrial DNA deletion breakpoints prevalent in human disease form G-quadruplexes, a class of DNA structures inefficiently unwound by the mitochondrial replicative Twinkle helicase.',\n",
              "  1),\n",
              " (2651,\n",
              "  7,\n",
              "  'Where do mitochondrial DNA deletion breakpoints tend to occur?',\n",
              "  'Association of G-quadruplex forming sequences with human mtDNA deletion breakpoints.',\n",
              "  0),\n",
              " (1524,\n",
              "  0,\n",
              "  'What is the gold standard treatment for Iatrogenic male incontinence?',\n",
              "  'The initial treatment for SUI that persists after 12 months consists of conservative measures such as pelvic floor muscle exercises and behavioral therapy.',\n",
              "  0),\n",
              " (1524,\n",
              "  1,\n",
              "  'What is the gold standard treatment for Iatrogenic male incontinence?',\n",
              "  'Properly selected and informed patients can also be treated efficiently with minimally invasive procedures such as the implantation of a male suburethral sling, although the experience with such devices is not extensive.',\n",
              "  1),\n",
              " (1524,\n",
              "  2,\n",
              "  'What is the gold standard treatment for Iatrogenic male incontinence?',\n",
              "  'However, the implantation of artificial urinary sphincter is the gold standard therapy.',\n",
              "  1),\n",
              " (1524,\n",
              "  3,\n",
              "  'What is the gold standard treatment for Iatrogenic male incontinence?',\n",
              "  'treatments such as periurethral injection of bulking agents, artificial urinary sphincter (AUS) implantation, and sub-urethral sling positioning.',\n",
              "  1),\n",
              " (1524,\n",
              "  4,\n",
              "  'What is the gold standard treatment for Iatrogenic male incontinence?',\n",
              "  'The artificial urethral sphincter has represented, until today, the gold standard but, in the recent years, sling systems have been investigated as minimally invasive alternative options.',\n",
              "  1),\n",
              " (1524,\n",
              "  5,\n",
              "  'What is the gold standard treatment for Iatrogenic male incontinence?',\n",
              "  'Today, three different sling procedures are commonly performed: bone-anchored, readjustable, and trans-obturator slings systems.',\n",
              "  0),\n",
              " (1524,\n",
              "  6,\n",
              "  'What is the gold standard treatment for Iatrogenic male incontinence?',\n",
              "  'The bone anchored suburethral synthetic sling is a simple and attractive procedure that can produce immediate good results with low morbidity, especially when strictly selected patients are treated.',\n",
              "  0),\n",
              " (1524,\n",
              "  7,\n",
              "  'What is the gold standard treatment for Iatrogenic male incontinence?',\n",
              "  'BAUS implantation is a safe, effective, minimally invasive option for iatrogenic male incontinence due to ISD.',\n",
              "  1),\n",
              " (1524,\n",
              "  8,\n",
              "  'What is the gold standard treatment for Iatrogenic male incontinence?',\n",
              "  'It compares favourably with AUS.',\n",
              "  0),\n",
              " (45,\n",
              "  0,\n",
              "  'What is the association of estrogen replacement therapy and intracranial meningioma risk?',\n",
              "  'The meta-analyses yielded significantly increased risks for all CNS tumors, glioma and meningioma in users of estrogen-only [1.35 (1.22-1.49), 1.23 (1.06-1.42) and 1.31 (1.20-1.43), respectively] but not estrogen-progestin HT [1.09 (0.99-1.19), 0.92 (0.78-1.08) and 1.05 (0.95-1.16), respectively]; these differences were statistically significant (p<0.005 for each tumor type).',\n",
              "  0),\n",
              " (45,\n",
              "  1,\n",
              "  'What is the association of estrogen replacement therapy and intracranial meningioma risk?',\n",
              "  'There was no significant difference between glioma and meningioma risk in users of estrogen-only HT.',\n",
              "  1),\n",
              " (45,\n",
              "  2,\n",
              "  'What is the association of estrogen replacement therapy and intracranial meningioma risk?',\n",
              "  'The totality of the available evidence suggests an increased risk of all CNS tumors (and of glioma and meningioma separately) in users of estrogen-only HT.',\n",
              "  0),\n",
              " (45,\n",
              "  3,\n",
              "  'What is the association of estrogen replacement therapy and intracranial meningioma risk?',\n",
              "  'Among premenopausal women, current use of oral contraceptives was associated with an increased risk of meningiomas (OR 1.8, 95% CI 1.1-2.9), while current use of hormone replacement therapy among postmenopausal women was not associated with a significant elevation in risk (OR 1.1, 95% CI 0.74-1.67).',\n",
              "  0),\n",
              " (45,\n",
              "  4,\n",
              "  'What is the association of estrogen replacement therapy and intracranial meningioma risk?',\n",
              "  'The relationship between current use of exogenous hormones and meningioma remains unclear, limited by the small numbers of patients currently on oral hormone medications and a lack of hormone receptor data for meningioma tumors.',\n",
              "  0),\n",
              " (45,\n",
              "  5,\n",
              "  'What is the association of estrogen replacement therapy and intracranial meningioma risk?',\n",
              "  'Ever use of estradiol-only therapy was associated with an increased risk of meningioma (standardized incidence ratio = 1.29, 95% confidence interval: 1.15, 1.44).',\n",
              "  0),\n",
              " (45,\n",
              "  6,\n",
              "  'What is the association of estrogen replacement therapy and intracranial meningioma risk?',\n",
              "  'Among women who had been using estradiol-only therapy for at least 3 years, the incidence of meningioma was 1.40-fold higher (95% confidence interval: 1.18, 1.64; P<0.001) than in the background population.',\n",
              "  0),\n",
              " (45,\n",
              "  7,\n",
              "  'What is the association of estrogen replacement therapy and intracranial meningioma risk?',\n",
              "  'Estradiol-only therapy was accompanied with a slightly increased risk of meningioma.',\n",
              "  0),\n",
              " (45,\n",
              "  8,\n",
              "  'What is the association of estrogen replacement therapy and intracranial meningioma risk?',\n",
              "  'Results from several prospective, large-scale studies indicate that postmenopausal hormone therapy may increase the risk for diagnosing meningioma by 30-80%, but there is no effect in regard to glioma.',\n",
              "  0),\n",
              " (45,\n",
              "  9,\n",
              "  'What is the association of estrogen replacement therapy and intracranial meningioma risk?',\n",
              "  'A retrospective study including more than 350,000 women, about 1400 of whom had developed meningioma, showed that the risk of meningioma was about twice as high in users of postmenopausal hormone replacement therapy as in non-users.',\n",
              "  0),\n",
              " (45,\n",
              "  10,\n",
              "  'What is the association of estrogen replacement therapy and intracranial meningioma risk?',\n",
              "  'Compared with never users of HRT, the relative risks (RRs) for all incident CNS tumours, gliomas, meningiomas and acoustic neuromas in current users of HRT were 1.20 (95% CI: 1.05-1.36), 1.09 (95% CI: 0.89-1.32), 1.34 (95% CI: 1.03-1.75) and 1.58 (95% CI: 1.02-2.45), respectively, and there was no significant difference in the relative risks by tumour type (heterogeneity p = 0.2).',\n",
              "  0),\n",
              " (45,\n",
              "  11,\n",
              "  'What is the association of estrogen replacement therapy and intracranial meningioma risk?',\n",
              "  'BACKGROUND: Previous studies on association of exogenous female sex hormones and risk for meningioma have yielded conflicting results.',\n",
              "  0),\n",
              " (45,\n",
              "  12,\n",
              "  'What is the association of estrogen replacement therapy and intracranial meningioma risk?',\n",
              "  'RESULTS: Postmenopausal hormonal treatment, use of contraceptives, or fertility treatment did not influence the risk of meningioma.',\n",
              "  0),\n",
              " (45,\n",
              "  13,\n",
              "  'What is the association of estrogen replacement therapy and intracranial meningioma risk?',\n",
              "  'CONCLUSIONS: Overall, we found little indication that reproductive factors or use of exogenous sex hormones affect meningioma risk.',\n",
              "  0),\n",
              " (45,\n",
              "  14,\n",
              "  'What is the association of estrogen replacement therapy and intracranial meningioma risk?',\n",
              "  'Although not definitive, available data suggest an association between the use of hormone replacement therapy and increased meningioma risk.',\n",
              "  1),\n",
              " (45,\n",
              "  15,\n",
              "  'What is the association of estrogen replacement therapy and intracranial meningioma risk?',\n",
              "  'BACKGROUND: The role of exogenous hormone exposures in the development of meningioma is unclear, but these exposures have been proposed as one hypothesis to explain the over-abundance of such tumors in women.',\n",
              "  0),\n",
              " (45,\n",
              "  16,\n",
              "  'What is the association of estrogen replacement therapy and intracranial meningioma risk?',\n",
              "  'RESULTS: Although risk of meningioma appeared modestly elevated in past OC users (OR = 1.5, 95% CI 0.8 - 2.7), and in current users (OR = 2.5, 95% CI 0.5 - 12.6), the confidence intervals were wide.',\n",
              "  0),\n",
              " (45,\n",
              "  17,\n",
              "  'What is the association of estrogen replacement therapy and intracranial meningioma risk?',\n",
              "  'Likewise, risk of meningioma was only weakly associated with past use of HRT (OR = 0.7, 95% CI 0.4 - 1.3), and not at all with current use of HRT (OR = 1.0, 95% CI 0.5 - 2.2).',\n",
              "  0),\n",
              " (45,\n",
              "  18,\n",
              "  'What is the association of estrogen replacement therapy and intracranial meningioma risk?',\n",
              "  ' Overall, in post menopausal women, HRT use appeared to confer a non-significant protective effect, and was not associated with low or high PR expressing meningiomas.CONCLUSION: This study found little evidence of associations between meningioma and exogenous hormone exposures in women but did suggest that some hormonal exposures may influence tumor biology in those women who develop meningioma.',\n",
              "  0),\n",
              " (45,\n",
              "  19,\n",
              "  'What is the association of estrogen replacement therapy and intracranial meningioma risk?',\n",
              "  'The increased odds ratios with African Americans was retained in post-menopausal women, while the protective odds ratios for pregnancy, smoking and oral contraceptives (OCs) became stronger in pre-menopausal women.',\n",
              "  0),\n",
              " (45,\n",
              "  20,\n",
              "  'What is the association of estrogen replacement therapy and intracranial meningioma risk?',\n",
              "  'The pattern by duration and timing of use does not suggest an etiologic role for OCs or hormone replacement therapy.',\n",
              "  0),\n",
              " (45,\n",
              "  21,\n",
              "  'What is the association of estrogen replacement therapy and intracranial meningioma risk?',\n",
              "  'The use of hormone replacement therapy in symptomatic postmenopausal women either with previously treated disease or with dormant tumors is discussed, but remains controversial.',\n",
              "  0),\n",
              " (45,\n",
              "  22,\n",
              "  'What is the association of estrogen replacement therapy and intracranial meningioma risk?',\n",
              "  'Although not definitive, available data suggest an association between the use of hormone replacement therapy and increased meningioma risk.',\n",
              "  1),\n",
              " (45,\n",
              "  23,\n",
              "  'What is the association of estrogen replacement therapy and intracranial meningioma risk?',\n",
              "  'OC use was associated with increased risk of a meningioma expressing less rather than more PR (OR = 3.2, 95% CI 1.3 - 8.0).',\n",
              "  0),\n",
              " (45,\n",
              "  24,\n",
              "  'What is the association of estrogen replacement therapy and intracranial meningioma risk?',\n",
              "  'In further analysis by hormone receptor status, there was some indication for an increased risk of progesterone receptor-positive meningiomas associated with oral contraceptive use (OR 1.39, 95% confidence interval 0.92-2.10) and other hormonal contraception (OR 1.50, 95% CI 0.95-2.36).',\n",
              "  0),\n",
              " (45,\n",
              "  25,\n",
              "  'What is the association of estrogen replacement therapy and intracranial meningioma risk?',\n",
              "  'Several studies indicate that use of hormone replacement therapy (HRT) is associated with an increased risk of intracranial meningioma, while associations between HRT use and risk of other brain tumors have been less explored.',\n",
              "  0),\n",
              " (45,\n",
              "  26,\n",
              "  'What is the association of estrogen replacement therapy and intracranial meningioma risk?',\n",
              "  'available data suggest an association between the use of hormone replacement therapy and increased meningioma risk.',\n",
              "  1),\n",
              " (45,\n",
              "  27,\n",
              "  'What is the association of estrogen replacement therapy and intracranial meningioma risk?',\n",
              "  'A significant positive association existed between meningioma risk and increased body mass index (p &lt; 0.01) while a significant negative association existed between meningioma risk and current smoking (p &lt; 0.01).',\n",
              "  0),\n",
              " (45,\n",
              "  28,\n",
              "  'What is the association of estrogen replacement therapy and intracranial meningioma risk?',\n",
              "  'Among premenopausal women, current use of oral contraceptives was associated with an increased risk of meningiomas (OR 1.8, 95% CI 1.1-2.9), while current use of hormone replacement therapy among postmenopausal women was not associated with a significant elevation in risk (OR 1.1, 95% CI 0.74-1.67).',\n",
              "  0),\n",
              " (45,\n",
              "  29,\n",
              "  'What is the association of estrogen replacement therapy and intracranial meningioma risk?',\n",
              "  'A higher risk of meningioma was observed among postmenopausal women who were current users of hormone replacement therapy (HR,',\n",
              "  0),\n",
              " (45,\n",
              "  30,\n",
              "  'What is the association of estrogen replacement therapy and intracranial meningioma risk?',\n",
              "  'Among premenopausal women, current use of oral contraceptives was associated with an increased risk of meningiomas (OR 1.8, 95% CI 1.1-2.9), while current use of hormone replacement therapy among postmenopausal women was not associated with a significant elevation in risk (OR 1.1, 95% CI 0.74-1.67).',\n",
              "  0),\n",
              " (45,\n",
              "  31,\n",
              "  'What is the association of estrogen replacement therapy and intracranial meningioma risk?',\n",
              "  'There was no association between use of fertility medications and meningioma risk.',\n",
              "  0),\n",
              " (45,\n",
              "  32,\n",
              "  'What is the association of estrogen replacement therapy and intracranial meningioma risk?',\n",
              "  'Although not definitive, available data suggest an association between the use of hormone replacement therapy and increased meningioma risk.',\n",
              "  1),\n",
              " (45,\n",
              "  33,\n",
              "  'What is the association of estrogen replacement therapy and intracranial meningioma risk?',\n",
              "  'A higher risk of meningioma was observed among postmenopausal women who were current users of hormone replacement therapy (HR, 1.79; 95% CI, 1.18-2.71) compared with never users.',\n",
              "  0),\n",
              " (45,\n",
              "  34,\n",
              "  'What is the association of estrogen replacement therapy and intracranial meningioma risk?',\n",
              "  'Among premenopausal women, current use of oral contraceptives was associated with an increased risk of meningiomas (OR 1.8, 95% CI 1.1-2.9), while current use of hormone replacement therapy among postmenopausal women was not associated with a significant elevation in risk (OR 1.1, 95% CI 0.74-1.67).',\n",
              "  0),\n",
              " (3225,\n",
              "  0,\n",
              "  'What is a SMR based BCI?',\n",
              "  ' sensorimotor rhythm-based brain-computer interfaces (SMR-BCI)',\n",
              "  1),\n",
              " (3225,\n",
              "  1,\n",
              "  'What is a SMR based BCI?',\n",
              "  'Here, we studied the immediate and therapeutic effects of BCI-based training to control pre-movement sensorimotor rhythm (SMR) amplitude on robot-assisted finger extension in people with stroke.',\n",
              "  0),\n",
              " (3225,\n",
              "  2,\n",
              "  'What is a SMR based BCI?',\n",
              "  ' Especially for BCIs based on the modulation of the Sensorimotor Rhythm (SMR)',\n",
              "  1),\n",
              " (3225,\n",
              "  3,\n",
              "  'What is a SMR based BCI?',\n",
              "  'Brain Computer Interface (BCI) and neurofeedback studies have investigated the impact of sensorimotor rhythm (SMR)',\n",
              "  1),\n",
              " (3225,\n",
              "  4,\n",
              "  'What is a SMR based BCI?',\n",
              "  'Modulation of sensorimotor rhythms (SMR) was suggested as a control signal for brain-computer interfaces (BCI).',\n",
              "  0),\n",
              " (3225,\n",
              "  5,\n",
              "  'What is a SMR based BCI?',\n",
              "  'Sensorimotor rhythm (SMR) based Brain-Computer Interfaces (BCI) typically require lengthy user training.',\n",
              "  1),\n",
              " (3225,\n",
              "  6,\n",
              "  'What is a SMR based BCI?',\n",
              "  'All brain-computer interface (BCI) groups that have published results of studies involving a large number of users performing BCI control based on the voluntary modulation of sensorimotor rhythms (SMR) report that BCI control could not be achieved by a non-negligible number of subjects (estimated 20% to 25%).',\n",
              "  1),\n",
              " (3225,\n",
              "  7,\n",
              "  'What is a SMR based BCI?',\n",
              "  'OBJECTIVE\\nConnect-Four, a new sensorimotor rhythm (SMR) based brain-computer interface (BCI) gaming application, was evaluated by four severely motor restricted end-users; two were in the locked-in state and had unreliable eye-movement.',\n",
              "  0),\n",
              " (3225,\n",
              "  8,\n",
              "  'What is a SMR based BCI?',\n",
              "  'OBJECTIVE\\nIt is well known that to acquire sensorimotor (SMR)-based brain-computer interface (BCI) control requires a training period before users can achieve their best possible performances.',\n",
              "  0),\n",
              " (2563,\n",
              "  0,\n",
              "  'What is host induced gene silencing?',\n",
              "  'Host-induced gene silencing (HIGS) is a transgenic technology used to silence fungal genes in planta during attempted infection and thereby reduces disease levels.',\n",
              "  1),\n",
              " (2563,\n",
              "  1,\n",
              "  'What is host induced gene silencing?',\n",
              "  \"HIGS relies on the host plant's ability to produce mobile small interfering RNA molecules, generated from long double-stranded RNA, which are complementary to targeted fungal genes.\",\n",
              "  1),\n",
              " (2563,\n",
              "  2,\n",
              "  'What is host induced gene silencing?',\n",
              "  'These molecules are transferred from the plant to invading fungi via an uncharacterised mechanism, to cause gene silencing.',\n",
              "  1),\n",
              " (2563,\n",
              "  3,\n",
              "  'What is host induced gene silencing?',\n",
              "  'We then discuss the developments and implications of combining HIGS with other methods of disease control.',\n",
              "  1),\n",
              " (4184,\n",
              "  0,\n",
              "  'Are interferons defensive proteins?',\n",
              "  'In response to viral infections, various pattern recognition receptors (PRRs) are activated for the production of type I interferon (IFN I).',\n",
              "  1),\n",
              " (4184,\n",
              "  1,\n",
              "  'Are interferons defensive proteins?',\n",
              "  ' activating interferon (IFN) production and positively regulating antiviral response in mammals.',\n",
              "  1),\n",
              " (4184,\n",
              "  2,\n",
              "  'Are interferons defensive proteins?',\n",
              "  'The innate immune system, in particular the type I interferon (IFN) response, is a powerful defence against virus infections.',\n",
              "  1),\n",
              " (4184,\n",
              "  3,\n",
              "  'Are interferons defensive proteins?',\n",
              "  'The interferon-induced GTP-binding protein Mx is responsible for a specific antiviral state against a broad spectrum of viral infections that are induced by type-I interferons (IFN α/β) in different vertebrates',\n",
              "  1),\n",
              " (1382,\n",
              "  0,\n",
              "  'Describe the usefulness of Macrostomum lignano in ion channel and stem cell research',\n",
              "  'Bioelectrical signals generated by ion channels play crucial roles in many cellular processes in both excitable and nonexcitable cells.',\n",
              "  1),\n",
              " (1382,\n",
              "  1,\n",
              "  'Describe the usefulness of Macrostomum lignano in ion channel and stem cell research',\n",
              "  'Some ion channels are directly implemented in chemical signaling pathways, the others are involved in regulation of cytoplasmic or vesicular ion concentrations, pH, cell volume, and membrane potentials.',\n",
              "  1),\n",
              " (1382,\n",
              "  2,\n",
              "  'Describe the usefulness of Macrostomum lignano in ion channel and stem cell research',\n",
              "  'Together with ion transporters and gap junction complexes, ion channels form steady-state voltage gradients across the cell membranes in nonexcitable cells.',\n",
              "  1),\n",
              " (1382,\n",
              "  3,\n",
              "  'Describe the usefulness of Macrostomum lignano in ion channel and stem cell research',\n",
              "  'These membrane potentials are involved in regulation of such processes as migration guidance, cell proliferation, and body axis patterning during development and regeneration.',\n",
              "  1),\n",
              " (1382,\n",
              "  4,\n",
              "  'Describe the usefulness of Macrostomum lignano in ion channel and stem cell research',\n",
              "  'While the importance of membrane potential in stem cell maintenance, proliferation, and differentiation is evident, the mechanisms of this bioelectric control of stem cell activity are still not well understood, and the role of specific ion channels in these processes remains unclear.',\n",
              "  1),\n",
              " (1382,\n",
              "  5,\n",
              "  'Describe the usefulness of Macrostomum lignano in ion channel and stem cell research',\n",
              "  'Here we introduce the flatworm Macrostomum lignano as a versatile model organism for addressing these topics.',\n",
              "  0),\n",
              " (1382,\n",
              "  6,\n",
              "  'Describe the usefulness of Macrostomum lignano in ion channel and stem cell research',\n",
              "  'We discuss biological and experimental properties of M. lignano, provide an overview of the recently developed experimental tools for this animal model, and demonstrate how manipulation of membrane potential influences regeneration in M. lignano.',\n",
              "  0),\n",
              " (1309,\n",
              "  0,\n",
              "  'What is the role of the histidine rich calcium binding protein (HRC) in cardiomyopathy?',\n",
              "  'The histidine-rich Ca-binding protein (HRC) is a Ca-storage protein in cardiac sarcoplasmic reticulum.',\n",
              "  1),\n",
              " (1309,\n",
              "  1,\n",
              "  'What is the role of the histidine rich calcium binding protein (HRC) in cardiomyopathy?',\n",
              "  'Recent transgenic studies revealed that this protein inhibits the maximal rates of sarcoplasmic reticulum Ca-transport, leading to cardiac dysfunction.',\n",
              "  1),\n",
              " (1309,\n",
              "  2,\n",
              "  'What is the role of the histidine rich calcium binding protein (HRC) in cardiomyopathy?',\n",
              "  'In view of the role of sarcoplasmic reticulum Ca-cycling in myocardial ischemia/reperfusion injury, we designed this study to gain further insight into the role of HRC during ischemia/reperfusion.',\n",
              "  1),\n",
              " (1309,\n",
              "  3,\n",
              "  'What is the role of the histidine rich calcium binding protein (HRC) in cardiomyopathy?',\n",
              "  'Our findings suggest that increased cardiac HRC expression protects against ischemia/reperfusion injury in the heart, resulting in improved recovery of function and reduced infarction.',\n",
              "  0),\n",
              " (1309,\n",
              "  4,\n",
              "  'What is the role of the histidine rich calcium binding protein (HRC) in cardiomyopathy?',\n",
              "  'The histidine-rich calcium binding protein (HRC) Ser96Ala polymorphism was shown to correlate with ventricular arrhythmias and sudden death only in dilated cardiomyopathy patients',\n",
              "  1),\n",
              " (1309,\n",
              "  5,\n",
              "  'What is the role of the histidine rich calcium binding protein (HRC) in cardiomyopathy?',\n",
              "  'The HRC(S96A) mutant exacerbated the inhibitory effects of HRC(WT) on the amplitude of Ca(2+) transients, prolongation of Ca(2+) decay time, and caffeine-induced sarcoplasmic reticulum Ca(2+) release.',\n",
              "  0),\n",
              " (1309,\n",
              "  6,\n",
              "  'What is the role of the histidine rich calcium binding protein (HRC) in cardiomyopathy?',\n",
              "  'Consistent with these findings, HRC(S96A) reduced maximal sarcoplasmic reticulum calcium uptake rate to a higher extent than HRC(WT).',\n",
              "  0),\n",
              " (1309,\n",
              "  7,\n",
              "  'What is the role of the histidine rich calcium binding protein (HRC) in cardiomyopathy?',\n",
              "  'Furthermore, the frequency of spontaneous Ca(2+) sparks, which was reduced by HRC(WT), was increased by mutant HRC(S96A) under resting conditions although there were no spontaneous Ca(2+) waves under stress conditions.',\n",
              "  0),\n",
              " (1309,\n",
              "  8,\n",
              "  'What is the role of the histidine rich calcium binding protein (HRC) in cardiomyopathy?',\n",
              "  'However, expression of the HRC(S96A) genetic variant in cardiomyocytes from a rat model of postmyocardial infarction heart failure induced dramatic disturbances of rhythmic Ca(2+) transients.',\n",
              "  0),\n",
              " (1309,\n",
              "  9,\n",
              "  'What is the role of the histidine rich calcium binding protein (HRC) in cardiomyopathy?',\n",
              "  'These findings indicate that the HRC Ser96Ala variant increases the propensity of arrhythmogenic Ca(2+) waves in the stressed failing heart, suggesting a link between this genetic variant and life-threatening ventricular arrhythmias in human carriers.',\n",
              "  1),\n",
              " (1384,\n",
              "  0,\n",
              "  'Which drug is benserazide usually co-administered with?',\n",
              "  'The present study utilizes in vivo amperometry to investigate the impact from unilateral 6-hydroxydopamine lesions and l-DOPA (4 mg/kg, including benserazide 15 mg/kg) -induced dyskinetic behavior on striatal basal extracellular glutamate concentration and potassium-evoked glutamate release in urethane-anesthetized rats.',\n",
              "  0),\n",
              " (1384,\n",
              "  1,\n",
              "  'Which drug is benserazide usually co-administered with?',\n",
              "  'Co-administration of L-Dopa with peripheral DDC inhibitors (carbidopa or benserazide) is the most effective symptomatic treatment for PD.',\n",
              "  1),\n",
              " (1384,\n",
              "  2,\n",
              "  'Which drug is benserazide usually co-administered with?',\n",
              "  'In experiment 1, l-DOPA-primed rats were pre-treated with Vehicle (0.9% NaCl), various doses of the partial 5-HT(1A) agonist, buspirone (0.25, 1.0 or 2.5 mg/kg, ip) or buspirone (2.5 mg/kg, ip)+the 5-HT(1A) antagonist, WAY100635 (0.5 mg/kg, ip) 5 min prior to l-DOPA (12 mg/kg+15 mg/kg benserazide, ip).',\n",
              "  0),\n",
              " (1384, 3, 'Which drug is benserazide usually co-administered with?', 'R', 0),\n",
              " (1384,\n",
              "  4,\n",
              "  'Which drug is benserazide usually co-administered with?',\n",
              "  'L-DOPA induced hyperalgesia occurred after conversion to dopamine because co-administration of benserazide, a DOPA decarboxylase inhibitor, completely abolished the L-DOPA-induced hyperalgesia.',\n",
              "  1),\n",
              " (1384,\n",
              "  5,\n",
              "  'Which drug is benserazide usually co-administered with?',\n",
              "  'First, animals were treated with levodopa (50 mg/kg with benserazide 12.5 mg/kg, twice daily), intraperitoneally (i.p.)',\n",
              "  0),\n",
              " (1384,\n",
              "  6,\n",
              "  'Which drug is benserazide usually co-administered with?',\n",
              "  'for 22 days.',\n",
              "  0),\n",
              " (1384,\n",
              "  7,\n",
              "  'Which drug is benserazide usually co-administered with?',\n",
              "  'Chronic L-DOPA-treated rats received the D1 receptor antagonist SCH23390 (0.01, 0.1, and 1.0 mg/kg; i.p.',\n",
              "  0),\n",
              " (1384,\n",
              "  8,\n",
              "  'Which drug is benserazide usually co-administered with?',\n",
              "  '), the D2 receptor antagonist Eticlopride (0.01, 0.1, and 1.0 mg/kg; i.p.',\n",
              "  0),\n",
              " (1384,\n",
              "  9,\n",
              "  'Which drug is benserazide usually co-administered with?',\n",
              "  '), a mixture of both antagonists (0.01, 0.1, 1.0 mg/kg each; i.p.',\n",
              "  0),\n",
              " (1384,\n",
              "  10,\n",
              "  'Which drug is benserazide usually co-administered with?',\n",
              "  '), or vehicle 30 min prior to L-DOPA (6 mg/kg; i.p.',\n",
              "  0),\n",
              " (1384,\n",
              "  11,\n",
              "  'Which drug is benserazide usually co-administered with?',\n",
              "  ')+Benserazide (15 mg/kg; i.p.',\n",
              "  0),\n",
              " (1384,\n",
              "  12,\n",
              "  'Which drug is benserazide usually co-administered with?',\n",
              "  ').',\n",
              "  0),\n",
              " (1384,\n",
              "  13,\n",
              "  'Which drug is benserazide usually co-administered with?',\n",
              "  'Nociceptive behaviors in mice after an intrathecal (i.t.)',\n",
              "  0),\n",
              " (1384,\n",
              "  14,\n",
              "  'Which drug is benserazide usually co-administered with?',\n",
              "  'administration of substance P were evaluated.',\n",
              "  0),\n",
              " (1384,\n",
              "  15,\n",
              "  'Which drug is benserazide usually co-administered with?',\n",
              "  'L-DOPA (i.t.)',\n",
              "  0),\n",
              " (1384,\n",
              "  16,\n",
              "  'Which drug is benserazide usually co-administered with?',\n",
              "  'dose-dependently attenuated the substance P-induced nociceptive behaviors.',\n",
              "  0),\n",
              " (1384,\n",
              "  17,\n",
              "  'Which drug is benserazide usually co-administered with?',\n",
              "  'Co-administration of benserazide (i.t.',\n",
              "  1),\n",
              " (1384,\n",
              "  18,\n",
              "  'Which drug is benserazide usually co-administered with?',\n",
              "  '), a DOPA decarboxylase inhibitor, abolished the antinociceptive effect of L-DOPA.',\n",
              "  0),\n",
              " (1384,\n",
              "  19,\n",
              "  'Which drug is benserazide usually co-administered with?',\n",
              "  'They were injected subcutaneously (s.c.) with L-DOPA methyl ester (125 mg per animal) plus benserazide (50 mg per animal; L-DOPA/benserazide) alone or in combination with (-)-OSU6162 (1.0, 3.0, 6.0 or 10 mg/kg, s.c.)',\n",
              "  0),\n",
              " (1384,\n",
              "  20,\n",
              "  'Which drug is benserazide usually co-administered with?',\n",
              "  'They were injected subcutaneously (s.c.) with L-Dopa methyl ester (125 mg per animal) plus benserazide (50 mg per animal; L-Dopa/benserazide) alone or in combination with JL-18 (at the doses of 0.1, 0.3, or 0.9 mg/kg, s.c.).',\n",
              "  0),\n",
              " (1384,\n",
              "  21,\n",
              "  'Which drug is benserazide usually co-administered with?',\n",
              "  'The administration of L-dopa (20 and 60 mg/kg p.o.)',\n",
              "  1),\n",
              " (1384,\n",
              "  22,\n",
              "  'Which drug is benserazide usually co-administered with?',\n",
              "  '+ benserazide (15 mg/kg p.o.)',\n",
              "  0),\n",
              " (1384,\n",
              "  23,\n",
              "  'Which drug is benserazide usually co-administered with?',\n",
              "  'resulted in dose-dependent increase of dialysate levels of L-dopa and 3-O-methyl-DOPA.',\n",
              "  0),\n",
              " (1384,\n",
              "  24,\n",
              "  'Which drug is benserazide usually co-administered with?',\n",
              "  'Tolcapone (30 mg/kg p.o.',\n",
              "  0),\n",
              " (1384,\n",
              "  25,\n",
              "  'Which drug is benserazide usually co-administered with?',\n",
              "  '), given as adjunct to both doses of L-dopa, markedly enhanced the elevation or extracellular L-dopa, while it completely prevented the formation of 3-O-methyl-DOPA.',\n",
              "  0),\n",
              " (1384,\n",
              "  26,\n",
              "  'Which drug is benserazide usually co-administered with?',\n",
              "  'In another experiment, the administration of L-dopa + benserazide (30 + 15 mg/kg p.o.)',\n",
              "  1),\n",
              " (1384,\n",
              "  27,\n",
              "  'Which drug is benserazide usually co-administered with?',\n",
              "  'resulted in increased extracellular levels of dopamine, DOPAC, HVA and 3-methoxytyramine.',\n",
              "  0),\n",
              " (1384,\n",
              "  28,\n",
              "  'Which drug is benserazide usually co-administered with?',\n",
              "  'In healthy subjects and in patients with parkinsonism plasma ALAAD level fell after administration of L-dopa + benserazide, but returned to previous levels within 90 min.',\n",
              "  0),\n",
              " (1384,\n",
              "  29,\n",
              "  'Which drug is benserazide usually co-administered with?',\n",
              "  'In a cross-sectional study blood was obtained, 2 h after dosing, from 104 patients with idiopathic parkinsonism, divided into four groups: no L-dopa treatment (group 1), L-dopa alone (group 2), L-dopa + benserazide (Madopar) (group 3) and L-dopa + carbidopa (Sinemet) (group 4).',\n",
              "  0),\n",
              " (2591,\n",
              "  0,\n",
              "  'What is the proteoform?',\n",
              "  'proteoforms - splice variants, single amino acid substitutions (SAP variants), and post-translational modifications (PTM) of proteins',\n",
              "  1),\n",
              " (2591,\n",
              "  1,\n",
              "  'What is the proteoform?',\n",
              "  'Although proteomics has rapidly developed in the past decade, researchers are still in the early stage of exploring the world of complex proteoforms, which are protein products with various primary structure alterations resulting from gene mutations, alternative splicing, post-translational modifications, and other biological processes',\n",
              "  1),\n",
              " (2591,\n",
              "  2,\n",
              "  'What is the proteoform?',\n",
              "  'Protein species (aka proteoforms) function at their molecular level, and diverse structures and biological roles of every proteoform come from often co-occurring proteolysis, amino acid variation and post-translational modifications.',\n",
              "  1),\n",
              " (2591,\n",
              "  3,\n",
              "  'What is the proteoform?',\n",
              "  'the identification of novel proteoforms is also made possible based on detection of novel translation initiation sites (cognate or near-cognate), novel transcript isoforms, sequence variation or novel (small) open reading frames in intergenic or un-translated genic regions by analyzing high-throughput sequencing data from RNAseq or ribosome profiling experiments.',\n",
              "  1),\n",
              " (3270,\n",
              "  0,\n",
              "  'Which is the target of belimumab in Systemic Lupus Erythematosus treatment?',\n",
              "  'Belimumab: a BLyS-specific inhibitor for systemic lupus erythematosus',\n",
              "  1),\n",
              " (3270,\n",
              "  1,\n",
              "  'Which is the target of belimumab in Systemic Lupus Erythematosus treatment?',\n",
              "  'To review the efficacy, safety, dosing, drug interactions, as well as economic and therapeutic considerations of belimumab, an investigational B-lymphocyte stimulator (BLyS) inhibitor.',\n",
              "  1),\n",
              " (3270,\n",
              "  2,\n",
              "  'Which is the target of belimumab in Systemic Lupus Erythematosus treatment?',\n",
              "  'Belimumab, an anti-BAFF monoclonal antibody',\n",
              "  0),\n",
              " (3270,\n",
              "  3,\n",
              "  'Which is the target of belimumab in Systemic Lupus Erythematosus treatment?',\n",
              "  'Here our crystal structure of the BAFF-belimumab Fab complex shows the precise epitope and the BAFF-neutralizing mechanism of belimumab, and demonstrates that the therapeutic activity of belimumab involves not only antagonizing the BAFF-receptor interaction, but also disrupting the formation of the more active BAFF 60-mer to favor the induction of the less active BAFF trimer through interaction with the flap region of BAFF',\n",
              "  0),\n",
              " (3270,\n",
              "  4,\n",
              "  'Which is the target of belimumab in Systemic Lupus Erythematosus treatment?',\n",
              "  'In recent years, a member of the tumor necrosis factor (TNF) family, soluble human B Lymphocyte Stimulator protein (BLyS), also referred to as B-cell activating factor (BAFF) and TNFSF13B has been studied extensively.',\n",
              "  0),\n",
              " (3270,\n",
              "  5,\n",
              "  'Which is the target of belimumab in Systemic Lupus Erythematosus treatment?',\n",
              "  'This protein is synthesized by myeloid cell lines, specifically interacts with B lymphocytes and increases their life-span',\n",
              "  0),\n",
              " (3270,\n",
              "  6,\n",
              "  'Which is the target of belimumab in Systemic Lupus Erythematosus treatment?',\n",
              "  'Belimumab, a fully human monoclonal antibody against B lymphocyte stimulator (BLyS), a B-cell survival factor, was licensed in 2011 for the treatment of autoantibody-positive SLE',\n",
              "  1),\n",
              " (3270,\n",
              "  7,\n",
              "  'Which is the target of belimumab in Systemic Lupus Erythematosus treatment?',\n",
              "  'Belimumab is a fully human monoclonal antibody directed against BAFF.',\n",
              "  1),\n",
              " (3270,\n",
              "  8,\n",
              "  'Which is the target of belimumab in Systemic Lupus Erythematosus treatment?',\n",
              "  ' Belimumab, a human monoclonal antibody specific for soluble BLyS, was ultimately approved by the United States Food and Drug Administration (FDA) in March 2011 for active autoantibody patients with systemic lupus erythematosus (SLE) despite standard therapy.',\n",
              "  1),\n",
              " (3270,\n",
              "  9,\n",
              "  'Which is the target of belimumab in Systemic Lupus Erythematosus treatment?',\n",
              "  'Targeted therapy with belimumab, the monoclonal antibody against BLyS, has shown clinical benefit in two large-scale, multicenter phase III trials leading to US Food and Drug Administration approval for patients with serologically positive SLE who have active disease despite standard therapy',\n",
              "  0),\n",
              " (222,\n",
              "  0,\n",
              "  'Is there any association between Jarid2 and miR-155 in Th17 cells?',\n",
              "  'Jarid2 links MicroRNA and chromatin in Th17 cells.',\n",
              "  0),\n",
              " (222,\n",
              "  1,\n",
              "  'Is there any association between Jarid2 and miR-155 in Th17 cells?',\n",
              "  'In this issue of Immunity, Escobar et\\xa0al.',\n",
              "  0),\n",
              " (222,\n",
              "  2,\n",
              "  'Is there any association between Jarid2 and miR-155 in Th17 cells?',\n",
              "  '(2014) bring microRNAs and chromatin together by showing how activation-induced miR-155 targets the chromatin protein Jarid2 to regulate proinflammatory cytokine production in T helper 17 cells.',\n",
              "  1),\n",
              " (222,\n",
              "  3,\n",
              "  'Is there any association between Jarid2 and miR-155 in Th17 cells?',\n",
              "  'miR-155 activates cytokine gene expression in Th17 cells by regulating the DNA-binding protein Jarid2 to relieve polycomb-mediated repression.',\n",
              "  1),\n",
              " (222,\n",
              "  4,\n",
              "  'Is there any association between Jarid2 and miR-155 in Th17 cells?',\n",
              "  'Mir155 was bound by Th17 cell transcription factors and was highly expressed during Th17 cell differentiation.',\n",
              "  0),\n",
              " (222,\n",
              "  5,\n",
              "  'Is there any association between Jarid2 and miR-155 in Th17 cells?',\n",
              "  'miR-155-deficient Th17 and T regulatory (Treg) cells expressed increased amounts of Jarid2, a DNA-binding protein that recruits the Polycomb Repressive Complex 2 (PRC2) to chromatin.',\n",
              "  1),\n",
              " (222,\n",
              "  6,\n",
              "  'Is there any association between Jarid2 and miR-155 in Th17 cells?',\n",
              "  'PRC2 binding to chromatin and H3K27 histone methylation was increased in miR-155-deficient cells, coinciding with failure to express Il22, Il10, Il9, and Atf3.',\n",
              "  0),\n",
              " (222,\n",
              "  7,\n",
              "  'Is there any association between Jarid2 and miR-155 in Th17 cells?',\n",
              "  'Defects in Th17 cell cytokine expression and Treg cell homeostasis in the absence of Mir155 could be partially suppressed by Jarid2 deletion.',\n",
              "  0),\n",
              " (222,\n",
              "  8,\n",
              "  'Is there any association between Jarid2 and miR-155 in Th17 cells?',\n",
              "  'Thus, miR-155 contributes to Th17 cell function by suppressing the inhibitory effects of Jarid2.',\n",
              "  1),\n",
              " (222,\n",
              "  9,\n",
              "  'Is there any association between Jarid2 and miR-155 in Th17 cells?',\n",
              "  'Thus, miR-155 contributes to Th17 cell function by suppressing the inhibitory effects of Jarid2.',\n",
              "  1),\n",
              " (222,\n",
              "  10,\n",
              "  'Is there any association between Jarid2 and miR-155 in Th17 cells?',\n",
              "  'Thus, miR-155 contributes to Th17 cell function by suppressing the inhibitory effects of Jarid2.',\n",
              "  0),\n",
              " (222,\n",
              "  11,\n",
              "  'Is there any association between Jarid2 and miR-155 in Th17 cells?',\n",
              "  'Defects in Th17 cell cytokine expression and Treg cell homeostasis in the absence of Mir155 could be partially suppressed by Jarid2 deletion.',\n",
              "  0),\n",
              " (222,\n",
              "  12,\n",
              "  'Is there any association between Jarid2 and miR-155 in Th17 cells?',\n",
              "  'Thus, miR-155 contributes to Th17 cell function by suppressing the inhibitory effects of Jarid2.',\n",
              "  0),\n",
              " (222,\n",
              "  13,\n",
              "  'Is there any association between Jarid2 and miR-155 in Th17 cells?',\n",
              "  'Thus, miR-155 contributes to Th17 cell function by suppressing the inhibitory effects of Jarid2.',\n",
              "  0),\n",
              " (4012,\n",
              "  0,\n",
              "  'Which subcortical brain structure is influenced the most by common genetic variants?',\n",
              "  'The highly complex structure of the human brain is strongly shaped by genetic influences.',\n",
              "  1),\n",
              " (4012,\n",
              "  1,\n",
              "  'Which subcortical brain structure is influenced the most by common genetic variants?',\n",
              "  'Subcortical brain regions form circuits with cortical areas to coordinate movement, learning, memory and motivation, and altered circuits can lead to abnormal behaviour and disease.',\n",
              "  1),\n",
              " (4012,\n",
              "  2,\n",
              "  'Which subcortical brain structure is influenced the most by common genetic variants?',\n",
              "  'To investigate how common genetic variants affect the structure of these brain regions, here we conduct genome-wide association studies of the volumes of seven subcortical regions and the intracranial volume derived from magnetic resonance images of 30,717 individuals from 50 cohorts.',\n",
              "  1),\n",
              " (4012,\n",
              "  3,\n",
              "  'Which subcortical brain structure is influenced the most by common genetic variants?',\n",
              "  'We identify five novel genetic variants influencing the volumes of the putamen and caudate nucleus.',\n",
              "  1),\n",
              " (4012,\n",
              "  4,\n",
              "  'Which subcortical brain structure is influenced the most by common genetic variants?',\n",
              "  'We also find stronger evidence for three loci with previously established influences on hippocampal volume and intracranial volume.',\n",
              "  0),\n",
              " (4012,\n",
              "  5,\n",
              "  'Which subcortical brain structure is influenced the most by common genetic variants?',\n",
              "  'These variants show specific volumetric effects on brain structures rather than global effects across structures.',\n",
              "  0),\n",
              " (4012,\n",
              "  6,\n",
              "  'Which subcortical brain structure is influenced the most by common genetic variants?',\n",
              "  'The strongest effects were found for the putamen, where a novel intergenic locus with replicable influence on volume (rs945270; P = 1.08\\u2009×\\u200910(-33); 0.52% variance explained) showed evidence of altering the expression of the KTN1 gene in both brain and blood tissue.',\n",
              "  1),\n",
              " (4012,\n",
              "  7,\n",
              "  'Which subcortical brain structure is influenced the most by common genetic variants?',\n",
              "  'Variants influencing putamen volume clustered near developmental genes that regulate apoptosis, axon guidance and vesicle transport.',\n",
              "  0),\n",
              " (4012,\n",
              "  8,\n",
              "  'Which subcortical brain structure is influenced the most by common genetic variants?',\n",
              "  'Identification of these genetic variants provides insight into the causes of variability in human brain development, and may help to determine mechanisms of neuropsychiatric dysfunction.',\n",
              "  0),\n",
              " (566,\n",
              "  0,\n",
              "  'Oxantel is used for periodontitis treatment. How does it work?',\n",
              "  'Oxantel disrupts polymicrobial biofilm development of periodontal pathogens',\n",
              "  1),\n",
              " (566,\n",
              "  1,\n",
              "  'Oxantel is used for periodontitis treatment. How does it work?',\n",
              "  'he anthelmintic drug oxantel has been shown to inhibit fumarate reductase (Frd) activity in some pathogenic bacteria and inhibit P. gingivalis homotypic biofilm formation.',\n",
              "  1),\n",
              " (566,\n",
              "  2,\n",
              "  'Oxantel is used for periodontitis treatment. How does it work?',\n",
              "  'Oxantel, a cholinergic anthelmintic and fumarate reductase inhibitor, significantly inhibited biofilm formation by P. gingivalis and disrupted established biofilms at concentrations below its MIC against planktonic cells.',\n",
              "  1),\n",
              " (1389,\n",
              "  0,\n",
              "  'Do A-type lamins bind euchromatin or heterochromatin?',\n",
              "  'These data reveal that the domain encoded by exon 9 is important to maintain telomere homeostasis and heterochromatin structure but does not play a role in DNA repair, thus pointing to other exons in the lamin A tail as responsible for the genomic instability phenotype in Lmna(Δ8-11/Δ8-11) mice',\n",
              "  1),\n",
              " (1389,\n",
              "  1,\n",
              "  'Do A-type lamins bind euchromatin or heterochromatin?',\n",
              "  'Lmna(Δ9/Δ9) MEFs exhibit telomere shortening and heterochromatin alterations but do not activate cathepsin L-mediated degradation of 53BP1 and maintain expression of BRCA1 and RAD51',\n",
              "  1),\n",
              " (1389,\n",
              "  2,\n",
              "  'Do A-type lamins bind euchromatin or heterochromatin?',\n",
              "  'Lamin A Δexon9 mutation leads to telomere and chromatin defects but not genomic instability',\n",
              "  0),\n",
              " (1389,\n",
              "  3,\n",
              "  'Do A-type lamins bind euchromatin or heterochromatin?',\n",
              "  'Caspase-6 gene disruption reveals a requirement for lamin A cleavage in apoptotic chromatin condensation',\n",
              "  0),\n",
              " (1389,\n",
              "  4,\n",
              "  'Do A-type lamins bind euchromatin or heterochromatin?',\n",
              "  'The A- and B-type nuclear lamin networks: microdomains involved in chromatin organization and transcription.',\n",
              "  0),\n",
              " (1389,\n",
              "  5,\n",
              "  'Do A-type lamins bind euchromatin or heterochromatin?',\n",
              "  'Comparative genomic hybridization (CGH) analyses of microdissected blebs, fluorescence in situ hybridization (FISH), and immunofluorescence localization of modified histones demonstrate that gene-rich euchromatin associates with the LA/C blebs',\n",
              "  1),\n",
              " (1389,\n",
              "  6,\n",
              "  'Do A-type lamins bind euchromatin or heterochromatin?',\n",
              "  'Lamin A/C, caspase-6, and chromatin configuration during meiosis resumption in the mouse oocyte',\n",
              "  0),\n",
              " (1389,\n",
              "  7,\n",
              "  'Do A-type lamins bind euchromatin or heterochromatin?',\n",
              "  'Our results demonstrated that these proteins were always present and that their distributions were related to oocyte maturity, determined by chromatin configuration and oocyte diameter',\n",
              "  0),\n",
              " (1389,\n",
              "  8,\n",
              "  'Do A-type lamins bind euchromatin or heterochromatin?',\n",
              "  'Histone acetylation controls, almost exclusively, euchromatin protein dynamics; lamin A expression regulates heterochromatin protein dynamics, and G9a regulates both euchromatin and heterochromatin protein dynamics.',\n",
              "  0),\n",
              " (1389,\n",
              "  9,\n",
              "  'Do A-type lamins bind euchromatin or heterochromatin?',\n",
              "  'Histone acetylation controls, almost exclusively, euchromatin protein dynamics; lamin A expression regulates heterochromatin protein dynamics, and G9a regulates both euchromatin and heterochromatin protein dynamics.',\n",
              "  0),\n",
              " (1389,\n",
              "  10,\n",
              "  'Do A-type lamins bind euchromatin or heterochromatin?',\n",
              "  'Other proteins that reversibly interact with DNA, such as the lamins and nuclear pores, may have a role in the organization of DNA into transcribable euchromatin and nontranscribable heterochromatin.',\n",
              "  1),\n",
              " (1389,\n",
              "  11,\n",
              "  'Do A-type lamins bind euchromatin or heterochromatin?',\n",
              "  'Histone acetylation controls, almost exclusively, euchromatin protein dynamics; lamin A expression regulates heterochromatin protein dynamics, and G9a regulates both euchromatin and heterochromatin protein dynamics.',\n",
              "  0),\n",
              " (1389,\n",
              "  12,\n",
              "  'Do A-type lamins bind euchromatin or heterochromatin?',\n",
              "  'Other proteins that reversibly interact with DNA, such as the lamins and nuclear pores, may have a role in the organization of DNA into transcribable euchromatin and nontranscribable heterochromatin.',\n",
              "  1),\n",
              " (1389,\n",
              "  13,\n",
              "  'Do A-type lamins bind euchromatin or heterochromatin?',\n",
              "  'Role for A-type lamins in herpesviral DNA targeting and heterochromatin modulation.',\n",
              "  0),\n",
              " (409,\n",
              "  0,\n",
              "  'Does a linker histone exist in the yeast genome?',\n",
              "  'Hho1p is a bona fide linker histone',\n",
              "  0),\n",
              " (409,\n",
              "  1,\n",
              "  'Does a linker histone exist in the yeast genome?',\n",
              "  'In Saccharomyces cerevisiae, HHO1 encodes a putative linker histone with very significant homology to histone H1',\n",
              "  1),\n",
              " (409,\n",
              "  2,\n",
              "  'Does a linker histone exist in the yeast genome?',\n",
              "  'HHO1p may play a similar role to linker histones, but at restricted locations in the chromatin',\n",
              "  1),\n",
              " (409,\n",
              "  3,\n",
              "  'Does a linker histone exist in the yeast genome?',\n",
              "  'The putative linker histone in Saccharomyces cerevisiae, Hho1p, has two regions of sequence (GI and GII) that are homologous to the single globular domains of linker histones H1 and H5 in higher eukaryotes.',\n",
              "  1),\n",
              " (409,\n",
              "  4,\n",
              "  'Does a linker histone exist in the yeast genome?',\n",
              "  'The Saccharomyces cerevisiae homologue of the linker histone H1, Hho1p, has two domains that are similar in sequence to the globular domain of H1 (and variants such as H5)',\n",
              "  1),\n",
              " (409,\n",
              "  5,\n",
              "  'Does a linker histone exist in the yeast genome?',\n",
              "  'Two homologous domains of similar structure but different stability in the yeast linker histone, Hho1p',\n",
              "  0),\n",
              " (409,\n",
              "  6,\n",
              "  'Does a linker histone exist in the yeast genome?',\n",
              "  'Saccharomyces cerevisiae encodes a single linker histone, Hho1p, with two globular domains.',\n",
              "  0),\n",
              " (409,\n",
              "  7,\n",
              "  'Does a linker histone exist in the yeast genome?',\n",
              "  'The Saccharomyces cerevisiae linker histone Hho1p, with two globular domains, can simultaneously bind to two four-way junction DNA molecules',\n",
              "  0),\n",
              " (409,\n",
              "  8,\n",
              "  'Does a linker histone exist in the yeast genome?',\n",
              "  'Here, we show in yeast, that the presence of yeast linker histone Hho1p represses expression of a pol II transcribed gene (MET15) embedded in the rDNA.',\n",
              "  0),\n",
              " (409,\n",
              "  9,\n",
              "  'Does a linker histone exist in the yeast genome?',\n",
              "  'Yeast linker histone Hho1p is required for efficient RNA polymerase I processivity and transcriptional silencing at the ribosomal DNA',\n",
              "  0),\n",
              " (409,\n",
              "  10,\n",
              "  'Does a linker histone exist in the yeast genome?',\n",
              "  'Saccharomyces cerevisiae linker histone Hho1p is not essential for cell viability, and very little is known about its function in vivo.',\n",
              "  0),\n",
              " (409,\n",
              "  11,\n",
              "  'Does a linker histone exist in the yeast genome?',\n",
              "  'Saccharomyces cerevisiae linker histone Hho1p functionally interacts with core histone H4 and negatively regulates the establishment of transcriptionally silent chromatin',\n",
              "  0),\n",
              " (409,\n",
              "  12,\n",
              "  'Does a linker histone exist in the yeast genome?',\n",
              "  ' Unlike canonical linker histones in higher eukaryotes that have a single conserved globular domain, Hho1p possesses two globular domains.',\n",
              "  0),\n",
              " (409,\n",
              "  13,\n",
              "  'Does a linker histone exist in the yeast genome?',\n",
              "  'We show that the carboxyl-terminal globular domain of Hho1p is dispensable for its function, suggesting that the mode of Hho1p action is similar to that of canonical linker histones',\n",
              "  0),\n",
              " (409,\n",
              "  14,\n",
              "  'Does a linker histone exist in the yeast genome?',\n",
              "  'To identify new proteins involved in spore nuclear organization, we purified chromatin from mature spores and discovered a significant enrichment of the linker histone (Hho1)',\n",
              "  0),\n",
              " (409,\n",
              "  15,\n",
              "  'Does a linker histone exist in the yeast genome?',\n",
              "  'Hho1 chromatin immunoprecipitation followed by sequencing (ChIP-seq) revealed increased genome-wide binding in mature spores and provides novel in vivo evidence of the linker histone binding to nucleosomal linker DNA',\n",
              "  0),\n",
              " (409,\n",
              "  16,\n",
              "  'Does a linker histone exist in the yeast genome?',\n",
              "  'One of the peculiarities of S. cerevisiae cells is the unusual and less abundant linker histone, Hho1p.',\n",
              "  0),\n",
              " (409,\n",
              "  17,\n",
              "  'Does a linker histone exist in the yeast genome?',\n",
              "  'Hho1p, the linker histone of Saccharomyces cerevisiae, is important for the proper chromatin organization in vivo',\n",
              "  0),\n",
              " (409,\n",
              "  18,\n",
              "  'Does a linker histone exist in the yeast genome?',\n",
              "  'Characteristically, linker histone depleted chromatin generally exhibited longer chromatin loops than the wild-type.',\n",
              "  1),\n",
              " (409,\n",
              "  19,\n",
              "  'Does a linker histone exist in the yeast genome?',\n",
              "  'Saccharomyces cerevisiae linker histone-Hho1p maintains chromatin loop organization during ageing.',\n",
              "  0),\n",
              " (409,\n",
              "  20,\n",
              "  'Does a linker histone exist in the yeast genome?',\n",
              "  'Database homology searching against the complete yeast genome has identified a gene, HHO1, (or YPL127C, formerly LPI17) which encodes a protein that has two regions that show similarity to the pea histone H1 globular domain.',\n",
              "  0),\n",
              " (409,\n",
              "  21,\n",
              "  'Does a linker histone exist in the yeast genome?',\n",
              "  'Database homology searching against the complete yeast genome has identified a gene, HHO1, (or YPL127C, formerly LPI17) which encodes a protein that has two regions that show similarity to the pea histone H1 globular domain.',\n",
              "  0),\n",
              " (409,\n",
              "  22,\n",
              "  'Does a linker histone exist in the yeast genome?',\n",
              "  'Biochemical studies to date have not been able to identify the linker histone H1 protein in the budding yeast Saccharomyces cerevisiae.',\n",
              "  0),\n",
              " (409,\n",
              "  23,\n",
              "  'Does a linker histone exist in the yeast genome?',\n",
              "  'Database homology searching against the complete yeast genome has identified a gene, HHO1, (or YPL127C, formerly LPI17) which encodes a protein that has two regions that show similarity to the pea histone H1 globular domain.',\n",
              "  0),\n",
              " (409,\n",
              "  24,\n",
              "  'Does a linker histone exist in the yeast genome?',\n",
              "  'Database homology searching against the complete yeast genome has identified a gene, HHO1, (or YPL127C, formerly LPI17) which encodes a protein that has two regions that show similarity to the pea histone H1 globular domain.',\n",
              "  0),\n",
              " (1938,\n",
              "  0,\n",
              "  'What are the side effects during statins administration in patients with atherosclerosis?',\n",
              "  'A history of side effects was reported by 40 (19.5%) of the patients, and mainly consisted of muscle complaints and gastrointestinal symptoms.',\n",
              "  1),\n",
              " (1938,\n",
              "  1,\n",
              "  'What are the side effects during statins administration in patients with atherosclerosis?',\n",
              "  'Treatment Options for Statin-Associated Muscle Symptoms.',\n",
              "  0),\n",
              " (1938,\n",
              "  2,\n",
              "  'What are the side effects during statins administration in patients with atherosclerosis?',\n",
              "  'Muscle symptoms are a clinically relevant side effect of statin treatment.',\n",
              "  0),\n",
              " (1938,\n",
              "  3,\n",
              "  'What are the side effects during statins administration in patients with atherosclerosis?',\n",
              "  'At least 5% of patients taking statins have statin-associated muscle symptoms (SAMS).',\n",
              "  1),\n",
              " (1938,\n",
              "  4,\n",
              "  'What are the side effects during statins administration in patients with atherosclerosis?',\n",
              "  'Muscle problems and other adverse symptoms associated with statin use are frequent reasons for non-adherence and discontinuation of statin therapy, which results in inadequate control of hyperlipidemia and increased cardiovascular risk.',\n",
              "  0),\n",
              " (1938,\n",
              "  5,\n",
              "  'What are the side effects during statins administration in patients with atherosclerosis?',\n",
              "  'It was estimated that rosuvastatin-associated absolute hazards of muscle-, liver- and renal-related adverse effects are lower than the corresponding vascular benefits in moderate vascular risk individuals.',\n",
              "  0),\n",
              " (1938,\n",
              "  6,\n",
              "  'What are the side effects during statins administration in patients with atherosclerosis?',\n",
              "  'Rosuvastatin-induced proteinuria appears to be of tubular origin, not relating to kidney injury.',\n",
              "  0),\n",
              " (1938,\n",
              "  7,\n",
              "  'What are the side effects during statins administration in patients with atherosclerosis?',\n",
              "  'Rosuvastatin increases the risk of new-onset diabetes by dose-dependently impairing insulin sensitivity.',\n",
              "  0),\n",
              " (1938,\n",
              "  8,\n",
              "  'What are the side effects during statins administration in patients with atherosclerosis?',\n",
              "  'Statins are associated with myopathy, transaminase elevations, and an increased risk of incident diabetes mellitus among some patients; connections between statins and other processes, such as renal and neurologic function, have also been studied with mixed results.',\n",
              "  0),\n",
              " (1938,\n",
              "  9,\n",
              "  'What are the side effects during statins administration in patients with atherosclerosis?',\n",
              "  'Muscle symptoms are a clinically relevant side effect of statin treatment.This review is based on pertinent publications retrieved by a selective literature search, and on the current recommendations of the European Atherosclerosis Society.At least 5% of patients taking statins have statin-associated muscle symptoms (SAMS).',\n",
              "  1),\n",
              " (1938,\n",
              "  10,\n",
              "  'What are the side effects during statins administration in patients with atherosclerosis?',\n",
              "  'Nevertheless, patients may develop side effects such as muscle pain and weakness years after starting statin therapy; however, the absolute number of patients affected by statin myopathy increases with treatment duration.',\n",
              "  1),\n",
              " (1938,\n",
              "  11,\n",
              "  'What are the side effects during statins administration in patients with atherosclerosis?',\n",
              "  \"Better understanding of the molecular mechanisms involved in statin-induced myopathy may help identify patient groups susceptible to statins' side effects, thereby increasing their safety..\",\n",
              "  1),\n",
              " (2999,\n",
              "  0,\n",
              "  'What is the percentage of individuals at risk of dominant medically actionable disease?',\n",
              "  '1 in 38 individuals at risk of a dominant medically actionable disease.',\n",
              "  1),\n",
              " (2999,\n",
              "  1,\n",
              "  'What is the percentage of individuals at risk of dominant medically actionable disease?',\n",
              "  'Our study shows that 1 in 38 healthy individuals (2.7%) has a (likely) pathogenic variant in one of 59 medically actionable dominant disease genes for which the American College of Medical Genetics and Genomics (ACMG) recommends disclosure.',\n",
              "  1),\n",
              " (2999,\n",
              "  2,\n",
              "  'What is the percentage of individuals at risk of dominant medically actionable disease?',\n",
              "  'Our study shows that 1 in 38 healthy individuals (2.7%) has a (likely) pathogenic variant in one of 59 medically actionable dominant disease genes for which the American College of Medical Genetics and Genomics (ACMG) recommends disclosure.',\n",
              "  1),\n",
              " (2999,\n",
              "  3,\n",
              "  'What is the percentage of individuals at risk of dominant medically actionable disease?',\n",
              "  '1 in 38 individuals at risk of a dominant medically actionable disease.Clinical genomic sequencing can identify pathogenic variants unrelated to the initial clinical question, but of medical relevance to the patients and their families.',\n",
              "  1),\n",
              " (1002,\n",
              "  0,\n",
              "  'The small molecule SEA0400 is an inhibitor of which ion antiporter/exchanger?',\n",
              "  'The plasma membrane Na(+)/Ca(2+) exchanger (NCX) is a bidirectional ion transporter that couples the translocation of Na(+) in one direction with that of Ca(2+) in the opposite direction.',\n",
              "  0),\n",
              " (1002,\n",
              "  1,\n",
              "  'The small molecule SEA0400 is an inhibitor of which ion antiporter/exchanger?',\n",
              "  'This system contributes to the regulation of intracellular Ca(2+) concentration via the forward mode (Ca(2+) efflux) or the reverse mode (Ca(2+) influx).',\n",
              "  0),\n",
              " (1002,\n",
              "  2,\n",
              "  'The small molecule SEA0400 is an inhibitor of which ion antiporter/exchanger?',\n",
              "  'Concerning the role of NCX in NO cytotoxicity, we have found, using the specific inhibitor of NCX 2-[4-[(2,5-difluorophenyl)methoxy]phenoxy]-5-ethoxyaniline (SEA0400), that NCX is involved in NO-induced cytotoxicity in cultured microglia, astrocytes, and neuronal cells.',\n",
              "  0),\n",
              " (1002,\n",
              "  3,\n",
              "  'The small molecule SEA0400 is an inhibitor of which ion antiporter/exchanger?',\n",
              "  'The Na(+)/Ca(2+)exchanger (NCX) principal function is taking 1 Ca(2+) out of the cytoplasm and introducing 3 Na(+).',\n",
              "  0),\n",
              " (1002,\n",
              "  4,\n",
              "  'The small molecule SEA0400 is an inhibitor of which ion antiporter/exchanger?',\n",
              "  'The increase of cytoplasmic Na(+) concentration induces the NCX reverse mode (NCX(REV)), favoring Ca(2+) influx.',\n",
              "  0),\n",
              " (1002,\n",
              "  5,\n",
              "  'The small molecule SEA0400 is an inhibitor of which ion antiporter/exchanger?',\n",
              "  'NCX(REV) can be inhibited by: KB-R7943 a non-specific compound that blocks voltage-dependent and store-operated Ca(2+) channels; SEA0400 that appears to be selective for NCX(REV), but difficult to obtain and SN-6, which efficacy has been shown only in cardiomyocytes.',\n",
              "  0),\n",
              " (1002,\n",
              "  6,\n",
              "  'The small molecule SEA0400 is an inhibitor of which ion antiporter/exchanger?',\n",
              "  'SEA0400 (1 μM), a pharmacological inhibitor of NCX, significantly shortened the MAP duration (P < .01) and reduced dispersion (P < .05).',\n",
              "  0),\n",
              " (1002,\n",
              "  7,\n",
              "  'The small molecule SEA0400 is an inhibitor of which ion antiporter/exchanger?',\n",
              "  'We have recently shown that the Na(+)/Ca(2+) exchanger (NCX) is involved in nitric oxide (NO)-induced cytotoxicity in cultured astrocytes and neurons.',\n",
              "  0),\n",
              " (1002,\n",
              "  8,\n",
              "  'The small molecule SEA0400 is an inhibitor of which ion antiporter/exchanger?',\n",
              "  'However, there is no in vivo evidence suggesting the role of NCX in neurodegenerative disorders associated with NO.',\n",
              "  0),\n",
              " (1002,\n",
              "  9,\n",
              "  'The small molecule SEA0400 is an inhibitor of which ion antiporter/exchanger?',\n",
              "  \"NO is implicated in the pathogenesis of neurodegenerative disorders such as Parkinson's disease.\",\n",
              "  0),\n",
              " (1002,\n",
              "  10,\n",
              "  'The small molecule SEA0400 is an inhibitor of which ion antiporter/exchanger?',\n",
              "  \"This study examined the effect of SEA0400, the specific NCX inhibitor, on 1-methyl-4-phenyl-1,2,3,6-tetrahydropyridine (MPTP)-induced dopaminergic neurotoxicity, a model of Parkinson's disease, in C57BL/6J mice.\",\n",
              "  0),\n",
              " (1002,\n",
              "  11,\n",
              "  'The small molecule SEA0400 is an inhibitor of which ion antiporter/exchanger?',\n",
              "  'The Na(+)/Ca(2+) exchanger (NCX) plays a role in the regulation of intracellular Ca(2+) levels, and nitric oxide (NO) is involved in many pathological conditions including neurodegenerative disorders.',\n",
              "  0),\n",
              " (1002,\n",
              "  12,\n",
              "  'The small molecule SEA0400 is an inhibitor of which ion antiporter/exchanger?',\n",
              "  'We have previously found that sodium nitroprusside (SNP), an NO donor, causes apoptotic-like cell death in cultured glial cells via NCX-mediated pathways and the mechanism for NO-induced cytotoxicity is cell type-dependent.',\n",
              "  0),\n",
              " (1002,\n",
              "  13,\n",
              "  'The small molecule SEA0400 is an inhibitor of which ion antiporter/exchanger?',\n",
              "  'The present study examined using the specific NCX inhibitor 2-[4-[(2,5-difluorophenyl)methoxy]phenoxy]-5-ethoxyaniline (SEA0400) whether NCX is involved in NO-induced injury in cultured neuronal cells.',\n",
              "  0),\n",
              " (1002,\n",
              "  14,\n",
              "  'The small molecule SEA0400 is an inhibitor of which ion antiporter/exchanger?',\n",
              "  'In view of the previous observation that NO stimulates the activity of the Na(+)/Ca(2+) exchanger (NCX), this study examines the involvement of NCX in cytotoxicity.',\n",
              "  0),\n",
              " (1002,\n",
              "  15,\n",
              "  'The small molecule SEA0400 is an inhibitor of which ion antiporter/exchanger?',\n",
              "  'The specific NCX inhibitor SEA0400 blocked SNP-induced phosphorylation of ERK, JNK and p38 MAPK, and decrease in cell viability.',\n",
              "  0),\n",
              " (1002,\n",
              "  16,\n",
              "  'The small molecule SEA0400 is an inhibitor of which ion antiporter/exchanger?',\n",
              "  'The sodium-calcium exchanger (NCX) is one of the transporters contributing to the control of intracellular calcium (Ca(2+)) concentration by normally mediating net Ca(2+) efflux.',\n",
              "  0),\n",
              " (1002,\n",
              "  17,\n",
              "  'The small molecule SEA0400 is an inhibitor of which ion antiporter/exchanger?',\n",
              "  'However, the reverse mode of the NCX can cause intracellular Ca(2+) concentration overload, which exacerbates the myocardial tissue injury resulting from ischemia.',\n",
              "  0),\n",
              " (1002,\n",
              "  18,\n",
              "  'The small molecule SEA0400 is an inhibitor of which ion antiporter/exchanger?',\n",
              "  'Although the NCX inhibitor SEA0400 has been shown to therapeutically reduce myocardial injury, no in vivo technique exists to monitor intracellular Ca(2+) fluctuations produced by this drug.',\n",
              "  0),\n",
              " (1002,\n",
              "  19,\n",
              "  'The small molecule SEA0400 is an inhibitor of which ion antiporter/exchanger?',\n",
              "  'We examined the involvement of the Na(+)/Ca(2+) exchanger in the automaticity of the pulmonary vein myocardium with a specific inhibitor, SEA0400.',\n",
              "  0),\n",
              " (1002,\n",
              "  20,\n",
              "  'The small molecule SEA0400 is an inhibitor of which ion antiporter/exchanger?',\n",
              "  'We investigated the expression of Na(+)/Ca(2+) exchanger (NCX) and the functional role of NCX in retinal damage by using NCX1-heterozygous deficient mice (NCX1(+/-)) and SEA0400 (2-[4-[(2,5-difluorophenyl)methoxy] phenoxy]-5-ethoxyaniline), a selective NCX inhibitor in vivo.',\n",
              "  1),\n",
              " (1002,\n",
              "  21,\n",
              "  'The small molecule SEA0400 is an inhibitor of which ion antiporter/exchanger?',\n",
              "  'SEA0400 is a selective inhibitor of the Na(+)/Ca(2+) exchanger having equal potencies to suppress both the forward and reverse mode operation of the Na(+)/Ca(2+) exchanger.',\n",
              "  1),\n",
              " (1002,\n",
              "  22,\n",
              "  'The small molecule SEA0400 is an inhibitor of which ion antiporter/exchanger?',\n",
              "  'Involvement of the Na+/Ca2+ exchanger in ouabain-induced inotropy and arrhythmogenesis was examined with a specific inhibitor, SEA0400.',\n",
              "  0),\n",
              " (1002,\n",
              "  23,\n",
              "  'The small molecule SEA0400 is an inhibitor of which ion antiporter/exchanger?',\n",
              "  'In right ventricular papillary muscle isolated from guinea-pig ventricle, 1 microM SEA0400, which specifically inhibits the Na+/Ca2+ exchanger by 80%, reduced the ouabain (1 microM)-induced positive inotropy by 40%, but had no effect on the inotropy induced by 100 microM isobutyl methylxantine.',\n",
              "  0),\n",
              " (1002,\n",
              "  24,\n",
              "  'The small molecule SEA0400 is an inhibitor of which ion antiporter/exchanger?',\n",
              "  'The effects of SEA0400, a selective inhibitor of the Na(+)/Ca(2+) exchanger (NCX), on Na(+)-dependent Ca(2+) uptake and catecholamine (CA) release were examined in bovine adrenal chromaffin cells that were loaded with Na(+) by treatment with ouabain and veratridine.',\n",
              "  1),\n",
              " (1002,\n",
              "  25,\n",
              "  'The small molecule SEA0400 is an inhibitor of which ion antiporter/exchanger?',\n",
              "  'SEA0400 inhibited Na(+)-dependent (45)Ca(2+) uptake and CA release, with the IC(50) values of 40 and 100 nM, respectively.',\n",
              "  1),\n",
              " (1002,\n",
              "  26,\n",
              "  'The small molecule SEA0400 is an inhibitor of which ion antiporter/exchanger?',\n",
              "  'Given the potential clinical benefit of inhibiting Na+/Ca2+ exchanger (NCX) activity during myocardial ischemia reperfusion (I/R), pharmacological approaches have been pursued to both inhibit and clarify the importance of this exchanger.',\n",
              "  0),\n",
              " (1002,\n",
              "  27,\n",
              "  'The small molecule SEA0400 is an inhibitor of which ion antiporter/exchanger?',\n",
              "  'SEA0400 was reported to have a potent NCX selectivity.',\n",
              "  0),\n",
              " (1002,\n",
              "  28,\n",
              "  'The small molecule SEA0400 is an inhibitor of which ion antiporter/exchanger?',\n",
              "  'Thus, we examined the effect of SEA0400 on NCX currents and I/R induced intracellular Ca2+ overload in mouse ventricular myocytes using patch clamp techniques and fluorescence measurements.',\n",
              "  0),\n",
              " (1002,\n",
              "  29,\n",
              "  'The small molecule SEA0400 is an inhibitor of which ion antiporter/exchanger?',\n",
              "  'The results suggested that SEA0400 is a potent NCX inhibitor, which can protect mouse cardiac myocytes from Ca2+ overload during I/R injuries.',\n",
              "  0),\n",
              " (1002,\n",
              "  30,\n",
              "  'The small molecule SEA0400 is an inhibitor of which ion antiporter/exchanger?',\n",
              "  'The effect of SEA0400, a novel Na+-Ca2+ exchanger inhibitor, on mechanical and electrophysiological parameters of coronary-perfused guinea-pig right ventricular tissue preparation was examined during no-flow ischemia and reperfusion.',\n",
              "  0),\n",
              " (1002,\n",
              "  31,\n",
              "  'The small molecule SEA0400 is an inhibitor of which ion antiporter/exchanger?',\n",
              "  'The Ca2+ overload by Ca2+ influx via Na+/Ca2+ exchanger (NCX) is a critical mechanism in myocardial ischemia/reperfusion injury.',\n",
              "  0),\n",
              " (1002,\n",
              "  32,\n",
              "  'The small molecule SEA0400 is an inhibitor of which ion antiporter/exchanger?',\n",
              "  'We investigated protective effects of a novel selective inhibitor of NCX, SEA0400, on cardiac function and energy metabolism during ischemia and reperfusion.',\n",
              "  0),\n",
              " (1002,\n",
              "  33,\n",
              "  'The small molecule SEA0400 is an inhibitor of which ion antiporter/exchanger?',\n",
              "  'The cardioprotective effects of SEA0400, a novel Na(+)-Ca(2+) exchanger inhibitor, were examined in isolated guinea pig myocardial tissue and ventricular myocytes.',\n",
              "  0),\n",
              " (1002,\n",
              "  34,\n",
              "  'The small molecule SEA0400 is an inhibitor of which ion antiporter/exchanger?',\n",
              "  'The effects of a new, potent, and selective inhibitor of the Na(+)/Ca(2+) exchange, SEA-0400 (SEA), on steady-state outward (forward exchange), inward (reverse exchange), and Ca(2+)/Ca(2+) transport exchange modes were studied in internally dialyzed squid giant axons from both the extra- and intracellular sides.',\n",
              "  0),\n",
              " (1002,\n",
              "  35,\n",
              "  'The small molecule SEA0400 is an inhibitor of which ion antiporter/exchanger?',\n",
              "  'Inhibition by SEA takes place preferentially from the intracellular side of the membrane.',\n",
              "  0),\n",
              " (1002,\n",
              "  36,\n",
              "  'The small molecule SEA0400 is an inhibitor of which ion antiporter/exchanger?',\n",
              "  'Using SEA0400, a potent and selective inhibitor of the Na+-Ca2+ exchanger (NCX), we examined whether NCX is involved in nitric oxide (NO)-induced disturbance of endoplasmic reticulum (ER) Ca2+ homeostasis followed by apoptosis in cultured rat microglia.',\n",
              "  0),\n",
              " (1002,\n",
              "  37,\n",
              "  'The small molecule SEA0400 is an inhibitor of which ion antiporter/exchanger?',\n",
              "  'Activation of the Na+/Ca2+ exchanger may contribute to Ca2+ overload during reperfusion after transient ischemia.',\n",
              "  0),\n",
              " (1002,\n",
              "  38,\n",
              "  'The small molecule SEA0400 is an inhibitor of which ion antiporter/exchanger?',\n",
              "  'We examined the effects of 2-[4-[(2,5-difluorophenyl) methoxy]phenoxy]-5-ethoxyaniline (SEA0400), a selective inhibitor of Na+/Ca2+ exchange, on a canine model of ischemia/reperfusion injury (myocardial stunning).',\n",
              "  1),\n",
              " (1002,\n",
              "  39,\n",
              "  'The small molecule SEA0400 is an inhibitor of which ion antiporter/exchanger?',\n",
              "  'SEA0400 (2-[4-[(2,5-difluorophenyl)methoxy]phenoxy]-5-ethoxyaniline) has recently been described as a potent and selective inhibitor of Na(+)-Ca(2+) exchange in cardiac, neuronal, and renal preparations.',\n",
              "  0),\n",
              " (1002,\n",
              "  40,\n",
              "  'The small molecule SEA0400 is an inhibitor of which ion antiporter/exchanger?',\n",
              "  'The inhibitory effects of SEA0400 were investigated on the cloned cardiac Na(+)-Ca(2+) exchanger, NCX1.1, expressed in Xenopus laevis oocytes to gain insight into its inhibitory mechanism.',\n",
              "  0),\n",
              " (1002,\n",
              "  41,\n",
              "  'The small molecule SEA0400 is an inhibitor of which ion antiporter/exchanger?',\n",
              "  'The effects of 2-[4-[(2,5-difluorophenyl) methoxy]phenoxy]-5-ethoxyaniline (SEA0400), a newly synthesized Na(+)-Ca(2+) exchanger (NCX) inhibitor, on the NCX current and other membrane currents were examined in isolated guinea-pig ventricular myocytes and compared with those of 2-[2-[4-(4-nitrobenzyloxy) phenyl]ethyl]isothiourea (KB-R7943).',\n",
              "  0),\n",
              " (1002,\n",
              "  42,\n",
              "  'The small molecule SEA0400 is an inhibitor of which ion antiporter/exchanger?',\n",
              "  'SEA0400 concentration-dependently inhibited the NCX current with a 10 fold higher potency than that of KB-R7943; 1 microM SEA0400 and 10 microM KB-R7943 inhibited the NCX current by more than 80%.',\n",
              "  0),\n",
              " (1002,\n",
              "  43,\n",
              "  'The small molecule SEA0400 is an inhibitor of which ion antiporter/exchanger?',\n",
              "  'KB-R7943, at 10 microM, inhibited the sodium current, L-type calcium current, delayed rectifier potassium current and inwardly rectifying potassium current by more than 50%, but SEA0400 (1 microM) had no significant effect on these currents.',\n",
              "  0),\n",
              " (1002,\n",
              "  44,\n",
              "  'The small molecule SEA0400 is an inhibitor of which ion antiporter/exchanger?',\n",
              "  'These results indicate that SEA0400 is a potent and highly selective inhibitor of NCX, and would be a powerful tool for further studies on the role of NCX in the heart and the therapeutic potential of its inhibition.',\n",
              "  0),\n",
              " (1002,\n",
              "  45,\n",
              "  'The small molecule SEA0400 is an inhibitor of which ion antiporter/exchanger?',\n",
              "  'The effect of the newly synthesized compound 2-[4-[(2,5-difluorophenyl)methoxy]phenoxy]-5-ethoxyaniline (SEA0400) on the Na+-Ca2+ exchanger (NCX) was investigated and compared against that of 2-[2-[4-(4-nitrobenzyloxy)phenyl]ethyl]isothiourea (KB-R7943).',\n",
              "  0),\n",
              " (1002,\n",
              "  46,\n",
              "  'The small molecule SEA0400 is an inhibitor of which ion antiporter/exchanger?',\n",
              "  'In addition, the effects of SEA0400 on reperfusion injury in vitro and in vivo were examined.',\n",
              "  0),\n",
              " (1002,\n",
              "  47,\n",
              "  'The small molecule SEA0400 is an inhibitor of which ion antiporter/exchanger?',\n",
              "  'SEA0400 was extremely more potent than KB-R7943 in inhibiting Na+-dependent Ca2+ uptake in cultured neurons, astrocytes, and microglia: IC50s of SEA0400 and KB-R7943 were 5 to 33 nM and 2 to 4 microM, respectively.',\n",
              "  0),\n",
              " (1002,\n",
              "  48,\n",
              "  'The small molecule SEA0400 is an inhibitor of which ion antiporter/exchanger?',\n",
              "  'SEA0400 at the concentration range that inhibited NCX exhibited negligible affinities for the Ca2+ channels, Na+ channels, K+ channels, norepinephrine transporter, and 14 receptors, and did not affect the activities of the Na+/H+ exchanger, Na+,K+-ATPase, Ca2+-ATPase, and five enzymes.',\n",
              "  0),\n",
              " (1002,\n",
              "  49,\n",
              "  'The small molecule SEA0400 is an inhibitor of which ion antiporter/exchanger?',\n",
              "  'These results indicate that SEA0400 is the most potent and selective inhibitor of NCX, and suggest that the compound may exert protective effects on postischemic brain damage.',\n",
              "  0),\n",
              " (2329,\n",
              "  0,\n",
              "  'What is the function of the TFIIS transcriptional factor (Dst1) in yeast?',\n",
              "  'TFIIS, an elongation factor encoded by DST1 in Saccharomyces cerevisiae, stimulates transcript cleavage in arrested RNA polymerase II.',\n",
              "  1),\n",
              " (2329,\n",
              "  1,\n",
              "  'What is the function of the TFIIS transcriptional factor (Dst1) in yeast?',\n",
              "  'Co-immunoprecipitation experiments showed that TFIIS can bind the Cdk8 module and SAGA in cell-free extracts',\n",
              "  1),\n",
              " (2329,\n",
              "  2,\n",
              "  'What is the function of the TFIIS transcriptional factor (Dst1) in yeast?',\n",
              "  'Members of the SAGA and Mediator complexes are partners of the transcription elongation factor TFIIS.',\n",
              "  1),\n",
              " (2329,\n",
              "  3,\n",
              "  'What is the function of the TFIIS transcriptional factor (Dst1) in yeast?',\n",
              "  'It is proposed that TFIIS and the Spt8-containing form of SAGA co-operate to rescue RNA polymerase II from unproductive elongation complexes, and that the Cdk8 module temporarily blocks transcription during transcript cleavage.',\n",
              "  1),\n",
              " (2329,\n",
              "  4,\n",
              "  'What is the function of the TFIIS transcriptional factor (Dst1) in yeast?',\n",
              "  \"TFIIS promotes the intrinsic ability of RNA polymerase II to cleave the 3'-end of the newly synthesized RNA.\",\n",
              "  0),\n",
              " (2329,\n",
              "  5,\n",
              "  'What is the function of the TFIIS transcriptional factor (Dst1) in yeast?',\n",
              "  'TFIIS and Rpb9 are essential when the cells are challenged with microtubule-destabilizing drugs;',\n",
              "  0),\n",
              " (2329,\n",
              "  6,\n",
              "  'What is the function of the TFIIS transcriptional factor (Dst1) in yeast?',\n",
              "  'While TFIIS has a pronounced effect on transcription elongation in vitro, the deletion of DST1 has no major effect on cell viability.',\n",
              "  0),\n",
              " (2329,\n",
              "  7,\n",
              "  'What is the function of the TFIIS transcriptional factor (Dst1) in yeast?',\n",
              "  'We propose that TFIIS is required under stress conditions and that TFIIS is important for the transition between initiation and elongation in vivo.',\n",
              "  0),\n",
              " (2329,\n",
              "  8,\n",
              "  'What is the function of the TFIIS transcriptional factor (Dst1) in yeast?',\n",
              "  'The eukaryotic transcript elongation factor TFIIS is encoded by a nonessential gene, PPR2, in Saccharomyces cerevisiae.',\n",
              "  0),\n",
              " (2329,\n",
              "  9,\n",
              "  'What is the function of the TFIIS transcriptional factor (Dst1) in yeast?',\n",
              "  'Genome-wide location analysis reveals a role of TFIIS in RNA polymerase III transcription.',\n",
              "  0),\n",
              " (2329,\n",
              "  10,\n",
              "  'What is the function of the TFIIS transcriptional factor (Dst1) in yeast?',\n",
              "  'TFIIS is a transcription elongation factor that stimulates transcript cleavage activity of arrested RNA polymerase II (Pol II)',\n",
              "  0),\n",
              " (2329,\n",
              "  11,\n",
              "  'What is the function of the TFIIS transcriptional factor (Dst1) in yeast?',\n",
              "  'Recent studies revealed that TFIIS has also a role in Pol II transcription initiation.',\n",
              "  0),\n",
              " (2329,\n",
              "  12,\n",
              "  'What is the function of the TFIIS transcriptional factor (Dst1) in yeast?',\n",
              "  ' TFIIS and Pol III occupancies correlated well genome-wide on this novel class of targets.',\n",
              "  0),\n",
              " (2329,\n",
              "  13,\n",
              "  'What is the function of the TFIIS transcriptional factor (Dst1) in yeast?',\n",
              "  'These data provide strong in vivo and in vitro evidence in favor of a role of TFIIS as a general Pol III transcription factor.',\n",
              "  0),\n",
              " (2329,\n",
              "  14,\n",
              "  'What is the function of the TFIIS transcriptional factor (Dst1) in yeast?',\n",
              "  'Transcription factor IIS (TFIIS) stimulates RNA cleavage by RNA polymerase II by allowing backtracked enzymes to resume transcription elongation.',\n",
              "  1),\n",
              " (2329,\n",
              "  15,\n",
              "  'What is the function of the TFIIS transcriptional factor (Dst1) in yeast?',\n",
              "  'TFIIS is required for the balanced expression of the genes encoding ribosomal components under transcriptional stress.',\n",
              "  0),\n",
              " (2329,\n",
              "  16,\n",
              "  'What is the function of the TFIIS transcriptional factor (Dst1) in yeast?',\n",
              "  'We discuss the effect of transcriptional stress on ribosome biogenesis and propose that TFIIS contributes to prevent a transcriptional imbalance between rDNA and RP genes.',\n",
              "  0),\n",
              " (2329,\n",
              "  17,\n",
              "  'What is the function of the TFIIS transcriptional factor (Dst1) in yeast?',\n",
              "  'Here, we investigated the role of nascent transcript cleavage stimulation activity on the maintenance of transcriptional fidelity in yeast.',\n",
              "  0),\n",
              " (2329,\n",
              "  18,\n",
              "  'What is the function of the TFIIS transcriptional factor (Dst1) in yeast?',\n",
              "  'TFIIS is a transcription elongation factor that stimulates transcript cleavage activity of arrested RNA polymerase II (Pol II).',\n",
              "  0),\n",
              " (2329,\n",
              "  19,\n",
              "  'What is the function of the TFIIS transcriptional factor (Dst1) in yeast?',\n",
              "  'A mild but systematic increase in RNA polymerase occupancy and a strict dependency on the transcript cleavage factor TFIIS (Dst1) also suggest a slower rate of translocation or higher probability of transcriptional stalling in this mutation.',\n",
              "  0),\n",
              " (2329,\n",
              "  20,\n",
              "  'What is the function of the TFIIS transcriptional factor (Dst1) in yeast?',\n",
              "  'Also, a DST1 and RPB9 double mutant had more severe transcriptional fidelity defect compared with the DST1 gene deletion mutant, suggesting that Rpb9 maintains transcriptional fidelity via two mechanisms, enhancement of S-II dependent cleavage stimulation and S-II independent function(s)..',\n",
              "  0),\n",
              " (2839,\n",
              "  0,\n",
              "  'What is DiseaseEnhancer?',\n",
              "  'DiseaseEnhancer: a resource of human disease-associated enhancer catalog.',\n",
              "  1),\n",
              " (2839,\n",
              "  1,\n",
              "  'What is DiseaseEnhancer?',\n",
              "  ' To facilitate better usage of published data and exploring enhancer deregulation in various human diseases, we created DiseaseEnhancer (http://biocc.hrbmu.edu.cn/DiseaseEnhancer/), a manually curated database for disease-associated enhancers.',\n",
              "  0),\n",
              " (2839,\n",
              "  2,\n",
              "  'What is DiseaseEnhancer?',\n",
              "  'As of July 2017, DiseaseEnhancer includes 847 disease-associated enhancers in 143 human diseases.',\n",
              "  1),\n",
              " (2839,\n",
              "  3,\n",
              "  'What is DiseaseEnhancer?',\n",
              "  'Database features include basic enhancer information (i.e.',\n",
              "  0),\n",
              " (2839,\n",
              "  4,\n",
              "  'What is DiseaseEnhancer?',\n",
              "  'genomic location and target genes); disease types; associated variants on the enhancer and their mediated phenotypes (i.e.',\n",
              "  1),\n",
              " (2839,\n",
              "  5,\n",
              "  'What is DiseaseEnhancer?',\n",
              "  'gain/loss of enhancer and the alterations of transcription factor bindings).',\n",
              "  1),\n",
              " (2839,\n",
              "  6,\n",
              "  'What is DiseaseEnhancer?',\n",
              "  'DiseaseEnhancer provides a promising avenue for researchers to facilitate the understanding of enhancer deregulation in disease pathogenesis, and identify new biomarkers for disease diagnosis and therapy.',\n",
              "  0),\n",
              " (2839,\n",
              "  7,\n",
              "  'What is DiseaseEnhancer?',\n",
              "  'To facilitate better usage of published data and exploring enhancer deregulation in various human diseases, we created DiseaseEnhancer (http://biocc.hrbmu.edu.cn/DiseaseEnhancer/), a manually curated database for disease-associated enhancers.',\n",
              "  0),\n",
              " (2839,\n",
              "  8,\n",
              "  'What is DiseaseEnhancer?',\n",
              "  'As of July 2017, DiseaseEnhancer includes 847 disease-associated enhancers in 143 human diseases.',\n",
              "  1),\n",
              " (1046,\n",
              "  0,\n",
              "  'What are the generic versions of Viagra',\n",
              "  'Sildenafil citrate (SIL)',\n",
              "  1),\n",
              " (1046,\n",
              "  1,\n",
              "  'What are the generic versions of Viagra',\n",
              "  'generic sildenafil (Unison Laboratories, Thailand) was proved to have the same bioequivalent as in the original formula.',\n",
              "  1),\n",
              " (1046,\n",
              "  2,\n",
              "  'What are the generic versions of Viagra',\n",
              "  'The authors conducted a 12-week case series to study the efficacy and safety of Elonza (generic sildenafil)',\n",
              "  1),\n",
              " (1046,\n",
              "  3,\n",
              "  'What are the generic versions of Viagra',\n",
              "  'Viagra tablets or generic versions containing sildenafil citrate',\n",
              "  1),\n",
              " (2574,\n",
              "  0,\n",
              "  'What organism causes scarlet fever also known as scarletina?',\n",
              "  'Scarlet fever is one of a variety of diseases caused by group A Streptococcus (GAS).',\n",
              "  1),\n",
              " (2574,\n",
              "  1,\n",
              "  'What organism causes scarlet fever also known as scarletina?',\n",
              "  'In a small epidemic of scarlet fever a healthy carrier of hemolytic streptococcus was detected; the organism carried was identical in its serological reactions with strains of hemolytic streptococci isolated from active cases of scarlet fever.',\n",
              "  0),\n",
              " (2574,\n",
              "  2,\n",
              "  'What organism causes scarlet fever also known as scarletina?',\n",
              "  'No morphological or cultural characteristics peculiar to the hemolytic streptococcus from scarlet fever can be demonstrated.',\n",
              "  0),\n",
              " (2574,\n",
              "  3,\n",
              "  'What organism causes scarlet fever also known as scarletina?',\n",
              "  'Application of multilocus enzyme electrophoresis and comparative sequencing of the gene (speA) encoding streptococcal pyrogenic exotoxin a (scarlet fever toxin) revealed that new waves of scarlet fever are associated with an increase in frequency of S. pyogenes clones carrying variant speA alleles.',\n",
              "  0),\n",
              " (2574,\n",
              "  4,\n",
              "  'What organism causes scarlet fever also known as scarletina?',\n",
              "  'Additionally, S. pyogenes induces toxin-mediated syndromes, i. e. scarlet fever, streptococcal toxic shock syndrome (STSS) and necrotizing fasciitis (NF).',\n",
              "  0),\n",
              " (2574,\n",
              "  5,\n",
              "  'What organism causes scarlet fever also known as scarletina?',\n",
              "  'Group A Streptococcus (GAS) is the only pathogen known to cause scarlet fever.',\n",
              "  0),\n",
              " (2574,\n",
              "  6,\n",
              "  'What organism causes scarlet fever also known as scarletina?',\n",
              "  'We present a case that illustrates the distinctive clinical spectrum of A. hemolyticum infections that may be confused with drug allergy, group A streptococcal scarlet fever, diphtheria, and even toxic shock syndrome.',\n",
              "  0),\n",
              " (2574,\n",
              "  7,\n",
              "  'What organism causes scarlet fever also known as scarletina?',\n",
              "  '1.',\n",
              "  0),\n",
              " (2574,\n",
              "  8,\n",
              "  'What organism causes scarlet fever also known as scarletina?',\n",
              "  'Hemolytic streptococcus has been found in 100 per cent of the throats of patients with scarlet fever during the 1st week of the disease.',\n",
              "  0),\n",
              " (2574,\n",
              "  9,\n",
              "  'What organism causes scarlet fever also known as scarletina?',\n",
              "  'Group C beta-hemolytic streptococci causing pharyngitis and scarlet fever.',\n",
              "  0),\n",
              " (2574,\n",
              "  10,\n",
              "  'What organism causes scarlet fever also known as scarletina?',\n",
              "  'Several outbreaks of scarlet fever caused by Streptococcus pyogenes were recently reported.',\n",
              "  0),\n",
              " (2574,\n",
              "  11,\n",
              "  'What organism causes scarlet fever also known as scarletina?',\n",
              "  'Little information is available on the differences in frequency of pyrogenic exotoxin genes between strains of group A streptococci that cause scarlet fever and those that cause pharyngotonsillitis in children in Taiwan.',\n",
              "  0),\n",
              " (2574,\n",
              "  12,\n",
              "  'What organism causes scarlet fever also known as scarletina?',\n",
              "  'Molecular characterization of Group A streptococcal isolates causing scarlet fever and pharyngitis among young children: a retrospective study from a northern Taiwan medical center.',\n",
              "  0),\n",
              " (2574,\n",
              "  13,\n",
              "  'What organism causes scarlet fever also known as scarletina?',\n",
              "  'group A streptococcus causing scarlet fever',\n",
              "  1),\n",
              " (2574,\n",
              "  14,\n",
              "  'What organism causes scarlet fever also known as scarletina?',\n",
              "  'scarlet-fever-related group A streptococcal,',\n",
              "  1),\n",
              " (2574,\n",
              "  15,\n",
              "  'What organism causes scarlet fever also known as scarletina?',\n",
              "  'Infection with group A beta-hemolytic streptococci (GABHS) is the most common bacterial cause of acute pharyngitis and tonsillitis beyond infancy.',\n",
              "  0),\n",
              " (2574,\n",
              "  16,\n",
              "  'What organism causes scarlet fever also known as scarletina?',\n",
              "  'scarlet fever is a local infection of the throat by a particular type of Streptococcus haemolyticus capable of producing a toxin which is absorbed and is the cause of the general manifestations of the disease',\n",
              "  1),\n",
              " (2574,\n",
              "  17,\n",
              "  'What organism causes scarlet fever also known as scarletina?',\n",
              "  'an epidemic by S.pyogenes type 1 with many cases of toxic shock was observed, the same type caused a scarlet fever epidemic without complications in eastern Germany.',\n",
              "  0),\n",
              " (2574,\n",
              "  18,\n",
              "  'What organism causes scarlet fever also known as scarletina?',\n",
              "  'clinical Streptococcus pyogenes strains causing scarlet fever and the streptococcal toxic shock-like syndrome (TSLS).',\n",
              "  0),\n",
              " (2574,\n",
              "  19,\n",
              "  'What organism causes scarlet fever also known as scarletina?',\n",
              "  'Streptococcus pyogenes (group A Streptococcus - GAS) is an important human pathogen which causes a variety of diseases, including tonsillopharyngitis, scarlet fever and rheumatic fever.',\n",
              "  1),\n",
              " (2574,\n",
              "  20,\n",
              "  'What organism causes scarlet fever also known as scarletina?',\n",
              "  'OF STREPTOCOCCUS : II.',\n",
              "  0),\n",
              " (2574,\n",
              "  21,\n",
              "  'What organism causes scarlet fever also known as scarletina?',\n",
              "  'ANTIGENIC RELATIONSHIPS BETWEEN STRAINS OF STREPTOCOCCUS HAEMOLYTICUS ISOLATED FROM SCARLET FEVER.',\n",
              "  0),\n",
              " (1313,\n",
              "  0,\n",
              "  'Mutations in which genes have been associated with Aicardi-Goutieres syndrome?',\n",
              "  'Aicardi-Goutières syndrome (AGS) is an inflammatory disorder caused by mutations in any of six genes (TREX1, RNASEH2A, RNASEH2B, RNASEH2C, SAMHD1, and ADAR).',\n",
              "  1),\n",
              " (1313,\n",
              "  1,\n",
              "  'Mutations in which genes have been associated with Aicardi-Goutieres syndrome?',\n",
              "  'Aicardi-Goutières syndrome (AGS) is an inflammatory disorder caused by mutations in any of six genes (TREX1, RNASEH2A, RNASEH2B, RNASEH2C, SAMHD1, and ADAR)',\n",
              "  1),\n",
              " (3890,\n",
              "  0,\n",
              "  'Where are integrins localized in a cell?',\n",
              "  'The extracellular domain of plasma membrane integrin αvβ3',\n",
              "  1),\n",
              " (3890,\n",
              "  1,\n",
              "  'Where are integrins localized in a cell?',\n",
              "  'Integrins are transmembrane glycoproteins that are broadly distributed in living organisms.',\n",
              "  1),\n",
              " (3890,\n",
              "  2,\n",
              "  'Where are integrins localized in a cell?',\n",
              "  'The immunofluorescence results showed that Bmintegrin β1 was located in the cell membrane',\n",
              "  1),\n",
              " (3269,\n",
              "  0,\n",
              "  'Is Li–Fraumeni syndrome a rare, autosomal recessive, hereditary disorder that predisposes carriers to cancer development?',\n",
              "  'Li-Fraumeni syndrome (LFS) is a rare cancer predisposition syndrome inherited in an autosomal dominant fashion that involves a germline mutation of tumor protein 53 (TP53).',\n",
              "  1),\n",
              " (3269,\n",
              "  1,\n",
              "  'Is Li–Fraumeni syndrome a rare, autosomal recessive, hereditary disorder that predisposes carriers to cancer development?',\n",
              "  'Li-Fraumeni syndrome (LFS), a multiorgan cancer predisposition caused by germline TP53 mutations, confers significant cancer risks for young people (15-39 years).',\n",
              "  0),\n",
              " (3269,\n",
              "  2,\n",
              "  'Is Li–Fraumeni syndrome a rare, autosomal recessive, hereditary disorder that predisposes carriers to cancer development?',\n",
              "  'Yet evidence of how individuals experience this condition and the psychosocial implications are lacking.',\n",
              "  0),\n",
              " (3269,\n",
              "  3,\n",
              "  'Is Li–Fraumeni syndrome a rare, autosomal recessive, hereditary disorder that predisposes carriers to cancer development?',\n",
              "  'Therefore, this systematic review assessed the psychosocial implications of living with, or at risk of, an autosomal dominant condition as a young person, to',\n",
              "  0),\n",
              " (3269,\n",
              "  4,\n",
              "  'Is Li–Fraumeni syndrome a rare, autosomal recessive, hereditary disorder that predisposes carriers to cancer development?',\n",
              "  'Li-Fraumeni syndrome (LFS) is a rare autosomal dominant disorder caused by a mutation in the p53 gene.',\n",
              "  0),\n",
              " (3269,\n",
              "  5,\n",
              "  'Is Li–Fraumeni syndrome a rare, autosomal recessive, hereditary disorder that predisposes carriers to cancer development?',\n",
              "  'Li-Fraumeni syndrome (LFS) is an autosomal dominant disorder occurring at a young age that predisposes individuals to multiple forms of cancer and to a heterogeneous spectrum of malignancies.',\n",
              "  0),\n",
              " (3269,\n",
              "  6,\n",
              "  'Is Li–Fraumeni syndrome a rare, autosomal recessive, hereditary disorder that predisposes carriers to cancer development?',\n",
              "  'Li-Fraumeni syndrome (LFS) is a rare hereditary autosomal dominant cancer disorder.',\n",
              "  1),\n",
              " (3269,\n",
              "  7,\n",
              "  'Is Li–Fraumeni syndrome a rare, autosomal recessive, hereditary disorder that predisposes carriers to cancer development?',\n",
              "  'Li-Fraumeni syndrome (LFS) is an autosomal dominant hereditary cancer disorder.',\n",
              "  0),\n",
              " (3269,\n",
              "  8,\n",
              "  'Is Li–Fraumeni syndrome a rare, autosomal recessive, hereditary disorder that predisposes carriers to cancer development?',\n",
              "  'Li-Fraumeni syndrome (LFS) is a rare, autosomal dominant, hereditary cancer predisposition disorder.',\n",
              "  1),\n",
              " (3269,\n",
              "  9,\n",
              "  'Is Li–Fraumeni syndrome a rare, autosomal recessive, hereditary disorder that predisposes carriers to cancer development?',\n",
              "  'Li-Fraumeni syndrome is a rare autosomal dominant cancer predisposition syndrome.',\n",
              "  1),\n",
              " (3269,\n",
              "  10,\n",
              "  'Is Li–Fraumeni syndrome a rare, autosomal recessive, hereditary disorder that predisposes carriers to cancer development?',\n",
              "  'Li-Fraumeni syndrome is an autosomal dominant disorder that is characterized by various types of cancer in childhood and adult cases.',\n",
              "  0),\n",
              " (3269,\n",
              "  11,\n",
              "  'Is Li–Fraumeni syndrome a rare, autosomal recessive, hereditary disorder that predisposes carriers to cancer development?',\n",
              "  'Li-Fraumeni syndrome (LFS) is a rare hereditary cancer syndrome associated with an autosomal dominant mutation inheritance in the TP53 tumor suppressor gene and a wide spectrum of cancer diagnoses.',\n",
              "  0),\n",
              " (3269,\n",
              "  12,\n",
              "  'Is Li–Fraumeni syndrome a rare, autosomal recessive, hereditary disorder that predisposes carriers to cancer development?',\n",
              "  'The Li-Fraumeni syndrome is an autosomal dominant disorder characterized by a high risk of developing osteosarcoma and has been found in up to 3% of children with osteosarcoma.',\n",
              "  0),\n",
              " (3269,\n",
              "  13,\n",
              "  'Is Li–Fraumeni syndrome a rare, autosomal recessive, hereditary disorder that predisposes carriers to cancer development?',\n",
              "  'Li-Fraumeni syndrome (LFS) is a rare, autosomal dominant, hereditary cancer predisposition disorder.',\n",
              "  1),\n",
              " (3269,\n",
              "  14,\n",
              "  'Is Li–Fraumeni syndrome a rare, autosomal recessive, hereditary disorder that predisposes carriers to cancer development?',\n",
              "  'Li-Fraumeni syndrome (LFS) is an inherited, autosomal-dominant condition that predisposes individuals to a wide-spectrum of tumors at an early age.',\n",
              "  0),\n",
              " (3269,\n",
              "  15,\n",
              "  'Is Li–Fraumeni syndrome a rare, autosomal recessive, hereditary disorder that predisposes carriers to cancer development?',\n",
              "  'The Li-Fraumeni syndrome (LFS) is an autosomal dominant hereditary disorder associated with different tumor types in childhood and young adults.',\n",
              "  0),\n",
              " (206,\n",
              "  0,\n",
              "  'Between which probes does the recurrent translocation breakpoint on chromosome 22 of neuroepithelioma lie?',\n",
              "  'The recurrent translocation breakpoint on chromosome 22 of neuroepithelioma has been localized between two probes, D22S1 and D22S15, by both in situ hybridization and somatic cell hybrids',\n",
              "  1),\n",
              " (206,\n",
              "  1,\n",
              "  'Between which probes does the recurrent translocation breakpoint on chromosome 22 of neuroepithelioma lie?',\n",
              "  'The recurrent translocation breakpoint on chromosome 22 of neuroepithelioma has been localized between two probes, D22S1 and D22S15, by both in situ hybridization and somatic cell hybrids.',\n",
              "  1),\n",
              " (2110,\n",
              "  0,\n",
              "  'Signaling of which pathways is inhibited by Dupilumab?',\n",
              "  'This study reviewed all studies about any roles of IL-4 that can directly and indirectly be played in the development of pemphigus and IL-4 inhibition with interferons and dupilumab therapy were introduced as a novel pemphigus treatment for patients who are in relapse phase of the disease.',\n",
              "  0),\n",
              " (2110,\n",
              "  1,\n",
              "  'Signaling of which pathways is inhibited by Dupilumab?',\n",
              "  'Dupilumab was also introduced as a possible treatment for patients with severe pemphigus.',\n",
              "  0),\n",
              " (2110,\n",
              "  2,\n",
              "  'Signaling of which pathways is inhibited by Dupilumab?',\n",
              "  'It can directly inhibit IL-4 by targeting IL-4 α-chain receptor.',\n",
              "  0),\n",
              " (2110,\n",
              "  3,\n",
              "  'Signaling of which pathways is inhibited by Dupilumab?',\n",
              "  'OBJECTIVE: To assess inhibition of interleukins 4 and 13 with dupilumab in patients with chronic sinusitis and nasal polyposis.',\n",
              "  0),\n",
              " (2110,\n",
              "  4,\n",
              "  'Signaling of which pathways is inhibited by Dupilumab?',\n",
              "  'BACKGROUND: Dupilumab, a fully human anti-interleukin-4 receptor α monoclonal antibody, inhibits interleukin-4 and interleukin-13 signalling, key drivers of type-2-mediated inflammation.',\n",
              "  1),\n",
              " (2110,\n",
              "  5,\n",
              "  'Signaling of which pathways is inhibited by Dupilumab?',\n",
              "  'Anti-IL-4 and IL-13 agents (dupilumab, lebrikizumab, and tralokinumab) which block different Th-2 inflammatory pathways and agents targeting the Th-17 inflammatory pathway in severe refractory asthma are under development.',\n",
              "  0),\n",
              " (2110,\n",
              "  6,\n",
              "  'Signaling of which pathways is inhibited by Dupilumab?',\n",
              "  'Best evidence of the clinical efficacy of novel immunologic approaches using biological agents in patients with AD is available for the anti-IL-4 receptor α-chain antibody dupilumab, but a number of studies are currently ongoing with other specific antagonists to immune system players.',\n",
              "  0),\n",
              " (2110,\n",
              "  7,\n",
              "  'Signaling of which pathways is inhibited by Dupilumab?',\n",
              "  'Background Dupilumab, a human monoclonal antibody against interleukin-4 receptor alpha, inhibits signaling of interleukin-4 and interleukin-13, type 2 cytokines that may be important drivers of atopic or allergic diseases such as atopic dermatitis.',\n",
              "  1),\n",
              " (2110,\n",
              "  8,\n",
              "  'Signaling of which pathways is inhibited by Dupilumab?',\n",
              "  'The world\\'s first prospective controlled studies with the biologic human anti-IL4R antibody dupilumab for the indication \"atopic dermatitis\" were published in 2014.',\n",
              "  0),\n",
              " (2110,\n",
              "  9,\n",
              "  'Signaling of which pathways is inhibited by Dupilumab?',\n",
              "  'An IL-4R antagonist, dupilumab, is the first drug that shows great promise in phase II trials.',\n",
              "  0),\n",
              " (2110,\n",
              "  10,\n",
              "  'Signaling of which pathways is inhibited by Dupilumab?',\n",
              "  'Dupilumab, a humanized monoclonal antibody to the interteukin-4R is the first antibody (i.e.',\n",
              "  0),\n",
              " (2110,\n",
              "  11,\n",
              "  'Signaling of which pathways is inhibited by Dupilumab?',\n",
              "  \"'biological') with published efficacy shown in controlled prospective studies in atopic dermatitis.\",\n",
              "  0),\n",
              " (2110,\n",
              "  12,\n",
              "  'Signaling of which pathways is inhibited by Dupilumab?',\n",
              "  'Dupilumab is a fully human monoclonal antibody directed against the IL-4 receptor α subunit that blocks the signaling of IL-4 and IL-13, both key cytokines in Th2-mediated pathways.',\n",
              "  0),\n",
              " (2110,\n",
              "  13,\n",
              "  'Signaling of which pathways is inhibited by Dupilumab?',\n",
              "  'Dupilumab inhibits interleukin-4 (IL-4) and interleukin-13 (IL-13) signaling and was previously found to be effective in asthma.',\n",
              "  1),\n",
              " (2110,\n",
              "  14,\n",
              "  'Signaling of which pathways is inhibited by Dupilumab?',\n",
              "  'Data from early-stage studies suggested that interleukin (IL)-4 and IL-13 are requisite drivers of atopic dermatitis, evidenced by marked improvement after treatment with dupilumab, a fully-human monoclonal antibody that blocks both pathways.',\n",
              "  0),\n",
              " (2110,\n",
              "  15,\n",
              "  'Signaling of which pathways is inhibited by Dupilumab?',\n",
              "  'Background Dupilumab, a human monoclonal antibody against interleukin-4 receptor alpha, inhibits signaling of interleukin-4 and interleukin-13, type 2 cytokines that may be important drivers of atopic or allergic diseases such as atopic dermatitis.',\n",
              "  1),\n",
              " (2110,\n",
              "  16,\n",
              "  'Signaling of which pathways is inhibited by Dupilumab?',\n",
              "  'The robust effects of dupilumab on skin inflammation and pruritus confirm the pathogenic role of IL-4 and IL-13 signaling in adult AD, and further support the application of Th2 cytokine antagonists in the treatment of this disease.',\n",
              "  0),\n",
              " (2110,\n",
              "  17,\n",
              "  'Signaling of which pathways is inhibited by Dupilumab?',\n",
              "  'Dupilumab, a fully human anti-interleukin-4 receptor α monoclonal antibody, inhibits interleukin-4 and interleukin-13 signalling, key drivers of type-2-mediated inflammation.',\n",
              "  1),\n",
              " (2110,\n",
              "  18,\n",
              "  'Signaling of which pathways is inhibited by Dupilumab?',\n",
              "  'Dupilumab is a fully human monoclonal antibody directed against the IL-4 receptor α subunit that blocks the signaling of IL-4 and IL-13, both key cytokines in Th2-mediated pathways',\n",
              "  0),\n",
              " (2110,\n",
              "  19,\n",
              "  'Signaling of which pathways is inhibited by Dupilumab?',\n",
              "  'In early-phase trials, dupilumab, a fully human mAb targeting IL-4 receptor �, markedly improved disease activity, but the effect of IL-4/IL-13 blockade on AD at the molecular level has not been characterized.OBJECTIVES: We sought to evaluate dupilumab modulation of the AD molecular signature.METHODS: We performed transcriptomic analyses of pretreatment and posttreatment skin biopsy specimens from patients with moderate-to-severe AD treated weekly with 150 or 300 mg of dupilumab or placebo.RESULTS: Exacerbation of the AD transcriptome was observed in placebo-treated patients.',\n",
              "  0),\n",
              " (2110,\n",
              "  20,\n",
              "  'Signaling of which pathways is inhibited by Dupilumab?',\n",
              "  'Dupilumab is a fully human monoclonal antibody directed against the IL-4 receptor α subunit that blocks the signaling of IL-4 and IL-13, both key cytokines in Th2-mediated pathways.',\n",
              "  0),\n",
              " (2110,\n",
              "  21,\n",
              "  'Signaling of which pathways is inhibited by Dupilumab?',\n",
              "  'Dupilumab inhibits interleukin-4 (IL-4) and interleukin-13 (IL-13) signaling and was previously found to be effective in asthma.',\n",
              "  0),\n",
              " (2110,\n",
              "  22,\n",
              "  'Signaling of which pathways is inhibited by Dupilumab?',\n",
              "  'The robust effects of dupilumab on skin inflammation and pruritus confirm the pathogenic role of IL-4 and IL-13 signaling in adult AD, and further support the application of Th2 cytokine antagonists in the treatment of this disease.',\n",
              "  0),\n",
              " (2110,\n",
              "  23,\n",
              "  'Signaling of which pathways is inhibited by Dupilumab?',\n",
              "  'In early-phase trials, dupilumab, a fully human mAb targeting IL-4 receptor á, markedly improved disease activity, but the effect of IL-4/IL-13 blockade on AD at the molecular level has not been characterized.OBJECTIVES: We sought to evaluate dupilumab modulation of the AD molecular signature.METHODS: We performed transcriptomic analyses of pretreatment and posttreatment skin biopsy specimens from patients with moderate-to-severe AD treated weekly with 150 or 300 mg of dupilumab or placebo.RESULTS: Exacerbation of the AD transcriptome was observed in placebo-treated patients.',\n",
              "  0),\n",
              " (3547,\n",
              "  0,\n",
              "  'What are the most common side effects of amantadine ER?',\n",
              "  'The most common side effects are hallucination, dizziness, orthostatic hypotension and pedal edema.',\n",
              "  1),\n",
              " (1625,\n",
              "  0,\n",
              "  'What is the use of emulsion PCR in Next Generation Sequencing?',\n",
              "  'Emulsion PCR-coupled target enrichment: an effective fishing method for high-throughput sequencing of poorly preserved ancient DNA',\n",
              "  0),\n",
              " (1625,\n",
              "  1,\n",
              "  'What is the use of emulsion PCR in Next Generation Sequencing?',\n",
              "  'We present a unified strategy in which emulsion PCR is coupled with target enrichment followed by next-generation sequencing.',\n",
              "  1),\n",
              " (1625,\n",
              "  2,\n",
              "  'What is the use of emulsion PCR in Next Generation Sequencing?',\n",
              "  'The method made it possible to obtain efficiently non-duplicated reads mapped to target sequences of interest, and this can achieve deep and reliable sequencing of ancient DNA from typical materials, even though poorly preserved.',\n",
              "  0),\n",
              " (1625,\n",
              "  3,\n",
              "  'What is the use of emulsion PCR in Next Generation Sequencing?',\n",
              "  'Three multiplex PCR reactions were carried out to amplify the coding exons of 328 genes including FBN1, TGFBR1 and TGFBR2.',\n",
              "  0),\n",
              " (1625,\n",
              "  4,\n",
              "  'What is the use of emulsion PCR in Next Generation Sequencing?',\n",
              "  'DNA fragments from different samples were ligated with barcoded sequencing adaptors.',\n",
              "  0),\n",
              " (1625,\n",
              "  5,\n",
              "  'What is the use of emulsion PCR in Next Generation Sequencing?',\n",
              "  'Template preparation and emulsion PCR, and Ion Sphere Particles enrichment were carried out using an Ion One Touch system.',\n",
              "  0),\n",
              " (1625,\n",
              "  6,\n",
              "  'What is the use of emulsion PCR in Next Generation Sequencing?',\n",
              "  'The ion sphere particles were sequenced on a 318 chip using the PGM platform.',\n",
              "  0),\n",
              " (1625,\n",
              "  7,\n",
              "  'What is the use of emulsion PCR in Next Generation Sequencing?',\n",
              "  'In our NGS approach, the 16,569-bp mtDNA is enriched by long-range PCR and the 108 nuclear genes (which represent 1301 amplicons and 680 kb) are enriched by RainDance emulsion PCR.',\n",
              "  0),\n",
              " (1625,\n",
              "  8,\n",
              "  'What is the use of emulsion PCR in Next Generation Sequencing?',\n",
              "  'Sequencing is performed on Illumina HiSeq 2000 or MiSeq platforms, and bioinformatics analysis is performed using commercial and in-house developed bioinformatics pipelines.',\n",
              "  0),\n",
              " (1625,\n",
              "  9,\n",
              "  'What is the use of emulsion PCR in Next Generation Sequencing?',\n",
              "  'However, pyrosequencing of emulsion PCR reactions, amplifying from only one molecule at a time, can generate megabases of clonally amplified loci at high coverage, thereby greatly simplifying allelic sequence determination.',\n",
              "  0),\n",
              " (1625,\n",
              "  10,\n",
              "  'What is the use of emulsion PCR in Next Generation Sequencing?',\n",
              "  \"During genomic PCR, in this 4-primer system, the outer set of primers containing the MID and the 454 adaptor sequences are incorporated into an amplicon generated by the inner HLA target-specific primers each containing a common sequence tag at the 5' end of the forward and reverse primers.\",\n",
              "  0),\n",
              " (1625,\n",
              "  11,\n",
              "  'What is the use of emulsion PCR in Next Generation Sequencing?',\n",
              "  'Pools of the resulting amplicons are used for emulsion PCR and clonal sequencing on the 454 Life Sciences GS FLX System, followed by genotyping with Conexio software.',\n",
              "  1),\n",
              " (1625,\n",
              "  12,\n",
              "  'What is the use of emulsion PCR in Next Generation Sequencing?',\n",
              "  'The purpose of our study was to compare NGS enrichment methods for a clinical assay targeting the nine genes known to be associated with aortopathy.',\n",
              "  0),\n",
              " (1625,\n",
              "  13,\n",
              "  'What is the use of emulsion PCR in Next Generation Sequencing?',\n",
              "  'RainDance emulsion PCR and SureSelect RNA-bait hybridization capture enrichment methods were directly compared by enriching DNA from eight samples.',\n",
              "  0),\n",
              " (1625,\n",
              "  14,\n",
              "  'What is the use of emulsion PCR in Next Generation Sequencing?',\n",
              "  'We show the efficacy by comparing the method to a commercial kit and further demonstrate that emulsion PCR can be used for bias free amplification and virtual immortalization of DNA template libraries.',\n",
              "  0),\n",
              " (1625,\n",
              "  15,\n",
              "  'What is the use of emulsion PCR in Next Generation Sequencing?',\n",
              "  'Here, we describe a novel approach for normalization of multiplex next generation sequencing libraries after emulsion PCR.',\n",
              "  1),\n",
              " (1625,\n",
              "  16,\n",
              "  'What is the use of emulsion PCR in Next Generation Sequencing?',\n",
              "  'Briefly, amplified libraries carrying unique barcodes are prepared by fluorescent tagging of complementary sequences and then resolved by high-speed flow cytometric sorting of labeled emulsion PCR beads.',\n",
              "  0),\n",
              " (1625,\n",
              "  17,\n",
              "  'What is the use of emulsion PCR in Next Generation Sequencing?',\n",
              "  'The protocol is simple and provides an even sequence distribution of multiplex libraries when sequencing the flow-sorted beads.',\n",
              "  0),\n",
              " (1625,\n",
              "  18,\n",
              "  'What is the use of emulsion PCR in Next Generation Sequencing?',\n",
              "  'Moreover, since many empty and mixed emulsion PCR beads are removed, the approach gives rise to a substantial increase in sequence quality and mean read length, as compared to that obtained by standard enrichment protocols.',\n",
              "  0),\n",
              " (1625,\n",
              "  19,\n",
              "  'What is the use of emulsion PCR in Next Generation Sequencing?',\n",
              "  'Based on the mechanism of emulsion PCR, a unique DNA template would only generate a unique sequence read after being amplified and sequenced on GS FLX.',\n",
              "  0),\n",
              " (1625,\n",
              "  20,\n",
              "  'What is the use of emulsion PCR in Next Generation Sequencing?',\n",
              "  'Prior to SOLiD sequencing reaction, the libraries were amplified with emulsion PCR and enriched with the P2 enrichment beads.',\n",
              "  1),\n",
              " (1625,\n",
              "  21,\n",
              "  'What is the use of emulsion PCR in Next Generation Sequencing?',\n",
              "  'The library samples were loaded to sequencing Chip for Work Flow Analysis (WFA) or sequencing running with default parameters.',\n",
              "  1),\n",
              " (962,\n",
              "  0,\n",
              "  'What is DeepCAGE?',\n",
              "  \"The cap analysis of gene expression (CAGE) technology has been established to detect transcriptional starting sites (TSSs) and expression levels by utilizing 5' cDNA tags and PCR.\",\n",
              "  1),\n",
              " (962,\n",
              "  1,\n",
              "  'What is DeepCAGE?',\n",
              "  'It has been reported that the amount of templates is proportional to the amplification efficiency of PCR.',\n",
              "  1),\n",
              " (962,\n",
              "  2,\n",
              "  'What is DeepCAGE?',\n",
              "  'CAGE has been used as a key technique for analyzing promoter activity and finding new transcripts including alternative spliced products and noncoding transcripts.',\n",
              "  1),\n",
              " (962,\n",
              "  3,\n",
              "  'What is DeepCAGE?',\n",
              "  'Here, we introduce more powerful tools such as deepCAGE, which can be utilized for high-throughput next-generation sequencing technology.',\n",
              "  0),\n",
              " (962,\n",
              "  4,\n",
              "  'What is DeepCAGE?',\n",
              "  'DeepCAGE can produce much deeper transcriptome datasets and can reveal more details of the regulatory network.',\n",
              "  1),\n",
              " (962,\n",
              "  5,\n",
              "  'What is DeepCAGE?',\n",
              "  \"Here, we present a new method for high-throughput sequencing of 5' cDNA tags-DeepCAGE: merging the Cap Analysis of Gene Expression method with ultra-high-throughput sequence technology.\",\n",
              "  0),\n",
              " (962,\n",
              "  6,\n",
              "  'What is DeepCAGE?',\n",
              "  'DeepCAGE can also detect promoters used only in a small subset of cells within the complex tissue',\n",
              "  0),\n",
              " (962,\n",
              "  7,\n",
              "  'What is DeepCAGE?',\n",
              "  'DeepCAGE can produce much deeper transcriptome datasets and can reveal more details of the regulatory network.',\n",
              "  1),\n",
              " (962,\n",
              "  8,\n",
              "  'What is DeepCAGE?',\n",
              "  'DeepCAGE can also detect promoters used only in a small subset of cells within the complex tissue.',\n",
              "  0),\n",
              " (962,\n",
              "  9,\n",
              "  'What is DeepCAGE?',\n",
              "  'Here, we introduce more powerful tools such as deepCAGE, which can be utilized for high-throughput next-generation sequencing technology.',\n",
              "  0),\n",
              " (962,\n",
              "  10,\n",
              "  'What is DeepCAGE?',\n",
              "  'Mammalian cells have at least 5-10 magnitudes more TSS than previously believed, and deeper sequencing is necessary to detect all active promoters in a given tissue.',\n",
              "  0),\n",
              " (962,\n",
              "  11,\n",
              "  'What is DeepCAGE?',\n",
              "  \"Here, we present a new method for high-throughput sequencing of 5' cDNA tags-DeepCAGE: merging the Cap Analysis of Gene Expression method with ultra-high-throughput sequence technology.\",\n",
              "  0),\n",
              " (962,\n",
              "  12,\n",
              "  'What is DeepCAGE?',\n",
              "  'Here, we introduce more powerful tools such as deepCAGE, which can be utilized for high-throughput next-generation sequencing technology.',\n",
              "  0),\n",
              " (962,\n",
              "  13,\n",
              "  'What is DeepCAGE?',\n",
              "  'Using these data, we present evidence indicating a key role for the Arnt2 transcription factor in hippocampus gene regulation.',\n",
              "  0),\n",
              " (962,\n",
              "  14,\n",
              "  'What is DeepCAGE?',\n",
              "  'DeepCAGE can also detect promoters used only in a small subset of cells within the complex tissue.',\n",
              "  0),\n",
              " (962,\n",
              "  15,\n",
              "  'What is DeepCAGE?',\n",
              "  'Here, we introduce more powerful tools such as deepCAGE, which can be utilized for high-throughput next-generation sequencing technology.',\n",
              "  0),\n",
              " (962,\n",
              "  16,\n",
              "  'What is DeepCAGE?',\n",
              "  'DeepCAGE can produce much deeper transcriptome datasets and can reveal more details of the regulatory network.',\n",
              "  0),\n",
              " (2212,\n",
              "  0,\n",
              "  'List clinical features of the IMAGe syndrome.',\n",
              "  'Recently, our group described gain-of-function mutations in the PCNA-binding site of CDKN1C that result in an undergrowth syndrome called IMAGe Syndrome (Intrauterine Growth Restriction, Metaphyseal dysplasia, Adrenal hypoplasia, and Genital anomalies), with life-threatening consequences.',\n",
              "  1),\n",
              " (2212,\n",
              "  1,\n",
              "  'List clinical features of the IMAGe syndrome.',\n",
              "  'IMAGe syndrome is an exceedingly rare condition first described in 1999.',\n",
              "  1),\n",
              " (2212,\n",
              "  2,\n",
              "  'List clinical features of the IMAGe syndrome.',\n",
              "  'Components of the syndrome are intrauterine growth retardation (IUGR), metaphyseal dysplasia, congenital adrenal hypoplasia and genital anomalies.',\n",
              "  1),\n",
              " (2212,\n",
              "  3,\n",
              "  'List clinical features of the IMAGe syndrome.',\n",
              "  'OBJECTIVE: Arboleda et al.',\n",
              "  0),\n",
              " (2212,\n",
              "  4,\n",
              "  'List clinical features of the IMAGe syndrome.',\n",
              "  'have recently shown that IMAGe (intra-uterine growth restriction, metaphyseal dysplasia, adrenal hypoplasia congenita and genital abnormalities) syndrome is caused by gain-of-function mutations of maternally expressed gene CDKN1C on chromosome 11p15.5.',\n",
              "  1),\n",
              " (2212,\n",
              "  5,\n",
              "  'List clinical features of the IMAGe syndrome.',\n",
              "  'Intriguing is that CDKN1C gain-of-function variations were recently found in patients with IMAGe syndrome (intrauterine growth restriction, metaphyseal dysplasia, congenital adrenal hypoplasia, and genital anomalies).',\n",
              "  1),\n",
              " (2139,\n",
              "  0,\n",
              "  'Which histone mutations have been associated with pediatric gliomas?',\n",
              "  'Detecting the H3F3A mutant allele found in high-grade pediatric glioma by real-time PCR',\n",
              "  0),\n",
              " (2139,\n",
              "  1,\n",
              "  'Which histone mutations have been associated with pediatric gliomas?',\n",
              "  ' It has been reported recently that about 80% of DIPG cases and 70% of midline glioblastomas contain a mutation at one allele of the H3F3A gene (encoding histone H3 variant H3.3), replacing the lysine 27 with methionine (K27M).',\n",
              "  0),\n",
              " (2139,\n",
              "  2,\n",
              "  'Which histone mutations have been associated with pediatric gliomas?',\n",
              "  'In order to facilitate diagnosis of DIPG patients, a quick and reliable method to identify the H3F3A K27M mutation is needed.',\n",
              "  0),\n",
              " (2139,\n",
              "  3,\n",
              "  'Which histone mutations have been associated with pediatric gliomas?',\n",
              "  'Using this optimized real-time PCR assay, we analyzed eleven samples, two of which containing H3F3A K27M mutation, and found that these two samples were differentially amplified from the nine others.',\n",
              "  0),\n",
              " (2139,\n",
              "  4,\n",
              "  'Which histone mutations have been associated with pediatric gliomas?',\n",
              "  'we were able to discern the H3F3A K27M mutation in a newly obtained pediatric brainstem glioblastoma sample whose H3.3 status was not known previously, and in three other DIPG samples as well as paraffin embedded samples.',\n",
              "  0),\n",
              " (2139,\n",
              "  5,\n",
              "  'Which histone mutations have been associated with pediatric gliomas?',\n",
              "  'These results demonstrate that we have developed a new reliable procedure for detecting the H3F3A K27M mutation in pediatric glioblastoma patient samples.',\n",
              "  0),\n",
              " (2139,\n",
              "  6,\n",
              "  'Which histone mutations have been associated with pediatric gliomas?',\n",
              "  'Moreover, approximately 30% of pediatric high grade gliomas (pedHGG) including GBM and DIPG harbor a lysine 27 mutation (K27M) in histone 3.3 (H3.3) which is correlated with poor outcome and was shown to influence EZH2 function',\n",
              "  1),\n",
              " (2139,\n",
              "  7,\n",
              "  'Which histone mutations have been associated with pediatric gliomas?',\n",
              "  'A lesson learned from the H3.3K27M mutation found in pediatric glioma: a new approach to the study of the function of histone modifications in vivo?',\n",
              "  0),\n",
              " (2139,\n",
              "  8,\n",
              "  'Which histone mutations have been associated with pediatric gliomas?',\n",
              "  'Recent studies on high-grade pediatric GBM have identified two recurrent mutations (K27M and G34R/V) in genes encoding histone H3 (H3F3A for H3.3 and HIST1H3B for H3.1)',\n",
              "  1),\n",
              " (2139,\n",
              "  9,\n",
              "  'Which histone mutations have been associated with pediatric gliomas?',\n",
              "  'Genome-wide studies using ChIP-seq on H3.3K27M patient samples indicate a global reduction of H3K27me3 on chromatin.',\n",
              "  0),\n",
              " (2139,\n",
              "  10,\n",
              "  'Which histone mutations have been associated with pediatric gliomas?',\n",
              "  'Remarkably, we also found a dramatic enrichment of H3K27me3 and EZH2 (the catalytic subunit H3K27 methyltransferase) at hundreds of gene loci in H3.3K27M patient cells',\n",
              "  0),\n",
              " (2139,\n",
              "  11,\n",
              "  'Which histone mutations have been associated with pediatric gliomas?',\n",
              "  'Two new studies show that the known histone H3 alteration p.Lys27Met in pediatric glioma leads to globally diminished trimethylation at histone H3 lysine 27.',\n",
              "  0),\n",
              " (2139,\n",
              "  12,\n",
              "  'Which histone mutations have been associated with pediatric gliomas?',\n",
              "  'The histone H3.3K27M mutation in pediatric glioma reprograms H3K27 methylation and gene expression.',\n",
              "  0),\n",
              " (2139,\n",
              "  13,\n",
              "  'Which histone mutations have been associated with pediatric gliomas?',\n",
              "  'Recent studies have identified a Lys 27-to-methionine (K27M) mutation at one allele of H3F3A, one of the two genes encoding histone H3 variant H3.3, in 60% of high-grade pediatric glioma cases.',\n",
              "  1),\n",
              " (2139,\n",
              "  14,\n",
              "  'Which histone mutations have been associated with pediatric gliomas?',\n",
              "  'These results indicate that H3.3K27M mutation reprograms epigenetic landscape and gene expression, which may drive tumorigenesis.',\n",
              "  0),\n",
              " (2139,\n",
              "  15,\n",
              "  'Which histone mutations have been associated with pediatric gliomas?',\n",
              "  'Here, we discuss vaccine treatment for children diagnosed with malignant glioma, through targeting EphA2, IL-13Rα2 and/or histone H3 K27M, while in adults, treatments with RINTEGA, Prophage Series G-100 and dendritic cells are explored.',\n",
              "  0),\n",
              " (2139,\n",
              "  16,\n",
              "  'Which histone mutations have been associated with pediatric gliomas?',\n",
              "  'Histone H3 lysine(27)-to-methionine (H3K27M) gain-of-function mutations occur in highly aggressive pediatric gliomas.',\n",
              "  0),\n",
              " (2139,\n",
              "  17,\n",
              "  'Which histone mutations have been associated with pediatric gliomas?',\n",
              "  'Sequencing of pediatric gliomas has identified missense mutations Lys27Met (K27M) and Gly34Arg/Val (G34R/V) in genes encoding histone H3.3 (H3F3A) and H3.1 (HIST3H1B).',\n",
              "  0),\n",
              " (2139,\n",
              "  18,\n",
              "  'Which histone mutations have been associated with pediatric gliomas?',\n",
              "  'Over 70% of diffuse intrinsic pediatric gliomas, an aggressive brainstem tumor, harbor heterozygous mutations that create a K27M amino acid substitution (methionine replaces lysine 27) in the tail of histone H3.3.',\n",
              "  0),\n",
              " (2139,\n",
              "  19,\n",
              "  'Which histone mutations have been associated with pediatric gliomas?',\n",
              "  'Recently, sequencing of tumor cells revealed that histone H3 is frequently mutated in pediatric HGG, with up to 78\\xa0% of diffuse intrinsic pontine gliomas (DIPGs) carrying K27M and 36\\xa0% of non-brainstem gliomas carrying either K27M or G34R/V mutations.',\n",
              "  0),\n",
              " (2139,\n",
              "  20,\n",
              "  'Which histone mutations have been associated with pediatric gliomas?',\n",
              "  'Use of human embryonic stem cells to model pediatric gliomas with H3.3K27M histone mutation.',\n",
              "  0),\n",
              " (2139,\n",
              "  21,\n",
              "  'Which histone mutations have been associated with pediatric gliomas?',\n",
              "  'Exon sequencing has identified a mutation in K27M of the histone H3.3 gene (H3F3A K27M and G34R/V) in about 20% of pediatric glioblastomas, but it remains to be seen whether these mutations can be considered specific for pediatric diffuse high-grade astrocytomas or also occur in other pediatric brain tumors.',\n",
              "  0),\n",
              " (2139,\n",
              "  22,\n",
              "  'Which histone mutations have been associated with pediatric gliomas?',\n",
              "  'K27M mutation in histone H3.3 defines clinically and biologically distinct subgroups of pediatric diffuse intrinsic pontine gliomas.',\n",
              "  0),\n",
              " (2139,\n",
              "  23,\n",
              "  'Which histone mutations have been associated with pediatric gliomas?',\n",
              "  'Recently, sequencing of tumor cells revealed that histone H3 is frequently mutated in pediatric HGG, with up to 78\\xa0% of diffuse intrinsic pontine gliomas (DIPGs) carrying K27M and 36\\xa0% of non-brainstem gliomas carrying either K27M or G34R/V mutations',\n",
              "  0),\n",
              " (2139,\n",
              "  24,\n",
              "  'Which histone mutations have been associated with pediatric gliomas?',\n",
              "  'Recent sequencing studies have shown that ~30\\xa0% of pediatric GBM and ~80\\xa0% of diffuse intrinsic pontine gliomas show K27M mutations in the H3F3A gene, a variant encoding histone H3.3',\n",
              "  0),\n",
              " (2139,\n",
              "  25,\n",
              "  'Which histone mutations have been associated with pediatric gliomas?',\n",
              "  'Histone H3 lysine(27)-to-methionine (H3K27M) gain-of-function mutations occur in highly aggressive pediatric gliomas',\n",
              "  0),\n",
              " (2139,\n",
              "  26,\n",
              "  'Which histone mutations have been associated with pediatric gliomas?',\n",
              "  'Sequencing of pediatric gliomas has identified missense mutations Lys27Met (K27M) and Gly34Arg/Val (G34R/V) in genes encoding histone H3.3 (H3F3A) and H3.1 (HIST3H1B)',\n",
              "  0),\n",
              " (2139,\n",
              "  27,\n",
              "  'Which histone mutations have been associated with pediatric gliomas?',\n",
              "  'Over 70% of diffuse intrinsic pediatric gliomas, an aggressive brainstem tumor, harbor heterozygous mutations that create a K27M amino acid substitution (methionine replaces lysine 27) in the tail of histone H3.3',\n",
              "  0),\n",
              " (2139,\n",
              "  28,\n",
              "  'Which histone mutations have been associated with pediatric gliomas?',\n",
              "  'Exon sequencing has identified a mutation in K27M of the histone H3.3 gene (H3F3A K27M and G34R/V) in about 20% of pediatric glioblastomas, but it remains to be seen whether these mutations can be considered specific for pediatric diffuse high-grade astrocytomas or also occur in other pediatric brain tumors',\n",
              "  0),\n",
              " (2139,\n",
              "  29,\n",
              "  'Which histone mutations have been associated with pediatric gliomas?',\n",
              "  'Pediatric brainstem gliomas often harbor oncogenic K27M mutation of histone H3.3',\n",
              "  0),\n",
              " (2139,\n",
              "  30,\n",
              "  'Which histone mutations have been associated with pediatric gliomas?',\n",
              "  'K27M mutation in histone H3.3 defines clinically and biologically distinct subgroups of pediatric diffuse intrinsic pontine gliomas',\n",
              "  0),\n",
              " (2139,\n",
              "  31,\n",
              "  'Which histone mutations have been associated with pediatric gliomas?',\n",
              "  'Over 70% of diffuse intrinsic pediatric gliomas, an aggressive brainstem tumor, harbor heterozygous mutations that create a K27M amino acid substitution (methionine replaces lysine 27) in the tail of histone H3.3.',\n",
              "  0),\n",
              " (2139,\n",
              "  32,\n",
              "  'Which histone mutations have been associated with pediatric gliomas?',\n",
              "  'Recent studies have identified a Lys 27-to-methionine (K27M) mutation at one allele of H3F3A, one of the two genes encoding histone H3 variant H3.3, in 60% of high-grade pediatric glioma cases.',\n",
              "  1),\n",
              " (2139,\n",
              "  33,\n",
              "  'Which histone mutations have been associated with pediatric gliomas?',\n",
              "  'We established a Drosophila animal model for the pathogenic histone H3K27M mutation and show that its overexpression resembles polycomb repressive complex 2 (PRC2) loss-of-function phenotypes, causing derepression of PRC2 target genes and developmental perturbations.',\n",
              "  0),\n",
              " (2139,\n",
              "  34,\n",
              "  'Which histone mutations have been associated with pediatric gliomas?',\n",
              "  'Recent sequencing studies have shown that ~30\\xa0% of pediatric GBM and ~80\\xa0% of diffuse intrinsic pontine gliomas show K27M mutations in the H3F3A gene, a variant encoding histone H3.3.',\n",
              "  0),\n",
              " (2139,\n",
              "  35,\n",
              "  'Which histone mutations have been associated with pediatric gliomas?',\n",
              "  'Evolutionary reconstruction indicates histone 3 (H3) K27M--including H3.2K27M--mutations potentially arise first and are invariably associated with specific, high-fidelity obligate partners throughout the tumour and its spread, from diagnosis to end-stage disease, suggesting mutual need for tumorigenesis.',\n",
              "  0),\n",
              " (2139,\n",
              "  36,\n",
              "  'Which histone mutations have been associated with pediatric gliomas?',\n",
              "  'Moreover, approximately 30% of pediatric high grade gliomas (pedHGG) including GBM and DIPG harbor a lysine 27 mutation (K27M) in histone 3.3 (H3.3) which is correlated with poor outcome and was shown to influence EZH2 function.The present study investigated the correlation of expression of EZH2 and other PRC2 genes (EZH1, SUZ12, EED) with overall survival of pediatric GBM patients and the cytotoxic impact of EZH2 inhibition by the novel agent Tazemetostat in pediatric GBM/DIPG cells harboring either a H3.3 mutation or a H3 wildtype.EZH2 gene expression does not c',\n",
              "  1),\n",
              " (2139,\n",
              "  37,\n",
              "  'Which histone mutations have been associated with pediatric gliomas?',\n",
              "  'Recently, sequencing of tumor cells revealed that histone H3 is frequently mutated in pediatric HGG, with up to 78\\xa0% of diffuse intrinsic pontine gliomas (DIPGs) carrying K27M and 36\\xa0% of non-brainstem gliomas carrying either K27M or G34R/V mutations.',\n",
              "  0),\n",
              " (2139,\n",
              "  38,\n",
              "  'Which histone mutations have been associated with pediatric gliomas?',\n",
              "  'Important insights have also been uncovered in the field of pediatric glioma, including the identification of recurrent mutation, fusion, and/or duplication events of the BRAF, FGFR1, MYB, and MYBL1 genes in pediatric low-grade gliomas, mutations affecting histone components (H3F3A p.K27M or p.G34) in pediatric high-grade gliomas, and aggressive subsets developing in midline central nervous system structures.',\n",
              "  0),\n",
              " (2139,\n",
              "  39,\n",
              "  'Which histone mutations have been associated with pediatric gliomas?',\n",
              "  'K27M-H3.3 is universally associated with short survival in DIPG, while patients wild-type for H3.3 show improved survival.',\n",
              "  0),\n",
              " (2139,\n",
              "  40,\n",
              "  'Which histone mutations have been associated with pediatric gliomas?',\n",
              "  'Taken together, our results suggest that mutations disrupting the histone code at H3K36, including H3.3 G34R/V, IDH1 and/or SETD2 mutations, are central to the genesis of hemispheric HGGs in older children and young adults.',\n",
              "  0),\n",
              " (2139,\n",
              "  41,\n",
              "  'Which histone mutations have been associated with pediatric gliomas?',\n",
              "  'Gliomas with H3F3A K27M from pediatric or young adult patients had similar, characteristic DNA methylation profiles.',\n",
              "  0),\n",
              " (2139,\n",
              "  42,\n",
              "  'Which histone mutations have been associated with pediatric gliomas?',\n",
              "  'K27M-H3.3 is universally associated with short survival in DIPG, while patients wild-type for H3.3 show improved survival.',\n",
              "  0),\n",
              " (2139,\n",
              "  43,\n",
              "  'Which histone mutations have been associated with pediatric gliomas?',\n",
              "  'Moreover, the gain of H3K27me3 and Ezh2 at gene promoters alters the expression of genes that are associated with various cancer pathways.',\n",
              "  0),\n",
              " (2139,\n",
              "  44,\n",
              "  'Which histone mutations have been associated with pediatric gliomas?',\n",
              "  'Use of human embryonic stem cells to model pediatric gliomas with H3.3K27M histone mutation.',\n",
              "  0),\n",
              " (2139,\n",
              "  45,\n",
              "  'Which histone mutations have been associated with pediatric gliomas?',\n",
              "  'The histone H3.3K27M mutation in pediatric glioma reprograms H3K27 methylation and gene expression.',\n",
              "  0),\n",
              " (2139,\n",
              "  46,\n",
              "  'Which histone mutations have been associated with pediatric gliomas?',\n",
              "  'K27M mutation in histone H3.3 defines clinically and biologically distinct subgroups of pediatric diffuse intrinsic pontine gliomas.',\n",
              "  0),\n",
              " (2139,\n",
              "  47,\n",
              "  'Which histone mutations have been associated with pediatric gliomas?',\n",
              "  'A lesson learned from the H3.3K27M mutation found in pediatric glioma: a new approach to the study of the function of histone modifications in vivo?',\n",
              "  0),\n",
              " (4066,\n",
              "  0,\n",
              "  'List proteins that promotes calcification.',\n",
              "  'several promoters and inhibitors of calcification (matrix Gla protein (MGP), fibroblast growth factor-23 (FGF-23), matrix metalloproteinases (MMP-2 and -9),',\n",
              "  1),\n",
              " (4066,\n",
              "  1,\n",
              "  'List proteins that promotes calcification.',\n",
              "  \"Primary familial brain calcification (PFBC), widely known as Fahr's disease, is a rare disorder caused by pathogenic variants in SLC20A2, PDGFB, PDGFRB, XPR1, or MYORG genes.\",\n",
              "  1),\n",
              " (4066,\n",
              "  2,\n",
              "  'List proteins that promotes calcification.',\n",
              "  ' calcification-promoting TNAP (tissue nonspecific alkaline phosphatase) activity.',\n",
              "  1),\n",
              " (2372,\n",
              "  0,\n",
              "  'Is there any linear-time and linear-space algorithm for the computation of avoided words in biological sequences?',\n",
              "  'The systematic search for avoided words is particularly useful for biological sequence analysis.',\n",
              "  0),\n",
              " (2372,\n",
              "  1,\n",
              "  'Is there any linear-time and linear-space algorithm for the computation of avoided words in biological sequences?',\n",
              "  'We present a linear-time and linear-space algorithm for the computation of avoided words of lengthkin a given sequencex.',\n",
              "  1),\n",
              " (2372,\n",
              "  2,\n",
              "  'Is there any linear-time and linear-space algorithm for the computation of avoided words in biological sequences?',\n",
              "  'We suggest a modification to this algorithm so that it computes all avoided words ofx, irrespective of their length, within the same time complexity.',\n",
              "  0),\n",
              " (2372,\n",
              "  3,\n",
              "  'Is there any linear-time and linear-space algorithm for the computation of avoided words in biological sequences?',\n",
              "  'We also present combinatorial results with regards to avoided words and absent words.',\n",
              "  0),\n",
              " (2372,\n",
              "  4,\n",
              "  'Is there any linear-time and linear-space algorithm for the computation of avoided words in biological sequences?',\n",
              "  'We present a linear-time and linear-space algorithm for the computation of avoided words of length <br>',\n",
              "  1),\n",
              " (2372,\n",
              "  5,\n",
              "  'Is there any linear-time and linear-space algorithm for the computation of avoided words in biological sequences?',\n",
              "  '<b>BACKGROUND</b>: The deviation of the observed frequency of a word <br><b>RESULTS</b>: In this article, we propose an [Formula: see text]-time and [Formula: see text]-space algorithm to compute all [Formula: see text]-avoided words of length <br><b>CONCLUSIONS</b>: The systematic search for avoided words is particularly useful for biological sequence analysis.',\n",
              "  0),\n",
              " (2372,\n",
              "  6,\n",
              "  'Is there any linear-time and linear-space algorithm for the computation of avoided words in biological sequences?',\n",
              "  'We present a linear-time and linear-space algorithm for the computation of avoided words of length',\n",
              "  1),\n",
              " (2372,\n",
              "  7,\n",
              "  'Is there any linear-time and linear-space algorithm for the computation of avoided words in biological sequences?',\n",
              "  'We present a linear-time and linear-space algorithm for the computation of avoided words of length k in a given sequence x.',\n",
              "  1),\n",
              " (2372,\n",
              "  8,\n",
              "  'Is there any linear-time and linear-space algorithm for the computation of avoided words in biological sequences?',\n",
              "  'Hence, computing all such words naïvely can be a very time-consuming procedure, in particular for large k.   RESULTS In this article, we propose an [Formula: see text]-time and [Formula: see text]-space algorithm to compute all [Formula: see text]-avoided words of length k in a given sequence of length n over a fixed-sized alphabet.',\n",
              "  1),\n",
              " (2372,\n",
              "  9,\n",
              "  'Is there any linear-time and linear-space algorithm for the computation of avoided words in biological sequences?',\n",
              "  'We also present a time-optimal [Formula: see text]-time algorithm to compute all [Formula: see text]-avoided words (of any length) in a sequence of length n over an integer alphabet of size [Formula: see text].',\n",
              "  0),\n",
              " (2372,\n",
              "  10,\n",
              "  'Is there any linear-time and linear-space algorithm for the computation of avoided words in biological sequences?',\n",
              "  'the deviation of the observed frequency of a word from its expected frequency in a given sequence is used to determine whether or not the word is this concept is particularly useful in dna linguistic analysis the value of the deviation of denoted by formula see text effectively characterises the extent of a word by its edge contrast in the context in which it occurs a word of length formula see text is a formula see text avoided word in if formula see text for a given threshold formula see text notice that such a word may be completely from hence computing all such words naïvely can be a very time consuming procedure in particular for large in this article we propose an formula see text time and formula see text space algorithm to compute all formula see text avoided words of length in a given sequence of length over a fixed sized alphabet we also present a time optimal formula see text time algorithm to compute all formula see text avoided words of any length in a sequence of length over an integer alphabet of size formula see text in addition we provide a tight asymptotic upper bound for the number of formula see text avoided words over an integer alphabet and the expected length of the longest one we make available an implementation of our algorithm experimental results using both real and synthetic data show the efficiency and applicability of our implementation in biological sequence analysis the systematic search for avoided words is particularly useful for biological sequence analysis we present a linear time and linear space algorithm for the computation of avoided words of length in a given sequence we suggest a modification to this algorithm so that it computes all avoided words of irrespective of their length within the same time complexity we also present combinatorial results with regards to avoided words and absent words.',\n",
              "  0),\n",
              " (4058,\n",
              "  0,\n",
              "  'Describe the role of epidermal CYLD inactivation in sebaceous and basaloid skin tumors',\n",
              "  'Epidermal CYLD inactivation sensitizes mice to the development of sebaceous and basaloid skin tumors.',\n",
              "  1),\n",
              " (4058,\n",
              "  1,\n",
              "  'Describe the role of epidermal CYLD inactivation in sebaceous and basaloid skin tumors',\n",
              "  'The deubiquitinase-encoding gene Cyld displays a dominant genetic linkage to a wide spectrum of skin-appendage tumors, which could be collectively designated as CYLD mutant-syndrome (CYLDm-syndrome).',\n",
              "  1),\n",
              " (4058,\n",
              "  2,\n",
              "  'Describe the role of epidermal CYLD inactivation in sebaceous and basaloid skin tumors',\n",
              "  'Despite recent advances, little is understood about the molecular mechanisms responsible for this painful and difficult-to-treat skin disease.',\n",
              "  1),\n",
              " (4058,\n",
              "  3,\n",
              "  'Describe the role of epidermal CYLD inactivation in sebaceous and basaloid skin tumors',\n",
              "  'Here, we generated a conditional mouse model with epidermis-targeted expression of a catalytically deficient CYLDm through K14-Cre-mediated deletion of exon 9 (hereafter refer to CyldEΔ9/Δ9 ).',\n",
              "  0),\n",
              " (4058,\n",
              "  4,\n",
              "  'Describe the role of epidermal CYLD inactivation in sebaceous and basaloid skin tumors',\n",
              "  'CyldEΔ9/Δ9 mice were born alive but developed hair and sebaceous gland abnormalities and dental defects at 100% and 60% penetrance, respectively.',\n",
              "  0),\n",
              " (4058,\n",
              "  5,\n",
              "  'Describe the role of epidermal CYLD inactivation in sebaceous and basaloid skin tumors',\n",
              "  'Upon topical challenge with DMBA/TPA, these animals primarily developed sebaceous and basaloid tumors resembling human CYLDm-syndrome as opposed to papilloma, which is most commonly induced in WT mice by this treatment.',\n",
              "  1),\n",
              " (4058,\n",
              "  6,\n",
              "  'Describe the role of epidermal CYLD inactivation in sebaceous and basaloid skin tumors',\n",
              "  'Molecular analysis revealed that TRAF6-K63-Ubiquitination (K63-Ub), c-Myc-K63-Ub, and phospho-c-Myc (S62) were markedly elevated in CyldEΔ9/Δ9 skin.',\n",
              "  0),\n",
              " (4058,\n",
              "  7,\n",
              "  'Describe the role of epidermal CYLD inactivation in sebaceous and basaloid skin tumors',\n",
              "  'Topical treatment with a pharmacological c-Myc inhibitor induced sebaceous and basal cell apoptosis in CyldEΔ9/Δ9 skin.',\n",
              "  0),\n",
              " (4058,\n",
              "  8,\n",
              "  'Describe the role of epidermal CYLD inactivation in sebaceous and basaloid skin tumors',\n",
              "  'Consistently, c-Myc activation was readily detected in human cylindroma and sebaceous adenoma.',\n",
              "  0),\n",
              " (4058,\n",
              "  9,\n",
              "  'Describe the role of epidermal CYLD inactivation in sebaceous and basaloid skin tumors',\n",
              "  'Taken together, our findings demonstrate that CyldEΔ9/Δ9 mice represent a disease-relevant animal model and identify TRAF6 and c-Myc as potential therapeutic targets for CYLDm-syndrome.',\n",
              "  1),\n",
              " (1324,\n",
              "  0,\n",
              "  'What is the effect of the absence of Saccharomyces cerevisiae Rrm3p?',\n",
              "  'Twice as many pause sites were identified in rrm3 compared with wild-type cells, as pausing in this strain occurred at both highly transcribed RNA polymerase II genes and the previously identified protein DNA complexes.',\n",
              "  0),\n",
              " (1324,\n",
              "  1,\n",
              "  'What is the effect of the absence of Saccharomyces cerevisiae Rrm3p?',\n",
              "  'ORFs of highly transcribed RNA polymerase II genes are a class of natural pause sites that are not exacerbated in rrm3 cells',\n",
              "  0),\n",
              " (1324,\n",
              "  2,\n",
              "  'What is the effect of the absence of Saccharomyces cerevisiae Rrm3p?',\n",
              "  'The DNA helicase Rrm3 promotes replication fork progression through >1000 discrete genomic regions and represses the cDNA-mediated mobility of the Ty1 retrotransposon.',\n",
              "  0),\n",
              " (1324,\n",
              "  3,\n",
              "  'What is the effect of the absence of Saccharomyces cerevisiae Rrm3p?',\n",
              "  'We explored the connection between DNA replication and Ty1 retromobility by investigating the basis of increased retromobility in an rrm3 mutant.',\n",
              "  0),\n",
              " (1324,\n",
              "  4,\n",
              "  'What is the effect of the absence of Saccharomyces cerevisiae Rrm3p?',\n",
              "  'Even though Ty1 cDNA levels are increased in the absence of RRM3, neither the level nor target-site specificity of cDNA integration was altered',\n",
              "  0),\n",
              " (1324,\n",
              "  5,\n",
              "  'What is the effect of the absence of Saccharomyces cerevisiae Rrm3p?',\n",
              "  'We propose that RNA:DNA hybrid regions within nascent retrotransposition events block replication in an rrm3 mutant, leading to chromosome breaks within Ty1 sequences',\n",
              "  0),\n",
              " (1324,\n",
              "  6,\n",
              "  'What is the effect of the absence of Saccharomyces cerevisiae Rrm3p?',\n",
              "  'We demonstrate that the inefficient mtDNA replication process of mutant yeast cells lacking the PIF1 DNA helicase is partly rescued in the absence of the DNA helicase RRM3.',\n",
              "  0),\n",
              " (1324,\n",
              "  7,\n",
              "  'What is the effect of the absence of Saccharomyces cerevisiae Rrm3p?',\n",
              "  'The rescue effect is likely due to the increase in the deoxynucleoside triphosphates (dNTPs) pool caused by the lack of RRM3',\n",
              "  0),\n",
              " (1324,\n",
              "  8,\n",
              "  'What is the effect of the absence of Saccharomyces cerevisiae Rrm3p?',\n",
              "  ' However, the essential nuclear function of Pfh1p could be supplied by Rrm3p.',\n",
              "  0),\n",
              " (1324,\n",
              "  9,\n",
              "  'What is the effect of the absence of Saccharomyces cerevisiae Rrm3p?',\n",
              "  'Expression of Rrm3p suppressed the accumulation of DNA damage foci but not the hydroxyurea sensitivity of cells depleted of nuclear Pfh1p.',\n",
              "  0),\n",
              " (1324,\n",
              "  10,\n",
              "  'What is the effect of the absence of Saccharomyces cerevisiae Rrm3p?',\n",
              "  'Together, these data demonstrate that Pfh1p has essential roles in the replication of both nuclear and mitochondrial DNA',\n",
              "  0),\n",
              " (1324,\n",
              "  11,\n",
              "  'What is the effect of the absence of Saccharomyces cerevisiae Rrm3p?',\n",
              "  ' In the absence of the Rrm3p helicase, there was a slight enhancement of fork arrest at the Ter sites.',\n",
              "  0),\n",
              " (1324,\n",
              "  12,\n",
              "  'What is the effect of the absence of Saccharomyces cerevisiae Rrm3p?',\n",
              "  'Simultaneous deletions of the TOF1 (or CSM3), and the RRM3 genes restored fork arrest by removing both the fork-releasing and fork-protection activities',\n",
              "  0),\n",
              " (1324,\n",
              "  13,\n",
              "  'What is the effect of the absence of Saccharomyces cerevisiae Rrm3p?',\n",
              "  \"Rrm3p is a 5'-to-3' DNA helicase that helps replication forks traverse protein-DNA complexes.\",\n",
              "  0),\n",
              " (1324,\n",
              "  14,\n",
              "  'What is the effect of the absence of Saccharomyces cerevisiae Rrm3p?',\n",
              "  'Its absence leads to increased fork stalling and breakage at over 1,000 specific sites located throughout the Saccharomyces cerevisiae genome',\n",
              "  0),\n",
              " (1324,\n",
              "  15,\n",
              "  'What is the effect of the absence of Saccharomyces cerevisiae Rrm3p?',\n",
              "  'These data suggest a model in which the stalled and broken forks generated in rrm3 cells activate a checkpoint response that provides time for fork repair and restart',\n",
              "  0),\n",
              " (1324,\n",
              "  16,\n",
              "  'What is the effect of the absence of Saccharomyces cerevisiae Rrm3p?',\n",
              "  'Lack of the yeast Rrm3p DNA helicase causes replication defects at multiple sites within ribosomal DNA (rDNA), including at the replication fork barrier (RFB)',\n",
              "  0),\n",
              " (1324,\n",
              "  17,\n",
              "  'What is the effect of the absence of Saccharomyces cerevisiae Rrm3p?',\n",
              "  \"The Saccharomyces cerevisiae RRM3 gene encodes a 5' to 3' DNA helicase.\",\n",
              "  1),\n",
              " (1324,\n",
              "  18,\n",
              "  'What is the effect of the absence of Saccharomyces cerevisiae Rrm3p?',\n",
              "  'While replication of most of the yeast genome was not dependent upon Rrm3p, in its absence, replication forks paused and often broke at an estimated 1400 discrete sites, including tRNA genes, centromeres, inactive replication origins, and transcriptional silencers.',\n",
              "  1),\n",
              " (1324,\n",
              "  19,\n",
              "  'What is the effect of the absence of Saccharomyces cerevisiae Rrm3p?',\n",
              "  'These replication defects were associated with activation of the intra-S phase checkpoint.',\n",
              "  1),\n",
              " (1324,\n",
              "  20,\n",
              "  'What is the effect of the absence of Saccharomyces cerevisiae Rrm3p?',\n",
              "  'Activation of the checkpoint was critical for viability of rrm3Delta cells, especially at low temperatures',\n",
              "  1),\n",
              " (1324,\n",
              "  21,\n",
              "  'What is the effect of the absence of Saccharomyces cerevisiae Rrm3p?',\n",
              "  'These data indicate that the Rrm3p DNA helicase helps replication forks traverse protein-DNA complexes, naturally occurring impediments that are encountered in each S phase',\n",
              "  0),\n",
              " (1324,\n",
              "  22,\n",
              "  'What is the effect of the absence of Saccharomyces cerevisiae Rrm3p?',\n",
              "  'In wild-type Saccharomyces cerevisiae, replication forks slowed during their passage through telomeric C(1-3)A/TG(1-3) tracts.',\n",
              "  0),\n",
              " (1324,\n",
              "  23,\n",
              "  'What is the effect of the absence of Saccharomyces cerevisiae Rrm3p?',\n",
              "  \"This slowing was greatly exacerbated in the absence of RRM3, shown here to encode a 5' to 3' DNA helicase\",\n",
              "  1),\n",
              " (1324,\n",
              "  24,\n",
              "  'What is the effect of the absence of Saccharomyces cerevisiae Rrm3p?',\n",
              "  ' Loss of Rrm3p also resulted in replication fork pausing at specific sites in subtelomeric DNA, such as at inactive replication origins, and at internal tracts of C(1-3)A/TG(1-3) DNA',\n",
              "  0),\n",
              " (1324,\n",
              "  25,\n",
              "  'What is the effect of the absence of Saccharomyces cerevisiae Rrm3p?',\n",
              "  'Local chromatin structure at the ribosomal DNA causes replication fork pausing and genome instability in the absence of the S. cerevisiae DNA helicase Rrm3p.',\n",
              "  0),\n",
              " (1697,\n",
              "  0,\n",
              "  'Which mitochondrial genes are regulated by thyroid hormone?',\n",
              "  'We used a polymerase chain reaction (PCR)-based mRNA differential display (DD) analysis to obtain a profile of thyroid hormone-responsive genes in osteoblast-like cells',\n",
              "  0),\n",
              " (1697,\n",
              "  1,\n",
              "  'Which mitochondrial genes are regulated by thyroid hormone?',\n",
              "  'At the 2-hour time point, 1 true-positive novel clone was identified and shown to be the mitochondrial gene, subunit 6 of ATP synthase (ATPase-6).',\n",
              "  1),\n",
              " (1697,\n",
              "  2,\n",
              "  'Which mitochondrial genes are regulated by thyroid hormone?',\n",
              "  'We studied the coordination of the two genomes by measuring transcript levels for nuclear (IV, Va, and VIc) and mitochondrial (II and III) subunits of cytochrome-c oxidase after altering the mitochondrial content of rat muscle and liver by altering the thyroid state of the animals.',\n",
              "  1),\n",
              " (1697,\n",
              "  3,\n",
              "  'Which mitochondrial genes are regulated by thyroid hormone?',\n",
              "  'Tissue levels of these mRNAs were generally decreased in hypothyroid animals and were up-regulated again after thyroid hormone (T3) treatment.',\n",
              "  1),\n",
              " (1697,\n",
              "  4,\n",
              "  'Which mitochondrial genes are regulated by thyroid hormone?',\n",
              "  'Following this procedure, we now report the identification of the mitochondrial NADH dehydrogenase subunit 3 (ND3) gene as target of thyroid hormone.',\n",
              "  1),\n",
              " (1697,\n",
              "  5,\n",
              "  'Which mitochondrial genes are regulated by thyroid hormone?',\n",
              "  'Sequencing and electrophoretic mobility shift assays confirmed the presence of a thyroid hormone receptor (TR)/c-erbA specific binding site in the mitochondrial ND3 gene.',\n",
              "  1),\n",
              " (3535,\n",
              "  0,\n",
              "  'What is RiboTag profiling?',\n",
              "  'RiboTag is a flexible tool for measuring the translational state of targeted cells in heterogeneous cell cultures.',\n",
              "  1),\n",
              " (3535,\n",
              "  1,\n",
              "  'What is RiboTag profiling?',\n",
              "  'Here we describe our adaptation of a ribosomal capture strategy that was designed to be used in transgenic mice expressing tagged ribosomal subunits (RiboTag) in specific cell types, thereby allowing measurement of translating RNAs from desired cell types within complex tissues.',\n",
              "  1),\n",
              " (3535,\n",
              "  2,\n",
              "  'What is RiboTag profiling?',\n",
              "  'Using this strategy we were able to isolate and analyze neuron-specific RNA despite the presence of glia by co-transfecting experimental plasmids with plasmids that selectively express RiboTag in neurons.',\n",
              "  0),\n",
              " (3535,\n",
              "  3,\n",
              "  'What is RiboTag profiling?',\n",
              "  'RiboTag immunoprecipitation was capable of recovering high integrity RNA from small numbers of transfected cells that can then be interrogated by a variety of methods (e.g., RT-qPCR, PCR array, RNA-Seq) and compared with basal RNA expression of the entire culture.',\n",
              "  1),\n",
              " (3535,\n",
              "  4,\n",
              "  'What is RiboTag profiling?',\n",
              "  'Additionally, we demonstrate how co-transfection of RiboTag with small hairpin RNA (shRNA) constructs can validate and accurately assess the degree of gene expression knockdown, and how RiboTag can be used to measure receptor-mediated gene regulation with transiently expressed designer receptors exclusively activated by designer drugs (DREADDs).',\n",
              "  1),\n",
              " (3535,\n",
              "  5,\n",
              "  'What is RiboTag profiling?',\n",
              "  'RiboTag co-transfection represents a convenient and powerful tool to isolate RNA from a specific subset of cultured cells with a variety of applications for experiments in vitro.',\n",
              "  1),\n",
              " (2993,\n",
              "  0,\n",
              "  'Which deep learning algorithm has been developed for variant calling?',\n",
              "  'Despite rapid advances in sequencing technologies, accurately calling genetic variants present in an individual genome from billions of short, errorful sequence reads remains challenging.',\n",
              "  1),\n",
              " (2993,\n",
              "  1,\n",
              "  'Which deep learning algorithm has been developed for variant calling?',\n",
              "  'Here we show that a deep convolutional neural network can call genetic variation in aligned next-generation sequencing read data by learning statistical relationships between images of read pileups around putative variant and true genotype calls.',\n",
              "  1),\n",
              " (2993,\n",
              "  2,\n",
              "  'Which deep learning algorithm has been developed for variant calling?',\n",
              "  'The approach, called DeepVariant, outperforms existing state-of-the-art tools.',\n",
              "  1),\n",
              " (2993,\n",
              "  3,\n",
              "  'Which deep learning algorithm has been developed for variant calling?',\n",
              "  'The learned model generalizes across genome builds and mammalian species, allowing nonhuman sequencing projects to benefit from the wealth of human ground-truth data.',\n",
              "  1),\n",
              " (2993,\n",
              "  4,\n",
              "  'Which deep learning algorithm has been developed for variant calling?',\n",
              "  'We further show that DeepVariant can learn to call variants in a variety of sequencing technologies and experimental designs, including deep whole genomes from 10X Genomics and Ion Ampliseq exomes, highlighting the benefits of using more automated and generalizable techniques for variant calling.',\n",
              "  1),\n",
              " (435,\n",
              "  0,\n",
              "  'What is the molecular function of psoralen photobinding on DNA?',\n",
              "  'The interaction of two water-soluble furocoumarins, 8-(omega-diethyl aminopropyloxy)psoralen hydrochloride (I) and its 5-isomer (II), with DNA has been investigated by spectroscopic, equilibrium dialysis, hydrodynamic and chiroptical techniques.',\n",
              "  1),\n",
              " (435,\n",
              "  1,\n",
              "  'What is the molecular function of psoralen photobinding on DNA?',\n",
              "  'Both compounds intercalate into the polynucleotide double helix.',\n",
              "  1),\n",
              " (435,\n",
              "  2,\n",
              "  'What is the molecular function of psoralen photobinding on DNA?',\n",
              "  'Both compounds bind very efficiently to DNA, the extent of this process being modulated by the nature of substituents at position 8',\n",
              "  1),\n",
              " (435,\n",
              "  3,\n",
              "  'What is the molecular function of psoralen photobinding on DNA?',\n",
              "  \"We have described an exonuclease III/photoreversal procedure to map, with base pair resolution, the bases which have photoreacted with 4,5',8-trimethylpsoralen (Me3-psoralen) forming either monoadducts or interstrand cross-links in DNA (20)\",\n",
              "  0),\n",
              " (435,\n",
              "  4,\n",
              "  'What is the molecular function of psoralen photobinding on DNA?',\n",
              "  'Psoralen and light treatment removed negative superhelical turns, and extensive treatments failed to produce positive superhelical turns in covalently closed plasmid DNA.',\n",
              "  0),\n",
              " (435,\n",
              "  5,\n",
              "  'What is the molecular function of psoralen photobinding on DNA?',\n",
              "  'Like psoralen, these compounds form a molecular complex with DNA, undergoing intercalation inside the double helix of the macromolecule',\n",
              "  1),\n",
              " (435,\n",
              "  6,\n",
              "  'What is the molecular function of psoralen photobinding on DNA?',\n",
              "  'The crystal and molecular structure of 4,4&apos;,5&apos;-trimethylazapsoralen, obtained by X ray diffraction, was also reported.',\n",
              "  0),\n",
              " (435,\n",
              "  7,\n",
              "  'What is the molecular function of psoralen photobinding on DNA?',\n",
              "  'Like psoralen, these compounds form a molecular complex with DNA, undergoing intercalation inside the double helix of the macromolecule.',\n",
              "  1),\n",
              " (1273,\n",
              "  0,\n",
              "  'What is Cerebral Cavernous Malformation?',\n",
              "  'Cerebral cavernous malformation (CCM) is a disease of the central nervous system causing hemorrhage-prone multiple lumen vascular malformations and very severe neurological consequences',\n",
              "  1),\n",
              " (1273,\n",
              "  1,\n",
              "  'What is Cerebral Cavernous Malformation?',\n",
              "  'Cerebral cavernous malformations (CMs) are vascular malformations of the central nervous system, which can be detected in the absence of any clinical symptoms.',\n",
              "  1),\n",
              " (1273,\n",
              "  2,\n",
              "  'What is Cerebral Cavernous Malformation?',\n",
              "  'Nodules and cysts with mixed signal intensity and a peripheral hemosiderin rim are considered brain magnetic resonance imaging (MRI) findings typical of CMs',\n",
              "  0),\n",
              " (1273,\n",
              "  3,\n",
              "  'What is Cerebral Cavernous Malformation?',\n",
              "  'Cerebral cavernous malformations (CCM) are enlarged vascular lesions affecting 0.1-0.5% of the population worldwide and causing hemorrhagic strokes, seizures, and neurological deficits',\n",
              "  0),\n",
              " (1273,\n",
              "  4,\n",
              "  'What is Cerebral Cavernous Malformation?',\n",
              "  'Although most cerebral cavernous malformations showed evolution of hemorrhage or no change in size or shape on follow-up MR images, cerebral cavernous malformations represented as mixture of subacute and chronic hemorrhage with hemosiderin rim (type II) have a higher frequency to rebleed than other types of cerebral cavernous malformations.',\n",
              "  0),\n",
              " (1273,\n",
              "  5,\n",
              "  'What is Cerebral Cavernous Malformation?',\n",
              "  'Mutations in Krev1 interaction trapped gene 1 (KRIT1) cause cerebral cavernous malformation, an autosomal dominant disease featuring malformation of cerebral capillaries resulting in cerebral hemorrhage, strokes, and seizures.',\n",
              "  0),\n",
              " (1273,\n",
              "  6,\n",
              "  'What is Cerebral Cavernous Malformation?',\n",
              "  'Cerebral cavernous malformation is a vascular disease of the brain causing headaches, seizures, and cerebral hemorrhage.',\n",
              "  1),\n",
              " (1273,\n",
              "  7,\n",
              "  'What is Cerebral Cavernous Malformation?',\n",
              "  'Cerebral cavernous malformation (CCM) is a common vascular disease in central nervous system that frequently predisposes to stroke, seizure, and cerebral hemorrhage',\n",
              "  0),\n",
              " (1273,\n",
              "  8,\n",
              "  'What is Cerebral Cavernous Malformation?',\n",
              "  'Cerebral cavernous malformation (CCM) is a disease of the central nervous system causing hemorrhage-prone multiple lumen vascular malformations and very severe neurological consequences.',\n",
              "  1),\n",
              " (1273,\n",
              "  9,\n",
              "  'What is Cerebral Cavernous Malformation?',\n",
              "  'Although most cerebral cavernous malformations showed evolution of hemorrhage or no change in size or shape on follow-up MR images, cerebral cavernous malformations represented as mixture of subacute and chronic hemorrhage with hemosiderin rim (type II) have a higher frequency to rebleed than other types of cerebral cavernous malformations.',\n",
              "  0),\n",
              " (1273,\n",
              "  10,\n",
              "  'What is Cerebral Cavernous Malformation?',\n",
              "  'Cerebral cavernous malformations represented as hemosiderin deposition without central core (type IV) have a lower tendency to rebleed than other types and do not need any treatment.',\n",
              "  0),\n",
              " (1273,\n",
              "  11,\n",
              "  'What is Cerebral Cavernous Malformation?',\n",
              "  'Adverse reaction of irradiation was observed in five of 22 patients treated with gamma knife surgery.',\n",
              "  0),\n",
              " (1273,\n",
              "  12,\n",
              "  'What is Cerebral Cavernous Malformation?',\n",
              "  'Although most cerebral cavernous malformations showed evolution of hemorrhage or no change in size or shape on follow-up MR images, cerebral cavernous malformations represented as mixture of subacute and chronic hemorrhage with hemosiderin rim (type II) have a higher frequency to rebleed than other types of cerebral cavernous malformations.',\n",
              "  0),\n",
              " (1273,\n",
              "  13,\n",
              "  'What is Cerebral Cavernous Malformation?',\n",
              "  'Cerebral cavernous malformations often remain clinically silent until a mutation carrier suffers a stroke or seizure.',\n",
              "  0),\n",
              " (1273,\n",
              "  14,\n",
              "  'What is Cerebral Cavernous Malformation?',\n",
              "  'Presymptomatic genetic testing has been valuable to follow and manage cerebral cavernous malformation mutation carriers.',\n",
              "  0),\n",
              " (1273,\n",
              "  15,\n",
              "  'What is Cerebral Cavernous Malformation?',\n",
              "  'Although most cerebral cavernous malformations showed evolution of hemorrhage or no change in size or shape on follow-up MR images, cerebral cavernous malformations represented as mixture of subacute and chronic hemorrhage with hemosiderin rim (type II) have a higher frequency to rebleed than other types of cerebral cavernous malformations.',\n",
              "  0),\n",
              " (1273,\n",
              "  16,\n",
              "  'What is Cerebral Cavernous Malformation?',\n",
              "  'Cerebral cavernous malformation is a vascular disease of the brain causing headaches, seizures, and cerebral hemorrhage.',\n",
              "  1),\n",
              " (254,\n",
              "  0,\n",
              "  'Which is the relation between sweating and anaerobic threshold?',\n",
              "  'It is concluded that the sweating response during upright recovery is significantly modified by exercise intensity and may likely be influenced by the nonthermal baroreceptor reflex adjustments postexercise.',\n",
              "  1),\n",
              " (254,\n",
              "  1,\n",
              "  'Which is the relation between sweating and anaerobic threshold?',\n",
              "  ' The purpose of this paper is to delineate the effects of training status, heat acclimation, environmental conditions and host factors on the sweating response to exercise.',\n",
              "  1),\n",
              " (254,\n",
              "  2,\n",
              "  'Which is the relation between sweating and anaerobic threshold?',\n",
              "  'These results would suggest that at a given exercise intensity in subjects with a higher aerobic capacity body temperature is maintained with a lower sweating rate than that in subjects with a lower aerobic capacity.',\n",
              "  1),\n",
              " (4145,\n",
              "  0,\n",
              "  'Is atenolol metabolized by CYP2D6?',\n",
              "  'The study analysed the prescribing and dispensing of CYP2D6 drugs (metoprolol, donepezil, galantamine, codeine, tamoxifen) together with CYP2D6-blocking SSRIs (paroxetine/fluoxetine) or SSRIs without significant CYP2D6 inhibition (citalopram/escitalopram/sertraline), and the related prescribing of CYP2D6-independent comparator drugs (atenolol, rivastigmine, propoxyphene, anastrozole).',\n",
              "  1),\n",
              " (3821,\n",
              "  0,\n",
              "  'Which is the master oncogenic transcription factor in T-cell acute lymphoblastic leukemia?',\n",
              "  'We identify CNOT3 as a tumor suppressor mutated in 7 of 89 (7.9%) adult T-ALLs, and its knockdown causes tumors in a sensitized Drosophila melanogaster model.',\n",
              "  0),\n",
              " (3821,\n",
              "  1,\n",
              "  'Which is the master oncogenic transcription factor in T-cell acute lymphoblastic leukemia?',\n",
              "  'In addition, we identify mutations affecting the ribosomal proteins RPL5 and RPL10 in 12 of 122 (9.8%) pediatric T-ALLs, with recurrent alterations of Arg98 in RPL10.',\n",
              "  0),\n",
              " (3821,\n",
              "  2,\n",
              "  'Which is the master oncogenic transcription factor in T-cell acute lymphoblastic leukemia?',\n",
              "  'ARID5B as a critical downstream target of the TAL1 complex that activates the oncogenic transcriptional program and promotes T-cell leukemogenesis.',\n",
              "  0),\n",
              " (3821,\n",
              "  3,\n",
              "  'Which is the master oncogenic transcription factor in T-cell acute lymphoblastic leukemia?',\n",
              "  'The oncogenic transcription factor TAL1/SCL induces an aberrant transcriptional program in T-cell acute lymphoblastic leukemia (T-ALL) cells.',\n",
              "  1),\n",
              " (3821,\n",
              "  4,\n",
              "  'Which is the master oncogenic transcription factor in T-cell acute lymphoblastic leukemia?',\n",
              "  'Our results indicate that ARID5B reinforces the oncogenic transcriptional program by positively regulating the TAL1-induced regulatory circuit and MYC in T-ALL, thereby contributing to T-cell leukemogenesis.',\n",
              "  0),\n",
              " (3821,\n",
              "  5,\n",
              "  'Which is the master oncogenic transcription factor in T-cell acute lymphoblastic leukemia?',\n",
              "  'TAL1 as a master oncogenic transcription factor in T-cell acute lymphoblastic leukemia.',\n",
              "  0),\n",
              " (3821,\n",
              "  6,\n",
              "  'Which is the master oncogenic transcription factor in T-cell acute lymphoblastic leukemia?',\n",
              "  'BACKGROUND: T-cell acute lymphoblastic leukemia is a hematologic malignancy characterized by T-cell proliferation, and in many cases, the ectopic expression of the oncogenic transcription factor T-cell acute lymphocytic leukemia protein 1 (TAL1).',\n",
              "  0),\n",
              " (3821,\n",
              "  7,\n",
              "  'Which is the master oncogenic transcription factor in T-cell acute lymphoblastic leukemia?',\n",
              "  'MiR-7 Functions as a Tumor Suppressor by Targeting the Oncogenes TAL1 in T-Cell Acute Lymphoblastic Leukemia.',\n",
              "  0),\n",
              " (3821,\n",
              "  8,\n",
              "  'Which is the master oncogenic transcription factor in T-cell acute lymphoblastic leukemia?',\n",
              "  'Aberrant expression of transcription factor oncogenes such as HOX11, HOX11L2, TAL1/SCL, LYL1, LMO1, and LMO2 can be detected in lymphoblasts from up to 80% of patients with acute T-cell lymphoblastic leukemia (T-ALL).',\n",
              "  0),\n",
              " (3821,\n",
              "  9,\n",
              "  'Which is the master oncogenic transcription factor in T-cell acute lymphoblastic leukemia?',\n",
              "  'The transcription factor TAL1/SCL is one of the most prevalent oncogenes in T-cell acute lymphoblastic leukemia (T-ALL), a malignant disorder resulting from leukemic transformation of thymus T-cell precursors.',\n",
              "  1),\n",
              " (3821,\n",
              "  10,\n",
              "  'Which is the master oncogenic transcription factor in T-cell acute lymphoblastic leukemia?',\n",
              "  'TAL1 as a master oncogenic transcription factor in T-cell acute lymphoblastic leukemia',\n",
              "  0),\n",
              " (3821,\n",
              "  11,\n",
              "  'Which is the master oncogenic transcription factor in T-cell acute lymphoblastic leukemia?',\n",
              "  'The transcription factor TAL1/SCL is one of the most prevalent oncogenes in T-cell acute lymphoblastic leukemia (T-ALL), a malignant disorder resulting from leukemic transformation of thymus T-cell precursors.',\n",
              "  0),\n",
              " (3821,\n",
              "  12,\n",
              "  'Which is the master oncogenic transcription factor in T-cell acute lymphoblastic leukemia?',\n",
              "  'The oncogenic transcription factor TAL1/SCL is aberrantly expressed in over 40% of cases of human T\\xa0cell acute lymphoblastic leukemia (T-ALL), emphasizing its importance in the molecular pathogenesis of T-ALL.',\n",
              "  1),\n",
              " (3821,\n",
              "  13,\n",
              "  'Which is the master oncogenic transcription factor in T-cell acute lymphoblastic leukemia?',\n",
              "  'H',\n",
              "  0),\n",
              " (3821,\n",
              "  14,\n",
              "  'Which is the master oncogenic transcription factor in T-cell acute lymphoblastic leukemia?',\n",
              "  'The Tal1 oncogene is a class II basic helix-loop-helix (bHLH) transcription factor, overexpressed in as much as 60% of T cell acute lymphoblastic leukemia cases.',\n",
              "  0),\n",
              " (3821,\n",
              "  15,\n",
              "  'Which is the master oncogenic transcription factor in T-cell acute lymphoblastic leukemia?',\n",
              "  'Li',\n",
              "  0),\n",
              " (3821,\n",
              "  16,\n",
              "  'Which is the master oncogenic transcription factor in T-cell acute lymphoblastic leukemia?',\n",
              "  'SCL/TAL1 (stem cell leukemia/T-cell acute lymphoblastic leukemia [T-ALL] 1) is an essential transcription factor in normal and malignant hematopoiesis.',\n",
              "  0),\n",
              " (3821,\n",
              "  17,\n",
              "  'Which is the master oncogenic transcription factor in T-cell acute lymphoblastic leukemia?',\n",
              "  'It',\n",
              "  0),\n",
              " (3821,\n",
              "  18,\n",
              "  'Which is the master oncogenic transcription factor in T-cell acute lymphoblastic leukemia?',\n",
              "  'Previous results indicated that miR-146b-5p is downregulated by TAL1, a transcription factor critical for early hematopoiesis that is frequently overexpressed in T-cell acute lymphoblastic leukemia (T-ALL) where it has an oncogenic role.',\n",
              "  0),\n",
              " (3821,\n",
              "  19,\n",
              "  'Which is the master oncogenic transcription factor in T-cell acute lymphoblastic leukemia?',\n",
              "  'H',\n",
              "  0),\n",
              " (3821,\n",
              "  20,\n",
              "  'Which is the master oncogenic transcription factor in T-cell acute lymphoblastic leukemia?',\n",
              "  'In this review, we focus on the oncogenic transcription factor TAL1 and the tumor-suppressor E-proteins and discuss the malignant cell state, the transcriptional circuit, and the consequence of molecular abnormalities in T-ALL.',\n",
              "  0),\n",
              " (3821,\n",
              "  21,\n",
              "  'Which is the master oncogenic transcription factor in T-cell acute lymphoblastic leukemia?',\n",
              "  'The oncogenic transcription factor TAL1/SCL is aberrantly expressed in over 40% of cases of human T\\xa0cell acute lymphoblastic leukemia (T-ALL), emphasizing its importance in the molecular pathogenesis of T-ALL.',\n",
              "  1),\n",
              " (3821,\n",
              "  22,\n",
              "  'Which is the master oncogenic transcription factor in T-cell acute lymphoblastic leukemia?',\n",
              "  'Oncogenic transcriptional program driven by TAL1 in T-cell acute lymphoblastic leukemia.',\n",
              "  1),\n",
              " (656,\n",
              "  0,\n",
              "  'What symptoms characterize the Muenke syndrome?',\n",
              "  'Muenke syndrome is characterized by coronal craniosynostosis (bilateral more often than unilateral), hearing loss, developmental delay, and carpal and/or tarsal bone coalition.',\n",
              "  0),\n",
              " (656,\n",
              "  1,\n",
              "  'What symptoms characterize the Muenke syndrome?',\n",
              "  'Tarsal coalition is a distinct feature of Muenke syndrome and has been reported since the initial description of the disorder in the 1990s.',\n",
              "  0),\n",
              " (656,\n",
              "  2,\n",
              "  'What symptoms characterize the Muenke syndrome?',\n",
              "  'Muenke syndrome caused by the FGFR3 Pro250Arg mutation is associated with craniosynostosis, hearing loss, and various bony anomalies.',\n",
              "  0),\n",
              " (656,\n",
              "  3,\n",
              "  'What symptoms characterize the Muenke syndrome?',\n",
              "  'Muenke syndrome caused by the FGFR3(P250R) mutation is an autosomal dominant disorder mostly identified with coronal suture synostosis, but it also presents with other craniofacial phenotypes that include mild to moderate midface hypoplasia.',\n",
              "  0),\n",
              " (656,\n",
              "  4,\n",
              "  'What symptoms characterize the Muenke syndrome?',\n",
              "  'Sensorineural hearing loss at lower frequencies was found only in patients with Muenke syndrome.',\n",
              "  0),\n",
              " (656,\n",
              "  5,\n",
              "  'What symptoms characterize the Muenke syndrome?',\n",
              "  ' Sensorineural hearing loss can occur in all 4 syndromes studied but is the primary cause of hearing loss in children and young adults with Muenke syndrome.',\n",
              "  0),\n",
              " (656,\n",
              "  6,\n",
              "  'What symptoms characterize the Muenke syndrome?',\n",
              "  'The facial features of children with FGFR3Pro250Arg mutation (Muenke syndrome) differ from those with the other eponymous craniosynostotic disorders.',\n",
              "  0),\n",
              " (656,\n",
              "  7,\n",
              "  'What symptoms characterize the Muenke syndrome?',\n",
              "  'We documented midfacial growth and position of the forehead after fronto-orbital advancement (FOA) in patients with the FGFR3 mutation.',\n",
              "  0),\n",
              " (656,\n",
              "  8,\n",
              "  'What symptoms characterize the Muenke syndrome?',\n",
              "  'The Muenke syndrome (MS) is characterized by unicoronal or bicoronal craniosynostosis, midfacial hypoplasia, ocular hypertelorism, and a variety of minor abnormalities associated with a mutation in the fibroblast growth factor receptor 3 (FGFR3) gene.',\n",
              "  0),\n",
              " (656,\n",
              "  9,\n",
              "  'What symptoms characterize the Muenke syndrome?',\n",
              "  'Muenke syndrome is characterized by considerable phenotypic variability: features may include coronal synostosis (more often bilateral than unilateral); synostosis of other sutures, all sutures (pansynostosis), or no sutures; or macrocephaly.',\n",
              "  0),\n",
              " (656,\n",
              "  10,\n",
              "  'What symptoms characterize the Muenke syndrome?',\n",
              "  'Bilateral coronal synostosis typically results in brachycephaly (broad skull), although turribrachycephaly (a \"tower-shaped\" skull) or a cloverleaf skull can be observed.',\n",
              "  0),\n",
              " (656,\n",
              "  11,\n",
              "  'What symptoms characterize the Muenke syndrome?',\n",
              "  'Unilateral coronal synostosis results in anterior plagiocephaly (asymmetry of the skull and face).',\n",
              "  0),\n",
              " (656,\n",
              "  12,\n",
              "  'What symptoms characterize the Muenke syndrome?',\n",
              "  'Other craniofacial findings typically include: ocular hypertelorism, ptosis or proptosis (usually mild), midface hypoplasia, temporal bossing, and a highly arched palate.',\n",
              "  0),\n",
              " (656,\n",
              "  13,\n",
              "  'What symptoms characterize the Muenke syndrome?',\n",
              "  'Strabismus is common.',\n",
              "  0),\n",
              " (656,\n",
              "  14,\n",
              "  'What symptoms characterize the Muenke syndrome?',\n",
              "  'Extracranial findings can include: hearing loss (in 33%-100% of affected individuals); developmental delay (~33%); intellectual disability; carpal bone and/or tarsal bone fusions; brachydactyly, broad toes, broad thumbs, and/or clinodactyly; and radiographic findings of thimble-like (short and broad) middle phalanges and/or cone-shaped epiphyses.',\n",
              "  1),\n",
              " (656,\n",
              "  15,\n",
              "  'What symptoms characterize the Muenke syndrome?',\n",
              "  'Phenotypic variability is considerable even within the same family.',\n",
              "  0),\n",
              " (656,\n",
              "  16,\n",
              "  'What symptoms characterize the Muenke syndrome?',\n",
              "  'Muenke syndrome is an autosomal dominant disorder characterized by coronal suture craniosynostosis, hearing loss, developmental delay, carpal and tarsal fusions, and the presence of the Pro250Arg mutation in the FGFR3 gene.',\n",
              "  1),\n",
              " (656,\n",
              "  17,\n",
              "  'What symptoms characterize the Muenke syndrome?',\n",
              "  'A majority of the patients (95%) demonstrated a mild-to-moderate, low frequency sensorineural hearing loss.',\n",
              "  0),\n",
              " (656,\n",
              "  18,\n",
              "  'What symptoms characterize the Muenke syndrome?',\n",
              "  'Increased digital markings were more severe posteriorly in Muenke patients than in non-Muenke patients.',\n",
              "  0),\n",
              " (656,\n",
              "  19,\n",
              "  'What symptoms characterize the Muenke syndrome?',\n",
              "  'The Muenke patients with unilateral coronal synostosis showed a somewhat more severe asymmetry in the anterior part of the skull than the non-Muenke patients.',\n",
              "  0),\n",
              " (656,\n",
              "  20,\n",
              "  'What symptoms characterize the Muenke syndrome?',\n",
              "  'Muenke syndrome is a genetically determined craniosynostosis that involves one or both coronal sutures.',\n",
              "  0),\n",
              " (656,\n",
              "  21,\n",
              "  'What symptoms characterize the Muenke syndrome?',\n",
              "  'In some patients it is associated with skeletal abnormalities such as thimble-like middle phalanges, coned epiphysis, and/or neurological impairment, namely sensorineural hearing loss or mental retardation.',\n",
              "  0),\n",
              " (656,\n",
              "  22,\n",
              "  'What symptoms characterize the Muenke syndrome?',\n",
              "  'Muenke syndrome is an autosomal dominant disorder characterized by coronal suture craniosynostosis, hearing loss, developmental delay, carpal and tarsal fusions, and the presence of the Pro250Arg mutation in the FGFR3 gene.',\n",
              "  1),\n",
              " (656,\n",
              "  23,\n",
              "  'What symptoms characterize the Muenke syndrome?',\n",
              "  'Muenke syndrome is characterized by various craniofacial deformities and is caused by an autosomal-dominant activating mutation in fibroblast growth factor receptor 3 (FGFR3(P250R)               ).',\n",
              "  0),\n",
              " (656,\n",
              "  24,\n",
              "  'What symptoms characterize the Muenke syndrome?',\n",
              "  'The Muenke syndrome (MS) is characterized by unicoronal or bicoronal craniosynostosis, midfacial hypoplasia, ocular hypertelorism, and a variety of minor abnormalities associated with a mutation in the fibroblast growth factor receptor 3 (FGFR3) gene.',\n",
              "  0),\n",
              " (656,\n",
              "  25,\n",
              "  'What symptoms characterize the Muenke syndrome?',\n",
              "  'Muenke syndrome and FGFR2-related isolated coronal synostosis are characterized only by uni- or bicoronal craniosynostosis; the remainder are characterized by bicoronal craniosynostosis or cloverleaf skull, distinctive facial features, and variable hand and foot findings.',\n",
              "  0),\n",
              " (656,\n",
              "  26,\n",
              "  'What symptoms characterize the Muenke syndrome?',\n",
              "  'Muenke syndrome is an autosomal-dominant craniosynostosis syndrome characterized by unilateral or bilateral coronal craniosynostosis, hearing loss, intellectual disability, and relatively subtle limb findings such as carpal bone fusion and tarsal bone fusion.',\n",
              "  0),\n",
              " (656,\n",
              "  27,\n",
              "  'What symptoms characterize the Muenke syndrome?',\n",
              "  'Muenke syndrome is an autosomal dominant disorder characterized by coronal suture craniosynostosis, hearing loss, developmental delay, carpal and tarsal fusions, and the presence of the Pro250Arg mutation in the FGFR3 gene',\n",
              "  1),\n",
              " (656,\n",
              "  28,\n",
              "  'What symptoms characterize the Muenke syndrome?',\n",
              "  'Muenke syndrome caused by the FGFR3 Pro250Arg mutation is associated with craniosynostosis, hearing loss, and various bony anomalies',\n",
              "  0),\n",
              " (656,\n",
              "  29,\n",
              "  'What symptoms characterize the Muenke syndrome?',\n",
              "  'Muenke syndrome is characterized by coronal suture synostosis, midface hypoplasia, subtle limb anomalies, and hearing loss',\n",
              "  0),\n",
              " (656,\n",
              "  30,\n",
              "  'What symptoms characterize the Muenke syndrome?',\n",
              "  'Muenke syndrome is an autosomal-dominant craniosynostosis syndrome characterized by unilateral or bilateral coronal craniosynostosis, hearing loss, intellectual disability, and relatively subtle limb findings such as carpal bone fusion and tarsal bone fusion',\n",
              "  0),\n",
              " (656,\n",
              "  31,\n",
              "  'What symptoms characterize the Muenke syndrome?',\n",
              "  'Muenke syndrome is characterized by coronal craniosynostosis (bilateral more often than unilateral), hearing loss, developmental delay, and carpal and/or tarsal bone coalition',\n",
              "  0),\n",
              " (656,\n",
              "  32,\n",
              "  'What symptoms characterize the Muenke syndrome?',\n",
              "  'Muenke syndrome is characterized by various craniofacial deformities and is caused by an autosomal-dominant activating mutation in fibroblast growth factor receptor 3 (FGFR3(P250R)               )',\n",
              "  0),\n",
              " (656,\n",
              "  33,\n",
              "  'What symptoms characterize the Muenke syndrome?',\n",
              "  'Muenke syndrome is an autosomal dominant disorder characterized by coronal suture craniosynostosis, hearing loss, developmental delay, carpal and tarsal fusions, and the presence of the Pro250Arg mutation in the FGFR3 gene',\n",
              "  1),\n",
              " (516,\n",
              "  0,\n",
              "  'Is there an association between TERT promoter mutation and survival of glioma patients?',\n",
              "  'Mutations lead to TERT upregulation and are associated with aggressive clinical behavior in glioblastomas.',\n",
              "  0),\n",
              " (516,\n",
              "  1,\n",
              "  'Is there an association between TERT promoter mutation and survival of glioma patients?',\n",
              "  \"Kaplan-Meier's survival analysis showed that TERT promoter mutation (P=0.037), Isocitrate dehydrogenase (IDH) mutation (P<0.001), and 1p/19q codeletion (P<0.001) were associated with favorable overall survival (OS).\",\n",
              "  0),\n",
              " (516,\n",
              "  2,\n",
              "  'Is there an association between TERT promoter mutation and survival of glioma patients?',\n",
              "  'In the subset of 116 IDH-mutated lower-grade gliomas lacking 1p/19q codeletion, 19 TERT promoter-mutated tumors exhibited longer progression-free survival (PFS) (P=0.027) and OS (P=0.004).',\n",
              "  0),\n",
              " (516,\n",
              "  3,\n",
              "  'Is there an association between TERT promoter mutation and survival of glioma patients?',\n",
              "  'Consistent with this observation, in the subset of 97 IDH-mutated astrocytomas, 14 TERT promoter-mutated tumors showed longer PFS (P=0.001) and OS (P=0.001).',\n",
              "  0),\n",
              " (516,\n",
              "  4,\n",
              "  'Is there an association between TERT promoter mutation and survival of glioma patients?',\n",
              "  'In contrast, among the subset of 74 IDH wild-type lower-grade gliomas with intact 1p/19q, TERT promoter mutation was associated with shorter PFS (P=0.001) and OS (P=0.001).',\n",
              "  1),\n",
              " (516,\n",
              "  5,\n",
              "  'Is there an association between TERT promoter mutation and survival of glioma patients?',\n",
              "  'Similarly, in the subset of 65 IDH wild-type astrocytomas, 16 TERT promoter-mutated tumors exhibited unfavorable PFS (P=0.007) and OS (P=0.008).',\n",
              "  0),\n",
              " (516,\n",
              "  6,\n",
              "  'Is there an association between TERT promoter mutation and survival of glioma patients?',\n",
              "  'Our results indicate that when combined with IDH status, TERT promoter mutation contributes to prognostic subgroups of lower-grade astrocytic tumors or 1p/19q intact lower-grade gliomas and this may further refine future molecular classification of lower-grade gliomas.',\n",
              "  0),\n",
              " (516,\n",
              "  7,\n",
              "  'Is there an association between TERT promoter mutation and survival of glioma patients?',\n",
              "  'RESULTS: TERTp-mut identified in 60.8% of gliomas (491 out of 807) was globally associated with poorer outcome (Hazard ratio (HR)=1.50).',\n",
              "  0),\n",
              " (516,\n",
              "  8,\n",
              "  'Is there an association between TERT promoter mutation and survival of glioma patients?',\n",
              "  'TERT promoter mutations lead to high transcriptional activity under hypoxia and temozolomide treatment and predict poor prognosis in gliomas.',\n",
              "  0),\n",
              " (516,\n",
              "  9,\n",
              "  'Is there an association between TERT promoter mutation and survival of glioma patients?',\n",
              "  'Patients with TERT promoter mutations had lower survival rates, even after adjusting for other known or potential risk factors, and the incidence of mutation was correlated with patient age.',\n",
              "  1),\n",
              " (516,\n",
              "  10,\n",
              "  'Is there an association between TERT promoter mutation and survival of glioma patients?',\n",
              "  'TERT promoter mutations maintained its ability of inducing high transcriptional activity even under hypoxic and TMZ treatment conditions, and the presence of mutations was associated with poor prognosis in glioma patients.',\n",
              "  1),\n",
              " (516,\n",
              "  11,\n",
              "  'Is there an association between TERT promoter mutation and survival of glioma patients?',\n",
              "  'Patients whose Grade III-IV gliomas exhibit TERT promoter mutations alone predominately have primary GBMs associated with poor median OS (11.5 months).',\n",
              "  0),\n",
              " (516,\n",
              "  12,\n",
              "  'Is there an association between TERT promoter mutation and survival of glioma patients?',\n",
              "  'Patients whose Grade III-IV gliomas exhibit IDH1/2 mutations alone predominately have astrocytic morphologies and exhibit a median OS of 57 months while patients whose tumors exhibit both TERT promoter and IDH1/2 mutations predominately exhibit oligodendroglial morphologies and exhibit median OS of 125 months.',\n",
              "  0),\n",
              " (516,\n",
              "  13,\n",
              "  'Is there an association between TERT promoter mutation and survival of glioma patients?',\n",
              "  'Patients with TERT promoter mutations had lower survival rates, even after adjusting for other known or potential risk factors, and the incidence of mutation was correlated with patient age.CONCLUSION: TERT promoter mutations were specific to gliomas.',\n",
              "  0),\n",
              " (516,\n",
              "  14,\n",
              "  'Is there an association between TERT promoter mutation and survival of glioma patients?',\n",
              "  'We defined, based on TERTp-mut and IDH mutation status, four prognostic groups: (1) TERTp-mut and IDH-mut associated with 1p19q codeletion, overall survival (OS)>17 years; (2) TERTp-wt and IDH-mut, associated with TP53 mutation, OS=97.5 months; (3) TERTp-wt and IDH-wt, with no specific association, OS=31.6 months; (4) TERTp-mut and IDH-wt, associated with EGFR amplification, OS=15.4 months.',\n",
              "  0),\n",
              " (516,\n",
              "  15,\n",
              "  'Is there an association between TERT promoter mutation and survival of glioma patients?',\n",
              "  'Mutations lead to TERT upregulation and are associated with aggressive clinical behavior in glioblastomas.',\n",
              "  0),\n",
              " (516,\n",
              "  16,\n",
              "  'Is there an association between TERT promoter mutation and survival of glioma patients?',\n",
              "  'In the subset of 116 IDH-mutated lower-grade gliomas lacking 1p/19q codeletion, 19 TERT promoter-mutated tumors exhibited longer progression-free survival (PFS) (P=0.027) and OS (P=0.004).',\n",
              "  0),\n",
              " (516,\n",
              "  17,\n",
              "  'Is there an association between TERT promoter mutation and survival of glioma patients?',\n",
              "  'The mean age at diagnosis was lowest (37 years) among patients who had gliomas with only IDH mutations and was highest (59 years) among patients who had gliomas with only TERT mutations.',\n",
              "  0),\n",
              " (516,\n",
              "  18,\n",
              "  'Is there an association between TERT promoter mutation and survival of glioma patients?',\n",
              "  'The molecular groups were independently associated with overall survival among patients with grade II or III gliomas but not among patients with grade IV gliomas.',\n",
              "  1),\n",
              " (516,\n",
              "  19,\n",
              "  'Is there an association between TERT promoter mutation and survival of glioma patients?',\n",
              "  'Mutations were detected in gliomas, but not in meningiomas, pituitary adenomas, cavernomas, intracranial metastases, normal brain tissues, or peripheral blood of glioma patients.',\n",
              "  0),\n",
              " (516,\n",
              "  20,\n",
              "  'Is there an association between TERT promoter mutation and survival of glioma patients?',\n",
              "  'Patients with TERT promoter mutations had lower survival rates, even after adjusting for other known or potential risk factors, and the incidence of mutation was correlated with patient age.',\n",
              "  1),\n",
              " (4210,\n",
              "  0,\n",
              "  'Is the zelda transcription factor a chromatin remodeller?',\n",
              "  'ncreasing the number of Zelda binding sites accelerates the kinetics of nuclei transcriptional activation regardless of their transcriptional past',\n",
              "  0),\n",
              " (4210,\n",
              "  1,\n",
              "  'Is the zelda transcription factor a chromatin remodeller?',\n",
              "  'Zelda facilitates transcriptional activation by accumulating in microenvironments where it could accelerate the duration of multiple pre-initiation steps.',\n",
              "  0),\n",
              " (4210,\n",
              "  2,\n",
              "  'Is the zelda transcription factor a chromatin remodeller?',\n",
              "  'Zelda acts through specific chromatin patterns of histone modifications to mark developmental enhancers and active promoters',\n",
              "  0),\n",
              " (4210,\n",
              "  3,\n",
              "  'Is the zelda transcription factor a chromatin remodeller?',\n",
              "  'Zelda overcomes the high intrinsic nucleosome barrier at enhancers during Drosophila zygotic genome activation.',\n",
              "  0),\n",
              " (4210,\n",
              "  4,\n",
              "  'Is the zelda transcription factor a chromatin remodeller?',\n",
              "  'The Drosophila genome activator Vielfaltig (Vfl), also known as Zelda (Zld), is thought to prime enhancers for activation by patterning transcription factors (TFs).',\n",
              "  0),\n",
              " (4210,\n",
              "  5,\n",
              "  'Is the zelda transcription factor a chromatin remodeller?',\n",
              "  'Such priming is accompanied by increased chromatin accessibility',\n",
              "  0),\n",
              " (4210,\n",
              "  6,\n",
              "  'Is the zelda transcription factor a chromatin remodeller?',\n",
              "  'early enhancers are characterized by an intrinsically high nucleosome barrier.',\n",
              "  0),\n",
              " (4210,\n",
              "  7,\n",
              "  'Is the zelda transcription factor a chromatin remodeller?',\n",
              "  'Zld tackles this nucleosome barrier through local depletion of nucleosomes with the effect being dependent on the number and position of Zld motifs.',\n",
              "  1),\n",
              " (4210,\n",
              "  8,\n",
              "  'Is the zelda transcription factor a chromatin remodeller?',\n",
              "  'Zelda is differentially required for chromatin accessibility, transcription factor binding, and gene expression in the early Drosophila embryo',\n",
              "  1),\n",
              " (4210,\n",
              "  9,\n",
              "  'Is the zelda transcription factor a chromatin remodeller?',\n",
              "  'Open chromatin is associated with Zelda-bound loci, as well as more generally with regions of active transcriptio',\n",
              "  0),\n",
              " (4210,\n",
              "  10,\n",
              "  'Is the zelda transcription factor a chromatin remodeller?',\n",
              "  'During this developmental transition, the zygotic genome is largely transcriptionally quiescent and undergoes significant chromatin remodeling.',\n",
              "  0),\n",
              " (4210,\n",
              "  11,\n",
              "  'Is the zelda transcription factor a chromatin remodeller?',\n",
              "  'In Drosophila, the DNA-binding protein Zelda (also known as Vielfaltig) is required for this transition and for transcriptional activation of the zygotic genome.',\n",
              "  1),\n",
              " (4210,\n",
              "  12,\n",
              "  'Is the zelda transcription factor a chromatin remodeller?',\n",
              "  'Zld facilitates binding of Dl to regulatory DNA, and that this is associated with increased chromatin accessibility.',\n",
              "  0),\n",
              " (4210,\n",
              "  13,\n",
              "  'Is the zelda transcription factor a chromatin remodeller?',\n",
              "  'Importantly, the change in chromatin accessibility is strongly correlated with the change in Zld binding',\n",
              "  0),\n",
              " (4210,\n",
              "  14,\n",
              "  'Is the zelda transcription factor a chromatin remodeller?',\n",
              "  'Zelda is differentially required for chromatin accessibility, transcription factor binding, and gene expression in the early Drosophila embryo.',\n",
              "  1),\n",
              " (4210,\n",
              "  15,\n",
              "  'Is the zelda transcription factor a chromatin remodeller?',\n",
              "  'We propose that both Zelda and GAGA factor function to specify sites of open chromatin and together facilitate the remodeling of the early embryonic genome.',\n",
              "  0),\n",
              " (4210,\n",
              "  16,\n",
              "  'Is the zelda transcription factor a chromatin remodeller?',\n",
              "  'is analysis highlighted a strong and specific enrichment of predicted ZGA-associated CRMs for Zelda, CBP, Trl binding sites, as well as for histone marks associated with active enhancers (H3K4me1) and for open chromatin regions.CO',\n",
              "  0),\n",
              " (4210,\n",
              "  17,\n",
              "  'Is the zelda transcription factor a chromatin remodeller?',\n",
              "  'We demonstrate that Zelda is essential for hundreds of regions of open chromatin.',\n",
              "  0),\n",
              " (4210,\n",
              "  18,\n",
              "  'Is the zelda transcription factor a chromatin remodeller?',\n",
              "  'Intriguingly, some Zelda sites still maintain these chromatin patterns in Drosophila embryos lacking maternal Zelda protein.',\n",
              "  0),\n",
              " (4210,\n",
              "  19,\n",
              "  'Is the zelda transcription factor a chromatin remodeller?',\n",
              "  'Zelda potentiates morphogen activity by increasing chromatin accessibility.',\n",
              "  0),\n",
              " (4210,\n",
              "  20,\n",
              "  'Is the zelda transcription factor a chromatin remodeller?',\n",
              "  'Zelda binds cis-regulatory elements (TAGteam heptamers), making chromatin accessible for gene transcription.',\n",
              "  0),\n",
              " (4210,\n",
              "  21,\n",
              "  'Is the zelda transcription factor a chromatin remodeller?',\n",
              "  'zinc-finger TF zelda (zld) is essential for the maternal-to-zygotic transition (MZT) in Drosophila melanogaster, where it directly binds over thousand cis-regulatory modules to regulate chromatin accessibility.',\n",
              "  0),\n",
              " (4210,\n",
              "  22,\n",
              "  'Is the zelda transcription factor a chromatin remodeller?',\n",
              "  'D. mela',\n",
              "  0),\n",
              " (4210,\n",
              "  23,\n",
              "  'Is the zelda transcription factor a chromatin remodeller?',\n",
              "  'This Zelda-mediated chromatin accessibility facilitates transcription-factor recruitment and early gene expression.',\n",
              "  1),\n",
              " (4210,\n",
              "  24,\n",
              "  'Is the zelda transcription factor a chromatin remodeller?',\n",
              "  'While analyzing chromatin immunoprecipitation data sets from 21 sequence-specific transcription factors active in the Drosophila embryo, we found that binding of all factors exhibits a dose-dependent relationship with \"TAGteam\" sequence motifs bound by the zinc finger protein Vielfaltig, also known as Zelda, a recently discovered activator of the zygotic genome.',\n",
              "  0),\n",
              " (4210,\n",
              "  25,\n",
              "  'Is the zelda transcription factor a chromatin remodeller?',\n",
              "  'These different timing classes each associate with binding sites for two transcription factors, GAGA-factor and Zelda, previously implicated in controlling chromatin accessibility at ZGA.',\n",
              "  0),\n",
              " (4210,\n",
              "  26,\n",
              "  'Is the zelda transcription factor a chromatin remodeller?',\n",
              "  'Together this reveals a distinct requirement for a chromatin remodeller in promoting the activity of the pioneer factor OCT4 and regulating the pluripotency network.',\n",
              "  0),\n",
              " (4210,\n",
              "  27,\n",
              "  'Is the zelda transcription factor a chromatin remodeller?',\n",
              "  'Chromatin accessibility at OCT4-bound sites requires the chromatin remodeller BRG1, which is recruited to these sites by OCT4 to support additional transcription factor binding and expression of the pluripotency-associated transcriptome.',\n",
              "  0),\n",
              " (4210,\n",
              "  28,\n",
              "  'Is the zelda transcription factor a chromatin remodeller?',\n",
              "  'The pioneer factor OCT4 requires the chromatin remodeller BRG1 to support gene regulatory element function in mouse embryonic stem cells.',\n",
              "  0),\n",
              " (4210,\n",
              "  29,\n",
              "  'Is the zelda transcription factor a chromatin remodeller?',\n",
              "  'Pioneer transcription factors recognise and bind their target sequences in inaccessible chromatin to establish new transcriptional networks throughout development and cellular reprogramming.',\n",
              "  0),\n",
              " (4210,\n",
              "  30,\n",
              "  'Is the zelda transcription factor a chromatin remodeller?',\n",
              "  'During this process, pioneer factors establish an accessible chromatin state to facilitate additional transcription factor binding, yet it remains unclear how different pioneer factors achieve this.',\n",
              "  0),\n",
              " (4210,\n",
              "  31,\n",
              "  'Is the zelda transcription factor a chromatin remodeller?',\n",
              "  'Here, we discover that the pluripotency-associated pioneer factor OCT4 binds chromatin to shape accessibility, transcription factor co-binding, and regulatory element function in mouse embryonic stem cells.',\n",
              "  0),\n",
              " (4210,\n",
              "  32,\n",
              "  'Is the zelda transcription factor a chromatin remodeller?',\n",
              "  'Open chromatin is associated with Zelda-bound loci, as well as more generally with regions of active transcription.',\n",
              "  0),\n",
              " (4210,\n",
              "  33,\n",
              "  'Is the zelda transcription factor a chromatin remodeller?',\n",
              "  'Unexpectedly, chromatin at a large subset of Zelda-bound regions remains open even in the absence of Zelda.',\n",
              "  0),\n",
              " (4210,\n",
              "  34,\n",
              "  'Is the zelda transcription factor a chromatin remodeller?',\n",
              "  'During this developmental transition, the zygotic genome is largely transcriptionally quiescent and undergoes significant chromatin remodeling.',\n",
              "  0),\n",
              " (4210,\n",
              "  35,\n",
              "  'Is the zelda transcription factor a chromatin remodeller?',\n",
              "  'Here we used formaldehyde-assisted isolation of regulatory elements to determine the role of Zelda in regulating regions of open chromatin in the early embryo.',\n",
              "  0),\n",
              " (4210,\n",
              "  36,\n",
              "  'Is the zelda transcription factor a chromatin remodeller?',\n",
              "  'Pioneer transcription factors can engage nucleosomal DNA, which leads to local chromatin remodeling and to the establishment of transcriptional competence.',\n",
              "  0),\n",
              " (4210,\n",
              "  37,\n",
              "  'Is the zelda transcription factor a chromatin remodeller?',\n",
              "  'Recently, we showed that Zelda acts through specific chromatin patterns of histone modifications to mark developmental enhancers and active promoters.',\n",
              "  0),\n",
              " (4210,\n",
              "  38,\n",
              "  'Is the zelda transcription factor a chromatin remodeller?',\n",
              "  'The zinc-finger TF zelda (zld) is essential for the maternal-to-zygotic transition (MZT) in Drosophila melanogaster, where it directly binds over thousand cis-regulatory modules to regulate chromatin accessibility.',\n",
              "  0),\n",
              " (4210,\n",
              "  39,\n",
              "  'Is the zelda transcription factor a chromatin remodeller?',\n",
              "  'Nonetheless, the extent to which Zelda influences chromatin accessibility across the genome is largely unknown.',\n",
              "  0),\n",
              " (4210,\n",
              "  40,\n",
              "  'Is the zelda transcription factor a chromatin remodeller?',\n",
              "  'We present evidence that Zld facilitates binding of Dl to regulatory DNA, and that this is associated with increased chromatin accessibility.',\n",
              "  0),\n",
              " (4210,\n",
              "  41,\n",
              "  'Is the zelda transcription factor a chromatin remodeller?',\n",
              "  'ZRF1 facilitates the remodeling of multiprotein complexes at chromatin and lies at the heart of signaling processes that occur at DNA damage sites and during transcriptional activation.',\n",
              "  0),\n",
              " (4210,\n",
              "  42,\n",
              "  'Is the zelda transcription factor a chromatin remodeller?',\n",
              "  'We postulate that ZRF1 operates in conjunction with cellular remodeling machines and suggest that on-site remodeling might be a hallmark of many chromatin-associated signaling pathways.',\n",
              "  0),\n",
              " (4210,\n",
              "  43,\n",
              "  'Is the zelda transcription factor a chromatin remodeller?',\n",
              "  'Reconstituted transcription reactions established that the Brahma (BRM) chromatin-remodeling complex is essential for Zeste-directed activation on nucleosomal templates.',\n",
              "  0),\n",
              " (2749,\n",
              "  0,\n",
              "  'What is the origin of human breast milk bacteria?',\n",
              "  'In the past years, results provided by different research groups suggest that certain bacteria from the maternal gastrointestinal tract could translocate through a mechanism involving mononuclear immune cells, migrate to the mammary glands via an endogenous cellular route (the bacterial entero-mammary pathway), and subsequently colonize the gastrointestinal tract of the breast-fed neonate.',\n",
              "  1),\n",
              " (2416,\n",
              "  0,\n",
              "  'List factors that promote lymphangiogenesis.',\n",
              "  'Lymphangiogenesis plays an important role in cancer progression and is regulated by a complex mechanism that includes vascular endothelial growth factor (VEGF) signaling.',\n",
              "  1),\n",
              " (2416,\n",
              "  1,\n",
              "  'List factors that promote lymphangiogenesis.',\n",
              "  'ur results demonstrate that NHT stimulates lymphangiogenesis via upregulation of VEGF-C and -D',\n",
              "  1),\n",
              " (2416,\n",
              "  2,\n",
              "  'List factors that promote lymphangiogenesis.',\n",
              "  'Vegfc is essential for secondary angiogenesis, giving rise to veins and lymphatics',\n",
              "  1),\n",
              " (2416,\n",
              "  3,\n",
              "  'List factors that promote lymphangiogenesis.',\n",
              "  'vegfc and vegfd cooperatively control lymphangiogenesis throughout the embryo, including during the formation of the trunk lymphatic vasculature.',\n",
              "  0),\n",
              " (2416,\n",
              "  4,\n",
              "  'List factors that promote lymphangiogenesis.',\n",
              "  'The signals for lymphangiogenesis, VEGF-C and its receptor VEGF-R3',\n",
              "  1),\n",
              " (2416,\n",
              "  5,\n",
              "  'List factors that promote lymphangiogenesis.',\n",
              "  'Vascular endothelial growth factor (VEGF) members VEGF-C and VEGF-D are both potent candidates for stimulating lymphangiogenesis,',\n",
              "  1),\n",
              " (2026,\n",
              "  0,\n",
              "  'What is the role of Lysine 2-hydroxyisobutyrylation?',\n",
              "  'Lysine 2-hydroxyisobutyrylation is a widely distributed active histone mark.',\n",
              "  0),\n",
              " (2026,\n",
              "  1,\n",
              "  'What is the role of Lysine 2-hydroxyisobutyrylation?',\n",
              "  'We report the identification of a new type of histone mark, lysine 2-hydroxyisobutyrylation (Khib), and identify the mark at 63 human and mouse histone Khib sites, including 27 unique lysine sites that are not known to be modified by lysine acetylation (Kac) and lysine crotonylation (Kcr).',\n",
              "  0),\n",
              " (2026,\n",
              "  2,\n",
              "  'What is the role of Lysine 2-hydroxyisobutyrylation?',\n",
              "  'This histone mark was initially identified by MS and then validated by chemical and biochemical methods.',\n",
              "  1),\n",
              " (2026,\n",
              "  3,\n",
              "  'What is the role of Lysine 2-hydroxyisobutyrylation?',\n",
              "  'Histone Khib shows distinct genomic distributions from histone Kac or histone Kcr during male germ cell differentiation.',\n",
              "  1),\n",
              " (2026,\n",
              "  4,\n",
              "  'What is the role of Lysine 2-hydroxyisobutyrylation?',\n",
              "  'Using chromatin immunoprecipitation sequencing, gene expression analysis and immunodetection, we show that in male germ cells, H4K8hib is associated with active gene transcription in meiotic and post-meiotic cells.',\n",
              "  0),\n",
              " (2026,\n",
              "  5,\n",
              "  'What is the role of Lysine 2-hydroxyisobutyrylation?',\n",
              "  'In addition, H4K8ac-associated genes are included in and constitute only a subfraction of H4K8hib-labeled genes.',\n",
              "  1),\n",
              " (2026,\n",
              "  6,\n",
              "  'What is the role of Lysine 2-hydroxyisobutyrylation?',\n",
              "  'The histone Khib mark is conserved and widely distributed, has high stoichiometry and induces a large structural change.',\n",
              "  1),\n",
              " (2026,\n",
              "  7,\n",
              "  'What is the role of Lysine 2-hydroxyisobutyrylation?',\n",
              "  'These findings suggest its critical role on the regulation of chromatin functions.',\n",
              "  0),\n",
              " (2026,\n",
              "  8,\n",
              "  'What is the role of Lysine 2-hydroxyisobutyrylation?',\n",
              "  'We report the identification of a new type of histone mark, lysine 2-hydroxyisobutyrylation (Khib), and identify the mark at 63 human and mouse histone Khib sites, including 27 unique lysine sites that are not known to be modified by lysine acetylation (Kac) and lysine crotonylation (Kcr).',\n",
              "  0),\n",
              " (2026,\n",
              "  9,\n",
              "  'What is the role of Lysine 2-hydroxyisobutyrylation?',\n",
              "  'Lysine 2-hydroxyisobutyrylation is a widely distributed active histone mark',\n",
              "  0),\n",
              " (2026,\n",
              "  10,\n",
              "  'What is the role of Lysine 2-hydroxyisobutyrylation?',\n",
              "  'We report the identification of a new type of histone mark, lysine 2-hydroxyisobutyrylation (Khib), and identify the mark at 63 human and mouse histone Khib sites, including 27 unique lysine sites that are not known to be modified by lysine acetylation (Kac) and lysine crotonylation (Kcr)',\n",
              "  0),\n",
              " (2026,\n",
              "  11,\n",
              "  'What is the role of Lysine 2-hydroxyisobutyrylation?',\n",
              "  'We report the identification of a new type of histone mark, lysine 2-hydroxyisobutyrylation (Khib), and identify the mark at 63 human and mouse histone Khib sites, including 27 unique lysine sites that are not known to be modified by lysine acetylation (Kac) and lysine crotonylation (Kcr).',\n",
              "  0),\n",
              " (2026,\n",
              "  12,\n",
              "  'What is the role of Lysine 2-hydroxyisobutyrylation?',\n",
              "  'The histone Khib mark is conserved and widely distributed, has high stoichiometry and induces a large structural change.',\n",
              "  1),\n",
              " (2026,\n",
              "  13,\n",
              "  'What is the role of Lysine 2-hydroxyisobutyrylation?',\n",
              "  'Lysine 2-hydroxyisobutyrylation is a widely distributed active histone mark.',\n",
              "  0),\n",
              " (4192,\n",
              "  0,\n",
              "  'Which disease is monitored in the BIOCURA cohort?',\n",
              "  'In RA patients eligible for biological treatment included in the BiOCURA registry, 53 inflammatory proteins were measured using xMAP® technology.',\n",
              "  0),\n",
              " (4192,\n",
              "  1,\n",
              "  'Which disease is monitored in the BIOCURA cohort?',\n",
              "  'Several studies have employed microarray-based profiling to predict response to tumor necrosis factor-alpha inhibitors (TNFi) in rheumatoid arthritis (RA); yet efforts to validate these targets have failed to show predictive abilities acceptable for clinical practice.METHODS: The eighty most extreme responders and nonresponders to TNFi therapy were selected from the observational BiOCURA cohort.',\n",
              "  1),\n",
              " (4192,\n",
              "  2,\n",
              "  'Which disease is monitored in the BIOCURA cohort?',\n",
              "  'In the metabolomic profiling, lipids, oxylipins, and amines were measured in serum samples of RA patients from the observational BiOCURA cohort, before start of biological treatment.',\n",
              "  1),\n",
              " (4192,\n",
              "  3,\n",
              "  'Which disease is monitored in the BIOCURA cohort?',\n",
              "  'In clinical practice, approximately one-third of patients with rheumatoid arthritis (RA) respond insufficiently to TNF-α inhibitors (TNFis).',\n",
              "  1),\n",
              " (4192,\n",
              "  4,\n",
              "  'Which disease is monitored in the BIOCURA cohort?',\n",
              "  \"The aim of the study was to explore the use of a metabolomics to identify predictors for the outcome of TNFi therapy, and study the metabolomic fingerprint in active RA irrespective of patients' response.\",\n",
              "  1),\n",
              " (4192,\n",
              "  5,\n",
              "  'Which disease is monitored in the BIOCURA cohort?',\n",
              "  'From the observational BiOCURA cohort, 40 adalimumab- (ADA) and 40 etanercept- (ETN) treated patients were selected to enter the discovery cohort and baseline serum profiling on 758 miRNAs was performed.',\n",
              "  0),\n",
              " (4192,\n",
              "  6,\n",
              "  'Which disease is monitored in the BIOCURA cohort?',\n",
              "  'Despite the success of TNF-alpha inhibitor (TNFi) treatment in rheumatoid arthritis (RA), a substantial number of patients necessitate discontinuation.',\n",
              "  1),\n",
              " (4192,\n",
              "  7,\n",
              "  'Which disease is monitored in the BIOCURA cohort?',\n",
              "  'Prediction thereof would be clinically relevant and guide the decision whether to start TNFi treatment.METHODS: Data were used from the observational BiOCURA cohort, in which patients initiating biological treatment were enrolled and followed up for one year.',\n",
              "  0),\n",
              " (4192, 8, 'Which disease is monitored in the BIOCURA cohort?', 'I', 0),\n",
              " (4192,\n",
              "  9,\n",
              "  'Which disease is monitored in the BIOCURA cohort?',\n",
              "  'BACKGROUND: Several studies have employed microarray-based profiling to predict response to tumor necrosis factor-alpha inhibitors (TNFi) in rheumatoid arthritis (RA); yet efforts to validate these targets have failed to show predictive abilities acceptable for clinical practice',\n",
              "  0),\n",
              " (1586,\n",
              "  0,\n",
              "  'What is the principle of ATAC (Assay for Transposase-Accessible Chromatin) technique?',\n",
              "  'In conclusion, we show that FAIRE-seq and ATAC-seq based open chromatin profiling, combined with motif discovery, is a straightforward approach to identify functional genomic regulatory regions, master regulators, and gene regulatory networks controlling complex in vivo processes.',\n",
              "  0),\n",
              " (1586,\n",
              "  1,\n",
              "  'What is the principle of ATAC (Assay for Transposase-Accessible Chromatin) technique?',\n",
              "  'To this end, we first compare two different approaches to detect open chromatin in vivo using the Drosophila eye primordium as a model system: FAIRE-seq, based on physical separation of open versus closed chromatin; and ATAC-seq, based on preferential integration of a transposon into open chromatin.',\n",
              "  0),\n",
              " (1586,\n",
              "  2,\n",
              "  'What is the principle of ATAC (Assay for Transposase-Accessible Chromatin) technique?',\n",
              "  'We describe an assay for transposase-accessible chromatin using sequencing (ATAC-seq), based on direct in vitro transposition of sequencing adaptors into native chromatin, as a rapid and sensitive method for integrative epigenomic analysis',\n",
              "  1),\n",
              " (1586,\n",
              "  3,\n",
              "  'What is the principle of ATAC (Assay for Transposase-Accessible Chromatin) technique?',\n",
              "  'ATAC-seq captures open chromatin sites using a simple two-step protocol with 500-50,000 cells and reveals the interplay between genomic locations of open chromatin, DNA-binding proteins, individual nucleosomes and chromatin compaction at nucleotide resolution.',\n",
              "  1),\n",
              " (1586,\n",
              "  4,\n",
              "  'What is the principle of ATAC (Assay for Transposase-Accessible Chromatin) technique?',\n",
              "  \"Using ATAC-seq maps of human CD4(+) T cells from a proband obtained on consecutive days, we demonstrated the feasibility of analyzing an individual's epigenome on a timescale compatible with clinical decision-making.\",\n",
              "  0),\n",
              " (1586,\n",
              "  5,\n",
              "  'What is the principle of ATAC (Assay for Transposase-Accessible Chromatin) technique?',\n",
              "  'We describe an assay for transposase-accessible chromatin using sequencing (ATAC-seq), based on direct in vitro transposition of sequencing adaptors into native chromatin, as a rapid and sensitive method for integrative epigenomic analysis.',\n",
              "  1),\n",
              " (1586,\n",
              "  6,\n",
              "  'What is the principle of ATAC (Assay for Transposase-Accessible Chromatin) technique?',\n",
              "  'ATAC-seq captures open chromatin sites using a simple two-step protocol with 500-50,000 cells and reveals the interplay between genomic locations of open chromatin, DNA-binding proteins, individual nucleosomes and chromatin compaction at nucleotide resolution.',\n",
              "  1),\n",
              " (1586,\n",
              "  7,\n",
              "  'What is the principle of ATAC (Assay for Transposase-Accessible Chromatin) technique?',\n",
              "  'ATAC-seq captures open chromatin sites using a simple two-step protocol with 500-50,000 cells and reveals the interplay between genomic locations of open chromatin, DNA-binding proteins, individual nucleosomes and chromatin compaction at nucleotide resolution.',\n",
              "  1),\n",
              " (1586,\n",
              "  8,\n",
              "  'What is the principle of ATAC (Assay for Transposase-Accessible Chromatin) technique?',\n",
              "  'We describe an assay for transposase-accessible chromatin using sequencing (ATAC-seq),',\n",
              "  0),\n",
              " (3852,\n",
              "  0,\n",
              "  'What is FeatureCounts used for?',\n",
              "  'featureCounts: an efficient general purpose program for assigning sequence reads to genomic features.',\n",
              "  1),\n",
              " (3852,\n",
              "  1,\n",
              "  'What is FeatureCounts used for?',\n",
              "  'We present featureCounts, a read summarization program suitable for counting reads generated from either RNA or genomic DNA sequencing experiments',\n",
              "  1),\n",
              " (3852,\n",
              "  2,\n",
              "  'What is FeatureCounts used for?',\n",
              "  'mmquant is a drop-in replacement of the widely used tools htseq-count and featureCounts that handles multi-mapping reads in an unabiased way.',\n",
              "  0),\n",
              " (3852,\n",
              "  3,\n",
              "  'What is FeatureCounts used for?',\n",
              "  'We also show the correlation for raw read counts reported by TPMCalculator, HTSeq and featureCounts',\n",
              "  0),\n",
              " (3852,\n",
              "  4,\n",
              "  'What is FeatureCounts used for?',\n",
              "  'It features six independent core-workflows comprising the state-of-the-art technology with dozens of popular cutting-edge tools such as Tophat-Cufflink-Cuffdiff, Subread-featureCounts-DESeq2, STAR-RSEM-EBSeq, Bowtie-eXpress-edgeR, kallisto-sleuth, HISAT-StringTie-Ballgown, and embeds itself in Snakemake, which is a modern pipeline management system.',\n",
              "  0),\n",
              " (3852,\n",
              "  5,\n",
              "  'What is FeatureCounts used for?',\n",
              "  'SULTS: We present featureCounts, a read summarization program suitable for counting reads generated from either RNA or genomic DNA sequencing experiments.',\n",
              "  1),\n",
              " (3852, 6, 'What is FeatureCounts used for?', 'f', 0),\n",
              " (3852,\n",
              "  7,\n",
              "  'What is FeatureCounts used for?',\n",
              "  'e present featureCounts, a read summarization program suitable for counting reads generated from either RNA or genomic DNA sequencing experiments.',\n",
              "  1),\n",
              " (3852,\n",
              "  8,\n",
              "  'What is FeatureCounts used for?',\n",
              "  't is considerably faster than existing methods (by an order of magnitude for gene-level summarization) and requires far less computer memory.',\n",
              "  0),\n",
              " (3852,\n",
              "  9,\n",
              "  'What is FeatureCounts used for?',\n",
              "  'Read summarization is required for a great variety of genomic analyses but has so far received relatively little attention in the literature.R',\n",
              "  0),\n",
              " (3852,\n",
              "  10,\n",
              "  'What is FeatureCounts used for?',\n",
              "  'The process of counting reads is called read summarization.',\n",
              "  0),\n",
              " (3852, 11, 'What is FeatureCounts used for?', 'marization.', 0),\n",
              " (3852,\n",
              "  12,\n",
              "  'What is FeatureCounts used for?',\n",
              "  'Read summarization is required for a great variety of genomic analyses but has so far received relatively little attention in the literature.RESULTS: We present featureCounts, a read summarization program suitable for counting reads generated from either RNA or genomic DNA se',\n",
              "  1),\n",
              " (1963,\n",
              "  0,\n",
              "  'Which protein complexes recognize centromeric (CEN) DNA in yeast?',\n",
              "  'The Schizosaccharomyces pombe centromere-linked genes, LYS1 and CYH1 on chromosome I and TPS13 and RAN1 on chromosome II, have been isolated',\n",
              "  1),\n",
              " (1963,\n",
              "  1,\n",
              "  'Which protein complexes recognize centromeric (CEN) DNA in yeast?',\n",
              "  'Histone H3 localizes to the centromeric DNA in budding yeast',\n",
              "  1),\n",
              " (1963,\n",
              "  2,\n",
              "  'Which protein complexes recognize centromeric (CEN) DNA in yeast?',\n",
              "  ' In budding yeast, as well as in other eukaryotes, the Cse4 histone variant (known in vertebrates as CENP-A) is believed to substitute for histone H3 at the centromeric nucleosome',\n",
              "  1),\n",
              " (1963,\n",
              "  3,\n",
              "  'Which protein complexes recognize centromeric (CEN) DNA in yeast?',\n",
              "  'Surprisingly, we observed a strong interaction of H3, as well as Cse4, H4, H2A, and H2B, but not histone chaperone Scm3 (HJURP in human) with the centromeric DNA.',\n",
              "  1),\n",
              " (1963,\n",
              "  4,\n",
              "  'Which protein complexes recognize centromeric (CEN) DNA in yeast?',\n",
              "  'H3 localizes to centromeric DNA at all stages of the cell cycle.',\n",
              "  1),\n",
              " (1963,\n",
              "  5,\n",
              "  'Which protein complexes recognize centromeric (CEN) DNA in yeast?',\n",
              "  'Our results favor a H3-Cse4 heterotypic octamer at the budding yeast centromere',\n",
              "  0),\n",
              " (1963,\n",
              "  6,\n",
              "  'Which protein complexes recognize centromeric (CEN) DNA in yeast?',\n",
              "  \"In budding yeast, a single right-handed cenH3/H4/H2A/H2B tetramer wraps the ∼80-bp Centromere DNA Element II (CDE II) sequence of each centromere into a 'hemisome'.\",\n",
              "  0),\n",
              " (1963,\n",
              "  7,\n",
              "  'Which protein complexes recognize centromeric (CEN) DNA in yeast?',\n",
              "  'Here, we show that Cse4 octamers remain intact under conditions of low salt and urea that dissociate H3 octamers.',\n",
              "  0),\n",
              " (2636,\n",
              "  0,\n",
              "  'What is the drug Tecfidera used against?',\n",
              "  'Dimethyl fumarate (Tecfidera™) is an effective therapy for relapsing forms of multiple sclerosis (MS).',\n",
              "  1),\n",
              " (2636,\n",
              "  1,\n",
              "  'What is the drug Tecfidera used against?',\n",
              "  'dimethyl fumarate (Tecfidera(®)) for multiple sclerosis.',\n",
              "  1),\n",
              " (2636,\n",
              "  2,\n",
              "  'What is the drug Tecfidera used against?',\n",
              "  'Tecifidera is approved for the treatment of relapsing-remitting multiple sclerosis',\n",
              "  1),\n",
              " (2636,\n",
              "  3,\n",
              "  'What is the drug Tecfidera used against?',\n",
              "  'Dimethyl fumarate (Tecfidera): a new oral agent for multiple sclerosis.',\n",
              "  1),\n",
              " (3373,\n",
              "  0,\n",
              "  'Is the drug Exubera currently (March 2020) available?',\n",
              "  'Despite discontinuation of the first inhalable insulin, Exubera®, due to suboptimal market acceptance, development of orally inhaled insulin delivery systems has been galvanized by the recent approval of Afrezza® and several others awaiting approval.',\n",
              "  1),\n",
              " (465,\n",
              "  0,\n",
              "  'Which eye condition is managed by the athens protocol?',\n",
              "  'Keratoconus management: long-term stability of topography-guided normalization combined with high-fluence CXL stabilization (the Athens Protocol).',\n",
              "  0),\n",
              " (465,\n",
              "  1,\n",
              "  'Which eye condition is managed by the athens protocol?',\n",
              "  'The Athens Protocol to arrest keratectasia progression and improve corneal regularity demonstrates safe and effective results as a keratoconus management option.',\n",
              "  0),\n",
              " (465,\n",
              "  2,\n",
              "  'Which eye condition is managed by the athens protocol?',\n",
              "  'Progressive potential for long-term flattening validates using caution in the surface normalization to avoid overcorrection.',\n",
              "  0),\n",
              " (465,\n",
              "  3,\n",
              "  'Which eye condition is managed by the athens protocol?',\n",
              "  'The management of cornea blindness from severe corneal scarring, with the Athens Protocol (transepithelial topography-guided PRK therapeutic remodeling, combined with same-day, collagen cross-linking).',\n",
              "  1),\n",
              " (465,\n",
              "  4,\n",
              "  'Which eye condition is managed by the athens protocol?',\n",
              "  ' To evaluate the safety and efficacy of combined transepithelial topography-guided photorefractive keratectomy (PRK) therapeutic remodeling, combined with same-day, collagen cross-linking (CXL).',\n",
              "  1),\n",
              " (465,\n",
              "  5,\n",
              "  'Which eye condition is managed by the athens protocol?',\n",
              "  'This protocol was used for the management of cornea blindness due to severe corneal scarring.',\n",
              "  1),\n",
              " (465,\n",
              "  6,\n",
              "  'Which eye condition is managed by the athens protocol?',\n",
              "  'Management of corneal ectasia after LASIK with combined, same-day, topography-guided partial transepithelial PRK and collagen cross-linking: the athens protocol.',\n",
              "  1),\n",
              " (465,\n",
              "  7,\n",
              "  'Which eye condition is managed by the athens protocol?',\n",
              "  'Corneal refractive power and symmetry changes following normalization of ectasias treated with partial topography-guided PTK combined with higher-fluence CXL (the Athens Protocol).',\n",
              "  0),\n",
              " (465,\n",
              "  8,\n",
              "  'Which eye condition is managed by the athens protocol?',\n",
              "  'To compare epithelial remodeling in keratoconic eyes that had photorefractive keratectomy and corneal collagen crosslinking (Athens protocol) with that in untreated keratoconic eyes and healthy eyes.',\n",
              "  0),\n",
              " (465,\n",
              "  9,\n",
              "  'Which eye condition is managed by the athens protocol?',\n",
              "  'To evaluate a series of patients with corneal ectasia after LASIK that underwent the Athens Protocol: combined topography-guided photorefractive keratectomy (PRK) to reduce or eliminate induced myopia and astigmatism followed by sequential, same-day ultraviolet A (UVA) corneal collagen cross-linking (CXL).',\n",
              "  1),\n",
              " (465,\n",
              "  10,\n",
              "  'Which eye condition is managed by the athens protocol?',\n",
              "  'PURPOSE: To compare epithelial remodeling in keratoconic eyes that had photorefractive keratectomy and corneal collagen crosslinking (Athens protocol) with that in untreated keratoconic eyes and healthy eyes.SETTING: Private clinical practice, Athens, Greece.DESIGN: Comparative case series.METHODS: Fourier-domain anterior segment optical coherence tomography (AS-OCT) was used to obtain in vivo 3-dimensional epithelial thickness maps and center, superior, inferior, maximum, minimum, mean, midperipheral, and variability data.RESULTS: Group A comprised 175 treated keratoconic eyes (Athens protocol); Group B, 193 untreated keratoconic eyes; and Group C, 160 healthy eyes.',\n",
              "  0),\n",
              " (465,\n",
              "  11,\n",
              "  'Which eye condition is managed by the athens protocol?',\n",
              "  'CONCLUSIONS: The Athens Protocol to arrest keratectasia progression and improve corneal regularity demonstrates safe and effective results as a keratoconus management option.',\n",
              "  0),\n",
              " (465,\n",
              "  12,\n",
              "  'Which eye condition is managed by the athens protocol?',\n",
              "  'To compare epithelial remodeling in keratoconic eyes that had photorefractive keratectomy and corneal collagen crosslinking (Athens protocol) with that in untreated keratoconic eyes and healthy eyes.',\n",
              "  0),\n",
              " (465,\n",
              "  13,\n",
              "  'Which eye condition is managed by the athens protocol?',\n",
              "  'Private clinical practice, Athens, Greece.',\n",
              "  0),\n",
              " (465,\n",
              "  14,\n",
              "  'Which eye condition is managed by the athens protocol?',\n",
              "  'To compare epithelial remodeling in keratoconic eyes that had photorefractive keratectomy and corneal collagen crosslinking (Athens protocol) with that in untreated keratoconic eyes and healthy eyes.',\n",
              "  0),\n",
              " (901,\n",
              "  0,\n",
              "  'Which disorder is rated by Palmini classification?',\n",
              "  'This study also provides an opportunity to compare the predictive value of the ILAE and Palmini et al classification schemes with regard to the type I focal cortical dysplasias.',\n",
              "  0),\n",
              " (901,\n",
              "  1,\n",
              "  'Which disorder is rated by Palmini classification?',\n",
              "  'Of the 91 patients, there were 50 patients with ILAE focal cortical dysplasia type Ib, 41 with ILAE focal cortical dysplasia type Ic, 63 with Palmini et al focal cortical dysplasia type IA, and 28 with Palmini et al focal cortical dysplasia type IB.',\n",
              "  0),\n",
              " (901,\n",
              "  2,\n",
              "  'Which disorder is rated by Palmini classification?',\n",
              "  'Crude analysis revealed no significant difference between patients with subtypes of ILAE focal cortical dysplasia type I or Palmini et al focal cortical dysplasia type I concerning postoperative outcome according to the Engel and ILAE scoring systems on seizure frequency.',\n",
              "  0),\n",
              " (901,\n",
              "  3,\n",
              "  'Which disorder is rated by Palmini classification?',\n",
              "  'Our findings revealed no significant difference concerning surgical outcome with respect to seizure frequency for the histologic subtypes of ILAE focal cortical dysplasia type I (Ib vs Ic) or Palmini et al focal cortical dysplasia type I (IA vs IB).',\n",
              "  0),\n",
              " (901,\n",
              "  4,\n",
              "  'Which disorder is rated by Palmini classification?',\n",
              "  'MCD were classified following the existing classification schemes (Barkovich et al., 2012.',\n",
              "  0),\n",
              " (901, 5, 'Which disorder is rated by Palmini classification?', 'Brain.', 0),\n",
              " (901,\n",
              "  6,\n",
              "  'Which disorder is rated by Palmini classification?',\n",
              "  '135, 1348-1369; Palmini et al., 2004.',\n",
              "  0),\n",
              " (901,\n",
              "  7,\n",
              "  'Which disorder is rated by Palmini classification?',\n",
              "  'Neurology.',\n",
              "  0),\n",
              " (901,\n",
              "  8,\n",
              "  'Which disorder is rated by Palmini classification?',\n",
              "  '62, S2-S8) and the ILAE classification for FCD recently proposed by Blümcke in 2011.',\n",
              "  0),\n",
              " (901,\n",
              "  9,\n",
              "  'Which disorder is rated by Palmini classification?',\n",
              "  'DISCUSSION: This short review provides an overview of the issues which account for the varied historical approaches to FCD classification and descriptions of gross pathologic findings associated with FCD and an overview of two more recently developed and widely used schema, the Palmini et al.',\n",
              "  0),\n",
              " (901,\n",
              "  10,\n",
              "  'Which disorder is rated by Palmini classification?',\n",
              "  '(Neurology 62: S2-8, 2004) and the International League Against Epilepsy (ILAE) classifications Blumcke et al.',\n",
              "  0),\n",
              " (901,\n",
              "  11,\n",
              "  'Which disorder is rated by Palmini classification?',\n",
              "  'Epilepsia 52: 158-174, 2011.',\n",
              "  0),\n",
              " (901,\n",
              "  12,\n",
              "  'Which disorder is rated by Palmini classification?',\n",
              "  ' In contrast to the former FCD classification by Palmini, which considered only histologic features, the novel ILAE classification also relies on magnetic resonance imaging (MRI) findings and presumed pathogenesis.',\n",
              "  0),\n",
              " (901,\n",
              "  13,\n",
              "  'Which disorder is rated by Palmini classification?',\n",
              "  'Rates of high frequency oscillations in patients with pathologically confirmed focal cortical dysplasia of Palmini type 1a and b were compared with those in type 2a and b.',\n",
              "  0),\n",
              " (901,\n",
              "  14,\n",
              "  'Which disorder is rated by Palmini classification?',\n",
              "  'Tuberous sclerosis complex (TSC) and severe cortical dysplasia (CD), or CD type II according to Palmini classification, share histopathologic similarities, specifically the presence of cytomegalic neurons and balloon cells.',\n",
              "  0),\n",
              " (901,\n",
              "  15,\n",
              "  'Which disorder is rated by Palmini classification?',\n",
              "  'This study also provides an opportunity to compare the predictive value of the ILAE and Palmini et al classification schemes with regard to the type I focal cortical dysplasias.',\n",
              "  0),\n",
              " (901,\n",
              "  16,\n",
              "  'Which disorder is rated by Palmini classification?',\n",
              "  'Ten of the 12 patients (83%) who had adequate tissue excised adjacent to the meningioangiomatosis demonstrated evidence of focal cortical dysplasia, with 6 of those (60%) classified as Palmini type IA, and 4 patients (40%) classified as Palmini type IIA.',\n",
              "  0),\n",
              " (901,\n",
              "  17,\n",
              "  'Which disorder is rated by Palmini classification?',\n",
              "  'Of the 91 patients, there were 50 patients with ILAE focal cortical dysplasia type Ib, 41 with ILAE focal cortical dysplasia type Ic, 63 with Palmini et al focal cortical dysplasia type IA, and 28 with Palmini et al focal cortical dysplasia type IB.',\n",
              "  0),\n",
              " (901,\n",
              "  18,\n",
              "  'Which disorder is rated by Palmini classification?',\n",
              "  'Our findings revealed no significant difference concerning surgical outcome with respect to seizure frequency for the histologic subtypes of ILAE focal cortical dysplasia type I (Ib vs Ic) or Palmini et al focal cortical dysplasia type I (IA vs IB).',\n",
              "  0),\n",
              " (901,\n",
              "  19,\n",
              "  'Which disorder is rated by Palmini classification?',\n",
              "  'Crude analysis revealed no significant difference between patients with subtypes of ILAE focal cortical dysplasia type I or Palmini et al focal cortical dysplasia type I concerning postoperative outcome according to the Engel and ILAE scoring systems on seizure frequency.',\n",
              "  0),\n",
              " (901,\n",
              "  20,\n",
              "  'Which disorder is rated by Palmini classification?',\n",
              "  \"Since early 2000, the definition of FCD has gradually been given a broader interpretation than the case described by Taylor et al., as shown in Palmini's classification (2004) or the newest classification (2011) proposed by the Neuropathology Task Force of the International League Against Epilepsy (ILAE).\",\n",
              "  0),\n",
              " (901,\n",
              "  21,\n",
              "  'Which disorder is rated by Palmini classification?',\n",
              "  \"Palmini's classification proposed in 2004 is now widely used to categorize FCD.\",\n",
              "  1),\n",
              " (901,\n",
              "  22,\n",
              "  'Which disorder is rated by Palmini classification?',\n",
              "  \"According to Palmini's classification system, these lesions were categorized as focal cortical dysplasia (FCD) type II.\",\n",
              "  1),\n",
              " (901,\n",
              "  23,\n",
              "  'Which disorder is rated by Palmini classification?',\n",
              "  'In contrast to the former FCD classification by Palmini, which considered only histologic features, the novel ILAE classification also relies on magnetic resonance imaging (MRI) findings and presumed pathogenesis.',\n",
              "  0),\n",
              " (901,\n",
              "  24,\n",
              "  'Which disorder is rated by Palmini classification?',\n",
              "  \"According to Palmini's classification system, these lesions were categorized as focal cortical dysplasia (FCD) type II.\",\n",
              "  1),\n",
              " (901,\n",
              "  25,\n",
              "  'Which disorder is rated by Palmini classification?',\n",
              "  \"According to Palmini's classification system, the following pathologic subgroups were identified: FCD type IA (3/38), FCD type IB (20/38), FCD type IIA (5/38) and FCD type IIB (5/38).\",\n",
              "  0),\n",
              " (901,\n",
              "  26,\n",
              "  'Which disorder is rated by Palmini classification?',\n",
              "  \"In addition, severe pathologic features (Palmini's classification, FCD type II) (p = 0.025) showed significant correlation with a better surgical outcome.\",\n",
              "  0),\n",
              " (901,\n",
              "  27,\n",
              "  'Which disorder is rated by Palmini classification?',\n",
              "  'Tuberous sclerosis complex (TSC) and severe cortical dysplasia (CD), or CD type II according to Palmini classification, share histopathologic similarities, specifically the presence of cytomegalic neurons and balloon cells.',\n",
              "  0),\n",
              " (901,\n",
              "  28,\n",
              "  'Which disorder is rated by Palmini classification?',\n",
              "  \"In detail, according to Palmini's classification, mild malformations of cortical development (mMCDs) were disclosed in three patients, focal cortical dysplasia (FCD) type Ia in three patients, and FCD type Ib in five patients.\",\n",
              "  0),\n",
              " (901,\n",
              "  29,\n",
              "  'Which disorder is rated by Palmini classification?',\n",
              "  'These results showed considerable improvement compared to a previous study evaluating the 2004 Palmini FCD classification.',\n",
              "  0),\n",
              " (901,\n",
              "  30,\n",
              "  'Which disorder is rated by Palmini classification?',\n",
              "  'In contrast to the former FCD classification by Palmini, which considered only histologic features, the novel ILAE classification also relies on magnetic resonance imaging (MRI) findings and presumed pathogenesis.',\n",
              "  0),\n",
              " (901,\n",
              "  31,\n",
              "  'Which disorder is rated by Palmini classification?',\n",
              "  'Rates of high frequency oscillations in patients with pathologically confirmed focal cortical dysplasia of Palmini type 1a and b were compared with those in type 2a and b.',\n",
              "  0),\n",
              " (901,\n",
              "  32,\n",
              "  'Which disorder is rated by Palmini classification?',\n",
              "  'Ten of the 12 patients (83%) who had adequate tissue excised adjacent to the meningioangiomatosis demonstrated evidence of focal cortical dysplasia, with 6 of those (60%) classified as Palmini type IA, and 4 patients (40%) classified as Palmini type IIA.',\n",
              "  0),\n",
              " (901,\n",
              "  33,\n",
              "  'Which disorder is rated by Palmini classification?',\n",
              "  'Numerous classifications of the complex structural abnormalities of focal cortical dysplasia have been proposed - from Taylor et al.',\n",
              "  1),\n",
              " (901,\n",
              "  34,\n",
              "  'Which disorder is rated by Palmini classification?',\n",
              "  'in 1971 to the last modification of Palmini classification made by Blumcke in 2011.',\n",
              "  0),\n",
              " (901,\n",
              "  35,\n",
              "  'Which disorder is rated by Palmini classification?',\n",
              "  'Tuberous sclerosis complex (TSC) and severe cortical dysplasia (CD), or CD type II according to Palmini classification, share histopathologic similarities, specifically the presence of cytomegalic neurons and balloon cells.',\n",
              "  0),\n",
              " (901,\n",
              "  36,\n",
              "  'Which disorder is rated by Palmini classification?',\n",
              "  'in 1971 to the last modification of Palmini classification made by Blumcke in 2011.',\n",
              "  0),\n",
              " (901,\n",
              "  37,\n",
              "  'Which disorder is rated by Palmini classification?',\n",
              "  'In general, three types of cortical dysplasia are recognized.Type I focal cortical dysplasia with mild symptomatic expression and late onset, is more often seen in adults, with changes present in the temporal lobe.Clinical symptoms are more severe in type II of cortical dysplasia usually seen in children.',\n",
              "  0),\n",
              " (901,\n",
              "  38,\n",
              "  'Which disorder is rated by Palmini classification?',\n",
              "  'In contrast to the former FCD classification by Palmini, which considered only histologic features, the novel ILAE classification also relies on magnetic resonance imaging (MRI) findings and presumed pathogenesis.',\n",
              "  0),\n",
              " (901,\n",
              "  39,\n",
              "  'Which disorder is rated by Palmini classification?',\n",
              "  'Since early 2000, the definition of FCD has gradually been given a broader interpretation than the case described by Taylor et al., as shown in Palmini&apos;s classification (2004) or the newest classification (2011) proposed by the Neuropathology Task Force of the International League Against Epilepsy (ILAE).',\n",
              "  0),\n",
              " (901,\n",
              "  40,\n",
              "  'Which disorder is rated by Palmini classification?',\n",
              "  'The ILAE classification describes 3 types of disease: Type I, Type II, and Type III.',\n",
              "  0),\n",
              " (901,\n",
              "  41,\n",
              "  'Which disorder is rated by Palmini classification?',\n",
              "  'in 1971 to the last modification of Palmini classification made by Blumcke in 2011.',\n",
              "  0),\n",
              " (901,\n",
              "  42,\n",
              "  'Which disorder is rated by Palmini classification?',\n",
              "  'In general, three types of cortical dysplasia are recognized.',\n",
              "  1),\n",
              " (1422,\n",
              "  0,\n",
              "  'Describe Hot water reflex epilepsy.',\n",
              "  'In this review we assess our currently available knowledge about reflex seizures with special emphasis on the difference between \"generalized\" reflex seizures induced by visual stimuli, thinking, praxis and language tasks, and \"focal\" seizures induced by startle, eating, music, hot water, somatosensory stimuli and orgasm.',\n",
              "  0),\n",
              " (1422,\n",
              "  1,\n",
              "  'Describe Hot water reflex epilepsy.',\n",
              "  'CONCLUSIONS: Intermittent clobazam prophylaxis prior to head water bath might be a preferred mode of treatment of pure HWE.',\n",
              "  1),\n",
              " (1422,\n",
              "  2,\n",
              "  'Describe Hot water reflex epilepsy.',\n",
              "  'Different types of reflex epilepsy such as eating, startle myoclonus, and hot water epilepsy were included in the study.',\n",
              "  0),\n",
              " (1422,\n",
              "  3,\n",
              "  'Describe Hot water reflex epilepsy.',\n",
              "  'Hot water epilepsy was triggered by contact with hot water either in the bath or by hand immersion, and VEEG showed fronto-parietal involvement.',\n",
              "  0),\n",
              " (1422,\n",
              "  4,\n",
              "  'Describe Hot water reflex epilepsy.',\n",
              "  'Hot water epilepsy is rarely described in European countries, where bathing epilepsy in younger children is more common and often confused with this type of epilepsy.',\n",
              "  0),\n",
              " (1422,\n",
              "  5,\n",
              "  'Describe Hot water reflex epilepsy.',\n",
              "  'We studied the anatomical correlates of reflex hot water epilepsy (HWE) using multimodality investigations viz.',\n",
              "  0),\n",
              " (1422,\n",
              "  6,\n",
              "  'Describe Hot water reflex epilepsy.',\n",
              "  'magnetic resonance imaging (MRI), electroencephalography (EEG), and single photon emission computed tomography (SPECT).',\n",
              "  0),\n",
              " (1422,\n",
              "  7,\n",
              "  'Describe Hot water reflex epilepsy.',\n",
              "  'The clinical and SPECT studies suggested temporal lobe as the seizure onset zone in some of the patients with HWE.',\n",
              "  0),\n",
              " (1422,\n",
              "  8,\n",
              "  'Describe Hot water reflex epilepsy.',\n",
              "  'Hot water epilepsy (HWE) refers to a specific type of reflex epilepsy precipitated by the stimulus of bathing in hot water.',\n",
              "  1),\n",
              " (1422,\n",
              "  9,\n",
              "  'Describe Hot water reflex epilepsy.',\n",
              "  'Pathogenesis is still unknown and temporal lobe has been thought to take part in the epileptogenesis.',\n",
              "  1),\n",
              " (1422,\n",
              "  10,\n",
              "  'Describe Hot water reflex epilepsy.',\n",
              "  'HWE can be symptomatic of focal cortical malformation, and few cases were reported.',\n",
              "  1),\n",
              " (1422,\n",
              "  11,\n",
              "  'Describe Hot water reflex epilepsy.',\n",
              "  'Hot water epilepsy is a reflex or sensory epilepsy in which seizures are triggered by the stimulus of bathing in hot water',\n",
              "  0),\n",
              " (1422,\n",
              "  12,\n",
              "  'Describe Hot water reflex epilepsy.',\n",
              "  'Hot water epilepsy (HWE) refers to a specific type of reflex epilepsy precipitated by the stimulus of bathing in hot water',\n",
              "  1),\n",
              " (1422,\n",
              "  13,\n",
              "  'Describe Hot water reflex epilepsy.',\n",
              "  'Hot water epilepsy was triggered by contact with hot water either in the bath or by hand immersion, and VEEG showed fronto-parietal involvement',\n",
              "  0),\n",
              " (1422,\n",
              "  14,\n",
              "  'Describe Hot water reflex epilepsy.',\n",
              "  'Hot water epilepsy: A rare form of reflex epilepsy',\n",
              "  0),\n",
              " (1422,\n",
              "  15,\n",
              "  'Describe Hot water reflex epilepsy.',\n",
              "  'Hot water epilepsy is a form of reflex epilepsy in childhood',\n",
              "  0),\n",
              " (1422,\n",
              "  16,\n",
              "  'Describe Hot water reflex epilepsy.',\n",
              "  'We studied the anatomical correlates of reflex hot water epilepsy (HWE) using multimodality investigations viz.',\n",
              "  0),\n",
              " (1422,\n",
              "  17,\n",
              "  'Describe Hot water reflex epilepsy.',\n",
              "  'magnetic resonance imaging (MRI), electroencephalography (EEG), and single photon emission computed tomography (SPECT)',\n",
              "  0),\n",
              " (1422,\n",
              "  18,\n",
              "  'Describe Hot water reflex epilepsy.',\n",
              "  'Hot water epilepsy is a reflex or sensory epilepsy in which seizures are triggered by the stimulus of bathing in hot water.',\n",
              "  0),\n",
              " (1422,\n",
              "  19,\n",
              "  'Describe Hot water reflex epilepsy.',\n",
              "  'The diagnosis of hot water epilepsy was supported by an ictal EEG.',\n",
              "  0),\n",
              " (1422,\n",
              "  20,\n",
              "  'Describe Hot water reflex epilepsy.',\n",
              "  'Hot water epilepsy, also known as bathing epilepsy or water-immersion epilepsy is, in the Caucasian population, a rare form of benign epilepsy, where seizures are provoked by immersion in a hot or even just a warm bath.',\n",
              "  0),\n",
              " (1422,\n",
              "  21,\n",
              "  'Describe Hot water reflex epilepsy.',\n",
              "  'This is the first comprehensive video publication of a seizure provoked by water-immersion in a Caucasian child.',\n",
              "  0),\n",
              " (1422,\n",
              "  22,\n",
              "  'Describe Hot water reflex epilepsy.',\n",
              "  'Hot water epilepsy (HWE) is a form of reflex or sensory epilepsy wherein seizures are precipitated by an unusual stimulus, the contact of hot water over the head and body.',\n",
              "  0),\n",
              " (1422,\n",
              "  23,\n",
              "  'Describe Hot water reflex epilepsy.',\n",
              "  'Hot water epilepsy (HWE) is a rare form of reflex epilepsy caused by bathing with hot water.',\n",
              "  0),\n",
              " (1422,\n",
              "  24,\n",
              "  'Describe Hot water reflex epilepsy.',\n",
              "  'In this paper, we describe three cases with hot water epilepsy.',\n",
              "  0),\n",
              " (1422,\n",
              "  25,\n",
              "  'Describe Hot water reflex epilepsy.',\n",
              "  '&quot;Hot water epilepsy&quot; (HWE), precipitated by a bath or shower in hot water, has been described infrequently in the literature.',\n",
              "  0),\n",
              " (1422,\n",
              "  26,\n",
              "  'Describe Hot water reflex epilepsy.',\n",
              "  'A video-EEG recording was taken while hot water was poured onto his chest.',\n",
              "  0),\n",
              " (1422,\n",
              "  27,\n",
              "  'Describe Hot water reflex epilepsy.',\n",
              "  'Hot water epilepsy is rarely described in European countries, where bathing epilepsy in younger children is more common and often confused with this type of epilepsy.',\n",
              "  0),\n",
              " (1422,\n",
              "  28,\n",
              "  'Describe Hot water reflex epilepsy.',\n",
              "  'Hot water epilepsy (HWE) or bathing epilepsy is one of the reflex epilepsies induced by hot water pouring over the head,',\n",
              "  0),\n",
              " (1422,\n",
              "  29,\n",
              "  'Describe Hot water reflex epilepsy.',\n",
              "  'These two patients are the youngest reported cases of reflex hot water seizures, and the only reported cases in which reflex hot water seizures subsequently manifested episodes of alternating hemiplegia.',\n",
              "  0),\n",
              " (1422,\n",
              "  30,\n",
              "  'Describe Hot water reflex epilepsy.',\n",
              "  'Hot water epilepsy is a reflex epilepsy.',\n",
              "  0),\n",
              " (1422,\n",
              "  31,\n",
              "  'Describe Hot water reflex epilepsy.',\n",
              "  'Seizures are provoked by hot water, and result from the association of both cutaneous and heat stimuli.',\n",
              "  0),\n",
              " (1422,\n",
              "  32,\n",
              "  'Describe Hot water reflex epilepsy.',\n",
              "  'Hot water epilepsy belongs to the group of reflex epilepsies.',\n",
              "  0),\n",
              " (1422,\n",
              "  33,\n",
              "  'Describe Hot water reflex epilepsy.',\n",
              "  'Seizures are provoked by hot water, due to the association of both cutaneous and heat stimuli.',\n",
              "  0),\n",
              " (1422,\n",
              "  34,\n",
              "  'Describe Hot water reflex epilepsy.',\n",
              "  'A patient with reflex epilepsy is described, in whom seizures were induced by bathing in hot water.',\n",
              "  0),\n",
              " (2641,\n",
              "  0,\n",
              "  'What do the trispecific HIV antibodies target?',\n",
              "  'We engineered trispecific antibodies (Abs) that allow a single molecule to interact with three independent HIV-1 envelope determinants: the CD4 binding site, the membrane-proximal external region (MPER), and the V1V2 glycan site.',\n",
              "  1),\n",
              " (857,\n",
              "  0,\n",
              "  'What is the Arnold-Chiari syndrome?',\n",
              "  'Arnold Chiari syndrome is a condition characterized by herniation of the cerebellar tonsils through the foramen magnum.',\n",
              "  1),\n",
              " (857,\n",
              "  1,\n",
              "  'What is the Arnold-Chiari syndrome?',\n",
              "  'The Arnold-Chiari malformation is very rare hindbrain abnormalities characterized by herniation of the hindbrain through the foramen magnum.',\n",
              "  1),\n",
              " (857,\n",
              "  2,\n",
              "  'What is the Arnold-Chiari syndrome?',\n",
              "  'The origin of Arnold-Chiari syndrome is connected with narrowness in the posterior fossa, particularly with narrowing of the arachnoid spaces.',\n",
              "  1),\n",
              " (857,\n",
              "  3,\n",
              "  'What is the Arnold-Chiari syndrome?',\n",
              "  'Arnold-Chiari Syndrome I is a malformation of the cervicomedullary junction, manifesting usually with downbeat nystagmus, palsy of the caudal cerebral nerves, headache, and vertigo.',\n",
              "  1),\n",
              " (2877,\n",
              "  0,\n",
              "  'Can enasidenib be used for the treatment of acute myeloid leukemia?',\n",
              "  'In August 2017, the United States Federal Drug Administration (FDA) approved enasidenib (Idhifa, Celgene/Agios) for adults with relapsed and refractory acute myelogenous leukemia (AML) with an IDH2 mutation.',\n",
              "  1),\n",
              " (3233,\n",
              "  0,\n",
              "  'What is aphasia?',\n",
              "  'Aphasia, the language disorder following brain damage, is frequently accompanied by deficits of working memor',\n",
              "  1),\n",
              " (3233,\n",
              "  1,\n",
              "  'What is aphasia?',\n",
              "  'People with aphasia (PWA) use pantomime, gesture in absence of speech,',\n",
              "  1),\n",
              " (679,\n",
              "  0,\n",
              "  'Which gene(s) should be genotyped in order to prescribe the drug Cetuximab (anti-EGFR)?',\n",
              "  'Treatment of metastatic colorectal cancer with targeted anti-EGFR therapeutics such as cetuximab extends survival in only 25% of patients who test wild-type for KRAS, while the majority of patients prove resistant (J Clin Oncol 28(7):1254-1261, 2010).',\n",
              "  0),\n",
              " (679,\n",
              "  1,\n",
              "  'Which gene(s) should be genotyped in order to prescribe the drug Cetuximab (anti-EGFR)?',\n",
              "  ' Somatic mutation of the EGFR signalling pathway is a prevalent mechanism of resistance to cetuximab (Nature 486(7404):532-536, 2012).',\n",
              "  1),\n",
              " (679,\n",
              "  2,\n",
              "  'Which gene(s) should be genotyped in order to prescribe the drug Cetuximab (anti-EGFR)?',\n",
              "  'If the human genome harbours variants that influence susceptibility of the EGFR pathway to oncogenic mutation, such variants could also be prognostic for cetuximab responsiveness.',\n",
              "  0),\n",
              " (679,\n",
              "  3,\n",
              "  'Which gene(s) should be genotyped in order to prescribe the drug Cetuximab (anti-EGFR)?',\n",
              "  'KRAS mutations are strong predictors for clinical outcomes of EGFR-targeted treatments such as cetuximab and panitumumab in metastatic colorectal cancer (mCRC).',\n",
              "  1),\n",
              " (679,\n",
              "  4,\n",
              "  'Which gene(s) should be genotyped in order to prescribe the drug Cetuximab (anti-EGFR)?',\n",
              "  'KRAS mutation is widely accepted as a strong, negative predictive marker for anti-epidermal growth factor receptor antibodies, including cetuximab and panitumumab.',\n",
              "  0),\n",
              " (679,\n",
              "  5,\n",
              "  'Which gene(s) should be genotyped in order to prescribe the drug Cetuximab (anti-EGFR)?',\n",
              "  'KRAS status is now a mandatory prerequisite in order to treat metastatic colorectal patients with anti-Epidermal Growth Factor Receptor (EGFR) antibodies, such as cetuximab or panitumumab.',\n",
              "  0),\n",
              " (679,\n",
              "  6,\n",
              "  'Which gene(s) should be genotyped in order to prescribe the drug Cetuximab (anti-EGFR)?',\n",
              "  'KRAS mutations are unambiguously linked to a lack of response to these targeted therapies.',\n",
              "  1),\n",
              " (679,\n",
              "  7,\n",
              "  'Which gene(s) should be genotyped in order to prescribe the drug Cetuximab (anti-EGFR)?',\n",
              "  'The EGFR-R497K polymorphism is a potential predictor for overall survival in HNSCC patients treated with cetuximab based therapy in the palliative setting.',\n",
              "  0),\n",
              " (679,\n",
              "  8,\n",
              "  'Which gene(s) should be genotyped in order to prescribe the drug Cetuximab (anti-EGFR)?',\n",
              "  'Laboratories are increasingly required to perform molecular tests for the detection of mutations in the KRAS gene in metastatic colorectal cancers to allow better clinical management and more effective treatment for these patients.',\n",
              "  0),\n",
              " (679,\n",
              "  9,\n",
              "  'Which gene(s) should be genotyped in order to prescribe the drug Cetuximab (anti-EGFR)?',\n",
              "  \"KRAS mutation status predicts a patient's likely response to the monoclonal antibody cetuximab.\",\n",
              "  0),\n",
              " (679,\n",
              "  10,\n",
              "  'Which gene(s) should be genotyped in order to prescribe the drug Cetuximab (anti-EGFR)?',\n",
              "  'KRAS mutation has been unambiguously identified as a marker of resistance to cetuximab-based treatment in metastatic colorectal cancer (mCRC) patients.',\n",
              "  1),\n",
              " (679,\n",
              "  11,\n",
              "  'Which gene(s) should be genotyped in order to prescribe the drug Cetuximab (anti-EGFR)?',\n",
              "  ' This study suggested EGF A+61G polymorphism to be a predictive marker for pCR, independent of KRAS mutation status, to cetuximab-based neoadjuvant chemoradiation of patients with locally advanced rectal cancer.',\n",
              "  0),\n",
              " (679,\n",
              "  12,\n",
              "  'Which gene(s) should be genotyped in order to prescribe the drug Cetuximab (anti-EGFR)?',\n",
              "  'Intratumoral gene expression levels of EGFR, VEGFR2 and NRP as well as polymorphisms in FCGR3A, CyclinD1 and EGFR could predict clinical outcome in mCRC patients enrolled in BOND2, independent of KRAS mutation status.',\n",
              "  0),\n",
              " (679,\n",
              "  13,\n",
              "  'Which gene(s) should be genotyped in order to prescribe the drug Cetuximab (anti-EGFR)?',\n",
              "  'Numerous clinical studies have shown that anti-EGFR therapies are effective only in a subset of patients with colorectal cancer.',\n",
              "  0),\n",
              " (679,\n",
              "  14,\n",
              "  'Which gene(s) should be genotyped in order to prescribe the drug Cetuximab (anti-EGFR)?',\n",
              "  'Even though mutations in the KRAS gene have been confirmed as negative predictors of the response to EGFR-targeted therapies, not all KRAS wild-type (wt-KRAS) patients will respond to treatment.',\n",
              "  0),\n",
              " (679,\n",
              "  15,\n",
              "  'Which gene(s) should be genotyped in order to prescribe the drug Cetuximab (anti-EGFR)?',\n",
              "  'Recent studies have demonstrated that additionally wild-type BRAF (wt-BRAF) genotype is required for response to panitumumab or cetuximab, suggesting that BRAF genotype criteria should be used together with KRAS genotype for selecting the patients who are about to benefit from the anti-EGFR therapy.',\n",
              "  0),\n",
              " (679,\n",
              "  16,\n",
              "  'Which gene(s) should be genotyped in order to prescribe the drug Cetuximab (anti-EGFR)?',\n",
              "  'KRAS mutation is a dramatic example of single nucleotide polymorphism, which is able to identify a priori patients that could receive or not an anti-EGFR monoclonal antibody such as cetuximab or panitumumab.',\n",
              "  0),\n",
              " (679,\n",
              "  17,\n",
              "  'Which gene(s) should be genotyped in order to prescribe the drug Cetuximab (anti-EGFR)?',\n",
              "  'While confirming the negative effect of KRAS mutations on outcome after cetuximab, we show that BRAF, NRAS, and PIK3CA exon 20 mutations are significantly associated with a low response rate.',\n",
              "  0),\n",
              " (679,\n",
              "  18,\n",
              "  'Which gene(s) should be genotyped in order to prescribe the drug Cetuximab (anti-EGFR)?',\n",
              "  'Cetuximab and panitumumab, mAbs targeting EGFR, are registered for metastatic colorectal carcinoma (mCRC) patients whose tumors express EGFR as determined by immunohistochemistry.',\n",
              "  0),\n",
              " (679,\n",
              "  19,\n",
              "  'Which gene(s) should be genotyped in order to prescribe the drug Cetuximab (anti-EGFR)?',\n",
              "  'However, this method is not predictive of treatment efficacy.',\n",
              "  0),\n",
              " (679,\n",
              "  20,\n",
              "  'Which gene(s) should be genotyped in order to prescribe the drug Cetuximab (anti-EGFR)?',\n",
              "  ' This study suggests that TP53 mutations are predictive of cetuximab sensitivity, particularly in patients without KRAS mutation, and that TP53 genotyping could have a clinical interest to select patients who should benefit from cetuximab-based CT.',\n",
              "  0),\n",
              " (679,\n",
              "  21,\n",
              "  'Which gene(s) should be genotyped in order to prescribe the drug Cetuximab (anti-EGFR)?',\n",
              "  'A KRAS mutation confers resistance to cetuximab, which reduces treatment options, especially in first-line.',\n",
              "  1),\n",
              " (679,\n",
              "  22,\n",
              "  'Which gene(s) should be genotyped in order to prescribe the drug Cetuximab (anti-EGFR)?',\n",
              "  'KRAS genotyping was recently introduced as predictive biomarker, since only tumors carrying a wildtype were found to respond to treatment with panitumumab.',\n",
              "  0),\n",
              " (4144,\n",
              "  0,\n",
              "  'Which particular intersex phenotype is related to steroid reductase?',\n",
              "  'Gender identity and role in a pedigree of Arabs with intersex due to 5 alpha reductase-2 deficiency',\n",
              "  1),\n",
              " (4144,\n",
              "  1,\n",
              "  'Which particular intersex phenotype is related to steroid reductase?',\n",
              "  'Between 1986 and 1995, a pedigree of six Arabs with male pseudohermaphroditism due to 5 alpha reductase-2 deficiency have been identified.',\n",
              "  1),\n",
              " (4144,\n",
              "  2,\n",
              "  'Which particular intersex phenotype is related to steroid reductase?',\n",
              "  'In the 20 yr since it was established that impairment of dihydrotestosterone formation is the cause of a rare form of human intersex, a wealth of information has accumulated about the genetics, endocrinology, and variable phenotypic manifestations, culminating in the cloning of cDNAs encoding two 5 alpha-reductase genes and documentation that mutations in the steroid 5 alpha-reductase 2 gene are the cause of 5 alpha-reductase deficiency.',\n",
              "  1),\n",
              " (4144,\n",
              "  3,\n",
              "  'Which particular intersex phenotype is related to steroid reductase?',\n",
              "  'It is also imperative to establish whether defects in steroid 5 alpha-reductase 2, perhaps in the heterozygous state, are responsible for a portion of cases of sporadic hypospadias, to determine whether 5 alpha-reductase plays a role in progesterone action in women, and to elucidate the relation between androgen action and gender role behavior.',\n",
              "  0),\n",
              " (4144,\n",
              "  4,\n",
              "  'Which particular intersex phenotype is related to steroid reductase?',\n",
              "  'Virilization of the external genitalia in the male fetus requires testosterone and dihydrotestosterone (DHT), which is formed from testosterone by the action of the enzyme, 5alpha-reductase type 2 (5alphaR-2).',\n",
              "  1),\n",
              " (4144,\n",
              "  5,\n",
              "  'Which particular intersex phenotype is related to steroid reductase?',\n",
              "  'Steroid 5-alpha-reductase 2 deficiency is an autosomal recessive disorder with clinical spectrum ranges from a male phenotype with hypospadia to a female phenotype with normal wolffian structures',\n",
              "  0),\n",
              " (4144,\n",
              "  6,\n",
              "  'Which particular intersex phenotype is related to steroid reductase?',\n",
              "  'Steroid 5-alpha-reductase 2 deficiency is a rare disorder leading to male pseudohermaphroditism, a condition characterized by incomplete differentiation of male genitalia in 46,XY patients.',\n",
              "  0),\n",
              " (4144,\n",
              "  7,\n",
              "  'Which particular intersex phenotype is related to steroid reductase?',\n",
              "  'H',\n",
              "  0),\n",
              " (4144,\n",
              "  8,\n",
              "  'Which particular intersex phenotype is related to steroid reductase?',\n",
              "  'ch impairs DHT formation (mutation within the 5alphaR-2 gene or 5alphaR-2 inhibitors) or normal function of the AR (mutation in the AR gene, antiandrogens) may result in insufficient androgen action in the male fetus and in subsequent undervirilization in the newborn.',\n",
              "  0),\n",
              " (4144,\n",
              "  9,\n",
              "  'Which particular intersex phenotype is related to steroid reductase?',\n",
              "  'Hypospadias may',\n",
              "  0),\n",
              " (4144,\n",
              "  10,\n",
              "  'Which particular intersex phenotype is related to steroid reductase?',\n",
              "  'oding sequences of the SRY gene (case 1) and 2 candidates for monogenic hypospadias, namely MAMLD1 (mastermind-like domain containing 1) and SRD5A2 (steroid-5-alpha-reductase, alpha polypeptide 2).',\n",
              "  0),\n",
              " (4144,\n",
              "  11,\n",
              "  'Which particular intersex phenotype is related to steroid reductase?',\n",
              "  'Sequencing of the entire SRY gene, incl',\n",
              "  0),\n",
              " (4144,\n",
              "  12,\n",
              "  'Which particular intersex phenotype is related to steroid reductase?',\n",
              "  'In 1974, a lack of 5α-dihydrotestosterone (5α-DHT), the most potent androgen across species except for fish, was shown to be the origin of a type of pseudohermaphrodism in which boys have female-like external genitalia.',\n",
              "  0),\n",
              " (4144,\n",
              "  13,\n",
              "  'Which particular intersex phenotype is related to steroid reductase?',\n",
              "  'Molecular studies were focused on coding sequences of the SRY gene (case 1) and 2 candidates for monogenic hypospadias, namely MAMLD1 (mastermind-like domain containing 1) and SRD5A2 (steroid-5-alpha-reductase, alpha polypeptide 2).',\n",
              "  0),\n",
              " (4144,\n",
              "  14,\n",
              "  'Which particular intersex phenotype is related to steroid reductase?',\n",
              "  'None of the identified polymorphisms cosegregated with the intersexual phenotype, thus, we cannot confirm that hypospadias may be associated with polymorphism in the coding sequence of the studied genes.',\n",
              "  0),\n",
              " (4144,\n",
              "  15,\n",
              "  'Which particular intersex phenotype is related to steroid reductase?',\n",
              "  'This animal was classified as a compound sex reversal (78,XX, SRY-negative) with the hypospadias syndrome.',\n",
              "  0),\n",
              " (4144,\n",
              "  16,\n",
              "  'Which particular intersex phenotype is related to steroid reductase?',\n",
              "  'Hypospadias in a male (78,XY; SRY-positive) dog and sex reversal female (78,XX; SRY-negative) dogs: clinical, histological and genetic studies.',\n",
              "  0),\n",
              " (4144,\n",
              "  17,\n",
              "  'Which particular intersex phenotype is related to steroid reductase?',\n",
              "  'Hence, steroid 5 alpha-reductase deficiency should be considered not only in sex reversed patients with female or ambiguous phenotypes, but also in those with mild symptoms of undermasculinization as encountered in patients with hypospadias and/or micropenis.',\n",
              "  0),\n",
              " (4144,\n",
              "  18,\n",
              "  'Which particular intersex phenotype is related to steroid reductase?',\n",
              "  'Phenotypic classification of male pseudohermaphroditism due to steroid 5 alpha-reductase 2 deficiency.',\n",
              "  1),\n",
              " (4144,\n",
              "  19,\n",
              "  'Which particular intersex phenotype is related to steroid reductase?',\n",
              "  'A classification based on the severity of the masculinization defect may be used for correlation of phenotypes with enzyme activities and genotypes, and for comparisons of phenotypes between different patients as the basis for clinical decisions to be made in patients with pseudohermaphroditism due to steroid 5 alpha-reductase 2 deficiency.',\n",
              "  0),\n",
              " (4144,\n",
              "  20,\n",
              "  'Which particular intersex phenotype is related to steroid reductase?',\n",
              "  'Inactivating mutations of the 5α-steroid reductase type-2 (SRD5A2) gene result in a broad spectrum of masculinization defects, ranging from a male phenotype with hypospadias to a female phenotype with Wolffian structures.',\n",
              "  0),\n",
              " (2309,\n",
              "  0,\n",
              "  'Which receptor does GDF15 bind?',\n",
              "  'GFRAL is the receptor for GDF15 and is required for the anti-obesity effects of the ligand.',\n",
              "  1),\n",
              " (2309,\n",
              "  1,\n",
              "  'Which receptor does GDF15 bind?',\n",
              "  'Here we show that GDF15 binds specifically to GDNF family receptor α-like (GFRAL) with high affinity, and that GFRAL requires association with the coreceptor RET to elicit intracellular signaling in response to GDF15 stimulation.',\n",
              "  1),\n",
              " (2099,\n",
              "  0,\n",
              "  'Which miRNA is targeted by SRY/Sox9?',\n",
              "  'Does the linear Sry transcript function as a ceRNA for miR-138?',\n",
              "  1),\n",
              " (2099,\n",
              "  1,\n",
              "  'Which miRNA is targeted by SRY/Sox9?',\n",
              "  'Recently, the sex determining region Y ( Sry) and the cerebellar degeneration-related protein 1 ( CDR1as) RNA transcripts have been described to function as a new class of post-transcriptional regulatory RNAs that behave as circular endogenous RNA sponges for the micro RNAs (miRNAs) miR-138 and miR-7, respectively.',\n",
              "  1),\n",
              " (2099,\n",
              "  2,\n",
              "  'Which miRNA is targeted by SRY/Sox9?',\n",
              "  ' it is reasonable to think that the linear Sry sense transcript could additionally act as a miRNA sponge, or as an endogenous competing RNA for miR-138.',\n",
              "  1),\n",
              " (2099,\n",
              "  3,\n",
              "  'Which miRNA is targeted by SRY/Sox9?',\n",
              "  'Results indicated that miR-138 directly targeted SRY-related high mobility group box 4 (SOX4) and hypoxia-inducible factor-1α (HIF-1α), and overexpression of SOX4 and HIF-1α effectively reversed the miR-138-mediated suppression of cell invasion.',\n",
              "  1),\n",
              " (2099,\n",
              "  4,\n",
              "  'Which miRNA is targeted by SRY/Sox9?',\n",
              "  'We further show that the testis-specific circRNA, sex-determining region Y (Sry), serves as a miR-138 sponge, suggesting that miRNA sponge effects achieved by circRNA formation are a general phenomenon.',\n",
              "  1),\n",
              " (1644,\n",
              "  0,\n",
              "  'Is Calcium/Calmodulin dependent protein kinase II (CaMKII) involved in cardiac arrhythmias and heart failure?',\n",
              "  'In human hypertrophy, both CaMKII and PKA functionally regulate RyR2 and may induce SR Ca(2+) leak.',\n",
              "  0),\n",
              " (1644,\n",
              "  1,\n",
              "  'Is Calcium/Calmodulin dependent protein kinase II (CaMKII) involved in cardiac arrhythmias and heart failure?',\n",
              "  'In the transition from hypertrophy to HF, the diastolic Ca(2+) leak increases and disturbed Ca(2+) cycling occurs.',\n",
              "  0),\n",
              " (1644,\n",
              "  2,\n",
              "  'Is Calcium/Calmodulin dependent protein kinase II (CaMKII) involved in cardiac arrhythmias and heart failure?',\n",
              "  'This is associated with an increase in CaMKII- but not PKA-dependent RyR2 phosphorylation.',\n",
              "  0),\n",
              " (1644,\n",
              "  3,\n",
              "  'Is Calcium/Calmodulin dependent protein kinase II (CaMKII) involved in cardiac arrhythmias and heart failure?',\n",
              "  'CaMKII inhibition may thus reflect a promising therapeutic target for the treatment of arrhythmias and contractile dysfunction.',\n",
              "  0),\n",
              " (1644,\n",
              "  4,\n",
              "  'Is Calcium/Calmodulin dependent protein kinase II (CaMKII) involved in cardiac arrhythmias and heart failure?',\n",
              "  'Ca(2+)/calmodulin-dependent protein kinase II (CaMKII) is an enzyme with important regulatory functions in the heart and brain, and its chronic activation can be pathological.',\n",
              "  0),\n",
              " (1644,\n",
              "  5,\n",
              "  'Is Calcium/Calmodulin dependent protein kinase II (CaMKII) involved in cardiac arrhythmias and heart failure?',\n",
              "  'CaMKII activation is seen in heart failure, and can directly induce pathological changes in ion channels, Ca(2+) handling and gene transcription.',\n",
              "  0),\n",
              " (1644,\n",
              "  6,\n",
              "  'Is Calcium/Calmodulin dependent protein kinase II (CaMKII) involved in cardiac arrhythmias and heart failure?',\n",
              "  'In the recent years, Ca(2+)/calmodulin-dependent protein kinase II (CaMKII) was suggested to be associated with cardiac hypertrophy and heart failure but also with arrhythmias both in animal models as well as in the human heart.',\n",
              "  0),\n",
              " (1644,\n",
              "  7,\n",
              "  'Is Calcium/Calmodulin dependent protein kinase II (CaMKII) involved in cardiac arrhythmias and heart failure?',\n",
              "  'Calcium-calmodulin-dependent protein kinase II (CaMKII) has emerged as a central mediator of cardiac stress responses which may serve several critical roles in the regulation of cardiac rhythm, cardiac contractility and growth.',\n",
              "  0),\n",
              " (1644,\n",
              "  8,\n",
              "  'Is Calcium/Calmodulin dependent protein kinase II (CaMKII) involved in cardiac arrhythmias and heart failure?',\n",
              "  'Sustained and excessive activation of CaMKII during cardiac disease has, however, been linked to arrhythmias, and maladaptive cardiac remodeling, eventually leading to heart failure (HF) and sudden cardiac death.',\n",
              "  0),\n",
              " (1644,\n",
              "  9,\n",
              "  'Is Calcium/Calmodulin dependent protein kinase II (CaMKII) involved in cardiac arrhythmias and heart failure?',\n",
              "  'Overexpression of Ca(2+)/calmodulin-dependent protein kinase II (CaMKII) in transgenic mice results in heart failure and arrhythmias.',\n",
              "  0),\n",
              " (1644,\n",
              "  10,\n",
              "  'Is Calcium/Calmodulin dependent protein kinase II (CaMKII) involved in cardiac arrhythmias and heart failure?',\n",
              "  'From recent studies, it appears evident that Ca(2+)/calmodulin-dependent protein kinase II (CaMKII) plays a central role in the arrhythmogenic processes in heart failure by sensing intracellular Ca(2+) and redox stress, affecting individual ion channels and thereby leading to electrical instability in the heart.',\n",
              "  0),\n",
              " (1644,\n",
              "  11,\n",
              "  'Is Calcium/Calmodulin dependent protein kinase II (CaMKII) involved in cardiac arrhythmias and heart failure?',\n",
              "  'CaMKII activation is proarrhythmic in heart failure where myocardium is stretched.',\n",
              "  0),\n",
              " (1644,\n",
              "  12,\n",
              "  'Is Calcium/Calmodulin dependent protein kinase II (CaMKII) involved in cardiac arrhythmias and heart failure?',\n",
              "  'The Ca-calmodulin dependent kinase II (CaMKII) seems to be involved in the development of heart failure and arrhythmias and may therefore be a promising target for the development of antiarrhythmic therapies.',\n",
              "  1),\n",
              " (1644,\n",
              "  13,\n",
              "  'Is Calcium/Calmodulin dependent protein kinase II (CaMKII) involved in cardiac arrhythmias and heart failure?',\n",
              "  'Ca(2+)/calmodulin-dependent protein kinase II (CaMKII) is up-regulated in heart failure and has been shown to cause I(Na) gating changes that mimic those induced by a point mutation in humans that is associated with combined long QT and Brugada syndromes.',\n",
              "  0),\n",
              " (1644,\n",
              "  14,\n",
              "  'Is Calcium/Calmodulin dependent protein kinase II (CaMKII) involved in cardiac arrhythmias and heart failure?',\n",
              "  'CaMKII-dependent phosphorylation of Na(V)1.5 at multiple sites (including Thr-594 and Ser-516) appears to be required to evoke loss-of-function changes in gating that could contribute to acquired Brugada syndrome-like effects in heart failure.',\n",
              "  0),\n",
              " (1644,\n",
              "  15,\n",
              "  'Is Calcium/Calmodulin dependent protein kinase II (CaMKII) involved in cardiac arrhythmias and heart failure?',\n",
              "  'Because CaMKII expression and activity are increased in cardiac hypertrophy, heart failure, and during arrhythmias both in animal models as well as in the human heart a clinical significance of CaMKII is implied.',\n",
              "  0),\n",
              " (1644,\n",
              "  16,\n",
              "  'Is Calcium/Calmodulin dependent protein kinase II (CaMKII) involved in cardiac arrhythmias and heart failure?',\n",
              "  'The multifunctional Ca(2+)- and calmodulin-dependent protein kinase II (CaMKII) is now recognized to play a central role in pathological events in the cardiovascular system.',\n",
              "  0),\n",
              " (1644,\n",
              "  17,\n",
              "  'Is Calcium/Calmodulin dependent protein kinase II (CaMKII) involved in cardiac arrhythmias and heart failure?',\n",
              "  'CaMKII has diverse downstream targets that promote vascular disease, heart failure, and arrhythmias, so improved understanding of CaMKII signaling has the potential to lead to new therapies for cardiovascular disease.',\n",
              "  0),\n",
              " (1644,\n",
              "  18,\n",
              "  'Is Calcium/Calmodulin dependent protein kinase II (CaMKII) involved in cardiac arrhythmias and heart failure?',\n",
              "  \"In our opinion, the multifunctional Ca and calmodulin-dependent protein kinase II (CaMKII) has emerged as a molecule to watch, in part because a solid body of accumulated data essentially satisfy Koch's postulates, showing that the CaMKII pathway is a core mechanism for promoting myocardial hypertrophy and heart failure.\",\n",
              "  0),\n",
              " (1644,\n",
              "  19,\n",
              "  'Is Calcium/Calmodulin dependent protein kinase II (CaMKII) involved in cardiac arrhythmias and heart failure?',\n",
              "  'Multiple groups have now confirmed the following: (1) that CaMKII activity is increased in hypertrophied and failing myocardium from animal models and patients; (2) CaMKII overexpression causes myocardial hypertrophy and HF and (3) CaMKII inhibition (by drugs, inhibitory peptides and gene deletion) improves myocardial hypertrophy and HF',\n",
              "  0),\n",
              " (1644,\n",
              "  20,\n",
              "  'Is Calcium/Calmodulin dependent protein kinase II (CaMKII) involved in cardiac arrhythmias and heart failure?',\n",
              "  ' In contrast, inhibiting the CaMKII pathway appears to reduce arrhythmias and improve myocardial responses to pathological stimuli.',\n",
              "  0),\n",
              " (1644,\n",
              "  21,\n",
              "  'Is Calcium/Calmodulin dependent protein kinase II (CaMKII) involved in cardiac arrhythmias and heart failure?',\n",
              "  'In this review, we discuss the important role of Ca(2+)/calmodulin-dependent protein kinase II (CaMKII) in the regulation of RyR2-mediated Ca(2+) release.',\n",
              "  0),\n",
              " (1644,\n",
              "  22,\n",
              "  'Is Calcium/Calmodulin dependent protein kinase II (CaMKII) involved in cardiac arrhythmias and heart failure?',\n",
              "  'In particular, we examine how pathological activation of CaMKII can lead to an increased risk of sudden arrhythmic death.',\n",
              "  0),\n",
              " (1644,\n",
              "  23,\n",
              "  'Is Calcium/Calmodulin dependent protein kinase II (CaMKII) involved in cardiac arrhythmias and heart failure?',\n",
              "  'Finally, we discuss how reduction of CaMKII-mediated RyR2 hyperactivity might reduce the risk of arrhythmias and may serve as a rationale for future pharmacotherapeutic approaches.',\n",
              "  0),\n",
              " (1644,\n",
              "  24,\n",
              "  'Is Calcium/Calmodulin dependent protein kinase II (CaMKII) involved in cardiac arrhythmias and heart failure?',\n",
              "  'Transgenic (TG) Ca(2+)/calmodulin-dependent protein kinase II (CaMKII) δ(C) mice develop systolic heart failure (HF).',\n",
              "  0),\n",
              " (1644,\n",
              "  25,\n",
              "  'Is Calcium/Calmodulin dependent protein kinase II (CaMKII) involved in cardiac arrhythmias and heart failure?',\n",
              "  'CaMKII regulates intracellular Ca(2+) handling proteins as well as sarcolemmal Na(+) channels.',\n",
              "  0),\n",
              " (1644,\n",
              "  26,\n",
              "  'Is Calcium/Calmodulin dependent protein kinase II (CaMKII) involved in cardiac arrhythmias and heart failure?',\n",
              "  'We hypothesized that CaMKII also contributes to diastolic dysfunction and arrhythmias via augmentation of the late Na(+) current (late I(Na)) in early HF (8-week-old TG mice).',\n",
              "  0),\n",
              " (1644,\n",
              "  27,\n",
              "  'Is Calcium/Calmodulin dependent protein kinase II (CaMKII) involved in cardiac arrhythmias and heart failure?',\n",
              "  'Thus, late I(Na) inhibition appears to be a promising option for diastolic dysfunction and arrhythmias in HF where CaMKII is found to be increased.',\n",
              "  0),\n",
              " (1644,\n",
              "  28,\n",
              "  'Is Calcium/Calmodulin dependent protein kinase II (CaMKII) involved in cardiac arrhythmias and heart failure?',\n",
              "  'We tested the hypothesis that increased RyR2 phosphorylation by Ca(2+)/calmodulin-dependent protein kinase II is both necessary and sufficient to promote lethal ventricular arrhythmias.',\n",
              "  0),\n",
              " (1644,\n",
              "  29,\n",
              "  'Is Calcium/Calmodulin dependent protein kinase II (CaMKII) involved in cardiac arrhythmias and heart failure?',\n",
              "  'CONCLUSIONS: our results suggest that Ca(2+)/calmodulin-dependent protein kinase II phosphorylation of RyR2 Ca(2+) release channels at S2814 plays an important role in arrhythmogenesis and sudden cardiac death in mice with heart failure.',\n",
              "  0),\n",
              " (1644,\n",
              "  30,\n",
              "  'Is Calcium/Calmodulin dependent protein kinase II (CaMKII) involved in cardiac arrhythmias and heart failure?',\n",
              "  'Excessive activation of calmodulin kinase II (CaMKII) causes arrhythmias and heart failure, but the cellular mechanisms for CaMKII-targeted proteins causing disordered cell membrane excitability and myocardial dysfunction remain uncertain.',\n",
              "  0),\n",
              " (1644,\n",
              "  31,\n",
              "  'Is Calcium/Calmodulin dependent protein kinase II (CaMKII) involved in cardiac arrhythmias and heart failure?',\n",
              "  'Transgenic (TG) Ca/calmodulin-dependent protein kinase II (CaMKII)delta(C) mice have heart failure and isoproterenol (ISO)-inducible arrhythmias.',\n",
              "  0),\n",
              " (1644,\n",
              "  32,\n",
              "  'Is Calcium/Calmodulin dependent protein kinase II (CaMKII) involved in cardiac arrhythmias and heart failure?',\n",
              "  'We hypothesized that CaMKII contributes to arrhythmias and underlying cellular events and that inhibition of CaMKII reduces cardiac arrhythmogenesis in vitro and in vivo.',\n",
              "  0),\n",
              " (1644,\n",
              "  33,\n",
              "  'Is Calcium/Calmodulin dependent protein kinase II (CaMKII) involved in cardiac arrhythmias and heart failure?',\n",
              "  'We conclude that CaMKII contributes to cardiac arrhythmogenesis in TG CaMKIIdelta(C) mice having heart failure and suggest the increased SR Ca leak as an important mechanism.',\n",
              "  0),\n",
              " (1644,\n",
              "  34,\n",
              "  'Is Calcium/Calmodulin dependent protein kinase II (CaMKII) involved in cardiac arrhythmias and heart failure?',\n",
              "  'Moreover, CaMKII inhibition reduces cardiac arrhythmias in vitro and in vivo and may therefore indicate a potential role for future antiarrhythmic therapies warranting further studies.',\n",
              "  0),\n",
              " (1644,\n",
              "  35,\n",
              "  'Is Calcium/Calmodulin dependent protein kinase II (CaMKII) involved in cardiac arrhythmias and heart failure?',\n",
              "  'Ca2+/calmodulin dependent protein kinase II (CaMKII) can phosphorylate RyR2 and modulate its activity.',\n",
              "  0),\n",
              " (1644,\n",
              "  36,\n",
              "  'Is Calcium/Calmodulin dependent protein kinase II (CaMKII) involved in cardiac arrhythmias and heart failure?',\n",
              "  'This phosphorylation positively modulates cardiac inotropic function but in extreme situations such as heart failure, elevated CaMKII activity can adversely increase Ca2+ release from the SR and lead to arrhythmogenesis.',\n",
              "  0),\n",
              " (1644,\n",
              "  37,\n",
              "  'Is Calcium/Calmodulin dependent protein kinase II (CaMKII) involved in cardiac arrhythmias and heart failure?',\n",
              "  'Calcium/calmodulin-dependent kinase II (CaMKII) is a multifunctional serine/threonine kinase expressed abundantly in the heart.',\n",
              "  0),\n",
              " (1644,\n",
              "  38,\n",
              "  'Is Calcium/Calmodulin dependent protein kinase II (CaMKII) involved in cardiac arrhythmias and heart failure?',\n",
              "  'CaMKII targets numerous proteins involved in excitation-contraction coupling and excitability, and its activation may simultaneously contribute to heart failure and cardiac arrhythmias.',\n",
              "  1),\n",
              " (1644,\n",
              "  39,\n",
              "  'Is Calcium/Calmodulin dependent protein kinase II (CaMKII) involved in cardiac arrhythmias and heart failure?',\n",
              "  'Under stress conditions, excessive CaMKII activity promotes heart failure and arrhythmias, in part through actions at Ca(2+) homeostatic proteins.',\n",
              "  0),\n",
              " (1644,\n",
              "  40,\n",
              "  'Is Calcium/Calmodulin dependent protein kinase II (CaMKII) involved in cardiac arrhythmias and heart failure?',\n",
              "  'Ca-calmodulin-dependent protein kinase II (CaMKII) was recently shown to alter Na(+) channel gating and recapitulate a human Na(+) channel genetic mutation that causes an unusual combined arrhythmogenic phenotype in patients: simultaneous long QT syndrome and Brugada syndrome.',\n",
              "  0),\n",
              " (1644,\n",
              "  41,\n",
              "  'Is Calcium/Calmodulin dependent protein kinase II (CaMKII) involved in cardiac arrhythmias and heart failure?',\n",
              "  'CaMKII is upregulated in heart failure where arrhythmias are common, and CaMKII inhibition can reduce arrhythmias.',\n",
              "  0),\n",
              " (1644,\n",
              "  42,\n",
              "  'Is Calcium/Calmodulin dependent protein kinase II (CaMKII) involved in cardiac arrhythmias and heart failure?',\n",
              "  'Thus, CaMKII-dependent channel modulation may contribute to acquired arrhythmic disease.',\n",
              "  0),\n",
              " (1644,\n",
              "  43,\n",
              "  'Is Calcium/Calmodulin dependent protein kinase II (CaMKII) involved in cardiac arrhythmias and heart failure?',\n",
              "  'In heart failure (HF), Ca(2+)/calmodulin kinase II (CaMKII) expression is increased.',\n",
              "  0),\n",
              " (1644,\n",
              "  44,\n",
              "  'Is Calcium/Calmodulin dependent protein kinase II (CaMKII) involved in cardiac arrhythmias and heart failure?',\n",
              "  'Altered Na(+) channel gating is linked to and may promote ventricular tachyarrhythmias (VTs) in HF.',\n",
              "  0),\n",
              " (1644,\n",
              "  45,\n",
              "  'Is Calcium/Calmodulin dependent protein kinase II (CaMKII) involved in cardiac arrhythmias and heart failure?',\n",
              "  'Calmodulin regulates Na(+) channel gating, in part perhaps via CaMKII.',\n",
              "  0),\n",
              " (1644,\n",
              "  46,\n",
              "  'Is Calcium/Calmodulin dependent protein kinase II (CaMKII) involved in cardiac arrhythmias and heart failure?',\n",
              "  'Thus, CaMKII-dependent regulation of Na(+) channel function may contribute to arrhythmogenesis in HF.',\n",
              "  0),\n",
              " (1644,\n",
              "  47,\n",
              "  'Is Calcium/Calmodulin dependent protein kinase II (CaMKII) involved in cardiac arrhythmias and heart failure?',\n",
              "  'Recent findings that CaMKII expression in the heart changes during hypertrophy, heart failure, myocardial ischemia, and infarction suggest that CaMKII may be a viable therapeutic target for patients suffering from common forms of heart disease.',\n",
              "  0),\n",
              " (1644,\n",
              "  48,\n",
              "  'Is Calcium/Calmodulin dependent protein kinase II (CaMKII) involved in cardiac arrhythmias and heart failure?',\n",
              "  ' Overexpression of Ca(2+)/calmodulin-dependent protein kinase II (CaMKII) in transgenic mice results in heart failure and arrhythmias.',\n",
              "  0),\n",
              " (1644,\n",
              "  49,\n",
              "  'Is Calcium/Calmodulin dependent protein kinase II (CaMKII) involved in cardiac arrhythmias and heart failure?',\n",
              "  'Transgenic (TG) Ca/calmodulin-dependent protein kinase II (CaMKII)delta(C) mice have heart failure and isoproterenol (ISO)-inducible arrhythmias.',\n",
              "  0),\n",
              " (1644,\n",
              "  50,\n",
              "  'Is Calcium/Calmodulin dependent protein kinase II (CaMKII) involved in cardiac arrhythmias and heart failure?',\n",
              "  'BACKGROUND: Transgenic (TG) Ca/calmodulin-dependent protein kinase II (CaMKII)delta(C) mice have heart failure and isoproterenol (ISO)-inducible arrhythmias.',\n",
              "  0),\n",
              " (1644,\n",
              "  51,\n",
              "  'Is Calcium/Calmodulin dependent protein kinase II (CaMKII) involved in cardiac arrhythmias and heart failure?',\n",
              "  'CaMKII targets numerous proteins involved in excitation-contraction coupling and excitability, and its activation may simultaneously contribute to heart failure and cardiac arrhythmias.',\n",
              "  1),\n",
              " (1644,\n",
              "  52,\n",
              "  'Is Calcium/Calmodulin dependent protein kinase II (CaMKII) involved in cardiac arrhythmias and heart failure?',\n",
              "  'Calcium/calmodulin-dependent protein kinase II contributes to cardiac arrhythmogenesis in heart failure.',\n",
              "  0),\n",
              " (1644,\n",
              "  53,\n",
              "  'Is Calcium/Calmodulin dependent protein kinase II (CaMKII) involved in cardiac arrhythmias and heart failure?',\n",
              "  'From recent studies, it appears evident that Ca(2+)/calmodulin-dependent protein kinase II (CaMKII) plays a central role in the arrhythmogenic processes in heart failure by sensing intracellular Ca(2+) and redox stress, affecting individual ion channels and thereby leading to electrical instability in the heart.',\n",
              "  0),\n",
              " (1644,\n",
              "  54,\n",
              "  'Is Calcium/Calmodulin dependent protein kinase II (CaMKII) involved in cardiac arrhythmias and heart failure?',\n",
              "  'Ryanodine receptor phosphorylation, calcium/calmodulin-dependent protein kinase II, and life-threatening ventricular arrhythmias.',\n",
              "  0),\n",
              " (1644,\n",
              "  55,\n",
              "  'Is Calcium/Calmodulin dependent protein kinase II (CaMKII) involved in cardiac arrhythmias and heart failure?',\n",
              "  'CaMKII targets numerous proteins involved in excitation-contraction coupling and excitability, and its activation may simultaneously contribute to heart failure and cardiac arrhythmias',\n",
              "  1),\n",
              " (1644,\n",
              "  56,\n",
              "  'Is Calcium/Calmodulin dependent protein kinase II (CaMKII) involved in cardiac arrhythmias and heart failure?',\n",
              "  'The Ca-calmodulin dependent kinase II (CaMKII) seems to be involved in the development of heart failure and arrhythmias and may therefore be a promising target for the development of antiarrhythmic therapies',\n",
              "  1),\n",
              " (1644,\n",
              "  57,\n",
              "  'Is Calcium/Calmodulin dependent protein kinase II (CaMKII) involved in cardiac arrhythmias and heart failure?',\n",
              "  'Calcium-calmodulin kinase II mediates digitalis-induced arrhythmias.',\n",
              "  0),\n",
              " (1644,\n",
              "  58,\n",
              "  'Is Calcium/Calmodulin dependent protein kinase II (CaMKII) involved in cardiac arrhythmias and heart failure?',\n",
              "  'Transgenic (TG) Ca/calmodulin-dependent protein kinase II (CaMKII)delta(C) mice have heart failure and isoproterenol (ISO)-inducible arrhythmias',\n",
              "  0),\n",
              " (3223,\n",
              "  0,\n",
              "  'Name three binding partners of cofilin 2.',\n",
              "  'Cofilin-2 (Cfl2) was identified as one of the potential targets of miR-301a using prediction databases, which we validated by luciferase assay and mutation of predicted binding sites.',\n",
              "  1),\n",
              " (3223,\n",
              "  1,\n",
              "  'Name three binding partners of cofilin 2.',\n",
              "  'In contrast to other isoforms, cofilin-2 efficiently binds and disassembles both ADP- and ATP/ADP-Pi-actin filaments.',\n",
              "  1),\n",
              " (3223,\n",
              "  2,\n",
              "  'Name three binding partners of cofilin 2.',\n",
              "  'We mapped surface-exposed cofilin-2-specific residues required for ATP-actin binding and propose that these residues function as an \"actin nucleotide-state sensor\" among ADF/cofilins.',\n",
              "  1),\n",
              " (2033,\n",
              "  0,\n",
              "  'Which two drugs were compared in the ARISTOTLE Trial?',\n",
              "  'METHODS: We analyzed patients from ARISTOTLE, a randomized trial of 18,201 patients with atrial fibrillation comparing the effects of apixaban versus warfarin on the risk of stroke or systemic embolism.',\n",
              "  0),\n",
              " (2033,\n",
              "  1,\n",
              "  'Which two drugs were compared in the ARISTOTLE Trial?',\n",
              "  ' In this study we evaluated the relationship between inflammatory biomarkers at baseline and future risk of cardiovascular events in the Apixaban for Reduction In Stroke and Other Thromboembolic Events in Atrial Fibrillation (ARISTOTLE) trial.METHODS: The ARISTOTLE trial randomised 18,201 patients with AF to apixaban or warfarin.',\n",
              "  1),\n",
              " (2033,\n",
              "  2,\n",
              "  'Which two drugs were compared in the ARISTOTLE Trial?',\n",
              "  'Moreover, apixaban was associated with decreased all-cause mortality (RR 0.89, 95% CI 0.81 to 0.99) although this finding was driven by the results of the ARISTOTLE trial.',\n",
              "  0),\n",
              " (2033,\n",
              "  3,\n",
              "  'Which two drugs were compared in the ARISTOTLE Trial?',\n",
              "  'Compared with warfarin, apixaban (in the ARISTOTLE trial) and edoxaban (in the ENGAGE-AF trial) were shown to be superior or non-inferior, respectively, for reduction in stroke or SE risk in patients with AF.',\n",
              "  0),\n",
              " (2033,\n",
              "  4,\n",
              "  'Which two drugs were compared in the ARISTOTLE Trial?',\n",
              "  'METHODS: A decision-analytic Markov model was constructed to assess the cost-effectiveness of apixaban versus warfarin, based on data from the Apixaban for Reduction in Stroke and Other Thromboembolic Events in AF (ARISTOTLE) trial.',\n",
              "  1),\n",
              " (2033,\n",
              "  5,\n",
              "  'Which two drugs were compared in the ARISTOTLE Trial?',\n",
              "  'The following phase 3 randomized trials with novel oral anticoagulants versus warfarin for stroke prevention in AF have been completed: the Randomized Evaluation of Long-term Anticoagulant therapy (RE-LY) trial with dabigatran, the Rivaroxaban Once daily oral direct Factor Xa inhibition Compared with vitamin K antagonism for prevention of stroke and Embolism Trial in Atrial Fibrillation (ROCKET-AF) trial with rivaroxaban, and the Apixaban for Reduction of Stroke and Other Thromboembolism Events in Atrial Fibrillation (ARISTOTLE) trial with apixaban.',\n",
              "  0),\n",
              " (2033,\n",
              "  6,\n",
              "  'Which two drugs were compared in the ARISTOTLE Trial?',\n",
              "  'The following phase 3 randomized trials with novel oral anticoagulants versus warfarin for stroke prevention in AF have been completed: the Randomized Evaluation of Long-term Anticoagulant therapy (RE-LY) trial with dabigatran, the Rivaroxaban Once daily oral direct Factor Xa inhibition Compared with vitamin K antagonism for prevention of stroke and Embolism Trial in Atrial Fibrillation (ROCKET-AF) trial with rivaroxaban, and the Apixaban for Reduction of Stroke and Other Thromboembolism Events in Atrial Fibrillation (ARISTOTLE) trial with apixaban',\n",
              "  0),\n",
              " (2033,\n",
              "  7,\n",
              "  'Which two drugs were compared in the ARISTOTLE Trial?',\n",
              "  'We sought to compare the efficacy and safety of the 3 new agents based on data from their published warfarin-controlled randomized trials, using the method of adjusted indirect comparisons.We included findings from 44 535 patients enrolled in 3 trials of the efficacy of dabigatran (Randomized Evaluation of Long-Term Anticoagulation Therapy [RELY]), apixaban (Apixaban for Reduction in Stroke and Other Thromboembolic Events in Atrial Fibrillation [ARISTOTLE]), and rivaroxaban (Rivaroxaban Once Daily Oral Direct Factor Xa Inhibition Compared With Vitamin K Antagonism for Prevention of Stroke and Embolism Trial in Atrial Fibrillation [ROCKET-AF]), each compared with warfarin.',\n",
              "  0),\n",
              " (2033,\n",
              "  8,\n",
              "  'Which two drugs were compared in the ARISTOTLE Trial?',\n",
              "  'A comparative analysis of three major clinical trials with factor Xa inhibitor oral anticoagulant (XOAC) drugs versus warfarin in atrial fibrillation-Rocket-AF (rivaroxaban), Aristotle (apixaban) and Engage AF Timi 48 (edoxaban; two different doses and sets of data)-was carried out.Data were extracted from the original reports (study level) and a meta-analysis was carried out.When compared with warfarin, XOAC therapy was associated with a decrease in haemorrhagic stroke, with a similar pattern for all regimens and meta-analysis showing a risk ratio of 0.488 (95% CI 0.',\n",
              "  0),\n",
              " (2033,\n",
              "  9,\n",
              "  'Which two drugs were compared in the ARISTOTLE Trial?',\n",
              "  'Apixaban compared with warfarin in patients with atrial fibrillation and previous stroke or transient ischaemic attack: a subgroup analysis of the ARISTOTLE trial.',\n",
              "  0),\n",
              " (2033,\n",
              "  10,\n",
              "  'Which two drugs were compared in the ARISTOTLE Trial?',\n",
              "  'Efficacy and safety of apixaban compared with warfarin for stroke prevention in patients with atrial fibrillation from East Asia: a subanalysis of the Apixaban for Reduction in Stroke and Other Thromboembolic Events in Atrial Fibrillation (ARISTOTLE) Trial.',\n",
              "  1),\n",
              " (2033,\n",
              "  11,\n",
              "  'Which two drugs were compared in the ARISTOTLE Trial?',\n",
              "  'Efficacy of apixaban when compared with warfarin in relation to renal function in patients with atrial fibrillation: insights from the ARISTOTLE trial.',\n",
              "  0),\n",
              " (2033,\n",
              "  12,\n",
              "  'Which two drugs were compared in the ARISTOTLE Trial?',\n",
              "  'A comparative analysis of three major clinical trials with factor Xa inhibitor oral anticoagulant (XOAC) drugs versus warfarin in atrial fibrillation-Rocket-AF (rivaroxaban), Aristotle (apixaban) and Engage AF Timi 48 (edoxaban; two different doses and sets of data)-was carried out.Data were extracted from the original reports (study level) and a meta-analysis was carried out.When compared with warfarin, XOAC therapy was associated with a decrease in haemorrhagic stroke, with a similar pattern for all regimens and meta-analysis showing a risk ratio of 0.488 (95% C',\n",
              "  0),\n",
              " (2033,\n",
              "  13,\n",
              "  'Which two drugs were compared in the ARISTOTLE Trial?',\n",
              "  'This study sought to characterize major bleeding on the basis of the components of the major bleeding definition, to explore major bleeding by location, to define 30-day mortality after a major bleeding event, and to identify factors associated with major bleeding.Apixaban was shown to reduce the risk of major hemorrhage among patients with atrial fibrillation in the ARISTOTLE (Apixaban for Reduction in Stroke and Other Thromboembolic Events in Atrial Fibrillation) trial.All patients who received at least 1 dose of a study drug were included.',\n",
              "  1),\n",
              " (2033,\n",
              "  14,\n",
              "  'Which two drugs were compared in the ARISTOTLE Trial?',\n",
              "  'Little is known about patients with 1 dose-reduction criterion who received the 5 mg twice daily dose of apixaban.To determine the frequency of 1 dose-reduction criterion and whether the effects of the 5 mg twice daily dose of apixaban on stroke or systemic embolism and bleeding varied among patients with 1 or no dose-reduction criteria.Among 18 201 patients in the ARISTOTLE trial, 17\\u202f322 were included in this analysis.',\n",
              "  0),\n",
              " (2033,\n",
              "  15,\n",
              "  'Which two drugs were compared in the ARISTOTLE Trial?',\n",
              "  'We sought to compare the efficacy and safety of the 3 new agents based on data from their published warfarin-controlled randomized trials, using the method of adjusted indirect comparisons.We included findings from 44 535 patients enrolled in 3 trials of the efficacy of dabigatran (Randomized Evaluation of Long-Term Anticoagulation Therapy [RELY]), apixaban (Apixaban for Reduction in Stroke and Other Thromboembolic Events in Atrial Fibrillation [ARISTOTLE]), and rivaroxaban (Rivaroxaban Once Daily Oral Direct Factor Xa Inhibition Compared With Vitamin K Antagonism for Prevention of Stroke and Embolism Trial i',\n",
              "  1),\n",
              " (2033,\n",
              "  16,\n",
              "  'Which two drugs were compared in the ARISTOTLE Trial?',\n",
              "  'This study aimed to evaluate the medical cost reductions associated with the use of individual NOACs instead of warfarin from the US payer perspective.Rates for efficacy and safety clinical events for warfarin were estimated as the weighted averages from the RE-LY, ROCKET-AF and ARISTOTLE trials, and event rates for NOACs were determined by applying trial hazard ratios or relative risk ratios to such weighted averages.',\n",
              "  0),\n",
              " (2033,\n",
              "  17,\n",
              "  'Which two drugs were compared in the ARISTOTLE Trial?',\n",
              "  'The ARISTOTLE trial showed apixaban was superior to warfarin for stroke prevention, significantly reduced all major bleeding, and resulted in a significant reduction in all-cause mortality.',\n",
              "  0),\n",
              " (2033,\n",
              "  18,\n",
              "  'Which two drugs were compared in the ARISTOTLE Trial?',\n",
              "  'The following phase 3 randomized trials with novel oral anticoagulants versus warfarin for stroke prevention in AF have been completed: the Randomized Evaluation of Long-term Anticoagulant therapy (RE-LY) trial with dabigatran, the Rivaroxaban Once daily oral direct Factor Xa inhibition Compared with vitamin K antagonism for prevention of stroke and Embolism Trial in Atrial Fibrillation (ROCKET-AF) trial with rivaroxaban, and the Apixaban for Reduction of Stroke and Other Thromboembolism Events in Atrial Fibrillation (ARISTOTLE) trial with apixaban.',\n",
              "  0),\n",
              " (2033,\n",
              "  19,\n",
              "  'Which two drugs were compared in the ARISTOTLE Trial?',\n",
              "  'In the ARISTOTLE trial, apixaban was superior to warfarin in the reduction of stroke or systemic embolism, major bleeding, intracranial hemorrhage, and all-cause mortality, with a similar reduction in the rate of ischemic stroke and better tolerability.',\n",
              "  0),\n",
              " (2033,\n",
              "  20,\n",
              "  'Which two drugs were compared in the ARISTOTLE Trial?',\n",
              "  'The results of the ARISTOTLE trial, which is evaluating apixaban against warfarin in ∼18\\u2009000 patients with AF, are expected to be available later this year.',\n",
              "  0),\n",
              " (2033,\n",
              "  21,\n",
              "  'Which two drugs were compared in the ARISTOTLE Trial?',\n",
              "  'The Medco data did not contain information for patients receiving apixaban as it was not on the market at the time of analysis.RESULTS: Stroke and MB rates among RW NVAF patients during warfarin exposure were higher compared with event rates in patients treated with warfarin in ARISTOTLE (stroke: 5.29 vs.',\n",
              "  0),\n",
              " (2033,\n",
              "  22,\n",
              "  'Which two drugs were compared in the ARISTOTLE Trial?',\n",
              "  'In the ARISTOTLE trial, apixaban when compared with warfarin reduced the rate of stroke, death, and bleeding.',\n",
              "  0),\n",
              " (2033,\n",
              "  23,\n",
              "  'Which two drugs were compared in the ARISTOTLE Trial?',\n",
              "  'The ARISTOTLE trial compared apixaban with warfarin in 18 201 patients with AF and ≥ 1 additional risk factor for stroke.',\n",
              "  0),\n",
              " (2033,\n",
              "  24,\n",
              "  'Which two drugs were compared in the ARISTOTLE Trial?',\n",
              "  'Compared with warfarin, apixaban (in the ARISTOTLE trial) and edoxaban (in the ENGAGE-AF trial) were shown to be superior or non-inferior, respectively, for reduction in stroke or SE risk in patients with AF.',\n",
              "  0),\n",
              " (2033,\n",
              "  25,\n",
              "  'Which two drugs were compared in the ARISTOTLE Trial?',\n",
              "  'Efficacy and safety of apixaban compared with warfarin according to age for stroke prevention in atrial fibrillation: observations from the ARISTOTLE trial.',\n",
              "  0),\n",
              " (2033,\n",
              "  26,\n",
              "  'Which two drugs were compared in the ARISTOTLE Trial?',\n",
              "  'Apixaban compared with warfarin in patients with atrial fibrillation and previous stroke or transient ischaemic attack: a subgroup analysis of the ARISTOTLE trial.',\n",
              "  0),\n",
              " (2033,\n",
              "  27,\n",
              "  'Which two drugs were compared in the ARISTOTLE Trial?',\n",
              "  'In the ARISTOTLE trial, the rate of stroke or systemic embolism was reduced by apixaban compared with warfarin in patients with atrial fibrillation (AF).',\n",
              "  0),\n",
              " (3796,\n",
              "  0,\n",
              "  'What does Retapamulin treat?',\n",
              "  ' the current study, we investigated the relationship between pleuromutilins, including valnemulin, tiamulin, and retapamulin, and 13 other antibiotics representing different mechanisms of action, against methicillin-susceptible and -resistant S. aureus',\n",
              "  0),\n",
              " (3796,\n",
              "  1,\n",
              "  'What does Retapamulin treat?',\n",
              "  'Clinical and bacteriological efficacy of twice daily topical retapamulin ointment 1% in the management of impetigo and other uncomplicated superficial skin infections',\n",
              "  0),\n",
              " (3796,\n",
              "  2,\n",
              "  'What does Retapamulin treat?',\n",
              "  'To determine clinical and bacteriological efficacy of retapamulin ointment 1% in treatment of patients with cutaneous bacterial infections caused by methicillin-resistant S. aureus (MRSA) and other bacteria.',\n",
              "  1),\n",
              " (3796,\n",
              "  3,\n",
              "  'What does Retapamulin treat?',\n",
              "  ' Fusidic acid, mupirocin, and retapamulin cover methicillin-susceptible S. aureus and streptococcal infections.',\n",
              "  1),\n",
              " (3796,\n",
              "  4,\n",
              "  'What does Retapamulin treat?',\n",
              "  'Treatment includes topical antibiotics such as mupirocin, retapamulin, and fusidic acid.',\n",
              "  0),\n",
              " (3796,\n",
              "  5,\n",
              "  'What does Retapamulin treat?',\n",
              "  'topical retapamulin ointment 1% versus oral linezolid in the treatment of secondarily infected traumatic lesions and impetigo due to methicillin-resistant Staphylococcus aureus',\n",
              "  1),\n",
              " (3796,\n",
              "  6,\n",
              "  'What does Retapamulin treat?',\n",
              "  ' To evaluate the clinical and bacteriological efficacy of topical retapamulin ointment 1% versus oral linezolid in the treatment of patients with secondarily infected traumatic lesions (SITLs; excluding abscesses) or impetigo due to methicillin-resistant Staphylococcus aureus (MRSA)',\n",
              "  0),\n",
              " (3796,\n",
              "  7,\n",
              "  'What does Retapamulin treat?',\n",
              "  'Retapamulin is a semisynthetic pleuromutilin derivative being developed as a topical antibiotic for treating bacterial infections of the skin.',\n",
              "  0),\n",
              " (3796,\n",
              "  8,\n",
              "  'What does Retapamulin treat?',\n",
              "  'OBJECTIVES: Retapamulin is the first agent of the pleuromutilin class formulated as a topical antibacterial for treating skin infections.',\n",
              "  0),\n",
              " (3796,\n",
              "  9,\n",
              "  'What does Retapamulin treat?',\n",
              "  'Retapamulin appears to be a much needed antimicrobial option for treating the AD population due to their common carriage of bacterial pathogens and frequency of infectious complications.',\n",
              "  0),\n",
              " (3796,\n",
              "  10,\n",
              "  'What does Retapamulin treat?',\n",
              "  'Retapamulin belongs to a newly developed class of antibiotics for the treatment of uncomplicated skin infections.',\n",
              "  0),\n",
              " (3796,\n",
              "  11,\n",
              "  'What does Retapamulin treat?',\n",
              "  'Retapamulin: what is the role of this topical antimicrobial in the treatment of bacterial infections in atopic dermatitis?',\n",
              "  0),\n",
              " (3796,\n",
              "  12,\n",
              "  'What does Retapamulin treat?',\n",
              "  'In vitro, retapamulin is highly potent against S. aureus and has a lower propensity to develop resistance than mupirocin.',\n",
              "  0),\n",
              " (3796,\n",
              "  13,\n",
              "  'What does Retapamulin treat?',\n",
              "  '(twice-daily) dosing of retapamulin is highly effective against impetigo due to methicillin- susceptible S. aureus and Streptococcus pyogenes and may play an important role in limiting the development of resistance against systemic agents.',\n",
              "  0),\n",
              " (3796,\n",
              "  14,\n",
              "  'What does Retapamulin treat?',\n",
              "  'In 2007, retapamulin was the first agent for human use approved in the pleuromutilin class of antibacterials in the United States (U.S.), and is the first topical antibacterial indicated to treat impetigo in over 20 years.',\n",
              "  0),\n",
              " (3796,\n",
              "  15,\n",
              "  'What does Retapamulin treat?',\n",
              "  'Retapamulin: an antibacterial with a novel mode of action in an age of emerging resistance to Staphylococcus aureus.',\n",
              "  0),\n",
              " ...]"
            ]
          },
          "metadata": {},
          "execution_count": 12
        }
      ],
      "source": [
        "training_set = []\n",
        "for d in q_train:\n",
        "    document_data = dataset[dataset['question'] == d]\n",
        "    training_set += list(document_data.itertuples(index=False, name=None))\n",
        "training_set"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 13,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "HeQ287RjOC5i",
        "outputId": "fccbaceb-cb0e-4111-9e0a-157e23a1632a"
      },
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "[(3953,\n",
              "  0,\n",
              "  'How many nucleotides long is the HOTAIR CNE?',\n",
              "  'HOTAIR was proposed to regulate either HoxD cluster genes in trans or HoxC cluster genes in cis, a mechanism that remains unclear.',\n",
              "  1),\n",
              " (3953,\n",
              "  1,\n",
              "  'How many nucleotides long is the HOTAIR CNE?',\n",
              "  'We have identified a 32-nucleotide conserved noncoding element (CNE) as HOTAIR ancient sequence that likely originated at the root of vertebrate.',\n",
              "  1),\n",
              " (3953,\n",
              "  2,\n",
              "  'How many nucleotides long is the HOTAIR CNE?',\n",
              "  'The second round of whole-genome duplication resulted in one copy of the CNE within HOTAIR and another copy embedded in noncoding transcript of HOXD11.',\n",
              "  1),\n",
              " (3953,\n",
              "  3,\n",
              "  'How many nucleotides long is the HOTAIR CNE?',\n",
              "  'Paralogous CNEs underwent compensatory mutations, exhibit sequence complementarity with respect to transcripts directionality, and have high affinity in\\xa0vitro.',\n",
              "  0),\n",
              " (3953,\n",
              "  4,\n",
              "  'How many nucleotides long is the HOTAIR CNE?',\n",
              "  'The HOTAIR CNE resembled a poised enhancer in stem cells and an active enhancer in HOTAIR-expressing cells.',\n",
              "  1),\n",
              " (3953,\n",
              "  5,\n",
              "  'How many nucleotides long is the HOTAIR CNE?',\n",
              "  'HOTAIR expression is positively correlated with HOXC11 in cis and negatively correlated with HOXD11 in trans.',\n",
              "  1),\n",
              " (1512,\n",
              "  0,\n",
              "  'Which are the subtypes of Pfeiffer syndrome?',\n",
              "  'Of 802 patients treated for craniosynostosis, 28 were identified with Pfeiffer syndrome: 17 were classified as type I (61 percent), seven were classified as type II (25 percent), and four were classified as type III (14 percent).',\n",
              "  0),\n",
              " (1512,\n",
              "  1,\n",
              "  'Which are the subtypes of Pfeiffer syndrome?',\n",
              "  'Pfeiffer syndrome is divided into three clinical subtypes.',\n",
              "  1),\n",
              " (1512,\n",
              "  2,\n",
              "  'Which are the subtypes of Pfeiffer syndrome?',\n",
              "  'Type 1 \"classic\" Pfeiffer syndrome involves individuals with mild manifestations including brachycephaly, midface hypoplasia and finger and toe abnormalities; it is associated with normal intelligence and generally good outcome.',\n",
              "  1),\n",
              " (1512,\n",
              "  3,\n",
              "  'Which are the subtypes of Pfeiffer syndrome?',\n",
              "  'Type 2 consists of cloverleaf skull, extreme proptosis, finger and toe abnormalities, elbow ankylosis or synostosis, developmental delay and neurological complications.',\n",
              "  1),\n",
              " (1512,\n",
              "  4,\n",
              "  'Which are the subtypes of Pfeiffer syndrome?',\n",
              "  'Type 3 is similar to type 2 but without a cloverleaf skull.',\n",
              "  1),\n",
              " (1512,\n",
              "  5,\n",
              "  'Which are the subtypes of Pfeiffer syndrome?',\n",
              "  'Recently, based on clinical findings, the disorder has been divided into three subtypes: type 1, characterized by mild expression; type 2, in which clover leaf skull deformity and multiple congenital anomalies are present at birth; and type 3, which is similar to type 2, but lacks the presence of the clover leaf skull at birth.',\n",
              "  0),\n",
              " (1512,\n",
              "  6,\n",
              "  'Which are the subtypes of Pfeiffer syndrome?',\n",
              "  'It is a clinically variable disorder and has been divided into three subtypes [Cohen, 1993: Am J Med Genet 45:300-307].',\n",
              "  0),\n",
              " (1512,\n",
              "  7,\n",
              "  'Which are the subtypes of Pfeiffer syndrome?',\n",
              "  'Type 1 represents the less severe cases, while types 2 and 3 are the more severe cases.',\n",
              "  0),\n",
              " (1512,\n",
              "  8,\n",
              "  'Which are the subtypes of Pfeiffer syndrome?',\n",
              "  'These latter types tend to have a higher risk for neurodevelopmental problems and a reduced life expectancy.',\n",
              "  0),\n",
              " (1512,\n",
              "  9,\n",
              "  'Which are the subtypes of Pfeiffer syndrome?',\n",
              "  'Classic Pfeiffer syndrome is designated type I.',\n",
              "  0),\n",
              " (1512,\n",
              "  10,\n",
              "  'Which are the subtypes of Pfeiffer syndrome?',\n",
              "  'Type 2 consists of cloverleaf skull with Pfeiffer hands and feet together with ankylosis of the elbows.',\n",
              "  0),\n",
              " (1512,\n",
              "  11,\n",
              "  'Which are the subtypes of Pfeiffer syndrome?',\n",
              "  'Such patients do poorly with an early death.',\n",
              "  0),\n",
              " (1512,\n",
              "  12,\n",
              "  'Which are the subtypes of Pfeiffer syndrome?',\n",
              "  'All reported instances to date have been sporadic.',\n",
              "  0),\n",
              " (1512,\n",
              "  13,\n",
              "  'Which are the subtypes of Pfeiffer syndrome?',\n",
              "  'Type 3 is similar to type 2 but without cloverleaf skull.',\n",
              "  1),\n",
              " (1512,\n",
              "  14,\n",
              "  'Which are the subtypes of Pfeiffer syndrome?',\n",
              "  'Ocular proptosis is severe in degree and the anterior cranial base is markedly short.',\n",
              "  0),\n",
              " (1512,\n",
              "  15,\n",
              "  'Which are the subtypes of Pfeiffer syndrome?',\n",
              "  'These patients also do poorly and tend to have an early death.',\n",
              "  0),\n",
              " (1512,\n",
              "  16,\n",
              "  'Which are the subtypes of Pfeiffer syndrome?',\n",
              "  'To date all cases have occurred sporadically.',\n",
              "  0),\n",
              " (1512,\n",
              "  17,\n",
              "  'Which are the subtypes of Pfeiffer syndrome?',\n",
              "  'Although these 3 clinical subtypes do not have status as separate entities, their diagnostic and prognostic implications are important.',\n",
              "  0),\n",
              " (1512,\n",
              "  18,\n",
              "  'Which are the subtypes of Pfeiffer syndrome?',\n",
              "  'Type 1 is commonly associated with normal intelligence, generally good outcome, and can be found dominantly inherited in some families.',\n",
              "  0),\n",
              " (1512,\n",
              "  19,\n",
              "  'Which are the subtypes of Pfeiffer syndrome?',\n",
              "  'Types 2 and 3 generally have severe neurological compromise, poor prognosis, early death, and sporadic occurrence.',\n",
              "  0),\n",
              " (1887,\n",
              "  0,\n",
              "  'List the three most abundant bacterial phyla present in mouse feces.',\n",
              "  'Amplicons representing the major phyla encountered in the rumen (Firmicutes, 43.7%; Proteobacteria, 28.7%; Bacteroidetes, 25.3%; Spirochea, 1.1%, and Synergistes, 1.1%) were recovered,',\n",
              "  1),\n",
              " (1887,\n",
              "  1,\n",
              "  'List the three most abundant bacterial phyla present in mouse feces.',\n",
              "  'alterations in the gut microbiota (increased Firmicutes and decreased Bacteroidetes)',\n",
              "  1),\n",
              " (1887,\n",
              "  2,\n",
              "  'List the three most abundant bacterial phyla present in mouse feces.',\n",
              "  'The taxonomic analysis of the metagenomic reads indicated that pygmy loris fecal microbiomes were dominated by Bacteroidetes and Proteobacteria phyla.',\n",
              "  1),\n",
              " (1887,\n",
              "  3,\n",
              "  'List the three most abundant bacterial phyla present in mouse feces.',\n",
              "  'Specifically, intermittent hypoxia-exposed mice showed a higher abundance of Firmicutes and a smaller abundance of Bacteroidetes and Proteobacteria phyla than controls.',\n",
              "  1),\n",
              " (678,\n",
              "  0,\n",
              "  'List types of avoided words in bacterial genomes',\n",
              "  'Avoidance of palindromic words in bacterial and archaeal genomes: a close connection with restriction enzymes.',\n",
              "  0),\n",
              " (678,\n",
              "  1,\n",
              "  'List types of avoided words in bacterial genomes',\n",
              "  'Short palindromic sequences (4, 5 and 6 bp palindromes) are avoided at a statistically significant level in the genomes of several bacteria, including the completely sequenced Haemophilus influenzae and Synechocystis sp.',\n",
              "  1),\n",
              " (678,\n",
              "  2,\n",
              "  'List types of avoided words in bacterial genomes',\n",
              "  'genomes and in the complete genome of the archaeon Methanococcus jannaschii.',\n",
              "  1),\n",
              " (678,\n",
              "  3,\n",
              "  'List types of avoided words in bacterial genomes',\n",
              "  'Palindromes corresponding to sites for restriction enzymes from other species are also avoided, albeit less significantly, suggesting that in the course of evolution bacterial DNA has been exposed to a wide spectrum of restriction enzymes, probably as the result of lateral transfer mediated by mobile genetic elements, such as plasmids and prophages',\n",
              "  1),\n",
              " (678,\n",
              "  4,\n",
              "  'List types of avoided words in bacterial genomes',\n",
              "  'Palindromic words appear to accumulate in DNA once it becomes isolated from restriction-modification systems, as demonstrated by the case of organellar genomes',\n",
              "  1),\n",
              " (678,\n",
              "  5,\n",
              "  'List types of avoided words in bacterial genomes',\n",
              "  'In certain cases, a comparison of avoided palindromic words in taxonomically related bacteria shows a pattern of relatedness of their R-M systems.',\n",
              "  0),\n",
              " (678,\n",
              "  6,\n",
              "  'List types of avoided words in bacterial genomes',\n",
              "  'Recognition sites for type II restriction and modification enzymes in genomes of several bacteria are recognized as semi-palindromic motifs and are avoided at a significant degree.',\n",
              "  1),\n",
              " (678,\n",
              "  7,\n",
              "  'List types of avoided words in bacterial genomes',\n",
              "  'On the basis of the set of most avoided words, and in accordance to the IUPAC coding standards, suggestions are made regarding potential recognition sequences.',\n",
              "  0),\n",
              " (678,\n",
              "  8,\n",
              "  'List types of avoided words in bacterial genomes',\n",
              "  'In certain cases, a comparison of avoided palindromic words in taxonomically related bacteria shows a pattern of relatedness of their R-M systems.',\n",
              "  0),\n",
              " (1051,\n",
              "  0,\n",
              "  \"What is the name of the stem loop present in the 3' end of genes encoding for selenoproteins?\",\n",
              "  'stem-loop structure called the selenocysteine incorporating sequence (SECIS)',\n",
              "  0),\n",
              " (1051,\n",
              "  1,\n",
              "  \"What is the name of the stem loop present in the 3' end of genes encoding for selenoproteins?\",\n",
              "  \"3'-UTR mRNA stem-loop termed SElenoCysteine Insertion Sequence (SECIS)\",\n",
              "  1),\n",
              " (1051,\n",
              "  2,\n",
              "  \"What is the name of the stem loop present in the 3' end of genes encoding for selenoproteins?\",\n",
              "  'Sec is inserted by a specific translational machinery that recognizes a stem-loop structure, the SECIS element',\n",
              "  0),\n",
              " (1051,\n",
              "  3,\n",
              "  \"What is the name of the stem loop present in the 3' end of genes encoding for selenoproteins?\",\n",
              "  \"Selenocysteine Insertion Sequence (SECIS) element in the 3'UTR of the transcript.\",\n",
              "  1),\n",
              " (1051,\n",
              "  4,\n",
              "  \"What is the name of the stem loop present in the 3' end of genes encoding for selenoproteins?\",\n",
              "  'The proximal stem-loop promotes Sec insertion',\n",
              "  0),\n",
              " (1051,\n",
              "  5,\n",
              "  \"What is the name of the stem loop present in the 3' end of genes encoding for selenoproteins?\",\n",
              "  'Selenocysteine is encoded by an in-frame UGA codon specified by a stem-loop structure, the Sec insertion sequence element (SECIS)',\n",
              "  0),\n",
              " (1051,\n",
              "  6,\n",
              "  \"What is the name of the stem loop present in the 3' end of genes encoding for selenoproteins?\",\n",
              "  \"3' untranslated region RNA stem loop called a SEC incorporation sequence (SECIS)\",\n",
              "  0),\n",
              " (1051,\n",
              "  7,\n",
              "  \"What is the name of the stem loop present in the 3' end of genes encoding for selenoproteins?\",\n",
              "  \"recoding of UGA as Sec depends on the selenocysteine insertion sequence (SECIS) element, a stem-loop structure in the 3' untranslated region of the transcript\",\n",
              "  0),\n",
              " (1051,\n",
              "  8,\n",
              "  \"What is the name of the stem loop present in the 3' end of genes encoding for selenoproteins?\",\n",
              "  \"this requires a dedicated machinery comprising a stem-loop structure in the 3' UTR RNA (the SECIS element)\",\n",
              "  0),\n",
              " (1051,\n",
              "  9,\n",
              "  \"What is the name of the stem loop present in the 3' end of genes encoding for selenoproteins?\",\n",
              "  'RNA stem-loop structure, the SECIS element in the 3 untranslated region of (UTR) selenoprotein mRNAs',\n",
              "  0),\n",
              " (1051,\n",
              "  10,\n",
              "  \"What is the name of the stem loop present in the 3' end of genes encoding for selenoproteins?\",\n",
              "  'recoding of the UGA stop codon to selenocysteine.',\n",
              "  0),\n",
              " (1051,\n",
              "  11,\n",
              "  \"What is the name of the stem loop present in the 3' end of genes encoding for selenoproteins?\",\n",
              "  \"In eukaryotes, this requires an RNA stem loop structure in the 3'-untranslated region, termed a selenocysteine insertion sequence (SECIS),\",\n",
              "  0),\n",
              " (1051,\n",
              "  12,\n",
              "  \"What is the name of the stem loop present in the 3' end of genes encoding for selenoproteins?\",\n",
              "  ' insertion into proteins is directed by translational recoding of specific UGA codons located upstream of a stem-loop structure known as Sec insertion sequence (SECIS) element',\n",
              "  0),\n",
              " (1051,\n",
              "  13,\n",
              "  \"What is the name of the stem loop present in the 3' end of genes encoding for selenoproteins?\",\n",
              "  \"In eukaryotes, incorporation of Sec requires a Sec insertion sequence (SECIS) element, a stem-loop structure located in the 3'-untranslated regions of selenoprotein mRNAs\",\n",
              "  0),\n",
              " (1051,\n",
              "  14,\n",
              "  \"What is the name of the stem loop present in the 3' end of genes encoding for selenoproteins?\",\n",
              "  \"selenocysteine insertion requires a cis-acting selenocysteine insertion sequence (SECIS) usually located in the 3'UTR of selenoprotein mRNAs\",\n",
              "  1),\n",
              " (1051,\n",
              "  15,\n",
              "  \"What is the name of the stem loop present in the 3' end of genes encoding for selenoproteins?\",\n",
              "  \"The Sec insertion sequence (SECIS) element, which is the stem-loop structure present in 3' untranslated regions (UTRs) of eukaryotic selenoprotein-encoding genes\",\n",
              "  0),\n",
              " (1051,\n",
              "  16,\n",
              "  \"What is the name of the stem loop present in the 3' end of genes encoding for selenoproteins?\",\n",
              "  \"For eukaryotic selenoprotein mRNAs, it has been proposed that a conserved stem-loop structure designated the Sec insertion sequence (SECIS) in the 3'-untranslated (3'-UTR) region is required for recognition of UGA as a Sec codon\",\n",
              "  0),\n",
              " (1051,\n",
              "  17,\n",
              "  \"What is the name of the stem loop present in the 3' end of genes encoding for selenoproteins?\",\n",
              "  \"3'-untranslated regions of selenoprotein genes contain a common stem-loop structure, selenocysteine insertion sequence (SECIS) element, that is necessary for decoding UGA as selenocysteine\",\n",
              "  0),\n",
              " (1051,\n",
              "  18,\n",
              "  \"What is the name of the stem loop present in the 3' end of genes encoding for selenoproteins?\",\n",
              "  'Analyses of eukaryotic selenocysteine insertion sequence (SECIS) elements via computer folding programs, mutagenesis studies, and chemical and enzymatic probing has led to the derivation of a predicted consensus structural model for these elements.',\n",
              "  0),\n",
              " (1051,\n",
              "  19,\n",
              "  \"What is the name of the stem loop present in the 3' end of genes encoding for selenoproteins?\",\n",
              "  'This model consists of a stem-loop or hairpin',\n",
              "  0),\n",
              " (1051,\n",
              "  20,\n",
              "  \"What is the name of the stem loop present in the 3' end of genes encoding for selenoproteins?\",\n",
              "  \"ECIS elements form stem-loop structures in the 3' untranslated regions (UTR) of eukaryotic mRNAs that encode selenoproteins\",\n",
              "  0),\n",
              " (1051,\n",
              "  21,\n",
              "  \"What is the name of the stem loop present in the 3' end of genes encoding for selenoproteins?\",\n",
              "  'We report a detailed experimental study of the secondary structures of the SECIS elements',\n",
              "  0),\n",
              " (1051,\n",
              "  22,\n",
              "  \"What is the name of the stem loop present in the 3' end of genes encoding for selenoproteins?\",\n",
              "  'It is characterized by a stem-loop structure',\n",
              "  0),\n",
              " (1051,\n",
              "  23,\n",
              "  \"What is the name of the stem loop present in the 3' end of genes encoding for selenoproteins?\",\n",
              "  \" in eukaryotic selenoprotein mRNAs, this stem-loop structure, the selenocysteine insertion sequence (SECIS) element, resides in the 3'-untranslated region\",\n",
              "  1),\n",
              " (1051,\n",
              "  24,\n",
              "  \"What is the name of the stem loop present in the 3' end of genes encoding for selenoproteins?\",\n",
              "  \"ECIS elements are stem-loop structures located in the 3' untranslated regions (UTRs) of eukaryotic selenoprotein mRNAs\",\n",
              "  0),\n",
              " (1051,\n",
              "  25,\n",
              "  \"What is the name of the stem loop present in the 3' end of genes encoding for selenoproteins?\",\n",
              "  \"eukaryotic selenocysteine UGA codons requires a stem-loop structure in the 3'UTR of mRNAs, the selenocysteine insertion sequence (SECIS) element\",\n",
              "  1),\n",
              " (1051,\n",
              "  26,\n",
              "  \"What is the name of the stem loop present in the 3' end of genes encoding for selenoproteins?\",\n",
              "  'stem-loops and critical nucleotides similar to those in the SECIS elements',\n",
              "  0),\n",
              " (2794,\n",
              "  0,\n",
              "  'Are ultraconserved enhancers important for normal development?',\n",
              "  'Ultraconserved Enhancers Are Required for Normal Development.',\n",
              "  1),\n",
              " (2794,\n",
              "  1,\n",
              "  'Are ultraconserved enhancers important for normal development?',\n",
              "  'However, initial deletion studies in mice revealed that loss of such extraordinarily constrained sequences had no immediate impact on viability.',\n",
              "  1),\n",
              " (2794,\n",
              "  2,\n",
              "  'Are ultraconserved enhancers important for normal development?',\n",
              "  'Here, we show that ultraconserved enhancers are required for normal development.',\n",
              "  1),\n",
              " (2794,\n",
              "  3,\n",
              "  'Are ultraconserved enhancers important for normal development?',\n",
              "  'Here, we show that ultraconserved enhancers are required for normal development.',\n",
              "  1),\n",
              " (463,\n",
              "  0,\n",
              "  'What is the effect of resveratrol on mTOR activity?',\n",
              "  'Resveratrol inhibits mTOR signaling by promoting the interaction between mTOR and DEPTOR',\n",
              "  0),\n",
              " (463,\n",
              "  1,\n",
              "  'What is the effect of resveratrol on mTOR activity?',\n",
              "  'Here, we show that RSV inhibits insulin- and leucine-stimulated mTOR signaling in C2C12 fibroblasts via a Sirt1-independent mechanism',\n",
              "  0),\n",
              " (463,\n",
              "  2,\n",
              "  'What is the effect of resveratrol on mTOR activity?',\n",
              "  'Treating C2C12 cells with RSV dramatically inhibited insulin-stimulated Akt, S6 kinase, and 4E-BP1 phosphorylation but had little effect on tyrosine phosphorylation of the insulin receptor and activation of the p44/42 MAPK signaling pathway',\n",
              "  0),\n",
              " (463,\n",
              "  3,\n",
              "  'What is the effect of resveratrol on mTOR activity?',\n",
              "  'RSV treatment also partially blocked mTOR and S6 kinase phosphorylation in TSC1/2-deficient mouse embryonic fibroblasts, suggesting the presence of an inhibitory site downstream of TSC1/2',\n",
              "  0),\n",
              " (463,\n",
              "  4,\n",
              "  'What is the effect of resveratrol on mTOR activity?',\n",
              "  'Taken together, our studies reveal that RSV inhibits leucine-stimulated mTORC1 activation by promoting mTOR/DEPTOR interaction and thus uncover a novel mechanism by which RSV negatively regulates mTOR activity',\n",
              "  1),\n",
              " (463,\n",
              "  5,\n",
              "  'What is the effect of resveratrol on mTOR activity?',\n",
              "  'Recent studies suggest that modulation of the mTOR signalling pathway could play an important role in mediating the beneficial effects of RSV.',\n",
              "  0),\n",
              " (463,\n",
              "  6,\n",
              "  'What is the effect of resveratrol on mTOR activity?',\n",
              "  'Resveratrol inhibits the mTOR mitogenic signaling evoked by oxidized LDL in smooth muscle cells',\n",
              "  0),\n",
              " (463,\n",
              "  7,\n",
              "  'What is the effect of resveratrol on mTOR activity?',\n",
              "  'esveratrol blocked the oxLDL-induced phosphorylation and activation of the PI3K/Akt/mTOR/p70S6K pathway and strongly inhibited both the DNA synthesis and proliferation of SMC',\n",
              "  0),\n",
              " (463,\n",
              "  8,\n",
              "  'What is the effect of resveratrol on mTOR activity?',\n",
              "  'At concentrations that inhibit mTOR, resveratrol suppresses cellular senescence',\n",
              "  0),\n",
              " (463,\n",
              "  9,\n",
              "  'What is the effect of resveratrol on mTOR activity?',\n",
              "  'Here we demonstrated that, at cytostatic, near-toxic concentrations, resveratrol inhibited S6 phosphorylation and prevented the senescence morphology in human cells',\n",
              "  0),\n",
              " (463,\n",
              "  10,\n",
              "  'What is the effect of resveratrol on mTOR activity?',\n",
              "  'Resveratrol downregulates PI3K/Akt/mTOR signaling pathways in human U251 glioma cells',\n",
              "  1),\n",
              " (463,\n",
              "  11,\n",
              "  'What is the effect of resveratrol on mTOR activity?',\n",
              "  'On the other hand, RSV significantly increased the association between mTOR and its inhibitor, DEPTOR',\n",
              "  0),\n",
              " (463,\n",
              "  12,\n",
              "  'What is the effect of resveratrol on mTOR activity?',\n",
              "  'Resveratrol inhibits mTOR signaling by targeting DEPTOR',\n",
              "  0),\n",
              " (463,\n",
              "  13,\n",
              "  'What is the effect of resveratrol on mTOR activity?',\n",
              "  'Recent studies suggest that suppressing the signaling pathway mediated by mTOR, a well-known energy sensor that integrates various hormonal, nutrient and environmental signals to regulate cell growth, metabolism and survival, could play an important role in mediating the beneficial effect of RSV',\n",
              "  0),\n",
              " (463,\n",
              "  14,\n",
              "  'What is the effect of resveratrol on mTOR activity?',\n",
              "  'Resveratrol engages AMPK to attenuate ERK and mTOR signaling in sensory neurons and inhibits incision-induced acute and chronic pain',\n",
              "  0),\n",
              " (463,\n",
              "  15,\n",
              "  'What is the effect of resveratrol on mTOR activity?',\n",
              "  'Our results demonstrate that resveratrol profoundly inhibits ERK and mTOR signaling in sensory neurons in a time- and concentration-dependent fashion and that these effects are mediated by AMPK activation and independent of sirtuin activity',\n",
              "  0),\n",
              " (463,\n",
              "  16,\n",
              "  'What is the effect of resveratrol on mTOR activity?',\n",
              "  'Resveratrol enhances the antitumor effects of temozolomide in glioblastoma via ROS-dependent AMPK-TSC-mTOR signaling pathway',\n",
              "  0),\n",
              " (463,\n",
              "  17,\n",
              "  'What is the effect of resveratrol on mTOR activity?',\n",
              "  'Resveratrol enhances the anti-tumor activity of the mTOR inhibitor rapamycin in multiple breast cancer cell lines mainly by suppressing rapamycin-induced AKT signaling.',\n",
              "  0),\n",
              " (463,\n",
              "  18,\n",
              "  'What is the effect of resveratrol on mTOR activity?',\n",
              "  'Resveratrol inhibits mTOR signaling by promoting the interaction between mTOR and DEPTOR.',\n",
              "  0),\n",
              " (463,\n",
              "  19,\n",
              "  'What is the effect of resveratrol on mTOR activity?',\n",
              "  'Consistent with the in vitro findings, resveratrol intervention in the PTEN knockout mouse model was associated with reduction in the prostatic levels of mTOR complex 1 (mTORC1) activity and increased expression of SIRT1.',\n",
              "  0),\n",
              " (463,\n",
              "  20,\n",
              "  'What is the effect of resveratrol on mTOR activity?',\n",
              "  'Resveratrol inhibited the phosphorylation of PI3K, AKT and mTOR.',\n",
              "  0),\n",
              " (463,\n",
              "  21,\n",
              "  'What is the effect of resveratrol on mTOR activity?',\n",
              "  'Resveratrol inhibits the mTOR mitogenic signaling evoked by oxidized LDL in smooth muscle cells.',\n",
              "  0),\n",
              " (463,\n",
              "  22,\n",
              "  'What is the effect of resveratrol on mTOR activity?',\n",
              "  'Resveratrol also alleviated the PI3K/Akt/mTOR signaling by down-regulation of Akt phosphorylation and up-regulation of PTEN expression.',\n",
              "  0),\n",
              " (463,\n",
              "  23,\n",
              "  'What is the effect of resveratrol on mTOR activity?',\n",
              "  'Resveratrol blocked the oxLDL-induced phosphorylation and activation of the PI3K/Akt/mTOR/p70S6K pathway and strongly inhibited both the DNA synthesis and proliferation of SMC.',\n",
              "  0),\n",
              " (463,\n",
              "  24,\n",
              "  'What is the effect of resveratrol on mTOR activity?',\n",
              "  'Resveratrol reduced phosphorylation of ribosomal protein S6 and the mTOR inhibitor rapamycin further enhanced resveratrol-induced cell death.',\n",
              "  0),\n",
              " (463,\n",
              "  25,\n",
              "  'What is the effect of resveratrol on mTOR activity?',\n",
              "  'Resveratrol downregulates PI3K/Akt/mTOR signaling pathways in human U251 glioma cells.',\n",
              "  1),\n",
              " (463,\n",
              "  26,\n",
              "  'What is the effect of resveratrol on mTOR activity?',\n",
              "  'Our results demonstrate that resveratrol profoundly inhibits ERK and mTOR signaling in sensory neurons in a time- and concentration-dependent fashion and that these effects are mediated by AMPK activation and independent of sirtuin activity.',\n",
              "  0),\n",
              " (463,\n",
              "  27,\n",
              "  'What is the effect of resveratrol on mTOR activity?',\n",
              "  'The SIRT1 activator resveratrol reduces, whereas SIRT1 inhibitor nicotinamide enhances mTOR activity in a SIRT1 dependent manner.',\n",
              "  0),\n",
              " (463,\n",
              "  28,\n",
              "  'What is the effect of resveratrol on mTOR activity?',\n",
              "  'We conclude that resveratrol is an inhibitor of global protein synthesis, and that this effect is mediated through modulation of mTOR-dependent and independent signaling.',\n",
              "  0),\n",
              " (463,\n",
              "  29,\n",
              "  'What is the effect of resveratrol on mTOR activity?',\n",
              "  'RSV treatment has no effect on the expression levels of mTOR, raptor and DEPTOR, but greatly promotes the interaction between mTOR and its inhibitor DEPTOR.',\n",
              "  0),\n",
              " (463,\n",
              "  30,\n",
              "  'What is the effect of resveratrol on mTOR activity?',\n",
              "  'Taken together, our studies reveal that RSV inhibits leucine-stimulated mTORC1 activation by promoting mTOR/DEPTOR interaction and thus uncover a novel mechanism by which RSV negatively regulates mTOR activity.',\n",
              "  1),\n",
              " (463,\n",
              "  31,\n",
              "  'What is the effect of resveratrol on mTOR activity?',\n",
              "  'Furthermore, the inhibitory effect of RSV on leucine-stimulated mTOR signaling was greatly reduced in cells in which the expression levels of DEPTOR were suppressed by RNAi',\n",
              "  0),\n",
              " (463,\n",
              "  32,\n",
              "  'What is the effect of resveratrol on mTOR activity?',\n",
              "  'Consistent with the in vitro findings, resveratrol intervention in the PTEN knockout mouse model was associated with reduction in the prostatic levels of mTOR complex 1 (mTORC1) activity and increased expression of SIRT1',\n",
              "  0),\n",
              " (463,\n",
              "  33,\n",
              "  'What is the effect of resveratrol on mTOR activity?',\n",
              "  'The SIRT1 activator resveratrol reduces, whereas SIRT1 inhibitor nicotinamide enhances mTOR activity in a SIRT1 dependent manner',\n",
              "  0),\n",
              " (463,\n",
              "  34,\n",
              "  'What is the effect of resveratrol on mTOR activity?',\n",
              "  'However, most frequently, RSV is found to inhibit the activity of the mTOR pathway proteins, and to activate AMPK and LKB1, which can suppress mTOR signalling',\n",
              "  0),\n",
              " (463,\n",
              "  35,\n",
              "  'What is the effect of resveratrol on mTOR activity?',\n",
              "  'Over the past few years, numerous studies have suggested that suppressing the activity of mammalian target of rapamycin (mTOR), a critical regulator of cell metabolism, growth, and proliferation, may provide a key mechanism underlying the anticarcinogenic properties of resveratrol',\n",
              "  0),\n",
              " (463,\n",
              "  36,\n",
              "  'What is the effect of resveratrol on mTOR activity?',\n",
              "  'Resveratrol blocks specifically this pathway, thereby inhibiting oxLDL-induced SMC proliferation.',\n",
              "  0),\n",
              " (463,\n",
              "  37,\n",
              "  'What is the effect of resveratrol on mTOR activity?',\n",
              "  'Resveratrol causes cell cycle arrest and induces apoptotic cell death in various types of cancer cells',\n",
              "  0),\n",
              " (463,\n",
              "  38,\n",
              "  'What is the effect of resveratrol on mTOR activity?',\n",
              "  'Resveratrol decreased both the expression and phosphorylation of Akt',\n",
              "  0),\n",
              " (463,\n",
              "  39,\n",
              "  'What is the effect of resveratrol on mTOR activity?',\n",
              "  ' Modulation of the AMPK, Akt and mTOR pathways',\n",
              "  0),\n",
              " (463,\n",
              "  40,\n",
              "  'What is the effect of resveratrol on mTOR activity?',\n",
              "  'RSV (2.5-5 μM) inhibited clonogenic survival of PC3 and 22RV1 cells but not of normal prostate PNT1A cells',\n",
              "  0),\n",
              " (463,\n",
              "  41,\n",
              "  'What is the effect of resveratrol on mTOR activity?',\n",
              "  'RSV enhanced IR-activation of ATM and AMPK but inhibited basal and IR-induced phosphorylation of Akt',\n",
              "  0),\n",
              " (463,\n",
              "  42,\n",
              "  'What is the effect of resveratrol on mTOR activity?',\n",
              "  'Our results suggest that RSV arrests cell cycle, promotes apoptosis and sensitizes PrCa cells to IR likely through a desirable dual action to activate the ATM-AMPK-p53-p21(cip1)/p27(kip1) and inhibit the Akt signalling pathways',\n",
              "  0),\n",
              " (463,\n",
              "  43,\n",
              "  'What is the effect of resveratrol on mTOR activity?',\n",
              "  'Resveratrol pre-treatment reduces early inflammatory responses induced by status epilepticus via mTOR signaling',\n",
              "  0),\n",
              " (463,\n",
              "  44,\n",
              "  'What is the effect of resveratrol on mTOR activity?',\n",
              "  'Over the past few years, numerous studies have suggested that suppressing the activity of mammalian target of rapamycin (mTOR), a critical regulator of cell metabolism, growth, and proliferation, may provide a key mechanism underlying the anticarcinogenic properties of resveratrol.',\n",
              "  0),\n",
              " (463,\n",
              "  45,\n",
              "  'What is the effect of resveratrol on mTOR activity?',\n",
              "  'It has been found that resveratrol targets multiple components of the phosphatidylinositol 3- kinase(PI3K)/Akt and mTOR signaling pathways, including PI3K, Akt, PTEN, and DEPTOR, suggesting that this natural compound and its derivatives may offer a promising new cancer treatment.',\n",
              "  1),\n",
              " (463,\n",
              "  46,\n",
              "  'What is the effect of resveratrol on mTOR activity?',\n",
              "  'Autophagic cell death induced by resveratrol depends on the Ca(2+)/AMPK/mTOR pathway in A549 cells',\n",
              "  0),\n",
              " (463,\n",
              "  47,\n",
              "  'What is the effect of resveratrol on mTOR activity?',\n",
              "  'In conclusion, we demonstrate that resveratrol-induced A549 cell death was mediated by the process of autophagic cell death via Ca(2+)/AMPK-mTOR signaling pathway.',\n",
              "  0),\n",
              " (1725,\n",
              "  0,\n",
              "  'What is a disordered protein?',\n",
              "  'Intrinsically disordered proteins (IDPs) are ubiquitously involved in cellular processes and often implicated in human pathological conditions.',\n",
              "  1),\n",
              " (1725,\n",
              "  1,\n",
              "  'What is a disordered protein?',\n",
              "  'The critical biological roles of these proteins, despite not adopting a well-defined fold, encouraged structural biologists to revisit their views on the protein structure-function paradigm.',\n",
              "  0),\n",
              " (1725,\n",
              "  2,\n",
              "  'What is a disordered protein?',\n",
              "  'Intrinsically disordered proteins (IDPs) frequently function in protein interaction networks that regulate crucial cellular signaling pathways.',\n",
              "  1),\n",
              " (1725,\n",
              "  3,\n",
              "  'What is a disordered protein?',\n",
              "  'Many IDPs undergo transitions from disordered conformational ensembles to folded structures upon binding to their cellular targets.',\n",
              "  1),\n",
              " (1725,\n",
              "  4,\n",
              "  'What is a disordered protein?',\n",
              "  'The intrinsically disordered protein (IDP) stathmin plays an important regulatory role in cytoskeletal maintenance through its helical binding to tubulin and microtubules.',\n",
              "  0),\n",
              " (1725,\n",
              "  5,\n",
              "  'What is a disordered protein?',\n",
              "  'However, it lacks a stable fold in the absence of its binding partner.',\n",
              "  0),\n",
              " (1725,\n",
              "  6,\n",
              "  'What is a disordered protein?',\n",
              "  'Intrinsically disordered proteins lack stable tertiary and/or secondary structures under physiological conditions in vitro.',\n",
              "  1),\n",
              " (1725,\n",
              "  7,\n",
              "  'What is a disordered protein?',\n",
              "  'Intrinsically disordered proteins undergo significant conformational transitions to well folded forms only on binding to partner.',\n",
              "  1),\n",
              " (1725,\n",
              "  8,\n",
              "  'What is a disordered protein?',\n",
              "  'DisCons is an openly accessible sequence analysis tool that identifies and highlights structurally disordered segments of proteins where the conformational flexibility is conserved across homologs, and therefore potentially functional.',\n",
              "  0),\n",
              " (2851,\n",
              "  0,\n",
              "  'Is TIAM1 favoring tumor progression in colorectal cancer (CRC)?',\n",
              "  'Here, we identify TIAM1 as a critical antagonist of CRC progression through inhibiting TAZ and YAP, effectors of WNT signaling.',\n",
              "  1),\n",
              " (2851,\n",
              "  1,\n",
              "  'Is TIAM1 favoring tumor progression in colorectal cancer (CRC)?',\n",
              "  'We demonstrate that TIAM1 shuttles between the cytoplasm and nucleus antagonizing TAZ/YAP by distinct mechanisms in the two compartments.',\n",
              "  0),\n",
              " (2851,\n",
              "  2,\n",
              "  'Is TIAM1 favoring tumor progression in colorectal cancer (CRC)?',\n",
              "  'In the cytoplasm, TIAM1 localizes to the destruction complex and promotes TAZ degradation by enhancing its interaction with βTrCP.',\n",
              "  0),\n",
              " (2851,\n",
              "  3,\n",
              "  'Is TIAM1 favoring tumor progression in colorectal cancer (CRC)?',\n",
              "  'Nuclear TIAM1 suppresses TAZ/YAP interaction with TEADs, inhibiting expression of TAZ/YAP target genes implicated in epithelial-mesenchymal transition, cell migration, and invasion, and consequently suppresses CRC cell migration and invasion.',\n",
              "  0),\n",
              " (2851,\n",
              "  4,\n",
              "  'Is TIAM1 favoring tumor progression in colorectal cancer (CRC)?',\n",
              "  'Importantly, high nuclear TIAM1 in clinical specimens associates with increased CRC patient survival.',\n",
              "  0),\n",
              " (2851,\n",
              "  5,\n",
              "  'Is TIAM1 favoring tumor progression in colorectal cancer (CRC)?',\n",
              "  'Together, our findings suggest that in CRC TIAM1 suppresses tumor progression by regulating YAP/TAZ activity.',\n",
              "  1),\n",
              " (2851,\n",
              "  6,\n",
              "  'Is TIAM1 favoring tumor progression in colorectal cancer (CRC)?',\n",
              "  'Together, our findings suggest that in CRC TIAM1 suppresses tumor progression by regulating YAP/TAZ activity.',\n",
              "  1),\n",
              " (2851,\n",
              "  7,\n",
              "  'Is TIAM1 favoring tumor progression in colorectal cancer (CRC)?',\n",
              "  'Nuclear TIAM1 suppresses TAZ/YAP interaction with TEADs, inhibiting expression of TAZ/YAP target genes implicated in epithelial-mesenchymal transition, cell migration, and invasion, and consequently suppresses CRC cell migration and invasion.',\n",
              "  0),\n",
              " (2851,\n",
              "  8,\n",
              "  'Is TIAM1 favoring tumor progression in colorectal cancer (CRC)?',\n",
              "  'Using an orthotopic xenograft model in nude mice, we confirmed that Tiam1 silencing could reduce tumor growth by subcutaneous injection and could suppress lung and liver metastases of colorectal cancer cells.',\n",
              "  1),\n",
              " (2851,\n",
              "  9,\n",
              "  'Is TIAM1 favoring tumor progression in colorectal cancer (CRC)?',\n",
              "  'Together, our findings suggest that in CRC TIAM1 suppresses tumor progression by regulating YAP/TAZ activity.<br>',\n",
              "  1),\n",
              " (1362,\n",
              "  0,\n",
              "  'Which protein is the main inhibitor of protein phosphatase 1 (PP1)?',\n",
              "  'The type 1 protein phosphatase (PP1) is a critical negative regulator of Ca(2+) cycling and contractility in the cardiomyocyte.',\n",
              "  0),\n",
              " (1362,\n",
              "  1,\n",
              "  'Which protein is the main inhibitor of protein phosphatase 1 (PP1)?',\n",
              "  'In particular, it mediates restoration of cardiac function to basal levels, after beta-adrenergic stimulation, by dephosphorylating key phospho-proteins.',\n",
              "  0),\n",
              " (1362,\n",
              "  2,\n",
              "  'Which protein is the main inhibitor of protein phosphatase 1 (PP1)?',\n",
              "  'PP1 is a holoenzyme comprised of its catalytic and auxiliary subunits.',\n",
              "  0),\n",
              " (1362,\n",
              "  3,\n",
              "  'Which protein is the main inhibitor of protein phosphatase 1 (PP1)?',\n",
              "  \"These regulatory proteins dictate PP1's subcellular localization, substrate specificity and activity.\",\n",
              "  0),\n",
              " (1362,\n",
              "  4,\n",
              "  'Which protein is the main inhibitor of protein phosphatase 1 (PP1)?',\n",
              "  'Amongst them, inhibitor-1 is of particular importance since it has been implicated as an integrator of multiple neurohormonal pathways, which finely regulate PP1 activity, at the level of the sarcoplasmic reticulum (SR).',\n",
              "  0),\n",
              " (1362,\n",
              "  5,\n",
              "  'Which protein is the main inhibitor of protein phosphatase 1 (PP1)?',\n",
              "  'In fact, perturbations in the regulation of PP1 by inhibitor-1 have been implicated in the pathogenesis of heart failure, suggesting that inhibitor-1-based therapeutic interventions may ameliorate cardiac dysfunction and remodeling in the failing heart.',\n",
              "  0),\n",
              " (1362,\n",
              "  6,\n",
              "  'Which protein is the main inhibitor of protein phosphatase 1 (PP1)?',\n",
              "  'Aberrant beta-adrenergic signaling and depressed calcium homeostasis, associated with an imbalance of protein kinase A and phosphatase-1 activities, are hallmarks of heart failure.',\n",
              "  0),\n",
              " (1362,\n",
              "  7,\n",
              "  'Which protein is the main inhibitor of protein phosphatase 1 (PP1)?',\n",
              "  'Phosphatase-1 is restrained by its endogenous inhibitor, protein phosphatase inhibitor-1 (PPI-1)',\n",
              "  0),\n",
              " (1362,\n",
              "  8,\n",
              "  'Which protein is the main inhibitor of protein phosphatase 1 (PP1)?',\n",
              "  'Using this technology, we showed that inhibition of the protein phosphatase 1, by its constitutively active inhibitor-1, significantly increases cardiac contractility and calcium handling.',\n",
              "  0),\n",
              " (1362,\n",
              "  9,\n",
              "  'Which protein is the main inhibitor of protein phosphatase 1 (PP1)?',\n",
              "  'Thus, long-term cardiac specific overexpression of the protein phosphatase 1 inhibitor-1 and the associated increases in cardiac contractility appear to herald changes in a rather small number of proteins, which may reflect important compensatory adaptations in a hyperdynamic heart [corrected]',\n",
              "  0),\n",
              " (1362,\n",
              "  10,\n",
              "  'Which protein is the main inhibitor of protein phosphatase 1 (PP1)?',\n",
              "  'Inhibitor-1 becomes a potent inhibitor of protein phosphatase 1 when phosphorylated by cAMP-dependent protein kinase at Thr(35).',\n",
              "  1),\n",
              " (1362,\n",
              "  11,\n",
              "  'Which protein is the main inhibitor of protein phosphatase 1 (PP1)?',\n",
              "  'Moreover, Ser(67) of inhibitor-1 serves as a substrate for cyclin-dependent kinase 5 in the brain.',\n",
              "  0),\n",
              " (1362,\n",
              "  12,\n",
              "  'Which protein is the main inhibitor of protein phosphatase 1 (PP1)?',\n",
              "  'Inhibitor 1 (I-1) is a protein inhibitor of protein phosphatase 1 (PP1), a major eukaryotic Ser/Thr phosphatase.',\n",
              "  1),\n",
              " (1362,\n",
              "  13,\n",
              "  'Which protein is the main inhibitor of protein phosphatase 1 (PP1)?',\n",
              "  'Nonphosphorylated I-1 is inactive, whereas phosphorylated I-1 is a potent PP1 inhibitor.',\n",
              "  0),\n",
              " (1362,\n",
              "  14,\n",
              "  'Which protein is the main inhibitor of protein phosphatase 1 (PP1)?',\n",
              "  'Ser67-phosphorylated inhibitor 1 is a potent protein phosphatase 1 inhibitor.',\n",
              "  0),\n",
              " (1362,\n",
              "  15,\n",
              "  'Which protein is the main inhibitor of protein phosphatase 1 (PP1)?',\n",
              "  'PP1 activity was inhibited by protamine, heparin, okadaic acid (IC50 50 nM) and mammalian inhibitor-1 (IC50 2 nM).',\n",
              "  0),\n",
              " (1362,\n",
              "  16,\n",
              "  'Which protein is the main inhibitor of protein phosphatase 1 (PP1)?',\n",
              "  'On the other hand.',\n",
              "  0),\n",
              " (1362,\n",
              "  17,\n",
              "  'Which protein is the main inhibitor of protein phosphatase 1 (PP1)?',\n",
              "  'The cDNA encoding human brain protein phosphatase inhibitor-1 (I-1) was expressed in Escherichia coli.',\n",
              "  0),\n",
              " (1362,\n",
              "  18,\n",
              "  'Which protein is the main inhibitor of protein phosphatase 1 (PP1)?',\n",
              "  'Following PKA phosphorylation at a threonine, recombinant human I-1 was indistinguishable from rabbit skeletal muscle I-1 as a potent and specific inhibitor of the type-1 protein serine/threonine phosphatase (PP1).',\n",
              "  0),\n",
              " (1362,\n",
              "  19,\n",
              "  'Which protein is the main inhibitor of protein phosphatase 1 (PP1)?',\n",
              "  'N-Terminal phosphopeptides of I-1 that retained the selectivity of intact human I-1 highlighted a functional domain that mediates PP1 inhibition.',\n",
              "  0),\n",
              " (1362,\n",
              "  20,\n",
              "  'Which protein is the main inhibitor of protein phosphatase 1 (PP1)?',\n",
              "  'Protein phosphatase inhibitor-1 was purified from bovine adipose tissue.',\n",
              "  0),\n",
              " (1362,\n",
              "  21,\n",
              "  'Which protein is the main inhibitor of protein phosphatase 1 (PP1)?',\n",
              "  \"The protein had an apparent molecular mass of 32 kDa by SDS/PAGE and a Stokes' radius of 3.4 nm.\",\n",
              "  0),\n",
              " (1362,\n",
              "  22,\n",
              "  'Which protein is the main inhibitor of protein phosphatase 1 (PP1)?',\n",
              "  'It was phosphorylated by cAMP-dependent protein kinase on a threonyl residue; this phosphorylation was necessary for inhibition of protein phosphatase-1.',\n",
              "  0),\n",
              " (1362,\n",
              "  23,\n",
              "  'Which protein is the main inhibitor of protein phosphatase 1 (PP1)?',\n",
              "  'Bovine adipose tissue inhibitor-1 was compared directly with rabbit skeletal muscle inhibitor-1 and with a 32000-Mr, dopamine- and cAMP-regulated phosphoprotein from bovine brain (DARPP-32), also an inhibitor of protein phosphatase-1.',\n",
              "  0),\n",
              " (1362,\n",
              "  24,\n",
              "  'Which protein is the main inhibitor of protein phosphatase 1 (PP1)?',\n",
              "  'Protein inhibitor-1 (I-1) specifically inhibits protein phosphatase 1 (PP1), the predominant PLB phosphatase in heart.',\n",
              "  0),\n",
              " (1362,\n",
              "  25,\n",
              "  'Which protein is the main inhibitor of protein phosphatase 1 (PP1)?',\n",
              "  'Inhibitor 1 (I-1) is a protein inhibitor of protein phosphatase 1 (PP1), the predominating Ser/Thr phosphatase in the heart.',\n",
              "  1),\n",
              " (1362,\n",
              "  26,\n",
              "  'Which protein is the main inhibitor of protein phosphatase 1 (PP1)?',\n",
              "  'Inhibitor-1 (I-1) is a selective inhibitor of protein phosphatase-1 (PP1) and regulates several PP1-dependent signaling pathways, including cardiac contractility and regulation of learning and memory.',\n",
              "  0),\n",
              " (1362,\n",
              "  27,\n",
              "  'Which protein is the main inhibitor of protein phosphatase 1 (PP1)?',\n",
              "  'Inhibitor 1 (I-1) is a protein inhibitor of protein phosphatase 1 (PP1), a major eukaryotic Ser/Thr phosphatase.',\n",
              "  1),\n",
              " (1362,\n",
              "  28,\n",
              "  'Which protein is the main inhibitor of protein phosphatase 1 (PP1)?',\n",
              "  'Inhibitor 1 (I-1) is a protein inhibitor of protein phosphatase 1 (PP1), the predominating Ser/Thr phosphatase in the heart',\n",
              "  1),\n",
              " (1362,\n",
              "  29,\n",
              "  'Which protein is the main inhibitor of protein phosphatase 1 (PP1)?',\n",
              "  'Protein inhibitor-1 (I-1) specifically inhibits protein phosphatase 1 (PP1), the predominant PLB phosphatase in heart',\n",
              "  0),\n",
              " (990,\n",
              "  0,\n",
              "  'Dracorhodin perchlorate was tested for treatment of which cancers?',\n",
              "  'Dracorhodin perchlorate induced human breast cancer MCF-7 apoptosis through mitochondrial pathways.',\n",
              "  0),\n",
              " (990,\n",
              "  1,\n",
              "  'Dracorhodin perchlorate was tested for treatment of which cancers?',\n",
              "  'It was reported that DP could induce apoptosis in human prostate cancer, human gastric tumor cells and human melanoma, but the cytotoxic effect of DP on human breast cancer was not investigated.',\n",
              "  1),\n",
              " (990,\n",
              "  2,\n",
              "  'Dracorhodin perchlorate was tested for treatment of which cancers?',\n",
              "  'CONCLUSION: Therefore DP was a candidate for anti-breast cancer, DP induced apoptosis of MCF-7 through mitochondrial pathway.',\n",
              "  0),\n",
              " (990,\n",
              "  3,\n",
              "  'Dracorhodin perchlorate was tested for treatment of which cancers?',\n",
              "  'Dracorhodin perchlorate has been recently shown to induce apoptotic cell death in cancer cells.',\n",
              "  0),\n",
              " (990,\n",
              "  4,\n",
              "  'Dracorhodin perchlorate was tested for treatment of which cancers?',\n",
              "  'However, the molecular mechanisms underlying these effects are unknown in human gastric tumor cells.',\n",
              "  0),\n",
              " (990,\n",
              "  5,\n",
              "  'Dracorhodin perchlorate was tested for treatment of which cancers?',\n",
              "  'Dracorhodin perchlorate suppresses proliferation and induces apoptosis in human prostate cancer cell line PC-3.',\n",
              "  0),\n",
              " (990,\n",
              "  6,\n",
              "  'Dracorhodin perchlorate was tested for treatment of which cancers?',\n",
              "  'The growth inhibition and pro-apoptosis effects of dracorhodin perchlorate on human prostate cancer PC-3 cell line were examined.',\n",
              "  0),\n",
              " (990,\n",
              "  7,\n",
              "  'Dracorhodin perchlorate was tested for treatment of which cancers?',\n",
              "  'Dracorhodin perchlorate, an anthocyanin red pigment, induces human premyelocytic leukemia HL-60 cell death through apoptotic pathway.',\n",
              "  0),\n",
              " (990,\n",
              "  8,\n",
              "  'Dracorhodin perchlorate was tested for treatment of which cancers?',\n",
              "  'Dracorhodin perchlorate, an anthocyanin red pigment, induces human melanoma A375-S2 cell death through the apoptotic pathway.',\n",
              "  0),\n",
              " (990,\n",
              "  9,\n",
              "  'Dracorhodin perchlorate was tested for treatment of which cancers?',\n",
              "  'Dracorhodin perchlorate inhibited proliferation of several tumor cell lines.',\n",
              "  0),\n",
              " (990,\n",
              "  10,\n",
              "  'Dracorhodin perchlorate was tested for treatment of which cancers?',\n",
              "  'The growth inhibition and pro-apoptosis effects of dracorhodin perchlorate on human prostate cancer PC-3 cell line were examined.',\n",
              "  0),\n",
              " (990,\n",
              "  11,\n",
              "  'Dracorhodin perchlorate was tested for treatment of which cancers?',\n",
              "  'It was reported that DP could induce apoptosis in human prostate cancer, human gastric tumor cells and human melanoma, but the cytotoxic effect of DP on human breast cancer was not investigated.',\n",
              "  1),\n",
              " (990,\n",
              "  12,\n",
              "  'Dracorhodin perchlorate was tested for treatment of which cancers?',\n",
              "  ' Dracorhodin perchlorate (DP) was a synthetic analogue of the antimicrobial anthocyanin red pigment dracorhodin.',\n",
              "  0),\n",
              " (990,\n",
              "  13,\n",
              "  'Dracorhodin perchlorate was tested for treatment of which cancers?',\n",
              "  'It was reported that DP could induce apoptosis in human prostate cancer, human gastric tumor cells and human melanoma, but the cytotoxic effect of DP on human breast cancer was not investigated.',\n",
              "  1),\n",
              " (990,\n",
              "  14,\n",
              "  'Dracorhodin perchlorate was tested for treatment of which cancers?',\n",
              "  ' The growth inhibition and pro-apoptosis effects of dracorhodin perchlorate on human prostate cancer PC-3 cell line were examined.',\n",
              "  0),\n",
              " (990,\n",
              "  15,\n",
              "  'Dracorhodin perchlorate was tested for treatment of which cancers?',\n",
              "  'After administration of 10-80 μmol/L dracorhodin perchlorate for 12-48 h, cell viability of PC-3 cells was measured by MTT colorimetry.',\n",
              "  0),\n",
              " (990,\n",
              "  16,\n",
              "  'Dracorhodin perchlorate was tested for treatment of which cancers?',\n",
              "  'It was reported that DP could induce apoptosis in human prostate cancer, human gastric tumor cells and human melanoma, but the cytotoxic effect of DP on human breast cancer was not investigated.',\n",
              "  1),\n",
              " (990,\n",
              "  17,\n",
              "  'Dracorhodin perchlorate was tested for treatment of which cancers?',\n",
              "  'This study would investigate whether DP was a candidate chemical of anti-human breast cancer.',\n",
              "  0),\n",
              " (990,\n",
              "  18,\n",
              "  'Dracorhodin perchlorate was tested for treatment of which cancers?',\n",
              "  'Dracorhodin perchlorate (DP) was a synthetic analogue of the antimicrobial anthocyanin red pigment dracorhodin.',\n",
              "  0),\n",
              " (990,\n",
              "  19,\n",
              "  'Dracorhodin perchlorate was tested for treatment of which cancers?',\n",
              "  'It was reported that DP could induce apoptosis in human prostate cancer, human gastric tumor cells and human melanoma, but the cytotoxic effect of DP on human breast cancer was not investigated.',\n",
              "  1),\n",
              " (990,\n",
              "  20,\n",
              "  'Dracorhodin perchlorate was tested for treatment of which cancers?',\n",
              "  'The growth inhibition and pro-apoptosis effects of dracorhodin perchlorate on human prostate cancer PC-3 cell line were examined.',\n",
              "  0),\n",
              " (990,\n",
              "  21,\n",
              "  'Dracorhodin perchlorate was tested for treatment of which cancers?',\n",
              "  'After administration of 10-80 μmol/L dracorhodin perchlorate for 12-48 h, cell viability of PC-3 cells was measured by MTT colorimetry.',\n",
              "  0),\n",
              " (1783,\n",
              "  0,\n",
              "  'What are the birth defects associated with Zika-virus infection?',\n",
              "  'the first time after the discovery of ZIKV, the Brazilian scientists are studying the possibility for the virus to cause severe congenital infection related to microcephaly and serious birth defects due to the time-spatial coincidence of the alarming increase of newborns with microcephaly and the Brazilian ZIKV epidemic.',\n",
              "  0),\n",
              " (1783,\n",
              "  1,\n",
              "  'What are the birth defects associated with Zika-virus infection?',\n",
              "  ' Although a causal relationship between Zika infection during pregnancy and microcephaly is strongly suspected, such a connection has not yet been scientifically proven.',\n",
              "  1),\n",
              " (1783,\n",
              "  2,\n",
              "  'What are the birth defects associated with Zika-virus infection?',\n",
              "  'a convincing association between Zika virus infection during pregnancy and birth defects, like microcephaly, among some of the newborns.',\n",
              "  1),\n",
              " (1783,\n",
              "  3,\n",
              "  'What are the birth defects associated with Zika-virus infection?',\n",
              "  'counseling pregnant women on the risks of fetal microcephaly and other birth defects',\n",
              "  0),\n",
              " (1783,\n",
              "  4,\n",
              "  'What are the birth defects associated with Zika-virus infection?',\n",
              "  'evidence suggests that congenital Zika virus infection is associated with microcephaly and other adverse pregnancy and infant outcomes.',\n",
              "  0),\n",
              " (1783,\n",
              "  5,\n",
              "  'What are the birth defects associated with Zika-virus infection?',\n",
              "  'the virus has been linked to congenital malformations, including microcephaly and other severe neurological diseases, such as Guillain-Barré syndrome.',\n",
              "  0),\n",
              " (1783,\n",
              "  6,\n",
              "  'What are the birth defects associated with Zika-virus infection?',\n",
              "  ' severe birth defects, such as microcephaly, have been linked to infection during early pregnancy.',\n",
              "  0),\n",
              " (1783,\n",
              "  7,\n",
              "  'What are the birth defects associated with Zika-virus infection?',\n",
              "  'n can be prenatally passed from a pregnant woman to her fetus.',\n",
              "  0),\n",
              " (1783,\n",
              "  8,\n",
              "  'What are the birth defects associated with Zika-virus infection?',\n",
              "  'There is sufficient evidence to conclude that intrauterine Zika virus infection is a cause of microcephaly and serious brain anomalies, but the full spectrum of anomalies has not been delineated.',\n",
              "  1),\n",
              " (1783,\n",
              "  9,\n",
              "  'What are the birth defects associated with Zika-virus infection?',\n",
              "  'Although the full spectrum of adverse reproductive outcomes caused by Zika virus infection is not yet determined, a distinctive phenotype-the congenital Zika syndrome-has emerged.',\n",
              "  1),\n",
              " (1783,\n",
              "  10,\n",
              "  'What are the birth defects associated with Zika-virus infection?',\n",
              "  'The current Zika virus (ZIKV) outbreak is associated with high numbers of human congenital birth defects',\n",
              "  0),\n",
              " (1783,\n",
              "  11,\n",
              "  'What are the birth defects associated with Zika-virus infection?',\n",
              "  'However, ZIKV infection in early pregnancy has been associated with severe birth defects, including microcephaly and other developmental issues.',\n",
              "  0),\n",
              " (1783,\n",
              "  12,\n",
              "  'What are the birth defects associated with Zika-virus infection?',\n",
              "  'Zika virus (ZIKV) infection is a new emerging threat around the globe which might be responsible for microcephaly and Guillain-Barre syndrome in the infants.',\n",
              "  0),\n",
              " (1783,\n",
              "  13,\n",
              "  'What are the birth defects associated with Zika-virus infection?',\n",
              "  'Zika virus infection during pregnancy is a cause of microcephaly and other severe birth defects',\n",
              "  1),\n",
              " (348,\n",
              "  0,\n",
              "  'What is the rate of survival after commotio cordis?',\n",
              "  'At their commotio cordis event, 216 study patients were 0.2-51 years old (mean age 15±9 years); 95% were males.',\n",
              "  1),\n",
              " (348,\n",
              "  1,\n",
              "  'What is the rate of survival after commotio cordis?',\n",
              "  'Death occurred in 156 individuals (72%), while the other 60 (28%) survived.',\n",
              "  0),\n",
              " (348,\n",
              "  2,\n",
              "  'What is the rate of survival after commotio cordis?',\n",
              "  'In the 2 groups, events were largely similar demographically, including frequency of survival (26% in U.S. vs 25%; P = .84),',\n",
              "  1),\n",
              " (348,\n",
              "  3,\n",
              "  'What is the rate of survival after commotio cordis?',\n",
              "  'Survival of commotio cordis has risen from 10% to 15% since 2001.',\n",
              "  1),\n",
              " (348,\n",
              "  4,\n",
              "  'What is the rate of survival after commotio cordis?',\n",
              "  'Only 21 (16%) individuals survived their event, with particularly prompt cardiopulmonary resuscitation/defibrillation (most commonly reversing ventricular fibrillation) the only identifiable factor associated with a favorable outcome.',\n",
              "  0),\n",
              " (348,\n",
              "  5,\n",
              "  'What is the rate of survival after commotio cordis?',\n",
              "  'Survival rates for commotio cordis are low, even with prompt CPR and defibrillation.',\n",
              "  1),\n",
              " (348,\n",
              "  6,\n",
              "  'What is the rate of survival after commotio cordis?',\n",
              "  'Survival is low, even when resuscitation is performed.',\n",
              "  1),\n",
              " (1750,\n",
              "  0,\n",
              "  'Is synapsin a phosphoprotein?',\n",
              "  'Synapsin is an evolutionarily conserved presynaptic phosphoprotein.',\n",
              "  1),\n",
              " (1750,\n",
              "  1,\n",
              "  'Is synapsin a phosphoprotein?',\n",
              "  'Synapsins as a family of presynaptic terminal phosphoprotein participates in neuronal development',\n",
              "  1),\n",
              " (1750,\n",
              "  2,\n",
              "  'Is synapsin a phosphoprotein?',\n",
              "  'Synapsin III (SynIII) is a phosphoprotein',\n",
              "  1),\n",
              " (1750,\n",
              "  3,\n",
              "  'Is synapsin a phosphoprotein?',\n",
              "  'The neuronal phosphoprotein synapsin III',\n",
              "  1),\n",
              " (1750,\n",
              "  4,\n",
              "  'Is synapsin a phosphoprotein?',\n",
              "  'Synapsin II is a member of the neuronal phosphoprotein family.',\n",
              "  1),\n",
              " (1750, 5, 'Is synapsin a phosphoprotein?', 'phosphoprotein synapsin', 0),\n",
              " (1482,\n",
              "  0,\n",
              "  'Which receptor(s) recognize lysosomal hydrolases in trans-Golgi network (TGN)?',\n",
              "  'The majority of the enzymes leave the TGN after modification with mannose-6-phosphate (M6P) residues, which are specifically recognized by M6P receptors (MPRs), ensuring their transport to the endosomal/lysosomal system.',\n",
              "  1),\n",
              " (1482,\n",
              "  1,\n",
              "  'Which receptor(s) recognize lysosomal hydrolases in trans-Golgi network (TGN)?',\n",
              "  'M6P receptors play a major role in the intracellular transport of newly synthesized lysosomal enzymes in mammalian cells',\n",
              "  0),\n",
              " (1482,\n",
              "  2,\n",
              "  'Which receptor(s) recognize lysosomal hydrolases in trans-Golgi network (TGN)?',\n",
              "  'Mannose 6-phosphate receptors (MPRs) are known to be shuttled between the trans-Golgi network (TGN) and endosomes, thereby several lysosomal hydrolases are delivered through the endocytic pathway into lysosomes.',\n",
              "  0),\n",
              " (1482,\n",
              "  3,\n",
              "  'Which receptor(s) recognize lysosomal hydrolases in trans-Golgi network (TGN)?',\n",
              "  'Most soluble lysosomal hydrolases are sorted in the trans-Golgi network (TGN) and delivered to the lysosomes by the mannose 6-phosphate receptor (M6PR).',\n",
              "  0),\n",
              " (1482,\n",
              "  4,\n",
              "  'Which receptor(s) recognize lysosomal hydrolases in trans-Golgi network (TGN)?',\n",
              "  'certain soluble lysosomal hydrolases, is sorted and trafficked to the lysosomes by sortilin.',\n",
              "  0),\n",
              " (1482,\n",
              "  5,\n",
              "  'Which receptor(s) recognize lysosomal hydrolases in trans-Golgi network (TGN)?',\n",
              "  'Mannose-6-phosphate receptors (MPRs) transport lysosomal hydrolases from the trans Golgi network (TGN) to endosomes.',\n",
              "  1),\n",
              " (1482,\n",
              "  6,\n",
              "  'Which receptor(s) recognize lysosomal hydrolases in trans-Golgi network (TGN)?',\n",
              "  'Recently, the multi-ligand receptor sortilin has also been implicated in this transport',\n",
              "  0),\n",
              " (1482,\n",
              "  7,\n",
              "  'Which receptor(s) recognize lysosomal hydrolases in trans-Golgi network (TGN)?',\n",
              "  'A shortcut to the lysosome: the mannose-6-phosphate-independent pathway',\n",
              "  0),\n",
              " (1482,\n",
              "  8,\n",
              "  'Which receptor(s) recognize lysosomal hydrolases in trans-Golgi network (TGN)?',\n",
              "  'several lines of evidence suggest the existence of alternative processes of lysosomal targeting.',\n",
              "  0),\n",
              " (1482,\n",
              "  9,\n",
              "  'Which receptor(s) recognize lysosomal hydrolases in trans-Golgi network (TGN)?',\n",
              "  'Among them, the two that are mediated by the M6P alternative receptors, lysosomal integral membrane protein (LIMP-2) and sortilin, have gained unequivocal support.',\n",
              "  0),\n",
              " (1482,\n",
              "  10,\n",
              "  'Which receptor(s) recognize lysosomal hydrolases in trans-Golgi network (TGN)?',\n",
              "  'delivery of mannose 6-phosphate receptors carrying lysosomal hydrolases from the trans-Golgi network (TGN) to the endosomal system',\n",
              "  1),\n",
              " (1482,\n",
              "  11,\n",
              "  'Which receptor(s) recognize lysosomal hydrolases in trans-Golgi network (TGN)?',\n",
              "  'The cation-independent mannose 6-phosphate receptor (CI-MPR) mediates sorting of lysosomal hydrolase precursors from the TGN to endosomes.',\n",
              "  0),\n",
              " (1482,\n",
              "  12,\n",
              "  'Which receptor(s) recognize lysosomal hydrolases in trans-Golgi network (TGN)?',\n",
              "  'mannose 6-phosphate (Man-6-P) recognition marker on lysosomal hydrolases.',\n",
              "  0),\n",
              " (1482,\n",
              "  13,\n",
              "  'Which receptor(s) recognize lysosomal hydrolases in trans-Golgi network (TGN)?',\n",
              "  'The delivery of mannose 6-phosphate receptors carrying lysosomal hydrolases from the trans-Golgi network (TGN) to the endosomal system is mediated by selective incorporation of the receptor-hydrolase complexes into vesicular transport carriers (TCs) that are coated with clathrin and the adaptor proteins, GGA and AP-1.',\n",
              "  1),\n",
              " (1482,\n",
              "  14,\n",
              "  'Which receptor(s) recognize lysosomal hydrolases in trans-Golgi network (TGN)?',\n",
              "  'Mannose-6-phosphate receptors (MPRs) transport lysosomal hydrolases from the trans Golgi network (TGN) to endosomes.',\n",
              "  0),\n",
              " (1482,\n",
              "  15,\n",
              "  'Which receptor(s) recognize lysosomal hydrolases in trans-Golgi network (TGN)?',\n",
              "  'Mannose 6-phosphate receptors carry newly synthesized lysosomal hydrolases from the trans-Golgi network to endosomes, then return to the trans-Golgi network for another round of enzyme delivery.',\n",
              "  0),\n",
              " (1482,\n",
              "  16,\n",
              "  'Which receptor(s) recognize lysosomal hydrolases in trans-Golgi network (TGN)?',\n",
              "  'In this way, the M6P receptors help package the hydrolases into vesicles that bud from the trans-Golgi network to deliver their contents to endosomes that ultimately will develop into mature lysosomes, where recently-delivered hydrolases may start digesting the endocyted material.',\n",
              "  0),\n",
              " (1482,\n",
              "  17,\n",
              "  'Which receptor(s) recognize lysosomal hydrolases in trans-Golgi network (TGN)?',\n",
              "  'However, LS fibroblasts displayed reduced mannose 6-phosphate receptor (MPR)-mediated re-uptake of the lysosomal enzyme arylsulfatase B.',\n",
              "  0),\n",
              " (1482,\n",
              "  18,\n",
              "  'Which receptor(s) recognize lysosomal hydrolases in trans-Golgi network (TGN)?',\n",
              "  'In addition, endosome-to-trans Golgi network (TGN) transport of MPRs was decreased significantly, leading to higher levels of cell surface MPRs and their enrichment in enlarged, retromer-positive endosomes in OCRL-depleted HeLa cells.',\n",
              "  0),\n",
              " (1482,\n",
              "  19,\n",
              "  'Which receptor(s) recognize lysosomal hydrolases in trans-Golgi network (TGN)?',\n",
              "  'The M6P groups are then recognized by two independent transmembrane M6P receptors, present in the trans-Golgi network: the cation-independent M6P receptor and/or the cation-dependent M6P receptor.',\n",
              "  0),\n",
              " (1482,\n",
              "  20,\n",
              "  'Which receptor(s) recognize lysosomal hydrolases in trans-Golgi network (TGN)?',\n",
              "  'Two related mannose 6-phosphate receptors (MPRs) recognize this feature in the trans Golgi network (TGN) and deliver the hydrolases to the late endosome.',\n",
              "  0),\n",
              " (1482,\n",
              "  21,\n",
              "  'Which receptor(s) recognize lysosomal hydrolases in trans-Golgi network (TGN)?',\n",
              "  'The cation-independent mannose 6-phosphate receptor (CIMPR) cycles between the trans-Golgi network (TGN) and endosomes to mediate sorting of lysosomal hydrolases.',\n",
              "  0),\n",
              " (1482,\n",
              "  22,\n",
              "  'Which receptor(s) recognize lysosomal hydrolases in trans-Golgi network (TGN)?',\n",
              "  'The M6P groups are then recognized by two independent transmembrane M6P receptors, present in the trans-Golgi network: the cation-independent M6P receptor and/or the cation-dependent M6P receptor',\n",
              "  0),\n",
              " (1482,\n",
              "  23,\n",
              "  'Which receptor(s) recognize lysosomal hydrolases in trans-Golgi network (TGN)?',\n",
              "  'Mannose-6-phosphate receptors (MPRs) transport lysosomal hydrolases from the trans Golgi network (TGN) to endosomes',\n",
              "  0),\n",
              " (1482,\n",
              "  24,\n",
              "  'Which receptor(s) recognize lysosomal hydrolases in trans-Golgi network (TGN)?',\n",
              "  'The delivery of mannose 6-phosphate receptors carrying lysosomal hydrolases from the trans-Golgi network (TGN) to the endosomal system is mediated by selective incorporation of the receptor-hydrolase complexes into vesicular transport carriers (TCs) that are coated with clathrin and the adaptor proteins, GGA and AP-1',\n",
              "  1),\n",
              " (1482,\n",
              "  25,\n",
              "  'Which receptor(s) recognize lysosomal hydrolases in trans-Golgi network (TGN)?',\n",
              "  'Mannose 6-phosphate receptors (MPRs) are known to be shuttled between the trans-Golgi network (TGN) and endosomes, thereby several lysosomal hydrolases are delivered through the endocytic pathway into lysosomes',\n",
              "  0),\n",
              " (1482,\n",
              "  26,\n",
              "  'Which receptor(s) recognize lysosomal hydrolases in trans-Golgi network (TGN)?',\n",
              "  'The cation-independent mannose 6-phosphate receptor (CIMPR) cycles between the trans-Golgi network (TGN) and endosomes to mediate sorting of lysosomal hydrolases',\n",
              "  0),\n",
              " (1482,\n",
              "  27,\n",
              "  'Which receptor(s) recognize lysosomal hydrolases in trans-Golgi network (TGN)?',\n",
              "  'Two related mannose 6-phosphate receptors (MPRs) recognize this feature in the trans Golgi network (TGN) and deliver the hydrolases to the late endosome',\n",
              "  0),\n",
              " (581,\n",
              "  0,\n",
              "  'Is CD99 encoded by MIC2 gene?',\n",
              "  'We report 2 unusual cytogenetic findings in a pediatric Ewing sarcoma, an insertion of the MIC2 gene encoding CD99 from Xp to 10p and a submicroscopic deletion of the well-known tumor supressor gene KLF6',\n",
              "  0),\n",
              " (581,\n",
              "  1,\n",
              "  'Is CD99 encoded by MIC2 gene?',\n",
              "  'We obtained the final diagnosis of ES/PNET by immunohistochemical molecular study with positive staining for the MIC2 gene product (CD99) and a Ewing sarcoma breakpoint region 1 (EWSR1) gene rearrangement',\n",
              "  0),\n",
              " (581,\n",
              "  2,\n",
              "  'Is CD99 encoded by MIC2 gene?',\n",
              "  'CD99, a transmembrane protein encoded by MIC2 gene is involved in multiple cellular events including cell adhesion and migration, apoptosis, cell differentiation and regulation of protein trafficking either in physiological or pathological conditions',\n",
              "  1),\n",
              " (581,\n",
              "  3,\n",
              "  'Is CD99 encoded by MIC2 gene?',\n",
              "  'CD99 is a 32-kDa transmembrane glycoprotein that is encoded by the MIC2 gene',\n",
              "  1),\n",
              " (581,\n",
              "  4,\n",
              "  'Is CD99 encoded by MIC2 gene?',\n",
              "  'The surgical specimens showed small round cell tumor with positive staining for MIC2 gene product (CD99)',\n",
              "  0),\n",
              " (581,\n",
              "  5,\n",
              "  'Is CD99 encoded by MIC2 gene?',\n",
              "  'CD99 is a 32-kDa transmembrane glycoprotein that is encoded by the MIC2 gene.',\n",
              "  1),\n",
              " (581,\n",
              "  6,\n",
              "  'Is CD99 encoded by MIC2 gene?',\n",
              "  'The leukocyte surface molecule CD99 is an integral membrane glycoprotein encoded by the E2/MIC2 gene.',\n",
              "  0),\n",
              " (581,\n",
              "  7,\n",
              "  'Is CD99 encoded by MIC2 gene?',\n",
              "  'Human CD99, which is encoded by the mic2 gene, is a ubiquitous 32 kDa transmembrane protein.',\n",
              "  0),\n",
              " (581,\n",
              "  8,\n",
              "  'Is CD99 encoded by MIC2 gene?',\n",
              "  'Human CD99 is a 32-kDa cell surface protein that is encoded by the MIC2 gene localized to the PAR1.',\n",
              "  0),\n",
              " (581,\n",
              "  9,\n",
              "  'Is CD99 encoded by MIC2 gene?',\n",
              "  'The tumors displayed intense immunoreactivity in a membranous pattern for CD99, the cell surface glycoprotein encoded by the MIC2 gene.',\n",
              "  0),\n",
              " (581,\n",
              "  10,\n",
              "  'Is CD99 encoded by MIC2 gene?',\n",
              "  'CD99, a transmembrane protein encoded by MIC2 gene is involved in multiple cellular events including cell adhesion and migration, apoptosis, cell differentiation and regulation of protein trafficking either in physiological or pathological conditions.',\n",
              "  1),\n",
              " (581,\n",
              "  11,\n",
              "  'Is CD99 encoded by MIC2 gene?',\n",
              "  'CD99, the product of the MIC2 gene, exhibits an erythroid-specific quantitative polymorphism coregulated with the polymorphism of the XG blood group gene.',\n",
              "  0),\n",
              " (581,\n",
              "  12,\n",
              "  'Is CD99 encoded by MIC2 gene?',\n",
              "  'CD99, the product of the MIC2 gene, exhibits an erythroid-specific quantitative polymorphism co-regulated with the Xga blood group polymorphism.',\n",
              "  0),\n",
              " (581,\n",
              "  13,\n",
              "  'Is CD99 encoded by MIC2 gene?',\n",
              "  'Homology searches resulted in finding homologous sequences (totally about 40% homology) in the human MIC2 gene product (CD99; 32-kDa) of T lymphocytes.',\n",
              "  0),\n",
              " (581,\n",
              "  14,\n",
              "  'Is CD99 encoded by MIC2 gene?',\n",
              "  \"Although considered a specific marker for Ewing's sarcoma/peripheral neuroectodermal tumour, the MIC2 gene product (CD99) has been immunolocalised in a variety of human tumours.\",\n",
              "  0),\n",
              " (581,\n",
              "  15,\n",
              "  'Is CD99 encoded by MIC2 gene?',\n",
              "  'MIC2, the gene encoding the CD99 antigen, is found in the pseudoautosomal region of both the X and Y chromosomes.',\n",
              "  0),\n",
              " (581,\n",
              "  16,\n",
              "  'Is CD99 encoded by MIC2 gene?',\n",
              "  'Human CD99 is a 32-kDa cell surface protein that is encoded by the MIC2 gene localized to the PAR1.',\n",
              "  0),\n",
              " (581,\n",
              "  17,\n",
              "  'Is CD99 encoded by MIC2 gene?',\n",
              "  'The tumors displayed intense immunoreactivity in a membranous pattern for CD99, the cell surface glycoprotein encoded by the MIC2 gene.',\n",
              "  0),\n",
              " (581,\n",
              "  18,\n",
              "  'Is CD99 encoded by MIC2 gene?',\n",
              "  'CD99 (MIC2) regulates the LFA-1/ICAM-1-mediated adhesion of lymphocytes, and its gene encodes both positive and negative regulators of cellular adhesion.',\n",
              "  0),\n",
              " (581,\n",
              "  19,\n",
              "  'Is CD99 encoded by MIC2 gene?',\n",
              "  'Relation of neurological marker expression and EWS gene fusion types in MIC2/CD99-positive tumors of the Ewing family.',\n",
              "  0),\n",
              " (581,\n",
              "  20,\n",
              "  'Is CD99 encoded by MIC2 gene?',\n",
              "  'The Ewing family of tumors (EFT) is characterized by high MIC2/CD99 expression and specific EWS/ETS gene rearrangements, resulting in different chimeric transcripts.',\n",
              "  0),\n",
              " (581,\n",
              "  21,\n",
              "  'Is CD99 encoded by MIC2 gene?',\n",
              "  'The tumors displayed intense immunoreactivity in a membranous pattern for CD99, the cell surface glycoprotein encoded by the MIC2 gene.',\n",
              "  0),\n",
              " (581,\n",
              "  22,\n",
              "  'Is CD99 encoded by MIC2 gene?',\n",
              "  \"Monoclonal antibody (MAb) HBA71, which was raised against Ewing's sarcoma cells, recognizes a cell-surface glycoprotein, p30/32MIC2, that is encoded by the MIC2 gene in the pseudoautosomal region of human chromosomes X and Y.\",\n",
              "  0),\n",
              " (581,\n",
              "  23,\n",
              "  'Is CD99 encoded by MIC2 gene?',\n",
              "  'Monoclonal antibodies (mAbs) directed against E2, a 32-kDa transmembrane protein encoded by the MIC2 gene located in the pseudoautosomal region, induce a transbilayer movement of phosphatidylserine and, to a lesser extent, phosphatidylethanolamine in human thymocytes and a Jurkat T lymphocytes.',\n",
              "  0),\n",
              " (581,\n",
              "  24,\n",
              "  'Is CD99 encoded by MIC2 gene?',\n",
              "  'Homology searches resulted in finding homologous sequences (totally about 40% homology) in the human MIC2 gene product (CD99; 32-kDa) of T lymphocytes.',\n",
              "  0),\n",
              " (581,\n",
              "  25,\n",
              "  'Is CD99 encoded by MIC2 gene?',\n",
              "  'CD99 is a 32-kDa transmembrane glycoprotein that is encoded by the MIC2 gene',\n",
              "  0),\n",
              " (581,\n",
              "  26,\n",
              "  'Is CD99 encoded by MIC2 gene?',\n",
              "  'CD99, a transmembrane protein encoded by MIC2 gene is involved in multiple cellular events including cell adhesion and migration, apoptosis, cell differentiation and regulation of protein trafficking either in physiological or pathological conditions',\n",
              "  1),\n",
              " (581,\n",
              "  27,\n",
              "  'Is CD99 encoded by MIC2 gene?',\n",
              "  'The surgical specimens showed small round cell tumor with positive staining for MIC2 gene product (CD99)',\n",
              "  0),\n",
              " (581,\n",
              "  28,\n",
              "  'Is CD99 encoded by MIC2 gene?',\n",
              "  'We report 2 unusual cytogenetic findings in a pediatric Ewing sarcoma, an insertion of the MIC2 gene encoding CD99 from Xp to 10p and a submicroscopic deletion of the well-known tumor supressor gene KLF6',\n",
              "  0),\n",
              " (581,\n",
              "  29,\n",
              "  'Is CD99 encoded by MIC2 gene?',\n",
              "  'MIC2, the gene encoding the CD99 antigen, is found in the pseudoautosomal region of both the X and Y chromosomes',\n",
              "  0),\n",
              " (581,\n",
              "  30,\n",
              "  'Is CD99 encoded by MIC2 gene?',\n",
              "  'Immunohistochemical analysis showed weak to moderate and partial staining for MIC2 (CD99) and WT1, respectively',\n",
              "  0),\n",
              " (581,\n",
              "  31,\n",
              "  'Is CD99 encoded by MIC2 gene?',\n",
              "  'Human CD99, which is encoded by the mic2 gene, is a ubiquitous 32 kDa transmembrane protein',\n",
              "  0),\n",
              " (581,\n",
              "  32,\n",
              "  'Is CD99 encoded by MIC2 gene?',\n",
              "  'The leukocyte surface molecule CD99 is an integral membrane glycoprotein encoded by the E2/MIC2 gene',\n",
              "  0),\n",
              " (581,\n",
              "  33,\n",
              "  'Is CD99 encoded by MIC2 gene?',\n",
              "  'The tumors displayed intense immunoreactivity in a membranous pattern for CD99, the cell surface glycoprotein encoded by the MIC2 gene',\n",
              "  0),\n",
              " (581,\n",
              "  34,\n",
              "  'Is CD99 encoded by MIC2 gene?',\n",
              "  'Human CD99 is a 32-kDa cell surface protein that is encoded by the MIC2 gene localized to the PAR1',\n",
              "  0),\n",
              " (581,\n",
              "  35,\n",
              "  'Is CD99 encoded by MIC2 gene?',\n",
              "  'MIC2, the gene encoding the CD99 antigen, is found in the pseudoautosomal region of both the X and Y chromosomes',\n",
              "  0),\n",
              " (581,\n",
              "  36,\n",
              "  'Is CD99 encoded by MIC2 gene?',\n",
              "  'CD99, the product of the MIC2 gene, exhibits an erythroid-specific quantitative polymorphism co-regulated with the Xga blood group polymorphism',\n",
              "  0),\n",
              " (1342,\n",
              "  0,\n",
              "  'Which E3 ubiquitin ligase mediates the ubiquitination and degradation of the interferon receptor type 1 (IFNAR1)?',\n",
              "  'IFNAR1 ubiquitination is facilitated by the βTrcp E3 ubiquitin ligase that is recruited to IFNAR1 upon its degron phosphorylation, which is induced by the ligand.',\n",
              "  0),\n",
              " (1342,\n",
              "  1,\n",
              "  'Which E3 ubiquitin ligase mediates the ubiquitination and degradation of the interferon receptor type 1 (IFNAR1)?',\n",
              "  'Both ligand-dependent and -independent pathways converge on phosphorylation of Ser(535) within the IFNAR1 degron leading to recruitment of beta-Trcp E3 ubiquitin ligase and concomitant ubiquitination and degradation.',\n",
              "  0),\n",
              " (1342,\n",
              "  2,\n",
              "  'Which E3 ubiquitin ligase mediates the ubiquitination and degradation of the interferon receptor type 1 (IFNAR1)?',\n",
              "  'The SCF(betaTrcp) (Skp1-Cullin1-F-box complex) E3 ubiquitin ligase that mediates IFNAR1 ubiquitination and degradation in cells can conjugate both types of chains in vitro.',\n",
              "  0),\n",
              " (1342,\n",
              "  3,\n",
              "  'Which E3 ubiquitin ligase mediates the ubiquitination and degradation of the interferon receptor type 1 (IFNAR1)?',\n",
              "  ' Levels of IFNAR1 (regulated via degradation mediated by the betaTrcp E3 ubiquitin ligase) and IFNalpha signaling were reduced in 1205Lu melanoma cell line that harbors activated BRAF and exhibits high levels of betaTrcp ubiquitin ligase.',\n",
              "  0),\n",
              " (1342,\n",
              "  4,\n",
              "  'Which E3 ubiquitin ligase mediates the ubiquitination and degradation of the interferon receptor type 1 (IFNAR1)?',\n",
              "  'IFNalpha promotes the phosphorylation of IFNAR1 on Ser535, followed by recruitment of the E3 ubiquitin ligase, beta-TrCP2 (beta-transducin repeats-containing protein 2), ubiquitination of IFNAR1 and proteolysis.',\n",
              "  1),\n",
              " (1342,\n",
              "  5,\n",
              "  'Which E3 ubiquitin ligase mediates the ubiquitination and degradation of the interferon receptor type 1 (IFNAR1)?',\n",
              "  'Ubiquitination, endocytosis, and lysosomal degradation of the IFNAR1 (interferon alpha receptor 1) subunit of the type I interferon (IFN) receptor is mediated by the SCFbeta-Trcp (Skp1-Cullin1-F-box protein beta transducin repeat-containing protein) E3 ubiquitin ligase in a phosphorylation-dependent manner.',\n",
              "  1),\n",
              " (1342,\n",
              "  6,\n",
              "  'Which E3 ubiquitin ligase mediates the ubiquitination and degradation of the interferon receptor type 1 (IFNAR1)?',\n",
              "  'Here we show that IFNAR1 interacts with the Homolog of Slimb (HOS) F-box protein in a phosphorylation-dependent manner, and that this interaction is promoted by interferon alpha (IFNalpha).',\n",
              "  1),\n",
              " (1342,\n",
              "  7,\n",
              "  'Which E3 ubiquitin ligase mediates the ubiquitination and degradation of the interferon receptor type 1 (IFNAR1)?',\n",
              "  'IFNAR1 is ubiquitinated by the Skp1-Cullin1-HOS-Roc1 (SCF(HOS)) ubiquitin ligase in vitro.',\n",
              "  0),\n",
              " (1342,\n",
              "  8,\n",
              "  'Which E3 ubiquitin ligase mediates the ubiquitination and degradation of the interferon receptor type 1 (IFNAR1)?',\n",
              "  'The SCF(betaTrcp) (Skp1-Cullin1-F-box complex) E3 ubiquitin ligase that mediates IFNAR1 ubiquitination and degradation in cells can conjugate both types of chains in vitro.',\n",
              "  0),\n",
              " (1342,\n",
              "  9,\n",
              "  'Which E3 ubiquitin ligase mediates the ubiquitination and degradation of the interferon receptor type 1 (IFNAR1)?',\n",
              "  'IFNAR1 ubiquitination is facilitated by the βTrcp E3 ubiquitin ligase that is recruited to IFNAR1 upon its degron phosphorylation, which is induced by the ligand.',\n",
              "  0),\n",
              " (1342,\n",
              "  10,\n",
              "  'Which E3 ubiquitin ligase mediates the ubiquitination and degradation of the interferon receptor type 1 (IFNAR1)?',\n",
              "  'IFNalpha promotes the phosphorylation of IFNAR1 on Ser535, followed by recruitment of the E3 ubiquitin ligase, beta-TrCP2 (beta-transducin repeats-containing protein 2), ubiquitination of IFNAR1 and proteolysis.',\n",
              "  1),\n",
              " (1342,\n",
              "  11,\n",
              "  'Which E3 ubiquitin ligase mediates the ubiquitination and degradation of the interferon receptor type 1 (IFNAR1)?',\n",
              "  'Both ligand-dependent and -independent pathways converge on phosphorylation of Ser(535) within the IFNAR1 degron leading to recruitment of beta-Trcp E3 ubiquitin ligase and concomitant ubiquitination and degradation.',\n",
              "  0),\n",
              " (1342,\n",
              "  12,\n",
              "  'Which E3 ubiquitin ligase mediates the ubiquitination and degradation of the interferon receptor type 1 (IFNAR1)?',\n",
              "  'Ubiquitination, endocytosis, and lysosomal degradation of the IFNAR1 (interferon alpha receptor 1) subunit of the type I interferon (IFN) receptor is mediated by the SCFbeta-Trcp (Skp1-Cullin1-F-box protein beta transducin repeat-containing protein) E3 ubiquitin ligase in a phosphorylation-dependent manner',\n",
              "  1),\n",
              " (1342,\n",
              "  13,\n",
              "  'Which E3 ubiquitin ligase mediates the ubiquitination and degradation of the interferon receptor type 1 (IFNAR1)?',\n",
              "  'The SCF(betaTrcp) (Skp1-Cullin1-F-box complex) E3 ubiquitin ligase that mediates IFNAR1 ubiquitination and degradation in cells can conjugate both types of chains in vitro',\n",
              "  0),\n",
              " (2663,\n",
              "  0,\n",
              "  'Is the petrous bone used in ancient DNA sampling?',\n",
              "  'Large-scale genomic analyses of ancient human populations have become feasible partly due to refined sampling methods.',\n",
              "  1),\n",
              " (2663,\n",
              "  1,\n",
              "  'Is the petrous bone used in ancient DNA sampling?',\n",
              "  'The inner part of petrous bones and the cementum layer in teeth roots are currently recognized as the best substrates for such research.',\n",
              "  1),\n",
              " (2663,\n",
              "  2,\n",
              "  'Is the petrous bone used in ancient DNA sampling?',\n",
              "  'Ancient DNA (aDNA) research involves invasive and destructive sampling procedures that are often incompatible with anthropological, anatomical, and bioarcheological analyses requiring intact skeletal remains.',\n",
              "  1),\n",
              " (2663,\n",
              "  3,\n",
              "  'Is the petrous bone used in ancient DNA sampling?',\n",
              "  'The osseous labyrinth inside the petrous bone has been shown to yield higher amounts of endogenous DNA than any other skeletal element; however, accessing this labyrinth in cases of a complete or reconstructed skull involves causing major structural damage to the cranial vault or base.',\n",
              "  1),\n",
              " (2663,\n",
              "  4,\n",
              "  'Is the petrous bone used in ancient DNA sampling?',\n",
              "  'first genome-wide ancient DNA from Anatolian Neolithic farmers, whose genetic material we obtained by extracting from petrous bones,',\n",
              "  1),\n",
              " (753,\n",
              "  0,\n",
              "  \"Which enzyme is deficient in Gaucher's disease?\",\n",
              "  \"Gaucher's disease is caused by deficient lysosomal glucocerebrosidase activity\",\n",
              "  1),\n",
              " (753,\n",
              "  1,\n",
              "  \"Which enzyme is deficient in Gaucher's disease?\",\n",
              "  'Gaucher\\'s disease is due to glucocerebrosidase deficiency which is responsible for the accumulation of non degraded glucosylceramide within the lysosomes of macrophages: these \"Gaucher cells\", overloaded and alternatively activated, release in patient\\'s plasma numerous compounds (cytokines, chemokines, hydrolases...) some of which contribute to the various tissue damages',\n",
              "  0),\n",
              " (753,\n",
              "  2,\n",
              "  \"Which enzyme is deficient in Gaucher's disease?\",\n",
              "  \"Gaucher's disease is an uncommon inborn recessive autosomal disease, due to a deficient activity of the lysosomal enzyme beta glucocerebrosidase\",\n",
              "  1),\n",
              " (753,\n",
              "  3,\n",
              "  \"Which enzyme is deficient in Gaucher's disease?\",\n",
              "  'Gaucher disease is an inborn recessive autosomal disease due to a partial deficiency of the lysosomal enzyme beta glucocerebrosidase.',\n",
              "  1),\n",
              " (753,\n",
              "  4,\n",
              "  \"Which enzyme is deficient in Gaucher's disease?\",\n",
              "  'The deficient activity leads to accumulation of the lipid glucocerebroside in the liver, the spleen and bone marrow with concomitant anemia and thrombocytopenia',\n",
              "  1),\n",
              " (753,\n",
              "  5,\n",
              "  \"Which enzyme is deficient in Gaucher's disease?\",\n",
              "  \"Alglucerase is a modified form of human placental glucocerebrosidase used as enzyme replacement therapy for patients with Gaucher's disease, in whom functional glucocerebrosidase is deficient\",\n",
              "  0),\n",
              " (753,\n",
              "  6,\n",
              "  \"Which enzyme is deficient in Gaucher's disease?\",\n",
              "  'The common mutations found in the lysosomal enzyme deficient in Gaucher disease, beta-glucocerebrosidase, earmark these proteins for destruction by the endoplasmic reticulum-localised protein folding machinery, resulting in enzyme insufficiency, lysosomal glycolipid storage and subsequent pathology.',\n",
              "  0),\n",
              " (753,\n",
              "  7,\n",
              "  \"Which enzyme is deficient in Gaucher's disease?\",\n",
              "  'Mutations in GBA, the gene encoding glucocerebrosidase, the enzyme deficient in Gaucher disease, are common risk factors for Parkinson disease, as patients with Parkinson disease are over five times more likely to carry GBA mutations than healthy controls.',\n",
              "  0),\n",
              " (753,\n",
              "  8,\n",
              "  \"Which enzyme is deficient in Gaucher's disease?\",\n",
              "  'It is an essential activator for glucocerebrosidase, the enzyme deficient in Gaucher disease.',\n",
              "  0),\n",
              " (753,\n",
              "  9,\n",
              "  \"Which enzyme is deficient in Gaucher's disease?\",\n",
              "  'GBA encodes for glucocerebrosidase (GCase), the enzyme deficient in the lysosomal storage disorder, Gaucher disease (GD).',\n",
              "  0),\n",
              " (753,\n",
              "  10,\n",
              "  \"Which enzyme is deficient in Gaucher's disease?\",\n",
              "  'The gene for glucocerebrosidase ( GBA), the enzyme deficient in Gaucher disease, is located in a gene-rich region on 1q21.',\n",
              "  0),\n",
              " (753,\n",
              "  11,\n",
              "  \"Which enzyme is deficient in Gaucher's disease?\",\n",
              "  'β-glucocerebrosidase, the enzyme deficient in Gaucher disease, also has an essential role in maintaining epidermal permeability function, by regulating the ratio of ceramides to glucosylceramides in the stratum corneum of the skin.',\n",
              "  0),\n",
              " (753,\n",
              "  12,\n",
              "  \"Which enzyme is deficient in Gaucher's disease?\",\n",
              "  'Recently, it was recognized that mutations in the gene encoding glucocerebrosidase, the enzyme deficient in Gaucher disease, are associated with an increased incidence of parkinsonism.',\n",
              "  0),\n",
              " (753,\n",
              "  13,\n",
              "  \"Which enzyme is deficient in Gaucher's disease?\",\n",
              "  'Mutations in glucocerebrosidase (GBA), the enzyme deficient in Gaucher disease, are also identified at an increased frequency among Parkinson probands, including those of Ashkenazi Jewish ancestry.',\n",
              "  0),\n",
              " (753,\n",
              "  14,\n",
              "  \"Which enzyme is deficient in Gaucher's disease?\",\n",
              "  'Structure/function relationships of acid beta-glucosidase, the enzyme deficient in Gaucher disease, were evaluated by characterizing the proteins expressed from cDNAs encoding normal and mutant enzymes.',\n",
              "  0),\n",
              " (753,\n",
              "  15,\n",
              "  \"Which enzyme is deficient in Gaucher's disease?\",\n",
              "  'Gaucher disease is an autosomal recessive disorder resulting from deficient activity of the lysosomal enzyme glucocerebrosidase (GBA, E.C.3.2.1.45)',\n",
              "  0),\n",
              " (753,\n",
              "  16,\n",
              "  \"Which enzyme is deficient in Gaucher's disease?\",\n",
              "  'GBA encodes for glucocerebrosidase (GCase), the enzyme deficient in the lysosomal storage disorder, Gaucher disease (GD)',\n",
              "  0),\n",
              " (753,\n",
              "  17,\n",
              "  \"Which enzyme is deficient in Gaucher's disease?\",\n",
              "  'LIMP-2 is required for the normal biogenesis and maintenance of lysosomes and endosomes and has been identified as the specific receptor for glucocerebrosidase, the enzyme deficient in Gaucher disease',\n",
              "  0),\n",
              " (753,\n",
              "  18,\n",
              "  \"Which enzyme is deficient in Gaucher's disease?\",\n",
              "  'Mutations in glucocerebrosidase (GCase), the enzyme deficient in Gaucher disease, are a common genetic risk factor for the development of Parkinson disease and related disorders, implicating the role of this lysosomal hydrolase in the disease etiology',\n",
              "  0),\n",
              " (753,\n",
              "  19,\n",
              "  \"Which enzyme is deficient in Gaucher's disease?\",\n",
              "  'This gene is involved in lysosomal mannose-6-phosphate-independent trafficking of β-glucocerebrosidase (GC), an enzyme deficient in Gaucher disease',\n",
              "  0),\n",
              " (753,\n",
              "  20,\n",
              "  \"Which enzyme is deficient in Gaucher's disease?\",\n",
              "  'Gaucher disease (GD) is a lysosomal storage disorder, caused by deficient activity of the enzyme glucocerebrosidase',\n",
              "  0),\n",
              " (753,\n",
              "  21,\n",
              "  \"Which enzyme is deficient in Gaucher's disease?\",\n",
              "  'Mutations in GBA, the gene encoding glucocerebrosidase, the enzyme deficient in Gaucher disease, are common risk factors for Parkinson disease, as patients with Parkinson disease are over five times more likely to carry GBA mutations than healthy controls',\n",
              "  0),\n",
              " (753,\n",
              "  22,\n",
              "  \"Which enzyme is deficient in Gaucher's disease?\",\n",
              "  'The common mutations found in the lysosomal enzyme deficient in Gaucher disease, beta-glucocerebrosidase, earmark these proteins for destruction by the endoplasmic reticulum-localised protein folding machinery, resulting in enzyme insufficiency, lysosomal glycolipid storage and subsequent pathology',\n",
              "  0),\n",
              " (753,\n",
              "  23,\n",
              "  \"Which enzyme is deficient in Gaucher's disease?\",\n",
              "  \"Replacement therapy for inherited enzyme deficiency--macrophage-targeted glucocerebrosidase for Gaucher's disease.\",\n",
              "  0),\n",
              " (753,\n",
              "  24,\n",
              "  \"Which enzyme is deficient in Gaucher's disease?\",\n",
              "  'Gaucher disease (GD) is an autosomal recessive lysosomal disorder caused by a deficiency of glucocerebrosidase.',\n",
              "  0),\n",
              " (753,\n",
              "  25,\n",
              "  \"Which enzyme is deficient in Gaucher's disease?\",\n",
              "  \"Gaucher's disease, the most prevalent of the sphingolipid storage disorders, is caused by a deficiency of the enzyme glucocerebrosidase (glucosylceramidase).\",\n",
              "  0),\n",
              " (753,\n",
              "  26,\n",
              "  \"Which enzyme is deficient in Gaucher's disease?\",\n",
              "  \"Pathogenic variants in the glucocerebrosidase gene (GBA) encoding the enzyme deficient in Gaucher's disease (GD) are associated with Parkinson's disease (PD).\",\n",
              "  0),\n",
              " (753,\n",
              "  27,\n",
              "  \"Which enzyme is deficient in Gaucher's disease?\",\n",
              "  \"Gaucher's disease is an uncommon inborn recessive autosomal disease, due to a deficient activity of the lysosomal enzyme beta glucocerebrosidase.\",\n",
              "  1),\n",
              " (753,\n",
              "  28,\n",
              "  \"Which enzyme is deficient in Gaucher's disease?\",\n",
              "  'Recently, it was recognized that mutations in the gene encoding glucocerebrosidase, the enzyme deficient in Gaucher disease, are associated with an increased incidence of parkinsonism.',\n",
              "  0),\n",
              " (753,\n",
              "  29,\n",
              "  \"Which enzyme is deficient in Gaucher's disease?\",\n",
              "  \"Gaucher's disease is caused by deficient lysosomal glucocerebrosidase activity.\",\n",
              "  0),\n",
              " (753,\n",
              "  30,\n",
              "  \"Which enzyme is deficient in Gaucher's disease?\",\n",
              "  \"Functional glucocerebrosidase is deficient in Gaucher's disease, an autosomal recessive lipid storage disorder that affects people of all ethnic backgrounds, but has a higher incidence among East European Jews (Ashkenazim).\",\n",
              "  0),\n",
              " (753,\n",
              "  31,\n",
              "  \"Which enzyme is deficient in Gaucher's disease?\",\n",
              "  \"We studied two patients with type IV glycogen storage disease 37 and 91 months after liver transplantation and a third patient with lysosomal glucocerebrosidase deficiency (type 1 Gaucher's disease), in whom tissue glucocerebroside deposition had decreased 26 months after liver replacement, to determine whether the migration of cells from the allograft (microchimerism) could explain the improved metabolism of enzyme-deficient tissues in the recipient.\",\n",
              "  0),\n",
              " (2298,\n",
              "  0,\n",
              "  'Which gene is the paralog of yeast UPC2?',\n",
              "  'Here, we examine the role of the related transcription factors Ecm22 and Upc2 in Saccharomyces cerevisiae filamentation.',\n",
              "  1),\n",
              " (2298,\n",
              "  1,\n",
              "  'Which gene is the paralog of yeast UPC2?',\n",
              "  'The zinc cluster proteins Upc2 and Ecm22 promote filamentation in Saccharomyces cerevisiae by sterol biosynthesis-dependent and -independent pathways.',\n",
              "  1),\n",
              " (2298,\n",
              "  2,\n",
              "  'Which gene is the paralog of yeast UPC2?',\n",
              "  'Upc2, a zinc-cluster transcription factor, is a regulator of ergosterol biosynthesis in yeast.',\n",
              "  0),\n",
              " (2298,\n",
              "  3,\n",
              "  'Which gene is the paralog of yeast UPC2?',\n",
              "  'The zinc cluster proteins Ecm22, Upc2, Sut1 and Sut2 have initially been identified as regulators of sterol import in the budding yeast Saccharomyces cerevisiae.',\n",
              "  1),\n",
              " (2298,\n",
              "  4,\n",
              "  'Which gene is the paralog of yeast UPC2?',\n",
              "  'Yeast studies defined a 7-bp consensus sterol-response element (SRE) common to genes involved in sterol biosynthesis and two transcription factors, Upc2 and Ecm22, which direct transcription of sterol biosynthetic genes.',\n",
              "  1),\n",
              " (2298,\n",
              "  5,\n",
              "  'Which gene is the paralog of yeast UPC2?',\n",
              "  'Ecm22 and Upc2 positively control the expression of FHN1, NPR1, PRR2 and sterol biosynthesis genes.',\n",
              "  1),\n",
              " (2298,\n",
              "  6,\n",
              "  'Which gene is the paralog of yeast UPC2?',\n",
              "  'RNA-seq analysis shows that hypoxic regulation of sterol synthesis genes in Y. lipolytica is predominantly mediated by Upc2.',\n",
              "  0),\n",
              " (2298,\n",
              "  7,\n",
              "  'Which gene is the paralog of yeast UPC2?',\n",
              "  'However, in yeasts such as Saccharomyces cerevisiae and Candida albicans sterol synthesis is instead regulated by Upc2, an unrelated transcription factor with a Gal4-type zinc finger.',\n",
              "  0),\n",
              " (2298,\n",
              "  8,\n",
              "  'Which gene is the paralog of yeast UPC2?',\n",
              "  'Recombinant endogenous promoter studies show that the UPC2 anaerobic AR1b elements act in trans to regulate ergosterol gene expression.',\n",
              "  0),\n",
              " (2298,\n",
              "  9,\n",
              "  'Which gene is the paralog of yeast UPC2?',\n",
              "  'One such factor could be the mammalian equivalent of the gene product of UPC2.',\n",
              "  0),\n",
              " (2298,\n",
              "  10,\n",
              "  'Which gene is the paralog of yeast UPC2?',\n",
              "  'Simultaneous deletion of ECM22 and UPC2 as well as mutation of the three Ste12-binding sites in the PRM1 promoter completely abolishes basal and pheromone-induced PRM1 expression, indicating that Ste12 and Ecm22/Upc2 control PRM1 transcription through distinct pathways.',\n",
              "  0),\n",
              " (2298,\n",
              "  11,\n",
              "  'Which gene is the paralog of yeast UPC2?',\n",
              "  'Ecm22 and Upc2 positively regulate basal expression of PRM1 and PRM4.',\n",
              "  0),\n",
              " (2298,\n",
              "  12,\n",
              "  'Which gene is the paralog of yeast UPC2?',\n",
              "  'Our results indicate that Upc2 must occupy UPC2 AR1b elements in order for ERG gene expression induction to take place.',\n",
              "  0),\n",
              " (2298,\n",
              "  13,\n",
              "  'Which gene is the paralog of yeast UPC2?',\n",
              "  'A yeast mutant (upc2-1) with a defect in the aerobic exclusion of sterols was found to have increased sensitivity to LiCl and NaCl.',\n",
              "  0),\n",
              " (2298,\n",
              "  14,\n",
              "  'Which gene is the paralog of yeast UPC2?',\n",
              "  'The AR1b elements are absolutely required for auto-induction of UPC2 gene expression and protein and require Upc2 and Ecm22 for function.',\n",
              "  0),\n",
              " (2298,\n",
              "  15,\n",
              "  'Which gene is the paralog of yeast UPC2?',\n",
              "  'Secretion of apoE was achieved only by the use of a mutant (upc2) strain of yeast with the phenotype of enhanced uptake and intracellular esterification of exogenous cholesterol.',\n",
              "  0),\n",
              " (2298,\n",
              "  16,\n",
              "  'Which gene is the paralog of yeast UPC2?',\n",
              "  'Gas chromatographic analysis of the nonsaponifiable fractions of the various strains showed that the major sterol for all YLR228c and UPC2 combinations was ergosterol, the consensus yeast sterol..',\n",
              "  0),\n",
              " (2298,\n",
              "  17,\n",
              "  'Which gene is the paralog of yeast UPC2?',\n",
              "  'In the pathogenic yeast Candida albicans, the zinc cluster transcription factor Upc2p has been shown to regulate the expression of ERG11 and other genes involved in ergosterol biosynthesis upon exposure to azole antifungals.',\n",
              "  0),\n",
              " (2298,\n",
              "  18,\n",
              "  'Which gene is the paralog of yeast UPC2?',\n",
              "  'By comparing the gene expression profiles of the fluconazole-resistant isolate and of strains carrying wild-type and mutated UPC2 alleles, we identified target genes that are controlled by Upc2p.',\n",
              "  0),\n",
              " (1607,\n",
              "  0,\n",
              "  'What is the effect of ranolazine in diastolic heart failure?',\n",
              "  'Data from in vitro and animal studies indicate that ranolazine improves diastolic function by inhibiting the late sodium current.',\n",
              "  1),\n",
              " (1607,\n",
              "  1,\n",
              "  'What is the effect of ranolazine in diastolic heart failure?',\n",
              "  'Ranolazine is an innovative anti-ischemic and antianginal agent that inhibits the late Na current, thereby reducing the Na-dependent Ca-overload, which improves diastolic tone and oxygen handling during myocardial ischemia.',\n",
              "  1),\n",
              " (1607,\n",
              "  2,\n",
              "  'What is the effect of ranolazine in diastolic heart failure?',\n",
              "  'In addition, ranolazine seems to exert beneficial effects on diastolic cardiac function.',\n",
              "  0),\n",
              " (1607,\n",
              "  3,\n",
              "  'What is the effect of ranolazine in diastolic heart failure?',\n",
              "  'Ranolazine improves cardiac diastolic dysfunction through modulation of myofilament calcium sensitivity',\n",
              "  1),\n",
              " (1607,\n",
              "  4,\n",
              "  'What is the effect of ranolazine in diastolic heart failure?',\n",
              "  'Diastolic dysfunction could be reversed by ranolazine, probably resulting from a direct effect on myofilaments, indicating that cardiac oxidative stress may mediate diastolic dysfunction through altering the contractile apparatus.',\n",
              "  1),\n",
              " (1607,\n",
              "  5,\n",
              "  'What is the effect of ranolazine in diastolic heart failure?',\n",
              "  'Ranolazine (10 microM) significantly reduced the PC-induced increase in LVEDP by 72 +/- 6% (n = 6, p < 0.001), reduced left ventricular wall stiffness, and attenuated the PC-induced increase of CPP by 53 +/- 10% (n = 6-7, p < 0.05).',\n",
              "  0),\n",
              " (1607,\n",
              "  6,\n",
              "  'What is the effect of ranolazine in diastolic heart failure?',\n",
              "  ' Ranolazine, a specific inhibitor of late INa, reduces Na influx and hence ameliorates disturbed Na and Ca homeostasis.',\n",
              "  1),\n",
              " (1607,\n",
              "  7,\n",
              "  'What is the effect of ranolazine in diastolic heart failure?',\n",
              "  'New clinical and experimental studies even point to potential antiarrhythmic effects, beneficial effects in diastolic heart failure, and under hyperglycemic conditions.',\n",
              "  0),\n",
              " (695,\n",
              "  0,\n",
              "  'What is the association between adiponectin and migraine?',\n",
              "  'Increase in body fat, especially in gluteofemoral region, elevates adiponectin and leptin secretion which in turn impair inflammatory processes that could be contributing to migraine risk.',\n",
              "  1),\n",
              " (695,\n",
              "  1,\n",
              "  'What is the association between adiponectin and migraine?',\n",
              "  'CONCLUSION: In this pilot study of women episodic migraineurs, the HMW\\u2009:\\u2009LMW ADP ratio level was associated with migraine severity and predictive of acute treatment response.',\n",
              "  1),\n",
              " (695,\n",
              "  2,\n",
              "  'What is the association between adiponectin and migraine?',\n",
              "  'ADP and the HMW\\u2009:\\u2009LMW ratio of ADP represent potential novel biomarkers and drug targets for episodic migraine.',\n",
              "  0),\n",
              " (695,\n",
              "  3,\n",
              "  'What is the association between adiponectin and migraine?',\n",
              "  'Specifically, neurotransmittors such as serotonin, peptides such as orexin, and adipocytokines such as adiponectin and leptin have been suggested to have roles in both feeding and migraine.',\n",
              "  0),\n",
              " (695,\n",
              "  4,\n",
              "  'What is the association between adiponectin and migraine?',\n",
              "  'Obesity can be considered as a proinflammatory state in which increased inflammatory mediators, vascular hyperreactivity, plasma calcitonin gene-related peptide (CGRP) concentrations and decreased adiponectin concentrations are observed.',\n",
              "  0),\n",
              " (695,\n",
              "  5,\n",
              "  'What is the association between adiponectin and migraine?',\n",
              "  'These alterations can cause an increase in the frequency of migraine attacks developed of central sensitization, and thereafter, chronic migraine.',\n",
              "  0),\n",
              " (695,\n",
              "  6,\n",
              "  'What is the association between adiponectin and migraine?',\n",
              "  'Specifically, neurotransmittors such as serotonin, peptides such as orexin, and adipocytokines such as adiponectin and leptin have been suggested to have roles in both feeding and migraine.',\n",
              "  0),\n",
              " (695,\n",
              "  7,\n",
              "  'What is the association between adiponectin and migraine?',\n",
              "  'Several adipocytokines appear to play an integral role in feeding and obesity--and have also been linked to pain.',\n",
              "  0),\n",
              " (695,\n",
              "  8,\n",
              "  'What is the association between adiponectin and migraine?',\n",
              "  'Among these proteins are adiponectin and leptin.',\n",
              "  0),\n",
              " (695,\n",
              "  9,\n",
              "  'What is the association between adiponectin and migraine?',\n",
              "  'The author reviews the regulation of adipose tissue and feeding and provides an in-depth examination of adiponectin and leptin and their association with migraine.',\n",
              "  1),\n",
              " (695,\n",
              "  10,\n",
              "  'What is the association between adiponectin and migraine?',\n",
              "  'CONCLUSION: Serum adiponectin levels are increased in women chronic daily headache (CDH) sufferers.',\n",
              "  1),\n",
              " (695,\n",
              "  11,\n",
              "  'What is the association between adiponectin and migraine?',\n",
              "  'n this review we discuss the basic science of adiponectin and its potential connection to the pathophysiology of migraine.',\n",
              "  0),\n",
              " (695,\n",
              "  12,\n",
              "  'What is the association between adiponectin and migraine?',\n",
              "  'Future research may focus on how adiponectin levels are potentially altered during migraine attacks, and how that information can be potentially translated into migraine therapy.',\n",
              "  0),\n",
              " (695,\n",
              "  13,\n",
              "  'What is the association between adiponectin and migraine?',\n",
              "  'Collectively, the results of our analysis suggest that a link between serum adiponectin and migraine remains elusive, at the best',\n",
              "  1),\n",
              " (695,\n",
              "  14,\n",
              "  'What is the association between adiponectin and migraine?',\n",
              "  'Future research may focus on how adiponectin levels are potentially altered during migraine attacks, and how that information can be potentially translated into migraine therapy.',\n",
              "  0),\n",
              " (3378,\n",
              "  0,\n",
              "  'What is Idiopathic toe walking?',\n",
              "  'BACKGROUND: Idiopathic toe walking (ITW) is a diagnosis of exclusion for children walking on their toes with no medical cause.',\n",
              "  1),\n",
              " (3378,\n",
              "  1,\n",
              "  'What is Idiopathic toe walking?',\n",
              "  'INTRODUCTION: Idiopathic toe-walking (ITW) is described as a gait pattern with no contact between the heels and the ground in children older than 3years.',\n",
              "  1),\n",
              " (3378,\n",
              "  2,\n",
              "  'What is Idiopathic toe walking?',\n",
              "  'Idiopathic toe walking is a relatively common developmental condition often leading to secondary problems such as pain and muscle contractures in the lower extremities.',\n",
              "  0),\n",
              " (3378,\n",
              "  3,\n",
              "  'What is Idiopathic toe walking?',\n",
              "  'BACKGROUND: Children with idiopathic toe-walking, a common pediatric condition, walk some or all of the time on their toes.',\n",
              "  0),\n",
              " (3378,\n",
              "  4,\n",
              "  'What is Idiopathic toe walking?',\n",
              "  'BACKGROUND: Idiopathic toe walking is a diagnosis of exclusion characterized by a persistent toe-toe gait pattern after three years of age.',\n",
              "  1),\n",
              " (3378,\n",
              "  5,\n",
              "  'What is Idiopathic toe walking?',\n",
              "  'Idiopathic toe walking (ITW) is a pathological gait pattern in which children walk on their tip toes with no orthopedic or neurological reason.',\n",
              "  1),\n",
              " (3378,\n",
              "  6,\n",
              "  'What is Idiopathic toe walking?',\n",
              "  'BACKGROUND\\n\\nIdiopathic toe walking (ITW) is an exclusionary diagnosis given to healthy children who persist in walking on their toes after they should typically have achieved a heel-toe gait.',\n",
              "  0),\n",
              " (3378,\n",
              "  7,\n",
              "  'What is Idiopathic toe walking?',\n",
              "  'BACKGROUND\\n\\nIdiopathic toe walking is a diagnosis of exclusion characterized by a persistent toe-toe gait pattern after three years of age.',\n",
              "  0),\n",
              " (3378,\n",
              "  8,\n",
              "  'What is Idiopathic toe walking?',\n",
              "  'Idiopathic toe walking is a diagnosis of exclusion characterized by a persistent toe-toe gait pattern after three years of age.',\n",
              "  1),\n",
              " (3378,\n",
              "  9,\n",
              "  'What is Idiopathic toe walking?',\n",
              "  'The diagnosis of idiopathic toe walking is a diagnosis of exclusion used for children with persistent toe walking and no associated medical condition .',\n",
              "  0),\n",
              " (3378,\n",
              "  10,\n",
              "  'What is Idiopathic toe walking?',\n",
              "  'Idiopathic toe-walking is a diagnosis of exclusion when a child presents with bilateral toe-to-toe gait.',\n",
              "  0),\n",
              " (3378,\n",
              "  11,\n",
              "  'What is Idiopathic toe walking?',\n",
              "  'BACKGROUND: Idiopathic toe walking is a diagnosis of exclusion characterized by a persistent toe-toe gait pattern after three years of age.',\n",
              "  0),\n",
              " (3378,\n",
              "  12,\n",
              "  'What is Idiopathic toe walking?',\n",
              "  'Idiopathic toe walking (ITW), considered abnormal after the age of 3 years, is a common complaint seen by medical professionals, especially orthopaedic surgeons and physiotherapists.',\n",
              "  0),\n",
              " (3378,\n",
              "  13,\n",
              "  'What is Idiopathic toe walking?',\n",
              "  'Idiopathic toe-walking is defined as persistent toe-walking in a normal child in the absence of developmental, neurological or neuromuscular conditions.',\n",
              "  0),\n",
              " (311,\n",
              "  0,\n",
              "  'Name a method for enrichment of arginine-methylated peptides.',\n",
              "  'To better study protein methylation, we have developed highly specific antibodies against monomethyl arginine; asymmetric dimethyl arginine; and monomethyl, dimethyl, and trimethyl lysine motifs.',\n",
              "  1),\n",
              " (311,\n",
              "  1,\n",
              "  'Name a method for enrichment of arginine-methylated peptides.',\n",
              "  'These antibodies were used to perform immunoaffinity purification of methyl peptides followed by LC-MS/MS analysis to identify and quantify arginine and lysine methylation sites in several model studies.',\n",
              "  1),\n",
              " (3881,\n",
              "  0,\n",
              "  'Which network analysis method can you use for prioritization of metabolic disease genes?',\n",
              "  'metPropagate: network-guided propagation of metabolomic information for prioritization of metabolic disease genes.',\n",
              "  0),\n",
              " (3881,\n",
              "  1,\n",
              "  'Which network analysis method can you use for prioritization of metabolic disease genes?',\n",
              "  'Many inborn errors of metabolism (IEMs) are amenable to treatment, therefore early diagnosis is imperative.',\n",
              "  0),\n",
              " (3881,\n",
              "  2,\n",
              "  'Which network analysis method can you use for prioritization of metabolic disease genes?',\n",
              "  'Whole-exome sequencing (WES) variant prioritization coupled with phenotype-guided clinical and bioinformatics expertise is typically used to identify disease-causing variants; however, it can be challenging to identify the causal candidate gene when a large number of rare and potentially pathogenic variants are detected.',\n",
              "  1),\n",
              " (3881,\n",
              "  3,\n",
              "  'Which network analysis method can you use for prioritization of metabolic disease genes?',\n",
              "  'Here, we present a network-based approach, metPropagate, that uses untargeted metabolomics (UM) data from a single patient and a group of controls to prioritize candidate genes in patients with suspected IEMs.',\n",
              "  1),\n",
              " (3881,\n",
              "  4,\n",
              "  'Which network analysis method can you use for prioritization of metabolic disease genes?',\n",
              "  'We validate metPropagate on 107 patients with IEMs diagnosed in Miller et al.',\n",
              "  0),\n",
              " (3881,\n",
              "  5,\n",
              "  'Which network analysis method can you use for prioritization of metabolic disease genes?',\n",
              "  '(2015) and 11 patients with both CNS and metabolic abnormalities.',\n",
              "  0),\n",
              " (3881,\n",
              "  6,\n",
              "  'Which network analysis method can you use for prioritization of metabolic disease genes?',\n",
              "  \"The metPropagate method ranks candidate genes by label propagation, a graph-smoothing algorithm that considers each gene's metabolic perturbation in addition to the network of interactions between neighbors.\",\n",
              "  0),\n",
              " (3881,\n",
              "  7,\n",
              "  'Which network analysis method can you use for prioritization of metabolic disease genes?',\n",
              "  'metPropagate was able to prioritize at least one causative gene in the top 20th percentile of candidate genes for 92% of patients with known IEMs.',\n",
              "  1),\n",
              " (3881,\n",
              "  8,\n",
              "  'Which network analysis method can you use for prioritization of metabolic disease genes?',\n",
              "  \"Applied to patients with suspected neurometabolic disease, metPropagate placed at least one causative gene in the top 20th percentile in 9/11 patients, and ranked the causative gene more highly than Exomiser's phenotype-based ranking in 6/11 patients.\",\n",
              "  0),\n",
              " (3881,\n",
              "  9,\n",
              "  'Which network analysis method can you use for prioritization of metabolic disease genes?',\n",
              "  'Interestingly, ranking by a weighted combination of metPropagate and Exomiser scores resulted in improved prioritization.',\n",
              "  0),\n",
              " (3881,\n",
              "  10,\n",
              "  'Which network analysis method can you use for prioritization of metabolic disease genes?',\n",
              "  'The results of this study indicate that network-based analysis of UM data can provide an additional mode of evidence to prioritize causal genes in patients with suspected IEMs.',\n",
              "  0),\n",
              " (3881,\n",
              "  11,\n",
              "  'Which network analysis method can you use for prioritization of metabolic disease genes?',\n",
              "  \"The metPropagate method ranks candidate genes by label propagation, a graph-smoothing algorithm that considers each gene's metabolic perturbation in addition to the network of interactions between neighbors.\",\n",
              "  0),\n",
              " (3881,\n",
              "  12,\n",
              "  'Which network analysis method can you use for prioritization of metabolic disease genes?',\n",
              "  'we present a network-based approach, metPropagate, that uses untargeted metabolomics (UM) data from a single patient and a group of controls to prioritize candidate genes in patients with suspected IEMs.',\n",
              "  1),\n",
              " (3881,\n",
              "  13,\n",
              "  'Which network analysis method can you use for prioritization of metabolic disease genes?',\n",
              "  'We valida',\n",
              "  0),\n",
              " (3881,\n",
              "  14,\n",
              "  'Which network analysis method can you use for prioritization of metabolic disease genes?',\n",
              "  'metPropagate: network-guided propagation of metabolomic information for prioritization of metabolic disease genes',\n",
              "  0),\n",
              " (3881,\n",
              "  15,\n",
              "  'Which network analysis method can you use for prioritization of metabolic disease genes?',\n",
              "  'Here, we present a network-based approach, metPropagate, that uses untargeted metabolomics (UM) data from a single patient and a group of controls to prioritize candidate genes in patients with suspected IEMs.',\n",
              "  1),\n",
              " (3881,\n",
              "  16,\n",
              "  'Which network analysis method can you use for prioritization of metabolic disease genes?',\n",
              "  \"Applied to patients with suspected neurometabolic disease, metPropagate placed at least one causative gene in the top 20th percentile in 9/11 patients, and ranked the causative gene more highly than Exomiser's phenotype-based ranking in 6/11 patients.\",\n",
              "  0),\n",
              " (3881,\n",
              "  17,\n",
              "  'Which network analysis method can you use for prioritization of metabolic disease genes?',\n",
              "  'metPropagate was able to prioritize at least one causative gene in the top 20th percentile of candidate genes for 92% of patients with known IEMs.',\n",
              "  0),\n",
              " (3881,\n",
              "  18,\n",
              "  'Which network analysis method can you use for prioritization of metabolic disease genes?',\n",
              "  'Interestingly, ranking by a weighted combination of metPropagate and Exomiser scores resulted in improved prioritization.',\n",
              "  0),\n",
              " (346,\n",
              "  0,\n",
              "  'Which is the most common cause of sudden cardiac death in young athletes?',\n",
              "  'The most common cause of death was hypertrophic cardiomyopathy (30 %), followed by coronary artery anomalies (9 %), and myocarditis (9 %).',\n",
              "  0),\n",
              " (346,\n",
              "  1,\n",
              "  'Which is the most common cause of sudden cardiac death in young athletes?',\n",
              "  'The most common cause of this, hypertrophic cardiomyopathy (HCM), is a genetic disorder responsible for more than a third of cases and is manageable.',\n",
              "  0),\n",
              " (346,\n",
              "  2,\n",
              "  'Which is the most common cause of sudden cardiac death in young athletes?',\n",
              "  'HCM is the most common cause of sudden death in young competitive athletes and preparticipation screening programs have to be implemented to avoid these tragic fatalities.',\n",
              "  0),\n",
              " (346,\n",
              "  3,\n",
              "  'Which is the most common cause of sudden cardiac death in young athletes?',\n",
              "  'Hypertrophic cardiomyopathy (HCM) is the most common cause of sudden cardiac death (SCD) in young people, including trained athletes.',\n",
              "  1),\n",
              " (346,\n",
              "  4,\n",
              "  'Which is the most common cause of sudden cardiac death in young athletes?',\n",
              "  'Hypertrophic cardiomyopathy (HCM) is regarded as the most common cause of sudden cardiac death in young people (including trained athletes).',\n",
              "  1),\n",
              " (346,\n",
              "  5,\n",
              "  'Which is the most common cause of sudden cardiac death in young athletes?',\n",
              "  'Hypertrophic cardiomyopathy (HCM) is the most common cause of sudden cardiac death in young people, including trained athletes.',\n",
              "  1),\n",
              " (346,\n",
              "  6,\n",
              "  'Which is the most common cause of sudden cardiac death in young athletes?',\n",
              "  'Hypertrophic cardiomyopathy is regarded as the most common cause of sudden cardiac death in young people (including trained athletes).',\n",
              "  1),\n",
              " (346,\n",
              "  7,\n",
              "  'Which is the most common cause of sudden cardiac death in young athletes?',\n",
              "  'The most common cause of sudden cardiac death in athletes is hypertrophic cardiomyopathy.',\n",
              "  1),\n",
              " (346,\n",
              "  8,\n",
              "  'Which is the most common cause of sudden cardiac death in young athletes?',\n",
              "  'Hypertrophic cardiomyopathy (HCM) is one of the most common inherited primary cardiac disorders and the most common cause of sudden cardiac death in young athletes.',\n",
              "  0),\n",
              " (346,\n",
              "  9,\n",
              "  'Which is the most common cause of sudden cardiac death in young athletes?',\n",
              "  'The most common cause of sudden cardiac death in individuals aged less than 35 years, including competitive athletes, is the inherited disorder hypertrophic cardiomyopathy (HCM).',\n",
              "  0),\n",
              " (346,\n",
              "  10,\n",
              "  'Which is the most common cause of sudden cardiac death in young athletes?',\n",
              "  'udden death in young competitive athletes is due to a variety of cardiovascular diseases (CVDs) and, most commonly, HCM.',\n",
              "  0),\n",
              " (346,\n",
              "  11,\n",
              "  'Which is the most common cause of sudden cardiac death in young athletes?',\n",
              "  'Hypertrophic cardiomyopathy (HCM) is the most common cause of death in the young, particularly in young competitive athletes.',\n",
              "  0),\n",
              " (346,\n",
              "  12,\n",
              "  'Which is the most common cause of sudden cardiac death in young athletes?',\n",
              "  'Hypertrophic cardiomyopathy (HC) is probably the most common cause of sudden cardiac death in youthful athletes, and this diagnosis has represented a contraindication to continued participation in competitive sports.',\n",
              "  0),\n",
              " (3744,\n",
              "  0,\n",
              "  'Has tocilizumab been assessed against Covid-19?',\n",
              "  'Preliminary clinical results have indicated that antagonism of the IL-6 receptor (IL-6R), including with the FDA-approved humanized monoclonal antibody tocilizumab, can improve the outcomes of patients with severe or critical COVID-19 while maintaining a good safety profile.',\n",
              "  1),\n",
              " (935,\n",
              "  0,\n",
              "  'List Kartagener Syndrome Triad.',\n",
              "  'The triad of situs inversus, bronchiectasis and sinusitis is known as Kartagener syndrome.',\n",
              "  1),\n",
              " (935,\n",
              "  1,\n",
              "  'List Kartagener Syndrome Triad.',\n",
              "  'BACKGROUND: KARTAGENER SYNDROME (KS) IS A RARE CONGENITAL DISEASE CHARACTERISED BY A CLINICAL TRIAD OF SYMPTOMS: situs inversus, chronic rhinosinusitis, and bronchiectasis.',\n",
              "  0),\n",
              " (935,\n",
              "  2,\n",
              "  'List Kartagener Syndrome Triad.',\n",
              "  \"Kartagener's syndrome is a very rare congenital malformation comprising of a classic triad of sinusitis, situs inversus and bronchiectasis.\",\n",
              "  0),\n",
              " (935,\n",
              "  3,\n",
              "  'List Kartagener Syndrome Triad.',\n",
              "  'We present a case of a patient with clinically definite ALS, who had earlier suffered from Kartagener syndrome, which is characterized by the triad comprising chronic sinusitis, bronchiectasis, and situs inversus.',\n",
              "  0),\n",
              " (935,\n",
              "  4,\n",
              "  'List Kartagener Syndrome Triad.',\n",
              "  'Kartagener syndrome (KS), an autosomal recessively inherited disease, is characterized by the triad of situs inversus, bronchiectasis and sinusitis.',\n",
              "  1),\n",
              " (935,\n",
              "  5,\n",
              "  'List Kartagener Syndrome Triad.',\n",
              "  'In 1933, Manes Kartagener, a Zurich pulmonary physician, reported four patients with the triad of sinusitis, bronchiectasis, and situs inversus.',\n",
              "  0),\n",
              " (935,\n",
              "  6,\n",
              "  'List Kartagener Syndrome Triad.',\n",
              "  'A case of a nine year and eight months old child with Kartagener\\'s syndrome (triad) is described: chronic maxillary sinusitis, bronchiectasis and \"situs inversus totalis\".',\n",
              "  0),\n",
              " (935,\n",
              "  7,\n",
              "  'List Kartagener Syndrome Triad.',\n",
              "  \"Kartagener's syndrome is an inherited disease characterized by a triad of symptoms--bronchiectasis, situs inversus and sinusitis--and is classified as an immotile cilia syndrome.\",\n",
              "  1),\n",
              " (935,\n",
              "  8,\n",
              "  'List Kartagener Syndrome Triad.',\n",
              "  \"Kartagener's syndrome is characterized by the clinical triad of bronchitis, sinusitis, and situs inversus.\",\n",
              "  0),\n",
              " (935,\n",
              "  9,\n",
              "  'List Kartagener Syndrome Triad.',\n",
              "  \"Kartagener's syndrome is a well known classical triad of presentations consisting of bronchiectasis, sinusitis and situs inversus.\",\n",
              "  0),\n",
              " (935,\n",
              "  10,\n",
              "  'List Kartagener Syndrome Triad.',\n",
              "  \"Kartagener's syndrome is a rare disorder characterized by the triad of situs inversus, including dextrocardia, bronchiectasis and paranasal sinusitis.\",\n",
              "  0),\n",
              " (935,\n",
              "  11,\n",
              "  'List Kartagener Syndrome Triad.',\n",
              "  \"Kartagener's syndrome is an inherited disease characterized by a triad of symptoms: bronchiectasis, situs inversus and sinusitis resulting from defective cilial motility.\",\n",
              "  0),\n",
              " (935,\n",
              "  12,\n",
              "  'List Kartagener Syndrome Triad.',\n",
              "  \"Ear, nose and throat symptoms and signs were studied in 15 patients with Kartagener's syndrome: a triad consisting of chronic rhinosinusitis, chronic bronchitis with bronchiectasis, and situs inversus.\",\n",
              "  0),\n",
              " (935,\n",
              "  13,\n",
              "  'List Kartagener Syndrome Triad.',\n",
              "  'Kartagener syndrome (a clinical variant of primary ciliary dyskinesia) is a recessive autossomical disease characterized by the triad of chronic sinusitis, bronchiectasis and situs inversus with dextrocardia',\n",
              "  0),\n",
              " (935,\n",
              "  14,\n",
              "  'List Kartagener Syndrome Triad.',\n",
              "  'Kartagener Syndrome is a rare autosomal recessive disorder consisting of triad of sinusitis, bronchiectasis and situs inversus with dextrocardia',\n",
              "  0),\n",
              " (935,\n",
              "  15,\n",
              "  'List Kartagener Syndrome Triad.',\n",
              "  'Kartagener syndrome (KS), an autosomal recessively inherited disease, is characterized by the triad of situs inversus, bronchiectasis and sinusitis',\n",
              "  1),\n",
              " (935,\n",
              "  16,\n",
              "  'List Kartagener Syndrome Triad.',\n",
              "  'Kartagener&apos;s syndrome is an inherited disease characterized by a triad of symptoms--bronchiectasis, situs inversus and sinusitis--and is classified as an immotile cilia syndrome',\n",
              "  0),\n",
              " (935,\n",
              "  17,\n",
              "  'List Kartagener Syndrome Triad.',\n",
              "  'Kartagener&apos;s syndrome is a very rare congenital malformation comprising of a classic triad of sinusitis, situs inversus and bronchiectasis',\n",
              "  0),\n",
              " (935,\n",
              "  18,\n",
              "  'List Kartagener Syndrome Triad.',\n",
              "  'A comprehensive clinicomorphological examination of 24 children with Zivert-Kartagener syndrome ascertained the complete triad (bronchiectasis, maldevelopment of the sinuses and transposition of the viscera) in all of them',\n",
              "  0),\n",
              " (935,\n",
              "  19,\n",
              "  'List Kartagener Syndrome Triad.',\n",
              "  'The triad situs viscerum inversus, bronchiectasis and sinusitis is known as Kartagener syndrome',\n",
              "  1),\n",
              " (935,\n",
              "  20,\n",
              "  'List Kartagener Syndrome Triad.',\n",
              "  'We present a case of a patient with clinically definite ALS, who had earlier suffered from Kartagener syndrome, which is characterized by the triad comprising chronic sinusitis, bronchiectasis, and situs inversus.',\n",
              "  0),\n",
              " (935,\n",
              "  21,\n",
              "  'List Kartagener Syndrome Triad.',\n",
              "  'Recent linkage and mutational analyses identified several genes that are responsible for Kartagener syndrome.',\n",
              "  0),\n",
              " (2588,\n",
              "  0,\n",
              "  'Are osteoclasts specialized in bone degradation?',\n",
              "  'osteoclast-mediated attack on bone',\n",
              "  1),\n",
              " (2588,\n",
              "  1,\n",
              "  'Are osteoclasts specialized in bone degradation?',\n",
              "  'Bone degradation is caused by osteoclasts, the normal bone-resorbing cells.',\n",
              "  1),\n",
              " (2588,\n",
              "  2,\n",
              "  'Are osteoclasts specialized in bone degradation?',\n",
              "  'Cathepsin K is a highly potent collagenase in osteoclasts and is responsible for bone degradation.',\n",
              "  1),\n",
              " (2588,\n",
              "  3,\n",
              "  'Are osteoclasts specialized in bone degradation?',\n",
              "  'In osteoclasts, Src controls podosome organization and bone degradation, which leads to an osteopetrotic phenotype in src(-/-) mice',\n",
              "  1),\n",
              " (2588,\n",
              "  4,\n",
              "  'Are osteoclasts specialized in bone degradation?',\n",
              "  'Bone degradation by osteoclasts depends on the formation of a sealing zone, composed of an interlinked network of podosomes, which delimits the degradation lacuna into which osteoclasts secrete acid and proteolytic enzymes.',\n",
              "  1),\n",
              " (551,\n",
              "  0,\n",
              "  'Which receptors are bound by Tasimelteon?',\n",
              "  'We present here data on the in vitro binding affinity of tasimelteon for both human melatonin receptors MT1 and MT2, as well as the extended screen of other receptors and enzymes.',\n",
              "  0),\n",
              " (551,\n",
              "  1,\n",
              "  'Which receptors are bound by Tasimelteon?',\n",
              "  'Results indicate that tasimelteon is a potent Dual Melatonin Receptor Agonist (DMRA) with 2.1-4.4 times greater affinity for the MT2 receptor believed to mediate circadian rhythm phase-shifting (Ki = 0.0692 nM and Ki = 0.17 nM in NIH-3T3 and CHO-K1 cells, respectively), than for the MT1 receptor (Ki = 0.304 nM and Ki = 0.35 nM, respectively).',\n",
              "  0),\n",
              " (551,\n",
              "  2,\n",
              "  'Which receptors are bound by Tasimelteon?',\n",
              "  'The drugs covered target GABAA (zaleplon-CR, lorediplon, EVT-201), orexin (filorexant, MIN-202), histamine-H1 (LY2624803), serotonin 5-HT2A (ITI-007), melatonin/serotonin5-HT1A (piromelatine) and melatonin (indication expansions of prolonged-release melatonin and tasimelteon for pediatric sleep and circadian rhythm disorders) receptors.',\n",
              "  0),\n",
              " (551,\n",
              "  3,\n",
              "  'Which receptors are bound by Tasimelteon?',\n",
              "  'Several melatonin receptors agonists (ramelteon, prolonged-release melatonin, agomelatine and tasimelteon) have recently become available for the treatment of insomnia, depression and circadian rhythms sleep-wake disorders.',\n",
              "  0),\n",
              " (551,\n",
              "  4,\n",
              "  'Which receptors are bound by Tasimelteon?',\n",
              "  'Tasimelteon (HETLIOZ™) is an orally bioavailable agonist of the melatonin MT1 and MT2 receptors that has been approved in the US for the treatment of non-24-hour sleep-wake disorder.',\n",
              "  1),\n",
              " (551,\n",
              "  5,\n",
              "  'Which receptors are bound by Tasimelteon?',\n",
              "  'Numerous physiological functions of the pineal gland hormone melatonin are mediated via activation of two G-protein-coupled receptors, MT1 and MT2.',\n",
              "  0),\n",
              " (551,\n",
              "  6,\n",
              "  'Which receptors are bound by Tasimelteon?',\n",
              "  'The melatonergic drugs on the market, ramelteon and agomelatine, as well as the most advanced drug candidates under clinical evaluation, tasimelteon and TIK-301, are high-affinity nonselective MT1/MT2 agonists.',\n",
              "  0),\n",
              " (551,\n",
              "  7,\n",
              "  'Which receptors are bound by Tasimelteon?',\n",
              "  'A new melatonin 1 and melatonin 2 agonist tasimelteon improves sleep in these patients, resetting their circadian sleep-wake clocks.',\n",
              "  0),\n",
              " (551,\n",
              "  8,\n",
              "  'Which receptors are bound by Tasimelteon?',\n",
              "  'A general suitability in mitigating other symptoms of major depressive disorder cannot be deduced from the actions of tasimelteon via the melatonin receptors MT1 and MT2.',\n",
              "  1),\n",
              " (551,\n",
              "  9,\n",
              "  'Which receptors are bound by Tasimelteon?',\n",
              "  'Tasimelteon (HETLIOZ™) is an orally bioavailable agonist of the melatonin MT1 and MT2 receptors that has been approved in the US for the treatment of non-24-hour sleep-wake disorder.',\n",
              "  1),\n",
              " (551,\n",
              "  10,\n",
              "  'Which receptors are bound by Tasimelteon?',\n",
              "  'Ramelteon and tasimelteon are new chrono-hypnotic agents, selective for melatonin MT1 and MT2 receptors.',\n",
              "  0),\n",
              " (551,\n",
              "  11,\n",
              "  'Which receptors are bound by Tasimelteon?',\n",
              "  'Results indicate that tasimelteon is a potent Dual Melatonin Receptor Agonist (DMRA) with 2.1-4.4 times greater affinity for the MT2 receptor believed to mediate circadian rhythm phase-shifting (Ki = 0.0692 nM and Ki = 0.17 nM in NIH-3T3 and CHO-K1 cells, respectively), than for the MT1 receptor (Ki = 0.304 nM and Ki = 0.35 nM, respectively).',\n",
              "  0),\n",
              " (551,\n",
              "  12,\n",
              "  'Which receptors are bound by Tasimelteon?',\n",
              "  'A general suitability in mitigating other symptoms of major depressive disorder cannot be deduced from the actions of tasimelteon via the melatonin receptors MT1 and MT2.',\n",
              "  1),\n",
              " (551,\n",
              "  13,\n",
              "  'Which receptors are bound by Tasimelteon?',\n",
              "  'We present here data on the in vitro binding affinity of tasimelteon for both human melatonin receptors MT1 and MT2, as well as the extended screen of other receptors and enzymes.',\n",
              "  0),\n",
              " (551,\n",
              "  14,\n",
              "  'Which receptors are bound by Tasimelteon?',\n",
              "  'Results indicate that tasimelteon is a potent Dual Melatonin Receptor Agonist (DMRA) with 2.1-4.4 times greater affinity for the MT2 receptor believed to mediate circadian rhythm phase-shifting (Ki = 0.0692 nM and Ki = 0.17 nM in NIH-3T3 and CHO-K1 cells, respectively), than for the MT1 receptor (Ki = 0.304 nM and Ki = 0.35 nM, respectively).',\n",
              "  0),\n",
              " (551,\n",
              "  15,\n",
              "  'Which receptors are bound by Tasimelteon?',\n",
              "  'We present here data on the in vitro binding affinity of tasimelteon for both human melatonin receptors MT1 and MT2, as well as the extended screen of other receptors and enzymes.',\n",
              "  0),\n",
              " (551,\n",
              "  16,\n",
              "  'Which receptors are bound by Tasimelteon?',\n",
              "  'Results indicate that tasimelteon is a potent Dual Melatonin Receptor Agonist (DMRA) with 2.1-4.4 times greater affinity for the MT2 receptor believed to mediate circadian rhythm phase-shifting (Ki = 0.0692 nM and Ki = 0.17 nM in NIH-3T3 and CHO-K1 cells, respectively), than for the MT1 receptor (Ki = 0.304 nM and Ki = 0.35 nM, respectively).',\n",
              "  0),\n",
              " (551,\n",
              "  17,\n",
              "  'Which receptors are bound by Tasimelteon?',\n",
              "  'Results indicate that tasimelteon is a potent Dual Melatonin Receptor Agonist (DMRA) with 2.1-4.4 times greater affinity for the MT2 receptor believed to mediate circadian rhythm phase-shifting (Ki = 0.0692 nM and Ki = 0.17 nM in NIH-3T3 and CHO-K1 cells, respectively), than for the MT1 receptor (Ki = 0.304 nM and Ki = 0.35 nM, respectively).',\n",
              "  0),\n",
              " (551,\n",
              "  18,\n",
              "  'Which receptors are bound by Tasimelteon?',\n",
              "  'Tasimelteon was also shown to have no appreciable affinity for more than 160 other pharmacologically relevant receptors and several enzymes.',\n",
              "  0),\n",
              " (551,\n",
              "  19,\n",
              "  'Which receptors are bound by Tasimelteon?',\n",
              "  ' Hetlioz(®) (tasimelteon) is the first approved treatment in the United States for Non-24-Hour Sleep-Wake Disorder (Non-24).',\n",
              "  1),\n",
              " (551,\n",
              "  20,\n",
              "  'Which receptors are bound by Tasimelteon?',\n",
              "  'We present here data on the in vitro binding affinity of tasimelteon for both human melatonin receptors MT1 and MT2, as well as the extended screen of other receptors and enzymes.',\n",
              "  0),\n",
              " (551,\n",
              "  21,\n",
              "  'Which receptors are bound by Tasimelteon?',\n",
              "  'Results indicate that tasimelteon is a potent Dual Melatonin Receptor Agonist (DMRA) with 2.1-4.4 times greater affinity for the MT2 receptor believed to mediate circadian rhythm phase-shifting (Ki = 0.0692 nM and Ki = 0.17 nM in NIH-3T3 and CHO-K1 cells, respectively), than for the MT1 receptor (Ki = 0.304 nM and Ki = 0.35 nM, respectively).',\n",
              "  0),\n",
              " (551,\n",
              "  22,\n",
              "  'Which receptors are bound by Tasimelteon?',\n",
              "  'We present here data on the in vitro binding affinity of tasimelteon for both human melatonin receptors MT1 and MT2, as well as the extended screen of other receptors and enzymes.',\n",
              "  0),\n",
              " (367,\n",
              "  0,\n",
              "  \"Is bapineuzumab effective for treatment of patients with Alzheimer's disease?\",\n",
              "  ' Thus far, results from two large phase 3 trial programs with bapineuzumab and solaneuzumab, respectively, have brought rather disappointing results.',\n",
              "  0),\n",
              " (367,\n",
              "  1,\n",
              "  \"Is bapineuzumab effective for treatment of patients with Alzheimer's disease?\",\n",
              "  'More recently, in phase III studies, bapineuzumab has been discontinued because it did not prove clinically effective (despite its significant effect on biomarkers), while solaneuzumab has been found effective in slowing AD progression.',\n",
              "  0),\n",
              " (367,\n",
              "  2,\n",
              "  \"Is bapineuzumab effective for treatment of patients with Alzheimer's disease?\",\n",
              "  'Passive immunotherapy with monoclonal antibodies (mAbs) against Aβ is in late clinical development but recently the two most advanced mAbs, Bapineuzumab and Solanezumab, targeting an N-terminal or central epitope, respectively, failed to meet their target of improving or stabilizing cognition and function.',\n",
              "  0),\n",
              " (367,\n",
              "  3,\n",
              "  \"Is bapineuzumab effective for treatment of patients with Alzheimer's disease?\",\n",
              "  \"The marginal effects observed in recent clinical studies of solanezumab, targeting monomeric Aβ, and bapineuzumab, targeting amyloid plaques, prompted expert comments that drug discovery efforts in Alzheimer's disease should focus on soluble forms of Aβ rather than fibrillar Aβ deposits found in amyloid plaques.\",\n",
              "  0),\n",
              " (367,\n",
              "  4,\n",
              "  \"Is bapineuzumab effective for treatment of patients with Alzheimer's disease?\",\n",
              "  'Phase III trials showed that bapineuzumab failed to improve cognitive and functional performances in AD patients, and was associated with a high incidence of amyloid-related imaging abnormalities (ARIA).',\n",
              "  1),\n",
              " (367,\n",
              "  5,\n",
              "  \"Is bapineuzumab effective for treatment of patients with Alzheimer's disease?\",\n",
              "  'Clinical trials on various drugs, including AN1792, bapineuzumab, and solanezumab, have been carried out; however, all trials have failed to demonstrate apparent clinical benefits.',\n",
              "  0),\n",
              " (367,\n",
              "  6,\n",
              "  \"Is bapineuzumab effective for treatment of patients with Alzheimer's disease?\",\n",
              "  'Despite the alteration in biochemical composition, all 3 immunized subjects exhibited continued cognitive decline.',\n",
              "  0),\n",
              " (367,\n",
              "  7,\n",
              "  \"Is bapineuzumab effective for treatment of patients with Alzheimer's disease?\",\n",
              "  'Despite negative topline phase 3 clinical trial results for bapineuzumab and solanezumab in mild to moderate AD, findings from these trials and recent advances suggest renewed optimism for anti-amyloid therapies.',\n",
              "  0),\n",
              " (367,\n",
              "  8,\n",
              "  \"Is bapineuzumab effective for treatment of patients with Alzheimer's disease?\",\n",
              "  \"The lack of progress in the development of disease-modifying therapy in Alzheimer's disease (AD) was highlighted recently by the cessation of a phase 3 clinical trial studying the effects of bapineuzumab on mild to moderate disease.\",\n",
              "  0),\n",
              " (367,\n",
              "  9,\n",
              "  \"Is bapineuzumab effective for treatment of patients with Alzheimer's disease?\",\n",
              "  'No treatment benefit was apparent, whereas several serious side effects occurred more commonly in the treatment group compared to placebo.',\n",
              "  0),\n",
              " (367,\n",
              "  10,\n",
              "  \"Is bapineuzumab effective for treatment of patients with Alzheimer's disease?\",\n",
              "  'Clinical studies using the N-terminal-directed anti-Aβ antibody bapineuzumab have demonstrated reduced brain PET-Pittsburg-B signals, suggesting the reduction of Aβ plaques, and reduced levels of total and phosphorylated tau protein in the CSF of treated AD patients.',\n",
              "  0),\n",
              " (367,\n",
              "  11,\n",
              "  \"Is bapineuzumab effective for treatment of patients with Alzheimer's disease?\",\n",
              "  'Preclinical studies using 3D6 (the murine form of bapineuzumab) have demonstrated resolution of Aβ plaque and vascular burdens, neuritic dystrophy, and preservation of synaptic density in the transgenic APP mouse models.',\n",
              "  0),\n",
              " (367,\n",
              "  12,\n",
              "  \"Is bapineuzumab effective for treatment of patients with Alzheimer's disease?\",\n",
              "  'The clinical results of the initial studies with bapineuzumab were equivocal in terms of cognitive benefit.',\n",
              "  0),\n",
              " (367,\n",
              "  13,\n",
              "  \"Is bapineuzumab effective for treatment of patients with Alzheimer's disease?\",\n",
              "  'The occurrence of vasogenic edema after bapineuzumab, and more rarely brain microhemorrhages (especially in Apo E ε4 carriers), has raised concerns on the safety of these antibodies directed against the N-terminus of the Aβ peptide.',\n",
              "  0),\n",
              " (367,\n",
              "  14,\n",
              "  \"Is bapineuzumab effective for treatment of patients with Alzheimer's disease?\",\n",
              "  ' The most advanced of these immunological approaches is bapineuzumab, composed of humanized anti-Aβ monoclonal antibodies, that has been tested in two Phase II trials, demonstrating to reduce Aβ burden in the brain of AD patients.',\n",
              "  1),\n",
              " (367,\n",
              "  15,\n",
              "  \"Is bapineuzumab effective for treatment of patients with Alzheimer's disease?\",\n",
              "  'However, the preliminary cognitive efficacy of bapineuzumab appears uncertain.',\n",
              "  0),\n",
              " (367,\n",
              "  16,\n",
              "  \"Is bapineuzumab effective for treatment of patients with Alzheimer's disease?\",\n",
              "  'The occurrence of vasogenic edema, especially in apolipoprotein E 4 carriers, may limit its clinical use and have led to abandon the highest dose of the drug (2 mg/kg).',\n",
              "  0),\n",
              " (367,\n",
              "  17,\n",
              "  \"Is bapineuzumab effective for treatment of patients with Alzheimer's disease?\",\n",
              "  'However, the preliminary cognitive efficacy of bapineuzumab, a humanized anti-Aβ monoclonal antibody, appears uncertain.',\n",
              "  0),\n",
              " (367,\n",
              "  18,\n",
              "  \"Is bapineuzumab effective for treatment of patients with Alzheimer's disease?\",\n",
              "  'Moreover, the occurrence of vasogenic edema and, more rarely, brain microhemorrhages, especially in apolipoprotein E ϵ4 carriers, have led to abandoning of the highest dose of the drug.',\n",
              "  0),\n",
              " (367,\n",
              "  19,\n",
              "  \"Is bapineuzumab effective for treatment of patients with Alzheimer's disease?\",\n",
              "  'However, the preliminary equivocal cognitive results obtained with bapineuzumab as well as the detrimental cognitive effects observed with semagacestat, a potent γ-secretase inhibitor, raise the possibility that targeting Aβ may not be clinically efficacious in AD.',\n",
              "  0),\n",
              " (367,\n",
              "  20,\n",
              "  \"Is bapineuzumab effective for treatment of patients with Alzheimer's disease?\",\n",
              "  'The patient received four bapineuzumab infusions over a 39 week period.',\n",
              "  0),\n",
              " (367,\n",
              "  21,\n",
              "  \"Is bapineuzumab effective for treatment of patients with Alzheimer's disease?\",\n",
              "  'During the course of this treatment, there was no remarkable change in cognitive impairment as determined by MMSE scores.',\n",
              "  0),\n",
              " (367,\n",
              "  22,\n",
              "  \"Is bapineuzumab effective for treatment of patients with Alzheimer's disease?\",\n",
              "  'Forty-eight days after the fourth bapineuzumab infusion was given, MRI revealed that the patient had developed lacunar infarcts and possible vasogenic edema, probably related to immunotherapy, but a subsequent MRI scan 38 days later demonstrated resolution of vasogenic edema.',\n",
              "  0),\n",
              " (367,\n",
              "  23,\n",
              "  \"Is bapineuzumab effective for treatment of patients with Alzheimer's disease?\",\n",
              "  'The patient expired due to acute congestive heart failure complicated by progressive AD and cerebrovascular accident 378 days after the first bapineuzumab infusion and 107 days after the end of therapy.',\n",
              "  0),\n",
              " (367,\n",
              "  24,\n",
              "  \"Is bapineuzumab effective for treatment of patients with Alzheimer's disease?\",\n",
              "  'Neuropathological and biochemical analysis did not produce evidence of lasting plaque regression or clearance of Aβ due to immunotherapy.',\n",
              "  0),\n",
              " (367,\n",
              "  25,\n",
              "  \"Is bapineuzumab effective for treatment of patients with Alzheimer's disease?\",\n",
              "  'These results suggest that, in this particular case, bapineuzumab immunotherapy neither resulted in detectable clearance of amyloid plaques nor prevented further cognitive impairment.',\n",
              "  0),\n",
              " (367,\n",
              "  26,\n",
              "  \"Is bapineuzumab effective for treatment of patients with Alzheimer's disease?\",\n",
              "  'Bapineuzumab has been shown to reduce Aβ burden in the brain of AD patients.',\n",
              "  0),\n",
              " (367,\n",
              "  27,\n",
              "  \"Is bapineuzumab effective for treatment of patients with Alzheimer's disease?\",\n",
              "  'However, its preliminary cognitive efficacy appears uncertain, particularly in ApoE ε4 carriers, and vasogenic edema may limit its clinical use.',\n",
              "  0),\n",
              " (367,\n",
              "  28,\n",
              "  \"Is bapineuzumab effective for treatment of patients with Alzheimer's disease?\",\n",
              "  \"Bapineuzumab appears capable of reducing the cerebral beta-amyloid peptide burden in patients with Alzheimer's disease.\",\n",
              "  1),\n",
              " (367,\n",
              "  29,\n",
              "  \"Is bapineuzumab effective for treatment of patients with Alzheimer's disease?\",\n",
              "  'However, particularly in APOE 4 carriers, its ability to slow disease progression remains uncertain, and vasogenic edema - a dose-limiting and potentially severe adverse reaction - may limit its clinical applicability.',\n",
              "  0),\n",
              " (367,\n",
              "  30,\n",
              "  \"Is bapineuzumab effective for treatment of patients with Alzheimer's disease?\",\n",
              "  'The first is a phase 2 study of passive immunotherapy with bapineuzumab, a humanized anti-Abeta monoclonal antibody directed against the N-terminus of Abeta.',\n",
              "  1),\n",
              " (367,\n",
              "  31,\n",
              "  \"Is bapineuzumab effective for treatment of patients with Alzheimer's disease?\",\n",
              "  'This trial showed no differences within dose cohorts on the primary efficacy analysis.',\n",
              "  0),\n",
              " (367,\n",
              "  32,\n",
              "  \"Is bapineuzumab effective for treatment of patients with Alzheimer's disease?\",\n",
              "  'Exploratory analyses showed potential treatment differences on cognitive and functional endpoints in study completers and apolipoprotein E epsilon4 noncarriers.',\n",
              "  0),\n",
              " (367,\n",
              "  33,\n",
              "  \"Is bapineuzumab effective for treatment of patients with Alzheimer's disease?\",\n",
              "  'A safety concern was the occurrence of reversible vasogenic edema.',\n",
              "  0),\n",
              " (367,\n",
              "  34,\n",
              "  \"Is bapineuzumab effective for treatment of patients with Alzheimer's disease?\",\n",
              "  'The first passive immunotherapy trial with bapineuzumab, a humanized monoclonal antibody against the end terminus of Abeta, also encountered some dose dependent adverse events during the Phase II portion of the study, vasogenic edema in 12 cases, which were significantly over represented in ApoE4 carriers.',\n",
              "  1),\n",
              " (1943,\n",
              "  0,\n",
              "  'What does the human ABCC gene  product do?',\n",
              "  'structure, function, and expression of the important drug resistance-conferring members belonging to three subfamilies of the human ABC family; these are ABCB1 (MDR1/P-glycoprotein of subfamily ABCB), subfamily ABCC (MRPs), and ABCG2 (BCRP of subfamily ABCG), which are expressed in various organs.',\n",
              "  1),\n",
              " (1943,\n",
              "  1,\n",
              "  'What does the human ABCC gene  product do?',\n",
              "  'Transport proteins, including members of the multidrug resistance protein (MRP)/ABCC subfamily, have been recognized to contribute to the latter function.',\n",
              "  0),\n",
              " (1943,\n",
              "  2,\n",
              "  'What does the human ABCC gene  product do?',\n",
              "  \"MRP5 (ABCC5) was identified as transmembrane transport protein for cyclic nucleotides, especially 3',5'-cyclic GMP (cGMP), indicating an additional role in signal transduction and a potential role in placenta development.\",\n",
              "  0),\n",
              " (1943,\n",
              "  3,\n",
              "  'What does the human ABCC gene  product do?',\n",
              "  \"According to the new nomenclature of human ABC transporter genes, the 'ABCC' gene sub-family comprises three classes involving multidrug resistance-associated proteins (MRPs), sulfonylurea receptors (SURs), and a cystic fibrosis transmembrane conductance regulator (CFTR).\",\n",
              "  0),\n",
              " (1943,\n",
              "  4,\n",
              "  'What does the human ABCC gene  product do?',\n",
              "  'With the addition of these two genes, the complete human ABCC subfamily has 12 identified members (ABCC1-12), nine from the multidrug resistance-like subgroup, two from the sulfonylurea receptor subgroup, and the CFTR gene.',\n",
              "  0),\n",
              " (1943,\n",
              "  5,\n",
              "  'What does the human ABCC gene  product do?',\n",
              "  'Identification and bioinformatic characterization of a multidrug resistance associated protein (ABCC) gene in Plasmodium berghei.',\n",
              "  0),\n",
              " (1943,\n",
              "  6,\n",
              "  'What does the human ABCC gene  product do?',\n",
              "  'One major group of transporters is known as multidrug resistance associated proteins (MRP; ABCC gene family).',\n",
              "  0),\n",
              " (1943,\n",
              "  7,\n",
              "  'What does the human ABCC gene  product do?',\n",
              "  'On the other hand, several human multidrug resistance proteins [human ATP-binding cassette transporter, subfamily C (ABCC)] cause resistance against nucleoside analogs and mediate transport of phosphorylated nucleoside derivatives out of the cells in an ATP-dependent manner.',\n",
              "  1),\n",
              " (1943,\n",
              "  8,\n",
              "  'What does the human ABCC gene  product do?',\n",
              "  \"According to the new nomenclature of human ABC transporter genes, the 'ABCC' gene sub-family comprises three classes involving multidrug resistance-associated proteins (MRPs), sulfonylurea receptors (SURs), and a cystic fibrosis transmembrane conductance regulator (CFTR).\",\n",
              "  0),\n",
              " (1943,\n",
              "  9,\n",
              "  'What does the human ABCC gene  product do?',\n",
              "  'Several MRP/ABCC members (MRPs 1-3) are associated with tumor resistance which is often caused by an increased efflux and decreased intracellular accumulation of natural product anticancer drugs and other anticancer agents.',\n",
              "  0),\n",
              " (1943,\n",
              "  10,\n",
              "  'What does the human ABCC gene  product do?',\n",
              "  'In vitro, the MRP/ABCC transporters can collectively confer resistance to natural product anticancer drugs and their conjugated metabolites, platinum compounds, folate antimetabolites, nucleoside and nucleotide analogs, arsenical and antimonial oxyanions, peptide-based agents, and in concert with alterations in phase II conjugating or biosynthetic enzymes, classical alkylating agents, alkylating agents.',\n",
              "  0),\n",
              " (1943,\n",
              "  11,\n",
              "  'What does the human ABCC gene  product do?',\n",
              "  'The human MRP/ABCC transporters except MRP9/ABCC12 are all able to transport organic anions, such as drugs conjugated to glutathione, sulphate or glucuronate.',\n",
              "  0),\n",
              " (1943,\n",
              "  12,\n",
              "  'What does the human ABCC gene  product do?',\n",
              "  'The ABCC multidrug resistance associated proteins (ABCC-MRP), a subclass of ABC transporters are involved in multiple physiological processes that include cellular homeostasis, metal detoxification, and transport of glutathione-conjugates.',\n",
              "  1),\n",
              " (1943,\n",
              "  13,\n",
              "  'What does the human ABCC gene  product do?',\n",
              "  'he ABCC subfamily of the ATP binding cassette (ABC) transporters, which were formerly known as multidrug resistance-related proteins (MRPs), consists of closely related members found in all eukaryotic organisms.',\n",
              "  1),\n",
              " (1943,\n",
              "  14,\n",
              "  'What does the human ABCC gene  product do?',\n",
              "  'The ATP-binding cassette (ABC) transporters are a superfamily of membrane proteins that are best known for their ability to transport a wide variety of exogenous and endogenous substances across membranes against a concentration gradient via ATP hydrolysis.',\n",
              "  1),\n",
              " (1943,\n",
              "  15,\n",
              "  'What does the human ABCC gene  product do?',\n",
              "  \"There are seven subfamilies of human ABC transporters, one of the largest being the 'C' subfamily (gene symbol ABCC).\",\n",
              "  0),\n",
              " (1943,\n",
              "  16,\n",
              "  'What does the human ABCC gene  product do?',\n",
              "  'Nine ABCC subfamily members, the so-called multidrug resistance proteins (MRPs) 1-9, have been implicated in mediating multidrug resistance in tumor cells to varying degrees as the efflux extrude chemotherapeutic compounds (or their metabolites) from malignant cells',\n",
              "  0),\n",
              " (1785,\n",
              "  0,\n",
              "  'Which R package is used for the detection of chromosomal abnormalities from microarray data?',\n",
              "  'CAFE: an R package for the detection of gross chromosomal abnormalities from gene expression microarray data',\n",
              "  1),\n",
              " (1785,\n",
              "  1,\n",
              "  'Which R package is used for the detection of chromosomal abnormalities from microarray data?',\n",
              "  'CAFE: an R package for the detection of gross chromosomal abnormalities from gene expression microarray data.',\n",
              "  1),\n",
              " (1785,\n",
              "  2,\n",
              "  'Which R package is used for the detection of chromosomal abnormalities from microarray data?',\n",
              "  'It is implemented as an R package that analyzes Affymetrix *.CEL files and comes with flexible plotting functions, easing visualization of chromosomal abnormalities.CAFE is available from https://bitbucket.org/cob87icW6z/cafe/ as both source and compiled packages for Linux and Windows.',\n",
              "  0),\n",
              " (1785,\n",
              "  3,\n",
              "  'Which R package is used for the detection of chromosomal abnormalities from microarray data?',\n",
              "  'SUMMARY: The current methods available to detect chromosomal abnormalities from DNA microarray expression data are cumbersome and inflexible.',\n",
              "  1),\n",
              " (1785,\n",
              "  4,\n",
              "  'Which R package is used for the detection of chromosomal abnormalities from microarray data?',\n",
              "  'CAFE has been developed to alleviate these issues.',\n",
              "  0),\n",
              " (1785,\n",
              "  5,\n",
              "  'Which R package is used for the detection of chromosomal abnormalities from microarray data?',\n",
              "  'It is implemented as an R package that analyzes Affymetrix *.CEL files and comes with flexible plotting functions, easing visualization of chromosomal abnormalities.AVAILABILITY AND IMPLEMENTATION: CAFE is available from https://bitbucket.org/cob87icW6z/cafe/ as both source and compiled packages for Linux and Windows.',\n",
              "  0),\n",
              " (1785,\n",
              "  6,\n",
              "  'Which R package is used for the detection of chromosomal abnormalities from microarray data?',\n",
              "  'SUMMARY: The current methods available to detect chromosomal abnormalities from DNA microarray expression data are cumbersome and inflexible.',\n",
              "  1),\n",
              " (1785,\n",
              "  7,\n",
              "  'Which R package is used for the detection of chromosomal abnormalities from microarray data?',\n",
              "  'CAFE has been developed to alleviate these issues.',\n",
              "  0),\n",
              " (1785,\n",
              "  8,\n",
              "  'Which R package is used for the detection of chromosomal abnormalities from microarray data?',\n",
              "  'CAFE: an R package for the detection of gross chromosomal abnormalities from gene expression microarray data.',\n",
              "  1),\n",
              " (2316,\n",
              "  0,\n",
              "  'Can the CEP290 gene mutations be targeted by AAV-mediated gene therapy?',\n",
              "  'The large size of the CEP290 gene prevents its use in adeno-associated virus (AAV)-mediated gene augmentation therapy.',\n",
              "  1),\n",
              " (3884,\n",
              "  0,\n",
              "  'Is there high nucleotide diversity in the Drosophila suzukii species?',\n",
              "  'Native to Asia, the soft-skinned fruit pest Drosophila suzukii has recently invaded the United States and Europe.',\n",
              "  1),\n",
              " (3884,\n",
              "  1,\n",
              "  'Is there high nucleotide diversity in the Drosophila suzukii species?',\n",
              "  'The eastern United States represents the most recent expansion of their range, and presents an opportunity to test alternative models of colonization history.',\n",
              "  1),\n",
              " (3884,\n",
              "  2,\n",
              "  'Is there high nucleotide diversity in the Drosophila suzukii species?',\n",
              "  'Here, we investigate the genetic population structure of this invasive fruit fly, with a focus on the eastern United States.',\n",
              "  1),\n",
              " (3884,\n",
              "  3,\n",
              "  'Is there high nucleotide diversity in the Drosophila suzukii species?',\n",
              "  'We sequenced six X-linked gene fragments from 246 individuals collected from a total of 12 populations.',\n",
              "  0),\n",
              " (3884,\n",
              "  4,\n",
              "  'Is there high nucleotide diversity in the Drosophila suzukii species?',\n",
              "  'We examine patterns of genetic diversity within and between populations and explore alternative colonization scenarios using approximate Bayesian computation.',\n",
              "  1),\n",
              " (3884,\n",
              "  5,\n",
              "  'Is there high nucleotide diversity in the Drosophila suzukii species?',\n",
              "  'Our results indicate high levels of nucleotide diversity in this species and suggest that the recent invasions of Europe and the continental United States are independent demographic events.',\n",
              "  1),\n",
              " (3872,\n",
              "  0,\n",
              "  'What is Corkscrew Esophagus?',\n",
              "  'Corkscrew esophagus (also referred as rosary bead esophagus) is a classic finding of diffuse esophageal spasm (DES) in barium studies reflecting abnormal contractions, leading to compartmentalization and curling of the esophagus, ultimately giving an appearance similar to a corkscrew or rosary beads.',\n",
              "  1),\n",
              " (3872,\n",
              "  1,\n",
              "  'What is Corkscrew Esophagus?',\n",
              "  \"Esophagography evidenced a disorder of esophagus motility with diffuse multiple spasm, reminiscent of the 'corkscrew esophagus'.\",\n",
              "  1),\n",
              " (3872,\n",
              "  2,\n",
              "  'What is Corkscrew Esophagus?',\n",
              "  'Corkscrew oesophagus is a radiological diagnosis and is characterised by twisted segments in the distal third of the oesophagus.',\n",
              "  0),\n",
              " (3872,\n",
              "  3,\n",
              "  'What is Corkscrew Esophagus?',\n",
              "  'Nonpropulsive esophageal contractions radiologically described as tertiary contractions or \"corkscrew\" esophagus suggest the presence of an underlying motility disorder and may lead to impaired acid clearance.',\n",
              "  0),\n",
              " (3872,\n",
              "  4,\n",
              "  'What is Corkscrew Esophagus?',\n",
              "  'Corkscrew esophagus (also referred as rosary bead esophagus) is a classic finding of diffuse esophageal spasm (DES) in barium studies reflecting abnormal contractions, leading to compartmentalization and curling of the esophagus, ultimately giving an appearance similar to a corkscrew or rosary beads.',\n",
              "  1),\n",
              " (3872,\n",
              "  5,\n",
              "  'What is Corkscrew Esophagus?',\n",
              "  \"Esophagography evidenced a disorder of esophagus motility with diffuse multiple spasm, reminiscent of the 'corkscrew esophagus'.\",\n",
              "  0),\n",
              " (3872,\n",
              "  6,\n",
              "  'What is Corkscrew Esophagus?',\n",
              "  'Corkscrew esophagus (also referred as rosary bead esophagus) is a classic finding of diffuse esophageal spasm (DES) in barium studies reflecting abnormal contractions, leading to compartmentalization and curling of the esophagus, ultimately giving an appearance similar to a corkscrew or rosary beads.',\n",
              "  1),\n",
              " (3872, 7, 'What is Corkscrew Esophagus?', 'We r', 0),\n",
              " (3872,\n",
              "  8,\n",
              "  'What is Corkscrew Esophagus?',\n",
              "  'Barium swallow examination revealed the esophagus to have the corkscrew appearance characteristic of diffuse esophageal spasm (DES).',\n",
              "  1),\n",
              " (2060,\n",
              "  0,\n",
              "  'Do brown fat cells produce heat?',\n",
              "  'WAT and BAT are both involved in energy balance.',\n",
              "  0),\n",
              " (2060,\n",
              "  1,\n",
              "  'Do brown fat cells produce heat?',\n",
              "  'WAT is mainly involved in the storage and mobilization of energy in the form of triglycerides, whereas BAT specializes in dissipating energy as heat during cold- or diet-induced thermogenesis.',\n",
              "  0),\n",
              " (2060,\n",
              "  2,\n",
              "  'Do brown fat cells produce heat?',\n",
              "  'Because brown adipose\\xa0tissue (BAT) dissipates energy in the form of heat, increasing energy expenditure by augmenting BAT-mediated thermogenesis may represent an approach to counter obesity and its complications.',\n",
              "  0),\n",
              " (2060,\n",
              "  3,\n",
              "  'Do brown fat cells produce heat?',\n",
              "  'Classic brown fat and inducible beige fat both dissipate chemical energy in the form of heat through the actions of mitochondrial uncoupling protein 1.',\n",
              "  0),\n",
              " (2060,\n",
              "  4,\n",
              "  'Do brown fat cells produce heat?',\n",
              "  'This nonshivering thermogenesis is crucial for mammals as a defense against cold and obesity/diabetes.',\n",
              "  0),\n",
              " (2060,\n",
              "  5,\n",
              "  'Do brown fat cells produce heat?',\n",
              "  'Mitochondrial uncoupling protein 1 in brown fat cells produces heat by dissipating the energy generated by fatty acid and glucose oxidation.',\n",
              "  1),\n",
              " (2060,\n",
              "  6,\n",
              "  'Do brown fat cells produce heat?',\n",
              "  'Brown fat biology and thermogenesis.',\n",
              "  1),\n",
              " (2060,\n",
              "  7,\n",
              "  'Do brown fat cells produce heat?',\n",
              "  'Brown fat (brown adipose tissue, BAT) primary function is to produce heat.',\n",
              "  1),\n",
              " (2060,\n",
              "  8,\n",
              "  'Do brown fat cells produce heat?',\n",
              "  'Brown fat cells were classified into 6 types: Type 1 cells are fat-depleted cells filled with granular cytoplasm and are believed to be produced after oxidation of fat for heat production.',\n",
              "  0),\n",
              " (2060,\n",
              "  9,\n",
              "  'Do brown fat cells produce heat?',\n",
              "  'Calorimetric measurements from cell suspensions showed that ATP increased basal heat production of isolated brown fat cells by approximately 40% but had no effect on the greater than fivefold increase in heat production seen with maximal adrenergic stimulation.',\n",
              "  0),\n",
              " (2060,\n",
              "  10,\n",
              "  'Do brown fat cells produce heat?',\n",
              "  'Classic brown fat and inducible beige fat both dissipate chemical energy in the form of heat through the actions of mitochondrial uncoupling protein 1.',\n",
              "  0),\n",
              " (2060,\n",
              "  11,\n",
              "  'Do brown fat cells produce heat?',\n",
              "  'Brown adipocytes oxidize fatty acids to produce heat in response to cold or to excessive energy intake; stimulation of brown fat development and function may thus counteract obesity.',\n",
              "  0),\n",
              " (2060,\n",
              "  12,\n",
              "  'Do brown fat cells produce heat?',\n",
              "  'The occurrence of Types 1 and/or 6 cells that has been revealed in 65 out of the total 180 samples (36%), suggests that the oxidation of fat for the thermogenesis proceeds in the brown fat tissue and that brown fat cells partially undergo fat depletion.',\n",
              "  0),\n",
              " (2060,\n",
              "  13,\n",
              "  'Do brown fat cells produce heat?',\n",
              "  'Brown fat cells were classified into 6 types: Type 1 cells are fat-depleted cells filled with granular cytoplasm and are believed to be produced after oxidation of fat for heat production.',\n",
              "  0),\n",
              " (2060,\n",
              "  14,\n",
              "  'Do brown fat cells produce heat?',\n",
              "  'In response to cold, both classical brown fat and the newly identified \"beige\" or \"brite\" cells are activated by β-adrenergic signaling and catabolize stored lipids and carbohydrates to produce heat via UCP1',\n",
              "  0),\n",
              " (2060,\n",
              "  15,\n",
              "  'Do brown fat cells produce heat?',\n",
              "  'The ability of brown adipocytes (fat cells) to dissipate energy as heat shows great promise for the treatment of obesity and other metabolic disorders',\n",
              "  0),\n",
              " (2060,\n",
              "  16,\n",
              "  'Do brown fat cells produce heat?',\n",
              "  'Inappropriate heat dissipation ignites brown fat thermogenesis in mice with a mutant thyroid hormone receptor α1',\n",
              "  0),\n",
              " (2060,\n",
              "  17,\n",
              "  'Do brown fat cells produce heat?',\n",
              "  'Brown fat and vascular heat dissipation: The new cautionary tail',\n",
              "  1),\n",
              " (2060,\n",
              "  18,\n",
              "  'Do brown fat cells produce heat?',\n",
              "  'Brown adipose produces heat as a defense against hypothermia and obesity, and the appearance of brown-like adipocytes within white adipose tissue depots is associated with improved metabolic phenotypes.',\n",
              "  0),\n",
              " (2060,\n",
              "  19,\n",
              "  'Do brown fat cells produce heat?',\n",
              "  'In the same manner, marked ability to produce a considerable amount of heat was evidenced in  brown fat tissue of children and teenagers.',\n",
              "  0),\n",
              " (2060,\n",
              "  20,\n",
              "  'Do brown fat cells produce heat?',\n",
              "  'Brown fat cells were classified into 6 types: Type 1 cells are fat-depleted cells filled with granular cytoplasm and are believed to be produced after oxidation of fat for heat production.',\n",
              "  0),\n",
              " (2060,\n",
              "  21,\n",
              "  'Do brown fat cells produce heat?',\n",
              "  'It is inferred that brown-adipose-tissue heat production is reduced during (and probably also some time after) anesthesia.',\n",
              "  0),\n",
              " (2060,\n",
              "  22,\n",
              "  'Do brown fat cells produce heat?',\n",
              "  'Parallel measurements of heat production and thermogenin content in brown fat cells during cold acclimation of rats.',\n",
              "  0),\n",
              " (2060,\n",
              "  23,\n",
              "  'Do brown fat cells produce heat?',\n",
              "  'The classical white adipose tissue builds up energy in the form of triglycerides and is useful for preventing fatigue during periods of low caloric intake and the brown adipose tissue instead of inducing fat accumulation can produce energy as heat.',\n",
              "  0),\n",
              " (2060,\n",
              "  24,\n",
              "  'Do brown fat cells produce heat?',\n",
              "  'In response to cold, both classical brown fat and the newly identified \"beige\" or \"brite\" cells are activated by β-adrenergic signaling and catabolize stored lipids and carbohydrates to produce heat via UCP1.',\n",
              "  0),\n",
              " (2060,\n",
              "  25,\n",
              "  'Do brown fat cells produce heat?',\n",
              "  'White adipose tissue stores energy reserves as fat, whereas the metabolic function of brown adipose tissue is lipid oxidation to produce heat.',\n",
              "  0),\n",
              " (2060,\n",
              "  26,\n",
              "  'Do brown fat cells produce heat?',\n",
              "  'The main function of brown adipose tissue (BAT) is to produce heat in response to cold.',\n",
              "  0),\n",
              " (2060,\n",
              "  27,\n",
              "  'Do brown fat cells produce heat?',\n",
              "  'Brown adipocytes oxidize fatty acids to produce heat in response to cold or caloric overfeeding.',\n",
              "  0),\n",
              " (2060,\n",
              "  28,\n",
              "  'Do brown fat cells produce heat?',\n",
              "  'Brown fat (brown adipose tissue, BAT) primary function is to produce heat.',\n",
              "  1),\n",
              " (2060,\n",
              "  29,\n",
              "  'Do brown fat cells produce heat?',\n",
              "  'Adipose tissue plays an active role in energy balance because it is not only a lipid storing and mobilizing tissue but consists of functionally specialized tissues able to produce heat (in brown adipose tissue) and to produce or release a vast number of so called adipokines or adipocytokines.',\n",
              "  0),\n",
              " (2060,\n",
              "  30,\n",
              "  'Do brown fat cells produce heat?',\n",
              "  'Brown adipose tissue (BAT), a specialized fat that dissipates energy to produce heat, plays an important role in the regulation of energy balance.',\n",
              "  0),\n",
              " (2060,\n",
              "  31,\n",
              "  'Do brown fat cells produce heat?',\n",
              "  'Brown adipose cells are specialized to dissipate chemical energy in the form of heat, as a physiological defence against cold and obesity.',\n",
              "  0),\n",
              " (2060,\n",
              "  32,\n",
              "  'Do brown fat cells produce heat?',\n",
              "  'In the present study, the thermogenesis of human brown fat tissue was suggested chiefly with regard to the occurrence of Types 1 and/or 6 cells.',\n",
              "  0),\n",
              " (2060,\n",
              "  33,\n",
              "  'Do brown fat cells produce heat?',\n",
              "  'In the same manner, marked ability to produce a considerable amount of heat was evidenced in  brown fat tissue of children and teenagers.',\n",
              "  0),\n",
              " (2060,\n",
              "  34,\n",
              "  'Do brown fat cells produce heat?',\n",
              "  'Adult humans have heat-producing and energy-consuming brown adipose tissue in the clavicular region of the neck.',\n",
              "  0),\n",
              " (2060,\n",
              "  35,\n",
              "  'Do brown fat cells produce heat?',\n",
              "  'Brown and beige adipocytes expend chemical energy to produce heat and are therefore important in regulating body temperature and body weight.',\n",
              "  0),\n",
              " (2060,\n",
              "  36,\n",
              "  'Do brown fat cells produce heat?',\n",
              "  \"In human perirenal brown fat tissue, darkly stained fat-depleted cells (D) occupy, with other cell types (CR, CR'), an important part in the reversible heat production cycle of the brown fat tissue.\",\n",
              "  0),\n",
              " (2060,\n",
              "  37,\n",
              "  'Do brown fat cells produce heat?',\n",
              "  'Brown fat is a specialized fat depot that can increase energy expenditure and produce heat.',\n",
              "  0),\n",
              " (2326,\n",
              "  0,\n",
              "  'How may CTCF mediate splicing?',\n",
              "  'Two different mechanisms convey DNA methylation information into the regulation of alternative splicing.',\n",
              "  0),\n",
              " (2326,\n",
              "  1,\n",
              "  'How may CTCF mediate splicing?',\n",
              "  'The first involves modulation of the elongation rate of RNA polymerase II (Pol II) by CCCTC-binding factor (CTCF) and methyl-CpG binding protein 2 (MeCP2); the second involves the formation of a protein bridge by heterochromatin protein 1 (HP1) that recruits splicing factors onto transcribed alternative exons.',\n",
              "  1),\n",
              " (2326,\n",
              "  2,\n",
              "  'How may CTCF mediate splicing?',\n",
              "  'Furthermore, we found mutations in the splicing factor SFPQ and in the nonclassic regulators of mRNA processing CTCF and RAD21.',\n",
              "  0),\n",
              " (2326,\n",
              "  3,\n",
              "  'How may CTCF mediate splicing?',\n",
              "  ' We show that CTCF bound to HTLV-1 acts as an enhancer blocker, regulates HTLV-1 mRNA splicing, and forms long-distance interactions with flanking host chromatin.',\n",
              "  0),\n",
              " (2326,\n",
              "  4,\n",
              "  'How may CTCF mediate splicing?',\n",
              "  ' Many CTCF sites are also involved in other functions such as modulation of RNA splicing and specific regulation of gene expression, and the relationship between these activities and loop formation is another unanswered question that should keep investigators occupied for some time.',\n",
              "  0),\n",
              " (2326,\n",
              "  5,\n",
              "  'How may CTCF mediate splicing?',\n",
              "  'TET-catalyzed oxidation of intragenic 5-methylcytosine regulates CTCF-dependent alternative splicing.',\n",
              "  0),\n",
              " (2326,\n",
              "  6,\n",
              "  'How may CTCF mediate splicing?',\n",
              "  'Intragenic 5-methylcytosine and CTCF mediate opposing effects on pre-mRNA splicing: CTCF promotes inclusion of weak upstream exons through RNA polymerase II pausing, whereas 5-methylcytosine evicts CTCF, leading to exon exclusion.',\n",
              "  1),\n",
              " (2326,\n",
              "  7,\n",
              "  'How may CTCF mediate splicing?',\n",
              "  'Here, we reveal the methylcytosine dioxygenases TET1 and TET2 as active regulators of CTCF-mediated alternative splicing through conversion of 5-methylcytosine to its oxidation derivatives.',\n",
              "  0),\n",
              " (2326,\n",
              "  8,\n",
              "  'How may CTCF mediate splicing?',\n",
              "  'A chromatin code for alternative splicing involving a putative association between CTCF and HP1α proteins.',\n",
              "  0),\n",
              " (2326,\n",
              "  9,\n",
              "  'How may CTCF mediate splicing?',\n",
              "  'Recent experiments have suggested a new complex network of splicing regulation involving chromatin, transcription and multiple protein factors.',\n",
              "  0),\n",
              " (2326,\n",
              "  10,\n",
              "  'How may CTCF mediate splicing?',\n",
              "  'In particular, the CCCTC-binding factor (CTCF), the Argonaute protein AGO1, and members of the heterochromatin protein 1 (HP1) family have been implicated in the regulation of splicing associated with chromatin and the elongation of RNAPII.',\n",
              "  1),\n",
              " (2326,\n",
              "  11,\n",
              "  'How may CTCF mediate splicing?',\n",
              "  'Our results show that a considerable number of alternative splicing events could have a chromatin-dependent regulation involving the association of HP1α and CTCF near regulated exons.',\n",
              "  0),\n",
              " (2326,\n",
              "  12,\n",
              "  'How may CTCF mediate splicing?',\n",
              "  'Additionally, we find further evidence for the involvement of HP1α and AGO1 in chromatin-related splicing regulation.',\n",
              "  0),\n",
              " (2326,\n",
              "  13,\n",
              "  'How may CTCF mediate splicing?',\n",
              "  'CTCF also functions in RNA polymerase II regulation and in doing so can influence co-transcriptional splicing events.',\n",
              "  0),\n",
              " (2326,\n",
              "  14,\n",
              "  'How may CTCF mediate splicing?',\n",
              "  ' CTCF is ubiquitously expressed and plays diverse roles in gene regulation, imprinting, insulation, intra/interchromosomal interactions, nuclear compartmentalisation, and alternative splicing.',\n",
              "  0),\n",
              " (2326,\n",
              "  15,\n",
              "  'How may CTCF mediate splicing?',\n",
              "  'However, there was no correlation between the KTS+/KTS- splicing variants of WT1 and the methylation status of the CpGs of the CTCF binding site.',\n",
              "  1),\n",
              " (2326,\n",
              "  16,\n",
              "  'How may CTCF mediate splicing?',\n",
              "  'CTCF-promoted RNA polymerase II pausing links DNA methylation to splicing.',\n",
              "  0),\n",
              " (2326,\n",
              "  17,\n",
              "  'How may CTCF mediate splicing?',\n",
              "  'Here we provide the first evidence that a DNA-binding protein, CCCTC-binding factor (CTCF), can promote inclusion of weak upstream exons by mediating local RNA polymerase II pausing both in a mammalian model system for alternative splicing, CD45, and genome-wide',\n",
              "  1),\n",
              " (4220,\n",
              "  0,\n",
              "  'What is known about aberrant proteolytic activity in disease?',\n",
              "  \"Research into Alzheimer's disease pathology and treatment has often focused on presenilin proteins.\",\n",
              "  1),\n",
              " (4220,\n",
              "  1,\n",
              "  'What is known about aberrant proteolytic activity in disease?',\n",
              "  'These proteins provide the key catalytic activity of the γ-secretase complex in the cleavage of amyloid-β precursor protein and resultant amyloid tangle deposition.',\n",
              "  1),\n",
              " (4220,\n",
              "  2,\n",
              "  'What is known about aberrant proteolytic activity in disease?',\n",
              "  'Over the last 25 years, screening novel drugs to control this aberrant proteolytic activity has yet to identify effective treatments for the disease.',\n",
              "  1),\n",
              " (4220,\n",
              "  3,\n",
              "  'What is known about aberrant proteolytic activity in disease?',\n",
              "  'the authors report aberrant proteolytic activity in the salivary glands of non-obese diabetic mice and the generation of a unique organ-specific 17 kDa fragment of the chemokine and adhesion molecule fractalkine.',\n",
              "  1),\n",
              " (4220,\n",
              "  4,\n",
              "  'What is known about aberrant proteolytic activity in disease?',\n",
              "  'survey the different roles of proteases in epidermal homeostasis (from processing enzymes to signalling molecules) and explore the spectrum of rare and common human skin disorders where proteolytic pathways are dysregulated.',\n",
              "  0),\n",
              " (4220,\n",
              "  5,\n",
              "  'What is known about aberrant proteolytic activity in disease?',\n",
              "  'These data indicate that aberrant proteolytic activity in the NOD SMG results in increased fractalkine cleavage and generation of a unique fractalkine fragment.',\n",
              "  1),\n",
              " (4220,\n",
              "  6,\n",
              "  'What is known about aberrant proteolytic activity in disease?',\n",
              "  'This specific cleavage may contribute to autoimmunity.',\n",
              "  0),\n",
              " (3242,\n",
              "  0,\n",
              "  'What is the Triad of Alport Syndrome?',\n",
              "  'PURPOSE: Alport syndrome is a rare condition characterized by the clinical triad of nephritic syndrome, sensorineural deafness, and ophthalmological alterations.',\n",
              "  1),\n",
              " (3242,\n",
              "  1,\n",
              "  'What is the Triad of Alport Syndrome?',\n",
              "  'Classical AS is a progressive renal disease presenting with a triad of progressive hematuric nephritis and typical extra-renal complications, such as sensorineural hearing loss (SNHL) and variable ocular anomalies.',\n",
              "  0),\n",
              " (3242,\n",
              "  2,\n",
              "  'What is the Triad of Alport Syndrome?',\n",
              "  'In 1927, Arthur C. Alport first published his description of a triad of symptoms in a family with hereditary congenital haemorrhagic nephritis, deafness and ocular changes.',\n",
              "  0),\n",
              " (3242,\n",
              "  3,\n",
              "  'What is the Triad of Alport Syndrome?',\n",
              "  'BACKGROUND: Alport syndrome refers to the clinical triad of hereditary nephritis, sensorineural deafness, and ocular abnormalities.',\n",
              "  0),\n",
              " (3242,\n",
              "  4,\n",
              "  'What is the Triad of Alport Syndrome?',\n",
              "  'Alport syndrome is an oculo-renal syndrome characterized by a triad of clinical findings consisting of hemorrhagic nephritis, sensorineural hearing loss and characteristic ocular findings.',\n",
              "  0),\n",
              " (3242,\n",
              "  5,\n",
              "  'What is the Triad of Alport Syndrome?',\n",
              "  'BACKGROUND\\n\\nAlport syndrome is an inherited Type IV collagenopathy characterised by renal failure, hearing loss and ophthalmic manifestations such as lenticonus and dot-and-fleck retinopathy.',\n",
              "  0),\n",
              " (3242,\n",
              "  6,\n",
              "  'What is the Triad of Alport Syndrome?',\n",
              "  'PURPOSE:\\n\\nAlport syndrome is a rare condition characterized by the clinical triad of nephritic syndrome, sensorineural deafness, and ophthalmological alterations.',\n",
              "  0),\n",
              " (3242,\n",
              "  7,\n",
              "  'What is the Triad of Alport Syndrome?',\n",
              "  'BACKGROUND\\n\\nAlport syndrome refers to the clinical triad of hereditary nephritis, sensorineural deafness, and ocular abnormalities.',\n",
              "  0),\n",
              " (3242,\n",
              "  8,\n",
              "  'What is the Triad of Alport Syndrome?',\n",
              "  'BACKGROUND Alport syndrome refers to the clinical triad of hereditary nephritis, sensorineural deafness, and ocular abnormalities.',\n",
              "  0),\n",
              " (3242,\n",
              "  9,\n",
              "  'What is the Triad of Alport Syndrome?',\n",
              "  'Alport syndrome is an oculo-renal syndrome characterized by a triad of clinical findings consisting of hemorrhagic nephritis, sensorineural hearing loss and characteristic ocular findings.',\n",
              "  0),\n",
              " (3242,\n",
              "  10,\n",
              "  'What is the Triad of Alport Syndrome?',\n",
              "  'In 1927, Arthur C. Alport first published his description of a triad of symptoms in a family with hereditary congenital haemorrhagic nephritis, deafness and ocular changes.',\n",
              "  0),\n",
              " (3242,\n",
              "  11,\n",
              "  'What is the Triad of Alport Syndrome?',\n",
              "  \"Epstein 's triad which is a syndrome with the combination of macrothrombocytopenia , deafness and nephritis , is similar to Alport 's syndrome\",\n",
              "  0),\n",
              " (3242,\n",
              "  12,\n",
              "  'What is the Triad of Alport Syndrome?',\n",
              "  \"The Alport 's syndrome is a disease characterized by a symptomatic triad: nephropathy , hypoacusia and ocular alterations\",\n",
              "  0),\n",
              " (3242,\n",
              "  13,\n",
              "  'What is the Triad of Alport Syndrome?',\n",
              "  'Alport first published his description of a triad of symptoms in a family with hereditary congenital haemorrhagic nephritis , deafness and ocular changes',\n",
              "  0),\n",
              " (3242,\n",
              "  14,\n",
              "  'What is the Triad of Alport Syndrome?',\n",
              "  'Alport syndrome refers to the clinical triad of hereditary nephritis , sensorineural deafness , and ocular abnormalities',\n",
              "  1),\n",
              " (3242,\n",
              "  15,\n",
              "  'What is the Triad of Alport Syndrome?',\n",
              "  'BACKGROUND\\nAlport syndrome is an inherited Type IV collagenopathy characterised by renal failure, hearing loss and ophthalmic manifestations such as lenticonus and dot-and-fleck retinopathy.',\n",
              "  0),\n",
              " (3242,\n",
              "  16,\n",
              "  'What is the Triad of Alport Syndrome?',\n",
              "  'PURPOSE:\\nAlport syndrome is a rare condition characterized by the clinical triad of nephritic syndrome, sensorineural deafness, and ophthalmological alterations.',\n",
              "  0),\n",
              " (3242,\n",
              "  17,\n",
              "  'What is the Triad of Alport Syndrome?',\n",
              "  'Alport syndrome is an oculo-renal syndrome characterized by a triad of clinical findings consisting of hemorrhagic nephritis, sensorineural hearing loss and characteristic ocular findings.',\n",
              "  0),\n",
              " (3242,\n",
              "  18,\n",
              "  'What is the Triad of Alport Syndrome?',\n",
              "  'PURPOSE:: Alport syndrome is a rare condition characterized by the clinical triad of nephritic syndrome, sensorineural deafness, and ophthalmological alterations.',\n",
              "  1),\n",
              " (3242,\n",
              "  19,\n",
              "  'What is the Triad of Alport Syndrome?',\n",
              "  'BACKGROUND: Alport syndrome refers to the clinical triad of hereditary nephritis, sensorineural deafness, and ocular abnormalities.',\n",
              "  0),\n",
              " (3242,\n",
              "  20,\n",
              "  'What is the Triad of Alport Syndrome?',\n",
              "  'Alport syndrome refers to the clinical triad of hereditary nephritis, sensorineural deafness, and ocular abnormalities.',\n",
              "  1),\n",
              " (3242,\n",
              "  21,\n",
              "  'What is the Triad of Alport Syndrome?',\n",
              "  'Alport syndrome is a rare condition characterized by the clinical triad of nephritic syndrome, sensorineural deafness, and ophthalmological alterations.',\n",
              "  1),\n",
              " (1144,\n",
              "  0,\n",
              "  'Which histone modification is primarily linked to elongating transcription?',\n",
              "  'Similarly, H3K36 trimethylation, a mark associated with transcription elongation, was specifically increased at the HD locus in the striatum and not in the cerebellum.',\n",
              "  1),\n",
              " (1144,\n",
              "  1,\n",
              "  'Which histone modification is primarily linked to elongating transcription?',\n",
              "  'y expressing Myc protein fused with the estrogen receptor (Myc-ER) in fibroblasts, we observed that Myc, binding to the regulatory elements of Suz12, Ezh2, and Eed, induces the acetylation of histones H3 and H4 and the recruitment of elongating RNA polymerase II at their promoters',\n",
              "  0),\n",
              " (1144,\n",
              "  2,\n",
              "  'Which histone modification is primarily linked to elongating transcription?',\n",
              "  'A basal level of JIL-1 binding can be defined that correlates best with the methylation of histone H3 at lysine 36, a mark that is placed co-transcriptionally',\n",
              "  1),\n",
              " (1144,\n",
              "  3,\n",
              "  'Which histone modification is primarily linked to elongating transcription?',\n",
              "  'RNA polymerase II signals for deacetylation through the methylation of histone H3 lysine 36 (H3K36), which provides the recruitment signal for the Rpd3S histone deacetylase complex (HDAC)',\n",
              "  1),\n",
              " (1144,\n",
              "  4,\n",
              "  'Which histone modification is primarily linked to elongating transcription?',\n",
              "  'Recent studies reviewed here demonstrate that histone deacetylation on the body of a transcribed gene is regulated via Set2-mediated methylation of histone H3-K36',\n",
              "  1),\n",
              " (1144,\n",
              "  5,\n",
              "  'Which histone modification is primarily linked to elongating transcription?',\n",
              "  'H3K56 acetylation: a chromatin mark associated with the elongating RNA polymerase II',\n",
              "  0),\n",
              " (1144,\n",
              "  6,\n",
              "  'Which histone modification is primarily linked to elongating transcription?',\n",
              "  'Furthermore, Rtt109 and H3K56 acetylation appear to correlate with actively transcribed genes and associate with the elongating form of polymerase II in yeast',\n",
              "  0),\n",
              " (1144,\n",
              "  7,\n",
              "  'Which histone modification is primarily linked to elongating transcription?',\n",
              "  'In addition to coordinating the processing of the nascent transcript, elongating RNA polymerase II recruits histone methyltransferases to methylate lysines 4 and 36 of histone H3 in nucleosomes in the body of actively transcribed genes',\n",
              "  1),\n",
              " (1144,\n",
              "  8,\n",
              "  'Which histone modification is primarily linked to elongating transcription?',\n",
              "  \"Set1, the yeast histone H3-lysine 4 (H3-K4) methylase, is recruited by the Pol II elongation machinery to a highly localized domain at the 5' portion of active mRNA coding regions.\",\n",
              "  0),\n",
              " (2621,\n",
              "  0,\n",
              "  'List drugs that are included in the Vosevi polypill.',\n",
              "  'On July 18, 2017, the US Food and Drug Administration (FDA) approved sofosbuvir/velpatasvir/voxilaprevir (SOF/VEL/VOX) (Vosevi) fixed-dose combination (FDC), an interferon-free, complete regimen for adult patients with chronic hepatitis C virus (HCV) infection without cirrhosis or with compensated cirrhosis (Child-Pugh A) who have: genotype 1, 2, 3, 4, 5, or 6 infection and have previously been treated with an HCV regimen containing an NS5A inhibitor; and genotype 1a or 3 infection and have previously been treated with an HCV regimen containing sofosbuvir without an NS5A inhibitor.',\n",
              "  1),\n",
              " (3491,\n",
              "  0,\n",
              "  'Which drugs used in the treatment of Systemic Lupus Erythematosus are targeting granulocytes?',\n",
              "  'After treatment with any of the xenobiotic compounds, immunolabeling demonstrated the additional presence of granulocytes in foci, and, at electron microscope level neutrophils, eosinophils and their precursors were clearly recognizable.',\n",
              "  1),\n",
              " (3491,\n",
              "  1,\n",
              "  'Which drugs used in the treatment of Systemic Lupus Erythematosus are targeting granulocytes?',\n",
              "  'Cerebrospinal fluid (CSF) examination in patients with ATM in the course of SLE indicate usually pleocytosis with prevalence of granulocytes, increased protein levels, low glucose levels, significantly hindering differential diagnosis in the early stage of the disease.',\n",
              "  1),\n",
              " (3491,\n",
              "  2,\n",
              "  'Which drugs used in the treatment of Systemic Lupus Erythematosus are targeting granulocytes?',\n",
              "  'Granulocytes isolated from 17 patients with systemic lupus erythematosus (SLE) had a significantly increased (P less than 0.001) mean initial rate of phagocytosis (0.91 min-u).',\n",
              "  1),\n",
              " (3491,\n",
              "  3,\n",
              "  'Which drugs used in the treatment of Systemic Lupus Erythematosus are targeting granulocytes?',\n",
              "  'Both syndromes were refractory to conventional treatment but responded to rituximab and cyclophosphamide.',\n",
              "  1),\n",
              " (3491,\n",
              "  4,\n",
              "  'Which drugs used in the treatment of Systemic Lupus Erythematosus are targeting granulocytes?',\n",
              "  'Although some Fc-independent loss of CD22 is expected from internalization by epratuzumab, the concurrent and prominent reduction of CD19, CD21, and CD79b is Fc dependent and results from their transfer from epratuzumab-opsonized B cells to FcγR-expressing monocytes, natural killer cells, and granulocytes via trogocytosis.',\n",
              "  1),\n",
              " (3491,\n",
              "  5,\n",
              "  'Which drugs used in the treatment of Systemic Lupus Erythematosus are targeting granulocytes?',\n",
              "  ' ANCA also occurs in 30%-40% of patients with eosinophilic granulomatosis with polyangiitis (EGPA) and anti-GBM disease, but is uncommon in other forms of vasculitis.',\n",
              "  0),\n",
              " (3330,\n",
              "  0,\n",
              "  'List angiotensin-converting-enzyme inhibitors.',\n",
              "  'angiotensin-converting enzyme inhibitor captopril',\n",
              "  1),\n",
              " (3330,\n",
              "  1,\n",
              "  'List angiotensin-converting-enzyme inhibitors.',\n",
              "  'enalapril, lisinopril, or ramipril',\n",
              "  1),\n",
              " (4074,\n",
              "  0,\n",
              "  'What is the function of HP1a in the nucleus?',\n",
              "  'Drosophila heterochromatin-associated protein 1 (HP1) is an abundant component of heterochromatin, a highly condensed compartment of the nucleus that comprises a major fraction of complex genomes.',\n",
              "  1),\n",
              " (4074,\n",
              "  1,\n",
              "  'What is the function of HP1a in the nucleus?',\n",
              "  'Both the HP1a hinge and chromo shadow domain independently target heterochromatin, while the HP1c chromo shadow domain is implicated solely in euchromatin localization.',\n",
              "  0),\n",
              " (4074,\n",
              "  2,\n",
              "  'What is the function of HP1a in the nucleus?',\n",
              "  'Recently, Drosophila melanogaster Piwi was recently reported to associate with chromatin and to interact directly with the Heterochromatin Protein 1 (HP1a)',\n",
              "  0),\n",
              " (4074,\n",
              "  3,\n",
              "  'What is the function of HP1a in the nucleus?',\n",
              "  'These results suggest that silkworm Piwi proteins function as a chromatin regulator in collaboration with HP1a and HP1b.',\n",
              "  0),\n",
              " (4074,\n",
              "  4,\n",
              "  'What is the function of HP1a in the nucleus?',\n",
              "  'Heterochromatin protein 1 (HP1) was first described in Drosophila melanogaster as a heterochromatin associated protein required for epigenetic gene silencing.',\n",
              "  1),\n",
              " (4074,\n",
              "  5,\n",
              "  'What is the function of HP1a in the nucleus?',\n",
              "  'Here we characterized the genome-wide effect of nuclear Piwi elimination on the presence of the heterochromatic H3K9me3 mark and HP1a, as well as on the transcription-associated mark H3K4me2.',\n",
              "  0),\n",
              " (4074,\n",
              "  6,\n",
              "  'What is the function of HP1a in the nucleus?',\n",
              "  'Our results demonstrate that a significant increase in the H3K4me2 level upon nuclear Piwi loss is not accompanied by the alterations in H3K9me3 and HP1a levels for several germline-expressed transposons, suggesting that in this case Piwi prevents transcription by a mechanism distinct from H3K9 methylation.',\n",
              "  0),\n",
              " (4074,\n",
              "  7,\n",
              "  'What is the function of HP1a in the nucleus?',\n",
              "  'Heterochromatin is highly enriched for repetitive sequences, and is defined epigenetically by methylation of histone H3 at lysine 9 and recruitment of its binding partner heterochromatin protein 1 (HP1)',\n",
              "  0),\n",
              " (4074,\n",
              "  8,\n",
              "  'What is the function of HP1a in the nucleus?',\n",
              "  'The large fraction of heterochromatin in Drosophila neurons is bound by both B-type lamin and HP1a.',\n",
              "  0),\n",
              " (4074,\n",
              "  9,\n",
              "  'What is the function of HP1a in the nucleus?',\n",
              "  'However, in Kc167 cell culture, the only Drosophilla cell type where LADs have previously been mapped, they are neither H3K9me2-enriched nor overlapped with the domains of heterochromatin protein 1a (HP1a)',\n",
              "  0),\n",
              " (4074,\n",
              "  10,\n",
              "  'What is the function of HP1a in the nucleus?',\n",
              "  'Strikingly, contrary to Kc167 cells of embryonic origin, in neurons and, to a lesser extent, in glia and the fat body, HP1a domains appear to overlap strongly with LADs in both the chromosome arms and pericentromeric regions.',\n",
              "  0),\n",
              " (4074,\n",
              "  11,\n",
              "  'What is the function of HP1a in the nucleus?',\n",
              "  'Heterochromatin protein 1a (HP1a) has been proposed to function downstream of Piwi-piRNA complex in Drosophila.',\n",
              "  0),\n",
              " (4074,\n",
              "  12,\n",
              "  'What is the function of HP1a in the nucleus?',\n",
              "  'The chromatin of all transgene-associated piRNA clusters contain high levels of trimethylated lysine 9 of histone H3 (H3K9me3) and HP1a',\n",
              "  0),\n",
              " (4074,\n",
              "  13,\n",
              "  'What is the function of HP1a in the nucleus?',\n",
              "  'Su(var) 3-9 protein was found to be localized in the nucleus in Sf9 cells, and interact with histone H3, and the heterochromatin protein 1a (HP1a) and HP1b.',\n",
              "  0),\n",
              " (4074,\n",
              "  14,\n",
              "  'What is the function of HP1a in the nucleus?',\n",
              "  'An important component of the telomeres is the heterochromatin protein 1a (HP1a).',\n",
              "  0),\n",
              " (4074,\n",
              "  15,\n",
              "  'What is the function of HP1a in the nucleus?',\n",
              "  'Heterochromatin protein 1a (HP1a) is a chromatin-associated protein important for the formation and maintenance of heterochromatin.',\n",
              "  0),\n",
              " (4074,\n",
              "  16,\n",
              "  'What is the function of HP1a in the nucleus?',\n",
              "  'Drosophila heterochromatin-associated protein 1 (HP1) is an abundant component of heterochromatin, a highly condensed compartment of the nucleus that comprises a major fraction of complex genomes.',\n",
              "  1),\n",
              " (4074, 17, 'What is the function of HP1a in the nucleus?', 'S', 0),\n",
              " (4074,\n",
              "  18,\n",
              "  'What is the function of HP1a in the nucleus?',\n",
              "  'ophila HP1a protein undergoes liquid-liquid demixing in vitro, and nucleates into foci that display liquid properties during the first stages of heterochromatin domain formation in early Drosophila embryos.',\n",
              "  0),\n",
              " (4074,\n",
              "  19,\n",
              "  'What is the function of HP1a in the nucleus?',\n",
              "  'Furthermore, in',\n",
              "  0),\n",
              " (4074,\n",
              "  20,\n",
              "  'What is the function of HP1a in the nucleus?',\n",
              "  'Expression of transposable elements in the germline is controlled by Piwi-interacting (pi) RNAs produced by genomic loci termed piRNA clusters and associated with Rhino, a heterochromatin protein 1 (HP1) homolog.',\n",
              "  0),\n",
              " (4074, 21, 'What is the function of HP1a in the nucleus?', 'Pre', 0),\n",
              " (4074,\n",
              "  22,\n",
              "  'What is the function of HP1a in the nucleus?',\n",
              "  'Heterochromatin protein 1 (HP1) is an epigenetic regulator of chromatin structure and genome function in eukaryotes.',\n",
              "  1),\n",
              " (4074,\n",
              "  23,\n",
              "  'What is the function of HP1a in the nucleus?',\n",
              "  '9 protein was found to be localized in the nucleus in Sf9 cells, and interact with histone H3, and the heterochromatin protein 1a (HP1a) and HP1b.',\n",
              "  0),\n",
              " (4074,\n",
              "  24,\n",
              "  'What is the function of HP1a in the nucleus?',\n",
              "  'A dose-depend',\n",
              "  0),\n",
              " (4074,\n",
              "  25,\n",
              "  'What is the function of HP1a in the nucleus?',\n",
              "  'Heterochromatin protein 1a (HP1a) is a highly conserved and versatile epigenetic factor that can both silence and activate transcription.',\n",
              "  0),\n",
              " (4074,\n",
              "  26,\n",
              "  'What is the function of HP1a in the nucleus?',\n",
              "  'Here, we show that Eyg forms a protein complex with heterochromatin protein 1a (HP1a).',\n",
              "  0),\n",
              " (4074,\n",
              "  27,\n",
              "  'What is the function of HP1a in the nucleus?',\n",
              "  'The recruitment of HP1a prevents transcription by favoring a closed, heterochromatin-like structure.',\n",
              "  0),\n",
              " (4074,\n",
              "  28,\n",
              "  'What is the function of HP1a in the nucleus?',\n",
              "  'Using position effect variegation (PEV) experiments, we demonstrated that eyg has a dose-dependent effect on heterochromatin gene silencing and identified a genetic interaction with HP1a in this process.',\n",
              "  0),\n",
              " (4074,\n",
              "  29,\n",
              "  'What is the function of HP1a in the nucleus?',\n",
              "  'HP1a is an evolutionarily conserved heterochromatin-associated protein, which plays an essential role in heterochromatin-mediated gene silencing.',\n",
              "  0),\n",
              " (4074,\n",
              "  30,\n",
              "  'What is the function of HP1a in the nucleus?',\n",
              "  'Heterochromatin Protein 1 (HP1a) is a well-known conserved protein involved in heterochromatin formation and gene silencing in different species including humans.',\n",
              "  0),\n",
              " (4074,\n",
              "  31,\n",
              "  'What is the function of HP1a in the nucleus?',\n",
              "  'Heterochromatin protein 1 (HP1a in Drosophila) is a conserved eukaryotic chromosomal protein that is prominently associated with pericentric heterochromatin and mediates the concomitant gene silencing.',\n",
              "  0),\n",
              " (4074,\n",
              "  32,\n",
              "  'What is the function of HP1a in the nucleus?',\n",
              "  'Heterochromatin protein 1a functions for piRNA biogenesis predominantly from pericentric and telomeric regions in Drosophila.',\n",
              "  0),\n",
              " (4074,\n",
              "  33,\n",
              "  'What is the function of HP1a in the nucleus?',\n",
              "  'Heterochromatin protein 1 (HP1) is an epigenetic regulator of chromatin structure and genome function in eukaryotes.',\n",
              "  1),\n",
              " (3252,\n",
              "  0,\n",
              "  'When was Afrezza approved by the FDA?',\n",
              "  'In contrary, MannKind Corporation started developing its ultra-rapid-acting insulin Afrezza in a bold bid, probably by managing the issues in which Exubera was not successful.',\n",
              "  1),\n",
              " (3252,\n",
              "  1,\n",
              "  'When was Afrezza approved by the FDA?',\n",
              "  'Afrezza has been marketed since February, 2015 by Sanofi after getting FDA approval in June 2014.',\n",
              "  1),\n",
              " (4134,\n",
              "  0,\n",
              "  'Which methods exist for efficient calculation of Elementary flux modes (EFMs) in genome-scale metabolic networks (GSMNs)?',\n",
              "  'Elementary flux modes (EFMs) analysis constitutes a fundamental tool in systems biology.',\n",
              "  0),\n",
              " (4134,\n",
              "  1,\n",
              "  'Which methods exist for efficient calculation of Elementary flux modes (EFMs) in genome-scale metabolic networks (GSMNs)?',\n",
              "  'However, the efficient calculation of EFMs in genome-scale metabolic networks (GSMNs) is still a challenge.',\n",
              "  1),\n",
              " (4134,\n",
              "  2,\n",
              "  'Which methods exist for efficient calculation of Elementary flux modes (EFMs) in genome-scale metabolic networks (GSMNs)?',\n",
              "  'We present a novel algorithm that uses a linear programming-based tree search and efficiently enumerates a subset of EFMs in GSMNs.RESULTS: Our approach is compared with the EFMEvolver approach, demonstrating a significant improvement in computation time.',\n",
              "  1),\n",
              " (4134,\n",
              "  3,\n",
              "  'Which methods exist for efficient calculation of Elementary flux modes (EFMs) in genome-scale metabolic networks (GSMNs)?',\n",
              "  'We also validate the usefulness of our new approach by studying the acetate overflow metabolism in the Escherichia coli bacteria.',\n",
              "  0),\n",
              " (4134,\n",
              "  4,\n",
              "  'Which methods exist for efficient calculation of Elementary flux modes (EFMs) in genome-scale metabolic networks (GSMNs)?',\n",
              "  'To do so, we computed 1 million EFMs for each energetic amino acid and then analysed the relevance of each energetic amino acid based on gene/protein expression data and the obtained EFMs.',\n",
              "  0),\n",
              " (4134,\n",
              "  5,\n",
              "  'Which methods exist for efficient calculation of Elementary flux modes (EFMs) in genome-scale metabolic networks (GSMNs)?',\n",
              "  'We found good agreement between previous experiments and the conclusions reached using EFMs.',\n",
              "  0),\n",
              " (4134,\n",
              "  6,\n",
              "  'Which methods exist for efficient calculation of Elementary flux modes (EFMs) in genome-scale metabolic networks (GSMNs)?',\n",
              "  'Finally, we also analysed the performance of our approach when applied to large GSMNs.AVAILABILITY AND IMPLEMENTATION: The stand-alone software TreeEFM is implemented in C++ and interacts with the open-source linear solver COIN-OR Linear program Solver (CLP).',\n",
              "  1),\n",
              " (4134,\n",
              "  7,\n",
              "  'Which methods exist for efficient calculation of Elementary flux modes (EFMs) in genome-scale metabolic networks (GSMNs)?',\n",
              "  'Elementary flux modes (EFMs) are a key tool for analyzing genome-scale metabolic networks, and several methods have been proposed to compute them.',\n",
              "  1),\n",
              " (4134,\n",
              "  8,\n",
              "  'Which methods exist for efficient calculation of Elementary flux modes (EFMs) in genome-scale metabolic networks (GSMNs)?',\n",
              "  'Among them, those based on solving linear programming (LP) problems are known to be very efficient if the main interest lies in computing large enough sets of EFMs.RESULTS: Here, we propose a new method called EFM-Ta that boosts the efficiency rate by analyzing the information provided by the LP solver.',\n",
              "  1),\n",
              " (4134,\n",
              "  9,\n",
              "  'Which methods exist for efficient calculation of Elementary flux modes (EFMs) in genome-scale metabolic networks (GSMNs)?',\n",
              "  'We base our method on a further study of the final tableau of the simplex method.',\n",
              "  0),\n",
              " (4134,\n",
              "  10,\n",
              "  'Which methods exist for efficient calculation of Elementary flux modes (EFMs) in genome-scale metabolic networks (GSMNs)?',\n",
              "  'By performing additional elementary steps and avoiding trivial solutions consisting of two cycles, we obtain many more EFMs for each LP problem posed, improving the efficiency rate of previously proposed methods by more than one order of magnitude.',\n",
              "  0),\n",
              " (1693,\n",
              "  0,\n",
              "  'List components of the CRSP/Med complex.',\n",
              "  ' Here we describe a new human factor, CRSP, that is required together with the TAF(II)s for transcriptional activation by Sp1.',\n",
              "  1),\n",
              " (1693,\n",
              "  1,\n",
              "  'List components of the CRSP/Med complex.',\n",
              "  'Purification of CRSP identifies a complex of approximate relative molecular mass 700,000 (M(r) approximately 700K) that contains nine subunits with M(r) values ranging from 33K to 200K.',\n",
              "  0),\n",
              " (1693,\n",
              "  2,\n",
              "  'List components of the CRSP/Med complex.',\n",
              "  'Cloning of genes encoding CRSP subunits reveals that CRSP33 is a homologue of the yeast mediator subunit Med7, whereas CRSP150 contains a domain conserved in yeast mediator subunit Rgr1.',\n",
              "  1),\n",
              " (1693,\n",
              "  3,\n",
              "  'List components of the CRSP/Med complex.',\n",
              "  'CRSP p200 is identical to the nuclear hormone-receptor co-activator subunit TRIP2/PBP.',\n",
              "  1),\n",
              " (1693,\n",
              "  4,\n",
              "  'List components of the CRSP/Med complex.',\n",
              "  'CRSPs 34, 77 and 130 are new proteins, but the amino terminus of CRSP70 is homologous to elongation factor TFIIS.',\n",
              "  1),\n",
              " (1693,\n",
              "  5,\n",
              "  'List components of the CRSP/Med complex.',\n",
              "  'CRSP contains unique subunits as well as polypeptides that are shared with other cofactor complexes.',\n",
              "  0),\n",
              " (1693,\n",
              "  6,\n",
              "  'List components of the CRSP/Med complex.',\n",
              "  'The multi-subunit, human CRSP coactivator-also known as Mediator (Med)-regulates transcription by mediating signals between enhancer-bound factors (activators) and the core transcriptional machinery.',\n",
              "  1),\n",
              " (1693,\n",
              "  7,\n",
              "  'List components of the CRSP/Med complex.',\n",
              "  'Interestingly, different activators are known to bind distinct subunits within the CRSP/Med complex.',\n",
              "  0),\n",
              " (1693,\n",
              "  8,\n",
              "  'List components of the CRSP/Med complex.',\n",
              "  'We have isolated a stable, endogenous CRSP/Med complex (CRSP/Med2) that specifically lacks both the Med220 and the Med70 subunits.',\n",
              "  0),\n",
              " (3133,\n",
              "  0,\n",
              "  'Which malignancies is Keytruda approved for before 2017?',\n",
              "  'FDA previously approved Keytruda for treatment of several types of malignancies, such as metastatic melanoma, metastatic non-small-cell lung cancer, recurrent or metastatic head and neck cancer, refractory Hodgkin lymphoma, and urothelial carcinoma, all of which carry positive programmed death-1/programmed death-ligand 1 biomarkers.',\n",
              "  1),\n",
              " (3789,\n",
              "  0,\n",
              "  'List enzymes that removes histone modifications.',\n",
              "  ', Lysine Specific Demethylases (LSD) removes methylated histone H3 lysine 4 (H3K4) and H3 lysine 9 (H3K9)',\n",
              "  1),\n",
              " (3789,\n",
              "  1,\n",
              "  'List enzymes that removes histone modifications.',\n",
              "  'A family of enzymes, namely histone deacetylases (HDACs), removes these PTMs.',\n",
              "  1),\n",
              " (3789,\n",
              "  2,\n",
              "  'List enzymes that removes histone modifications.',\n",
              "  ' Histone deacetylases (HDACs), a specific epigenetic group of enzymes, dynamically and reversibly removes acetyl groups from histone tails projecting from the nucleosome.',\n",
              "  1),\n",
              " (101,\n",
              "  0,\n",
              "  'What is the principle of the PAR-CLIP methodology?',\n",
              "  'We developed a cell-based crosslinking approach to determine at high resolution and transcriptome-wide the binding sites of cellular RBPs and miRNPs.',\n",
              "  0),\n",
              " (101,\n",
              "  1,\n",
              "  'What is the principle of the PAR-CLIP methodology?',\n",
              "  'To gain insight into the complexity of snoRNA processing and the functional relevance of snoRNA-derived small RNAs, we sequence long and short RNAs, small RNAs that co-precipitate with the Argonaute 2 protein and RNA fragments obtained in photoreactive nucleotide-enhanced crosslinking and immunoprecipitation (PAR-CLIP) of core snoRNA-associated proteins.',\n",
              "  0),\n",
              " (101,\n",
              "  2,\n",
              "  'What is the principle of the PAR-CLIP methodology?',\n",
              "  'A large amount of miRNA-target interactions (MTIs) have been identified by the crosslinking and immunoprecipitation (CLIP) and the photoactivatable-ribonucleoside-enhanced CLIP (PAR-CLIP) along with the next-generation sequencing (NGS)',\n",
              "  0),\n",
              " (101,\n",
              "  3,\n",
              "  'What is the principle of the PAR-CLIP methodology?',\n",
              "  'PAR-CLIP shows high efficiency of RNA co-immunoprecipitation, but it also lead to T to C conversion in miRNA-RNA-protein crosslinking regions',\n",
              "  1),\n",
              " (101,\n",
              "  4,\n",
              "  'What is the principle of the PAR-CLIP methodology?',\n",
              "  'A recent method, PAR-CLIP, uses photoreactive nucleosides to crosslink RBPs to target RNAs in cells prior to immunoprecipitation.',\n",
              "  1),\n",
              " (101,\n",
              "  5,\n",
              "  'What is the principle of the PAR-CLIP methodology?',\n",
              "  'In this article, we review crosslinking and immunoprecipitation (CLIP) methods adapted for large-scale identification of target RNA-binding sites and the respective RNA recognition elements.',\n",
              "  0),\n",
              " (101,\n",
              "  6,\n",
              "  'What is the principle of the PAR-CLIP methodology?',\n",
              "  'CLIP methods have the potential to detect hundreds of thousands of binding sites in single experiments although the separation of signal from noise can be challenging.',\n",
              "  0),\n",
              " (101,\n",
              "  7,\n",
              "  'What is the principle of the PAR-CLIP methodology?',\n",
              "  'We focus on photoactivatable ribonucleoside-enhanced CLIP, which relies on the intracellular incorporation of photoactivatable ribonucleoside analogs into nascent transcripts, and yields characteristic sequence changes upon crosslinking that facilitate the separation of signal from noise.',\n",
              "  0),\n",
              " (101,\n",
              "  8,\n",
              "  'What is the principle of the PAR-CLIP methodology?',\n",
              "  'Photo-Activatable Ribonucleoside-enhanced CrossLinking and ImmunoPrecipitation (PAR-CLIP) method was recently developed for global identification of RNAs interacting with proteins.',\n",
              "  0),\n",
              " (101,\n",
              "  9,\n",
              "  'What is the principle of the PAR-CLIP methodology?',\n",
              "  'The strength of this versatile method results from induction of specific T to C transitions at sites of interaction.',\n",
              "  0),\n",
              " (101,\n",
              "  10,\n",
              "  'What is the principle of the PAR-CLIP methodology?',\n",
              "  'PAR-CLIP reveals a collection of RNAs bound to a protein whereas SILAC-based RNA pull-downs identify a group of proteins bound to an RNA.',\n",
              "  0),\n",
              " (101,\n",
              "  11,\n",
              "  'What is the principle of the PAR-CLIP methodology?',\n",
              "  'Here we present a step-by-step protocol and guidelines for the computational analysis for the large-scale identification of miRNA target sites in cultured cells by photoactivatable ribonucleoside enhanced crosslinking and immunoprecipitation (PAR-CLIP) of AGO proteins.',\n",
              "  0),\n",
              " (101,\n",
              "  12,\n",
              "  'What is the principle of the PAR-CLIP methodology?',\n",
              "  'In particular, PAR-CLIP utilizes a photoactivatable nucleoside for more efficient crosslinking.',\n",
              "  1),\n",
              " (101,\n",
              "  13,\n",
              "  'What is the principle of the PAR-CLIP methodology?',\n",
              "  'rosslinking and immunoprecipitation (CLIP) protocols have made it possible to identify transcriptome-wide RNA-protein interaction sites',\n",
              "  0),\n",
              " (101,\n",
              "  14,\n",
              "  'What is the principle of the PAR-CLIP methodology?',\n",
              "  'In this issue of Molecular Cell, Mukherjee et al.',\n",
              "  0),\n",
              " (101,\n",
              "  15,\n",
              "  'What is the principle of the PAR-CLIP methodology?',\n",
              "  '(2011) and Lebedeva et al.',\n",
              "  0),\n",
              " (101,\n",
              "  16,\n",
              "  'What is the principle of the PAR-CLIP methodology?',\n",
              "  \"(2011) identify transcriptome-wide HuR-RNA interactions using PAR-CLIP, unveiling HuR's nuclear role in pre-mRNA processing.\",\n",
              "  0),\n",
              " (101,\n",
              "  17,\n",
              "  'What is the principle of the PAR-CLIP methodology?',\n",
              "  'ross-linking and immunoprecipitation (CLIP) is increasingly used to map transcriptome-wide binding sites of RNA-binding proteins.',\n",
              "  0),\n",
              " (101,\n",
              "  18,\n",
              "  'What is the principle of the PAR-CLIP methodology?',\n",
              "  'We developed a method for CLIP data analysis, and applied it to compare CLIP with photoactivatable ribonucleoside-enhanced CLIP (PAR-CLIP) and to uncover how differences in cross-linking and ribonuclease digestion affect the identified sites.',\n",
              "  0),\n",
              " (101,\n",
              "  19,\n",
              "  'What is the principle of the PAR-CLIP methodology?',\n",
              "  'In order to increase the specificity and positional resolution, a strategy referred to as CLIP (UV cross-linking and immunoprecipitation) was introduced.',\n",
              "  0),\n",
              " (101,\n",
              "  20,\n",
              "  'What is the principle of the PAR-CLIP methodology?',\n",
              "  'Recently, PAR-CLIP was introduced that uses photoreactive ribonucleoside analogs for cross-linking',\n",
              "  0),\n",
              " (101,\n",
              "  21,\n",
              "  'What is the principle of the PAR-CLIP methodology?',\n",
              "  'AR-CliP--a method to identify transcriptome-wide the binding sites of RNA binding proteins',\n",
              "  0),\n",
              " (101,\n",
              "  22,\n",
              "  'What is the principle of the PAR-CLIP methodology?',\n",
              "  'We developed a powerful cell-based crosslinking approach to determine at high resolution and transcriptome-wide the binding sites of cellular RBPs and miRNPs that we term PAR-CliP (Photoactivatable-Ribonucleoside-Enhanced Crosslinking and Immunoprecipitation)',\n",
              "  1),\n",
              " (101,\n",
              "  23,\n",
              "  'What is the principle of the PAR-CLIP methodology?',\n",
              "  'The method relies on the incorporation of photoreactive ribonucleoside analogs, such as 4-thiouridine (4-SU) and 6-thioguanosine (6-SG) into nascent RNA transcripts by living cells.',\n",
              "  0),\n",
              " (101,\n",
              "  24,\n",
              "  'What is the principle of the PAR-CLIP methodology?',\n",
              "  'One characteristic feature of cDNA libraries prepared by PAR-CliP is that the precise position of crosslinking can be identified by mutations residing in the sequenced cDNA.',\n",
              "  1),\n",
              " (3333,\n",
              "  0,\n",
              "  'What is known about autosomal dominant Alzheimer’s disease?',\n",
              "  'The first autosomal dominant mutation in the amyloid precursor protein (APP) gene was described in 1991.',\n",
              "  1),\n",
              " (3333,\n",
              "  1,\n",
              "  'What is known about autosomal dominant Alzheimer’s disease?',\n",
              "  'Later, AD was also associated with mutated early-onset (presenilin 1/2, PSEN1/2 and APP) and late-onset (apolipoprotein E, ApoE) genes.',\n",
              "  1),\n",
              " (3333,\n",
              "  2,\n",
              "  'What is known about autosomal dominant Alzheimer’s disease?',\n",
              "  'Genome-wide association and linkage analysis studies with identified multiple genomic areas have implications for the treatment of AD.',\n",
              "  1),\n",
              " (3333,\n",
              "  3,\n",
              "  'What is known about autosomal dominant Alzheimer’s disease?',\n",
              "  'we apply the proposed methodology to the data of several biomarkers collected by the Dominantly Inherited Alzheimer Network, an international family-clustered registry to study autosomal dominant Alzheimer disease which is a rare form of Alzheimer disease caused by mutations in any of the three genes including the amyloid precursor protein, presenilin 1 and presenilin 2.',\n",
              "  1),\n",
              " (3333,\n",
              "  4,\n",
              "  'What is known about autosomal dominant Alzheimer’s disease?',\n",
              "  'We estimate the accuracy of several cerebrospinal fluid and neuroimaging biomarkers in differentiating three diagnostic and genetic populations: normal non-mutation carriers, asymptomatic mutation carriers, and symptomatic mutation carriers.',\n",
              "  1),\n",
              " (1184,\n",
              "  0,\n",
              "  'List the three most prevalent pathogenic species of Borrelia in Europe.',\n",
              "  ' The Borrelia species causing Lyme borreliosis in Europe express different OspA serotypes on their surface, B. burgdorferi (serotype 1), B. afzelii (serotype 2), B. garinii (serotypes, 3, 5 and 6) and B. bavariensis (serotype 4), while only B. burgdorferi is present in the US.',\n",
              "  1),\n",
              " (1184,\n",
              "  1,\n",
              "  'List the three most prevalent pathogenic species of Borrelia in Europe.',\n",
              "  'Borrelia afzelii is the predominant Borrelia species causing Lyme borreliosis in Europe.',\n",
              "  0),\n",
              " (1184,\n",
              "  2,\n",
              "  'List the three most prevalent pathogenic species of Borrelia in Europe.',\n",
              "  'Moreover we detected Borrelia spielmanii, Borrelia miyamotoi, Babesia divergens, and Babesia venatorum for the first time in Danish ticks.',\n",
              "  0),\n",
              " (1184,\n",
              "  3,\n",
              "  'List the three most prevalent pathogenic species of Borrelia in Europe.',\n",
              "  'We successfully determined the prevalence of expected (Borrelia burgdorferi sensu lato',\n",
              "  0),\n",
              " (1184,\n",
              "  4,\n",
              "  'List the three most prevalent pathogenic species of Borrelia in Europe.',\n",
              "  'Borrelia miyamotoi is a newly described emerging pathogen transmitted to people by Ixodes species ticks and found in temperate regions of North America, Europe, and Asia.',\n",
              "  0),\n",
              " (1184,\n",
              "  5,\n",
              "  'List the three most prevalent pathogenic species of Borrelia in Europe.',\n",
              "  'There is limited understanding of large scale entomological risk patterns of B. miyamotoi and of Borreila burgdorferi sensu stricto (ss), the agent of Lyme disease, in western North America.',\n",
              "  0),\n",
              " (1184,\n",
              "  6,\n",
              "  'List the three most prevalent pathogenic species of Borrelia in Europe.',\n",
              "  'In France as elsewhere in Europe the most prevalent TBD in humans is Lyme borreliosis, caused by different bacterial species belonging to Borrelia burgdorferi sensu lato complex',\n",
              "  0),\n",
              " (1184,\n",
              "  7,\n",
              "  'List the three most prevalent pathogenic species of Borrelia in Europe.',\n",
              "  'B. miyamotoi belonging to the relapsing fever group and transmitted by the same Ixodes species has been involved in human disease in Russia, the USA and the Netherlands.',\n",
              "  0),\n",
              " (1184,\n",
              "  8,\n",
              "  'List the three most prevalent pathogenic species of Borrelia in Europe.',\n",
              "  'Laboratory diagnosis of Lyme disease is based on the serological detection of antibodies against the etiologic agent Borrelia burgdorferi.',\n",
              "  0),\n",
              " (1184,\n",
              "  9,\n",
              "  'List the three most prevalent pathogenic species of Borrelia in Europe.',\n",
              "  'B. burgdorferi sl complex, currently comprising at least 19 genospecies, includes the main pathogenic species responsible for human disease in Europe: B. burgdorferi sensu stricto (ss), B. afzelii, and B. garinii.',\n",
              "  1),\n",
              " (1184,\n",
              "  10,\n",
              "  'List the three most prevalent pathogenic species of Borrelia in Europe.',\n",
              "  'The European red squirrel (Sciurus vulgaris) has long been suspected to be a reservoir host of the agents of Lyme borreliosis, in particular B. burgdorferi sensu stricto (s.s.).',\n",
              "  0),\n",
              " (1184,\n",
              "  11,\n",
              "  'List the three most prevalent pathogenic species of Borrelia in Europe.',\n",
              "  'n adults (n = 227), the prevalence of infection with B. burgdorferi s.l.',\n",
              "  0),\n",
              " (1184,\n",
              "  12,\n",
              "  'List the three most prevalent pathogenic species of Borrelia in Europe.',\n",
              "  'was 27.3%, with 18.9% B. burgdorferi s.s., 11.9% B. afzelii, and 3.5% B. garinii.',\n",
              "  1),\n",
              " (1184,\n",
              "  13,\n",
              "  'List the three most prevalent pathogenic species of Borrelia in Europe.',\n",
              "  ': The overall estimated prevalence of B. burgdorferi s.l was 9.7%',\n",
              "  0),\n",
              " (1184,\n",
              "  14,\n",
              "  'List the three most prevalent pathogenic species of Borrelia in Europe.',\n",
              "  ' The most prevalent genospecies was B. afzelii which was detected in 53.5% of Borrelia-positive ticks, followed by B. garinii and B. valaisiana with 26.2% and 5.5%, respectively.',\n",
              "  1),\n",
              " (1184,\n",
              "  15,\n",
              "  'List the three most prevalent pathogenic species of Borrelia in Europe.',\n",
              "  'Also, B. bavariensis and B. burgdorferi s.s. DNA in single I. ricinus ti',\n",
              "  0),\n",
              " (1184,\n",
              "  16,\n",
              "  'List the three most prevalent pathogenic species of Borrelia in Europe.',\n",
              "  ' Among all the analysed antigens those of B. burgdorferi s.s. were the most frequent cause of immunological reaction, followed by antigens of B. afzelii and B. garinii.',\n",
              "  1),\n",
              " (1184,\n",
              "  17,\n",
              "  'List the three most prevalent pathogenic species of Borrelia in Europe.',\n",
              "  'Reaction to antigens of B. spielmanii was rarely detected.',\n",
              "  0),\n",
              " (1184,\n",
              "  18,\n",
              "  'List the three most prevalent pathogenic species of Borrelia in Europe.',\n",
              "  'Overall, 15.8% ticks were infected with B. burgdorferi s. I. Borrelia afzelii (43.1%) was the predominant genospecies, followed by Borrelia valaisiana (14.7%), Borrelia garinii (13.7%) and Borrelia burgdorferi sensu stricto (6.3%).',\n",
              "  0),\n",
              " (1184,\n",
              "  19,\n",
              "  'List the three most prevalent pathogenic species of Borrelia in Europe.',\n",
              "  'In cases where the genospecies could be identified, B. garinii was most frequently found (8x), followed with B. burgdorferi s.s. (4×) and B. afzelii (3×).',\n",
              "  0),\n",
              " (1184,\n",
              "  20,\n",
              "  'List the three most prevalent pathogenic species of Borrelia in Europe.',\n",
              "  'In Europe, Ixodes ricinus is the vector of many pathogens of medical and veterinary relevance, among them Borrelia burgdorferi sensu lato and tick-borne encephalitis virus, which have been the subject of numerous investigations',\n",
              "  0),\n",
              " (1184,\n",
              "  21,\n",
              "  'List the three most prevalent pathogenic species of Borrelia in Europe.',\n",
              "  'The minimum prevalence of Borrelia burgdorferi sensu lato was 2.5%.',\n",
              "  0),\n",
              " (1184,\n",
              "  22,\n",
              "  'List the three most prevalent pathogenic species of Borrelia in Europe.',\n",
              "  'Borr.',\n",
              "  0),\n",
              " (1184,\n",
              "  23,\n",
              "  'List the three most prevalent pathogenic species of Borrelia in Europe.',\n",
              "  'afzelii, Borr.',\n",
              "  0),\n",
              " (1184,\n",
              "  24,\n",
              "  'List the three most prevalent pathogenic species of Borrelia in Europe.',\n",
              "  'burgdorferi sensu stricto, Borr.',\n",
              "  0),\n",
              " (1184,\n",
              "  25,\n",
              "  'List the three most prevalent pathogenic species of Borrelia in Europe.',\n",
              "  'garinii, Borr.',\n",
              "  0),\n",
              " (1184,\n",
              "  26,\n",
              "  'List the three most prevalent pathogenic species of Borrelia in Europe.',\n",
              "  'lusitaniae, and Borr.',\n",
              "  0),\n",
              " (1184,\n",
              "  27,\n",
              "  'List the three most prevalent pathogenic species of Borrelia in Europe.',\n",
              "  'valaisiana were identified by hybridization.',\n",
              "  0),\n",
              " (511,\n",
              "  0,\n",
              "  'Is Alpers disease inherited in an autosomal recessive mode?',\n",
              "  'Alpers-Huttenlocher syndrome (AHS) is a very rare autosomal recessive disorder',\n",
              "  1),\n",
              " (511,\n",
              "  1,\n",
              "  'Is Alpers disease inherited in an autosomal recessive mode?',\n",
              "  'Alpers syndrome is an autosomal recessive mitochondrial DNA depletion disorder that affects children and young adults',\n",
              "  1),\n",
              " (511,\n",
              "  2,\n",
              "  'Is Alpers disease inherited in an autosomal recessive mode?',\n",
              "  \"Alpers' syndrome is a fatal neurogenetic disorder first described more than 70 years ago.\",\n",
              "  1),\n",
              " (511,\n",
              "  3,\n",
              "  'Is Alpers disease inherited in an autosomal recessive mode?',\n",
              "  'It is an autosomal recessive, developmental mitochondrial DNA depletion disorder characterized by deficiency in mitochondrial DNA polymerase gamma (POLG) catalytic activity, refractory seizures, neurodegeneration, and liver disease',\n",
              "  1),\n",
              " (511,\n",
              "  4,\n",
              "  'Is Alpers disease inherited in an autosomal recessive mode?',\n",
              "  'Histopathological findings in both patients ((a) chronic hepatitis with prominent bile duct proliferation, fatty change, and fibrosis; (b) in the brain a patchy destruction of the cerebral cortex, predominantly involving striate cortex) were characteristic of progressive neuronal degeneration of childhood with liver disease--Alpers-Huttenlocher syndrome--a rare autosomal recessive disorder usually seen in infants and young children',\n",
              "  0),\n",
              " (511,\n",
              "  5,\n",
              "  'Is Alpers disease inherited in an autosomal recessive mode?',\n",
              "  'Histopathological findings in both patients ((a) chronic hepatitis with prominent bile duct proliferation, fatty change, and fibrosis; (b) in the brain a patchy destruction of the cerebral cortex, predominantly involving striate cortex) were characteristic of progressive neuronal degeneration of childhood with liver disease--Alpers-Huttenlocher syndrome--a rare autosomal recessive disorder usually seen in infants and young children.',\n",
              "  0),\n",
              " (511,\n",
              "  6,\n",
              "  'Is Alpers disease inherited in an autosomal recessive mode?',\n",
              "  'Alpers syndrome is a rare autosomal recessive hepatocerebral degenerative disorder.',\n",
              "  0),\n",
              " (511,\n",
              "  7,\n",
              "  'Is Alpers disease inherited in an autosomal recessive mode?',\n",
              "  'Alpers disease is a recessive mitochondrial disorder caused by mutations in POLG1 and characterized primarily by progressive neurological and hepatic degeneration.',\n",
              "  0),\n",
              " (511,\n",
              "  8,\n",
              "  'Is Alpers disease inherited in an autosomal recessive mode?',\n",
              "  'Alpers syndrome is an autosomal recessive mitochondrial DNA depletion disorder that affects children and young adults.',\n",
              "  1),\n",
              " (511,\n",
              "  9,\n",
              "  'Is Alpers disease inherited in an autosomal recessive mode?',\n",
              "  'We conclude that Alpers disease can be a cause of rapidly progressive liver failure in early childhood.',\n",
              "  0),\n",
              " (511,\n",
              "  10,\n",
              "  'Is Alpers disease inherited in an autosomal recessive mode?',\n",
              "  'Although the cause of this autosomal recessive disease is not known, it does not appear to be related to peroxisomal dysfunction.',\n",
              "  0),\n",
              " (2898,\n",
              "  0,\n",
              "  'What is evaluated with the SAD PERSONS scale?',\n",
              "  'METHOD: We compared the predictive accuracy of the Manchester Self-Harm Rule (MSHR), ReACT Self-Harm Rule (ReACT), SAD PERSONS Scale (SPS) and Modified SAD PERSONS Scale (MSPS) in an unselected sample of patients attending hospital following self-harm.',\n",
              "  0),\n",
              " (2898,\n",
              "  1,\n",
              "  'What is evaluated with the SAD PERSONS scale?',\n",
              "  'Clinicians completed a standard suicide risk instrument (modified SAD PERSONS scale), a 10-point Likert scale assessment of judgment of patient suicide risk (Clinician Prediction Scale), and a measure of their emotional responses to the patient (Therapist Response Questionnaire-Suicide Form).',\n",
              "  0),\n",
              " (2898,\n",
              "  2,\n",
              "  'What is evaluated with the SAD PERSONS scale?',\n",
              "  'AimsTo evaluate the performance of risk scales (Manchester Self-Harm Rule, ReACT Self-Harm Rule, SAD PERSONS scale, Modified SAD PERSONS scale, Barratt Impulsiveness Scale); and patient and clinician estimates of risk in identifying patients who repeat self-harm within 6 months.',\n",
              "  0),\n",
              " (2898,\n",
              "  3,\n",
              "  'What is evaluated with the SAD PERSONS scale?',\n",
              "  'Predicting suicide with the SAD PERSONS scale.',\n",
              "  1),\n",
              " (2898,\n",
              "  4,\n",
              "  'What is evaluated with the SAD PERSONS scale?',\n",
              "  'The SAD PERSONS suicide risk assessment scale is widely implemented in clinical settings despite limited supporting evidence.',\n",
              "  0),\n",
              " (2898,\n",
              "  5,\n",
              "  'What is evaluated with the SAD PERSONS scale?',\n",
              "  'This article aims to determine the ability of the SAD PERSONS scale (SPS) to predict future suicide in the emergency department.',\n",
              "  1),\n",
              " (2898,\n",
              "  6,\n",
              "  'What is evaluated with the SAD PERSONS scale?',\n",
              "  'CONCLUSIONS: Although widely used in educational and clinical settings, these findings do not support the use of the SPS and Modified SPS to predict suicide in adults seen by psychiatric services in the emergency department.',\n",
              "  0),\n",
              " (2898,\n",
              "  7,\n",
              "  'What is evaluated with the SAD PERSONS scale?',\n",
              "  ' For the outcome suicide attempt SAD PERSONS Scale had a sensitivity of 15% (95% CI 8-24) and specificity of 97% (96-98), and the Manchester Self-Harm Rule (MSHR) a sensitivity of 97% (97-97) and a specificity of 20% (20-21).',\n",
              "  0),\n",
              " (2898,\n",
              "  8,\n",
              "  'What is evaluated with the SAD PERSONS scale?',\n",
              "  'ReACT, which is a modification of MSHR, had a similar low specificity, as did the Sodersjukhuset Self Harm Rule.',\n",
              "  0),\n",
              " (2898,\n",
              "  9,\n",
              "  'What is evaluated with the SAD PERSONS scale?',\n",
              "  'OBJECTIVE\\nThe SAD PERSONS scale is a widely used risk assessment tool for suicidal behavior despite a paucity of supporting data.',\n",
              "  0),\n",
              " (2898,\n",
              "  10,\n",
              "  'What is evaluated with the SAD PERSONS scale?',\n",
              "  'SAD PERSONS did not predict suicide attempts better than chance (area under the curve =0.572; 95% confidence interval [CI], 0.51-0.64; P value nonsignificant).',\n",
              "  0),\n",
              " (2898,\n",
              "  11,\n",
              "  'What is evaluated with the SAD PERSONS scale?',\n",
              "  'CONCLUSION\\nIn their current form, SAD PERSONS and MSPS do not accurately predict future suicide attempts.',\n",
              "  0),\n",
              " (2898,\n",
              "  12,\n",
              "  'What is evaluated with the SAD PERSONS scale?',\n",
              "  'CONCLUSIONS\\nThe Chinese SAD PERSONS Scale is a brief instrument with acceptable psychometric properties for self-harm prediction.',\n",
              "  0),\n",
              " (2898,\n",
              "  13,\n",
              "  'What is evaluated with the SAD PERSONS scale?',\n",
              "  'The SAD PERSONS suicide risk assessment scale is widely implemented in clinical settings despite limited supporting evidence.',\n",
              "  0),\n",
              " (2898,\n",
              "  14,\n",
              "  'What is evaluated with the SAD PERSONS scale?',\n",
              "  'OBJECTIVE The SAD PERSONS scale is a widely used risk assessment tool for suicidal behavior despite a paucity of supporting data.',\n",
              "  0),\n",
              " (2898,\n",
              "  15,\n",
              "  'What is evaluated with the SAD PERSONS scale?',\n",
              "  'CONCLUSIONS The Chinese SAD PERSONS Scale is a brief instrument with acceptable psychometric properties for self-harm prediction.',\n",
              "  0),\n",
              " (2898,\n",
              "  16,\n",
              "  'What is evaluated with the SAD PERSONS scale?',\n",
              "  'The SAD PERSONS scale (SPS) is widely used for suicide risk assessment in clinical and educational settings.',\n",
              "  0),\n",
              " (2898,\n",
              "  17,\n",
              "  'What is evaluated with the SAD PERSONS scale?',\n",
              "  \"Beck's Medical Lethality Scale (BMLS) was administered to assess the degree of medical injury, and the SAD PERSONS mnemonic scale was used to evaluate suicide risk.\",\n",
              "  1),\n",
              " (2898,\n",
              "  18,\n",
              "  'What is evaluated with the SAD PERSONS scale?',\n",
              "  'The SAD PERSONS scale for suicide risk assessment: a systematic review.The SAD PERSONS scale (SPS) is widely used for suicide risk assessment in clinical and educational settings.',\n",
              "  0),\n",
              " (2898,\n",
              "  19,\n",
              "  'What is evaluated with the SAD PERSONS scale?',\n",
              "  'The SAD PERSONS suicide risk assessment scale is widely implemented in clinical settings despite limited supporting evidence.',\n",
              "  0),\n",
              " (2898,\n",
              "  20,\n",
              "  'What is evaluated with the SAD PERSONS scale?',\n",
              "  '<AbstractText Label=\"CONCLUSIONS\" NlmCategory=\"CONCLUSIONS\">The Chinese SAD PERSONS Scale is a brief instrument with acceptable psychometric properties for self-harm prediction.',\n",
              "  0),\n",
              " (2898,\n",
              "  21,\n",
              "  'What is evaluated with the SAD PERSONS scale?',\n",
              "  'Predicting suicide attempts with the SAD PERSONS scale: a longitudinal analysis.<AbstractText Label=\"OBJECTIVE\" NlmCategory=\"OBJECTIVE\">The SAD PERSONS scale is a widely used risk assessment tool for suicidal behavior despite a paucity of supporting data.',\n",
              "  1),\n",
              " (2898,\n",
              "  22,\n",
              "  'What is evaluated with the SAD PERSONS scale?',\n",
              "  'Predicting suicide with the SAD PERSONS scale.<AbstractText Label=\"BACKGROUND\">Suicide is a major public health issue, and a priority requirement is accurately identifying high-risk individuals.',\n",
              "  1),\n",
              " (2898,\n",
              "  23,\n",
              "  'What is evaluated with the SAD PERSONS scale?',\n",
              "  'Subjects completed a series of depression/suicide screening tools: the Columbia Suicide Severity Scale, SAD PERSONS scale, Patient Health Questionnaire 9, and Beck Scale for Suicidal Ideation.',\n",
              "  0),\n",
              " (2898,\n",
              "  24,\n",
              "  'What is evaluated with the SAD PERSONS scale?',\n",
              "  '<b>OBJECTIVE</b>: The SAD PERSONS scale is a widely used risk assessment tool for suicidal behavior despite a paucity of supporting data.',\n",
              "  0),\n",
              " (2898,\n",
              "  25,\n",
              "  'What is evaluated with the SAD PERSONS scale?',\n",
              "  'Both SAD PERSONS and MSPS showed poor predictive ability for future suicide attempts.',\n",
              "  0),\n",
              " (2898,\n",
              "  26,\n",
              "  'What is evaluated with the SAD PERSONS scale?',\n",
              "  'SAD PERSONS did not predict suicide attempts better than chance (area under the curve =0.572; 95% confidence interval [CI], 0.51-0.64; P value nonsignificant).',\n",
              "  0),\n",
              " (1124,\n",
              "  0,\n",
              "  'What is the role of the RUNX1-MYEF2 complex?',\n",
              "  'A novel complex, RUNX1-MYEF2, represses hematopoietic genes in erythroid cells.',\n",
              "  1),\n",
              " (1124,\n",
              "  1,\n",
              "  'What is the role of the RUNX1-MYEF2 complex?',\n",
              "  'Functional analysis shows that a subset of the target genes is suppressed by RUNX1 via the newly identified partner MYEF2.',\n",
              "  1),\n",
              " (1124,\n",
              "  2,\n",
              "  'What is the role of the RUNX1-MYEF2 complex?',\n",
              "  'Knockdown of Myef2 expression in developing zebrafish results in a reduced number of HSC.',\n",
              "  1),\n",
              " (1124,\n",
              "  3,\n",
              "  'What is the role of the RUNX1-MYEF2 complex?',\n",
              "  'Functional analysis shows that a subset of the target genes is suppressed by RUNX1 via the newly identified partner MYEF2.',\n",
              "  0),\n",
              " (1124,\n",
              "  4,\n",
              "  'What is the role of the RUNX1-MYEF2 complex?',\n",
              "  'Knockdown of Myef2 expression in developing zebrafish results in a reduced number of HSC.',\n",
              "  1),\n",
              " (1124,\n",
              "  5,\n",
              "  'What is the role of the RUNX1-MYEF2 complex?',\n",
              "  'Chromatin immunoprecipitation followed by sequencing (ChIP-seq) and microarray expression analysis were used to show that RUNX1 binds approximately 9,000 target sites in erythroid cells and is primarily active in the undifferentiated state.',\n",
              "  1),\n",
              " (1124,\n",
              "  6,\n",
              "  'What is the role of the RUNX1-MYEF2 complex?',\n",
              "  'Functional analysis shows that a subset of the target genes is suppressed by RUNX1 via the newly identified partner MYEF2.',\n",
              "  0),\n",
              " (1643,\n",
              "  0,\n",
              "  'What is the localization of the protein encoded by the gene DNAJC11?',\n",
              "  ' DnaJC11, respectively.',\n",
              "  1),\n",
              " (1643,\n",
              "  1,\n",
              "  'What is the localization of the protein encoded by the gene DNAJC11?',\n",
              "  'The first three are outer membrane proteins, CHCHD3 has been assigned to the matrix space, and the other two proteins have not been described in mitochondria previously.',\n",
              "  1),\n",
              " (1086,\n",
              "  0,\n",
              "  'Carbapenemase-producing gram-negative bacteria is a major health concern because their resistance to antibiotics.\\nList the most frequent carbapenemases found in Enterobacteriaceae.',\n",
              "  'The most frequent carbapenemases in Enterobacteriaceae were OXA-48, KPC and VIM-1; in P. aeruginosa it was VIM-2 and in A. baumannii OXA-23.',\n",
              "  1),\n",
              " (1086,\n",
              "  1,\n",
              "  'Carbapenemase-producing gram-negative bacteria is a major health concern because their resistance to antibiotics.\\nList the most frequent carbapenemases found in Enterobacteriaceae.',\n",
              "  'bla(IMP), bla(VIM), bla(SPM), bla(SIM) and bla(GIM), and of bla(KPC), bla(NDM) and bla(OXA-',\n",
              "  0),\n",
              " (1086,\n",
              "  2,\n",
              "  'Carbapenemase-producing gram-negative bacteria is a major health concern because their resistance to antibiotics.\\nList the most frequent carbapenemases found in Enterobacteriaceae.',\n",
              "  'Detection of NDM-1, VIM-1, KPC, OXA-48, and OXA-162 carbapenemases by matrix-assisted laser desorption ionization-time of flight mass spectrometry.',\n",
              "  0),\n",
              " (1086,\n",
              "  3,\n",
              "  'Carbapenemase-producing gram-negative bacteria is a major health concern because their resistance to antibiotics.\\nList the most frequent carbapenemases found in Enterobacteriaceae.',\n",
              "  'The Check-MDR CT102 DNA microarray enables detection of the most prevalent carbapenemases (NDM, VIM, KPC, OXA-48 and IMP)',\n",
              "  1),\n",
              " (1086,\n",
              "  4,\n",
              "  'Carbapenemase-producing gram-negative bacteria is a major health concern because their resistance to antibiotics.\\nList the most frequent carbapenemases found in Enterobacteriaceae.',\n",
              "  '63 KPC producers, 47 MBL producers, and 31 KPC and MBL producers',\n",
              "  0),\n",
              " (1086,\n",
              "  5,\n",
              "  'Carbapenemase-producing gram-negative bacteria is a major health concern because their resistance to antibiotics.\\nList the most frequent carbapenemases found in Enterobacteriaceae.',\n",
              "  'OXA-48 producers were poorly detected, but all systems reliably detected isolates with KPC and most with metallo-carbapenemases.',\n",
              "  0),\n",
              " (1086,\n",
              "  6,\n",
              "  'Carbapenemase-producing gram-negative bacteria is a major health concern because their resistance to antibiotics.\\nList the most frequent carbapenemases found in Enterobacteriaceae.',\n",
              "  'Producers of IMP-, VIM-, and KPC-type carbapenemases with high levels of resistance to cephalosporins and to carbapenems were detected at 1x10(1) CFU/ml.',\n",
              "  0),\n",
              " (1086,\n",
              "  7,\n",
              "  'Carbapenemase-producing gram-negative bacteria is a major health concern because their resistance to antibiotics.\\nList the most frequent carbapenemases found in Enterobacteriaceae.',\n",
              "  'The OXA-48 producers were not detected on ChromID ESBL medium unless coexpressing ESBLs, whereas carbapenemase-producing isolates with MICs of <4 microg/ml were not detected on CHROMagar KPC medium.',\n",
              "  0),\n",
              " (1086,\n",
              "  8,\n",
              "  'Carbapenemase-producing gram-negative bacteria is a major health concern because their resistance to antibiotics.\\nList the most frequent carbapenemases found in Enterobacteriaceae.',\n",
              "  'The most frequent carbapenemase was KPC-2 or -3 (73 strains), followed by VIM-1 (14), IMP-1 (11), SME-2 (5), and NMC-A (1).',\n",
              "  1),\n",
              " (1086,\n",
              "  9,\n",
              "  'Carbapenemase-producing gram-negative bacteria is a major health concern because their resistance to antibiotics.\\nList the most frequent carbapenemases found in Enterobacteriaceae.',\n",
              "  'All serine carbapenemases were detected in the United States, while MbetaL-producing strains were isolated in Europe.',\n",
              "  0),\n",
              " (1086,\n",
              "  10,\n",
              "  'Carbapenemase-producing gram-negative bacteria is a major health concern because their resistance to antibiotics.\\nList the most frequent carbapenemases found in Enterobacteriaceae.',\n",
              "  'Carbapenemase-producing Enterobacteriaceae showed high rates of resistance to most antimicrobial agents tested.',\n",
              "  0),\n",
              " (1086,\n",
              "  11,\n",
              "  'Carbapenemase-producing gram-negative bacteria is a major health concern because their resistance to antibiotics.\\nList the most frequent carbapenemases found in Enterobacteriaceae.',\n",
              "  'By using this methodology, isolates producing KPC, GES, Sme, IMI, and NMC-A carbapenemases were successfully distinguished from those producing other classes of beta-lactamases',\n",
              "  1),\n",
              " (1086,\n",
              "  12,\n",
              "  'Carbapenemase-producing gram-negative bacteria is a major health concern because their resistance to antibiotics.\\nList the most frequent carbapenemases found in Enterobacteriaceae.',\n",
              "  'KPC-2 and KPC-3 were the most frequently occurring carbapenemases (24 isolates, 20.2%) i',\n",
              "  0),\n",
              " (1086,\n",
              "  13,\n",
              "  'Carbapenemase-producing gram-negative bacteria is a major health concern because their resistance to antibiotics.\\nList the most frequent carbapenemases found in Enterobacteriaceae.',\n",
              "  'K. pneumoniae strains with known KPC (n\\u2003=\\u200331) or VIM (n\\u2003=\\u200320) carbapenemases',\n",
              "  0),\n",
              " (1086,\n",
              "  14,\n",
              "  'Carbapenemase-producing gram-negative bacteria is a major health concern because their resistance to antibiotics.\\nList the most frequent carbapenemases found in Enterobacteriaceae.',\n",
              "  'Carbapenemases including Klebsiella pneumoniae carbapenemases (KPC) are widespread among clinical isolates in the family Enterobacteriaceae.',\n",
              "  0),\n",
              " (1086,\n",
              "  15,\n",
              "  'Carbapenemase-producing gram-negative bacteria is a major health concern because their resistance to antibiotics.\\nList the most frequent carbapenemases found in Enterobacteriaceae.',\n",
              "  'Outbreak of vim-1-carbapenemase-producing Enterobacter cloacae in a pediatric intensive care unit.',\n",
              "  0),\n",
              " (1086,\n",
              "  16,\n",
              "  'Carbapenemase-producing gram-negative bacteria is a major health concern because their resistance to antibiotics.\\nList the most frequent carbapenemases found in Enterobacteriaceae.',\n",
              "  'mergence of pediatric infections caused by carbapenemases-producing Enterobacteriaceae is a critical issue as they are resistant to most β-lactam antibiotics.',\n",
              "  0),\n",
              " (1086,\n",
              "  17,\n",
              "  'Carbapenemase-producing gram-negative bacteria is a major health concern because their resistance to antibiotics.\\nList the most frequent carbapenemases found in Enterobacteriaceae.',\n",
              "  'SME-2-producing S. marcescens were isolated in the New York City area, Texas, and Ohio, while NMC-A was found in one E. cloacae strain from New York.',\n",
              "  0),\n",
              " (1086,\n",
              "  18,\n",
              "  'Carbapenemase-producing gram-negative bacteria is a major health concern because their resistance to antibiotics.\\nList the most frequent carbapenemases found in Enterobacteriaceae.',\n",
              "  'In contrast, metallo-beta-lactamases were prevalent in Europe.',\n",
              "  0),\n",
              " (1086,\n",
              "  19,\n",
              "  'Carbapenemase-producing gram-negative bacteria is a major health concern because their resistance to antibiotics.\\nList the most frequent carbapenemases found in Enterobacteriaceae.',\n",
              "  'IMP-1-producing E. cloacae (11 isolates) were detected in Turkey, while VIM-1-producing strains were found in Italy (Enterobacter spp.)',\n",
              "  0),\n",
              " (1086,\n",
              "  20,\n",
              "  'Carbapenemase-producing gram-negative bacteria is a major health concern because their resistance to antibiotics.\\nList the most frequent carbapenemases found in Enterobacteriaceae.',\n",
              "  'and Greece (Klebsiella pneumoniae).',\n",
              "  0),\n",
              " (1086,\n",
              "  21,\n",
              "  'Carbapenemase-producing gram-negative bacteria is a major health concern because their resistance to antibiotics.\\nList the most frequent carbapenemases found in Enterobacteriaceae.',\n",
              "  'd several simple confirmatory methods that allow the recognition of bacteria producing class A carbapenemases, including KPC, Sme, IMI, NMC-A, and GES,',\n",
              "  1),\n",
              " (4183,\n",
              "  0,\n",
              "  'Is Tranexamic acid effective for intracerebral haemorrhage?',\n",
              "  'Tranexamic acid did not increase the risk of post-intracerebral haemorrhage seizures in the first 90\\u2009days.',\n",
              "  1),\n",
              " (4183,\n",
              "  1,\n",
              "  'Is Tranexamic acid effective for intracerebral haemorrhage?',\n",
              "  'The primary outcome was not different between the two groups: 26 (52%) patients in the placebo group and 22 (44%) in the tranexamic acid group had intracerebral haemorrhage growth (odds ratio [OR] 0·72 [95% CI 0·32-1·59], p=0·41).',\n",
              "  0),\n",
              " (4183,\n",
              "  2,\n",
              "  'Is Tranexamic acid effective for intracerebral haemorrhage?',\n",
              "  'There was no evidence of a difference in the proportions of patients who died or had thromboembolic complications between the groups: eight (16%) in the placebo group vs 13 (26%) in the tranexamic acid group died and two (4%) vs one (2%) had thromboembolic complications.',\n",
              "  0),\n",
              " (4183,\n",
              "  3,\n",
              "  'Is Tranexamic acid effective for intracerebral haemorrhage?',\n",
              "  'None of the deaths was considered related to study medication.INTERPRETATION: Our study does not provide evidence that tranexamic acid prevents intracerebral haemorrhage growth, although the treatment was safe with no increase in thromboembolic complications.',\n",
              "  1),\n",
              " (4183,\n",
              "  4,\n",
              "  'Is Tranexamic acid effective for intracerebral haemorrhage?',\n",
              "  \"CONCLUSIONS: Tranexamic acid did not affect a patient's functional status at 90 days after ICH, despite there being significant modest reductions in early death (by 7 days), haematoma expansion and SAEs, which is consistent with an antifibrinolytic effect.\",\n",
              "  0),\n",
              " (4183,\n",
              "  5,\n",
              "  'Is Tranexamic acid effective for intracerebral haemorrhage?',\n",
              "  'INTERPRETATION: Functional status 90 days after intracerebral haemorrhage did not differ significantly between patients who received tranexamic acid and those who received placebo, despite a reduction in early deaths and serious adverse events.',\n",
              "  1),\n",
              " (4183,\n",
              "  6,\n",
              "  'Is Tranexamic acid effective for intracerebral haemorrhage?',\n",
              "  'Of these, two randomised controlled trials (RCTs) comparing intravenous tranexamic acid to placebo (n\\u2009=\\u200954) reported no significant difference in death or dependency.',\n",
              "  1),\n",
              " (4183,\n",
              "  7,\n",
              "  'Is Tranexamic acid effective for intracerebral haemorrhage?',\n",
              "  'Three observational studies (n\\u2009=\\u2009281) suggested less haematoma growth with rapid tranexamic acid infusion.',\n",
              "  1),\n",
              " (2662,\n",
              "  0,\n",
              "  'Does oncogene-induced DNA replication stress inhibit genomic instability?',\n",
              "  'Oncogene-induced DNA replication stress is thought to drive genomic instability in cancer.',\n",
              "  1),\n",
              " (2662,\n",
              "  1,\n",
              "  'Does oncogene-induced DNA replication stress inhibit genomic instability?',\n",
              "  'We propose that single-stranded DNA generated in response to oncogene-induced replication stress compromises the repair of deaminated cytosines and other damaged bases, leading to the observed SNS mutator phenotype.',\n",
              "  1),\n",
              " (1898,\n",
              "  0,\n",
              "  'Is Stat4 a transcription factor?',\n",
              "  'transcription factors T-bet and STAT4',\n",
              "  1),\n",
              " (1898,\n",
              "  1,\n",
              "  'Is Stat4 a transcription factor?',\n",
              "  'STAT4 is a member of the signal transducer and activator of transcription (STAT) family of molecules that localizes to the cytoplasm.',\n",
              "  1),\n",
              " (1898,\n",
              "  2,\n",
              "  'Is Stat4 a transcription factor?',\n",
              "  'STAT4 regulates various genes expression as a transcription factor after it is phosphorylated, dimerizes and translocates to the nucleus.',\n",
              "  1),\n",
              " (1898,\n",
              "  3,\n",
              "  'Is Stat4 a transcription factor?',\n",
              "  'STAT4 is a latent cytosolic factor that encodes a transcription factor transmitting signals stimulated by cytokines.',\n",
              "  1),\n",
              " (1898,\n",
              "  4,\n",
              "  'Is Stat4 a transcription factor?',\n",
              "  ' To investigate the role of signal transduction and activation of transcription 4 (STAT4) in the development and progression of human hepatocellular carcinoma (HCC)',\n",
              "  1),\n",
              " (2736,\n",
              "  0,\n",
              "  'What causes leishmaniasis?',\n",
              "  'we will focus on the shape and form ofLeishmaniaspp., a group of very successful protozoan parasites that cause a range of diseases from self-healing cutaneous leishmaniasis to visceral leishmaniasis',\n",
              "  1),\n",
              " (2736,\n",
              "  1,\n",
              "  'What causes leishmaniasis?',\n",
              "  'In conclusion, our data demonstrates that in susceptible mice L. (L.) amazonensis, a causative agent of tegumentary leishmaniasis, causes pathological changes similar to those produced by Leishmania (L.) infantum in both humans and canids.',\n",
              "  0),\n",
              " (2736,\n",
              "  2,\n",
              "  'What causes leishmaniasis?',\n",
              "  'American Tegumentary Leishmaniasis is caused by parasites of the genus Leishmania, and causes significant health problems throughout the Americas.',\n",
              "  0),\n",
              " (2736,\n",
              "  3,\n",
              "  'What causes leishmaniasis?',\n",
              "  'Leishmaniasis is a tropical infection caused by the protozoan, belonging to the group of Leishmania which causes Old World and New World disease.',\n",
              "  0),\n",
              " (2736,\n",
              "  4,\n",
              "  'What causes leishmaniasis?',\n",
              "  'Visceral leishmaniasis (VL) is caused by the protozoan parasite Leishmania donovani and transmitted by the bite of infected sandfly Phlebotomus argentipes.',\n",
              "  0),\n",
              " (2736,\n",
              "  5,\n",
              "  'What causes leishmaniasis?',\n",
              "  'Leishmania donovani is a causative agent of visceral leishmaniasis in South East Asia and Eastern Africa.',\n",
              "  1),\n",
              " (2736,\n",
              "  6,\n",
              "  'What causes leishmaniasis?',\n",
              "  'Leishmaniasis is a neglected tropical disease caused by Leishmania protozoa and associated with three main clinical presentations: cutaneous, mucocutaneous and visceral leishmaniasis.',\n",
              "  0),\n",
              " (2736,\n",
              "  7,\n",
              "  'What causes leishmaniasis?',\n",
              "  'L. (L.)\\xa0donovani usually causes visceral leishmaniasis, but in this case, the patient manifested cutaneous leishmaniasis.',\n",
              "  0),\n",
              " (2736,\n",
              "  8,\n",
              "  'What causes leishmaniasis?',\n",
              "  'Leishmania donovani in India causes visceral infection (kala-azar) and dermal infection (post-kala-azar dermal leishmaniasis).',\n",
              "  0),\n",
              " (2736,\n",
              "  9,\n",
              "  'What causes leishmaniasis?',\n",
              "  'Leishmaniasis is a clinically heterogeneous syndrome caused by intracellular protozoan parasites of the genus Leishmania.',\n",
              "  0),\n",
              " (2736,\n",
              "  10,\n",
              "  'What causes leishmaniasis?',\n",
              "  'Leishmaniasis is a parasitic infection caused by many species of the protozoa Leishmania.',\n",
              "  0),\n",
              " (2736,\n",
              "  11,\n",
              "  'What causes leishmaniasis?',\n",
              "  'Here we report an imported case of leishmaniasis by Leishmania (Leishmania) donovani infection from Sri Lanka.',\n",
              "  0),\n",
              " (2736,\n",
              "  12,\n",
              "  'What causes leishmaniasis?',\n",
              "  'Finally, Leishmania species cause leishmaniasis, a disease that ranges from self-healing but scarring cutaneous lesions to fatal visceral leishmaniasis in which parasites disseminate to the liver, spleen, and bone marrow.',\n",
              "  1),\n",
              " (2736,\n",
              "  13,\n",
              "  'What causes leishmaniasis?',\n",
              "  'Leishmania is a digenetic protozoan parasite causing leishmaniasis in humans.',\n",
              "  1),\n",
              " (2736,\n",
              "  14,\n",
              "  'What causes leishmaniasis?',\n",
              "  'In general, Leishmania species causing cutaneous leishmaniasis (CL) are more sensitive to Sb(III) than the species responsible for visceral leishmaniasis (VL).',\n",
              "  0),\n",
              " (2736,\n",
              "  15,\n",
              "  'What causes leishmaniasis?',\n",
              "  'Leishmania parasites cause leishmaniasis in humans and animals worldwide.',\n",
              "  0),\n",
              " (2736,\n",
              "  16,\n",
              "  'What causes leishmaniasis?',\n",
              "  'Leishmania is an intracellular protozoan parasite which causes Leishmaniasis, a global health problem affecting millions of people throughout 89 different countries in the world.',\n",
              "  0),\n",
              " (2736,\n",
              "  17,\n",
              "  'What causes leishmaniasis?',\n",
              "  'Leishmania donovani is an intracellular protozoan parasite that causes leishmaniasis, which can range from a self-healing cutaneous disease to a fatal visceral disease depending on the infecting species.',\n",
              "  1),\n",
              " (2736,\n",
              "  18,\n",
              "  'What causes leishmaniasis?',\n",
              "  'Leishmania major causes leishmaniasis and is grouped within the Trypanosomatidae family, which also includes the etiologic agent for African sleeping sickness, Trypanosoma brucei.',\n",
              "  0),\n",
              " (2736,\n",
              "  19,\n",
              "  'What causes leishmaniasis?',\n",
              "  'Leishmania major is an important trypanosomatid pathogen that causes leishmaniasis, which is a serious disease in much of the Old World.',\n",
              "  0),\n",
              " (2736,\n",
              "  20,\n",
              "  'What causes leishmaniasis?',\n",
              "  'Leishmania, a protozoan parasite that causes leishmaniasis, affects 1-2 million people every year worldwide.',\n",
              "  0),\n",
              " (2736,\n",
              "  21,\n",
              "  'What causes leishmaniasis?',\n",
              "  'The intracellular protozoan parasite Leishmania causes leishmaniasis, which is the second biggest killer worldwide among parasitic diseases, after malaria.',\n",
              "  0),\n",
              " (2736,\n",
              "  22,\n",
              "  'What causes leishmaniasis?',\n",
              "  'The leishmaniases are a group of diseases caused by species of Leishmania and transmitted by the bite of the female sandfly.',\n",
              "  0),\n",
              " (2736,\n",
              "  23,\n",
              "  'What causes leishmaniasis?',\n",
              "  'Sri Lankan cutaneous leishmaniasis is caused by Leishmania donovani zymodeme MON-37',\n",
              "  0),\n",
              " (2736,\n",
              "  24,\n",
              "  'What causes leishmaniasis?',\n",
              "  'Tegumentary leishmaniasis in Latin America is caused mainly by Leishmania viannia braziliensis complex parasites.',\n",
              "  0),\n",
              " (2736,\n",
              "  25,\n",
              "  'What causes leishmaniasis?',\n",
              "  'L. braziliensis and Leishmania viannia peruviana are the 2 predominant Leishmania species in Peru.',\n",
              "  0),\n",
              " (2736,\n",
              "  26,\n",
              "  'What causes leishmaniasis?',\n",
              "  'Identification of Leishmania species causing cutaneous leishmaniasis using Random Amplified Polymorphic DNA (RAPD-PCR) in Kharve, Iran.',\n",
              "  0),\n",
              " (1763,\n",
              "  0,\n",
              "  'Data from which major epigenome projects are contained in the DeepBlue epigenomic data server?',\n",
              "  'Here we present the DeepBlue Epigenomic Data Server, which streamlines epigenomic data analysis as well as software development.',\n",
              "  1),\n",
              " (1763,\n",
              "  1,\n",
              "  'Data from which major epigenome projects are contained in the DeepBlue epigenomic data server?',\n",
              "  'DeepBlue provides a comprehensive programmatic interface for finding, selecting, filtering, summarizing and downloading region sets.',\n",
              "  1),\n",
              " (1763,\n",
              "  2,\n",
              "  'Data from which major epigenome projects are contained in the DeepBlue epigenomic data server?',\n",
              "  'It contains data from four major epigenome projects, namely ENCODE, ROADMAP, BLUEPRINT and DEEP.',\n",
              "  1),\n",
              " (234,\n",
              "  0,\n",
              "  'What clinical use aptamers may have?',\n",
              "  'In murine tumor models, the aptamer-targeted siRNA-mediated NMD inhibition in tumor cells led to significant inhibition of tumor growth, which was superior to best-in-class \"conventional\" cancer vaccination protocols.',\n",
              "  0),\n",
              " (234,\n",
              "  1,\n",
              "  'What clinical use aptamers may have?',\n",
              "  'Tumor-targeted NMD inhibition forms the basis of a simple, broadly useful, and clinically feasible approach to enhance the antigenicity of disseminated tumors leading to their immune recognition and rejection.',\n",
              "  1),\n",
              " (234,\n",
              "  2,\n",
              "  'What clinical use aptamers may have?',\n",
              "  'This study is focused on one aptamer (termed A2).',\n",
              "  0),\n",
              " (234,\n",
              "  3,\n",
              "  'What clinical use aptamers may have?',\n",
              "  'Transfection of this molecule into HPV16-transformed cells resulted in inhibition of cell proliferation (shown using real-time cell electronic sensing and MTT assays) due to the induction of apoptosis (as demonstrated by Annexin V/propidium iodide staining).',\n",
              "  0),\n",
              " (234,\n",
              "  4,\n",
              "  'What clinical use aptamers may have?',\n",
              "  'Transfection of cells with A2 was correlated with the loss of E7 and the induction of apoptosis.',\n",
              "  0),\n",
              " (234,\n",
              "  5,\n",
              "  'What clinical use aptamers may have?',\n",
              "  'Aptamers specific for a number of cellular and viral proteins have been documented previously; one aptamer (Macugen) is approved for clinical use and several others are in clinical trials.',\n",
              "  0),\n",
              " (234,\n",
              "  6,\n",
              "  'What clinical use aptamers may have?',\n",
              "  'The high binding affinity and specificity of aptamers allows for isolation of an artificial ligand for theoretically any therapeutic target of interest.',\n",
              "  0),\n",
              " (234,\n",
              "  7,\n",
              "  'What clinical use aptamers may have?',\n",
              "  'Aptamers are a special class of nucleic acid molecules that are beginning to be investigated for clinical use.',\n",
              "  0),\n",
              " (234,\n",
              "  8,\n",
              "  'What clinical use aptamers may have?',\n",
              "  'These small RNA/DNA molecules can form secondary and tertiary structures capable of specifically binding proteins or other cellular targets; they are essentially a chemical equivalent of antibodies.',\n",
              "  0),\n",
              " (234,\n",
              "  9,\n",
              "  'What clinical use aptamers may have?',\n",
              "  'Aptamers have the advantage of being highly specific, relatively small in size, and non-immunogenic.',\n",
              "  0),\n",
              " (234,\n",
              "  10,\n",
              "  'What clinical use aptamers may have?',\n",
              "  'In the last two decades, many aptamers have been clinically developed as inhibitors for targets such as vascular endothelial growth factor (VEGF) and thrombin.',\n",
              "  1),\n",
              " (234,\n",
              "  11,\n",
              "  'What clinical use aptamers may have?',\n",
              "  'The first aptamer based therapeutic was FDA approved in 2004 for the treatment of age-related macular degeneration and several other aptamers are currently being evaluated in clinical trials.',\n",
              "  0),\n",
              " (234,\n",
              "  12,\n",
              "  'What clinical use aptamers may have?',\n",
              "  'Preclinical studies using aptamer-siRNA chimeras and aptamer targeted nanoparticle therapeutics have been very successful in mouse models of cancer and HIV.',\n",
              "  0),\n",
              " (234,\n",
              "  13,\n",
              "  'What clinical use aptamers may have?',\n",
              "  'In summary aptamers are in several stages of development, from pre-clinical studies to clinical trials and even as FDA approved therapeutics.',\n",
              "  0),\n",
              " (234,\n",
              "  14,\n",
              "  'What clinical use aptamers may have?',\n",
              "  'In subcutaneous and metastatic tumour models, tumour-targeted delivery of NMD factor-specific siRNAs conjugated to oligonucleotide aptamer ligands led to significant inhibition of tumour growth that was superior to that of vaccination with granulocyte-macrophage colony-stimulating factor (GM-CSF)-expressing irradiated tumour cells, and could be further enhanced by co-stimulation.',\n",
              "  0),\n",
              " (234,\n",
              "  15,\n",
              "  'What clinical use aptamers may have?',\n",
              "  'An assortment of non-viral carriers including liposomes, peptides, polymers, and aptamers are being evaluated for their ability to shepherd siRNA to the target tissue and cross the plasma membrane barrier into the cell.',\n",
              "  0),\n",
              " (234,\n",
              "  16,\n",
              "  'What clinical use aptamers may have?',\n",
              "  'In addition, a new thromboxane receptor antagonist, NCX-4016, a newly discovered protease-activated receptor antagonist that targets thrombin-induced platelet aggregation, and anti-von Willebrand factor aptamers show tremendous promise in refining antiplatelet therapy by targeting different receptors and molecules.',\n",
              "  0),\n",
              " (234,\n",
              "  17,\n",
              "  'What clinical use aptamers may have?',\n",
              "  'First-in-human evaluation of anti von Willebrand factor therapeutic aptamer ARC1779 in healthy volunteers',\n",
              "  0),\n",
              " (234,\n",
              "  18,\n",
              "  'What clinical use aptamers may have?',\n",
              "  'ARC1779 is a therapeutic aptamer antagonist of the A1 domain of von Willebrand Factor (vWF), the ligand for receptor glycoprotein 1b on platelets.',\n",
              "  0),\n",
              " (234,\n",
              "  19,\n",
              "  'What clinical use aptamers may have?',\n",
              "  'ARC1779 is being developed as a novel antithrombotic agent for use in patients with acute coronary syndromes.',\n",
              "  0),\n",
              " (234,\n",
              "  20,\n",
              "  'What clinical use aptamers may have?',\n",
              "  'This is the first-in-human evaluation of a novel aptamer antagonist of vWF.',\n",
              "  0),\n",
              " (234,\n",
              "  21,\n",
              "  'What clinical use aptamers may have?',\n",
              "  'ARC1779 produced dose- and concentration-dependent inhibition of vWF activity and platelet function with duration of effect suitable for the intended clinical use in acute coronary syndromes.',\n",
              "  0),\n",
              " (234,\n",
              "  22,\n",
              "  'What clinical use aptamers may have?',\n",
              "  'Wet age-related macular degeneration and diabetic retinopathy are pathological consequences of vascular endothelial growth factor (VEGF) release as a reaction to deficiency of oxygen and nutrients in the macular cells.',\n",
              "  1),\n",
              " (234,\n",
              "  23,\n",
              "  'What clinical use aptamers may have?',\n",
              "  'FDA recently approved two drugs specifically aimed at VEGF, bevacizumab, a humanized monoclonal antibody, and pegaptinib, a pegylated aptamer with application in ophthalmic pathologies.',\n",
              "  0),\n",
              " (234,\n",
              "  24,\n",
              "  'What clinical use aptamers may have?',\n",
              "  'Delivery of double-stranded DNA thioaptamers into HIV-1 infected cells for antiviral activity',\n",
              "  0),\n",
              " (234,\n",
              "  25,\n",
              "  'What clinical use aptamers may have?',\n",
              "  'Our lead thioaptamer, R12-2, targets the RNase H domain of the HIV-1 reverse transcriptase (RT) and inhibits viral infection in U373-MAGI-CCR5 cells.',\n",
              "  1),\n",
              " (234,\n",
              "  26,\n",
              "  'What clinical use aptamers may have?',\n",
              "  'In this report, we demonstrate that monothioate-modified DNA duplex oligonucleotides can be efficiently delivered into cells by liposome-based transfection agents to inhibit HIV replication.',\n",
              "  0),\n",
              " (234,\n",
              "  27,\n",
              "  'What clinical use aptamers may have?',\n",
              "  'Nucleic acid aptamers in therapeutic anticoagulation',\n",
              "  0),\n",
              " (234,\n",
              "  28,\n",
              "  'What clinical use aptamers may have?',\n",
              "  'Pegaptanib sodium for the treatment of neovascular age-related macular degeneration',\n",
              "  0),\n",
              " (234,\n",
              "  29,\n",
              "  'What clinical use aptamers may have?',\n",
              "  'Pegaptanib sodium is one of the first therapeutics belonging to the class of compounds known as aptamers.',\n",
              "  0),\n",
              " (234,\n",
              "  30,\n",
              "  'What clinical use aptamers may have?',\n",
              "  'The article highlights and summarises the results of the multi-centre, randomised, sham-controlled clinical trials with pegaptanib sodium to treat subfoveal choroidal neovascularisation in AMD.',\n",
              "  0),\n",
              " (234,\n",
              "  31,\n",
              "  'What clinical use aptamers may have?',\n",
              "  'In addition, the safety profile is reviewed.',\n",
              "  0),\n",
              " (234,\n",
              "  32,\n",
              "  'What clinical use aptamers may have?',\n",
              "  'Regulatable aptamers in medicine: focus on antithrombotic strategies',\n",
              "  0),\n",
              " (234,\n",
              "  33,\n",
              "  'What clinical use aptamers may have?',\n",
              "  'After a decade of intensive research, technology development and initial clinical evaluation, aptamers have now demonstrated broad potential as direct protein ligands and inhibitors, and thus represent an exciting class of compounds for the development of new therapeutic and diagnostic agents.',\n",
              "  0),\n",
              " (234,\n",
              "  34,\n",
              "  'What clinical use aptamers may have?',\n",
              "  'Two R-aptamers (R10-60 and D-R15-8) with the predominant shared characteristics were selected for further study on primary human chronic lymphocytic leukemia (CLL) B cells, which are well known to be difficult to transfect and activate.',\n",
              "  1),\n",
              " (234,\n",
              "  35,\n",
              "  'What clinical use aptamers may have?',\n",
              "  'Together, these findings suggest that the sequence compositions in R-aptamers that promote multimerization and contain optimal ISS CpG motifs facilitate the delivery of ISS-ODN to CLL cells and enhance the activation of these cells.',\n",
              "  0),\n",
              " (234,\n",
              "  36,\n",
              "  'What clinical use aptamers may have?',\n",
              "  'In particular, the use of aptamers for the alleviation of blood clotting and the treatment of AIDS are considered.',\n",
              "  0),\n",
              " (234,\n",
              "  37,\n",
              "  'What clinical use aptamers may have?',\n",
              "  'We synthesized an RNA aptamer probe specific for human CD4 using a reported sequence and investigated the potential use of this probe in cell phenotyping.',\n",
              "  0),\n",
              " (234,\n",
              "  38,\n",
              "  'What clinical use aptamers may have?',\n",
              "  'Studies in cultured cells demonstrated that the synthetic CD4 aptamer had a nearly identical cell-binding specificity as the standard CD4 antibody.',\n",
              "  0),\n",
              " (234,\n",
              "  39,\n",
              "  'What clinical use aptamers may have?',\n",
              "  'Fluorescent microscopy confirmed that the aptamer and antibody generated the same CD4 staining pattern in cells without competing with one another.',\n",
              "  0),\n",
              " (234,\n",
              "  40,\n",
              "  'What clinical use aptamers may have?',\n",
              "  'transfection of HPV16-transformed cells with A2 appeared to result in the loss of E7 and rise in pRb levels',\n",
              "  0),\n",
              " (41,\n",
              "  0,\n",
              "  'Which factors activate zygotic gene expression during the maternal-to-zygotic transition in zebrafish?',\n",
              "  'Nanog, Pou5f1 and SoxB1 activate zygotic gene expression during the maternal-to-zygotic transition.',\n",
              "  1),\n",
              " (41,\n",
              "  1,\n",
              "  'Which factors activate zygotic gene expression during the maternal-to-zygotic transition in zebrafish?',\n",
              "  'Our results demonstrate that maternal Nanog, Pou5f1 and SoxB1 are required to initiate the zygotic developmental program and induce clearance of the maternal program by activating miR-430 expression.',\n",
              "  1),\n",
              " (41,\n",
              "  2,\n",
              "  'Which factors activate zygotic gene expression during the maternal-to-zygotic transition in zebrafish?',\n",
              "  'Here we show that Nanog, Pou5f1 (also called Oct4) and SoxB1 regulate zygotic gene activation in zebrafish.',\n",
              "  1),\n",
              " (41,\n",
              "  3,\n",
              "  'Which factors activate zygotic gene expression during the maternal-to-zygotic transition in zebrafish?',\n",
              "  'Here we show that Nanog, Pou5f1 (also called Oct4) and SoxB1 regulate zygotic gene activation in zebrafish.',\n",
              "  1),\n",
              " (41,\n",
              "  4,\n",
              "  'Which factors activate zygotic gene expression during the maternal-to-zygotic transition in zebrafish?',\n",
              "  'We identified several hundred genes directly activated by maternal factors, constituting the first wave of zygotic transcription.',\n",
              "  1),\n",
              " (3919,\n",
              "  0,\n",
              "  'Describe SWItchMiner (SWIM)',\n",
              "  'SWIM: a computational tool to unveiling crucial nodes in complex biological networks.',\n",
              "  1),\n",
              " (3919,\n",
              "  1,\n",
              "  'Describe SWItchMiner (SWIM)',\n",
              "  'SWItchMiner (SWIM) is a wizard-like software implementation of a procedure, previously described, able to extract information contained in complex networks.',\n",
              "  1),\n",
              " (3919,\n",
              "  2,\n",
              "  'Describe SWItchMiner (SWIM)',\n",
              "  'Specifically, SWIM allows unearthing the existence of a new class of hubs, called \"fight-club hubs\", characterized by a marked negative correlation with their first nearest neighbors.',\n",
              "  1),\n",
              " (3919,\n",
              "  3,\n",
              "  'Describe SWItchMiner (SWIM)',\n",
              "  'Among them, a special subset of genes, called \"switch genes\", appears to be characterized by an unusual pattern of intra- and inter-module connections that confers them a crucial topological role, interestingly mirrored by the evidence of their clinic-biological relevance.',\n",
              "  1),\n",
              " (3919,\n",
              "  4,\n",
              "  'Describe SWItchMiner (SWIM)',\n",
              "  'Here, we applied SWIM to a large panel of cancer datasets from The Cancer Genome Atlas, in order to highlight switch genes that could be critically associated with the drastic changes in the physiological state of cells or tissues induced by the cancer development.',\n",
              "  1),\n",
              " (3919,\n",
              "  5,\n",
              "  'Describe SWItchMiner (SWIM)',\n",
              "  'We discovered that switch genes are found in all cancers we studied and they encompass protein coding genes and non-coding RNAs, recovering many known key cancer players but also many new potential biomarkers not yet characterized in cancer context.',\n",
              "  0),\n",
              " (3919,\n",
              "  6,\n",
              "  'Describe SWItchMiner (SWIM)',\n",
              "  'Furthermore, SWIM is amenable to detect switch genes in different organisms and cell conditions, with the potential to uncover important players in biologically relevant scenarios, including but not limited to human cancer.',\n",
              "  0),\n",
              " (1241,\n",
              "  0,\n",
              "  'Is poly (ADP- ribosylation) involved in transcriptional control?',\n",
              "  'Histone phosphorylation, ubiquitylation, SUMOylation and poly-ADP-ribosylation, as well as ATP-dependent nucleosome remodeling complexes, play equally pivotal roles in the maintenance of transcriptional fidelity',\n",
              "  1),\n",
              " (1241,\n",
              "  1,\n",
              "  'Is poly (ADP- ribosylation) involved in transcriptional control?',\n",
              "  'oly(ADP-ribose) polymerase-1 (PARP-1; EC 2.4.2.30) is an abundant nuclear protein that is involved in DNA repair, cell cycle control, programmed cell death and transcriptional regulation.',\n",
              "  0),\n",
              " (1241,\n",
              "  2,\n",
              "  'Is poly (ADP- ribosylation) involved in transcriptional control?',\n",
              "  'Many lines of evidence suggest that poly(ADP-ribose) polymerase-1 (Parp-1) is involved in transcriptional regulation of various genes as a coactivator or a corepressor by modulating chromatin structure',\n",
              "  1),\n",
              " (1241,\n",
              "  3,\n",
              "  'Is poly (ADP- ribosylation) involved in transcriptional control?',\n",
              "  'These results suggest that Parp-1 is required to maintain transcriptional regulation of a wide variety of genes on a genome-wide scale',\n",
              "  0),\n",
              " (1241,\n",
              "  4,\n",
              "  'Is poly (ADP- ribosylation) involved in transcriptional control?',\n",
              "  'PARP-1 was identified as a part of the mH2A1.1 nucleosome complex and was found to be associated with the Hsp70.1 promoter',\n",
              "  1),\n",
              " (1241,\n",
              "  5,\n",
              "  'Is poly (ADP- ribosylation) involved in transcriptional control?',\n",
              "  'Upon heat shock, the Hsp70.1 promoter-bound PARP-1 is released to activate transcription through ADP-ribosylation of other Hsp70.1 promoter-bound proteins',\n",
              "  0),\n",
              " (1241,\n",
              "  6,\n",
              "  'Is poly (ADP- ribosylation) involved in transcriptional control?',\n",
              "  'Cycloheximide-induced cells were treated with two chemical inhibitors of poly(ADP-ribose) polymerase.',\n",
              "  1),\n",
              " (1241,\n",
              "  7,\n",
              "  'Is poly (ADP- ribosylation) involved in transcriptional control?',\n",
              "  '3-Aminobenzamide inhibited 75% of PAP gene induction and 4-hydroxyquinazolone, the highly specific inhibitor of the enzyme, blocked almost completely PAP expression, suggesting that ADP-ribosylation was indeed required for the upregulation of PAP gene expression by cycloheximide',\n",
              "  0),\n",
              " (1241,\n",
              "  8,\n",
              "  'Is poly (ADP- ribosylation) involved in transcriptional control?',\n",
              "  ' inhibitors of poly(ADP-ribose) polymerase suppressed UV-induced HIV-1 gene expression but not tat-mediated expression',\n",
              "  1),\n",
              " (1241,\n",
              "  9,\n",
              "  'Is poly (ADP- ribosylation) involved in transcriptional control?',\n",
              "  'oly(ADP-ribose) polymerase inhibitors suppress UV-induced human immunodeficiency virus type 1 gene expression',\n",
              "  0),\n",
              " (1110,\n",
              "  0,\n",
              "  'Describe Malgaigne fracture.',\n",
              "  'Fractures of the transverse processes in the lumbar vertebrae occur as the result of major forces such as direct blunt trauma, violent lateral flexion-extension forces, avulsion of the psoas muscle, or Malgaigne fractures of the pelvis.',\n",
              "  0),\n",
              " (1110,\n",
              "  1,\n",
              "  'Describe Malgaigne fracture.',\n",
              "  'The anteroposterior radiographic view revealed ununited fractures at the left superior and inferior pubic ramus, noted as a type I Malgaigne fracture.',\n",
              "  0),\n",
              " (1110,\n",
              "  2,\n",
              "  'Describe Malgaigne fracture.',\n",
              "  'Posterior urethral injuries occur more commonly with vertically applied forces, which typically create Malgaigne-type fractures.',\n",
              "  0),\n",
              " (1110,\n",
              "  3,\n",
              "  'Describe Malgaigne fracture.',\n",
              "  'Patients with bilateral pubic rami fractures or Malgaigne fractures were particularly prone to additional injuries; therefore, abdominal CT examinations are recommended in these patients.',\n",
              "  1),\n",
              " (1110,\n",
              "  4,\n",
              "  'Describe Malgaigne fracture.',\n",
              "  'Pelvic fractures, especially those of a Malgaigne type, frequently occur in patients who are polytraumatized.',\n",
              "  1),\n",
              " (1110,\n",
              "  5,\n",
              "  'Describe Malgaigne fracture.',\n",
              "  'The authours report the case of a 27-year-old man who had suffered a severe polytrauma with blunt thoracic injury, fracture of the lumbar spine, Malgaigne-type fracture of the pelvis and fracture of the femoral shaft on the right side, 10 years before.',\n",
              "  0),\n",
              " (1110,\n",
              "  6,\n",
              "  'Describe Malgaigne fracture.',\n",
              "  'The paper deals with the displacement of the uterus observed during management of Malgaigne type pelvic fracture in a 33 years old female.',\n",
              "  0),\n",
              " (1110,\n",
              "  7,\n",
              "  'Describe Malgaigne fracture.',\n",
              "  'This problem was first described by Malgaigne in 1857 in his classic article on double vertical fractures of the pelvis, in which he described a 34-year-old woman with a pelvic fracture who subsequently died during childbirth.',\n",
              "  0),\n",
              " (1110,\n",
              "  8,\n",
              "  'Describe Malgaigne fracture.',\n",
              "  'Eleven cases of sacroiliac dislocation and/or fracture (Malgaigne pattern) were successfully reduced and stabilized using two threaded compression rods.',\n",
              "  0),\n",
              " (1110,\n",
              "  9,\n",
              "  'Describe Malgaigne fracture.',\n",
              "  'We compared the mechanical performance of the Orthofix pelvic external fixator with that of the Pittsburgh triangular frame (PTF) on eight fresh-frozen cadaver pelves with experimentally created Malgaigne (double-vertical) fracture/dislocations.',\n",
              "  0),\n",
              " (1110,\n",
              "  10,\n",
              "  'Describe Malgaigne fracture.',\n",
              "  'A 48-year-old woman developed symptomatic superior and inferior pubic rami fractures with a concomitant subluxation of the ipsilateral sacroiliac joint three years after pelvic irradiation for a gynecologic malignancy.',\n",
              "  1),\n",
              " (1110,\n",
              "  11,\n",
              "  'Describe Malgaigne fracture.',\n",
              "  'Twenty-six of the thirty-two cadavera that were examined radiographically and dissected had a double break in the pelvic ring (Malgaigne pattern).',\n",
              "  0),\n",
              " (1110,\n",
              "  12,\n",
              "  'Describe Malgaigne fracture.',\n",
              "  ' Initial hypotension, Malgaigne and bilateral pubic rami fractures, and blood requirements exceeding 2,000 cc were suggestive of abdominal trauma.',\n",
              "  1),\n",
              " (1110,\n",
              "  13,\n",
              "  'Describe Malgaigne fracture.',\n",
              "  ' Malgaigne-type fractures with sacroiliac disruptions were created in four cadaver pelvises.',\n",
              "  1),\n",
              " (1110,\n",
              "  14,\n",
              "  'Describe Malgaigne fracture.',\n",
              "  'The authors report 7 cases involving the surgical treatment of unstable pelvic ring fractures (Malgaigne type) using two compression bars, as described by Tile.',\n",
              "  0),\n",
              " (1110,\n",
              "  15,\n",
              "  'Describe Malgaigne fracture.',\n",
              "  'Malgaigne fracture is referred to pelvic ring disruption at two sites, and is often treated surgically for its instability.',\n",
              "  0),\n",
              " (1110,\n",
              "  16,\n",
              "  'Describe Malgaigne fracture.',\n",
              "  'Sacral fractures are generally accompanied by pelvic ring fractures.',\n",
              "  0),\n",
              " (1110,\n",
              "  17,\n",
              "  'Describe Malgaigne fracture.',\n",
              "  'Malgaigne fracture is referred to pelvic ring disruption at two sites, and is often treated surgically for its instability.',\n",
              "  0),\n",
              " (1110,\n",
              "  18,\n",
              "  'Describe Malgaigne fracture.',\n",
              "  'This problem was first described by Malgaigne in 1857 in his classic article on double vertical fractures of the pelvis, in which he described a 34-year-old woman with a pelvic fracture who subsequently died during childbirth',\n",
              "  0),\n",
              " (1110,\n",
              "  19,\n",
              "  'Describe Malgaigne fracture.',\n",
              "  'The authors report 7 cases involving the surgical treatment of unstable pelvic ring fractures (Malgaigne type) using two compression bars, as described by Tile',\n",
              "  0),\n",
              " (1110,\n",
              "  20,\n",
              "  'Describe Malgaigne fracture.',\n",
              "  'Another less frequently reported complication of pelvic fractures is the potential association with pregnancy and subsequent childbirth.',\n",
              "  0),\n",
              " (1110,\n",
              "  21,\n",
              "  'Describe Malgaigne fracture.',\n",
              "  'This problem was first described by Malgaigne in 1857 in his classic article on double vertical fractures of the pelvis, in which he described a 34-year-old woman with a pelvic fracture who subsequently died during childbirth.',\n",
              "  0),\n",
              " (1110,\n",
              "  22,\n",
              "  'Describe Malgaigne fracture.',\n",
              "  'This problem was first described by Malgaigne in 1857 in his classic article on double vertical fractures of the pelvis, in which he described a 34-year-old woman with a pelvic fracture who subsequently died during childbirth.',\n",
              "  0),\n",
              " (1110,\n",
              "  23,\n",
              "  'Describe Malgaigne fracture.',\n",
              "  'A review of the literature reveals numerous articles addressing potential problems with parturition subsequent to a pelvic fracture and several articles dealing with pelvic fractures occurring during pregnancy.',\n",
              "  0),\n",
              " (1110,\n",
              "  24,\n",
              "  'Describe Malgaigne fracture.',\n",
              "  'The authors report 7 cases involving the surgical treatment of unstable pelvic ring fractures (Malgaigne type) using two compression bars, as described by Tile.',\n",
              "  0),\n",
              " (1110,\n",
              "  25,\n",
              "  'Describe Malgaigne fracture.',\n",
              "  'This problem was first described by Malgaigne in 1857 in his classic article on double vertical fractures of the pelvis, in which he described a 34-year-old woman with a pelvic fracture who subsequently died during childbirth.',\n",
              "  0),\n",
              " (3477,\n",
              "  0,\n",
              "  'How is transcriptional elongation affected by nucleosome positioning?',\n",
              "  'Nucleosome arrays inhibit both initiation and elongation of transcripts',\n",
              "  1),\n",
              " (3477,\n",
              "  1,\n",
              "  'How is transcriptional elongation affected by nucleosome positioning?',\n",
              "  ' Both transcript initiation and elongation were inhibited, the extent of the inhibition being directly proportional to the number of nucleosome cores reconstituted onto the pT207-18 DNA templates.',\n",
              "  1),\n",
              " (3477,\n",
              "  2,\n",
              "  'How is transcriptional elongation affected by nucleosome positioning?',\n",
              "  'neither partial nor complete dissociation of the histone octamer is essential for transcription elongation through arrays of nucleosome cores in vitro.',\n",
              "  0),\n",
              " (3477,\n",
              "  3,\n",
              "  'How is transcriptional elongation affected by nucleosome positioning?',\n",
              "  'The high affinity of histones for single-stranded DNA observed in titration experiments performed using the purified (+) and (-) strands of the NX1 fragment suggests that nucleosome dissociation is not due to the formation of segments of single-stranded DNA by RNA polymerase in the elongation process',\n",
              "  0),\n",
              " (3477,\n",
              "  4,\n",
              "  'How is transcriptional elongation affected by nucleosome positioning?',\n",
              "  'These effects are especially pronounced on chromatin templates, where both transcription initiation and elongation are virtually halted.',\n",
              "  1),\n",
              " (3477,\n",
              "  5,\n",
              "  'How is transcriptional elongation affected by nucleosome positioning?',\n",
              "  'The inhibition of transcription elongation appears to result from a dramatic increase in premature termination of transcripts.',\n",
              "  0),\n",
              " (3477,\n",
              "  6,\n",
              "  'How is transcriptional elongation affected by nucleosome positioning?',\n",
              "  'By using this system, we show that BRG1, the enzymatic motor of the SWI-SNF chromatin-remodelling complex, is recruited to the positioned nucleosome in a transcription elongation-dependent manner and facilitates traversal of the nucleosome by RNAPII.',\n",
              "  1),\n",
              " (3477,\n",
              "  7,\n",
              "  'How is transcriptional elongation affected by nucleosome positioning?',\n",
              "  'in order to elongate their products, both DNA and RNA polymerases must be able to overcome the inhibition presented by chromatin',\n",
              "  1),\n",
              " (3477,\n",
              "  8,\n",
              "  'How is transcriptional elongation affected by nucleosome positioning?',\n",
              "  ' We suggest a three-step model in which nucleosome remodelers, general transcription factors, and the transcriptional elongation machinery are primarily involved in generating the nucleosome positioning pattern in\\xa0vivo.',\n",
              "  0),\n",
              " (3477,\n",
              "  9,\n",
              "  'How is transcriptional elongation affected by nucleosome positioning?',\n",
              "  'The independent effect of lack of TFIIS on nucleosome occupancy and fuzziness supports the existence of alternative chromatin dynamics during transcription elongation.',\n",
              "  0),\n",
              " (3812,\n",
              "  0,\n",
              "  'Can secondary glioblastoma be caused by brain irradiation?',\n",
              "  'Prolonged survival in secondary glioblastoma following local injection of targeted alpha therapy with',\n",
              "  1),\n",
              " (3812,\n",
              "  1,\n",
              "  'Can secondary glioblastoma be caused by brain irradiation?',\n",
              "  '[Radiation induced glioblastoma: a case report].',\n",
              "  0),\n",
              " (3812,\n",
              "  2,\n",
              "  'Can secondary glioblastoma be caused by brain irradiation?',\n",
              "  'We report a surgical case of a 54-year-old woman with a radiation induced glioblastoma.',\n",
              "  0),\n",
              " (3812,\n",
              "  3,\n",
              "  'Can secondary glioblastoma be caused by brain irradiation?',\n",
              "  'Glioblastoma multiforme following cranial irradiation and chemotherapy for acute lymphocytic leukaemia.',\n",
              "  0),\n",
              " (3812,\n",
              "  4,\n",
              "  'Can secondary glioblastoma be caused by brain irradiation?',\n",
              "  'The occurrence of glioblastoma multiforme following radiation and chemotherapy in acute lymphocytic leukaemia (ALL) is rare.',\n",
              "  0),\n",
              " (3812,\n",
              "  5,\n",
              "  'Can secondary glioblastoma be caused by brain irradiation?',\n",
              "  'Glioblastoma multiforme following cranial irradiation and chemotherapy for acute lymphocytic leukaemia.',\n",
              "  0),\n",
              " (3812,\n",
              "  6,\n",
              "  'Can secondary glioblastoma be caused by brain irradiation?',\n",
              "  'R',\n",
              "  0),\n",
              " (3812,\n",
              "  7,\n",
              "  'Can secondary glioblastoma be caused by brain irradiation?',\n",
              "  'exact cause for the development of glioblastoma multiforme following therapy for ALL is not clear.',\n",
              "  1),\n",
              " (3812,\n",
              "  8,\n",
              "  'Can secondary glioblastoma be caused by brain irradiation?',\n",
              "  'A gen',\n",
              "  0),\n",
              " (3812,\n",
              "  9,\n",
              "  'Can secondary glioblastoma be caused by brain irradiation?',\n",
              "  'ndary malignant and benign brain tumors such as astrocytoma, meningioma and glioblastoma have been described in long-term survivors of conventional myeloablative alloBMT.',\n",
              "  0),\n",
              " (3812,\n",
              "  10,\n",
              "  'Can secondary glioblastoma be caused by brain irradiation?',\n",
              "  'Here w',\n",
              "  0),\n",
              " (3812,\n",
              "  11,\n",
              "  'Can secondary glioblastoma be caused by brain irradiation?',\n",
              "  'The authors consider irradiation-induced glioblastomas secondary to primarily verified medulloblastomas in patients who had previously undergone craniospinal irradiation as a component of combined treatment after tumor resection.',\n",
              "  0),\n",
              " (3812,\n",
              "  12,\n",
              "  'Can secondary glioblastoma be caused by brain irradiation?',\n",
              "  'The authors analyzed patterns of occurrence of irradiation-induced glioblastomas depending on the molecular genetic group and clinical characteristics of patients after primary surgery.',\n",
              "  0),\n",
              " (3812,\n",
              "  13,\n",
              "  'Can secondary glioblastoma be caused by brain irradiation?',\n",
              "  'Secondary brain tumors rarely arise after cranial irradiation; among them, meningiomas and glioblastomas are the most common and secondary oligodendroglial tumors the most rare.',\n",
              "  1),\n",
              " (3812,\n",
              "  14,\n",
              "  'Can secondary glioblastoma be caused by brain irradiation?',\n",
              "  'Secondary glioblastoma multiforme (sGBM) can occur after a long latency period following radiation treatment of various diseases including brain tumors, leukemia, and more benign disorders like tinea capitis.',\n",
              "  1),\n",
              " (3812,\n",
              "  15,\n",
              "  'Can secondary glioblastoma be caused by brain irradiation?',\n",
              "  'Irradiation, however, acts as an oncogenic factor as a delayed effect and it is rare that glioblastoma multiforme develops during the remission period of ALL.',\n",
              "  0),\n",
              " ...]"
            ]
          },
          "metadata": {},
          "execution_count": 13
        }
      ],
      "source": [
        "devtest_set = []\n",
        "for d in q_devtest:\n",
        "    document_data = dataset[dataset['question'] == d]\n",
        "    devtest_set += list(document_data.itertuples(index=False, name=None))\n",
        "devtest_set"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 14,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "E0G0hmLOOC5i",
        "outputId": "006a3f3e-6972-4095-c828-6170cdba6bd1"
      },
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "[(269,\n",
              "  0,\n",
              "  'What medication were compared in the ROCKET AF Trial?',\n",
              "  'METHODS AND RESULTS: Patients randomized in the Rivaroxaban Once Daily Oral Direct Factor Xa Inhibition Compared With Vitamin K Antagonism for Prevention of Stroke and Embolism Trial in Atrial Fibrillation (ROCKET-AF) trial (n = 14 264) were grouped by baseline AF category: paroxysmal or persistent.',\n",
              "  0),\n",
              " (269,\n",
              "  1,\n",
              "  'What medication were compared in the ROCKET AF Trial?',\n",
              "  'Multivariable adjustment was performed to compare thrombo-embolic events, bleeding, and death between groups, in high-risk subgroups, and across treatment assignment (rivaroxaban or warfarin).',\n",
              "  0),\n",
              " (269,\n",
              "  2,\n",
              "  'What medication were compared in the ROCKET AF Trial?',\n",
              "  'This review focuses on the direct Factor Xa inhibitor rivaroxaban, summarizing the properties that make rivaroxaban appropriate for anticoagulant therapy in this indication (including its predictable pharmacokinetic and pharmacodynamic profile and once-daily dosing regimen) and describing data from the Phase III ROCKET AF trial, which showed once-daily rivaroxaban to be noninferior to warfarin for the prevention of stroke in patients with nonvalvular AF.',\n",
              "  0),\n",
              " (269,\n",
              "  3,\n",
              "  'What medication were compared in the ROCKET AF Trial?',\n",
              "  'Subanalyses of ROCKET AF data showed rivaroxaban to have consistent efficacy and safety across a wide range of patients, and studies to confirm these results in real-world settings are underway.',\n",
              "  0),\n",
              " (269,\n",
              "  4,\n",
              "  'What medication were compared in the ROCKET AF Trial?',\n",
              "  'Rivaroxaban for stroke prevention in East Asian patients from the ROCKET AF trial.',\n",
              "  0),\n",
              " (269,\n",
              "  5,\n",
              "  'What medication were compared in the ROCKET AF Trial?',\n",
              "  'CONCLUSIONS: Observed relative efficacy and safety of rivaroxaban versus warfarin were similar among patients within and outside East Asia.',\n",
              "  0),\n",
              " (269,\n",
              "  6,\n",
              "  'What medication were compared in the ROCKET AF Trial?',\n",
              "  'Rivaroxaban versus warfarin in Japanese patients with nonvalvular atrial fibrillation in relation to the CHADS2 score: a subgroup analysis of the J-ROCKET AF trial.',\n",
              "  0),\n",
              " (269,\n",
              "  7,\n",
              "  'What medication were compared in the ROCKET AF Trial?',\n",
              "  'BACKGROUND: Results from a trial of rivaroxaban versus warfarin in 1280 Japanese patients with atrial fibrillation (J-ROCKET AF) revealed that rivaroxaban was noninferior to warfarin with respect to the principal safety outcome.',\n",
              "  0),\n",
              " (269,\n",
              "  8,\n",
              "  'What medication were compared in the ROCKET AF Trial?',\n",
              "  'Rivaroxaban, an oral direct factor Xa-inhibitor was non-inferior to adjusted dose warfarin in the prevention of stroke and embolism among patients with atrial fibrillation (AF) in the ROCKET-AF trial and has been approved for stroke prevention in AF.',\n",
              "  1),\n",
              " (269,\n",
              "  9,\n",
              "  'What medication were compared in the ROCKET AF Trial?',\n",
              "  'The ROCKET AF trial compared rivaroxaban (20 mg/day; 15 mg/day in patients with creatinine clearance 30-49 ml/min) with dose-adjusted warfarin (international normalized ratio 2-3) in 14,264 patients with AF and a prior history of stroke or at least two other additional risk factors for stroke.',\n",
              "  0),\n",
              " (269,\n",
              "  10,\n",
              "  'What medication were compared in the ROCKET AF Trial?',\n",
              "  'Factors associated with major bleeding events: insights from the ROCKET AF trial (rivaroxaban once-daily oral direct factor Xa inhibition compared with vitamin K antagonism for prevention of stroke and embolism trial in atrial fibrillation).',\n",
              "  1),\n",
              " (269,\n",
              "  11,\n",
              "  'What medication were compared in the ROCKET AF Trial?',\n",
              "  'In Rivaroxaban Once Daily Oral Direct Factor Xa Inhibitor Compared With Vitamin K Antagonism for Prevention of Stroke and Embolism Trial in Atrial Fibrillation (ROCKET AF) trial, rivaroxaban was noninferior to dose-adjusted warfarin in preventing stroke or systemic embolism among patients with nonvalvular atrial fibrillation at moderate to high stroke risk.',\n",
              "  0),\n",
              " (269,\n",
              "  12,\n",
              "  'What medication were compared in the ROCKET AF Trial?',\n",
              "  'Different clinical trials have shown the benefits of new oral anticoagulants over warfarin, but patients included in the ROCKET-AF trial were found to be at a higher risk of AF-related complications.',\n",
              "  0),\n",
              " (269,\n",
              "  13,\n",
              "  'What medication were compared in the ROCKET AF Trial?',\n",
              "  'The ROCKET AF trial demonstrated the noninferiority of rivaroxaban compared with warfarin for the prevention of stroke and systemic embolism, with a similar rate of major bleeding and a substantial reduction in intracranial hemorrhage.',\n",
              "  1),\n",
              " (269,\n",
              "  14,\n",
              "  'What medication were compared in the ROCKET AF Trial?',\n",
              "  'Rivaroxaban for stroke prevention in atrial fibrillation: a critical review of the ROCKET AF trial.',\n",
              "  0),\n",
              " (269,\n",
              "  15,\n",
              "  'What medication were compared in the ROCKET AF Trial?',\n",
              "  'Outcomes after cardioversion and atrial fibrillation ablation in patients treated with rivaroxaban and warfarin in the ROCKET AF trial.',\n",
              "  0),\n",
              " (269,\n",
              "  16,\n",
              "  'What medication were compared in the ROCKET AF Trial?',\n",
              "  'The overall analysis of the rivaroxaban versus warfarin in Japanese patients with atrial fibrillation (J-ROCKET AF) trial revealed that rivaroxaban was not inferior to warfarin with respect to the primary safety outcome.',\n",
              "  0),\n",
              " (269,\n",
              "  17,\n",
              "  'What medication were compared in the ROCKET AF Trial?',\n",
              "  'Medical costs in the US of clinical events associated with oral anticoagulant (OAC) use compared to warfarin among non-valvular atrial fibrillation patients ≥75 and <75 years of age, based on the ARISTOTLE, RE-LY, and ROCKET-AF trials.',\n",
              "  0),\n",
              " (269,\n",
              "  18,\n",
              "  'What medication were compared in the ROCKET AF Trial?',\n",
              "  'In patients at risk of stroke due to atrial fibrillation, rivaroxaban was noninferior compared to warfarin in preventing stroke/systemic embolism in the Rivaroxaban Once Daily Oral Direct Factor Xa Inhibition Compared with Vitamin K Antagonism for Prevention of Stroke and Embolism Trial in Atrial Fibrillation (ROCKET-AF) trial and was associated with a similar risk of major bleeding; the incidence of intracranial hemorrhage was 33% lower with rivaroxaban',\n",
              "  0),\n",
              " (269,\n",
              "  19,\n",
              "  'What medication were compared in the ROCKET AF Trial?',\n",
              "  'Outcomes of discontinuing rivaroxaban compared with warfarin in patients with nonvalvular atrial fibrillation: analysis from the ROCKET AF trial (Rivaroxaban Once-Daily, Oral, Direct Factor Xa Inhibition Compared With Vitamin K Antagonism for Prevention of Stroke and Embolism Trial in Atrial Fibrillation)',\n",
              "  1),\n",
              " (269,\n",
              "  20,\n",
              "  'What medication were compared in the ROCKET AF Trial?',\n",
              "  'The Phase III ROCKET AF (Rivaroxaban Once-Daily Oral Direct Factor Xa Inhibition Compared with Vitamin K Antagonism for Prevention of Stroke and Embolism Trial in Atrial Fibrillation) trial showed that rivaroxaban, an oral, direct Factor Xa inhibitor, was noninferior to warfarin for the reduction of stroke or systemic embolism in patients with AF',\n",
              "  0),\n",
              " (269,\n",
              "  21,\n",
              "  'What medication were compared in the ROCKET AF Trial?',\n",
              "  'In Rivaroxaban Once Daily Oral Direct Factor Xa Inhibitor Compared With Vitamin K Antagonism for Prevention of Stroke and Embolism Trial in Atrial Fibrillation (ROCKET AF) trial, rivaroxaban was noninferior to dose-adjusted warfarin in preventing stroke or systemic embolism among patients with nonvalvular atrial fibrillation at moderate to high stroke risk',\n",
              "  0),\n",
              " (269,\n",
              "  22,\n",
              "  'What medication were compared in the ROCKET AF Trial?',\n",
              "  'We investigated clinical characteristics and outcomes of patients with significant valvular disease (SVD) in the Rivaroxaban Once Daily Oral Direct Factor Xa Inhibition Compared with Vitamin K Antagonism for Prevention of Stroke and Embolism Trial in Atrial Fibrillation (ROCKET AF) trial',\n",
              "  0),\n",
              " (269,\n",
              "  23,\n",
              "  'What medication were compared in the ROCKET AF Trial?',\n",
              "  'Efficacy and safety of rivaroxaban compared with warfarin among elderly patients with nonvalvular atrial fibrillation in the Rivaroxaban Once Daily, Oral, Direct Factor Xa Inhibition Compared With Vitamin K Antagonism for Prevention of Stroke and Embolism Trial in Atrial Fibrillation (ROCKET AF)',\n",
              "  0),\n",
              " (269,\n",
              "  24,\n",
              "  'What medication were compared in the ROCKET AF Trial?',\n",
              "  'The oral factor Xa inhibitor rivaroxaban was noninferior to warfarin in the Rivaroxaban Once Daily, Oral, Direct Factor Xa Inhibition Compared With Vitamin K Antagonism for Prevention of Stroke and Embolism Trial in Atrial Fibrillation (ROCKET AF)',\n",
              "  0),\n",
              " (269,\n",
              "  25,\n",
              "  'What medication were compared in the ROCKET AF Trial?',\n",
              "  'We studied the use and outcomes of digoxin in patients in the Rivaroxaban Once Daily Oral Direct Factor Xa Inhibition Compared with Vitamin K Antagonism for Prevention of Stroke and Embolism Trial in Atrial Fibrillation (ROCKET AF)',\n",
              "  0),\n",
              " (269,\n",
              "  26,\n",
              "  'What medication were compared in the ROCKET AF Trial?',\n",
              "  'Outcomes of discontinuing rivaroxaban compared with warfarin in patients with nonvalvular atrial fibrillation: analysis from the ROCKET AF trial (Rivaroxaban Once-Daily, Oral, Direct Factor Xa Inhibition Compared With Vitamin K Antagonism for Prevention of Stroke and Embolism Trial in Atrial Fibrillation).',\n",
              "  1),\n",
              " (269,\n",
              "  27,\n",
              "  'What medication were compared in the ROCKET AF Trial?',\n",
              "  'Relationship between time in therapeutic range and comparative treatment effect of rivaroxaban and warfarin: results from the ROCKET AF trial.',\n",
              "  0),\n",
              " (269,\n",
              "  28,\n",
              "  'What medication were compared in the ROCKET AF Trial?',\n",
              "  'Efficacy and safety of rivaroxaban compared with warfarin among elderly patients with nonvalvular atrial fibrillation in the Rivaroxaban Once Daily, Oral, Direct Factor Xa Inhibition Compared With Vitamin K Antagonism for Prevention of Stroke and Embolism Trial in Atrial Fibrillation (ROCKET AF).',\n",
              "  0),\n",
              " (269,\n",
              "  29,\n",
              "  'What medication were compared in the ROCKET AF Trial?',\n",
              "  'Outcomes of temporary interruption of rivaroxaban compared with warfarin in patients with nonvalvular atrial fibrillation: results from the rivaroxaban once daily, oral, direct factor Xa inhibition compared with vitamin K antagonism for prevention of stroke and embolism trial in atrial fibrillation (ROCKET AF).',\n",
              "  0),\n",
              " (269,\n",
              "  30,\n",
              "  'What medication were compared in the ROCKET AF Trial?',\n",
              "  ' Health care providers should consider renal function, concomitant medication, polymorbidity and age prior to prescribing rivaroxaban.',\n",
              "  0),\n",
              " (269,\n",
              "  31,\n",
              "  'What medication were compared in the ROCKET AF Trial?',\n",
              "  'Care has to be taken when prescribing rivaroxaban to patients who are different from those included in the ROCKET AF trial.',\n",
              "  0),\n",
              " (269,\n",
              "  32,\n",
              "  'What medication were compared in the ROCKET AF Trial?',\n",
              "  'Health care providers should consider renal function, concomitant medication, polymorbidity and age prior to prescribing rivaroxaban.',\n",
              "  0),\n",
              " (269,\n",
              "  33,\n",
              "  'What medication were compared in the ROCKET AF Trial?',\n",
              "  'Care has to be taken when prescribing rivaroxaban to patients who are different from those included in the ROCKET AF trial.',\n",
              "  0),\n",
              " (269,\n",
              "  34,\n",
              "  'What medication were compared in the ROCKET AF Trial?',\n",
              "  'A separate trial, J-ROCKET AF, compared the safety of a Japan-specific rivaroxaban dose with warfarin administered according to Japanese guidelines in Japanese patients with AF.J-ROCKET AF was a prospective, randomized, double-blind, phase III trial.',\n",
              "  0),\n",
              " (269,\n",
              "  35,\n",
              "  'What medication were compared in the ROCKET AF Trial?',\n",
              "  'A separate trial, J-ROCKET AF, compared the safety of a Japan-specific rivaroxaban dose with warfarin administered according to Japanese guidelines in Japanese patients with AF.',\n",
              "  0),\n",
              " (269,\n",
              "  36,\n",
              "  'What medication were compared in the ROCKET AF Trial?',\n",
              "  'J-ROCKET AF was a prospective, randomized, double-blind, phase III trial.',\n",
              "  0),\n",
              " (956,\n",
              "  0,\n",
              "  'List programs suitable for pharmacophore modelling',\n",
              "  'Pharmacophore models based on thermolysin binding modes and activity profiles were generated using the LigandScout program',\n",
              "  0),\n",
              " (956,\n",
              "  1,\n",
              "  'List programs suitable for pharmacophore modelling',\n",
              "  'In this study, pharmacophore hypotheses based on known inhibitors were generated using common feature pharmacophore generation protocol available in Discovery Studio program',\n",
              "  0),\n",
              " (956,\n",
              "  2,\n",
              "  'List programs suitable for pharmacophore modelling',\n",
              "  ' A pharmacophore-based screening was then carried out using the program Catalyst',\n",
              "  0),\n",
              " (956,\n",
              "  3,\n",
              "  'List programs suitable for pharmacophore modelling',\n",
              "  'The PharmaGist employed algorithm to identify the best pharmacophores by computing multiple flexible alignments between the input ligands.',\n",
              "  0),\n",
              " (956,\n",
              "  4,\n",
              "  'List programs suitable for pharmacophore modelling',\n",
              "  'The multiple alignments are generated by combining alignments pair-wise between one of the gliptin input ligands, which acts as pivot and the other gliptin as ligand.',\n",
              "  0),\n",
              " (956,\n",
              "  5,\n",
              "  'List programs suitable for pharmacophore modelling',\n",
              "  'The resulting multiple alignments reveal spatial arrangements of consensus features shared by different subsets of input ligands.',\n",
              "  0),\n",
              " (956,\n",
              "  6,\n",
              "  'List programs suitable for pharmacophore modelling',\n",
              "  'The best pharmacophore model has been derived using both pair-wise and multiple alignment methods, which have been weighted in Pharmacophore Generation process',\n",
              "  0),\n",
              " (956,\n",
              "  7,\n",
              "  'List programs suitable for pharmacophore modelling',\n",
              "  'Pharmacophore models of c-Met kinase inhibitors have been developed based on 22 diverse compounds by using HypoGen algorithm implemented in Discovery studio program package.',\n",
              "  0),\n",
              " (956,\n",
              "  8,\n",
              "  'List programs suitable for pharmacophore modelling',\n",
              "  'chemical feature based pharmacophore models of MMP-1, MMP-8 and MMP-13 inhibitors have been developed with the aid of HypoGen module within Catalyst program package',\n",
              "  0),\n",
              " (956,\n",
              "  9,\n",
              "  'List programs suitable for pharmacophore modelling',\n",
              "  'Genetic Algorithm Similarity Program (GASP) was used to derive a 3D pharmacophore model which was used in effective alignment of data set.',\n",
              "  1),\n",
              " (956,\n",
              "  10,\n",
              "  'List programs suitable for pharmacophore modelling',\n",
              "  'Active site complimenting structure-based pharmacophore models were developed using Discovery Studio 2.5 program and validated using a dataset of known HDAC8 inhibitors.',\n",
              "  0),\n",
              " (956,\n",
              "  11,\n",
              "  'List programs suitable for pharmacophore modelling',\n",
              "  ' pharmacophore model has been generated from the training set by means of the MOE (molecular operating environment) program.',\n",
              "  1),\n",
              " (956,\n",
              "  12,\n",
              "  'List programs suitable for pharmacophore modelling',\n",
              "  'A training set of 20 compounds with known experimental activity was used to generate pharmacophore hypotheses using 3D QSAR Pharmacophore Generation module available in Discovery studio.',\n",
              "  0),\n",
              " (956,\n",
              "  13,\n",
              "  'List programs suitable for pharmacophore modelling',\n",
              "  'Pharmacophore model was developed for the first time, on a training data set of 22 CCR3 antagonists, using CATALYST HypoRefine program.',\n",
              "  0),\n",
              " (956,\n",
              "  14,\n",
              "  'List programs suitable for pharmacophore modelling',\n",
              "  'The CatalystHipHop approach was used to generate a pharmacophore model for cyclooxygenase-2 (COX-2) inhibitors based on a training set of 15 active inhibitors (1â15).',\n",
              "  1),\n",
              " (956,\n",
              "  15,\n",
              "  'List programs suitable for pharmacophore modelling',\n",
              "  ' Molecular modeling study, including fitting to a 3D-pharmacophore model of the designed molecules (10a-g), with HCV NS3 protease hypothesis using catalyst program was fulfilled.',\n",
              "  1),\n",
              " (956,\n",
              "  16,\n",
              "  'List programs suitable for pharmacophore modelling',\n",
              "  'pharmacophore model (Hypo1) with a well prediction capacity for CysLT(1) antagonists was developed using Catalyst/HypoGen program.',\n",
              "  0),\n",
              " (956,\n",
              "  17,\n",
              "  'List programs suitable for pharmacophore modelling',\n",
              "  'Three pharmacophore models of opioid agonists were generated by the Catalyst/HypoGen program',\n",
              "  0),\n",
              " (956,\n",
              "  18,\n",
              "  'List programs suitable for pharmacophore modelling',\n",
              "  'The superposition of the ligands was performed with the tool Pharmacophore Elucidation (MOE-package',\n",
              "  0),\n",
              " (956,\n",
              "  19,\n",
              "  'List programs suitable for pharmacophore modelling',\n",
              "  'Known PDE-5 inhibitors were used to construct a three dimensional quantitative structure-activity relationship (3D QSAR) model by HypoGen program.',\n",
              "  0),\n",
              " (956,\n",
              "  20,\n",
              "  'List programs suitable for pharmacophore modelling',\n",
              "  'pharmacophore models were generated for AT1 and ET(A) receptors based on highly selective AT1 and ET(A) antagonists using the program Catalyst/HipHop',\n",
              "  0),\n",
              " (956,\n",
              "  21,\n",
              "  'List programs suitable for pharmacophore modelling',\n",
              "  ' In this study, chemical feature based pharmacophore models of inhibitors of 5-LOX have been developed with the aid of HipHop and HypoGen modules within Catalyst program package',\n",
              "  1),\n",
              " (956,\n",
              "  22,\n",
              "  'List programs suitable for pharmacophore modelling',\n",
              "  'A data set consisting of 24 inhibitors was selected on the basis of the information content of the structures and activity data as required by the Catalyst/HypoGen program.',\n",
              "  0),\n",
              " (1193,\n",
              "  0,\n",
              "  'What is the effect of thapsigargin treatment?',\n",
              "  ' the sarcoplasmic/endoplasmic Ca(2+)-ATPase (SERCA) inhibitor thapsigargin',\n",
              "  1),\n",
              " (1193,\n",
              "  1,\n",
              "  'What is the effect of thapsigargin treatment?',\n",
              "  'Treatment of endoplasmic stress inducers, thapsigargin',\n",
              "  1),\n",
              " (1193,\n",
              "  2,\n",
              "  'What is the effect of thapsigargin treatment?',\n",
              "  'the two unrelated reversible autophagy inhibitors 3-methyladenine (3MA) and thapsigargin (TG) both blocked cargo sequestration completely.',\n",
              "  0),\n",
              " (1193,\n",
              "  3,\n",
              "  'What is the effect of thapsigargin treatment?',\n",
              "  'pretreatment with the endoplasmic reticulum Ca(2+) pump inhibitor thapsigargin (TG)',\n",
              "  1),\n",
              " (1193,\n",
              "  4,\n",
              "  'What is the effect of thapsigargin treatment?',\n",
              "  ' pharmacological ER stress agents (thapsigargin and tunicamycin).',\n",
              "  0),\n",
              " (1193,\n",
              "  5,\n",
              "  'What is the effect of thapsigargin treatment?',\n",
              "  ' thapsigargin-induced calcium depletion of sarcoplasmic reticular stores',\n",
              "  0),\n",
              " (1193,\n",
              "  6,\n",
              "  'What is the effect of thapsigargin treatment?',\n",
              "  'Addition of thapsigargin (Tg), that induces store-depletion and activates TRPC1-mediated Ca(2+) entry, potentiated the Cl(-) current, which was inhibited by the addition of a non-specific TRPC channel blocker SKF96365 or removal of external Ca(2+).',\n",
              "  0),\n",
              " (1193,\n",
              "  7,\n",
              "  'What is the effect of thapsigargin treatment?',\n",
              "  'we demonstrate that thapsigargin (TG) and tunicamycin (TM), two ERS inducers',\n",
              "  0),\n",
              " (1193,\n",
              "  8,\n",
              "  'What is the effect of thapsigargin treatment?',\n",
              "  'hapsigargin, a sarco/endoplasmic reticulum Ca(2+)-ATPase (SERCA) Ca(2+) pump inhibitor',\n",
              "  1),\n",
              " (1193,\n",
              "  9,\n",
              "  'What is the effect of thapsigargin treatment?',\n",
              "  'Thapsigargin-induced ER stress',\n",
              "  1),\n",
              " (1193,\n",
              "  10,\n",
              "  'What is the effect of thapsigargin treatment?',\n",
              "  'SERCA2a inhibitor (thapsigargin)',\n",
              "  0),\n",
              " (1193,\n",
              "  11,\n",
              "  'What is the effect of thapsigargin treatment?',\n",
              "  'he ER stressors, thapsigargin',\n",
              "  0),\n",
              " (445,\n",
              "  0,\n",
              "  'Is there any relationship between histone ubiquitylation and splicing?',\n",
              "  'histone H2B-specific deubiquitinase and demonstrate that H2B deubiquitination by USP49 is required for efficient cotranscriptional splicing of a large set of exons.',\n",
              "  1),\n",
              " (445,\n",
              "  1,\n",
              "  'Is there any relationship between histone ubiquitylation and splicing?',\n",
              "  'H2B monoubiquitylation (H2BK123ub1) marks introns in Saccharomyces cerevisiae',\n",
              "  1),\n",
              " (445,\n",
              "  2,\n",
              "  'Is there any relationship between histone ubiquitylation and splicing?',\n",
              "  'H2B ubiquitination by facilitating splicing',\n",
              "  1),\n",
              " (445,\n",
              "  3,\n",
              "  'Is there any relationship between histone ubiquitylation and splicing?',\n",
              "  'pre-mRNA splicing plays a critical role in histone H2B ubiquitination',\n",
              "  1),\n",
              " (445,\n",
              "  4,\n",
              "  'Is there any relationship between histone ubiquitylation and splicing?',\n",
              "  'unanticipated functional link between histone H2B ubiquitination and pre-mRNA splicing',\n",
              "  1),\n",
              " (2566,\n",
              "  0,\n",
              "  'What is the role of tankyrases in response to Double Strand Breaks (DSBs)?',\n",
              "  'Tankyrases Promote Homologous Recombination and Check Point Activation in Response to DSBs.',\n",
              "  1),\n",
              " (2566,\n",
              "  1,\n",
              "  'What is the role of tankyrases in response to Double Strand Breaks (DSBs)?',\n",
              "  'Tankyrases Promote Homologous Recombination and Check Point Activation in Response to DSBs.',\n",
              "  1),\n",
              " (307,\n",
              "  0,\n",
              "  'What family do mDia proteins belong in?',\n",
              "  'mDia proteins are members of the formin family of actin nucleating proteins that polymerize linear actin filaments.',\n",
              "  1),\n",
              " (307,\n",
              "  1,\n",
              "  'What family do mDia proteins belong in?',\n",
              "  'mDia1, a linear actin nucleator of the Formin family',\n",
              "  1),\n",
              " (307,\n",
              "  2,\n",
              "  'What family do mDia proteins belong in?',\n",
              "  'the mammalian Diaphanous members of the formin family of proteins (mDia) are two key players in the formation of filopodia and neurites.',\n",
              "  1),\n",
              " (307,\n",
              "  3,\n",
              "  'What family do mDia proteins belong in?',\n",
              "  'mDia proteins are mammalian homologues of Drosophila diaphanous and belong to the formin family proteins that catalyze actin nucleation and polymerization.',\n",
              "  1),\n",
              " (307,\n",
              "  4,\n",
              "  'What family do mDia proteins belong in?',\n",
              "  ' The mDia family of formins',\n",
              "  1),\n",
              " (307,\n",
              "  5,\n",
              "  'What family do mDia proteins belong in?',\n",
              "  'interacts with the formin protein mDia1 (DIAPH1).',\n",
              "  0),\n",
              " (307,\n",
              "  6,\n",
              "  'What family do mDia proteins belong in?',\n",
              "  'Formin-family proteins, in the active state, form actin-based structures such as stress fibres.',\n",
              "  0),\n",
              " (307,\n",
              "  7,\n",
              "  'What family do mDia proteins belong in?',\n",
              "  'Their activation mechanisms, however, are largely unknown except that mDia and its closely related formins can be activated by direct binding of the small GTPase Rho or Cdc42.',\n",
              "  0),\n",
              " (307,\n",
              "  8,\n",
              "  'What family do mDia proteins belong in?',\n",
              "  'Mouse Diaphanous-related formins (mDias) are members of the formin protein family that nucleate actin polymerization and subsequently promote filamentous actin (F-actin) elongation by monomer addition to fast-growing barbed ends.',\n",
              "  0),\n",
              " (307,\n",
              "  9,\n",
              "  'What family do mDia proteins belong in?',\n",
              "  'Diaphanous related formins (DRFs) are part of the formin protein family that control morphogenesis, embryonic differentiation, cytokinesis, and cell polarity.',\n",
              "  0),\n",
              " (307,\n",
              "  10,\n",
              "  'What family do mDia proteins belong in?',\n",
              "  'Diaphanous related formins (DRFs) are cytoskeleton remodeling proteins that mediate specific upstream GTPase signals to regulate cellular processes such as cytokinesis, cell polarity, and organelle motility.',\n",
              "  0),\n",
              " (1494,\n",
              "  0,\n",
              "  'What is the role of the Ada O6-alkylguanine alkyltransferase in bacteria?',\n",
              "  'multifunctional 39 kDa Escherichia coli Ada protein (O6-methylguanine-DNA methyltransferase) (EC 2.1.1.63), product of the ada gene',\n",
              "  0),\n",
              " (1494,\n",
              "  1,\n",
              "  'What is the role of the Ada O6-alkylguanine alkyltransferase in bacteria?',\n",
              "  'The Ada protein accepts stoichiometrically the alkyl group from O6-alkylguanine in DNA at the Cys-321 residue and from alkyl phosphotriester at the Cys-69 residue.',\n",
              "  1),\n",
              " (1494,\n",
              "  2,\n",
              "  'What is the role of the Ada O6-alkylguanine alkyltransferase in bacteria?',\n",
              "  'This protein functions in DNA repair by direct dealkylation of mutagenic O6-alkylguanine.',\n",
              "  0),\n",
              " (1494,\n",
              "  3,\n",
              "  'What is the role of the Ada O6-alkylguanine alkyltransferase in bacteria?',\n",
              "  'The protein methylated at Cys-69 becomes a transcriptional activator of the genes in the ada regulon, including its own.',\n",
              "  1),\n",
              " (1494,\n",
              "  4,\n",
              "  'What is the role of the Ada O6-alkylguanine alkyltransferase in bacteria?',\n",
              "  'O6-Alkylguanine DNA-alkyltransferase (ATase) repairs toxic, mutagenic and carcinogenic O6-alkylguanine (O6-alkG) lesions in DNA',\n",
              "  0),\n",
              " (1494,\n",
              "  5,\n",
              "  'What is the role of the Ada O6-alkylguanine alkyltransferase in bacteria?',\n",
              "  'O6-Methylguanine is removed from DNA via the transfer of the methyl group to a cysteine acceptor site present in the DNA repair protein O6-alkylguanine-DNA alkyltransferase.',\n",
              "  0),\n",
              " (1494,\n",
              "  6,\n",
              "  'What is the role of the Ada O6-alkylguanine alkyltransferase in bacteria?',\n",
              "  'DNA repair by O6-alkylguanine-DNA alkyltransferases plays a major role in removing lesions responsible for GC-->AT transitions induced by CCNU, influencing their ultimate distribution with respect to sequence context.',\n",
              "  0),\n",
              " (1494,\n",
              "  7,\n",
              "  'What is the role of the Ada O6-alkylguanine alkyltransferase in bacteria?',\n",
              "  'alkyltransferase (ATase)',\n",
              "  0),\n",
              " (1494,\n",
              "  8,\n",
              "  'What is the role of the Ada O6-alkylguanine alkyltransferase in bacteria?',\n",
              "  'the ada ATase induced by exposure to low doses of a methylating agent.',\n",
              "  0),\n",
              " (1494,\n",
              "  9,\n",
              "  'What is the role of the Ada O6-alkylguanine alkyltransferase in bacteria?',\n",
              "  'The Escherichia coli Ada and Ogt DNA methyltransferases (MTases) are known to transfer simple alkyl groups from O6-alkylguanine and O4-alkylthymine, directly restoring these alkylated DNA lesions to guanine and thymine.',\n",
              "  0),\n",
              " (1494,\n",
              "  10,\n",
              "  'What is the role of the Ada O6-alkylguanine alkyltransferase in bacteria?',\n",
              "  'During exponential growth, the spontaneous rate of G:C to A:T transitions and G:C to C:G transversions was elevated about fourfold in ada ogt double mutant versus wild-type E. coli.',\n",
              "  0),\n",
              " (1494,\n",
              "  11,\n",
              "  'What is the role of the Ada O6-alkylguanine alkyltransferase in bacteria?',\n",
              "  'compared with the wild type, stationary populations of the MTase-deficient E. coli (under lactose selection) displayed increased G:C to A:T and A:T to G:C transitions (10- and 3-fold, respectively) and increased G:C to C:G, A:T to C:G, and A:T to T:A transversions (10-, 2.5-, and 1.7-fold, respectively).',\n",
              "  1),\n",
              " (1494,\n",
              "  12,\n",
              "  'What is the role of the Ada O6-alkylguanine alkyltransferase in bacteria?',\n",
              "  'ATases are able to repair O6-alkylguanine (O6-AlkG) and O4-alkylthymine (O4-AlkT)',\n",
              "  0),\n",
              " (1494,\n",
              "  13,\n",
              "  'What is the role of the Ada O6-alkylguanine alkyltransferase in bacteria?',\n",
              "  'The inducible resistance to alkylation mutagenesis and killing in Escherichia coli (the adaptive response) is controlled by the ada gene.',\n",
              "  0),\n",
              " (1494,\n",
              "  14,\n",
              "  'What is the role of the Ada O6-alkylguanine alkyltransferase in bacteria?',\n",
              "  'The Ada protein acts both as a positive regulator of the response and as a DNA repair enzyme, correcting premutagenic O6-alkylguanine in DNA by suicidal transfer of the alkyl group to one of its own cysteine residues.',\n",
              "  1),\n",
              " (1494,\n",
              "  15,\n",
              "  'What is the role of the Ada O6-alkylguanine alkyltransferase in bacteria?',\n",
              "  'Forward mutations induced by 1-(2-chloroethyl)-3-cyclohexyl-1-nitrosourea (CCNU) in the lacl gene of Escherichia coli were recovered from bacteria proficient (Ogt+ Ada+) and deficient (Ogt- Ada-) in O6-alkylguanine-DNA alkyltransferase activity.',\n",
              "  0),\n",
              " (1494,\n",
              "  16,\n",
              "  'What is the role of the Ada O6-alkylguanine alkyltransferase in bacteria?',\n",
              "  'Spontaneous mutagenesis in O6-alkylguanine-DNA alkyltransferase-proficient and -deficient (ada ogt mutants) Escherichia coli was studied in two ways: in bacteria growing in nonselective liquid medium and in bacteria resting on selective agar plates.',\n",
              "  0),\n",
              " (1494,\n",
              "  17,\n",
              "  'What is the role of the Ada O6-alkylguanine alkyltransferase in bacteria?',\n",
              "  'We now show by Southern analysis that the mutation involves a gross deletion covering at least the ogt and fnr genes and that no O6-alkylguanine-DNA-alkyltransferase activity is present in cell-free extracts of an ada::Tn10 derivative of these bacteria.',\n",
              "  0),\n",
              " (1494,\n",
              "  18,\n",
              "  'What is the role of the Ada O6-alkylguanine alkyltransferase in bacteria?',\n",
              "  'O6-Alkylguanine DNA-alkyltransferase (ATase) repairs toxic, mutagenic and carcinogenic O6-alkylguanine (O6-alkG) lesions in DNA by a highly conserved reaction involving the stoichiometric transfer of the alkyl group to the active centre cysteine residue of the ATase protein.',\n",
              "  1),\n",
              " (1494,\n",
              "  19,\n",
              "  'What is the role of the Ada O6-alkylguanine alkyltransferase in bacteria?',\n",
              "  'Although the human O6-alkylguanine-DNA alkyltransferase (AGT) is very sensitive to inactivation by O6-benzylguanine (BG) or 2,4-diamino-6-benzyloxy-5-nitrosopyrimidine (5-nitroso-BP), the equivalent protein formed by the carboxyl terminal domain of the product of the Escherichia coli ada gene (Ada-C) is unaffected by these inhibitors.',\n",
              "  0),\n",
              " (1494,\n",
              "  20,\n",
              "  'What is the role of the Ada O6-alkylguanine alkyltransferase in bacteria?',\n",
              "  'A mutant of Bacillus subtilis defective in the constitutive activity of O6-alkylguanine-DNA alkyltransferase was isolated from a strain (ada-1) deficient in the adaptive response to DNA alkylation.',\n",
              "  0),\n",
              " (1494,\n",
              "  21,\n",
              "  'What is the role of the Ada O6-alkylguanine alkyltransferase in bacteria?',\n",
              "  'Mutations in the Ada O6-alkylguanine-DNA alkyltransferase conferring sensitivity to inactivation by O6-benzylguanine and 2,4-diamino-6-benzyloxy-5-nitrosopyrimidine.',\n",
              "  0),\n",
              " (1494,\n",
              "  22,\n",
              "  'What is the role of the Ada O6-alkylguanine alkyltransferase in bacteria?',\n",
              "  'The protein O 6-alkylguanine-DNA alkyltransferase(alkyltransferase) is involved in the repair of O 6-alkylguanine and O 4-alkylthymine in DNA and plays an important role in most organisms in attenuating the cytotoxic and mutagenic effects of certain classes of alkylating agents.',\n",
              "  0),\n",
              " (1494,\n",
              "  23,\n",
              "  'What is the role of the Ada O6-alkylguanine alkyltransferase in bacteria?',\n",
              "  'O6-Alkylguanine-DNA alkyltransferase (EC 2.1.1.63) repairs O6-alkylguanine lesions in DNA.',\n",
              "  0),\n",
              " (1494,\n",
              "  24,\n",
              "  'What is the role of the Ada O6-alkylguanine alkyltransferase in bacteria?',\n",
              "  'Spontaneous mutagenesis in O6-alkylguanine-DNA alkyltransferase-proficient and -deficient (ada ogt mutants) Escherichia coli was studied in two ways: in bacteria growing in nonselective liquid medium and in bacteria resting on selective agar plates',\n",
              "  0),\n",
              " (1494,\n",
              "  25,\n",
              "  'What is the role of the Ada O6-alkylguanine alkyltransferase in bacteria?',\n",
              "  'The protein O 6-alkylguanine-DNA alkyltransferase(alkyltransferase) is involved in the repair of O 6-alkylguanine and O 4-alkylthymine in DNA and plays an important role in most organisms in attenuating the cytotoxic and mutagenic effects of certain classes of alkylating agents',\n",
              "  0),\n",
              " (1494,\n",
              "  26,\n",
              "  'What is the role of the Ada O6-alkylguanine alkyltransferase in bacteria?',\n",
              "  'In summary, this paper presents for the first time evidence that DNA repair by O6-alkylguanine-DNA alkyltransferases plays a major role in removing lesions responsible for GC-->AT transitions induced by CCNU, influencing their ultimate distribution with respect to sequence context.',\n",
              "  0),\n",
              " (1494,\n",
              "  27,\n",
              "  'What is the role of the Ada O6-alkylguanine alkyltransferase in bacteria?',\n",
              "  'Forward mutations induced by 1-(2-chloroethyl)-3-cyclohexyl-1-nitrosourea (CCNU) in the lacl gene of Escherichia coli were recovered from bacteria proficient (Ogt+ Ada+) and deficient (Ogt- Ada-) in O6-alkylguanine-DNA alkyltransferase activity',\n",
              "  0),\n",
              " (1494,\n",
              "  28,\n",
              "  'What is the role of the Ada O6-alkylguanine alkyltransferase in bacteria?',\n",
              "  'Repair of O6-benzylguanine by the Escherichia coli Ada and Ogt and the human O6-alkylguanine-DNA alkyltransferases.',\n",
              "  0),\n",
              " (1494,\n",
              "  29,\n",
              "  'What is the role of the Ada O6-alkylguanine alkyltransferase in bacteria?',\n",
              "  'O6-Alkylguanine DNA-alkyltransferase (ATase) repairs toxic, mutagenic and carcinogenic O6-alkylguanine (O6-alkG) lesions in DNA by a highly conserved reaction involving the stoichiometric transfer of the alkyl group to the active centre cysteine residue of the ATase protein',\n",
              "  0),\n",
              " (1494,\n",
              "  30,\n",
              "  'What is the role of the Ada O6-alkylguanine alkyltransferase in bacteria?',\n",
              "  'O6-Alkylguanine-DNA alkyltransferase (EC 2.1.1.63) repairs O6-alkylguanine lesions in DNA',\n",
              "  0),\n",
              " (1916,\n",
              "  0,\n",
              "  'What is Bexsero?',\n",
              "  'A capsular group B meningococcal vaccine - 4CMenB (Bexsero) - has recently been licensed in the European Union, Canada and Australia.',\n",
              "  1),\n",
              " (1916,\n",
              "  1,\n",
              "  'What is Bexsero?',\n",
              "  'The 4-component meningococcal serogroup B vaccine 4CMenB (Bexsero) is the first vaccine against this serogroup and has been approved by licensing authorities in Europe, Canada and Australia.',\n",
              "  1),\n",
              " (1916,\n",
              "  2,\n",
              "  'What is Bexsero?',\n",
              "  \"Implementation of meningococcal B vaccination (Bexsero®) in France: Physicians' perceptions and experiences of a few months after marketing approval\",\n",
              "  0),\n",
              " (1916,\n",
              "  3,\n",
              "  'What is Bexsero?',\n",
              "  'In December 2013, the French public health authorities recommended the use of Bexsero® (meningococcus B vaccine) in areas with endemic risk and for patients at risk for invasive meningococcal B disease.',\n",
              "  0),\n",
              " (1916,\n",
              "  4,\n",
              "  'What is Bexsero?',\n",
              "  'Bexsero, a novel vaccine against meningococcus',\n",
              "  0),\n",
              " (1916,\n",
              "  5,\n",
              "  'What is Bexsero?',\n",
              "  'Meningococcus B is the most prevalent Neisseria meningitidis serogroup isolated in Hungary.',\n",
              "  0),\n",
              " (1916,\n",
              "  6,\n",
              "  'What is Bexsero?',\n",
              "  'Bexsero is one of the vaccines developed against it, which has been available in Hungary since the summer of 2014.',\n",
              "  0),\n",
              " (1916,\n",
              "  7,\n",
              "  'What is Bexsero?',\n",
              "  'Vaccinating Italian infants with a new multicomponent vaccine (Bexsero®) against meningococcal B disease: A cost-effectiveness analysis.',\n",
              "  0),\n",
              " (1916,\n",
              "  8,\n",
              "  'What is Bexsero?',\n",
              "  'The European Medicines Agency has approved a multicomponent serogroup B meningococcal vaccine (Bexsero®) for use in individuals of 2 months of age and older.',\n",
              "  0),\n",
              " (1916,\n",
              "  9,\n",
              "  'What is Bexsero?',\n",
              "  'Our results suggest that vaccinating infants in Italy with Bexsero® has the ability to significantly reduce meningococcal disease and, if the probable underestimation of disease incidence is considered, routine vaccination is advisable.',\n",
              "  0),\n",
              " (1916,\n",
              "  10,\n",
              "  'What is Bexsero?',\n",
              "  'Since the antigens selected for Bexsero® are also harbored by meningococci belonging to other serogroups there may be the potential for Bexsero® to offer a certain level of protection against non-B serogroups.',\n",
              "  0),\n",
              " (1916,\n",
              "  11,\n",
              "  'What is Bexsero?',\n",
              "  '[Bexsero, a novel vaccine against meningococcus].',\n",
              "  0),\n",
              " (1916,\n",
              "  12,\n",
              "  'What is Bexsero?',\n",
              "  'The new available vaccines are Bexsero® and Trumenba®.',\n",
              "  0),\n",
              " (1916,\n",
              "  13,\n",
              "  'What is Bexsero?',\n",
              "  'Surface-expressed protein antigens such as factor H-binding protein (fHbp), Neisserial adhesin A (NadA), Neisserial heparin-binding antigen (NHBA) and Porin protein A (PorA); all express sequence variability that can affect their function as protective immunogens when used in meningococcal serogroup B vaccines like the recently-approved 4CMenB (Bexsero(®)).',\n",
              "  0),\n",
              " (1916,\n",
              "  14,\n",
              "  'What is Bexsero?',\n",
              "  'Proteinaceous meningococcal B vaccines are under development, and the most advanced, Bexsero, is currently being evaluated by the European Medicines Agency.',\n",
              "  0),\n",
              " (1916,\n",
              "  15,\n",
              "  'What is Bexsero?',\n",
              "  'Among them, the Reverse Vaccinology approach was successfully applied to the development of an innovative vaccine against Neisseria meningitidis serogroup B, now available on the market with the commercial name BEXSERO® (Novartis Vaccines).',\n",
              "  0),\n",
              " (1916,\n",
              "  16,\n",
              "  'What is Bexsero?',\n",
              "  'Recently, a vaccine, 4CMenB (Bexsero(®)), containing three recombinant proteins, and outer membrane vesicles (OMV) derived from a serogroup B meningococcal strain (MenB) has been licensed in Europe and Australia and is indicated for persons aged 2 mo or older.',\n",
              "  0),\n",
              " (1916,\n",
              "  17,\n",
              "  'What is Bexsero?',\n",
              "  'Multicomponent meningococcal serogroup B vaccine (4CMenB; Bexsero(®)) is a unique vaccine containing four main immunogenic components: three recombinant proteins combined with outer membrane vesicles derived from meningococcal NZ98/254 strain',\n",
              "  0),\n",
              " (1916,\n",
              "  18,\n",
              "  'What is Bexsero?',\n",
              "  'Recently, a universal serogroup B meningococcal vaccine, Bexsero(®), was licensed in Europe, Australia and United States, following several clinical studies demonstrating its immunogenicity and safety',\n",
              "  0),\n",
              " (1916,\n",
              "  19,\n",
              "  'What is Bexsero?',\n",
              "  'Like any drug, Bexsero(®) and Trumemba(®) will require close observation to assess their impact on meningococcal epidemiology.',\n",
              "  0),\n",
              " (1916,\n",
              "  20,\n",
              "  'What is Bexsero?',\n",
              "  '<CopyrightInformation>Copyright © 2015 Elsevier Masson SAS',\n",
              "  0),\n",
              " (1916,\n",
              "  21,\n",
              "  'What is Bexsero?',\n",
              "  'Bexsero, a new vaccine against Neisseria meningitidis serogroup B (MenB), is composed of 3 main recombinant proteins and an outer membrane vesicle component',\n",
              "  1),\n",
              " (1916,\n",
              "  22,\n",
              "  'What is Bexsero?',\n",
              "  'The meningococcal 4CMenB vaccine (Bexsero; Novartis) contains four antigens that can elicit serum bactericidal activity, one of which is factor H (FH)-binding protein (FHbp)',\n",
              "  0),\n",
              " (1916,\n",
              "  23,\n",
              "  'What is Bexsero?',\n",
              "  'In December 2013 Bexsero® became available in Germany for vaccination against serogroup B meningococci (MenB)',\n",
              "  0),\n",
              " (1916,\n",
              "  24,\n",
              "  'What is Bexsero?',\n",
              "  'Bexsero® is based on surface protein antigens expressed by about 80% of circulating serogroup B meningococci in Germany',\n",
              "  0),\n",
              " (1916,\n",
              "  25,\n",
              "  'What is Bexsero?',\n",
              "  'Re-evaluating cost effectiveness of universal meningitis vaccination (Bexsero) in England: modelling study.',\n",
              "  0),\n",
              " (1916,\n",
              "  26,\n",
              "  'What is Bexsero?',\n",
              "  'Bexsero: a multicomponent vaccine for prevention of meningococcal disease.',\n",
              "  0),\n",
              " (1916,\n",
              "  27,\n",
              "  'What is Bexsero?',\n",
              "  'OBJECTIVE: To use mathematical and economic models to predict the epidemiological and economic impact of vaccination with Bexsero, designed to protect against group B meningococcal disease, to help inform vaccine policy in the United Kingdom.DESIGN: Modelling study.SETTING: England.POPULATION: People aged 0-99.INTERVENTIONS: Incremental impact of introductory vaccine strategies simulated with a transmission dynamic model of meningococcal infection and vaccination including potential herd effects.',\n",
              "  0),\n",
              " (1916,\n",
              "  28,\n",
              "  'What is Bexsero?',\n",
              "  'Preventing secondary cases of invasive meningococcal capsular group B (MenB) disease using a recently-licensed, multi-component, protein-based vaccine (Bexsero(�)).',\n",
              "  0),\n",
              " (1916,\n",
              "  29,\n",
              "  'What is Bexsero?',\n",
              "  'Modelled evaluation of multi-component meningococcal vaccine (Bexsero®) for the prevention of invasive meningococcal disease in infants and adolescents in the UK.',\n",
              "  0),\n",
              " (1916,\n",
              "  30,\n",
              "  'What is Bexsero?',\n",
              "  'BACKGROUND: 4CMenB (Bexsero), a vaccine developed against invasive meningococcal disease caused by capsular group B strains (MenB), was recently licensed for use by the European Medicines Agency.',\n",
              "  0),\n",
              " (1916,\n",
              "  31,\n",
              "  'What is Bexsero?',\n",
              "  'Bexsero is the first meningococcal B vaccine to be approved in the European Union.',\n",
              "  0),\n",
              " (1916,\n",
              "  32,\n",
              "  'What is Bexsero?',\n",
              "  'Quantification by LC-MS(E) of outer membrane vesicle proteins of the Bexsero® vaccine.',\n",
              "  0),\n",
              " (1916,\n",
              "  33,\n",
              "  'What is Bexsero?',\n",
              "  'We previously demonstrated the immunogenicity and tolerability of the serogroup B meningococcal vaccine, 4CMenB (Bexsero), in 11-17 y-olds randomized to receive 1, 2, or 3 doses at 1, 2, or 6 mo intervals.',\n",
              "  0),\n",
              " (1916,\n",
              "  34,\n",
              "  'What is Bexsero?',\n",
              "  'This policy statement provides recommendations for the prevention of serogroup B meningococcal disease through the use of 2 newly licensed serogroup B meningococcal vaccines: MenB-FHbp (Trumenba; Wyeth Pharmaceuticals, a subsidiary of Pfizer, Philadelphia, PA) and MenB-4C (Bexsero; Novartis Vaccines, Siena, Italy).',\n",
              "  0),\n",
              " (1916,\n",
              "  35,\n",
              "  'What is Bexsero?',\n",
              "  'An outbreak of Neisseria meningitidis serotype B infection occurred at a small residential university; public health announced an organizational vaccination program with the 4-component Meningococcal B (4CMenB) vaccine (Bexsero(TM), Novartis/GlaxoSmithKline Inc.) several days later.',\n",
              "  0),\n",
              " (1916,\n",
              "  36,\n",
              "  'What is Bexsero?',\n",
              "  'Bexsero, a new vaccine against serogroup B meningococcal disease (MenB), was licensed in Europe in January 2013.',\n",
              "  1),\n",
              " (1916,\n",
              "  37,\n",
              "  'What is Bexsero?',\n",
              "  'Recently approved in the EU, US, Australia, and Canada, 4CMenB (Bexsero(®), GSK Vaccines) is a multi-component meningococcal B (MenB) vaccine containing 3 surface exposed recombinant proteins (fHbp, NadA, and NHBA) and New Zealand strain outer membrane vesicles (NZ OMV) containing PorA 1.4.',\n",
              "  0),\n",
              " (1916,\n",
              "  38,\n",
              "  'What is Bexsero?',\n",
              "  'In Germany, Bexsero is recommended for persons at increased risk of invasive meningococcal disease, but not for universal childhood vaccination.',\n",
              "  0),\n",
              " (1916,\n",
              "  39,\n",
              "  'What is Bexsero?',\n",
              "  'To support decision making we adapted the independently developed model for England to the German setting to predict the potential health impact and cost-effectiveness of universal vaccination with Bexsero(®) against MenB disease.',\n",
              "  0),\n",
              " (1916,\n",
              "  40,\n",
              "  'What is Bexsero?',\n",
              "  'This has resulted in a multicomponent, recombinant, meningococcal serogroup B vaccine: 4CMenB (Bexsero(®), Novartis Vaccines & Diagnostics, NC, USA), containing four main immunogenic components: two recombinant fusion proteins (Neisseria heparin-binding antigen-GNA1030 and factor H-binding protein-GNA2091); recombinant Neisserial adhesion A; and detergent-treated outer membrane vesicles derived from the meningococcal NZ98/254 strain, where porin A 1.4 is the major immunodominant antigen.',\n",
              "  0),\n",
              " (1916,\n",
              "  41,\n",
              "  'What is Bexsero?',\n",
              "  'BACKGROUND: 4CMenB (Bexsero), a vaccine developed against invasive meningococcal disease caused by capsular group B strains (MenB), was recently licensed for use by the European Medicines Agency.',\n",
              "  0),\n",
              " (1916,\n",
              "  42,\n",
              "  'What is Bexsero?',\n",
              "  'In January 2015, FDA licensed a second MenB vaccine (MenB-4C [Bexsero, Novartis Vaccines]) as a 2-dose series.',\n",
              "  0),\n",
              " (1916,\n",
              "  43,\n",
              "  'What is Bexsero?',\n",
              "  'Bexsero, a new vaccine against Neisseria meningitidis serogroup B (MenB), is composed of 3 main recombinant proteins and an outer membrane vesicle component.',\n",
              "  0),\n",
              " (1916,\n",
              "  44,\n",
              "  'What is Bexsero?',\n",
              "  'Preventing secondary cases of invasive meningococcal capsular group B (MenB) disease using a recently-licensed, multi-component, protein-based vaccine (Bexsero(®)).',\n",
              "  0),\n",
              " (1916,\n",
              "  45,\n",
              "  'What is Bexsero?',\n",
              "  '[Bexsero, a novel vaccine against meningococcus].',\n",
              "  0),\n",
              " (1916,\n",
              "  46,\n",
              "  'What is Bexsero?',\n",
              "  \"Modelling the cost-effectiveness of catch-up 'MenB' (Bexsero) vaccination in England.\",\n",
              "  0),\n",
              " (1916,\n",
              "  47,\n",
              "  'What is Bexsero?',\n",
              "  'Re-evaluating cost effectiveness of universal meningitis vaccination (Bexsero) in England: modelling study.',\n",
              "  0),\n",
              " (1916,\n",
              "  48,\n",
              "  'What is Bexsero?',\n",
              "  'The novel vaccine Bexsero (previously 4CMenB) has been developed and proven safe and immunogenic in clinical trials.',\n",
              "  0),\n",
              " (1916,\n",
              "  49,\n",
              "  'What is Bexsero?',\n",
              "  'Multicomponent meningococcal serogroup B vaccine (4CMenB; Bexsero(®)): a review of its use in primary and booster vaccination.',\n",
              "  0),\n",
              " (1916,\n",
              "  50,\n",
              "  'What is Bexsero?',\n",
              "  'Bexsero, the recently licensed multicomponent vaccine against serogroup B Neisseria meningitidis (MenB).',\n",
              "  1),\n",
              " (1916,\n",
              "  51,\n",
              "  'What is Bexsero?',\n",
              "  'Recommendation for anti-meningococcal vaccination among these workers has been recently updated upon the licensure in Europe of Bexsero® vaccine.',\n",
              "  0),\n",
              " (1916,\n",
              "  52,\n",
              "  'What is Bexsero?',\n",
              "  'Clinical recommendations for the use of Bexsero have been published in several countries.',\n",
              "  0),\n",
              " (471,\n",
              "  0,\n",
              "  'Is there a crystal structure of the full-length of the flaviviridae NS5(Methyltransferase - RNA depended RNA Polymerase) ?',\n",
              "  ' flavivirus NS5 harbors a methyltransferase (MTase) in its N-terminal ≈ 265 residues and an RNA-dependent RNA polymerase (RdRP) within the C-terminal part.',\n",
              "  0),\n",
              " (471,\n",
              "  1,\n",
              "  'Is there a crystal structure of the full-length of the flaviviridae NS5(Methyltransferase - RNA depended RNA Polymerase) ?',\n",
              "  'One of the major interests and challenges in NS5 is to understand the interplay between RdRP and MTase as a unique natural fusion protein in viral genome replication and cap formation.',\n",
              "  0),\n",
              " (471,\n",
              "  2,\n",
              "  'Is there a crystal structure of the full-length of the flaviviridae NS5(Methyltransferase - RNA depended RNA Polymerase) ?',\n",
              "  'Here, we report the first crystal structure of the full-length flavivirus NS5 from Japanese encephalitis virus.',\n",
              "  1),\n",
              " (471,\n",
              "  3,\n",
              "  'Is there a crystal structure of the full-length of the flaviviridae NS5(Methyltransferase - RNA depended RNA Polymerase) ?',\n",
              "  '(DENV) nonstructural protein 5 (NS5) is composed of two globular domains separated by a 10-residue linker.',\n",
              "  0),\n",
              " (471,\n",
              "  4,\n",
              "  'Is there a crystal structure of the full-length of the flaviviridae NS5(Methyltransferase - RNA depended RNA Polymerase) ?',\n",
              "  \"The N-terminal domain participates in the synthesis of a mRNA cap 1 structure ((7Me)GpppA(2'OMe)) at the 5' end of the viral genome and possesses guanylyltransferase, guanine-N7-methyltransferase, and nucleoside-2'O-methyltransferase activities.\",\n",
              "  0),\n",
              " (471,\n",
              "  5,\n",
              "  'Is there a crystal structure of the full-length of the flaviviridae NS5(Methyltransferase - RNA depended RNA Polymerase) ?',\n",
              "  'The C-terminal domain is an RNA-dependent RNA polymerase responsible for viral RNA synthesis.',\n",
              "  0),\n",
              " (471,\n",
              "  6,\n",
              "  'Is there a crystal structure of the full-length of the flaviviridae NS5(Methyltransferase - RNA depended RNA Polymerase) ?',\n",
              "  'Although crystal structures of the two isolated domains have been obtained, there are no structural data for full-length NS5.',\n",
              "  1),\n",
              " (471,\n",
              "  7,\n",
              "  'Is there a crystal structure of the full-length of the flaviviridae NS5(Methyltransferase - RNA depended RNA Polymerase) ?',\n",
              "  'It is also unclear whether the two NS5 domains interact with each other to form a stable structure in which the relative orientation of the two domains is fixed.',\n",
              "  1),\n",
              " (471,\n",
              "  8,\n",
              "  'Is there a crystal structure of the full-length of the flaviviridae NS5(Methyltransferase - RNA depended RNA Polymerase) ?',\n",
              "  'To investigate the structure and dynamics of DENV type 3 NS5 in solution, we conducted small-angle X-ray scattering experiments with the full-length protein.',\n",
              "  1),\n",
              " (471,\n",
              "  9,\n",
              "  'Is there a crystal structure of the full-length of the flaviviridae NS5(Methyltransferase - RNA depended RNA Polymerase) ?',\n",
              "  'NS5 was found to be monomeric and well-folded under the conditions tested.',\n",
              "  0),\n",
              " (471,\n",
              "  10,\n",
              "  'Is there a crystal structure of the full-length of the flaviviridae NS5(Methyltransferase - RNA depended RNA Polymerase) ?',\n",
              "  'West Nile virus (WNV) NS5 protein contains a methyltransferase (MTase) domain involved in RNA capping and an RNA-dependent RNA polymerase (RdRp) domain essential for virus replication.',\n",
              "  0),\n",
              " (471,\n",
              "  11,\n",
              "  'Is there a crystal structure of the full-length of the flaviviridae NS5(Methyltransferase - RNA depended RNA Polymerase) ?',\n",
              "  'Crystal structures of individual WNV MTase and RdRp domains have been solved; however, the structure of full-length NS5 has not been determined.',\n",
              "  1),\n",
              " (471,\n",
              "  12,\n",
              "  'Is there a crystal structure of the full-length of the flaviviridae NS5(Methyltransferase - RNA depended RNA Polymerase) ?',\n",
              "  'To gain more insight into the structure of NS5 and interactions between the MTase and RdRp domains, we generated a panel of seven monoclonal antibodies (mAbs) to the NS5 protein of WNV (Kunjin strain) and mapped their binding sites using a series of truncated NS5 proteins and synthetic peptides.',\n",
              "  0),\n",
              " (471,\n",
              "  13,\n",
              "  'Is there a crystal structure of the full-length of the flaviviridae NS5(Methyltransferase - RNA depended RNA Polymerase) ?',\n",
              "  'Binding sites of four mAbs (5D4, 4B6, 5C11 and 6A10) were mapped to residues 354-389 in the fingers subdomain of the RdRp.',\n",
              "  0),\n",
              " (471,\n",
              "  14,\n",
              "  'Is there a crystal structure of the full-length of the flaviviridae NS5(Methyltransferase - RNA depended RNA Polymerase) ?',\n",
              "  'This is consistent with the ability of these mAbs to inhibit RdRp activity in vitro and suggests that this region represents a potential target for RdRp inhibitors.',\n",
              "  0),\n",
              " (471,\n",
              "  15,\n",
              "  'Is there a crystal structure of the full-length of the flaviviridae NS5(Methyltransferase - RNA depended RNA Polymerase) ?',\n",
              "  'Using a series of synthetic peptides, we also identified a linear epitope (bound by mAb 5H1) that mapped to a 13 aa stretch surrounding residues 47 and 49 in the MTase domain, a region predicted to interact with the palm subdomain of the RdRp.',\n",
              "  0),\n",
              " (471,\n",
              "  16,\n",
              "  'Is there a crystal structure of the full-length of the flaviviridae NS5(Methyltransferase - RNA depended RNA Polymerase) ?',\n",
              "  'The failure of one mAb (7G6) to bind both N- and C-terminally truncated NS5 recombinants indicates that the antibody recognizes a conformational epitope that requires the presence of residues in both the MTase and RdRp domains.',\n",
              "  0),\n",
              " (4224,\n",
              "  0,\n",
              "  'Which R/Bioconductor package has been developed for visualizing differential amino acid group usage in proteomics?',\n",
              "  'dagLogo: An R/Bioconductor package for identifying and visualizing differential amino acid group usage in proteomics data',\n",
              "  1),\n",
              " (4224,\n",
              "  1,\n",
              "  'Which R/Bioconductor package has been developed for visualizing differential amino acid group usage in proteomics?',\n",
              "  'Sequence logos have been widely used as graphical representations of conserved nucleic acid and protein motifs.',\n",
              "  0),\n",
              " (4224,\n",
              "  2,\n",
              "  'Which R/Bioconductor package has been developed for visualizing differential amino acid group usage in proteomics?',\n",
              "  'Due to the complexity of the amino acid (AA) alphabet, rich post-translational modification, and diverse subcellular localization of proteins, few versatile tools are available for effective identification and visualization of protein motifs.',\n",
              "  0),\n",
              " (4224,\n",
              "  3,\n",
              "  'Which R/Bioconductor package has been developed for visualizing differential amino acid group usage in proteomics?',\n",
              "  'In addition, various reduced AA alphabets based on physicochemical, structural, or functional properties have been valuable in the study of protein alignment, folding, structure prediction, and evolution.',\n",
              "  0),\n",
              " (4224,\n",
              "  4,\n",
              "  'Which R/Bioconductor package has been developed for visualizing differential amino acid group usage in proteomics?',\n",
              "  'However, there is lack of tools for applying reduced AA alphabets to the identification and visualization of statistically significant motifs.',\n",
              "  0),\n",
              " (4224,\n",
              "  5,\n",
              "  'Which R/Bioconductor package has been developed for visualizing differential amino acid group usage in proteomics?',\n",
              "  'To fill this gap, we developed an R/Bioconductor package dagLogo, which has several advantages over existing tools.',\n",
              "  0),\n",
              " (4224,\n",
              "  6,\n",
              "  'Which R/Bioconductor package has been developed for visualizing differential amino acid group usage in proteomics?',\n",
              "  'First, dagLogo allows various formats for input sets and provides comprehensive options to build optimal background models.',\n",
              "  1),\n",
              " (4224,\n",
              "  7,\n",
              "  'Which R/Bioconductor package has been developed for visualizing differential amino acid group usage in proteomics?',\n",
              "  'It implements different reduced AA alphabets to group AAs of similar properties.',\n",
              "  1),\n",
              " (4224,\n",
              "  8,\n",
              "  'Which R/Bioconductor package has been developed for visualizing differential amino acid group usage in proteomics?',\n",
              "  'Furthermore, dagLogo provides statistical and visual solutions for differential AA (or AA group) usage analysis of both large and small data sets.',\n",
              "  1),\n",
              " (4224,\n",
              "  9,\n",
              "  'Which R/Bioconductor package has been developed for visualizing differential amino acid group usage in proteomics?',\n",
              "  'Case studies showed that dagLogo can better identify and visualize conserved protein sequence patterns from different types of inputs and can potentially reveal the biological patterns that could be missed by other logo generators.',\n",
              "  1),\n",
              " (1949,\n",
              "  0,\n",
              "  'Which R package is used for the analysis of genome-wide DNA methylation profiles?',\n",
              "  'methylKit: a comprehensive R package for the analysis of genome-wide DNA methylation profiles.',\n",
              "  1),\n",
              " (1949,\n",
              "  1,\n",
              "  'Which R package is used for the analysis of genome-wide DNA methylation profiles?',\n",
              "  ' Here, we describe an R package, methylKit, that rapidly analyzes genome-wide cytosine epigenetic profiles from high-throughput methylation and hydroxymethylation sequencing experiments.',\n",
              "  0),\n",
              " (1949,\n",
              "  2,\n",
              "  'Which R package is used for the analysis of genome-wide DNA methylation profiles?',\n",
              "  'methylKit includes functions for clustering, sample quality visualization, differential methylation analysis and annotation features, thus automating and simplifying many of the steps for discerning statistically significant bases or regions of DNA methylation.',\n",
              "  1),\n",
              " (1949,\n",
              "  3,\n",
              "  'Which R package is used for the analysis of genome-wide DNA methylation profiles?',\n",
              "  'Here, we describe an R package, methylKit, that rapidly analyzes genome-wide cytosine epigenetic profiles from high-throughput methylation and hydroxymethylation sequencing experiments.',\n",
              "  0),\n",
              " (1949,\n",
              "  4,\n",
              "  'Which R package is used for the analysis of genome-wide DNA methylation profiles?',\n",
              "  'methylKit: a comprehensive R package for the analysis of genome-wide DNA methylation profiles',\n",
              "  1),\n",
              " (1949,\n",
              "  5,\n",
              "  'Which R package is used for the analysis of genome-wide DNA methylation profiles?',\n",
              "  'Here, we describe an R package, methylKit, that rapidly analyzes genome-wide cytosine epigenetic profiles from high-throughput methylation and hydroxymethylation sequencing experiments.',\n",
              "  0),\n",
              " (1949,\n",
              "  6,\n",
              "  'Which R package is used for the analysis of genome-wide DNA methylation profiles?',\n",
              "  'methylKit includes functions for clustering, sample quality visualization, differential methylation analysis and annotation features, thus automating and simplifying many of the steps for discerning statistically significant bases or regions of DNA methylation',\n",
              "  1),\n",
              " (1949,\n",
              "  7,\n",
              "  'Which R package is used for the analysis of genome-wide DNA methylation profiles?',\n",
              "  'Here, we describe an R package, methylKit, that rapidly analyzes genome-wide cytosine epigenetic profiles from high-throughput methylation and hydroxymethylation sequencing experiments.',\n",
              "  0),\n",
              " (1949,\n",
              "  8,\n",
              "  'Which R package is used for the analysis of genome-wide DNA methylation profiles?',\n",
              "  'methylKit includes functions for clustering, sample quality visualization, differential methylation analysis and annotation features, thus automating and simplifying many of the steps for discerning statistically significant bases or regions of DNA methylation.',\n",
              "  1),\n",
              " (1949,\n",
              "  9,\n",
              "  'Which R package is used for the analysis of genome-wide DNA methylation profiles?',\n",
              "  'methylKit: a comprehensive R package for the analysis of genome-wide DNA methylation profiles.',\n",
              "  0),\n",
              " (2517,\n",
              "  0,\n",
              "  'Does wheat belongs to the genus Avena, yes or no?',\n",
              "  'oat seedlings (Avena sativa)',\n",
              "  1),\n",
              " (2517,\n",
              "  1,\n",
              "  'Does wheat belongs to the genus Avena, yes or no?',\n",
              "  'wild green-oat (Avena sativa)',\n",
              "  1),\n",
              " (2517,\n",
              "  2,\n",
              "  'Does wheat belongs to the genus Avena, yes or no?',\n",
              "  ' Oat (Avena sativa L.)',\n",
              "  1),\n",
              " (2517,\n",
              "  3,\n",
              "  'Does wheat belongs to the genus Avena, yes or no?',\n",
              "  'Oat (Avena sativa L.)',\n",
              "  1),\n",
              " (2517,\n",
              "  4,\n",
              "  'Does wheat belongs to the genus Avena, yes or no?',\n",
              "  'Avena (Oats)',\n",
              "  0),\n",
              " (2517,\n",
              "  5,\n",
              "  'Does wheat belongs to the genus Avena, yes or no?',\n",
              "  'wild oats (Avena fatua L.)',\n",
              "  0),\n",
              " (2517,\n",
              "  6,\n",
              "  'Does wheat belongs to the genus Avena, yes or no?',\n",
              "  'oats (genus Avena)',\n",
              "  0),\n",
              " (2517,\n",
              "  7,\n",
              "  'Does wheat belongs to the genus Avena, yes or no?',\n",
              "  'Avena sativa L. and A. byzantina C. Koch)',\n",
              "  0),\n",
              " (2517,\n",
              "  8,\n",
              "  'Does wheat belongs to the genus Avena, yes or no?',\n",
              "  ' oat (Avena sativa L.).',\n",
              "  1),\n",
              " (2517,\n",
              "  9,\n",
              "  'Does wheat belongs to the genus Avena, yes or no?',\n",
              "  'oat (Avena sativa L.) and wheat (Triticum aestivum',\n",
              "  0),\n",
              " (4127,\n",
              "  0,\n",
              "  'What is BEL(Biological Expression Language) used for?',\n",
              "  'The BioCreative-V community proposed a challenging task of automatic extraction of causal relation network in Biological Expression Language (BEL) from the biomedical literature.',\n",
              "  0),\n",
              " (4127,\n",
              "  1,\n",
              "  'What is BEL(Biological Expression Language) used for?',\n",
              "  ' The BioCreative community has organized a shared task to evaluate the robustness of the causal relationship extraction algorithms in Biological Expression Language (BEL) from biomedical literature.',\n",
              "  0),\n",
              " (4127,\n",
              "  2,\n",
              "  'What is BEL(Biological Expression Language) used for?',\n",
              "  'Biological expression language (BEL) is a syntax representation allowing for the structured representation of a broad range of biological relationships.',\n",
              "  1),\n",
              " (4127,\n",
              "  3,\n",
              "  'What is BEL(Biological Expression Language) used for?',\n",
              "  'It is used in various situations to extract such knowledge and transform it into BEL networks.',\n",
              "  0),\n",
              " (4127,\n",
              "  4,\n",
              "  'What is BEL(Biological Expression Language) used for?',\n",
              "  'Biological Expression Language (BEL) assembles knowledge networks from biological relations across multiple modes and scales.',\n",
              "  0),\n",
              " (4127,\n",
              "  5,\n",
              "  'What is BEL(Biological Expression Language) used for?',\n",
              "  'or network representation, the Biological Expression Language (BEL) is well designed to collate findings from the scientific literature into biological network models.',\n",
              "  0),\n",
              " (4127, 6, 'What is BEL(Biological Expression Language) used for?', 'To', 0),\n",
              " (4127,\n",
              "  7,\n",
              "  'What is BEL(Biological Expression Language) used for?',\n",
              "  'Biological expression language (BEL) is one of the most popular languages to represent the causal and correlative relationships among biological events.',\n",
              "  0),\n",
              " (4127,\n",
              "  8,\n",
              "  'What is BEL(Biological Expression Language) used for?',\n",
              "  'his work, we demonstrate how disease-related epigenetic knowledge can be systematically captured and integrated with heterogeneous information into a functional context using Biological Expression Language (BEL).',\n",
              "  0),\n",
              " (4127, 9, 'What is BEL(Biological Expression Language) used for?', 'Thi', 0),\n",
              " (4127,\n",
              "  10,\n",
              "  'What is BEL(Biological Expression Language) used for?',\n",
              "  'Summary: Biological Expression Language (BEL) assembles knowledge networks from biological relations across multiple modes and',\n",
              "  0),\n",
              " (4127,\n",
              "  11,\n",
              "  'What is BEL(Biological Expression Language) used for?',\n",
              "  'e-edge relationships are described using the Biological Expression Language (BEL), which allows for the semantic representation of life science relationships in a computable format.',\n",
              "  0),\n",
              " (4127,\n",
              "  12,\n",
              "  'What is BEL(Biological Expression Language) used for?',\n",
              "  'The Networ',\n",
              "  0),\n",
              " (4127,\n",
              "  13,\n",
              "  'What is BEL(Biological Expression Language) used for?',\n",
              "  'Biological expression language (BEL) is one of the main formal representation models of biological networks.',\n",
              "  1),\n",
              " (4127,\n",
              "  14,\n",
              "  'What is BEL(Biological Expression Language) used for?',\n",
              "  'logical expression language (BEL) is a syntax representation allowing for the structured representation of a broad range of biological relationships.',\n",
              "  1),\n",
              " (4127, 15, 'What is BEL(Biological Expression Language) used for?', 'It', 0),\n",
              " (4127,\n",
              "  16,\n",
              "  'What is BEL(Biological Expression Language) used for?',\n",
              "  'semi-automated knowledge extraction workflow is presented that was developed to allow users to extract causal and correlative relationships from scientific literature and to transcribe them into the computable and human readable Biological Expression Language (BEL).',\n",
              "  0),\n",
              " (4127,\n",
              "  17,\n",
              "  'What is BEL(Biological Expression Language) used for?',\n",
              "  'Biological expression language (BEL) is a syntax representation allowing for the structured representation of a broad range of biological relationships.',\n",
              "  1),\n",
              " (4127,\n",
              "  18,\n",
              "  'What is BEL(Biological Expression Language) used for?',\n",
              "  'The extraction of complex relationships and their conversion to biological expression language (BEL) overview of the BioCreative VI (2017) BEL track.',\n",
              "  0),\n",
              " (4127,\n",
              "  19,\n",
              "  'What is BEL(Biological Expression Language) used for?',\n",
              "  'Biological expression language (BEL) is one of the most popular languages to represent the causal and correlative relationships among biological events.',\n",
              "  0),\n",
              " (4127,\n",
              "  20,\n",
              "  'What is BEL(Biological Expression Language) used for?',\n",
              "  'For network representation, the Biological Expression Language (BEL) is well designed to collate findings from the scientific literature into biological network models.',\n",
              "  0),\n",
              " (4127,\n",
              "  21,\n",
              "  'What is BEL(Biological Expression Language) used for?',\n",
              "  'Biological expression language (BEL) is one of the main formal representation models of biological networks.',\n",
              "  1),\n",
              " (4127,\n",
              "  22,\n",
              "  'What is BEL(Biological Expression Language) used for?',\n",
              "  'BEL was designed to capture relationships not only between proteins or chemicals, but also complex events such as biological processes or disease states.',\n",
              "  0),\n",
              " (4127,\n",
              "  23,\n",
              "  'What is BEL(Biological Expression Language) used for?',\n",
              "  'BEL is an advanced knowledge representation format which has been designed to be both human readable and machine processable.',\n",
              "  0),\n",
              " (4127,\n",
              "  24,\n",
              "  'What is BEL(Biological Expression Language) used for?',\n",
              "  'Here, we describe the new corpora in the systems biology modeling language BEL for training and testing biological relationship extraction systems that we prepared for the BioCreative V BEL track.',\n",
              "  0),\n",
              " (1593,\n",
              "  0,\n",
              "  'Which factors are considered in the ABCD2 score?',\n",
              "  \"The 'accuracy' of age, blood pressure, clinical features, duration and diabetes (ABCD(2)) scoring by non-stroke specialists referring patients to a daily Rapid Access Stroke Prevention (RASP) service is unclear, as is the accuracy of ABCD(2) scoring by trainee residents.\",\n",
              "  0),\n",
              " (1593,\n",
              "  1,\n",
              "  'Which factors are considered in the ABCD2 score?',\n",
              "  ' Classification of low risk was based on an age, blood pressure, clinical features, duration of symptoms and diabetes (ABCD2) score <4 and the absence of high risk features, including known carotid disease, crescendo TIA, or atrial fibrillation.',\n",
              "  1),\n",
              " (1593,\n",
              "  2,\n",
              "  'Which factors are considered in the ABCD2 score?',\n",
              "  'Our goal was to compare the very early predictive accuracy of the most relevant clinical scores [age, blood pressure, clinical features and duration of symptoms (ABCD) score, ABCD and diabetes (ABCD2) score, ABCD and brain infarction on imaging score, ABCD2 and brain infarction on imaging score, ABCD and prior TIA within 1 week of the index event (ABCD3) score, California Risk Score, Essen Stroke Risk Score and Stroke Prognosis Instrument II] in consecutive transient ischemic attack (TIA) patients.',\n",
              "  0),\n",
              " (1593,\n",
              "  3,\n",
              "  'Which factors are considered in the ABCD2 score?',\n",
              "  'We examined the association of the age, blood pressure, clinical features, duration of symptoms and diabetes (ABCD2) score, a validated risk prediction model for stroke after TIA, and the presence of ICS or ECS.',\n",
              "  1),\n",
              " (1593,\n",
              "  4,\n",
              "  'Which factors are considered in the ABCD2 score?',\n",
              "  'BACKGROUND: The ABCD2 score (Age, Blood pressure, Clinical features, Duration of symptoms and Diabetes) is used to identify patients having a transient ischemic attack who are at high risk for imminent stroke.',\n",
              "  1),\n",
              " (1593,\n",
              "  5,\n",
              "  'Which factors are considered in the ABCD2 score?',\n",
              "  'BACKGROUND AND PURPOSE: The risk of stroke after a transient ischaemic attack (TIA) can be predicted by scores incorporating age, blood pressure, clinical features, duration (ABCD-score), and diabetes (ABCD2-score).',\n",
              "  0),\n",
              " (1593,\n",
              "  6,\n",
              "  'Which factors are considered in the ABCD2 score?',\n",
              "  'BACKGROUND: Modifications to the age, blood pressure, clinical symptoms, duration of symptoms, and diabetes (ABCD2) score, which incorporate history of hypertension and acute hyperglycemia in addition to acute blood pressure (BP) elevation and history of diabetes, have been proposed to increase the predictive value of the score.',\n",
              "  0),\n",
              " (1593,\n",
              "  7,\n",
              "  'Which factors are considered in the ABCD2 score?',\n",
              "  'ABCD2 (Age, Blood pressure, Clinical features, Duration, Diabetes) score and magnetic resonance imaging abnormalities help to identify patients at high risk of stroke.',\n",
              "  1),\n",
              " (1593,\n",
              "  8,\n",
              "  'Which factors are considered in the ABCD2 score?',\n",
              "  'METHODS: We prospectively calculated the ABCD(2) score (age [> or = 60 years: 1 point]; blood pressure [systolic >140 mm Hg or diastolic >90 mm Hg: 1[; clinical features [unilateral weakness: 2, speech disturbance without weakness: 1, other symptom: 0]; duration of symptoms [ <10 minutes: 0, 10-59 minutes: 1, > or = 60 minutes: 2]; diabetes mellitus [yes: 1]) in consecutive TIA patients hospitalized in 3 tertiary care neurology departments across 2 different racial populations (white and Asian).',\n",
              "  0),\n",
              " (1593,\n",
              "  9,\n",
              "  'Which factors are considered in the ABCD2 score?',\n",
              "  'Early stroke risk is especially high in TIA patients with a high ABCD2 score of 4 or more (A age over 60 years [1 point]: B blood pressure > 140/90 mmHg [1 point]: C Clinical features, including unilateral weakness [2 points] and speech disturbance without weakness [1 point] D2: Diabetes [1 point] and Duration of symptoms [1 point for < 60 min and 2 points for > 60 min]), acute ischemic lesions on diffusion weighted image, > 50% carotid stenosis, severe intracranial artery stenosis, microembolic signals on transcranial Doppler, atrial fibrillation, or hypercoagulable states.',\n",
              "  0),\n",
              " (1593,\n",
              "  10,\n",
              "  'Which factors are considered in the ABCD2 score?',\n",
              "  'STUDY OBJECTIVE: We evaluate, in admitted patients with transient ischemic attack, the accuracy of the ABCD(2) (age [A], blood pressure [B], clinical features [weakness/speech disturbance] [C], transient ischemic attack duration [D], and diabetes history [D]) score in predicting ischemic stroke within 7 days.',\n",
              "  0),\n",
              " (1593,\n",
              "  11,\n",
              "  'Which factors are considered in the ABCD2 score?',\n",
              "  'Our goal was to compare the very early predictive accuracy of the most relevant clinical scores [age, blood pressure, clinical features and duration of symptoms (ABCD) score, ABCD and diabetes (ABCD2) score, ABCD and brain infarction on imaging score, ABCD2 and brain infarction on imaging score, ABCD and prior TIA within 1 week of the index event (ABCD3) score, California Risk Score, Essen Stroke Risk Score and Stroke Prognosis Instrument II] in consecutive transient ischemic attack (TIA) patients.',\n",
              "  0),\n",
              " (1593,\n",
              "  12,\n",
              "  'Which factors are considered in the ABCD2 score?',\n",
              "  'The risk of stroke after a transient ischaemic attack (TIA) can be predicted by scores incorporating age, blood pressure, clinical features, duration (ABCD-score), and diabetes (ABCD2-score).',\n",
              "  1),\n",
              " (1593,\n",
              "  13,\n",
              "  'Which factors are considered in the ABCD2 score?',\n",
              "  'BACKGROUND AND PURPOSE: The risk of stroke after a transient ischaemic attack (TIA) can be predicted by scores incorporating age, blood pressure, clinical features, duration (ABCD-score), and diabetes (ABCD2-score).',\n",
              "  0),\n",
              " (573,\n",
              "  0,\n",
              "  'Are there telemedicine applications for chronic pain management?',\n",
              "  'An integrated cognitive-behavioral and physical therapy group protocol has been developed and then implemented at remote sites using videoconferencing technology to provide pain management for veterans.',\n",
              "  0),\n",
              " (573,\n",
              "  1,\n",
              "  'Are there telemedicine applications for chronic pain management?',\n",
              "  'Tele-pain management: use of videoconferencing technology in the delivery of an integrated cognitive-behavioral and physical therapy group intervention.',\n",
              "  0),\n",
              " (573,\n",
              "  2,\n",
              "  'Are there telemedicine applications for chronic pain management?',\n",
              "  'It is feasible to provide treatment to women veterans living in rural areas by utilizing video-teleconferencing technology between larger VA medical centers and facilities at CBOCs in more rural settings',\n",
              "  0),\n",
              " (573,\n",
              "  3,\n",
              "  'Are there telemedicine applications for chronic pain management?',\n",
              "  'The results suggest that a smartphone-delivered intervention with diaries and personalized feedback can reduce catastrophizing and prevent increases in functional impairment and symptom levels in women with chronic widespread pain following inpatient rehabilitation.',\n",
              "  1),\n",
              " (573,\n",
              "  4,\n",
              "  'Are there telemedicine applications for chronic pain management?',\n",
              "  ' Of the studies available, there are very few randomized trials of telehealth pain care and only one general overview of e-health and chronic pain, which dedicates just a few paragraphs to telehealth.',\n",
              "  1),\n",
              " (573,\n",
              "  5,\n",
              "  'Are there telemedicine applications for chronic pain management?',\n",
              "  'therapy adaptation and the resultant specification for the SMART2 project-a technology-based self-management system for assisting long-term health conditions, including chronic pain',\n",
              "  0),\n",
              " (573,\n",
              "  6,\n",
              "  'Are there telemedicine applications for chronic pain management?',\n",
              "  'Results showed the use of videoconferencing for this group of patients is useable and satisfactory for both patients and staff, that the patients save time and money, and that for a system where videoconferencing equipment is already in use, it is also cost effective.',\n",
              "  1),\n",
              " (573,\n",
              "  7,\n",
              "  'Are there telemedicine applications for chronic pain management?',\n",
              "  'Staff were able to identify new patient problems.',\n",
              "  0),\n",
              " (573,\n",
              "  8,\n",
              "  'Are there telemedicine applications for chronic pain management?',\n",
              "  'This pilot study indicates that telemedicine follow-up consultations for chronic pain patients are feasible and cost-saving.',\n",
              "  1),\n",
              " (573,\n",
              "  9,\n",
              "  'Are there telemedicine applications for chronic pain management?',\n",
              "  'Patients and anesthesiologists were highly satisfied with telemedicine consultation.',\n",
              "  1),\n",
              " (4139,\n",
              "  0,\n",
              "  'What methodology does the FoundationOne CDx test use?',\n",
              "  'Genetic alterations and TMB were determined by FoundationOne CDx next generation sequencing (NGS) and the association with clinicopathologic features was analyzed.',\n",
              "  1),\n",
              " (145,\n",
              "  0,\n",
              "  'Are there drugs for Tick-borne Encephalitis?',\n",
              "  ' tick-borne encephalitis (TBE) have been detected in Bulgaria.',\n",
              "  1),\n",
              " (145,\n",
              "  1,\n",
              "  'Are there drugs for Tick-borne Encephalitis?',\n",
              "  'Considering the remarkable increase in TBE morbidity in Europe over the past two decades, we conducted a study of TBE among patients with acute viral meningitis who were hospitalised in Bulgaria during 2009 to 2012.',\n",
              "  1),\n",
              " (145,\n",
              "  2,\n",
              "  'Are there drugs for Tick-borne Encephalitis?',\n",
              "  'Viruses belonging to the Flaviviridae family primarily spread through arthropod vectors, and are the major causes of illness and death around the globe.',\n",
              "  1),\n",
              " (145,\n",
              "  3,\n",
              "  'Are there drugs for Tick-borne Encephalitis?',\n",
              "  'The Flaviviridae family consists of 3 genera which include the Flavivirus genus (type species, yellow fever virus) as the largest genus, the Hepacivirus (type species, hepatitis C virus) and the Pestivirus (type species, bovine virus diarrhea).',\n",
              "  1),\n",
              " (145,\n",
              "  4,\n",
              "  'Are there drugs for Tick-borne Encephalitis?',\n",
              "  'The flaviviruses (Flavivirus genus) are small RNA viruses transmitted by mosquitoes and ticks that take over host cell machinery in order to propagate.',\n",
              "  0),\n",
              " (145,\n",
              "  5,\n",
              "  'Are there drugs for Tick-borne Encephalitis?',\n",
              "  'However, hepaciviruses and pestiviruses are not antropod-borne.',\n",
              "  0),\n",
              " (145,\n",
              "  6,\n",
              "  'Are there drugs for Tick-borne Encephalitis?',\n",
              "  'Despite the extensive research and public health concern associated with flavivirus diseases, to date, there is no specific treatment available for any flavivirus infections, though commercially available vaccines for yellow fever, Japanese encephalitis and tick-born encephalitis exist.',\n",
              "  1),\n",
              " (805,\n",
              "  0,\n",
              "  'Which intermediate filament (IF) protein can be used as a non-specific marker of the neuronal precursor cells of the subventricular zone?',\n",
              "  'Nestin is a unique intermediate filament protein.',\n",
              "  1),\n",
              " (805,\n",
              "  1,\n",
              "  'Which intermediate filament (IF) protein can be used as a non-specific marker of the neuronal precursor cells of the subventricular zone?',\n",
              "  \"While it is robustly expressed in developing brain, postnatal expression is limited to the brain's subventricular zone (SVZ)\",\n",
              "  1),\n",
              " (805,\n",
              "  2,\n",
              "  'Which intermediate filament (IF) protein can be used as a non-specific marker of the neuronal precursor cells of the subventricular zone?',\n",
              "  'nestin, a marker protein for precursor cells in the subventricular zone',\n",
              "  1),\n",
              " (805,\n",
              "  3,\n",
              "  'Which intermediate filament (IF) protein can be used as a non-specific marker of the neuronal precursor cells of the subventricular zone?',\n",
              "  'adult subventricular zone (SVZ) stem and progenitor cells express nestin',\n",
              "  0),\n",
              " (805,\n",
              "  4,\n",
              "  'Which intermediate filament (IF) protein can be used as a non-specific marker of the neuronal precursor cells of the subventricular zone?',\n",
              "  'the typical protein of neural progenitors, nestin',\n",
              "  0),\n",
              " (805,\n",
              "  5,\n",
              "  'Which intermediate filament (IF) protein can be used as a non-specific marker of the neuronal precursor cells of the subventricular zone?',\n",
              "  'the nonspecific precursor cell marker Nestin',\n",
              "  0),\n",
              " (805,\n",
              "  6,\n",
              "  'Which intermediate filament (IF) protein can be used as a non-specific marker of the neuronal precursor cells of the subventricular zone?',\n",
              "  'In the subventricular zone, this effect was exerted selectively on a precursor subpopulation expressing nestin but not neuronal or glial cell-specific proteins.',\n",
              "  0),\n",
              " (805,\n",
              "  7,\n",
              "  'Which intermediate filament (IF) protein can be used as a non-specific marker of the neuronal precursor cells of the subventricular zone?',\n",
              "  'Nestin is an intermediate filament protein expressed in neuroepithelial stem cells during development and it is later replaced by cell specific neuronal or glial filaments.',\n",
              "  0),\n",
              " (805,\n",
              "  8,\n",
              "  'Which intermediate filament (IF) protein can be used as a non-specific marker of the neuronal precursor cells of the subventricular zone?',\n",
              "  'The results indicate that administration of thyroid hormone and retinoic acid increases the expression of Ki67, a nuclear antigen associated with cell proliferation, and of nestin, a marker protein for precursor cells in the subventricular zone of adult male rats.',\n",
              "  1),\n",
              " (805,\n",
              "  9,\n",
              "  'Which intermediate filament (IF) protein can be used as a non-specific marker of the neuronal precursor cells of the subventricular zone?',\n",
              "  'Only a minority of stem cells expressed nestin, a marker for neural precursor cells.',\n",
              "  0),\n",
              " (805,\n",
              "  10,\n",
              "  'Which intermediate filament (IF) protein can be used as a non-specific marker of the neuronal precursor cells of the subventricular zone?',\n",
              "  'In addition, bone marrow transplantation promoted proliferation of ependymal and subependymal cells, identified by nestin (a neuroepithelial stem cell marker), within the ventricular zone and subventricular zone (VZ/SVZ).',\n",
              "  0),\n",
              " (805,\n",
              "  11,\n",
              "  'Which intermediate filament (IF) protein can be used as a non-specific marker of the neuronal precursor cells of the subventricular zone?',\n",
              "  'The results indicate that administration of thyroid hormone and retinoic acid increases the expression of Ki67, a nuclear antigen associated with cell proliferation, and of nestin, a marker protein for precursor cells in the subventricular zone of adult male rats',\n",
              "  1),\n",
              " (805,\n",
              "  12,\n",
              "  'Which intermediate filament (IF) protein can be used as a non-specific marker of the neuronal precursor cells of the subventricular zone?',\n",
              "  'In all animals, 20-40% of the newly generated cells in the dentate gyrus and subventricular zone expressed the neural progenitor cell markers Musashi1 or Nestin',\n",
              "  0),\n",
              " (2349,\n",
              "  0,\n",
              "  'Do chromatin features predict genes associated with eQTLs?',\n",
              "  'Using the random forest classifier, we found that genomic proximity plus five TF and chromatin features are able to predict>90% of target genes within 1 megabase of eQTLs',\n",
              "  1),\n",
              " (2349,\n",
              "  1,\n",
              "  'Do chromatin features predict genes associated with eQTLs?',\n",
              "  'Using the random forest classifier, we found that genomic proximity plus five TF and chromatin features are able to predict >90% of target genes within 1 megabase of eQTLs.',\n",
              "  1),\n",
              " (2349,\n",
              "  2,\n",
              "  'Do chromatin features predict genes associated with eQTLs?',\n",
              "  'Cell type-specific gene expression in humans involves complex interactions between regulatory factors and DNA at enhancers and promoters.',\n",
              "  0),\n",
              " (2349,\n",
              "  3,\n",
              "  'Do chromatin features predict genes associated with eQTLs?',\n",
              "  'Mapping studies for expression quantitative trait loci (eQTLs), transcription factors (TFs) and chromatin markers have become widely used tools for identifying gene regulatory elements, but prediction of target genes remains a major challenge.',\n",
              "  0),\n",
              " (2349,\n",
              "  4,\n",
              "  'Do chromatin features predict genes associated with eQTLs?',\n",
              "  'Here, we integrate genome-wide data on TF-binding sites, chromatin markers and functional annotations to predict genes associated with human eQTLs.',\n",
              "  1),\n",
              " (2349,\n",
              "  5,\n",
              "  'Do chromatin features predict genes associated with eQTLs?',\n",
              "  'Using the random forest classifier, we found that genomic proximity plus five TF and chromatin features are able to predict >90% of target genes within 1 megabase of eQTLs.',\n",
              "  1),\n",
              " (2349,\n",
              "  6,\n",
              "  'Do chromatin features predict genes associated with eQTLs?',\n",
              "  'Despite being regularly used to map target genes, proximity is not a good indicator of eQTL targets for genes 150 kilobases away, but insulators, TF co-occurrence, open chromatin and functional similarities between TFs and genes are better indicators.',\n",
              "  1),\n",
              " (2349,\n",
              "  7,\n",
              "  'Do chromatin features predict genes associated with eQTLs?',\n",
              "  'Using all six features in the classifier achieved an area under the specificity and sensitivity curve of 0.91, much better compared with at most 0.75 for using any single feature.',\n",
              "  0),\n",
              " (2349,\n",
              "  8,\n",
              "  'Do chromatin features predict genes associated with eQTLs?',\n",
              "  'We hope this study will not only provide validation of eQTL-mapping studies, but also provide insight into the molecular mechanisms explaining how genetic variation can influence gene expression.',\n",
              "  0),\n",
              " (2801,\n",
              "  0,\n",
              "  'Is ibudilast effective for multiple sclerosis?',\n",
              "  'Ibudilast slowed brain atrophy in PPMS and SPMS patients in a multicenter phase 2b study.',\n",
              "  1),\n",
              " (2801,\n",
              "  1,\n",
              "  'Is ibudilast effective for multiple sclerosis?',\n",
              "  ' Ibudilast inhibits several cyclic nucleotide phosphodiesterases, macrophage migration inhibitory factor, and toll-like receptor 4 and can cross the blood-brain barrier, with potential salutary effects in progressive multiple sclerosis.',\n",
              "  0),\n",
              " (2801,\n",
              "  2,\n",
              "  'Is ibudilast effective for multiple sclerosis?',\n",
              "  'CONCLUSIONS: In a phase 2 trial involving patients with progressive multiple sclerosis, ibudilast was associated with slower progression of brain atrophy than placebo but was associated with higher rates of gastrointestinal side effects, headache, and depression.',\n",
              "  1),\n",
              " (2801,\n",
              "  3,\n",
              "  'Is ibudilast effective for multiple sclerosis?',\n",
              "  'Specifically, the current evidence regarding treatment of progressive MS with ocrelizumab, simvastatin, ibudilast, alpha-lipoic acid, high-dose biotin, siponimod, and cell-based therapies are discussed.',\n",
              "  0),\n",
              " (2801,\n",
              "  4,\n",
              "  'Is ibudilast effective for multiple sclerosis?',\n",
              "  'Based on our knowledge of pathophysiology, three therapeutic strategies are proposed: anti-inflammatory (ocrelizumab, siponimod…); remyelinating (opicinumab); and neuroprotective (high-dose biotin, ibudilast, simvastatin…).',\n",
              "  0),\n",
              " (2801,\n",
              "  5,\n",
              "  'Is ibudilast effective for multiple sclerosis?',\n",
              "  'Current article provides an overview of the pharmacology of IBD with a focus on preclinical and clinical data supporting its potential neuroprotective benefits for neurological conditions, including multiple sclerosis, neuropathic pain, medication overuse headache, stroke, opioid, alcohol and methamphetamine abuse.',\n",
              "  0),\n",
              " (2801,\n",
              "  6,\n",
              "  'Is ibudilast effective for multiple sclerosis?',\n",
              "  'Design, rationale, and baseline characteristics of the randomized double-blind phase II clinical trial of ibudilast in progressive multiple sclerosis.',\n",
              "  1),\n",
              " (2801,\n",
              "  7,\n",
              "  'Is ibudilast effective for multiple sclerosis?',\n",
              "  'METHODS: SPRINT-MS is a randomized, placebo-controlled, phase II trial of ibudilast in patients with PMS.',\n",
              "  1),\n",
              " (2801,\n",
              "  8,\n",
              "  'Is ibudilast effective for multiple sclerosis?',\n",
              "  'CONCLUSION: SPRINT-MS is designed to evaluate the safety and efficacy of ibudilast as a treatment for PMS while simultaneously validating five different imaging biomarkers as outcome metrics for use in future phase II proof-of-concept PMS trials.',\n",
              "  0),\n",
              " (2801,\n",
              "  9,\n",
              "  'Is ibudilast effective for multiple sclerosis?',\n",
              "  'Ibudilast for the treatment of multiple sclerosis.',\n",
              "  0),\n",
              " (2801,\n",
              "  10,\n",
              "  'Is ibudilast effective for multiple sclerosis?',\n",
              "  'AREAS COVERED: This article reviews various studies looking at ibudilast as a potential therapy for MS.',\n",
              "  0),\n",
              " (2801,\n",
              "  11,\n",
              "  'Is ibudilast effective for multiple sclerosis?',\n",
              "  'It summarizes prior and current clinical trials of ibudilast in MS as well as its pharmacology.EXPERT OPINION: Although ibudilast has not been found to decrease the focal inflammatory activity in relapsing MS, it was shown to have an effect on preserving brain volume and disability progression.',\n",
              "  0),\n",
              " (2801,\n",
              "  12,\n",
              "  'Is ibudilast effective for multiple sclerosis?',\n",
              "  'Ibudilast may have a role in the treatment of progressive MS phenotypes.',\n",
              "  0),\n",
              " (2801,\n",
              "  13,\n",
              "  'Is ibudilast effective for multiple sclerosis?',\n",
              "  'Adverse events with ibudilast included gastrointestinal symptoms, headache, and depression.<br><b>CONCLUSIONS</b>: In a phase 2 trial involving patients with progressive multiple sclerosis, ibudilast was associated with slower progression of brain atrophy than placebo but was associated with higher rates of gastrointestinal side effects, headache, and depression.',\n",
              "  1),\n",
              " (2801,\n",
              "  14,\n",
              "  'Is ibudilast effective for multiple sclerosis?',\n",
              "  'It summarizes prior and current clinical trials of ibudilast in MS as well as its pharmacology.<br><b>EXPERT OPINION</b>: Although ibudilast has not been found to decrease the focal inflammatory activity in relapsing MS, it was shown to have an effect on preserving brain volume and disability progression.',\n",
              "  0),\n",
              " (2752,\n",
              "  0,\n",
              "  'Which algorithm has been developed for the automatic extraction of co-expressed gene clusters from gene expression data?',\n",
              "  'Clust: automatic extraction of optimal co-expressed gene clusters from gene expression data.',\n",
              "  1),\n",
              " (2752,\n",
              "  1,\n",
              "  'Which algorithm has been developed for the automatic extraction of co-expressed gene clusters from gene expression data?',\n",
              "  'We present clust, a method that solves these problems by extracting clusters matching the biological expectations of co-expressed genes and outperforms widely used methods.',\n",
              "  1),\n",
              " (2752,\n",
              "  2,\n",
              "  'Which algorithm has been developed for the automatic extraction of co-expressed gene clusters from gene expression data?',\n",
              "  'Additionally, clust can simultaneously cluster multiple datasets, enabling users to leverage the large quantity of public expression data for novel comparative analysis.',\n",
              "  1),\n",
              " (2752,\n",
              "  3,\n",
              "  'Which algorithm has been developed for the automatic extraction of co-expressed gene clusters from gene expression data?',\n",
              "  'Clust is available at https://github.com/BaselAbujamous/clust .',\n",
              "  1),\n",
              " (2752,\n",
              "  4,\n",
              "  'Which algorithm has been developed for the automatic extraction of co-expressed gene clusters from gene expression data?',\n",
              "  'We present clust, a method that solves these problems by extracting clusters matching the biological expectations of co-expressed genes and outperforms widely used methods.',\n",
              "  1),\n",
              " (2752,\n",
              "  5,\n",
              "  'Which algorithm has been developed for the automatic extraction of co-expressed gene clusters from gene expression data?',\n",
              "  'Additionally, clust can simultaneously cluster multiple datasets, enabling users to leverage the large quantity of public expression data for novel comparative analysis.',\n",
              "  0),\n",
              " (2750,\n",
              "  0,\n",
              "  'Which species is the carrier of the SFTS ( severe fever with thrombocytopenia syndrome) virus?',\n",
              "  'Seroprevalence in animal species were: goats (66.8%), cattle (28.2%), dogs (7.4%), pigs (4.7%), chickens (1.2%), geese (1.7%), rodents (4.4%) and hedgehogs (2.7%).',\n",
              "  1),\n",
              " (2750,\n",
              "  1,\n",
              "  'Which species is the carrier of the SFTS ( severe fever with thrombocytopenia syndrome) virus?',\n",
              "  'The possibility that SFTSV transmission may occur by both the transstadial and transovarial routes was suggested by the fact that viral RNA was detected in H. longicornis at all developmental stages.',\n",
              "  1),\n",
              " (2750,\n",
              "  2,\n",
              "  'Which species is the carrier of the SFTS ( severe fever with thrombocytopenia syndrome) virus?',\n",
              "  'Tick-derived sequences shared over 95.6% identity with human- and animal-derived isolates.',\n",
              "  1),\n",
              " (2750,\n",
              "  3,\n",
              "  'Which species is the carrier of the SFTS ( severe fever with thrombocytopenia syndrome) virus?',\n",
              "  'This study provides evidence that implicates ticks as not only vectors but also, reservoirs of SFTSV.',\n",
              "  1),\n",
              " (2750,\n",
              "  4,\n",
              "  'Which species is the carrier of the SFTS ( severe fever with thrombocytopenia syndrome) virus?',\n",
              "  'Severe fever with thrombocytopenia syndrome (SFTS) is a new emerging zoonosis.',\n",
              "  1),\n",
              " (2750,\n",
              "  5,\n",
              "  'Which species is the carrier of the SFTS ( severe fever with thrombocytopenia syndrome) virus?',\n",
              "  'A survey of reptile-associated ticks and their infection status with severe fever with thrombocytopenia syndrome (SFTS) virus was conducted to determine the relative abundance and distribution among lizards, skinks, and snakes in the Republic of Korea (ROK).',\n",
              "  0),\n",
              " (2750,\n",
              "  6,\n",
              "  'Which species is the carrier of the SFTS ( severe fever with thrombocytopenia syndrome) virus?',\n",
              "  'In total, 84 ixodid ticks belonging to two genera (Ixodes and Amblyomma) were collected from 28/132 (21.2%) lizards, skinks, and snakes.',\n",
              "  0),\n",
              " (2750,\n",
              "  7,\n",
              "  'Which species is the carrier of the SFTS ( severe fever with thrombocytopenia syndrome) virus?',\n",
              "  'These results implicate lizards and snakes as potential hosts of SFTS virus.',\n",
              "  0),\n",
              " (2750,\n",
              "  8,\n",
              "  'Which species is the carrier of the SFTS ( severe fever with thrombocytopenia syndrome) virus?',\n",
              "  'Haemophysalis longicornis ticks have been considered the vector of severe fever with thrombocytopenia syndrome virus (SFTSV).',\n",
              "  0),\n",
              " (2750,\n",
              "  9,\n",
              "  'Which species is the carrier of the SFTS ( severe fever with thrombocytopenia syndrome) virus?',\n",
              "  'The patient gradually recovered with treatments of corticosteroids and immunoglobulin.<br><b>CONCLUSION</b>: These findings provide further evidence of SFTS viral transmission from H. longicornis to human.<br>',\n",
              "  0),\n",
              " (3536,\n",
              "  0,\n",
              "  'Is there an increased risk of meningiomas in atomic bomb survivors?',\n",
              "  'RESULTS: Meningioma was the most common tumor among clinically diagnosed tumors, followed by neuroepithelial tumor, schwannoma, and pituitary tumor.',\n",
              "  0),\n",
              " (3536,\n",
              "  1,\n",
              "  'Is there an increased risk of meningiomas in atomic bomb survivors?',\n",
              "  'The predominance of meningiomas over neuroepithelial tumors in the Japanese population was noteworthy and warrants further investigation.',\n",
              "  1),\n",
              " (3536,\n",
              "  2,\n",
              "  'Is there an increased risk of meningiomas in atomic bomb survivors?',\n",
              "  'Risk increases, although not statistically significant, were seen for meningiomas (ERR(Sv) = 0.6, 95% CI = -0.01 to 1.8), gliomas (ERR(Sv) = 0.6, 95% CI = -0.2 to 2.0), other nervous system tumors (ERR(Sv) = 0.5, 95% CI = <-0.2 to 2.2), and pituitary tumors (ERR(Sv) = 1.0, 95% CI = <-0.2 to 3.5).',\n",
              "  0),\n",
              " (3536,\n",
              "  3,\n",
              "  'Is there an increased risk of meningiomas in atomic bomb survivors?',\n",
              "  'High incidence of meningioma among Hiroshima atomic bomb survivors.',\n",
              "  1),\n",
              " (3536,\n",
              "  4,\n",
              "  'Is there an increased risk of meningiomas in atomic bomb survivors?',\n",
              "  'The incidence of meningioma among Hiroshima atomic bomb survivors has increased since 1975.',\n",
              "  1),\n",
              " (3536,\n",
              "  5,\n",
              "  'Is there an increased risk of meningiomas in atomic bomb survivors?',\n",
              "  'There was a significant correlation between the incidence and the dose of radiation to the brain.',\n",
              "  0),\n",
              " (3536,\n",
              "  6,\n",
              "  'Is there an increased risk of meningiomas in atomic bomb survivors?',\n",
              "  'The present findings strongly suggest that meningioma is one of the tumors induced by atomic bombing in Hiroshima.',\n",
              "  0),\n",
              " (3536,\n",
              "  7,\n",
              "  'Is there an increased risk of meningiomas in atomic bomb survivors?',\n",
              "  'Incidence of intracranial meningiomas in Nagasaki atomic-bomb survivors.',\n",
              "  1),\n",
              " (3536,\n",
              "  8,\n",
              "  'Is there an increased risk of meningiomas in atomic bomb survivors?',\n",
              "  'The analysis showed a high correlation between incidence of meningiomas and distance from the hypocenter.',\n",
              "  0),\n",
              " (3536,\n",
              "  9,\n",
              "  'Is there an increased risk of meningiomas in atomic bomb survivors?',\n",
              "  'The incidence among Nagasaki atomic-bomb survivors over 40 years of age, especially in those proximally exposed, appears to be increasing, in inverse proportion to the exposure distance, since 1981, 36 years after the explosion of the atomic bomb.',\n",
              "  0),\n",
              " (3536,\n",
              "  10,\n",
              "  'Is there an increased risk of meningiomas in atomic bomb survivors?',\n",
              "  'The incidence of meningioma among Hiroshima atomic bomb survivors has increased since 1975.',\n",
              "  1),\n",
              " (3536,\n",
              "  11,\n",
              "  'Is there an increased risk of meningiomas in atomic bomb survivors?',\n",
              "  'The present findings strongly suggest that meningioma is one of the tumors induced by atomic bombing in Hiroshima.',\n",
              "  0),\n",
              " (3536,\n",
              "  12,\n",
              "  'Is there an increased risk of meningiomas in atomic bomb survivors?',\n",
              "  'The incidences of meningioma among the survivors of Hiroshima in 5-year intervals since 1975 were 5.3, 7.4, 10.1, and 14.9, respectively.',\n",
              "  0),\n",
              " (3536,\n",
              "  13,\n",
              "  'Is there an increased risk of meningiomas in atomic bomb survivors?',\n",
              "  'The analysis showed a high correlation between incidence of meningiomas and distance from the hypocenter.',\n",
              "  0),\n",
              " (666,\n",
              "  0,\n",
              "  'Name the major classes of small  non coding RNAs in mammalians?',\n",
              "  'Some of these RNA classes, in particular microRNAs and snoRNAs, undergo maturation processes that lead to the production of shorter RNAs.',\n",
              "  1),\n",
              " (3897,\n",
              "  0,\n",
              "  \"Can Freund's complete adjuvant induce arthritis?\",\n",
              "  \"complete Freund's adjuvant (CFA) induced RA\",\n",
              "  1),\n",
              " (3897,\n",
              "  1,\n",
              "  \"Can Freund's complete adjuvant induce arthritis?\",\n",
              "  \"The RA model was established using Freund's complete adjuvant,\",\n",
              "  1),\n",
              " (3897,\n",
              "  2,\n",
              "  \"Can Freund's complete adjuvant induce arthritis?\",\n",
              "  \" Rheumatoid arthritis (RA) was induced by Freund's Complete Adjuvant (FCA; 1\\xa0mg/0.1\\xa0ml paraffin oil), injected subcutaneously on days 0, 30 and 40\",\n",
              "  1),\n",
              " (3897,\n",
              "  3,\n",
              "  \"Can Freund's complete adjuvant induce arthritis?\",\n",
              "  \"The rats were made arthritic using a subcutaneous injection with 0.1\\u2009ml complete Freund's adjuvant (CFA) into the footpad of the left hind paw.\",\n",
              "  1),\n",
              " (940,\n",
              "  0,\n",
              "  'What is the main application of SWATH-MS in proteomics?',\n",
              "  'it is a valuable resource for the selection of candidate proteotypic peptides for targeted proteomic experiments via Selected Reaction Monitoring (SRM) or SWATH-MS',\n",
              "  1),\n",
              " (940,\n",
              "  1,\n",
              "  'What is the main application of SWATH-MS in proteomics?',\n",
              "  'SWATH-MS is a data-independent acquisition method that generates, in a single measurement, a complete recording of the fragment ion spectra of all the analytes in a biological sample for which the precursor ions are within a predetermined m/z versus retention time window.',\n",
              "  1),\n",
              " (940,\n",
              "  2,\n",
              "  'What is the main application of SWATH-MS in proteomics?',\n",
              "  \"ew strategies, such as SWATH™ MS, which allows us to systematically characterize and quantify query sample sets of 'any protein of interest' in complex biological samples,\",\n",
              "  1),\n",
              " (940,\n",
              "  3,\n",
              "  'What is the main application of SWATH-MS in proteomics?',\n",
              "  'Here we present a new strategy that systematically queries sample sets for the presence and quantity of essentially any protein of interest.',\n",
              "  1),\n",
              " (709,\n",
              "  0,\n",
              "  'What is HOCOMOCO?',\n",
              "  'HOCOMOCO: a comprehensive collection of human transcription factor binding sites models.',\n",
              "  0),\n",
              " (709,\n",
              "  1,\n",
              "  'What is HOCOMOCO?',\n",
              "  'We present the Homo sapiens comprehensive model collection (HOCOMOCO, http://autosome.ru/HOCOMOCO/, http://cbrc.kaust.edu.sa/hocomoco/) containing carefully hand-curated TFBS models constructed by integration of binding sequences obtained by both low- and high-throughput methods.',\n",
              "  1),\n",
              " (709,\n",
              "  2,\n",
              "  'What is HOCOMOCO?',\n",
              "  'To construct position weight matrices to represent these TFBS models, we used ChIPMunk software in four computational modes, including newly developed periodic positional prior mode associated with DNA helix pitch.',\n",
              "  0),\n",
              " (709,\n",
              "  3,\n",
              "  'What is HOCOMOCO?',\n",
              "  'We selected only one TFBS model per TF, unless there was a clear experimental evidence for two rather distinct TFBS models.',\n",
              "  0),\n",
              " (709,\n",
              "  4,\n",
              "  'What is HOCOMOCO?',\n",
              "  'We assigned a quality rating to each model.',\n",
              "  0),\n",
              " (709,\n",
              "  5,\n",
              "  'What is HOCOMOCO?',\n",
              "  'HOCOMOCO contains 426 systematically curated TFBS models for 401 human TFs, where 172 models are based on more than one data source.',\n",
              "  1),\n",
              " (709,\n",
              "  6,\n",
              "  'What is HOCOMOCO?',\n",
              "  'We present the Homo sapiens comprehensive model collection (HOCOMOCO, http://autosome.ru/HOCOMOCO/, http://cbrc.kaust.edu.sa/hocomoco/) containing carefully hand-curated TFBS models constructed by integration of binding sequences obtained by both low- and high-throughput methods',\n",
              "  1),\n",
              " (709,\n",
              "  7,\n",
              "  'What is HOCOMOCO?',\n",
              "  'HOCOMOCO contains 426 systematically curated TFBS models for 401 human TFs, where 172 models are based on more than one data source',\n",
              "  1),\n",
              " (709,\n",
              "  8,\n",
              "  'What is HOCOMOCO?',\n",
              "  'HOCOMOCO: a comprehensive collection of human transcription factor binding sites models',\n",
              "  0),\n",
              " (709,\n",
              "  9,\n",
              "  'What is HOCOMOCO?',\n",
              "  'We show that integration of TFBS data from various types of experiments into a single model typically results in the improved model quality probably due to partial correction of source specific technique bias.',\n",
              "  0),\n",
              " (709,\n",
              "  10,\n",
              "  'What is HOCOMOCO?',\n",
              "  'We present the Homo sapiens comprehensive model collection (HOCOMOCO, http://autosome.ru/HOCOMOCO/, http://cbrc.kaust.edu.sa/hocomoco/) containing carefully hand-curated TFBS models constructed by integration of binding sequences obtained by both low- and high-throughput methods.',\n",
              "  1),\n",
              " (709,\n",
              "  11,\n",
              "  'What is HOCOMOCO?',\n",
              "  'To construct position weight matrices to represent these TFBS models, we used ChIPMunk software in four computational modes, including newly developed periodic positional prior mode associated with DNA helix pitch.',\n",
              "  0),\n",
              " (557,\n",
              "  0,\n",
              "  'List Parkin binding partners',\n",
              "  'In addition to two known binding partners (HSP90, CDC37), 12 proteins were identified using the TAP assay; four of which are mitochondrially localized (GRP75, HSP60, LRPPRC, and TUFM).',\n",
              "  1),\n",
              " (557,\n",
              "  1,\n",
              "  'List Parkin binding partners',\n",
              "  'We find that parkin binds PICK1 via a PDZ-mediated interaction, which predominantly promotes PICK1 monoubiquitination rather than polyubiquitination.',\n",
              "  1),\n",
              " (557,\n",
              "  2,\n",
              "  'List Parkin binding partners',\n",
              "  'We have identified the 20S proteasomal subunit alpha4 (synonyms: PSMA7, XAPC7, subunit alpha type 7) as a new interacting partner of parkin.',\n",
              "  1),\n",
              " (557,\n",
              "  3,\n",
              "  'List Parkin binding partners',\n",
              "  'we cloned parkin-binding protein using a yeast two-hybrid system and identified a putative G protein-coupled receptor protein,which we named the Pael receptor (Pael-R)',\n",
              "  1),\n",
              " (557,\n",
              "  4,\n",
              "  'List Parkin binding partners',\n",
              "  \", whose interactions with CASK are reviewed here, include the Parkinson's disease molecule parkin,\",\n",
              "  1),\n",
              " (2335,\n",
              "  0,\n",
              "  'What is the function of the gene MDA5?',\n",
              "  'Sensing of viral RNA by the cytosolic receptors RIG-I and melanoma differentiation-associated gene 5 (MDA5) leads to innate antiviral response.',\n",
              "  1),\n",
              " (2335,\n",
              "  1,\n",
              "  'What is the function of the gene MDA5?',\n",
              "  'The host innate immune response to RNA virus infection primarily involves pathogen-sensing toll-like receptors (TLRs) TLR3 and TLR7 and retinoic acid-inducible gene I-like receptor RIG-I and melanoma differentiation associated gene 5 (MDA5)',\n",
              "  0),\n",
              " (2335,\n",
              "  2,\n",
              "  'What is the function of the gene MDA5?',\n",
              "  'RNA PRRs are comprised of TLR3, TLR7, TLR8, RIG-I, MDA5, NLRP3, NOD2, and some other minorities.',\n",
              "  0),\n",
              " (2335,\n",
              "  3,\n",
              "  'What is the function of the gene MDA5?',\n",
              "  'Melanoma differentiation-associated gene 5 (MDA5) is a pattern recognition receptor that recognizes cytoplasmic viral double-stranded RNA (dsRNA) and initiates rapid innate antiviral responses.',\n",
              "  1),\n",
              " (2335,\n",
              "  4,\n",
              "  'What is the function of the gene MDA5?',\n",
              "  'MDA5 forms a filament-like multimer along the dsRNA leading to oligomerization, which in turn activates the adaptor protein mitochondrial antiviral signaling protein (MAVS) to provide a signal platform for the induction of type I interferon (IFN) and proinflammatory cytokines.',\n",
              "  1),\n",
              " (2335,\n",
              "  5,\n",
              "  'What is the function of the gene MDA5?',\n",
              "  'The conformational switch of MDA5 causes antiviral defense, but excessive activation of the MDA5-MAVS pathway may result in autoimmune diseases.',\n",
              "  1),\n",
              " (2335,\n",
              "  6,\n",
              "  'What is the function of the gene MDA5?',\n",
              "  'Retinoic acid-inducible gene-I (RIG-I)-like receptors (RLRs), including melanoma differentiation-associated gene 5 (MDA5) and RIG-I, are crucial for host recognition of non-self RNAs, especially viral RNA.',\n",
              "  1),\n",
              " (4107,\n",
              "  0,\n",
              "  'Which molecule is targeted by Upadacitinib?',\n",
              "  'We summarise the development and recent application of Janus kinase (JAK) inhibitors in the treatment of IMIDs, including first-generation pan-JAK inhibitors (tofacitinib, baricitinib, ruxolitinib, peficitinib) and second-generation selective JAK inhibitors (decernotinib, filgotinib, upadacitinib).',\n",
              "  0),\n",
              " (4107,\n",
              "  1,\n",
              "  'Which molecule is targeted by Upadacitinib?',\n",
              "  'Upadacitinib is a Janus kinase 1 inhibitor currently being evaluated in phase III rheumatoid arthritis trials.',\n",
              "  0),\n",
              " (4107,\n",
              "  2,\n",
              "  'Which molecule is targeted by Upadacitinib?',\n",
              "  'BACKGROUND AND OBJECTIVES: Upadacitinib is a janus kinase (JAK) 1 inhibitor being developed for the treatment of rheumatoid arthritis (RA) and other inflammatory diseases.',\n",
              "  0),\n",
              " (4107,\n",
              "  3,\n",
              "  'Which molecule is targeted by Upadacitinib?',\n",
              "  'RESULTS: In randomised controlled trials primary efficacy endpoints were met for tofacitinib (JAK 1/3 inhibitor-phase III), upadacitinib (JAK 1 inhibitor-phase II) and AJM300 (α4-integrin antagonist-phase II) in ulcerative colitis.',\n",
              "  0),\n",
              " (4107,\n",
              "  4,\n",
              "  'Which molecule is targeted by Upadacitinib?',\n",
              "  'Upadacitinib is a Janus kinase 1 inhibitor under development for the treatment of several inflammatory disorders including rheumatoid arthritis (RA).',\n",
              "  1),\n",
              " (4107,\n",
              "  5,\n",
              "  'Which molecule is targeted by Upadacitinib?',\n",
              "  'Upadacitinib selectively inhibits Janus Kinase 1 (JAK1) which could potentially reduce JAK2 and JAK3-related side effects.',\n",
              "  0),\n",
              " (4107,\n",
              "  6,\n",
              "  'Which molecule is targeted by Upadacitinib?',\n",
              "  'BACKGROUND\\nPhase 2 studies with upadacitinib, a selective Janus kinase 1 (JAK1) inhibitor, have shown safety and efficacy in the treatment of patients with active rheumatoid arthritis.',\n",
              "  0),\n",
              " (4107,\n",
              "  7,\n",
              "  'Which molecule is targeted by Upadacitinib?',\n",
              "  'BACKGROUND\\nUpadacitinib is a selective inhibitor of Janus kinase 1 and was efficacious in phase 2 studies in patients with moderate-to-severe rheumatoid arthritis.',\n",
              "  0),\n",
              " (4107,\n",
              "  8,\n",
              "  'Which molecule is targeted by Upadacitinib?',\n",
              "  'Upadacitinib is a novel selective oral Janus kinase 1 (JAK) inhibitor being developed for treatment of several inflammatory diseases.',\n",
              "  0),\n",
              " (4107,\n",
              "  9,\n",
              "  'Which molecule is targeted by Upadacitinib?',\n",
              "  'BACKGROUND AND OBJECTIVES\\nUpadacitinib is a janus kinase (JAK) 1 inhibitor being developed for the treatment of rheumatoid arthritis (RA) and other inflammatory diseases.',\n",
              "  0),\n",
              " (4107,\n",
              "  10,\n",
              "  'Which molecule is targeted by Upadacitinib?',\n",
              "  'AIMS\\nUpadacitinib (ABT-494) is a selective Janus kinase 1 inhibitor being developed for treatment of auto-immune inflammatory disorders.',\n",
              "  0),\n",
              " (4107,\n",
              "  11,\n",
              "  'Which molecule is targeted by Upadacitinib?',\n",
              "  'RESULTS\\nIn randomised controlled trials primary efficacy endpoints were met for tofacitinib (JAK 1/3 inhibitor-phase III), upadacitinib (JAK 1 inhibitor-phase II) and AJM300 (α4-integrin antagonist-phase II) in ulcerative colitis.',\n",
              "  0),\n",
              " (4107,\n",
              "  12,\n",
              "  'Which molecule is targeted by Upadacitinib?',\n",
              "  'AIMS Upadacitinib (ABT-494) is a selective Janus kinase 1 inhibitor being developed for treatment of auto-immune inflammatory disorders.',\n",
              "  0),\n",
              " (4107,\n",
              "  13,\n",
              "  'Which molecule is targeted by Upadacitinib?',\n",
              "  'BACKGROUND Upadacitinib is a selective inhibitor of Janus kinase 1 and was efficacious in phase 2 studies in patients with moderate-to-severe rheumatoid arthritis.',\n",
              "  0),\n",
              " (4107,\n",
              "  14,\n",
              "  'Which molecule is targeted by Upadacitinib?',\n",
              "  'Upadacitinib selectively inhibits Janus Kinase 1 (JAK1) which could potentially reduce JAK2 and JAK3-related side effects.',\n",
              "  0),\n",
              " (4107,\n",
              "  15,\n",
              "  'Which molecule is targeted by Upadacitinib?',\n",
              "  'Expert commentary: Upadacitinib has displayed a rapid and favorable efficacy profile in RA but despite being a selective JAK1 inhibitor appears to have a similar safety profile to less-selective Jakinibs.',\n",
              "  0),\n",
              " (4107,\n",
              "  16,\n",
              "  'Which molecule is targeted by Upadacitinib?',\n",
              "  'BACKGROUND AND OBJECTIVES Upadacitinib is a janus kinase (JAK) 1 inhibitor being developed for the treatment of rheumatoid arthritis (RA) and other inflammatory diseases.',\n",
              "  0),\n",
              " (4107,\n",
              "  17,\n",
              "  'Which molecule is targeted by Upadacitinib?',\n",
              "  'Upadacitinib is a Janus kinase 1 inhibitor under development for the treatment of several inflammatory disorders including rheumatoid arthritis (RA).',\n",
              "  1),\n",
              " (4107,\n",
              "  18,\n",
              "  'Which molecule is targeted by Upadacitinib?',\n",
              "  'Areas covered: In this paper, we review a newly developed oral selective JAK inhibitor, upadacitinib for the treatment of RA.',\n",
              "  0),\n",
              " (4107,\n",
              "  19,\n",
              "  'Which molecule is targeted by Upadacitinib?',\n",
              "  'Upadacitinib selectively inhibits Janus Kinase 1 (JAK1) which could potentially reduce JAK2 and JAK3-related side effects.',\n",
              "  0),\n",
              " (4107,\n",
              "  20,\n",
              "  'Which molecule is targeted by Upadacitinib?',\n",
              "  '<b>AIMS</b>: Upadacitinib (ABT-494) is a selective Janus kinase 1 inhibitor being developed for treatment of auto-immune inflammatory disorders.',\n",
              "  0),\n",
              " (4107,\n",
              "  21,\n",
              "  'Which molecule is targeted by Upadacitinib?',\n",
              "  'Expert commentary: Upadacitinib has displayed a rapid and favorable efficacy profile in RA but despite being a selective JAK1 inhibitor appears to have a similar safety profile to less-selective Jakinibs.',\n",
              "  0),\n",
              " (4107,\n",
              "  22,\n",
              "  'Which molecule is targeted by Upadacitinib?',\n",
              "  'Assessment of effect of CYP3A inhibition, CYP induction, OATP1B inhibition, and high-fat meal on pharmacokinetics of the JAK1 inhibitor upadacitinib.',\n",
              "  0),\n",
              " (4107,\n",
              "  23,\n",
              "  'Which molecule is targeted by Upadacitinib?',\n",
              "  'Upadacitinib (ABT-494) is a selective Janus kinase 1 inhibitor being developed for treatment of auto-immune inflammatory disorders.',\n",
              "  0),\n",
              " (4107,\n",
              "  24,\n",
              "  'Which molecule is targeted by Upadacitinib?',\n",
              "  'Areas covered: In this paper, we review a newly developed oral selective JAK inhibitor, upadacitinib for the treatment of RA.',\n",
              "  0),\n",
              " (4107,\n",
              "  25,\n",
              "  'Which molecule is targeted by Upadacitinib?',\n",
              "  'OBJECTIVES: We assessed the relative efficacy and safety of once-daily administration of 15 and 30\\u202fmg upadacitinib (a\\xa0JAK1-selective inhibitor) in patients with active rheumatoid arthritis (RA)',\n",
              "  0),\n",
              " (4107,\n",
              "  26,\n",
              "  'Which molecule is targeted by Upadacitinib?',\n",
              "  ' In contrast, efficacy of filgotinib, a selective JAK1 inhibitor, in CD patients was demonstrated in the randomized, double-blinded, placebo-controlled phase II FITZROY study.',\n",
              "  0),\n",
              " (4107,\n",
              "  27,\n",
              "  'Which molecule is targeted by Upadacitinib?',\n",
              "  'Upadacitinib also showed promising results in a phase II trial in moderate to severe CD.',\n",
              "  0),\n",
              " (4107,\n",
              "  28,\n",
              "  'Which molecule is targeted by Upadacitinib?',\n",
              "  'Upadacitinib is a Janus kinase\\xa01 inhibitor developed for treatment of moderate to severe rheumatoid arthritis (RA) and was recently approved by the US Food and Drug Administration for this indication in adults who have had an inadequate response or intolerance to methotrexate.',\n",
              "  0),\n",
              " (4107,\n",
              "  29,\n",
              "  'Which molecule is targeted by Upadacitinib?',\n",
              "  'BJECTIVE: We sought to evaluate the safety and efficacy of multiple doses of the selective Janus kinase 1 inhibitor upadacitinib in patients with moderate to severe atopic dermatitis.',\n",
              "  0),\n",
              " (4107,\n",
              "  30,\n",
              "  'Which molecule is targeted by Upadacitinib?',\n",
              "  'The aim of this study was to characterize the effects of upadacitinib, a Janus kinase 1 inhibitor, on in vivo activity of different cytochrome P450 (CYP) enzymes using a cocktail approach.',\n",
              "  0),\n",
              " (4107,\n",
              "  31,\n",
              "  'Which molecule is targeted by Upadacitinib?',\n",
              "  'thods: Structure-based hypotheses were used to design the JAK1 selective inhibitor upadacitinib.',\n",
              "  0),\n",
              " (4107, 32, 'Which molecule is targeted by Upadacitinib?', 'J', 0),\n",
              " (4107,\n",
              "  33,\n",
              "  'Which molecule is targeted by Upadacitinib?',\n",
              "  'S: In randomised controlled trials primary efficacy endpoints were met for tofacitinib (JAK 1/3 inhibitor-phase III), upadacitinib (JAK 1 inhibitor-phase II) and AJM300 (α4-integrin antagonist-phase II) in ulcerative colitis.',\n",
              "  0),\n",
              " (4107, 34, 'Which molecule is targeted by Upadacitinib?', 'Ozani', 0),\n",
              " (4107,\n",
              "  35,\n",
              "  'Which molecule is targeted by Upadacitinib?',\n",
              "  'Both these new compounds are active on the majority of four JAK family members (JAK1, JAK2, JAK3, and TYK2), whereas the most recent emerging approach is directed toward the development of JAK1 selective inhibitors (upadacitinib and filgotinib) with the aim to improve the safety profile by minimizing the effects on JAK3 and, especially, JAK2.',\n",
              "  0),\n",
              " (4107,\n",
              "  36,\n",
              "  'Which molecule is targeted by Upadacitinib?',\n",
              "  'Upadacitinib (ABT-494) is a selective JAK1 inhibitor that was engineered to address the hypothesis that greater JAK1 selectivity over other JAK family members will translate into a more favorable benefit:risk profile.',\n",
              "  0),\n",
              " (4107,\n",
              "  37,\n",
              "  'Which molecule is targeted by Upadacitinib?',\n",
              "  'Upadacitinib is a JAK inhibitor engineered to be selective for JAK1, and has recently been approved for use in patients with moderate-to-severe RA.',\n",
              "  0),\n",
              " (4107,\n",
              "  38,\n",
              "  'Which molecule is targeted by Upadacitinib?',\n",
              "  'e. Upadacitinib selectively targets JAK1 dependent disease drivers such as IL-6 and IFNγ, while reducing effects on reticulocytes and natural killer (NK) cells, which potentially contributed to the tolerability issues of tofacitinib.Me',\n",
              "  0),\n",
              " (4107, 39, 'Which molecule is targeted by Upadacitinib?', 'mia.', 0),\n",
              " (4107,\n",
              "  40,\n",
              "  'Which molecule is targeted by Upadacitinib?',\n",
              "  'Upadacitinib (ABT-494) is a selective JAK1 inhibitor that was engineered to address the hypothesis that greater JAK1 selectivity over other JAK family members will translate into a more favorable benefit:risk profil',\n",
              "  0),\n",
              " (4107,\n",
              "  41,\n",
              "  'Which molecule is targeted by Upadacitinib?',\n",
              "  'tructure-based hypotheses were used to design the JAK1 selective inhibitor upadacitinib.',\n",
              "  0),\n",
              " (4107,\n",
              "  42,\n",
              "  'Which molecule is targeted by Upadacitinib?',\n",
              "  \"We evaluated the efficacy and safety of upadacitinib, an oral selective Janus kinase 1 inhibitor, in a randomized trial of patients with Crohn's disease (CD).M\",\n",
              "  0),\n",
              " (4107, 43, 'Which molecule is targeted by Upadacitinib?', 'sk profile.', 0),\n",
              " (4107,\n",
              "  44,\n",
              "  'Which molecule is targeted by Upadacitinib?',\n",
              "  'Upadacitinib selectively targets JAK1 dependent disease drivers such as IL-6 and IFNγ, while reducing effects on reticulocytes and natural killer (NK) cells, which potentially contributed to the tolerability issues of tofacitinib.Methods: Structure-based hypotheses were used to design the JAK1 selective in',\n",
              "  0),\n",
              " (4107,\n",
              "  45,\n",
              "  'Which molecule is targeted by Upadacitinib?',\n",
              "  'AIMS: Upadacitinib (ABT-494) is a selective Janus kinase 1 inhibitor being developed for treatment of auto-immune inflammatory diso',\n",
              "  0),\n",
              " (4107,\n",
              "  46,\n",
              "  'Which molecule is targeted by Upadacitinib?',\n",
              "  'Upadacitinib is a selective Janus kinase (JAK) 1 inhibitor being developed for treatment of rheumatoid arthritis.',\n",
              "  1),\n",
              " (4107,\n",
              "  47,\n",
              "  'Which molecule is targeted by Upadacitinib?',\n",
              "  'o physiological consequences.',\n",
              "  0),\n",
              " (4107,\n",
              "  48,\n",
              "  'Which molecule is targeted by Upadacitinib?',\n",
              "  'Upadacitinib is ~\\u200960 fold selective for JAK1 over JAK2, and\\u2009>\\u2009100 fold selective',\n",
              "  0),\n",
              " (4107,\n",
              "  49,\n",
              "  'Which molecule is targeted by Upadacitinib?',\n",
              "  'Preferential Inhibition of JAK1 Relative to JAK3 by Upadacitinib: Exposure-Response Analyses of Ex Vivo Data From 2 Phase 1 Clinical Trials and Comparison to Tofacitinib.',\n",
              "  0),\n",
              " (4107,\n",
              "  50,\n",
              "  'Which molecule is targeted by Upadacitinib?',\n",
              "  'Here, we demonstrated that Upadacitinib, a JAK-1 inhibitor, inhibited the proliferation of cytokine-dependent ATL cell lines and the expression of p-STAT5.',\n",
              "  0),\n",
              " (4107,\n",
              "  51,\n",
              "  'Which molecule is targeted by Upadacitinib?',\n",
              "  'Selective JAK1 Inhibitors for the Treatment of Atopic Dermatitis: Focus on Upadacitinib and Abrocitinib.',\n",
              "  0),\n",
              " (4107,\n",
              "  52,\n",
              "  'Which molecule is targeted by Upadacitinib?',\n",
              "  'Upadacitinib and abrocitinib are oral small molecules that inhibit the JAK/STAT pathway by selectively blocking JAK1.',\n",
              "  0),\n",
              " (4107,\n",
              "  53,\n",
              "  'Which molecule is targeted by Upadacitinib?',\n",
              "  'Several more selective JAK-1 inhibitors, including filgotinib and upadacitinib, have also shown positive results in phase II studies and are currently enrolling in phase III development programs.',\n",
              "  0),\n",
              " (4107,\n",
              "  54,\n",
              "  'Which molecule is targeted by Upadacitinib?',\n",
              "  'Emerging data with selective JAK1 inhibitors upadacitinib and filgotinib looks very promising',\n",
              "  0),\n",
              " (4107,\n",
              "  55,\n",
              "  'Which molecule is targeted by Upadacitinib?',\n",
              "  'OBJECTIVE: To evaluate the efficacy, including capacity for inhibition of radiographic progression, and safety of upadacitinib, a JAK1-selective inhibitor, as compared to placebo or adalimumab in patients with rheumatoid arthritis (RA) who have experienced an inadequate response to methotrexate (MTX).',\n",
              "  0),\n",
              " (4107,\n",
              "  56,\n",
              "  'Which molecule is targeted by Upadacitinib?',\n",
              "  'BACKGROUND: Phase 2 studies with upadacitinib, a selective Janus kinase 1 (JAK1) inhibitor, have shown safety and efficacy in the treatment of patients with active rheumatoid arthritis.',\n",
              "  0),\n",
              " (4107,\n",
              "  57,\n",
              "  'Which molecule is targeted by Upadacitinib?',\n",
              "  'BACKGROUND: Upadacitinib is a selective inhibitor of Janus kinase 1 and was efficacious in phase 2 studies in patients with moderate-to-severe rheumatoid arthritis.',\n",
              "  0),\n",
              " (4107,\n",
              "  58,\n",
              "  'Which molecule is targeted by Upadacitinib?',\n",
              "  'Upadacitinib selectively inhibits Janus Kinase 1 (JAK1) which could potentially reduce JAK2 and JAK3-related side effects.',\n",
              "  0),\n",
              " (4107,\n",
              "  59,\n",
              "  'Which molecule is targeted by Upadacitinib?',\n",
              "  'Upadacitinib is a selective Janus Kinase 1 inhibitor which is being developed for the treatment of several inflammatory diseases including rheumatoid arthritis.',\n",
              "  1),\n",
              " (4107,\n",
              "  60,\n",
              "  'Which molecule is targeted by Upadacitinib?',\n",
              "  'Upadacitinib is a novel selective Janus kinase 1 inhibitor developed for treatment of rheumatoid arthritis and other autoimmune diseases.',\n",
              "  0),\n",
              " (4107,\n",
              "  61,\n",
              "  'Which molecule is targeted by Upadacitinib?',\n",
              "  'Upadacitinib is a selective Janus kinase 1 inhibitor being developed for the treatment of several inflammatory autoimmune diseases, including rheumatoid arthritis.',\n",
              "  1),\n",
              " (4107,\n",
              "  62,\n",
              "  'Which molecule is targeted by Upadacitinib?',\n",
              "  'BACKGROUND: Upadacitinib, an oral Janus kinase (JAK)1-selective inhibitor, showed efficacy in combination with stable background conventional synthetic disease-modifying antirheumatic drugs (csDMARDs) in patients with rheumatoid arthritis who had an inadequate response to DMARDs.',\n",
              "  0),\n",
              " (4107,\n",
              "  63,\n",
              "  'Which molecule is targeted by Upadacitinib?',\n",
              "  'This phase 1 study characterized the effect of multiple doses of upadacitinib, an oral Janus kinase 1 selective inhibitor, on the pharmacokinetics of the cytochrome P450 (CYP) 2B6 substrate bupropion.',\n",
              "  0),\n",
              " (4107,\n",
              "  64,\n",
              "  'Which molecule is targeted by Upadacitinib?',\n",
              "  'To date, three JAK inhibitors have been tested in patients with moderate-to-severe CD: tofacitinib (pan-JAK inhibitor), filgotinib (JAK1 inhibitor) and upadacitinib (JAK1 inhibitor).',\n",
              "  0),\n",
              " (2671,\n",
              "  0,\n",
              "  \"Is a mutation of the  ZIKV's membrane protein prM responsible for the microcephaly in new-born infants?\",\n",
              "  'Here we show that a single serine-to-asparagine substitution [Ser139→Asn139(S139N)] in the viral polyprotein substantially increased ZIKV infectivity in both human and mouse neural progenitor cells (NPCs) and led to more severe microcephaly in the mouse fetus, as well as higher mortality rates in neonatal mice.',\n",
              "  1),\n",
              " (2671,\n",
              "  1,\n",
              "  \"Is a mutation of the  ZIKV's membrane protein prM responsible for the microcephaly in new-born infants?\",\n",
              "  'A single mutation in the prM protein of Zika virus contributes to fetal microcephaly.',\n",
              "  1),\n",
              " (204,\n",
              "  0,\n",
              "  'What is the molecular function of the Chd1 protein?',\n",
              "  'The DNA-binding domain of the Chd1 chromatin-remodelling enzyme contains SANT and SLIDE domains',\n",
              "  1),\n",
              " (204,\n",
              "  1,\n",
              "  'What is the molecular function of the Chd1 protein?',\n",
              "  'The ATP-dependent chromatin-remodelling enzyme Chd1 is a 168-kDa protein consisting of a double chromodomain, Snf2-related ATPase domain, and a C-terminal DNA-binding domain.',\n",
              "  1),\n",
              " (204,\n",
              "  2,\n",
              "  'What is the molecular function of the Chd1 protein?',\n",
              "  'The presence of these domains in ISWI and Chd1 chromatin-remodelling enzymes may provide a means of efficiently harnessing the action of the Snf2-related ATPase domain for the purpose of nucleosome spacing and provide an explanation for partial redundancy between these proteins.',\n",
              "  0),\n",
              " (204,\n",
              "  3,\n",
              "  'What is the molecular function of the Chd1 protein?',\n",
              "  'Here we identify the chromatin remodelling protein Chd1 (chromo-ATPase/helicase-DNA binding domain 1) as a component of SAGA and SLIK',\n",
              "  0),\n",
              " (204,\n",
              "  4,\n",
              "  'What is the molecular function of the Chd1 protein?',\n",
              "  'one of the two chromodomains of Chd1 specifically interacts with the methylated lysine 4 mark on histone H3 that is associated with transcriptional activity',\n",
              "  1),\n",
              " (204,\n",
              "  5,\n",
              "  'What is the molecular function of the Chd1 protein?',\n",
              "  'Human but not yeast CHD1 binds directly and selectively to histone H3 methylated at lysine 4 via its tandem chromodomains',\n",
              "  0),\n",
              " (204,\n",
              "  6,\n",
              "  'What is the molecular function of the Chd1 protein?',\n",
              "  'In the current study, we identify human CHD1, an ATP-dependent chromatin remodeling protein, as a factor that directly and selectively recognizes histone H3 methylated on lysine 4',\n",
              "  1),\n",
              " (204,\n",
              "  7,\n",
              "  'What is the molecular function of the Chd1 protein?',\n",
              "  'human CHD1 binds to methylated H3K4 in a manner that requires both of its tandem chromodomains.',\n",
              "  0),\n",
              " (204,\n",
              "  8,\n",
              "  'What is the molecular function of the Chd1 protein?',\n",
              "  'Our findings indicate that yeast and human CHD1 have diverged in their ability to discriminate covalently modified histones and link histone modification-recognition and non-covalent chromatin remodeling activities within a single human protein.',\n",
              "  0),\n",
              " (204,\n",
              "  9,\n",
              "  'What is the molecular function of the Chd1 protein?',\n",
              "  'We find that the yeast Isw1a, Isw2, and Chd1 enzymes preferentially move nucleosomes toward more central locations on short DNA fragments whereas Isw1b does not.',\n",
              "  0),\n",
              " (204,\n",
              "  10,\n",
              "  'What is the molecular function of the Chd1 protein?',\n",
              "  'Analysis of nucleosome repositioning by yeast ISWI and Chd1 chromatin remodeling complexes.',\n",
              "  0),\n",
              " (204,\n",
              "  11,\n",
              "  'What is the molecular function of the Chd1 protein?',\n",
              "  'However, a key difference is that the Isw1a, Isw2, and Chd1 enzymes are unable to move nucleosomes to positions closer than 15 bp from a DNA end, whereas Isw1b can.',\n",
              "  0),\n",
              " (204,\n",
              "  12,\n",
              "  'What is the molecular function of the Chd1 protein?',\n",
              "  'The ISWI and CHD1 chromatin remodelling activities influence ADH2 expression and chromatin organization',\n",
              "  0),\n",
              " (204,\n",
              "  13,\n",
              "  'What is the molecular function of the Chd1 protein?',\n",
              "  'Here we show that the absence of the Isw1 and Chd1 ATP-dependent chromatin remodelling activities delays the maximal expression of ADH2 without impairing the chromatin remodelling that occurs upon activation.',\n",
              "  0),\n",
              " (204,\n",
              "  14,\n",
              "  'What is the molecular function of the Chd1 protein?',\n",
              "  'Thus, the ISWI and Chd1 remodelling factors are not only involved in transcription-related chromatin remodelling, but also are required to maintain a specific chromatin configuration across the yeast genome.',\n",
              "  0),\n",
              " (204,\n",
              "  15,\n",
              "  'What is the molecular function of the Chd1 protein?',\n",
              "  'the lack of Chd1 combined with the absence of Isw1 and Isw2 impairs nucleosome spacing along the ADH2 gene, and genome-wide in S. cerevisiae',\n",
              "  0),\n",
              " (204,\n",
              "  16,\n",
              "  'What is the molecular function of the Chd1 protein?',\n",
              "  'Through inclusion of Chd1 sequences in homology searches SLIDE domains were identified in CHD6-9 proteins.',\n",
              "  0),\n",
              " (204,\n",
              "  17,\n",
              "  'What is the molecular function of the Chd1 protein?',\n",
              "  'The molecular motor protein CHD1 has been implicated in the regulation of transcription and in the transcription-independent genome-wide incorporation of H3.3 into paternal chromatin in Drosophila melanogaster.',\n",
              "  0),\n",
              " (204,\n",
              "  18,\n",
              "  'What is the molecular function of the Chd1 protein?',\n",
              "  'Interestingly, changes in CHD1 protein levels did not correlate with changes in the levels of the euchromatin mark H3K4me3 or elongating RNA Polymerase II.',\n",
              "  0),\n",
              " (204,\n",
              "  19,\n",
              "  'What is the molecular function of the Chd1 protein?',\n",
              "  'CHD1 is known to bind H3K4me3 in mammalian cells, and Neurospora CHD1 is required for proper regulation of the frequency (frq) gene.',\n",
              "  0),\n",
              " (204,\n",
              "  20,\n",
              "  'What is the molecular function of the Chd1 protein?',\n",
              "  'The molecular motor protein CHD1 has been implicated in the regulation of transcription and in the transcription-independent genome-wide incorporation of H3.3 into paternal chromatin in Drosophila melanogaster.',\n",
              "  0),\n",
              " (204,\n",
              "  21,\n",
              "  'What is the molecular function of the Chd1 protein?',\n",
              "  'Thus, while CHD1 is localized to transcriptionally active regions of the genome, it can function to alter the levels of HP1a, perhaps through changes in methylation of H3K9.',\n",
              "  0),\n",
              " (204,\n",
              "  22,\n",
              "  'What is the molecular function of the Chd1 protein?',\n",
              "  'Histone H3K4 and K36 methylation, Chd1 and Rpd3S oppose the functions of Saccharomyces cerevisiae Spt4-Spt5 in transcription.',\n",
              "  0),\n",
              " (204,\n",
              "  23,\n",
              "  'What is the molecular function of the Chd1 protein?',\n",
              "  'Interestingly, changes in CHD1 protein levels did not correlate with changes in the levels of the euchromatin mark H3K4me3 or elongating RNA Polymerase II.',\n",
              "  0),\n",
              " (204,\n",
              "  24,\n",
              "  'What is the molecular function of the Chd1 protein?',\n",
              "  'The molecular motor protein CHD1 has been implicated in the regulation of transcription and in the transcription-independent genome-wide incorporation of H3.3 into paternal chromatin in Drosophila melanogaster.',\n",
              "  0),\n",
              " (204,\n",
              "  25,\n",
              "  'What is the molecular function of the Chd1 protein?',\n",
              "  'Interestingly, changes in CHD1 protein levels did not correlate with changes in the levels of the euchromatin mark H3K4me3 or elongating RNA Polymerase II.',\n",
              "  0),\n",
              " (204,\n",
              "  26,\n",
              "  'What is the molecular function of the Chd1 protein?',\n",
              "  'Thus, while CHD1 is localized to transcriptionally active regions of the genome, it can function to alter the levels of HP1a, perhaps through changes in methylation of H3K9.',\n",
              "  0),\n",
              " (204,\n",
              "  27,\n",
              "  'What is the molecular function of the Chd1 protein?',\n",
              "  ' Chromodomain from heterochromatin protein 1 and polycomb protein is known to be a lysine-methylated histone H3 tail-binding module.',\n",
              "  0),\n",
              " (204,\n",
              "  28,\n",
              "  'What is the molecular function of the Chd1 protein?',\n",
              "  'Chromo-helicase/ATPase DNA-binding protein 1 (CHD1) is an ATP-dependent chromatin remodeling factor, containing two tandem chromodomains.',\n",
              "  1),\n",
              " (204,\n",
              "  29,\n",
              "  'What is the molecular function of the Chd1 protein?',\n",
              "  'Over-expression of an ATPase inactive form of CHD1 did not result in severe chromosomal defects, suggesting that the ATPase activity is required for this in vivo phenotype.',\n",
              "  0),\n",
              " (204,\n",
              "  30,\n",
              "  'What is the molecular function of the Chd1 protein?',\n",
              "  'Interestingly, changes in CHD1 protein levels did not correlate with changes in the levels of the euchromatin mark H3K4me3 or elongating RNA Polymerase II.',\n",
              "  0),\n",
              " (204,\n",
              "  31,\n",
              "  'What is the molecular function of the Chd1 protein?',\n",
              "  'Interestingly, changes in CHD1 protein levels did not correlate with changes in the levels of the euchromatin mark H3K4me3 or elongating RNA Polymerase II.',\n",
              "  0),\n",
              " (731,\n",
              "  0,\n",
              "  'What kind of bonds are connecting keratin molecules?',\n",
              "  ' Feathers and hair consist of cornified epidermal keratinocytes in which proteins are crosslinked via disulfide bonds between cysteine residues of structural proteins to establish mechanical resilience.',\n",
              "  1),\n",
              " (731,\n",
              "  1,\n",
              "  'What kind of bonds are connecting keratin molecules?',\n",
              "  ' The method involves the generation of thiols by controlled reduction of cystine disulfide bonds in keratin',\n",
              "  1),\n",
              " (731,\n",
              "  2,\n",
              "  'What kind of bonds are connecting keratin molecules?',\n",
              "  'ossible chemical bonds formed between alpha-keratins and KAbetaPs may derive from electrostatic interactions in addition to cross-linking through disulphide bonds.',\n",
              "  0),\n",
              " (731,\n",
              "  3,\n",
              "  'What kind of bonds are connecting keratin molecules?',\n",
              "  'The method involves the generation of thiols by controlled reduction of cystine disulfide bonds in the keratin',\n",
              "  1),\n",
              " (731,\n",
              "  4,\n",
              "  'What kind of bonds are connecting keratin molecules?',\n",
              "  'e solubility test in the dithiothreitol/urea extraction buffer, the amino acid composition analysis and studies on keratin secondary structures suggest that the improved stability in water of thermally treated mats can be ascribed to the formation of amide bonds between acid and basic groups of some amino acid side chains.',\n",
              "  0),\n",
              " (731,\n",
              "  5,\n",
              "  'What kind of bonds are connecting keratin molecules?',\n",
              "  's the energy involved in the formation of disulfide bonds is much greater than that of hydrogen bonds or van der Waals interactions the structural transition is likely to be dominated by the requirement that the bonded cysteine residues occur at closely equivalent axial positions.',\n",
              "  1),\n",
              " (731,\n",
              "  6,\n",
              "  'What kind of bonds are connecting keratin molecules?',\n",
              "  'The interface of the K5-K14 coiled-coil heterodimer has asymmetric salt bridges, hydrogen bonds and hydrophobic contacts, and its surface exhibits a notable charge polarization.',\n",
              "  0),\n",
              " (731,\n",
              "  7,\n",
              "  'What kind of bonds are connecting keratin molecules?',\n",
              "  \"A trans-dimer homotypic disulfide bond involving Cys367 in K14's stutter region occurs in the crystal and in skin keratinocytes, where it is concentrated in a keratin filament cage enveloping the nucleus.\",\n",
              "  0),\n",
              " (731,\n",
              "  8,\n",
              "  'What kind of bonds are connecting keratin molecules?',\n",
              "  ' the alpha helices which are stabilized by hydrogen bonds and the alpha-helical coiled coils which are stabilized by hydrophobic interactions, is more sensitive to radiation than the supramolecular architecture of the keratin filament and the filament packing within the keratin associated proteins matrix, which is stabilized by disulphide bonds.',\n",
              "  0),\n",
              " (731,\n",
              "  9,\n",
              "  'What kind of bonds are connecting keratin molecules?',\n",
              "  ' A good solvent attacks the disulfide bonds between cystine molecules and hydrates the hair shaft.',\n",
              "  1),\n",
              " (473,\n",
              "  0,\n",
              "  'What is Prudent Diet?',\n",
              "  'Long-term diet was assessed by using FFQs every 4 y since 1986.',\n",
              "  0),\n",
              " (473,\n",
              "  1,\n",
              "  'What is Prudent Diet?',\n",
              "  'Prudent (high in vegetables) and Western (high in meats) patterns were identified by using a principal component analysis.',\n",
              "  1),\n",
              " (473,\n",
              "  2,\n",
              "  'What is Prudent Diet?',\n",
              "  'The Prudent dietary pattern was characterised by high intakes of vegetables, fruits, whole grain products and low intakes of refined grain products and the Western dietary pattern, by high intakes of refined grain products, desserts, sweets and processed meats.',\n",
              "  1),\n",
              " (473,\n",
              "  3,\n",
              "  'What is Prudent Diet?',\n",
              "  'aerobic exercise combined with diet recommendations (saturated/trans fat intake less than 10% of total calories and cholesterol less than 300 mg/day and/or fiber intake ≥ 25 g/day in women and ≥ 35 grams per day in men',\n",
              "  1),\n",
              " (473,\n",
              "  4,\n",
              "  'What is Prudent Diet?',\n",
              "  ' a prudent pattern (high in fish, peas, honey, nuts, juice, dry fruits, vegetable oil, liver and organic meat, and coconuts and low in hydrogenated fat and non-leafy vegetables)',\n",
              "  1),\n",
              " (473,\n",
              "  5,\n",
              "  'What is Prudent Diet?',\n",
              "  ' The prudent pattern was characterized by higher intakes of fruits, vegetables, legumes, fish, poultry, and whole grains, while the Western pattern was characterized by higher intakes of red and processed meats, sweets and desserts, french fries, and refined grains.',\n",
              "  1),\n",
              " (3364,\n",
              "  0,\n",
              "  'Is PRDM9 essential for meiosis?',\n",
              "  'Our findings do not identify the nature of the underlying DNA sequences, but argue against the proposed role of Prdm9 as an essential transcription factor in mouse meiosis',\n",
              "  1),\n",
              " (3364,\n",
              "  1,\n",
              "  'Is PRDM9 essential for meiosis?',\n",
              "  'PRDM9 gene polymorphism may not be associated with defective spermatogenesis in the Chinese Han population',\n",
              "  0),\n",
              " (3364,\n",
              "  2,\n",
              "  'Is PRDM9 essential for meiosis?',\n",
              "  'PRDM9 is essential for the progression through early meiotic prophase, including double strand break repair, homologous chromosome pairing, and sex body formation during spermatogenesis.',\n",
              "  1),\n",
              " (3364,\n",
              "  3,\n",
              "  'Is PRDM9 essential for meiosis?',\n",
              "  'PRDM9 (PR domain-containing protein 9) is a meiosis-specific protein that trimethylates H3K4 and controls the activation of recombination hot spots.',\n",
              "  1),\n",
              " (3364,\n",
              "  4,\n",
              "  'Is PRDM9 essential for meiosis?',\n",
              "  'It is an essential enzyme in the progression of early meiotic prophase.',\n",
              "  0),\n",
              " (3364,\n",
              "  5,\n",
              "  'Is PRDM9 essential for meiosis?',\n",
              "  'In many eukaryotes, sites of meiotic recombination, also called hotspots, are regions of accessible chromatin, but in many vertebrates, their location follows a distinct pattern and is specified by PR domain-containing protein 9 (PRDM9).',\n",
              "  0),\n",
              " (3364,\n",
              "  6,\n",
              "  'Is PRDM9 essential for meiosis?',\n",
              "  ' We found that although the post-SET zinc finger and the KRAB domains are not essential for the methyltransferase activity of PRDM9 in cell culture, the KRAB domain mutant mice show only residual PRDM9 methyltransferase activity and undergo meiotic arrest.',\n",
              "  0),\n",
              " (3364,\n",
              "  7,\n",
              "  'Is PRDM9 essential for meiosis?',\n",
              "  'In aggregate, our data indicate that domains typically involved in regulation of gene expression do not serve that role in PRDM9, but are likely involved in setting the proper chromatin environment for initiation and completion of homologous recombination.',\n",
              "  1),\n",
              " (3364,\n",
              "  8,\n",
              "  'Is PRDM9 essential for meiosis?',\n",
              "  'PRDM9 Methyltransferase Activity Is Essential for Meiotic DNA Double-Strand Break Formation at Its Binding Sites.',\n",
              "  1),\n",
              " (3195,\n",
              "  0,\n",
              "  'What are the 3 types of immunoglobulin heavy chain containing antibodies found in human breast milk?',\n",
              "  'This is a descriptive cross-sectional study of a convenience sample of 67 donated colostrum samples at different days after delivery, both raw and pasteurized.',\n",
              "  1),\n",
              " (3195,\n",
              "  1,\n",
              "  'What are the 3 types of immunoglobulin heavy chain containing antibodies found in human breast milk?',\n",
              "  'Antibody profiles were analyzed at different times during breastfeeding, and total and specific antibodies (IgM, IgA, and IgG subclasses)',\n",
              "  1),\n",
              " (3397,\n",
              "  0,\n",
              "  'What is the basis of the methidiumpropyl-EDTA sequencing (MPE-seq) method?',\n",
              "  'methidiumpropyl-EDTA sequencing (MPE-seq), a method for the genome-wide characterization of chromatin that involves the digestion of nuclei withMPE-Fe(II) followed by massively parallel sequencing.',\n",
              "  1),\n",
              " (3397,\n",
              "  1,\n",
              "  'What is the basis of the methidiumpropyl-EDTA sequencing (MPE-seq) method?',\n",
              "  'MPE-seq provides a unique and straightforward means for the genome-wide analysis of chromatin structure with minimal DNA sequence bias.',\n",
              "  1),\n",
              " (3534,\n",
              "  0,\n",
              "  \"Is Huntington's disease caused by a dominate or recessive gene?\",\n",
              "  \"Huntington's Disease (HD) is an autosomal dominant neurodegenerative disease characterized by gradual deterioration of motor and cognitive functions and development of psychiatric deficits.\",\n",
              "  1),\n",
              " (3534,\n",
              "  1,\n",
              "  \"Is Huntington's disease caused by a dominate or recessive gene?\",\n",
              "  'Animal models provide powerful means to study the pathological',\n",
              "  0),\n",
              " (3534,\n",
              "  2,\n",
              "  \"Is Huntington's disease caused by a dominate or recessive gene?\",\n",
              "  \"Huntington's disease (HD) is an inherited NDD caused by autosomal-dominant expanded CAG trinucleotide repeat mutation in the gene coding for Huntingtin (Htt).\",\n",
              "  1),\n",
              " (3534,\n",
              "  3,\n",
              "  \"Is Huntington's disease caused by a dominate or recessive gene?\",\n",
              "  \" Huntington's disease (HD) is an autosomal-dominant, progressive neurodegenerative disease characterized by movement disorder, psychiatric symptoms and cognitive decline.\",\n",
              "  1),\n",
              " (3534,\n",
              "  4,\n",
              "  \"Is Huntington's disease caused by a dominate or recessive gene?\",\n",
              "  \"The D4S10 locus, defined by the probe G8 and linked to the gene for Huntington's disease (HD), has permitted us to identify individuals with a high probability of being homozygous for this autosomal dominant neurodegenerative disorder.\",\n",
              "  0),\n",
              " (3534,\n",
              "  5,\n",
              "  \"Is Huntington's disease caused by a dominate or recessive gene?\",\n",
              "  \"Huntington's disease is a fatal autosomal dominant neurodegenerative disease caused by an abnormal CAG expansion in huntingtin's gene.\",\n",
              "  1),\n",
              " (3534,\n",
              "  6,\n",
              "  \"Is Huntington's disease caused by a dominate or recessive gene?\",\n",
              "  \"Huntington's disease is a fatal autosomal dominant neurodegenerative disease caused by an abnormal CAG expansion in huntingtin's gene\",\n",
              "  0),\n",
              " (3534,\n",
              "  7,\n",
              "  \"Is Huntington's disease caused by a dominate or recessive gene?\",\n",
              "  \"Huntington's disease is an autosomal dominant neurodegenerative disease caused by a toxic gain of function mutation in the huntingtin gene (Htt)\",\n",
              "  1),\n",
              " (3534,\n",
              "  8,\n",
              "  \"Is Huntington's disease caused by a dominate or recessive gene?\",\n",
              "  \"Huntington's disease (HD) is a late onset, incurable, autosomal dominantly-inherited, progressive neuropsychiatric disease, characterised by chorea, changes in personality, mood and behaviour, and dementia.\",\n",
              "  0),\n",
              " (3534,\n",
              "  9,\n",
              "  \"Is Huntington's disease caused by a dominate or recessive gene?\",\n",
              "  \"Huntington's disease (HD) is a genetically dominant neurodegenerative condition caused by an unique mutation in the disease gene huntingtin.\",\n",
              "  0),\n",
              " (3534,\n",
              "  10,\n",
              "  \"Is Huntington's disease caused by a dominate or recessive gene?\",\n",
              "  'Huntington disease is an autosomal-dominant neurodegenerative disease of mid-life onset caused by expansion of a polymorphic trinucleotide (CAG) repeat.',\n",
              "  0),\n",
              " (3534,\n",
              "  11,\n",
              "  \"Is Huntington's disease caused by a dominate or recessive gene?\",\n",
              "  'Huntington disease (HD) is a well-defined autosomal dominant neurodegenerative disease caused by CAG repeat expansions in HD gene.',\n",
              "  0),\n",
              " (2007,\n",
              "  0,\n",
              "  'Is the mouse Sry gene locus free of repetitive sequences?',\n",
              "  'We demonstrate that the presence of long inverted repeats (IR) flanking the mouse Sry gene leads to the formation of the Sry circular transcript in cultured cells',\n",
              "  1),\n",
              " (2007,\n",
              "  1,\n",
              "  'Is the mouse Sry gene locus free of repetitive sequences?',\n",
              "  'Circularization requires the presence of both IR.',\n",
              "  0),\n",
              " (2007,\n",
              "  2,\n",
              "  'Is the mouse Sry gene locus free of repetitive sequences?',\n",
              "  'As few as 400 complementary nt are necessary for this process',\n",
              "  0),\n",
              " (2007,\n",
              "  3,\n",
              "  'Is the mouse Sry gene locus free of repetitive sequences?',\n",
              "  'The presence of the IR does not significantly stimulate intermolecular annealing and trans-splicing in vivo',\n",
              "  0),\n",
              " (2007,\n",
              "  4,\n",
              "  'Is the mouse Sry gene locus free of repetitive sequences?',\n",
              "  'We have found that in an in vitro assay, the SRY protein binds to several sites of the Sry gene and especially to a (CA)25 sequence and to a (CAG)30 repeat',\n",
              "  1),\n",
              " (2007,\n",
              "  5,\n",
              "  'Is the mouse Sry gene locus free of repetitive sequences?',\n",
              "  'The Q-rich domain of the mouse sex determining gene, Sry, is encoded by an in-frame insertion of a repetitive sequence composed of mostly CAG repeats.',\n",
              "  1),\n",
              " (2007,\n",
              "  6,\n",
              "  'Is the mouse Sry gene locus free of repetitive sequences?',\n",
              "  'Inverted repeat structure of the Sry locus in mice.',\n",
              "  0),\n",
              " (2007,\n",
              "  7,\n",
              "  'Is the mouse Sry gene locus free of repetitive sequences?',\n",
              "  'We performed separate amplifications of DXZ4 repetitive satellite sequences on the X chromosome, and SRY gene - testis determined factor on the Y chromosome, using nested PCR',\n",
              "  0),\n",
              " (2007,\n",
              "  8,\n",
              "  'Is the mouse Sry gene locus free of repetitive sequences?',\n",
              "  'Detailed analysis of the Sry genomic locus reveals a further difference in that the mouse Sry open reading frame lies within 2.8 kilobases of unique sequence at the center of a large inverted repeat.',\n",
              "  1),\n",
              " (2007,\n",
              "  9,\n",
              "  'Is the mouse Sry gene locus free of repetitive sequences?',\n",
              "  'Detailed analysis of the Sry genomic locus reveals a further difference in that the mouse Sry open reading frame lies within 2.8 kilobases of unique sequence at the center of a large inverted repeat.',\n",
              "  1),\n",
              " (2007,\n",
              "  10,\n",
              "  'Is the mouse Sry gene locus free of repetitive sequences?',\n",
              "  'The mouse genomic Sry locus is characterized by two arms of a large inverted repeat, flanking a unique region that, between an acceptor and a donor splice site, contains a single exon encoding the Sry protein.',\n",
              "  0),\n",
              " (2007,\n",
              "  11,\n",
              "  'Is the mouse Sry gene locus free of repetitive sequences?',\n",
              "  'Recombination involving the repeat region may have led to an 11-kilobase deletion, precisely excising Sry in a line of XY female mice.',\n",
              "  0),\n",
              " (2007,\n",
              "  12,\n",
              "  'Is the mouse Sry gene locus free of repetitive sequences?',\n",
              "  'Repetitive element analysis revealed numerous LINE-L1 elements at regions where conservation is lost among the Sry copies.',\n",
              "  0),\n",
              " (2007,\n",
              "  13,\n",
              "  'Is the mouse Sry gene locus free of repetitive sequences?',\n",
              "  'Inverted repeat structure of the Sry locus in mice.',\n",
              "  0),\n",
              " (891,\n",
              "  0,\n",
              "  'Is NOD1 activated in inflammation?',\n",
              "  'Nod1 and Nod2 control bacterial infections and inflammation',\n",
              "  0),\n",
              " (891,\n",
              "  1,\n",
              "  'Is NOD1 activated in inflammation?',\n",
              "  'The Nod proteins Nod1 and Nod2 are two NLR family members that trigger immune defense in response to bacterial peptidoglycan',\n",
              "  0),\n",
              " (891,\n",
              "  2,\n",
              "  'Is NOD1 activated in inflammation?',\n",
              "  'Nod proteins fight off bacterial infections by stimulating proinflammatory signaling and cytokine networks and by inducing antimicrobial effectors, such as nitric oxide and antimicrobial peptides.',\n",
              "  1),\n",
              " (891,\n",
              "  3,\n",
              "  'Is NOD1 activated in inflammation?',\n",
              "  'Nod1 is also critically implicated in shaping adaptive immune responses towards bacterial-derived constituents.',\n",
              "  0),\n",
              " (891,\n",
              "  4,\n",
              "  'Is NOD1 activated in inflammation?',\n",
              "  'Together, Nod1 and Nod2 represent central players in the control of immune responses to bacterial infections and inflammation.',\n",
              "  1),\n",
              " (891,\n",
              "  5,\n",
              "  'Is NOD1 activated in inflammation?',\n",
              "  'The innate immune receptor Nod1 protects the intestine from inflammation-induced tumorigenesis.',\n",
              "  0),\n",
              " (891,\n",
              "  6,\n",
              "  'Is NOD1 activated in inflammation?',\n",
              "  'we show that Nod1 deficiency results in the increased development of both colitis-associated and Apc tumor suppressor-related colon tumors.',\n",
              "  0),\n",
              " (891,\n",
              "  7,\n",
              "  'Is NOD1 activated in inflammation?',\n",
              "  'In the absence of Nod1 signaling, there is a greater disruption of the intestinal epithelial cell barrier due to chemically induced injury as manifested by increased surface epithelial apoptosis early on during chemically induced colitis and increased intestinal permeability.',\n",
              "  0),\n",
              " (891,\n",
              "  8,\n",
              "  'Is NOD1 activated in inflammation?',\n",
              "  'The increased intestinal permeability is associated with enhanced inflammatory cytokine production and epithelial cell proliferation in Nod1-deficient mice as compared with wild-type mice.',\n",
              "  1),\n",
              " (891,\n",
              "  9,\n",
              "  'Is NOD1 activated in inflammation?',\n",
              "  'Depletion of the gut microbiota suppressed tumor development in Nod1-deficient mice, thus highlighting a link between the commensal bacteria within the intestine and the host innate immune Nod1 signaling pathway in the regulation inflammation-mediated colon cancer development.',\n",
              "  0),\n",
              " (891,\n",
              "  10,\n",
              "  'Is NOD1 activated in inflammation?',\n",
              "  ' NOD1 protein is expressed in the eye and promotes ocular inflammation in a dose- and time-dependent fashion.',\n",
              "  0),\n",
              " (891,\n",
              "  11,\n",
              "  'Is NOD1 activated in inflammation?',\n",
              "  'NOD1 expression in the eye and functional contribution to IL-1beta-dependent ocular inflammation in mice',\n",
              "  0),\n",
              " (891,\n",
              "  12,\n",
              "  'Is NOD1 activated in inflammation?',\n",
              "  \"Polymorphisms in NOD1 are associated with autoinflammatory diseases characterized by uveitis such as Crohn's disease and sarcoidosis\",\n",
              "  0),\n",
              " (891,\n",
              "  13,\n",
              "  'Is NOD1 activated in inflammation?',\n",
              "  'NOD1 is homologous to NOD2, which is responsible for an autosomal dominant form of uveitis.',\n",
              "  0),\n",
              " (891,\n",
              "  14,\n",
              "  'Is NOD1 activated in inflammation?',\n",
              "  'Nonetheless, the role of NOD1 in intraocular inflammation has not been explored.',\n",
              "  0),\n",
              " (891,\n",
              "  15,\n",
              "  'Is NOD1 activated in inflammation?',\n",
              "  'Nod1 and Nod2 regulation of inflammation in the Salmonella colitis model',\n",
              "  0),\n",
              " (891,\n",
              "  16,\n",
              "  'Is NOD1 activated in inflammation?',\n",
              "  'mice deficient for both Nod1 and Nod2 had attenuated inflammatory pathology, reduced levels of inflammatory cytokines, and increased colonization of the mucosal tissue',\n",
              "  1),\n",
              " (891,\n",
              "  17,\n",
              "  'Is NOD1 activated in inflammation?',\n",
              "  'The present study has demonstrated an unexpected role of Nod1 in the development of site-specific vascular inflammation, especially coronary arteritis.',\n",
              "  0),\n",
              " (891,\n",
              "  18,\n",
              "  'Is NOD1 activated in inflammation?',\n",
              "  'Nod1 ligands induce site-specific vascular inflammation',\n",
              "  0),\n",
              " (891,\n",
              "  19,\n",
              "  'Is NOD1 activated in inflammation?',\n",
              "  'Phenotyping of Nod1/2 double deficient mice and characterization of Nod1/2 in systemic inflammation and associated renal disease',\n",
              "  0),\n",
              " (891,\n",
              "  20,\n",
              "  'Is NOD1 activated in inflammation?',\n",
              "  'The present study analyzed Nod1 and Nod2 double deficient (Nod1/2 DKO) mice under physiological and inflammatory conditions',\n",
              "  1),\n",
              " (891,\n",
              "  21,\n",
              "  'Is NOD1 activated in inflammation?',\n",
              "  \"Several inflammatory disorders, such as Crohn's disease and asthma, are linked to genetic changes in either Nod1 or Nod2.\",\n",
              "  0),\n",
              " (891,\n",
              "  22,\n",
              "  'Is NOD1 activated in inflammation?',\n",
              "  'Systematic analysis of Nod1/2 DKO mice revealed a possible role of Nod1/2 in the development of renal disease during systemic inflammation.',\n",
              "  0),\n",
              " (891,\n",
              "  23,\n",
              "  'Is NOD1 activated in inflammation?',\n",
              "  'NOD1 and NOD2 Signaling in Infection and Inflammation',\n",
              "  0),\n",
              " (891,\n",
              "  24,\n",
              "  'Is NOD1 activated in inflammation?',\n",
              "  'NOD1 engagement generates an inflammatory response via activation of NFκB and MAPK pathways.',\n",
              "  0),\n",
              " (891,\n",
              "  25,\n",
              "  'Is NOD1 activated in inflammation?',\n",
              "  'In addition we profile novel inhibitors of RIP2 and NOD1 itself, which specifically inhibit NOD1 ligand induced inflammatory signalling in the vasculature.',\n",
              "  0),\n",
              " (891,\n",
              "  26,\n",
              "  'Is NOD1 activated in inflammation?',\n",
              "  'This data supports the potential utility of NOD1 and RIP2 as therapeutic targets in human disease where vascular inflammation is a clinical feature, such as in sepsis and septic shock.',\n",
              "  0),\n",
              " (891,\n",
              "  27,\n",
              "  'Is NOD1 activated in inflammation?',\n",
              "  'NOD1 expression elicited by iE-DAP in first trimester human trophoblast cells and its potential role in infection-associated inflammation',\n",
              "  0),\n",
              " (891,\n",
              "  28,\n",
              "  'Is NOD1 activated in inflammation?',\n",
              "  'This study aimed to investigate the expression and function of NOD1 in first trimester trophoblast cells, and evaluate the potential role of trophoblast cells in infection-associated inflammation',\n",
              "  0),\n",
              " (891,\n",
              "  29,\n",
              "  'Is NOD1 activated in inflammation?',\n",
              "  'NOD1 may have a role in mediating infection-associated inflammation.',\n",
              "  0),\n",
              " (891,\n",
              "  30,\n",
              "  'Is NOD1 activated in inflammation?',\n",
              "  'Once iE-DAP is recognized by NOD1, the inflammatory response may be induced via NOD1-RICK-NF-κB-mediated pathways.',\n",
              "  0),\n",
              " (891,\n",
              "  31,\n",
              "  'Is NOD1 activated in inflammation?',\n",
              "  'NOD1/2(-/-) mice were protected from HFD-induced inflammation, lipid accumulation, and peripheral insulin intolerance.',\n",
              "  0),\n",
              " (891,\n",
              "  32,\n",
              "  'Is NOD1 activated in inflammation?',\n",
              "  'In contrast, Nod1 gene-deficient mice developed enhanced joint inflammation with concomitant elevated levels of proinflammatory cytokines and cartilage damage, consistent with a model in which Nod1 controls the inflammatory reaction.',\n",
              "  0),\n",
              " (891,\n",
              "  33,\n",
              "  'Is NOD1 activated in inflammation?',\n",
              "  'These data indicate that the NLR family members Nod1 and Nod2 have different functions in controlling inflammation, and that intracellular Nod1-Nod2 interactions may determine the severity of arthritis in this experimental model.',\n",
              "  0),\n",
              " (891,\n",
              "  34,\n",
              "  'Is NOD1 activated in inflammation?',\n",
              "  'Whereas the lymphotoxin pathway was critical for the induction of the B cell chemoattractant CXCL13 in response to Nod1 agonists, B cell accumulation within the spleen following Nod1-induced systemic inflammation was independent of the lymphotoxin pathway.',\n",
              "  0),\n",
              " (891,\n",
              "  35,\n",
              "  'Is NOD1 activated in inflammation?',\n",
              "  'The effect of NOD1 and NOD2 activation on inflammation and the insulin signalling pathway was also assessed.',\n",
              "  0),\n",
              " (891,\n",
              "  36,\n",
              "  'Is NOD1 activated in inflammation?',\n",
              "  'Nonetheless, the role of NOD1 in intraocular inflammation has not been explored.',\n",
              "  0),\n",
              " (891,\n",
              "  37,\n",
              "  'Is NOD1 activated in inflammation?',\n",
              "  'A key role for the endothelium in NOD1 mediated vascular inflammation: comparison to TLR4 responses.',\n",
              "  0),\n",
              " (891,\n",
              "  38,\n",
              "  'Is NOD1 activated in inflammation?',\n",
              "  'We previously demonstrated that human first-trimester trophoblasts express Nod1 and Nod2, which trigger inflammation upon stimulation.',\n",
              "  0),\n",
              " (891,\n",
              "  39,\n",
              "  'Is NOD1 activated in inflammation?',\n",
              "  'In addition, recent studies have revealed a role for intracellular NOD1 receptors in the regulation of vascular inflammation and metabolism.',\n",
              "  0),\n",
              " (891,\n",
              "  40,\n",
              "  'Is NOD1 activated in inflammation?',\n",
              "  'In conclusion, the present findings describe an important role for NOD1 in the development of insulin resistance and inflammation in pregnancies complicated by GDM.',\n",
              "  0),\n",
              " (891,\n",
              "  41,\n",
              "  'Is NOD1 activated in inflammation?',\n",
              "  'Phenotyping of Nod1/2 double deficient mice and characterization of Nod1/2 in systemic inflammation and associated renal disease.',\n",
              "  0),\n",
              " (891,\n",
              "  42,\n",
              "  'Is NOD1 activated in inflammation?',\n",
              "  'Nod1 activation by bacterial iE-DAP induces maternal-fetal inflammation and preterm labor.',\n",
              "  0),\n",
              " (891,\n",
              "  43,\n",
              "  'Is NOD1 activated in inflammation?',\n",
              "  'The nucleotide-binding oligomerisation domain (NOD) intracellular molecules recognise a wide range of microbial products, as well as other intracellular danger signals, thereby initiating inflammation through activation of nuclear factor κB (NFκB).',\n",
              "  0),\n",
              " (891,\n",
              "  44,\n",
              "  'Is NOD1 activated in inflammation?',\n",
              "  'Nod1 ligands induce site-specific vascular inflammation.',\n",
              "  0),\n",
              " (891,\n",
              "  45,\n",
              "  'Is NOD1 activated in inflammation?',\n",
              "  'The nucleotide oligomerization domain (NOD) intracellular molecules recognize a wide range of microbial products as well as other intracellular danger signals, thereby initiating inflammation through activation of nuclear factor KB (NFKB), a central regulator of the terminal processes of human labor and delivery.',\n",
              "  0),\n",
              " (891,\n",
              "  46,\n",
              "  'Is NOD1 activated in inflammation?',\n",
              "  'CONCLUSIONS: We identify NOD proteins as innate immune components that are involved in diet-induced inflammation and insulin intolerance.',\n",
              "  0),\n",
              " (891,\n",
              "  47,\n",
              "  'Is NOD1 activated in inflammation?',\n",
              "  'Nod1 and Nod2 regulation of inflammation in the Salmonella colitis model.',\n",
              "  0),\n",
              " (891,\n",
              "  48,\n",
              "  'Is NOD1 activated in inflammation?',\n",
              "  'In particular, muramyl peptides trigger inflammation, contribute to host defense against microbial infections, and modulate the adaptive immune response to antigens.',\n",
              "  0),\n",
              " (891,\n",
              "  49,\n",
              "  'Is NOD1 activated in inflammation?',\n",
              "  'Systemic and tissue-specific inflammation was assessed using enzyme-linked immunosorbent assays in NOD ligand-injected mice.',\n",
              "  0),\n",
              " (891,\n",
              "  50,\n",
              "  'Is NOD1 activated in inflammation?',\n",
              "  'CONCLUSIONS: We identify NOD proteins as innate immune components that are involved in diet-induced inflammation and insulin intolerance.',\n",
              "  0),\n",
              " (891,\n",
              "  51,\n",
              "  'Is NOD1 activated in inflammation?',\n",
              "  'Acute activation of NOD proteins by mimetics of bacterial PGNs causes whole-body insulin resistance, bolstering the concept that innate immune responses to distinctive bacterial cues directly lead to insulin resistance.',\n",
              "  0),\n",
              " (891,\n",
              "  52,\n",
              "  'Is NOD1 activated in inflammation?',\n",
              "  'Interferon-gamma (IFN gamma), which is a potent pro-inflammatory cytokine in intestinal mucosal inflammation, activates CARD4/NOD1 mRNA transcription in a time- and dose-dependent manner and results in augmentation of CARD4/NOD1 protein in SW480 cells.',\n",
              "  0),\n",
              " (891,\n",
              "  53,\n",
              "  'Is NOD1 activated in inflammation?',\n",
              "  'These studies suggest that the Th1 cytokine, IFN gamma, activates CARD4/NOD1 transcription and regulate innate immune mechanisms in the condition of intestinal mucosal inflammation.',\n",
              "  0),\n",
              " (891,\n",
              "  54,\n",
              "  'Is NOD1 activated in inflammation?',\n",
              "  'NOD1 activation triggers inflammation,',\n",
              "  0),\n",
              " (891,\n",
              "  55,\n",
              "  'Is NOD1 activated in inflammation?',\n",
              "  'In contrast to enhanced leptin mRNA by LPS and TNFα, NOD1 activation suppressed leptin mRNA in adipocytes, suggesting the differential effects of NOD1 activation in adipocytes.',\n",
              "  0),\n",
              " (891,\n",
              "  56,\n",
              "  'Is NOD1 activated in inflammation?',\n",
              "  'Overall, our results suggest that NOD1 represents a novel target for adipose inflammation in obesity.',\n",
              "  0),\n",
              " (891,\n",
              "  57,\n",
              "  'Is NOD1 activated in inflammation?',\n",
              "  'NOD1 activation triggers inflammation, antimicrobial mechanisms and autophagy in both epithelial cells and murine macrophages.',\n",
              "  0),\n",
              " (891,\n",
              "  58,\n",
              "  'Is NOD1 activated in inflammation?',\n",
              "  'NOD1 is an intracellular immune receptor that senses peptidoglycan from Gram-negative bacteria and responds by inducing autophagy and activating NF-κB, leading to inflammation-mediated bacterial clearance; however chronic pathogens can evade NOD1-mediated clearance by altering peptidoglycan structure.',\n",
              "  0),\n",
              " (891,\n",
              "  59,\n",
              "  'Is NOD1 activated in inflammation?',\n",
              "  'Bacterial NOD1 activation is positioned as a component of metabolic endotoxemia that can contribute to hyperlipidemia, systemic inflammation and insulin resistance by acting directly on adipocytes.',\n",
              "  0),\n",
              " (891,\n",
              "  60,\n",
              "  'Is NOD1 activated in inflammation?',\n",
              "  'NOD1 ligand also caused inflammation and insulin resistance directly in primary hepatocytes from WT, but not NOD1(-/-), mice.',\n",
              "  0),\n",
              " (891,\n",
              "  61,\n",
              "  'Is NOD1 activated in inflammation?',\n",
              "  'We conclude that NOD1 activation reduced AHR in allergen-induced lung inflammation, which was accompanied by a reduction of allergen-specific T-cell proliferation.',\n",
              "  0),\n",
              " (891,\n",
              "  62,\n",
              "  'Is NOD1 activated in inflammation?',\n",
              "  'The effect of NOD1 and NOD2 activation on inflammation and the insulin signalling pathway was also assessed.',\n",
              "  0),\n",
              " (891,\n",
              "  63,\n",
              "  'Is NOD1 activated in inflammation?',\n",
              "  'Interferon-gamma (IFN gamma), which is a potent pro-inflammatory cytokine in intestinal mucosal inflammation, activates CARD4/NOD1 mRNA transcription in a time- and dose-dependent manner and results in augmentation of CARD4/NOD1 protein in SW480 cells.',\n",
              "  0),\n",
              " (1566,\n",
              "  0,\n",
              "  'What was the purpose of the FANTOM4 project?',\n",
              "  'The international Functional Annotation Of the Mammalian Genomes 4 (FANTOM4) research collaboration set out to better understand the transcriptional network that regulates macrophage differentiation and to uncover novel components of the transcriptome employing a series of high-throughput experiments.',\n",
              "  1),\n",
              " (1566,\n",
              "  1,\n",
              "  'What was the purpose of the FANTOM4 project?',\n",
              "  'The international Functional Annotation Of the Mammalian Genomes 4 (FANTOM4) research collaboration set out to better understand the transcriptional network that regulates macrophage differentiation and to uncover novel components of the transcriptome employing a series of high-throughput experiments',\n",
              "  1),\n",
              " (1566,\n",
              "  2,\n",
              "  'What was the purpose of the FANTOM4 project?',\n",
              "  'Using deep sequencing (deepCAGE), the FANTOM4 study measured the genome-wide dynamics of transcription-start-site usage in the human monocytic cell line THP-1 throughout a time course of growth arrest and differentiation.',\n",
              "  1),\n",
              " (1566,\n",
              "  3,\n",
              "  'What was the purpose of the FANTOM4 project?',\n",
              "  'The international Functional Annotation Of the Mammalian Genomes 4 (FANTOM4) research collaboration set out to better understand the transcriptional network that regulates macrophage differentiation and to uncover novel components of the transcriptome employing a series of high-throughput experiments.',\n",
              "  1),\n",
              " (497,\n",
              "  0,\n",
              "  'Which deiodinase is known to be present in liver?',\n",
              "  'Iodothyronine deiodinase in vitro activity studies in the chicken showed the presence of type I and type III iodothyronine deiodinase activity in both liver and kidney.',\n",
              "  1),\n",
              " (497,\n",
              "  1,\n",
              "  'Which deiodinase is known to be present in liver?',\n",
              "  'In embryonic chicken liver (ECL) two types of iodothyronine deiodinases are expressed: D1 and D3.',\n",
              "  1),\n",
              " (497,\n",
              "  2,\n",
              "  'Which deiodinase is known to be present in liver?',\n",
              "  'In liver homogenates, D1 activity was not correlated with age, whereas D3 activity showed a strong negative correlation with age (r -0.84), with high D3 activities in preterm infants and (except in 1 infant of 35 weeks) absent D3 activity in full-term infants.',\n",
              "  1),\n",
              " (497,\n",
              "  3,\n",
              "  'Which deiodinase is known to be present in liver?',\n",
              "  'In microsomes, D1 activities amounted to 4.3-60 pmol/min/mg protein in fetal livers and to 170-313 pmol/min/mg protein in adult livers, whereas microsomal D3 activities were 0.15-1.45 pmol/min/mg protein in fetuses and <0.1 pmol/min/mg protein in all but one adult.',\n",
              "  1),\n",
              " (497,\n",
              "  4,\n",
              "  'Which deiodinase is known to be present in liver?',\n",
              "  'high D1 and D3 activities in fetal human liver, and high D1 and mostly absent D3 activities in adult human liver.',\n",
              "  1),\n",
              " (1981,\n",
              "  0,\n",
              "  'Does the hERG gene code for a protein which is part of a sodium channel?',\n",
              "  ' The human ether-à-go-go-related gene (hERG 1a) potassium channel is critical for cardiac repolarization',\n",
              "  0),\n",
              " (1981,\n",
              "  1,\n",
              "  'Does the hERG gene code for a protein which is part of a sodium channel?',\n",
              "  'Human ether-a-go-go-related gene (hERG) channels conduct delayed rectifier K(+) current.',\n",
              "  1),\n",
              " (1981,\n",
              "  2,\n",
              "  'Does the hERG gene code for a protein which is part of a sodium channel?',\n",
              "  'The KvLQT1 and minK genes code the slowly activating, delayed rectifier (Iks) potassium channel, the HERG gene code the rapidly activating, delayed rectifier (Ikr) potassium channel of the heart, while the SCN5A gene codes a cardiac sodium channel.',\n",
              "  0),\n",
              " (1981,\n",
              "  3,\n",
              "  'Does the hERG gene code for a protein which is part of a sodium channel?',\n",
              "  'The molecular basis of inherited disorders caused by a mutation in either the gene coding for a particular potassium channel called HERG-or another gene, SCN5A, which codes for the sodium channel and disruption of which results in a loss of inactivation of the Na+ current.',\n",
              "  0),\n",
              " (1981,\n",
              "  4,\n",
              "  'Does the hERG gene code for a protein which is part of a sodium channel?',\n",
              "  'The aim of this study was to test whether a recently reported polymorphism in the HERG gene coding for the rapidly activating delayed rectifier K+ channel has influence on myocardial repolarization.',\n",
              "  0),\n",
              " (1981,\n",
              "  5,\n",
              "  'Does the hERG gene code for a protein which is part of a sodium channel?',\n",
              "  'Mutations in KvLQT1, minK and HERG genes affects repolarising, rectifier potassium currents, while SCN5A mutations cause delayed inactivation and reopening of the cardiac sodium channel, which initiates the depolarisation of cardiac cells.',\n",
              "  0),\n",
              " (1981,\n",
              "  6,\n",
              "  'Does the hERG gene code for a protein which is part of a sodium channel?',\n",
              "  \"A human genetic defect associated with 'long Q-T syndrome', an abnormality of cardiac rhythm involving the repolarization of the action potential, was recently found to lie in the HERG gene, which codes for a potassium channel.\",\n",
              "  0),\n",
              " (1981,\n",
              "  7,\n",
              "  'Does the hERG gene code for a protein which is part of a sodium channel?',\n",
              "  'Therefore, matrine and oxymatrine may have the potential to cure LQT2 as a potassium channel activator by promoting hERG channel activation and increasing hERG channel expression.',\n",
              "  0),\n",
              " (1981,\n",
              "  8,\n",
              "  'Does the hERG gene code for a protein which is part of a sodium channel?',\n",
              "  'The human delta1261 mutation of the HERG potassium channel results in a truncated protein that contains a subunit interaction domain and decreases the channel expression.',\n",
              "  0),\n",
              " (1981,\n",
              "  9,\n",
              "  'Does the hERG gene code for a protein which is part of a sodium channel?',\n",
              "  'HERG (human eag-related gene) encodes an inward-rectifier potassium channel formed by the assembly of four subunits.',\n",
              "  0),\n",
              " (1981,\n",
              "  10,\n",
              "  'Does the hERG gene code for a protein which is part of a sodium channel?',\n",
              "  'The human ether-?-go-go-related gene (HERG) encodes the pore-forming subunit of the rapidly activating delayed rectifier potassium channel in the heart.',\n",
              "  0),\n",
              " (1981,\n",
              "  11,\n",
              "  'Does the hERG gene code for a protein which is part of a sodium channel?',\n",
              "  'The human ether-a-go-go-related gene (hERG) encodes the rapidly activating, delayed rectifier potassium channel (IKr) important for cardiac repolarization.',\n",
              "  1),\n",
              " (1981,\n",
              "  12,\n",
              "  'Does the hERG gene code for a protein which is part of a sodium channel?',\n",
              "  'Role of glycosylation in cell surface expression and stability of HERG potassium channels.',\n",
              "  0),\n",
              " (1981,\n",
              "  13,\n",
              "  'Does the hERG gene code for a protein which is part of a sodium channel?',\n",
              "  'The human ERG protein (HERG or Kv 11.1) encoded by the human ether-a-go-go-related gene (herg) is the pore-forming subunit of the cardiac delayed rectifier potassium current (IKr) responsible for action potential (AP) repolarization',\n",
              "  0),\n",
              " (1981,\n",
              "  14,\n",
              "  'Does the hERG gene code for a protein which is part of a sodium channel?',\n",
              "  'Human ether-a-go-go related gene (herg) encoding HERG K(+) channel has been demonstrated in many previous studies with its association to cell cycle progression and growth in tumor cells',\n",
              "  0),\n",
              " (1981,\n",
              "  15,\n",
              "  'Does the hERG gene code for a protein which is part of a sodium channel?',\n",
              "  \"A human genetic defect associated with 'long Q-T syndrome', an abnormality of cardiac rhythm involving the repolarization of the action potential, was recently found to lie in the HERG gene, which codes for a potassium channel.\",\n",
              "  0),\n",
              " (1981,\n",
              "  16,\n",
              "  'Does the hERG gene code for a protein which is part of a sodium channel?',\n",
              "  'Drug-induced long QT syndrome: hERG K+ channel block and disruption of protein trafficking by fluoxetine and norfluoxetine.',\n",
              "  0),\n",
              " (1981,\n",
              "  17,\n",
              "  'Does the hERG gene code for a protein which is part of a sodium channel?',\n",
              "  'OBJECTIVES: The aim of this study was to test whether a recently reported polymorphism in the HERG gene coding for the rapidly activating delayed rectifier K+ channel has influence on myocardial repolarization.',\n",
              "  0),\n",
              " (1981,\n",
              "  18,\n",
              "  'Does the hERG gene code for a protein which is part of a sodium channel?',\n",
              "  'The aim of this study was to test whether the K897T polymorphism of the KCNH2 (HERG) gene coding for the rapidly activating delayed rectifier K+ channel influences cardiac repolarization assessed by principal component analysis (PCA) of T-wave morphology.',\n",
              "  0),\n",
              " (1981,\n",
              "  19,\n",
              "  'Does the hERG gene code for a protein which is part of a sodium channel?',\n",
              "  'The KvLQT1 and minK genes code the slowly activating, delayed rectifier (Iks) potassium channel, the HERG gene code the rapidly activating, delayed rectifier (Ikr) potassium channel of the heart, while the SCN5A gene codes a cardiac sodium channel.',\n",
              "  0),\n",
              " (1981,\n",
              "  20,\n",
              "  'Does the hERG gene code for a protein which is part of a sodium channel?',\n",
              "  'All code for subunits of sodium or potassium channels: two a subunits of the potassium channels (QVLQT1 for LQT1, HERG for LQT2), the a subunit of the sodium channel INa (SCN5A for LQT3), and two regulatory subunits of potassium channels (KCNE1 for LQT5 regulating the KvLQT1 channel and MiRP1 regulating HERG).',\n",
              "  0),\n",
              " (1981,\n",
              "  21,\n",
              "  'Does the hERG gene code for a protein which is part of a sodium channel?',\n",
              "  'The corresponding genes code for potassium channels KVLQT1 (LQT1) and HERG (LQT2) and the sodium channel SCN5A (LQT3).',\n",
              "  0),\n",
              " (1981,\n",
              "  22,\n",
              "  'Does the hERG gene code for a protein which is part of a sodium channel?',\n",
              "  'The molecular basis of inherited disorders caused by a mutation in either the gene coding for a particular potassium channel called HERG-or another gene, SCN5A, which codes for the sodium channel and disruption of which results in a loss of inactivation of the Na+ current.',\n",
              "  0),\n",
              " (1981,\n",
              "  23,\n",
              "  'Does the hERG gene code for a protein which is part of a sodium channel?',\n",
              "  'There may also be correlation between the strength of binding of the medicinal substance to the potassium channel coded by the HERG gene and prolongation of the QT interval.',\n",
              "  0),\n",
              " (1981,\n",
              "  24,\n",
              "  'Does the hERG gene code for a protein which is part of a sodium channel?',\n",
              "  \"We demonstrate that the mRNA 3'UTR of ppk29 affects neuronal firing rates and associated heat-induced seizures by acting as a natural antisense transcript (NAT) that regulates the neuronal mRNA levels of seizure (sei), the Drosophila homolog of the human Ether-à-go-go Related Gene (hERG) potassium channel.\",\n",
              "  0),\n",
              " (1981,\n",
              "  25,\n",
              "  'Does the hERG gene code for a protein which is part of a sodium channel?',\n",
              "  'Mutations in KvLQT1, minK and HERG genes affects repolarising, rectifier potassium currents, while SCN5A mutations cause delayed inactivation and reopening of the cardiac sodium channel, which initiates the depolarisation of cardiac cells.',\n",
              "  0),\n",
              " (1981,\n",
              "  26,\n",
              "  'Does the hERG gene code for a protein which is part of a sodium channel?',\n",
              "  'Among the congenital forms, particularly interest is focused on the potassium channel coded by the HERG gene located on chromosome 7 and with a key role in the normal electric cardiac activity.',\n",
              "  0),\n",
              " (1981,\n",
              "  27,\n",
              "  'Does the hERG gene code for a protein which is part of a sodium channel?',\n",
              "  'By employing heterologous expression and making comparisons to cells expressing wild-type human-ether-a-go-go-related protein (HERG), a potassium channel that contributes to I(Kr) current in ventricular cardiomyocytes, we demonstrate activation of an elevated endoplasmic reticulum (ER) stress response by the mutant I593R HERG potassium channel implicated in long QT syndrome type 2.',\n",
              "  0),\n",
              " (1981,\n",
              "  28,\n",
              "  'Does the hERG gene code for a protein which is part of a sodium channel?',\n",
              "  'Correction of defective protein trafficking of a mutant HERG potassium channel in human long QT syndrome.',\n",
              "  0),\n",
              " (1981,\n",
              "  29,\n",
              "  'Does the hERG gene code for a protein which is part of a sodium channel?',\n",
              "  'Mutations in the human ether-à-go-go-related gene (HERG), which encodes a delayed-rectifier potassium channel,',\n",
              "  1),\n",
              " (1981,\n",
              "  30,\n",
              "  'Does the hERG gene code for a protein which is part of a sodium channel?',\n",
              "  's HERG and KvLQT1 potassium channel genes',\n",
              "  0),\n",
              " (1981,\n",
              "  31,\n",
              "  'Does the hERG gene code for a protein which is part of a sodium channel?',\n",
              "  'HERG encodes the cardiac I(Kr) potassium channel.',\n",
              "  0),\n",
              " (1981,\n",
              "  32,\n",
              "  'Does the hERG gene code for a protein which is part of a sodium channel?',\n",
              "  'block of the cardiac potassium channel human ether-à-go-go-related gene (hERG)',\n",
              "  1),\n",
              " (1981,\n",
              "  33,\n",
              "  'Does the hERG gene code for a protein which is part of a sodium channel?',\n",
              "  'The human ether-a-go-go-related gene (hERG) encodes the pore-forming α-subunit of the rapidly activating delayed rectifier K(+) channel in the heart, which plays a critical role in cardiac action potential repolarization.',\n",
              "  1),\n",
              " (1981,\n",
              "  34,\n",
              "  'Does the hERG gene code for a protein which is part of a sodium channel?',\n",
              "  'Effects of donepezil on hERG potassium channels.',\n",
              "  0),\n",
              " (1981,\n",
              "  35,\n",
              "  'Does the hERG gene code for a protein which is part of a sodium channel?',\n",
              "  'Human ether-a-go-go related-gene K⁺ channels (hERG) participate in the regulation of tumor cell proliferation and apoptosis.',\n",
              "  0),\n",
              " (1981,\n",
              "  36,\n",
              "  'Does the hERG gene code for a protein which is part of a sodium channel?',\n",
              "  'HERG channel activity is up-regulated by growth factors',\n",
              "  0),\n",
              " (1981,\n",
              "  37,\n",
              "  'Does the hERG gene code for a protein which is part of a sodium channel?',\n",
              "  ' hERG potassium channels',\n",
              "  0),\n",
              " (1981,\n",
              "  38,\n",
              "  'Does the hERG gene code for a protein which is part of a sodium channel?',\n",
              "  'HERG, a K+ channel gene.',\n",
              "  0),\n",
              " (821,\n",
              "  0,\n",
              "  'Is K-63 linked protein ubiquitination related to proteasomal degradation?',\n",
              "  'In contrast to K48-linked polyubiquitin chains, K63-linked polyubiquitin chains function in nonproteasomal biological processes.',\n",
              "  1),\n",
              " (821,\n",
              "  1,\n",
              "  'Is K-63 linked protein ubiquitination related to proteasomal degradation?',\n",
              "  'Modification of proteins by the addition of lysine (K)-63-linked polyubiquitin (polyUb) chains is suggested to play important roles in a variety of cellular events, including DNA repair, signal transduction, and receptor endocytosis.',\n",
              "  1),\n",
              " (821,\n",
              "  2,\n",
              "  'Is K-63 linked protein ubiquitination related to proteasomal degradation?',\n",
              "  'Ubiquitination is best known for its role in targeting proteins for degradation by the proteasome, but evidence of the nonproteolytic functions of ubiquitin is also rapidly accumulating.',\n",
              "  0),\n",
              " (821,\n",
              "  3,\n",
              "  'Is K-63 linked protein ubiquitination related to proteasomal degradation?',\n",
              "  'One example of the regulatory, rather than proteolytic, function of ubiquitin is provided by study of the tumor necrosis factor (TNF) receptor-associated factor (TRAF) proteins, which function as ubiquitin ligases to synthesize lysine 63 (K(63))-linked polyubiquitin chains to mediate protein kinase activation through a proteasome-independent mechanism.',\n",
              "  1),\n",
              " (821,\n",
              "  4,\n",
              "  'Is K-63 linked protein ubiquitination related to proteasomal degradation?',\n",
              "  'Some TRAF proteins, such as TRAF2 and TRAF3, have recently been shown to have a positive role in the canonical pathway that activates nuclear factor kappaB (NF-kappaB) through IkappaB kinase beta (IKKbeta), but a negative role in the noncanonical pathway that activates NF-kappaB through IKKalpha.',\n",
              "  1),\n",
              " (821,\n",
              "  5,\n",
              "  'Is K-63 linked protein ubiquitination related to proteasomal degradation?',\n",
              "  'These opposing roles of TRAF proteins may be linked to their ability to synthesize distinct forms of polyubiquitin chains.',\n",
              "  0),\n",
              " (821,\n",
              "  6,\n",
              "  'Is K-63 linked protein ubiquitination related to proteasomal degradation?',\n",
              "  'Indeed, the TRAF2-interacting protein RIP can mediate IKK activation when it is modified by K(63) polyubiquitin chains, but is targeted to degradation by the proteasome when it is K(48)-polyubiquitinted by the NF-kappaB inhibitor A20.',\n",
              "  1),\n",
              " (821,\n",
              "  7,\n",
              "  'Is K-63 linked protein ubiquitination related to proteasomal degradation?',\n",
              "  'Thus, ubiquitin chains are dynamic switches that can influence signaling outputs in dramatically different ways.',\n",
              "  0),\n",
              " (821,\n",
              "  8,\n",
              "  'Is K-63 linked protein ubiquitination related to proteasomal degradation?',\n",
              "  'Importantly, although Lys-48-linked ubiquitin chains appear to trigger proteasomal degradation, the presence of Lys-63-linked ubiquitin chains suggests that ubiquitination of IP(3)Rs may have physiological consequences beyond signaling for degradation.',\n",
              "  0),\n",
              " (821,\n",
              "  9,\n",
              "  'Is K-63 linked protein ubiquitination related to proteasomal degradation?',\n",
              "  'Chains of ubiquitin linked via lysine 48 (K48) are associated with protein degradation while chains linked via lysine 63 (K63) are associated with intracellular signaling.',\n",
              "  0),\n",
              " (821,\n",
              "  10,\n",
              "  'Is K-63 linked protein ubiquitination related to proteasomal degradation?',\n",
              "  'Lys(48)-linked chains target proteins for proteasomal degradation, and Lys(63)-linked chains function in signal transduction, endocytosis and DNA repair',\n",
              "  0),\n",
              " (821,\n",
              "  11,\n",
              "  'Is K-63 linked protein ubiquitination related to proteasomal degradation?',\n",
              "  'Remarkably, the attached Lys-48- and Lys-63-linked ubiquitin chains are homogeneous and are segregated to separate IP(3)R subunits, and Lys-48-linked ubiquitin chains, but not Lys-63-linked chains, are required for IP(3)R degradation',\n",
              "  0),\n",
              " (821,\n",
              "  12,\n",
              "  'Is K-63 linked protein ubiquitination related to proteasomal degradation?',\n",
              "  'Activated inositol 1,4,5-trisphosphate receptors are modified by homogeneous Lys-48- and Lys-63-linked ubiquitin chains, but only Lys-48-linked chains are required for degradation.',\n",
              "  0),\n",
              " (607,\n",
              "  0,\n",
              "  'Is microRNA(miRNA) 30 involved in post-ischemic cardiac remodeling?',\n",
              "  'The myocardium of the failing heart undergoes a number of structural alterations, most notably hypertrophy of cardiac myocytes and an increase in extracellular matrix proteins, often seen as primary fibrosi',\n",
              "  1),\n",
              " (607,\n",
              "  1,\n",
              "  'Is microRNA(miRNA) 30 involved in post-ischemic cardiac remodeling?',\n",
              "  'Connective tissue growth factor (CTGF) is a key molecule in the process of fibrosis and therefore seems an attractive therapeutic target',\n",
              "  1),\n",
              " (607,\n",
              "  2,\n",
              "  'Is microRNA(miRNA) 30 involved in post-ischemic cardiac remodeling?',\n",
              "  'CTGF is importantly regulated by 2 major cardiac microRNAs (miRNAs), miR-133 and miR-30.',\n",
              "  1),\n",
              " (607,\n",
              "  3,\n",
              "  'Is microRNA(miRNA) 30 involved in post-ischemic cardiac remodeling?',\n",
              "  'the expression of both miRNAs was inversely related to the amount of CTGF in 2 rodent models of heart disease and in human pathological left ventricular hypertrophy.',\n",
              "  0),\n",
              " (607,\n",
              "  4,\n",
              "  'Is microRNA(miRNA) 30 involved in post-ischemic cardiac remodeling?',\n",
              "  'Second, in cultured cardiomyocytes and fibroblasts, knockdown of these miRNAs increased CTGF levels.',\n",
              "  1),\n",
              " (607,\n",
              "  5,\n",
              "  'Is microRNA(miRNA) 30 involved in post-ischemic cardiac remodeling?',\n",
              "  'Third, overexpression of miR-133 or miR-30c decreased CTGF levels, which was accompanied by decreased production of collagens.',\n",
              "  0),\n",
              " (607,\n",
              "  6,\n",
              "  'Is microRNA(miRNA) 30 involved in post-ischemic cardiac remodeling?',\n",
              "  'miR-30 importantly limit the production of CTGF',\n",
              "  0),\n",
              " (607,\n",
              "  7,\n",
              "  'Is microRNA(miRNA) 30 involved in post-ischemic cardiac remodeling?',\n",
              "  'miR-30 directly downregulate CTGF, a key profibrotic protein, and thereby establish an important role for these miRNAs in the control of structural changes in the extracellular matrix of the myocardium.',\n",
              "  1),\n",
              " (3377,\n",
              "  0,\n",
              "  'Is KAT2A involved in Acute myeloid leukemia (AML)?',\n",
              "  'Acute myeloid leukemia (AML) is an aggressive cancer with a poor prognosis, for which mainstream treatments have not changed for decades.',\n",
              "  0),\n",
              " (3377,\n",
              "  1,\n",
              "  'Is KAT2A involved in Acute myeloid leukemia (AML)?',\n",
              "  'To identify additional therapeutic targets in AML, we optimize a genome-wide clustered regularly interspaced short palindromic repeats (CRISPR) screening platform and use it to identify genetic vulnerabilities in AML cells.',\n",
              "  0),\n",
              " (3377,\n",
              "  2,\n",
              "  'Is KAT2A involved in Acute myeloid leukemia (AML)?',\n",
              "  'We identify 492 AML-specific cell-essential genes, including several established therapeutic targets such as DOT1L, BCL2, and MEN1, and many other genes including clinically actionable candidates.',\n",
              "  0),\n",
              " (3377,\n",
              "  3,\n",
              "  'Is KAT2A involved in Acute myeloid leukemia (AML)?',\n",
              "  'We validate selected genes using genetic and pharmacological inhibition, and chose KAT2A as a candidate for downstream study.',\n",
              "  0),\n",
              " (3377,\n",
              "  4,\n",
              "  'Is KAT2A involved in Acute myeloid leukemia (AML)?',\n",
              "  'KAT2A inhibition demonstrated anti-AML activity by inducing myeloid differentiation and apoptosis, and suppressed the growth of primary human AMLs of diverse genotypes while sparing normal hemopoietic stem-progenitor cells.',\n",
              "  1),\n",
              " (3377,\n",
              "  5,\n",
              "  'Is KAT2A involved in Acute myeloid leukemia (AML)?',\n",
              "  'Our results propose that KAT2A inhibition should be investigated as a therapeutic strategy in AML and provide a large number of genetic vulnerabilities of this leukemia that can be pursued in downstream studies.',\n",
              "  1),\n",
              " (3377,\n",
              "  6,\n",
              "  'Is KAT2A involved in Acute myeloid leukemia (AML)?',\n",
              "  'Our results propose that KAT2A inhibition should be investigated as a therapeutic strategy in AML and provide a large number of genetic vulnerabilities of this leukemia that can be pursued in downstream studies.',\n",
              "  1),\n",
              " (3377,\n",
              "  7,\n",
              "  'Is KAT2A involved in Acute myeloid leukemia (AML)?',\n",
              "  'KAT2A inhibition demonstrated anti-AML activity by inducing myeloid differentiation and apoptosis, and suppressed the growth of primary human AMLs of diverse genotypes while sparing normal hemopoietic stem-progenitor cells.',\n",
              "  1),\n",
              " (3377,\n",
              "  8,\n",
              "  'Is KAT2A involved in Acute myeloid leukemia (AML)?',\n",
              "  'Our results propose that KAT2A inhibition should be investigated as a therapeutic strategy in AML and provide a large number of genetic vulnerabilities of this leukemia that can be pursued in downstream studies.',\n",
              "  0),\n",
              " (3377,\n",
              "  9,\n",
              "  'Is KAT2A involved in Acute myeloid leukemia (AML)?',\n",
              "  'KAT2A inhibition demonstrated anti-AML activity by inducing myeloid differentiation and apoptosis, and suppressed the growth of primary human AMLs of diverse genotypes while sparing normal hemopoietic stem-progenitor cells.',\n",
              "  1),\n",
              " (747,\n",
              "  0,\n",
              "  'Is Hirschsprung disease one of the characteristics of the Mowat-Wilson syndrome?',\n",
              "  'Mowat-Wilson syndrome is a genetic disease caused by heterozygous mutations or deletions of the zinc finger E-box-binding homeobox 2 (ZEB2) gene.',\n",
              "  0),\n",
              " (747,\n",
              "  1,\n",
              "  'Is Hirschsprung disease one of the characteristics of the Mowat-Wilson syndrome?',\n",
              "  'The syndrome is characterized by typical facial features, moderate-to-severe mental retardation, epilepsy and variable congenital malformations, including Hirschsprung disease, genital anomalies, congenital heart disease, agenesis of the corpus callosum, and eye defects',\n",
              "  1),\n",
              " (747,\n",
              "  2,\n",
              "  'Is Hirschsprung disease one of the characteristics of the Mowat-Wilson syndrome?',\n",
              "  \"Mowat-Wilson syndrome (MWS) is a rare genetic condition where variable and multiple congenital anomalies including Hirschsprung's disease, intellectual disability, and prominent facial features are present\",\n",
              "  1),\n",
              " (747,\n",
              "  3,\n",
              "  'Is Hirschsprung disease one of the characteristics of the Mowat-Wilson syndrome?',\n",
              "  'Individuals with Mowat-Wilson syndrome (MWS; OMIM#235730) have characteristic facial features, a variety of congenital anomalies such as Hirschsprung disease, and intellectual disabilities caused by mutation or deletion of ZEB2 gene',\n",
              "  0),\n",
              " (747,\n",
              "  4,\n",
              "  'Is Hirschsprung disease one of the characteristics of the Mowat-Wilson syndrome?',\n",
              "  'Mowat-Wilson syndrome is a genetic disease characterized by typical facial features, Hirschsprung disease and multiple congenital abnormalities',\n",
              "  0),\n",
              " (747,\n",
              "  5,\n",
              "  'Is Hirschsprung disease one of the characteristics of the Mowat-Wilson syndrome?',\n",
              "  'Mowat-Wilson syndrome (MWS) is a severe intellectual disability (ID)-distinctive facial gestalt-multiple congenital anomaly syndrome, commonly associating microcephaly, epilepsy, corpus callosum agenesis, conotruncal heart defects, urogenital malformations and Hirschsprung disease (HSCR)',\n",
              "  0),\n",
              " (747,\n",
              "  6,\n",
              "  'Is Hirschsprung disease one of the characteristics of the Mowat-Wilson syndrome?',\n",
              "  ' The prevalence of Mowat-Wilson syndrome is currently unknown, but it seems that Mowat-Wilson syndrome is underdiagnosed, particularly in patients without Hirschsprung disease.',\n",
              "  0),\n",
              " (747,\n",
              "  7,\n",
              "  'Is Hirschsprung disease one of the characteristics of the Mowat-Wilson syndrome?',\n",
              "  'Mowat-Wilson syndrome is a mental retardation-multiple congenital anomaly syndrome characterized by a typical facies, developmental delay, epilepsy, and variable congenital malformations, including Hirschsprung disease, urogenital anomalies, congenital heart disease, and agenesis of the corpus callosum.',\n",
              "  1),\n",
              " (747,\n",
              "  8,\n",
              "  'Is Hirschsprung disease one of the characteristics of the Mowat-Wilson syndrome?',\n",
              "  '\"Mowat-Wilson\" syndrome with and without Hirschsprung disease is a distinct, recognizable multiple congenital anomalies-mental retardation syndrome caused by mutations in the zinc finger homeo box 1B gene.',\n",
              "  0),\n",
              " (747,\n",
              "  9,\n",
              "  'Is Hirschsprung disease one of the characteristics of the Mowat-Wilson syndrome?',\n",
              "  'Mowat-Wilson syndrome (MWS) is a recently delineated mental retardation; a multiple congenital anomaly syndrome characterised by a typical facial gestalt, Hirschsprung disease or severe constipation, genitourinary anomaly, congenital heart defects, agenesis of corpus callosum and eye defects.',\n",
              "  0),\n",
              " (747,\n",
              "  10,\n",
              "  'Is Hirschsprung disease one of the characteristics of the Mowat-Wilson syndrome?',\n",
              "  'We report a girl who had Hirschsprung disease in association with distinct facial appearance, microcephaly, agenesis of the corpus callosum and mental retardation (Mowat-Wilson syndrome).',\n",
              "  0),\n",
              " (747,\n",
              "  11,\n",
              "  'Is Hirschsprung disease one of the characteristics of the Mowat-Wilson syndrome?',\n",
              "  'Mowat-Wilson syndrome (MWS) is characterized by severe mental retardation with seizures, specific facial dysmorphism, Hirschsprung disease, anomalies of the corpus callosum, and genitourinary and cardiac malformations.',\n",
              "  0),\n",
              " (747,\n",
              "  12,\n",
              "  'Is Hirschsprung disease one of the characteristics of the Mowat-Wilson syndrome?',\n",
              "  'BACKGROUND/PURPOSE: Patients with zinc finger homeo box 1B (ZFHX1B) mutations or deletions develop multiple congenital anomalies including Hirschsprung disease, known as Mowat-Wilson syndrome (MWS).',\n",
              "  0),\n",
              " (747,\n",
              "  13,\n",
              "  'Is Hirschsprung disease one of the characteristics of the Mowat-Wilson syndrome?',\n",
              "  'Severe clinical course of Hirschsprung disease in a Mowat-Wilson syndrome patient.',\n",
              "  0),\n",
              " (747,\n",
              "  14,\n",
              "  'Is Hirschsprung disease one of the characteristics of the Mowat-Wilson syndrome?',\n",
              "  'Mowat-Wilson syndrome (MWS) is a multiple congenital anomaly syndrome characterized by a distinct facial phenotype (high forehead, frontal bossing, large eyebrows, medially flaring and sparse in the middle part, hypertelorism, deep set but large eyes, large and uplifted ear lobes, with a central depression, saddle nose with prominent rounded nasal tip, prominent columella, open mouth, with M-shaped upper lip, frequent smiling, and a prominent but narrow and triangular pointed chin), moderate-to-severe intellectual deficiency, epilepsy and variable congenital malformations including Hirschsprung disease (HSCR), genitourinary anomalies (in particular hypospadias in males), congenital heart defects, agenesis of the corpus callosum and eye anomalies.',\n",
              "  0),\n",
              " (747,\n",
              "  15,\n",
              "  'Is Hirschsprung disease one of the characteristics of the Mowat-Wilson syndrome?',\n",
              "  'Mowat-Wilson syndrome (MWS) is a multiple congenital anomaly syndrome characterized by a distinct facial phenotype, Hirschsprung disease, microcephaly and mental retardation.',\n",
              "  0),\n",
              " (747,\n",
              "  16,\n",
              "  'Is Hirschsprung disease one of the characteristics of the Mowat-Wilson syndrome?',\n",
              "  'Mowat-Wilson syndrome is a genetic disease characterized by typical facial features, Hirschsprung disease and multiple congenital abnormalities.',\n",
              "  0),\n",
              " (747,\n",
              "  17,\n",
              "  'Is Hirschsprung disease one of the characteristics of the Mowat-Wilson syndrome?',\n",
              "  'Supernumerary intestinal muscle coat in a patient with Hirschsprung disease/Mowat-Wilson syndrome.',\n",
              "  0),\n",
              " (747,\n",
              "  18,\n",
              "  'Is Hirschsprung disease one of the characteristics of the Mowat-Wilson syndrome?',\n",
              "  'We present the 1st case report of an additional enteric smooth muscle layer in a patient with Mowat-Wilson syndrome and Hirschsprung disease.',\n",
              "  0),\n",
              " (747,\n",
              "  19,\n",
              "  'Is Hirschsprung disease one of the characteristics of the Mowat-Wilson syndrome?',\n",
              "  'Mowat-Wilson\" syndrome with and without Hirschsprung disease is a distinct, recognizable multiple congenital anomalies-mental retardation syndrome caused by mutations in the zinc finger homeo box 1B gene.',\n",
              "  0),\n",
              " (747,\n",
              "  20,\n",
              "  'Is Hirschsprung disease one of the characteristics of the Mowat-Wilson syndrome?',\n",
              "  'Mowat-Wilson syndrome (MWS) is an autosomal dominant intellectual disability syndrome characterised by unique facial features and congenital anomalies such as Hirschsprung disease, congenital heart defects, corpus callosum agenesis and urinary tract anomalies.',\n",
              "  0),\n",
              " (747,\n",
              "  21,\n",
              "  'Is Hirschsprung disease one of the characteristics of the Mowat-Wilson syndrome?',\n",
              "  'Mowat-Wilson syndrome (MWS) is a mental retardation syndrome associated with distinctive facial features, microcephaly, epilepsy, and a variable spectrum of congenital anomalies, including Hirschsprung disease (HSCR), agenesis of the corpus callosum, genitourinary abnormalities, and congenital heart disease.',\n",
              "  1),\n",
              " (747,\n",
              "  22,\n",
              "  'Is Hirschsprung disease one of the characteristics of the Mowat-Wilson syndrome?',\n",
              "  'Mowat-Wilson syndrome is a mental retardation-multiple congenital anomaly syndrome characterized by a typical facies, developmental delay, epilepsy, and variable congenital malformations, including Hirschsprung disease, urogenital anomalies, congenital heart disease, and agenesis of the corpus callosum',\n",
              "  1),\n",
              " (747,\n",
              "  23,\n",
              "  'Is Hirschsprung disease one of the characteristics of the Mowat-Wilson syndrome?',\n",
              "  'Mowat-Wilson syndrome is a genetic disorder characterized by a distinct facial appearance, moderate-to-severe mental retardation, microcephaly, agenesis of the corpus callosum, Hirschsprung disease, congenital heart disease, and genital anomalies',\n",
              "  0),\n",
              " (747,\n",
              "  24,\n",
              "  'Is Hirschsprung disease one of the characteristics of the Mowat-Wilson syndrome?',\n",
              "  'We present the 1st case report of an additional enteric smooth muscle layer in a patient with Mowat-Wilson syndrome and Hirschsprung disease',\n",
              "  0),\n",
              " (747,\n",
              "  25,\n",
              "  'Is Hirschsprung disease one of the characteristics of the Mowat-Wilson syndrome?',\n",
              "  'Mowat-Wilson syndrome (MWS) is an autosomal dominant intellectual disability syndrome characterised by unique facial features and congenital anomalies such as Hirschsprung disease, congenital heart defects, corpus callosum agenesis and urinary tract anomalies',\n",
              "  0),\n",
              " (747,\n",
              "  26,\n",
              "  'Is Hirschsprung disease one of the characteristics of the Mowat-Wilson syndrome?',\n",
              "  'Mowat-Wilson syndrome (MWS) is characterized by severe mental retardation with seizures, specific facial dysmorphism, Hirschsprung disease, anomalies of the corpus callosum, and genitourinary and cardiac malformations',\n",
              "  0),\n",
              " (747,\n",
              "  27,\n",
              "  'Is Hirschsprung disease one of the characteristics of the Mowat-Wilson syndrome?',\n",
              "  'zfhz1b is the causative gene for Mowat-Wilson syndrome, in which patients demonstrate developmental delay and Hirschsprung disease, as well as other anomalies.',\n",
              "  0),\n",
              " (747,\n",
              "  28,\n",
              "  'Is Hirschsprung disease one of the characteristics of the Mowat-Wilson syndrome?',\n",
              "  'Mowat-Wilson syndrome (MWS) is a mental retardation syndrome associated with distinctive facial features, microcephaly, epilepsy, and a variable spectrum of congenital anomalies, including Hirschsprung disease (HSCR), agenesis of the corpus callosum, genitourinary abnormalities, and congenital heart disease',\n",
              "  0),\n",
              " (747,\n",
              "  29,\n",
              "  'Is Hirschsprung disease one of the characteristics of the Mowat-Wilson syndrome?',\n",
              "  \"Outcomes of Hirschsprung's disease associated with Mowat-Wilson syndrome.\",\n",
              "  0),\n",
              " (747,\n",
              "  30,\n",
              "  'Is Hirschsprung disease one of the characteristics of the Mowat-Wilson syndrome?',\n",
              "  'Mowat-Wilson syndrome (MWS) is a multiple congenital anomaly syndrome characterized by a distinct facial phenotype, Hirschsprung disease, microcephaly and mental retardation',\n",
              "  0),\n",
              " (2698,\n",
              "  0,\n",
              "  'Which method is Proseek based on?',\n",
              "  'We used proximity extension immunoassay (PEA, Proseek Multiplex, Olink) to assess the serum levels of ninety-two inflammation-related proteins in Czech patients with SLE (n\\xa0=\\xa075) and age-matched healthy control subjects (n\\xa0=\\xa023).',\n",
              "  1),\n",
              " (2698,\n",
              "  1,\n",
              "  'Which method is Proseek based on?',\n",
              "  'Using highly-sensitive Proximity Extension ImmunoAssay on 92 cancer biomarkers (Proseek',\n",
              "  1),\n",
              " (2423,\n",
              "  0,\n",
              "  \"Is Solanezumab effective for Alzheimer's Disease?\",\n",
              "  'An analysis of publicly available data from the Phase II studies for bapineuzumab and solanezumab indicates that neither compound produced compelling evidence of drug-like behavior that would justify their progression into pivotal trials.',\n",
              "  0),\n",
              " (2423,\n",
              "  1,\n",
              "  \"Is Solanezumab effective for Alzheimer's Disease?\",\n",
              "  \"Notably, a recent study of solanezumab, an amyloid β monoclonal antibody, raises hope for the further therapeutic potential of immunotherapy, not only in Alzheimer's disease, but also for other neurodegenerative disorders, including Parkinson's disease.\",\n",
              "  1),\n",
              " (2423,\n",
              "  2,\n",
              "  \"Is Solanezumab effective for Alzheimer's Disease?\",\n",
              "  \"For example, Eli Lilly announced a major change to its closely watched clinical trial for the Alzheimer's drug solanezumab which failed to reach statistical significance.\",\n",
              "  0),\n",
              " (2423,\n",
              "  3,\n",
              "  \"Is Solanezumab effective for Alzheimer's Disease?\",\n",
              "  'Areas covered: This contradiction prompted us to review all study phases of Intravenous Immunoglobulin (IVIG), Bapineuzumab, Solanezumab, Avagacestat and Dimebolin to shed more light on these recent failures.',\n",
              "  0),\n",
              " (2423,\n",
              "  4,\n",
              "  \"Is Solanezumab effective for Alzheimer's Disease?\",\n",
              "  \" Results from phase III clinical trials in mild-to-moderate Alzheimer's disease (AD) patients with two monoclonal antibodies bapineuzumab and solanezumab and intravenous immunoglobulin have been disappointing.\",\n",
              "  1),\n",
              " (2423,\n",
              "  5,\n",
              "  \"Is Solanezumab effective for Alzheimer's Disease?\",\n",
              "  'Subsequent analysis of pooled data from both phase III trials with solanezumab showed a reduction in cognitive decline in patients with mild AD.',\n",
              "  1),\n",
              " (2423,\n",
              "  6,\n",
              "  \"Is Solanezumab effective for Alzheimer's Disease?\",\n",
              "  'Secondary analyses of EXPEDITION studies suggested a smaller functional effect of solanezumab relative to cognition.',\n",
              "  0),\n",
              " (2423,\n",
              "  7,\n",
              "  \"Is Solanezumab effective for Alzheimer's Disease?\",\n",
              "  'An increasing effect of solanezumab over 18\\xa0months was shown for cognition and function.',\n",
              "  0),\n",
              " (2423,\n",
              "  8,\n",
              "  \"Is Solanezumab effective for Alzheimer's Disease?\",\n",
              "  \"RESULTS: In the mild AD population, less cognitive and functional decline was observed with solanezumab (n\\xa0=\\xa0659) versus placebo (n\\xa0=\\xa0663), measured by Alzheimer's Disease Assessment Scale Cognitive subscale, Mini-Mental State Examination, and Alzheimer's Disease Cooperative Study-Activities of Daily Living functional scale Instrumental ADLs.\",\n",
              "  0),\n",
              " (2423,\n",
              "  9,\n",
              "  \"Is Solanezumab effective for Alzheimer's Disease?\",\n",
              "  \"The promising results obtained with aducanumab and solanezumab against Alzheimer's disease (AD) strengthen the vaccine approach to prevent AD, despite of the many clinical setbacks.\",\n",
              "  0),\n",
              " (2423,\n",
              "  10,\n",
              "  \"Is Solanezumab effective for Alzheimer's Disease?\",\n",
              "  'CONCLUSIONS: Solanezumab, a humanized monoclonal antibody that binds amyloid, failed to improve cognition or functional ability.',\n",
              "  1),\n",
              " (2423,\n",
              "  11,\n",
              "  \"Is Solanezumab effective for Alzheimer's Disease?\",\n",
              "  \"Results from phase III clinical trials in mild-to-moderate Alzheimer's disease (AD) patients with two monoclonal antibodies bapineuzumab and solanezumab and intravenous immunoglobulin have been disappointing.\",\n",
              "  1),\n",
              " (2014,\n",
              "  0,\n",
              "  'Which protein mediates the replacement of H2A by H2A.Z in the yeast Saccharomyces cerevisiae?',\n",
              "  'The chromatin remodeler SWR1 mediates site-specific incorporation of H2A.Z by a multi-step histone replacement reaction, evicting histone H2A-H2B from the canonical nucleosome and depositing the H2A.Z-H2B dimer',\n",
              "  1),\n",
              " (2014,\n",
              "  1,\n",
              "  'Which protein mediates the replacement of H2A by H2A.Z in the yeast Saccharomyces cerevisiae?',\n",
              "  'he multisubunit nucleosome-remodeling enzyme complex SWR1, conserved from yeast to mammals, catalyzes the ATP-dependent replacement of histone H2A in canonical nucleosomes with H2A.Z.',\n",
              "  1),\n",
              " (2014,\n",
              "  2,\n",
              "  'Which protein mediates the replacement of H2A by H2A.Z in the yeast Saccharomyces cerevisiae?',\n",
              "  'Our results show that the Swr1-Z domain can deliver the H2A.Z-H2B dimer to the DNA-(H3-H4)2 tetrasome to form the nucleosome by a histone chaperone mechanism.',\n",
              "  0),\n",
              " (2014,\n",
              "  3,\n",
              "  'Which protein mediates the replacement of H2A by H2A.Z in the yeast Saccharomyces cerevisiae?',\n",
              "  'H2A.Z is deposited into chromatin by the SWR1 complex and is subject to acetylation of its four N-terminal tail lysine residues by the NuA4 and SAGA histone acetyltransferase complexes.',\n",
              "  0),\n",
              " (2014,\n",
              "  4,\n",
              "  'Which protein mediates the replacement of H2A by H2A.Z in the yeast Saccharomyces cerevisiae?',\n",
              "  'H2A.Z deposition is controlled by SWR-C chromatin remodeling enzymes that catalyze the nucleosomal exchange of canonical H2A with H2A.Z.',\n",
              "  0),\n",
              " (2014,\n",
              "  5,\n",
              "  'Which protein mediates the replacement of H2A by H2A.Z in the yeast Saccharomyces cerevisiae?',\n",
              "  'Saccharomyces cerevisiae Swr1, a Swi2/Snf2-related ATPase, is the catalytic core of a multisubunit chromatin remodeling enzyme, called the SWR1 complex, that efficiently replaces conventional histone H2A in nucleosomes with histone H2A.Z.',\n",
              "  1),\n",
              " (2014,\n",
              "  6,\n",
              "  'Which protein mediates the replacement of H2A by H2A.Z in the yeast Saccharomyces cerevisiae?',\n",
              "  'The system demonstrates ATP- and SWR1-complex-dependent replacement of histone H2A for histone H2A.Z on a preassembled nucleosome array.',\n",
              "  0),\n",
              " (2014,\n",
              "  7,\n",
              "  'Which protein mediates the replacement of H2A by H2A.Z in the yeast Saccharomyces cerevisiae?',\n",
              "  'We identified a complex containing 13 different polypeptides associated with a soluble pool of H2A.Z in Saccharomyces cerevisiae.',\n",
              "  0),\n",
              " (2014,\n",
              "  8,\n",
              "  'Which protein mediates the replacement of H2A by H2A.Z in the yeast Saccharomyces cerevisiae?',\n",
              "  'This complex was designated SWR1-Com in reference to the Swr1p subunit, a Swi2/Snf2-paralog.',\n",
              "  0),\n",
              " (2014,\n",
              "  9,\n",
              "  'Which protein mediates the replacement of H2A by H2A.Z in the yeast Saccharomyces cerevisiae?',\n",
              "  'The SWR1 complex replaces the canonical histone H2A with the variant H2A.Z (Htz1 in yeast) at specific chromatin regions.',\n",
              "  0),\n",
              " (2014,\n",
              "  10,\n",
              "  'Which protein mediates the replacement of H2A by H2A.Z in the yeast Saccharomyces cerevisiae?',\n",
              "  'In Saccharomyces cerevisiae, deposition of histone H2AZ is mediated by the multiprotein SWR1 complex, which catalyzes ATP-dependent exchange of nucleosomal histone H2A for H2AZ.',\n",
              "  0),\n",
              " (2014,\n",
              "  11,\n",
              "  'Which protein mediates the replacement of H2A by H2A.Z in the yeast Saccharomyces cerevisiae?',\n",
              "  'In Saccharomyces cerevisiae, chromatin deposition of histone H2AZ is mediated by the fourteen-subunit SWR1 complex, which catalyzes ATP-dependent exchange of nucleosomal histone H2A for H2AZ.',\n",
              "  0),\n",
              " (2014,\n",
              "  12,\n",
              "  'Which protein mediates the replacement of H2A by H2A.Z in the yeast Saccharomyces cerevisiae?',\n",
              "  'This incorporation is mediated by the conserved SWR1 complex, which replaces histone H2A in canonical nucleosomes with H2A.Z in an ATP-dependent manner.',\n",
              "  0),\n",
              " (2014,\n",
              "  13,\n",
              "  'Which protein mediates the replacement of H2A by H2A.Z in the yeast Saccharomyces cerevisiae?',\n",
              "  'Saccharomyces cerevisiae Swr1, a Swi2/Snf2-related ATPase, is the catalytic core of a multisubunit chromatin remodeling enzyme, called the SWR1 complex, that efficiently replaces conventional histone H2A in nucleosomes with histone H2A.Z.',\n",
              "  1),\n",
              " (2014,\n",
              "  14,\n",
              "  'Which protein mediates the replacement of H2A by H2A.Z in the yeast Saccharomyces cerevisiae?',\n",
              "  'In Saccharomyces cerevisiae, chromatin deposition of histone H2AZ is mediated by the fourteen-subunit SWR1 complex, which catalyzes ATP-dependent exchange of nucleosomal histone H2A for H2AZ.',\n",
              "  0),\n",
              " (2014,\n",
              "  15,\n",
              "  'Which protein mediates the replacement of H2A by H2A.Z in the yeast Saccharomyces cerevisiae?',\n",
              "  'In Saccharomyces cerevisiae, deposition of histone H2AZ is mediated by the multiprotein SWR1 complex, which catalyzes ATP-dependent exchange of nucleosomal histone H2A for H2AZ.',\n",
              "  0),\n",
              " (2014,\n",
              "  16,\n",
              "  'Which protein mediates the replacement of H2A by H2A.Z in the yeast Saccharomyces cerevisiae?',\n",
              "  'The Swc4p protein, encoded by an essential gene, is shared by two chromatin-remodeling complexes in Saccharomyces cerevisiae cells: NuA4 (nucleosome acetyltransferase of H4) and SWR1.',\n",
              "  0),\n",
              " (2014,\n",
              "  17,\n",
              "  'Which protein mediates the replacement of H2A by H2A.Z in the yeast Saccharomyces cerevisiae?',\n",
              "  'The Saccharomyces cerevisiae protein Yaf9 is a subunit of both the essential histone acetyltransferase complex NuA4 and the ATP-dependent chromatin remodeling complex SWR1-C, which deposits histone variant H2A.Z into euchromatin.',\n",
              "  0),\n",
              " (2014,\n",
              "  18,\n",
              "  'Which protein mediates the replacement of H2A by H2A.Z in the yeast Saccharomyces cerevisiae?',\n",
              "  'Here, we identify and characterize SERRATED LEAVES AND EARLY FLOWERING (SEF), an Arabidopsis (Arabidopsis thaliana) homolog of the yeast SWC6 protein, a conserved subunit of the SWR1/SRCAP complex.',\n",
              "  0),\n",
              " (2014,\n",
              "  19,\n",
              "  'Which protein mediates the replacement of H2A by H2A.Z in the yeast Saccharomyces cerevisiae?',\n",
              "  'The SWR1/SRCAP complex is a chromatin-remodeling complex that has been shown to be involved in substitution of histone H2A by the histone variant H2A.Z in yeast (Saccharomyces cerevisiae) and animals.',\n",
              "  0),\n",
              " (2014,\n",
              "  20,\n",
              "  'Which protein mediates the replacement of H2A by H2A.Z in the yeast Saccharomyces cerevisiae?',\n",
              "  'The Swc4p protein, encoded by an essential gene, is shared by two chromatin-remodeling complexes in Saccharomyces cerevisiae cells: NuA4 (nucleosome acetyltransferase of H4) and SWR1.',\n",
              "  0),\n",
              " (2014,\n",
              "  21,\n",
              "  'Which protein mediates the replacement of H2A by H2A.Z in the yeast Saccharomyces cerevisiae?',\n",
              "  'The SWR1 complex (SWR1C) in yeast catalyzes the replacement of nucleosomal H2A with the H2AZ variant, which ensures full activation of underlying genes.',\n",
              "  0),\n",
              " (2014,\n",
              "  22,\n",
              "  'Which protein mediates the replacement of H2A by H2A.Z in the yeast Saccharomyces cerevisiae?',\n",
              "  'In addition, H2AZ, a substrate of SWR1C, interacts with both PIE1 and AtSWC2.',\n",
              "  0),\n",
              " (2014,\n",
              "  23,\n",
              "  'Which protein mediates the replacement of H2A by H2A.Z in the yeast Saccharomyces cerevisiae?',\n",
              "  'The incorporation of H2AZ into chromatin is dependent on the SWR1 complex, which catalyses the replacement of conventional histone H2A with H2AZ.',\n",
              "  0),\n",
              " (2014,\n",
              "  24,\n",
              "  'Which protein mediates the replacement of H2A by H2A.Z in the yeast Saccharomyces cerevisiae?',\n",
              "  'H2A histone-fold and DNA elements in nucleosome activate SWR1-mediated H2A.Z replacement in budding yeast.',\n",
              "  0),\n",
              " (2014,\n",
              "  25,\n",
              "  'Which protein mediates the replacement of H2A by H2A.Z in the yeast Saccharomyces cerevisiae?',\n",
              "  'They report that reversal of H2A.Z replacement is mediated by SWR1 and related INO80 on an H2A.Z nucleosome carrying H3K56Q.',\n",
              "  0),\n",
              " (2014,\n",
              "  26,\n",
              "  'Which protein mediates the replacement of H2A by H2A.Z in the yeast Saccharomyces cerevisiae?',\n",
              "  'The chromatin remodeler SWR1 mediates site-specific incorporation of H2A.Z by a multi-step histone replacement reaction, evicting histone H2A-H2B from the canonical nucleosome and depositing the H2A.Z-H2B dimer.',\n",
              "  1),\n",
              " (2014,\n",
              "  27,\n",
              "  'Which protein mediates the replacement of H2A by H2A.Z in the yeast Saccharomyces cerevisiae?',\n",
              "  'This incorporation is mediated by the conserved SWR1 complex, which replaces histone H2A in canonical nucleosomes with H2A.Z in an ATP-dependent manner.',\n",
              "  0),\n",
              " (2014,\n",
              "  28,\n",
              "  'Which protein mediates the replacement of H2A by H2A.Z in the yeast Saccharomyces cerevisiae?',\n",
              "  'Here, we show that the JmjC domain protein Msc1 is a novel component of the fission yeast Swr1 complex and is required for Swr1-mediated incorporation of H2A.Z into nucleosomes at gene promoters.',\n",
              "  0),\n",
              " (4036,\n",
              "  0,\n",
              "  'Is colistin an antibiotic?',\n",
              "  'all antibiotics tested, apart from colistin,',\n",
              "  1),\n",
              " (4036,\n",
              "  1,\n",
              "  'Is colistin an antibiotic?',\n",
              "  'Among the selected antibiotics, there were 12 fluoroquinolone antibiotics (tosufloxacin, levofloxacin, sparfloxacin, clinafloxacin, pazufloxacin, gatifloxacin, enrofloxacin, lomefloxacin, norfloxacin, fleroxacin, flumequine, ciprofloxacin), 15 beta-lactam or cephalosporin antibiotics (cefmenoxime, cefotaxime, ceftizoxime, cefotiam, cefdinir, cefoperazone, cefpiramide, cefamandole, cefixime, ceftibuten, cefmetazole, cephalosporin C, aztreonam, piperacillintazobactam, mezlocillin), 3 tetracycline antibiotics (meclocycline, doxycycline, tetracycline), 2 membrane-acting agents (colistin and clofoctol),',\n",
              "  1),\n",
              " (4036,\n",
              "  2,\n",
              "  'Is colistin an antibiotic?',\n",
              "  ' Mice received an antibiotic cocktail (kanamycin, gentamicin, colistin, metronidazole, and vancomycin) for 96\\u2009h.',\n",
              "  1),\n",
              " (1023,\n",
              "  0,\n",
              "  'Which is the underlying mechanism for exon skipping used to treat Duchenne muscular dystrophy?',\n",
              "  'Antisense-mediated exon skipping therapy is a promising therapeutic approach that uses short DNA-like molecules called antisense oligonucleotides (AOs) to skip over/splice out the mutated part of the gene to produce a shortened but functional dystrophin protein',\n",
              "  1),\n",
              " (1023,\n",
              "  1,\n",
              "  'Which is the underlying mechanism for exon skipping used to treat Duchenne muscular dystrophy?',\n",
              "  'many DMD patients need exon skipping of multiple exons in order to restore the reading frame, depending on how many base pairs the mutated exon(s) and adjacent exons have.',\n",
              "  1),\n",
              " (1023,\n",
              "  2,\n",
              "  'Which is the underlying mechanism for exon skipping used to treat Duchenne muscular dystrophy?',\n",
              "  'Theoretically, multiple exon skipping could be used to treat approximately 90%, 80%, and 98% of DMD patients with deletion, duplication, and nonsense mutations, respectively',\n",
              "  1),\n",
              " (1023,\n",
              "  3,\n",
              "  'Which is the underlying mechanism for exon skipping used to treat Duchenne muscular dystrophy?',\n",
              "  'Antisense oligonucleotides (AONs) that bind to complementary sequences of the dystrophin pre-mRNA to induce skipping of the targeted exon by modulating pre-mRNA splicing are promising therapeutic agents for DMD.',\n",
              "  1),\n",
              " (1023,\n",
              "  4,\n",
              "  'Which is the underlying mechanism for exon skipping used to treat Duchenne muscular dystrophy?',\n",
              "  'Such AONs can restore the open reading frame of the DMD gene and produce internally deleted, yet partially functional dystrophin protein isoforms in skeletal muscle',\n",
              "  1),\n",
              " (1023,\n",
              "  5,\n",
              "  'Which is the underlying mechanism for exon skipping used to treat Duchenne muscular dystrophy?',\n",
              "  \"Of all novel molecular interventions currently being investigated for Duchenne muscular dystrophy, perhaps the most promising method aiming to restore dystrophin expression to diseased cells is known as 'exon skipping' or splice-modulation, whereby antisense oligonucleotides eliminate the deleterious effects of DMD mutations by modulating dystrophin pre-messenger RNA splicing, such that functional dystrophin protein is produced.\",\n",
              "  0),\n",
              " (1023,\n",
              "  6,\n",
              "  'Which is the underlying mechanism for exon skipping used to treat Duchenne muscular dystrophy?',\n",
              "  'We previously conducted a proof of principle; dose escalation study in Duchenne muscular dystrophy (DMD) patients using the morpholino splice-switching oligonucleotide AVI-4658 (eteplirsen) that induces skipping of dystrophin exon 51 in patients with relevant deletions, restores the open reading frame and induces dystrophin protein expression after intramuscular (i.m.)',\n",
              "  0),\n",
              " (1023,\n",
              "  7,\n",
              "  'Which is the underlying mechanism for exon skipping used to treat Duchenne muscular dystrophy?',\n",
              "  'injection',\n",
              "  0),\n",
              " (1023,\n",
              "  8,\n",
              "  'Which is the underlying mechanism for exon skipping used to treat Duchenne muscular dystrophy?',\n",
              "  'Using antisense oligonucleotides (AONs), the disrupted DMD reading frame is restored, allowing generation of partially functional dystrophin and conversion of a severe Duchenne into a milder Becker muscular dystrophy phenotype.',\n",
              "  0),\n",
              " (1023,\n",
              "  9,\n",
              "  'Which is the underlying mechanism for exon skipping used to treat Duchenne muscular dystrophy?',\n",
              "  'Using antisense oligonucleotides (AONs) targeting specific exons the DMD reading frame is restored and partially functional dystrophins are produced.',\n",
              "  0),\n",
              " (1099,\n",
              "  0,\n",
              "  'Are there any HCV replication inhibitors available?',\n",
              "  'We report here the discovery of the first small-molecule HCV infectivity inhibitor, GS-563253, also called HCV infectivity inhibitor 1 (HCV II-1).',\n",
              "  1),\n",
              " (1099,\n",
              "  1,\n",
              "  'Are there any HCV replication inhibitors available?',\n",
              "  'Resistance to mericitabine (prodrug of HCV NS5B polymerase inhibitor PSI-6130) is rare and conferred by the NS5B S282T mutation.',\n",
              "  1),\n",
              " (1099,\n",
              "  2,\n",
              "  'Are there any HCV replication inhibitors available?',\n",
              "  \"We tested the ability of NA808 to inhibit SPT's enzymatic activity in FLR3-1 replicon cells\",\n",
              "  0),\n",
              " (1099,\n",
              "  3,\n",
              "  'Are there any HCV replication inhibitors available?',\n",
              "  'The SPT inhibitor NA808 prevents replication of HCV genotypes 1a, 1b, 2a, 3a, and 4a in cultured hepatocytes and in mice with humanized livers.',\n",
              "  0),\n",
              " (1099,\n",
              "  4,\n",
              "  'Are there any HCV replication inhibitors available?',\n",
              "  'Vaniprevir (phase III clinical trials) and MK-5172 (phase II clinical trials) are two potent antiviral compounds that target NS3/4A protease',\n",
              "  0),\n",
              " (1099,\n",
              "  5,\n",
              "  'Are there any HCV replication inhibitors available?',\n",
              "  'treatment for hepatitis C virus (HCV) infection has been significantly improved with the approval of the first two HCV NS3/4A protease inhibitors, telaprevir (Incivek) and boceprevir (Victrelis).',\n",
              "  1),\n",
              " (1099,\n",
              "  6,\n",
              "  'Are there any HCV replication inhibitors available?',\n",
              "  'Combination therapy with telaprevir and BMS-788329 (NS5A inhibitor) reduced serum HCV RNA to undetectable levels.',\n",
              "  0),\n",
              " (1099,\n",
              "  7,\n",
              "  'Are there any HCV replication inhibitors available?',\n",
              "  'The presence of an NS3-V36A telaprevir resistance mutation resulted in poor response to telaprevir monotherapy but showed significant HCV reduction when telaprevir was combined with BMS-788329.',\n",
              "  0),\n",
              " (1099,\n",
              "  8,\n",
              "  'Are there any HCV replication inhibitors available?',\n",
              "  'However, a BMS-788329-resistant strain emerged at low frequency.',\n",
              "  0),\n",
              " (1099,\n",
              "  9,\n",
              "  'Are there any HCV replication inhibitors available?',\n",
              "  'Infection with a BMS-788329-resistant NS5A-L31V mutation rapidly resulted in gain of an additional NS5A-Y93A mutation that conferred telaprevir resistance during combination therapy',\n",
              "  0),\n",
              " (1099,\n",
              "  10,\n",
              "  'Are there any HCV replication inhibitors available?',\n",
              "  'HCV NS5A replication complex inhibitors, exemplified by Daclatasvir (BMS-790052), represent a new class of DAA',\n",
              "  1),\n",
              " (1099,\n",
              "  11,\n",
              "  'Are there any HCV replication inhibitors available?',\n",
              "  'ACH-806 (or GS-9132) is a novel, small-molecule inhibitor specific for hepatitis C virus (HCV).',\n",
              "  1),\n",
              " (1099,\n",
              "  12,\n",
              "  'Are there any HCV replication inhibitors available?',\n",
              "  'Telaprevir and boceprevir are the first two protease inhibitor (PI) DAAs to be approved for combination therapy with pegylated interferon (PEG-IFN) and ribavirin (RBV).',\n",
              "  0),\n",
              " (1099,\n",
              "  13,\n",
              "  'Are there any HCV replication inhibitors available?',\n",
              "  'symmetrical bidentate structure of the NS5A inhibitor BMS-790052, a series of new monodentate molecules were designed',\n",
              "  0),\n",
              " (1099,\n",
              "  14,\n",
              "  'Are there any HCV replication inhibitors available?',\n",
              "  'In vitro, boceprevir is more active than telaprevir against the HCV G3 NS3/4A enzyme in cell-based and biochemical assays and against G3 isolates in replicon assays',\n",
              "  0),\n",
              " (1099,\n",
              "  15,\n",
              "  'Are there any HCV replication inhibitors available?',\n",
              "  'Alisporivir is the most advanced host-targeting antiviral in clinical development.',\n",
              "  0),\n",
              " (1099,\n",
              "  16,\n",
              "  'Are there any HCV replication inhibitors available?',\n",
              "  'Alisporivir blocks HCV replication by neutralizing the peptidyl-prolyl isomerase activity of the abundant host cytosolic protein, cyclophilin A',\n",
              "  0),\n",
              " (1099,\n",
              "  17,\n",
              "  'Are there any HCV replication inhibitors available?',\n",
              "  'Interestingly, the NS5A inhibitor daclatasvir (BMS-790052) caused a decrease in serum HCV RNA levels by about two orders of magnitude within 6 h of administration',\n",
              "  0),\n",
              " (2399,\n",
              "  0,\n",
              "  'What are the side effects of deflazacort in DMD patients?',\n",
              "  'Side effects were growth failure and weight gain, facial fullness, blood pressure, bone health, cataracts, gastrointestinal symptoms, behavior, hypertrichosis, and need for medication interventions.',\n",
              "  1),\n",
              " (2130,\n",
              "  0,\n",
              "  'Have studies shown that there is no link between DNA methylation patterns and Post Traumatic Stress Disorder?',\n",
              "  'Using pre-deployment SKA2 methylation levels and childhood trauma exposure, we found that the previously published suicide prediction rule significantly predicted post-deployment PTSD symptoms (AUC=0.66, 95% CI: 0.53-0.79) with an optimal sensitivity of 0.81 and specificity of 0.91.',\n",
              "  0),\n",
              " (2130,\n",
              "  1,\n",
              "  'Have studies shown that there is no link between DNA methylation patterns and Post Traumatic Stress Disorder?',\n",
              "  'Permutation analysis using random methylation loci supported these findings.',\n",
              "  0),\n",
              " (2130,\n",
              "  2,\n",
              "  'Have studies shown that there is no link between DNA methylation patterns and Post Traumatic Stress Disorder?',\n",
              "  'Together, these data establish the importance of SKA2 for cortisol stress responsivity and the development of PTSD and provide further evidence that SKA2 is a promising biomarker for stress-related disorders including PTSD.',\n",
              "  0),\n",
              " (2130,\n",
              "  3,\n",
              "  'Have studies shown that there is no link between DNA methylation patterns and Post Traumatic Stress Disorder?',\n",
              "  ' Results provide novel support for PTSD-related accelerated aging in DNAm and extend the evidence base of known DNAm age correlates to the domains of neural integrity and cognition.',\n",
              "  1),\n",
              " (2130,\n",
              "  4,\n",
              "  'Have studies shown that there is no link between DNA methylation patterns and Post Traumatic Stress Disorder?',\n",
              "  'We investigated serum DNA methylation patterns in genomic repetitive elements, LINE-1 and Alu, for post-traumatic stress disorder (PTSD) cases and controls who were US military service members recently deployed to Afghanistan or Iraq.',\n",
              "  0),\n",
              " (2130,\n",
              "  5,\n",
              "  'Have studies shown that there is no link between DNA methylation patterns and Post Traumatic Stress Disorder?',\n",
              "  'In light of its role in glucocorticoid receptor transactivation, we investigated whether SKA2 DNA methylation influences cortisol stress reactivity and is involved in the development of post-traumatic stress disorder (PTSD).',\n",
              "  0),\n",
              " (2130,\n",
              "  6,\n",
              "  'Have studies shown that there is no link between DNA methylation patterns and Post Traumatic Stress Disorder?',\n",
              "  'These results suggest that alterations in global methylation pattern are involved in behavioural adaptation to environmental stress and pinpoint Dlgap2 as a possible target in PTSD.',\n",
              "  0),\n",
              " (2130,\n",
              "  7,\n",
              "  'Have studies shown that there is no link between DNA methylation patterns and Post Traumatic Stress Disorder?',\n",
              "  'Here we examined whether there was a link between an established rat model of post-traumatic stress disorder (PTSD) and Bdnf DNA methylation',\n",
              "  0),\n",
              " (2130,\n",
              "  8,\n",
              "  'Have studies shown that there is no link between DNA methylation patterns and Post Traumatic Stress Disorder?',\n",
              "  'We investigated serum DNA methylation patterns in genomic repetitive elements, LINE-1 and Alu, for post-traumatic stress disorder (PTSD) cases and controls who were US military service members recently deployed to Afghanistan or Iraq.Cases (n = 75) had a postdeployment diagnosis of PTSD',\n",
              "  1),\n",
              " (2130,\n",
              "  9,\n",
              "  'Have studies shown that there is no link between DNA methylation patterns and Post Traumatic Stress Disorder?',\n",
              "  'DNA methylation in repetitive elements and post-traumatic stress disorder: a case-control study of US military service members',\n",
              "  1),\n",
              " (2130,\n",
              "  10,\n",
              "  'Have studies shown that there is no link between DNA methylation patterns and Post Traumatic Stress Disorder?',\n",
              "  'Here we examined whether there was a link between an established rat model of post-traumatic stress disorder (PTSD) and Bdnf DNA methylation.',\n",
              "  0),\n",
              " (2130,\n",
              "  11,\n",
              "  'Have studies shown that there is no link between DNA methylation patterns and Post Traumatic Stress Disorder?',\n",
              "  'DNA methylation in repetitive elements and post-traumatic stress disorder: a case-control study of US military service members.',\n",
              "  1),\n",
              " (2130,\n",
              "  12,\n",
              "  'Have studies shown that there is no link between DNA methylation patterns and Post Traumatic Stress Disorder?',\n",
              "  'AIM: We investigated serum DNA methylation patterns in genomic repetitive elements, LINE-1 and Alu, for post-traumatic stress disorder (PTSD) cases and controls who were US military service members recently deployed to Afghanistan or Iraq.',\n",
              "  0),\n",
              " (2130,\n",
              "  13,\n",
              "  'Have studies shown that there is no link between DNA methylation patterns and Post Traumatic Stress Disorder?',\n",
              "  'Together, these results suggest that psychosocial stress may alter global and gene-specific DNA methylation patterns potentially associated with peripheral immune dysregulation.',\n",
              "  0),\n",
              " (2130,\n",
              "  14,\n",
              "  'Have studies shown that there is no link between DNA methylation patterns and Post Traumatic Stress Disorder?',\n",
              "  'DNA methylation in vulnerability to post-traumatic stress in rats: evidence for the role of the post-synaptic density protein Dlgap2.',\n",
              "  0),\n",
              " (2130,\n",
              "  15,\n",
              "  'Have studies shown that there is no link between DNA methylation patterns and Post Traumatic Stress Disorder?',\n",
              "  'Subjects with PTSD showed a higher DNA methylation of four CpG sites at the BDNF promoter compared with those without PTSD',\n",
              "  1),\n",
              " (2130,\n",
              "  16,\n",
              "  'Have studies shown that there is no link between DNA methylation patterns and Post Traumatic Stress Disorder?',\n",
              "  'Cumulatively, the data suggest that epigenetic variation at SKA2 mediates vulnerability to suicidal behaviors and PTSD through dysregulation of the HPA axis in response to stress.',\n",
              "  0),\n",
              " (3690,\n",
              "  0,\n",
              "  'What is a zoonotic virus?',\n",
              "  'Rift Valley fever virus (RVFV) is a mosquito-borne, zoonotic virus that infects ruminants, including cattle, sheep, goats, camels, and buffalo.',\n",
              "  1),\n",
              " (3690,\n",
              "  1,\n",
              "  'What is a zoonotic virus?',\n",
              "  'Hepatitis E virus (HEV) is a zoonotic virus which circulates in pigs and wild boars as main reservoir species.',\n",
              "  0),\n",
              " (3690,\n",
              "  2,\n",
              "  'What is a zoonotic virus?',\n",
              "  'Rodents and bats are now widely recognised as important sources of zoonotic virus infections in other mammals, including humans.',\n",
              "  0),\n",
              " (3690,\n",
              "  3,\n",
              "  'What is a zoonotic virus?',\n",
              "  'Nipah virus is a zoonotic virus harbored by bats and lethal to humans.',\n",
              "  1),\n",
              " (3690,\n",
              "  4,\n",
              "  'What is a zoonotic virus?',\n",
              "  'A zoonotic disease is a disease that can be passed from animals to humans.',\n",
              "  1),\n",
              " (3690,\n",
              "  5,\n",
              "  'What is a zoonotic virus?',\n",
              "  'Zoonotic viruses may adapt to a human host eventually becoming endemic in humans, but before doing so punctuated outbreaks of the zoonotic virus may be observed.',\n",
              "  1),\n",
              " (3690,\n",
              "  6,\n",
              "  'What is a zoonotic virus?',\n",
              "  'Monkeypox virus (MPXV), a close relative of Variola virus, is a zoonotic virus with an unknown reservoir.',\n",
              "  1),\n",
              " (683,\n",
              "  0,\n",
              "  'What do studies show about the effect of Induced hypothermia in premature babies?',\n",
              "  'Therapeutic hypothermia is a recognized treatment for term infants with hypoxic-ischemic encephalopathy (HIE) in reducing rate of death or neurodevelopmental disabilities.',\n",
              "  1),\n",
              " (683,\n",
              "  1,\n",
              "  'What do studies show about the effect of Induced hypothermia in premature babies?',\n",
              "  'Little is known about applications of this treatment to preterm newborns.',\n",
              "  0),\n",
              " (683,\n",
              "  2,\n",
              "  'What do studies show about the effect of Induced hypothermia in premature babies?',\n",
              "  'Randomised studies have demonstrated the efficacy of hypothermia for the treatment of perinatal hypoxic-ischaemic encephalopathy (HIE) in term or late preterm infants.',\n",
              "  1),\n",
              " (683,\n",
              "  3,\n",
              "  'What do studies show about the effect of Induced hypothermia in premature babies?',\n",
              "  'We aimed to establish the feasibility and safety of mild hypothermia in preterm neonates with NEC and MODS as a prelude to a randomized trial.',\n",
              "  1),\n",
              " (683,\n",
              "  4,\n",
              "  'What do studies show about the effect of Induced hypothermia in premature babies?',\n",
              "  'METHODS: This was a prospective, nonrandomized pilot study of 15 preterm infants who were referred for surgical intervention of advanced NEC and failure of at least 3 organs.',\n",
              "  1),\n",
              " (683,\n",
              "  5,\n",
              "  'What do studies show about the effect of Induced hypothermia in premature babies?',\n",
              "  'Whole-body cooling was achieved by ambient temperature adjustment with or without cooling mattress',\n",
              "  0),\n",
              " (683,\n",
              "  6,\n",
              "  'What do studies show about the effect of Induced hypothermia in premature babies?',\n",
              "  'Mild hypothermia for 48 hours in preterm neonates with severe NEC seems both feasible and safe.',\n",
              "  0),\n",
              " (683,\n",
              "  7,\n",
              "  'What do studies show about the effect of Induced hypothermia in premature babies?',\n",
              "  'Additional investigation of the efficacy of this therapeutic intervention in this population is warranted.',\n",
              "  1),\n",
              " (5,\n",
              "  0,\n",
              "  'Does metformin interfere thyroxine absorption?',\n",
              "  'LT4 absorption is unchanged by concomitant metformin ingestion.',\n",
              "  1),\n",
              " (5,\n",
              "  1,\n",
              "  'Does metformin interfere thyroxine absorption?',\n",
              "  'It has been hypothesized that metformin may suppress serum thyrotropin (TSH) concentrations by enhancing LT4 absorption or by directly affecting the hypothalamic-pituitary axis.',\n",
              "  1),\n",
              " (3738,\n",
              "  0,\n",
              "  'List 3 human diseases caused by  viruses in the family Paramyxoviridae.',\n",
              "  ': Previous controversy was generated over the hypothesis that a paramyxovirus such as measles',\n",
              "  0),\n",
              " (3738,\n",
              "  1,\n",
              "  'List 3 human diseases caused by  viruses in the family Paramyxoviridae.',\n",
              "  'Measles are a systemic infectious disease caused by a single stranded ribonucleic acid virus (measles virus) from the paramyxovirus family.',\n",
              "  0),\n",
              " (3738,\n",
              "  2,\n",
              "  'List 3 human diseases caused by  viruses in the family Paramyxoviridae.',\n",
              "  'Mumps is a Paramyxoviridae virus.',\n",
              "  0),\n",
              " (3738,\n",
              "  3,\n",
              "  'List 3 human diseases caused by  viruses in the family Paramyxoviridae.',\n",
              "  'Since chronic paramyxovirus infection with measles is known to be accompanied by increased production of antiviral antibodies',\n",
              "  0),\n",
              " (3738,\n",
              "  4,\n",
              "  'List 3 human diseases caused by  viruses in the family Paramyxoviridae.',\n",
              "  'Nipah virus (NiV) is a highly pathogenic paramyxovirus which causes fatal encephalitis in up to 75% of infected humans.',\n",
              "  0),\n",
              " (3738,\n",
              "  5,\n",
              "  'List 3 human diseases caused by  viruses in the family Paramyxoviridae.',\n",
              "  'Measles is caused by the paramyxovirus which spreads through airborne droplets',\n",
              "  0),\n",
              " (3738,\n",
              "  6,\n",
              "  'List 3 human diseases caused by  viruses in the family Paramyxoviridae.',\n",
              "  'Newcastle disease virus (NDV) is classified as a member of the superfamily Mononegavirales in the family Paramyxoviridae.',\n",
              "  1),\n",
              " (3738,\n",
              "  7,\n",
              "  'List 3 human diseases caused by  viruses in the family Paramyxoviridae.',\n",
              "  'Many of these identified genes were also required for infection by two other NS RNA viruses, the lymphocytic choriomeningitis virus of the Arenaviridae family and human parainfluenza virus type 3 of the Paramyxoviridae family.',\n",
              "  0),\n",
              " (3738,\n",
              "  8,\n",
              "  'List 3 human diseases caused by  viruses in the family Paramyxoviridae.',\n",
              "  'Measles virus ( MeV) , a highly contagious member of the Paramyxoviridae family , causes measles in humans .',\n",
              "  1),\n",
              " (3738,\n",
              "  9,\n",
              "  'List 3 human diseases caused by  viruses in the family Paramyxoviridae.',\n",
              "  'The human metapneumovirus ( hMPV ) is a newly reported respiratory virus belonging to the Paramyxoviridae family that has been associated with bronchiolitis and pneumonia in young children .',\n",
              "  0),\n",
              " (3738,\n",
              "  10,\n",
              "  'List 3 human diseases caused by  viruses in the family Paramyxoviridae.',\n",
              "  'Two members of the morbillivirus genus of the family Paramyxoviridae , canine distemper virus ( CDV ) and measles virus ( MV) , are well-known for their ability to cause a chronic demyelinating disease of the CNS in their natural hosts , dogs and humans , respectively .',\n",
              "  0),\n",
              " (3738,\n",
              "  11,\n",
              "  'List 3 human diseases caused by  viruses in the family Paramyxoviridae.',\n",
              "  'Respiratory syncytial virus ( RSV ) belongs to the family Paramyxoviridae and is the single most important cause of serious lower respiratory tract infections in young children , yet no highly effective treatment or vaccine is available.',\n",
              "  0),\n",
              " (3738,\n",
              "  12,\n",
              "  'List 3 human diseases caused by  viruses in the family Paramyxoviridae.',\n",
              "  'Human respiratory syncytial virus ( RSV) , a member of the Paramyxoviridae family , is the most important viral agent of pediatric respiratory tract disease worldwide.',\n",
              "  0),\n",
              " (3738,\n",
              "  13,\n",
              "  'List 3 human diseases caused by  viruses in the family Paramyxoviridae.',\n",
              "  'Respiratory syncytial virus ( RSV) , an RNA virus in the family Paramyxoviridae , causes respiratory disease in humans .',\n",
              "  1),\n",
              " (3738,\n",
              "  14,\n",
              "  'List 3 human diseases caused by  viruses in the family Paramyxoviridae.',\n",
              "  'Measles virus ( MV ) is a member of the Paramyxoviridae family and an important human pathogen causing strong immunosuppression in affected individuals and a considerable number of deaths worldwide.',\n",
              "  0),\n",
              " (3738,\n",
              "  15,\n",
              "  'List 3 human diseases caused by  viruses in the family Paramyxoviridae.',\n",
              "  'Hendra ( HeV ) and Nipah ( NiV ) viruses ( genus Henipavirus ( HNV; family Paramyxoviridae ) are emerging zoonotic agents that can cause severe respiratory distress and acute encephalitis in humans .',\n",
              "  1),\n",
              " (3738,\n",
              "  16,\n",
              "  'List 3 human diseases caused by  viruses in the family Paramyxoviridae.',\n",
              "  'Many of the common respiratory illnesses of infancy and childhood are caused by viruses of the Paramyxoviridae family, in particular measles virus, respiratory syncytial (RS) virus and parainfluenzavirus type 3 (PI3).',\n",
              "  1),\n",
              " (3738,\n",
              "  17,\n",
              "  'List 3 human diseases caused by  viruses in the family Paramyxoviridae.',\n",
              "  'Here we report simultaneous outbreaks of two distinct human respiratory viruses, human metapneumovirus (MPV; Pneumoviridae: Metapneumovirus) and human respirovirus 3 (HRV3; Paramyxoviridae; Respirovirus, formerly known as parainfluenza virus 3), in two chimpanzee (Pan troglodytes schweinfurthii) communities in the same forest in Uganda in December 2016 and January 2017.',\n",
              "  0),\n",
              " (3738,\n",
              "  18,\n",
              "  'List 3 human diseases caused by  viruses in the family Paramyxoviridae.',\n",
              "  'The large virus family Paramyxoviridae includes some of the most significant human and livestock viruses, such as measles-, distemper-, mumps-, parainfluenza-, Newcastle disease-, respiratory syncytial virus and metapneumoviruses.',\n",
              "  0),\n",
              " (3178,\n",
              "  0,\n",
              "  'What is the cause of Krabbe disease?',\n",
              "  'Krabbe disease (KD) is a rare disease caused by the deficiency of β-galactocerebrosidase.',\n",
              "  1),\n",
              " (3178,\n",
              "  1,\n",
              "  'What is the cause of Krabbe disease?',\n",
              "  'Globoid cell leukodystrophy (GLD), or Krabbe disease, is an inherited, neurologic disorder that results from deficiency of a lysosomal enzyme, galactosylceramidase.',\n",
              "  1),\n",
              " (3178,\n",
              "  2,\n",
              "  'What is the cause of Krabbe disease?',\n",
              "  'Krabbe disease is a rare neurodegenerative genetic disorder caused by deficiency of galactocerebrosidase.',\n",
              "  1),\n",
              " (3462,\n",
              "  0,\n",
              "  'What is the mechanism of action of Erdafitinib?',\n",
              "  'Safety, pharmacokinetic, and pharmacodynamics of erdafitinib, a pan-fibroblast growth factor receptor (FGFR) tyrosine kinase inhibitor, in patients with advanced or refractory solid tumors.',\n",
              "  1),\n",
              " (3462,\n",
              "  1,\n",
              "  'What is the mechanism of action of Erdafitinib?',\n",
              "  'Introduction This phase 1, open-label, multicenter, single-arm, dose-escalation study aimed to evaluate safety, pharmacokinetics (PK), and pharmacodynamics of erdafitinib (JNJ-42756493), an oral selective pan-fibroblast growth factor receptor (FGFR) tyrosine kinase inhibitor, and to determine the recommended phase 2 dose in Japanese patients with advanced or refractory solid tumors.',\n",
              "  1),\n",
              " (3462,\n",
              "  2,\n",
              "  'What is the mechanism of action of Erdafitinib?',\n",
              "  'Findings from a phase II study indicate clinical efficacy with erdafitinib in patients with FGFR-altered inoperable or metastatic urothelial carcinoma.',\n",
              "  0),\n",
              " (3462,\n",
              "  3,\n",
              "  'What is the mechanism of action of Erdafitinib?',\n",
              "  'Robust responses were seen with this investigational pan-FGFR inhibitor, including in patients who did not respond to prior immunotherapy.',\n",
              "  0),\n",
              " (3462,\n",
              "  4,\n",
              "  'What is the mechanism of action of Erdafitinib?',\n",
              "  'In a phase 1 clinical trial, erdafitinib, a pan FGFR inhibitor, was well tolerated with a manageable toxicity profile.',\n",
              "  0),\n",
              " (3462,\n",
              "  5,\n",
              "  'What is the mechanism of action of Erdafitinib?',\n",
              "  'Co-cultured fibroblasts promoted SW620 and HCT116 CRC spheroid invasion, and this was prevented by the SRC and FGFR kinase inhibitors Dasatinib and Erdafitinib, respectively.',\n",
              "  0),\n",
              " (3462,\n",
              "  6,\n",
              "  'What is the mechanism of action of Erdafitinib?',\n",
              "  'Discovery and Pharmacological Characterization of JNJ-42756493 (Erdafitinib), a Functionally Selective Small-Molecule FGFR Family Inhibitor.',\n",
              "  0),\n",
              " (3462,\n",
              "  7,\n",
              "  'What is the mechanism of action of Erdafitinib?',\n",
              "  'JNJ-42756493, erdafitinib, is an orally active small molecule with potent tyrosine kinase inhibitory activity against all four FGFR family members and selectivity versus other highly related kinases.',\n",
              "  0),\n",
              " (3462,\n",
              "  8,\n",
              "  'What is the mechanism of action of Erdafitinib?',\n",
              "  'Oncogenic Characterization and Pharmacologic Sensitivity of Activating Fibroblast Growth Factor Receptor (FGFR) Genetic Alterations to the Selective FGFR Inhibitor Erdafitinib.',\n",
              "  0),\n",
              " (3462,\n",
              "  9,\n",
              "  'What is the mechanism of action of Erdafitinib?',\n",
              "  'Pathway inhibition in response to FGFR inhibitor treatment was assessed in cell lines (both in vitro and in vivo) and in samples from patients treated with the FGFR inhibitor JNJ-42756493 (erdafitinib).',\n",
              "  0),\n",
              " (3462,\n",
              "  10,\n",
              "  'What is the mechanism of action of Erdafitinib?',\n",
              "  'Tumor cells expressing these FGFR activating mutants displayed sensitivity to the selective FGFR inhibitor erdafitinib and resulted in suppression of FGFR phosphorylation and downstream signal transduction.',\n",
              "  0),\n",
              " (3462,\n",
              "  11,\n",
              "  'What is the mechanism of action of Erdafitinib?',\n",
              "  'Erdafitinib, a tyrosine kinase inhibitor of FGFR1-4, has shown antitumor activity in preclinical models and in a phase 1 study involving patients with  __i_tag__ FGFR __end_i_tag__  alterations.',\n",
              "  0),\n",
              " (3462,\n",
              "  12,\n",
              "  'What is the mechanism of action of Erdafitinib?',\n",
              "  'Erdafitinib (Balversa™, Janssen Pharmaceutical Companies) is a pan-fibroblast growth factor receptor (FGFR) inhibitor that was recently approved in the USA for the treatment of locally advanced or metastatic FGFR3 or FGFR2 urothelial carcinoma.',\n",
              "  0),\n",
              " (3462,\n",
              "  13,\n",
              "  'What is the mechanism of action of Erdafitinib?',\n",
              "  'Multicenter Phase I Study of Erdafitinib (JNJ-42756493), Oral Pan-Fibroblast Growth Factor Receptor Inhibitor, in Patients with Advanced or Refractory Solid Tumors.',\n",
              "  0),\n",
              " (3462,\n",
              "  14,\n",
              "  'What is the mechanism of action of Erdafitinib?',\n",
              "  'JNJ-42756493 , erdafitinib , is an orally active small molecule with potent tyrosine kinase inhibitory activity against all four FGFR family members and selectivity versus other highly related kinases',\n",
              "  0),\n",
              " (3462,\n",
              "  15,\n",
              "  'What is the mechanism of action of Erdafitinib?',\n",
              "  'This review covers the preclinical and clinical evidence for erdafitinib , summarizes the results of other FGFR inhibitors tested in UC and explores future perspectives of FGFR inhibition in UC',\n",
              "  0),\n",
              " (3462,\n",
              "  16,\n",
              "  'What is the mechanism of action of Erdafitinib?',\n",
              "  'Erdafitinib was the first orally effective FGFR antagonist approved by the FDA (2019) for the treatment of advanced cancer, that of the urinary bladder.',\n",
              "  0),\n",
              " (3462,\n",
              "  17,\n",
              "  'What is the mechanism of action of Erdafitinib?',\n",
              "  'Erdafitinib, a potent oral fibroblast growth factor receptor inhibitor, is a low extraction ratio drug highly bound to alpha-1-acid glycoprotein (AGP) with free fraction (fu ) varying across populations.',\n",
              "  0),\n",
              " (3462,\n",
              "  18,\n",
              "  'What is the mechanism of action of Erdafitinib?',\n",
              "  'Introduction This phase 1, open-label, multicenter, single-arm, dose-escalation study aimed to evaluate safety, pharmacokinetics (PK), and pharmacodynamics of erdafitinib (JNJ-42756493), an oral selective pan-fibroblast growth factor receptor (FGFR) tyrosine kinase inhibitor, and to determine the recommended phase 2 dose in Japanese patients with advanced or refractory solid tumors.',\n",
              "  1),\n",
              " (3462,\n",
              "  19,\n",
              "  'What is the mechanism of action of Erdafitinib?',\n",
              "  'BACKGROUND AND OBJECTIVES\\nErdafitinib, an oral selective pan-fibroblast growth factor receptor (FGFR) kinase inhibitor, is primarily metabolized by cytochrome P450 (CYP) 2C9 and 3A4.',\n",
              "  0),\n",
              " (3462,\n",
              "  20,\n",
              "  'What is the mechanism of action of Erdafitinib?',\n",
              "  'Erdafitinib, a potent oral fibroblast growth factor receptor inhibitor, is a low extraction ratio drug highly bound to alpha-1-acid glycoprotein (AGP) with free fraction (fu ) varying across populations.',\n",
              "  0),\n",
              " (3462,\n",
              "  21,\n",
              "  'What is the mechanism of action of Erdafitinib?',\n",
              "  'Introduction This phase 1, open-label, multicenter, single-arm, dose-escalation study aimed to evaluate safety, pharmacokinetics (PK), and pharmacodynamics of erdafitinib (JNJ-42756493), an oral selective pan-fibroblast growth factor receptor (FGFR) tyrosine kinase inhibitor, and to determine the recommended phase 2 dose in Japanese patients with advanced or refractory solid tumors.',\n",
              "  1),\n",
              " (3462,\n",
              "  22,\n",
              "  'What is the mechanism of action of Erdafitinib?',\n",
              "  'Erdafitinib, an oral selective pan-fibroblast growth factor receptor (FGFR) kinase inhibitor, is primarily metabolized by cytochrome P450 (CYP) 2C9 and 3A4.',\n",
              "  1),\n",
              " (1795,\n",
              "  0,\n",
              "  'What is regioneR?',\n",
              "  'regioneR: an R/Bioconductor package for the association analysis of genomic regions based on permutation tests',\n",
              "  0),\n",
              " (1795,\n",
              "  1,\n",
              "  'What is regioneR?',\n",
              "  'regioneR is an R package that implements a permutation test framework specifically designed to work with genomic regions.',\n",
              "  1),\n",
              " (1795,\n",
              "  2,\n",
              "  'What is regioneR?',\n",
              "  'In addition to the predefined randomization and evaluation strategies, regioneR is fully customizable allowing the use of custom strategies to adapt it to specific questions.',\n",
              "  1),\n",
              " (1795,\n",
              "  3,\n",
              "  'What is regioneR?',\n",
              "  'Finally, it also implements a novel function to evaluate the local specificity of the detected association.',\n",
              "  0),\n",
              " (1795,\n",
              "  4,\n",
              "  'What is regioneR?',\n",
              "  'Randomization based approaches implicitly take into account the complexity of the genome without the need of assuming an underlying statistical model.regioneR is an R package that implements a permutation test framework specifically designed to work with genomic regions',\n",
              "  1),\n",
              " (1795,\n",
              "  5,\n",
              "  'What is regioneR?',\n",
              "  'In addition to the predefined randomization and evaluation strategies, regioneR is fully customizable allowing the use of custom strategies to adapt it to specific questions',\n",
              "  1),\n",
              " (1795,\n",
              "  6,\n",
              "  'What is regioneR?',\n",
              "  'Finally, it also implements a novel function to evaluate the local specificity of the detected association.regioneR is an R package released under Artistic-2.0 License',\n",
              "  1),\n",
              " (1795,\n",
              "  7,\n",
              "  'What is regioneR?',\n",
              "  'The source code and documents are freely available through Bioconductor (http://www.bioconductor.org/packages/regioneR).rmalinverni@carrerasresearch.org.<CopyrightInformation>© The Author 2015',\n",
              "  0),\n",
              " (1673,\n",
              "  0,\n",
              "  'Which microRNAs are involved in exercise adaptation?',\n",
              "  'Some miRNAs as miR-1, miR-133 and miR-208a are highly expressed in the heart and strongly associated with the development of cardiac hypertrophy.',\n",
              "  1),\n",
              " (1673,\n",
              "  1,\n",
              "  'Which microRNAs are involved in exercise adaptation?',\n",
              "  'Recent data indicate that these miRNAs as well as miR-206 change their expression quickly in response to physical activity.',\n",
              "  1),\n",
              " (1673,\n",
              "  2,\n",
              "  'Which microRNAs are involved in exercise adaptation?',\n",
              "  'the miR-494 content significantly decreased after endurance exercise in C57BL/6J mice, accompanied by an increase in expression of mtTFA and Foxj3 proteins.',\n",
              "  1),\n",
              " (1673,\n",
              "  3,\n",
              "  'Which microRNAs are involved in exercise adaptation?',\n",
              "  'These results suggest that miR-494 regulates mitochondrial biogenesis by downregulating mtTFA and Foxj3 during myocyte differentiation and skeletal muscle adaptation to physical exercise.',\n",
              "  0),\n",
              " (1673,\n",
              "  4,\n",
              "  'Which microRNAs are involved in exercise adaptation?',\n",
              "  '1) c-miRNA up-regulated by acute exercise before and after sustained training (miR-146a and miR-222), (2) c-miRNA responsive to acute exercise before but not after sustained training (miR-21 and miR-221), (3) c-miRNA responsive only to sustained training (miR-20a),',\n",
              "  1),\n",
              " (1673,\n",
              "  5,\n",
              "  'Which microRNAs are involved in exercise adaptation?',\n",
              "  'MicroRNA 1 expression was decreased independent of the training modality, and was paralleled by an increased expression of IGF-1 representing a potential target.',\n",
              "  0),\n",
              " (1673,\n",
              "  6,\n",
              "  'Which microRNAs are involved in exercise adaptation?',\n",
              "  'we investigated the expression of these myomiRs, including miR-1, miR-133a, miR-133b and miR-206 in muscle biopsies from vastus lateralis of healthy young males (n = 10) in relation to a hyperinsulinaemic–euglycaemic clamp as well as acute endurance exercise before and after 12 weeks of endurance training.',\n",
              "  1),\n",
              " (1673,\n",
              "  7,\n",
              "  'Which microRNAs are involved in exercise adaptation?',\n",
              "  'In resting biopsies, endurance training for 12 weeks decreased basal expression of all four myomiRs (P < 0.05)',\n",
              "  0),\n",
              " (1673,\n",
              "  8,\n",
              "  'Which microRNAs are involved in exercise adaptation?',\n",
              "  'These results suggest that miR-494 regulates mitochondrial biogenesis by downregulating mtTFA and Foxj3 during myocyte differentiation and skeletal muscle adaptation to physical exercise.',\n",
              "  0),\n",
              " (164,\n",
              "  0,\n",
              "  'What is the enzymatic activity of the breast cancer associated gene BRCA1?',\n",
              "  'E3-ubiquitin ligase activity is the only known enzymatic activity of BRCA1, which is mediated by the N-terminal RING finger domain.',\n",
              "  1),\n",
              " (164,\n",
              "  1,\n",
              "  'What is the enzymatic activity of the breast cancer associated gene BRCA1?',\n",
              "  'BRCA1 nuclear transport and ubiquitin E3 ligase enzymatic activity are tightly regulated by the BRCA1 dimeric binding partner BARD1 and further modulated by cancer mutations and diverse signaling pathways.',\n",
              "  1),\n",
              " (164,\n",
              "  2,\n",
              "  'What is the enzymatic activity of the breast cancer associated gene BRCA1?',\n",
              "  'E3 ligase activity of BRCA1 is not essential for mammalian cell viability or homology-directed repair of double-strand DNA breaks.',\n",
              "  0),\n",
              " (164,\n",
              "  3,\n",
              "  'What is the enzymatic activity of the breast cancer associated gene BRCA1?',\n",
              "  'The protein encoded by BRCA1 interacts in vivo with the related BARD1 protein to form a heterodimeric complex that acts as a ubiquitin E3 ligase.',\n",
              "  1),\n",
              " (164,\n",
              "  4,\n",
              "  'What is the enzymatic activity of the breast cancer associated gene BRCA1?',\n",
              "  'Because the enzymatic activity of the BRCA1/BARD1 heterodimer is conserved over a broad phylogenetic range, it is thought to be critical for the central functions of BRCA1.',\n",
              "  1),\n",
              " (164,\n",
              "  5,\n",
              "  'What is the enzymatic activity of the breast cancer associated gene BRCA1?',\n",
              "  'A mechanism for transcriptional repression dependent on the BRCA1 E3 ubiquitin ligase.',\n",
              "  0),\n",
              " (164,\n",
              "  6,\n",
              "  'What is the enzymatic activity of the breast cancer associated gene BRCA1?',\n",
              "  'In this study, we identified an ubiquitin-dependent mechanism by which BRCA1 inhibits transcription.',\n",
              "  0),\n",
              " (164,\n",
              "  7,\n",
              "  'What is the enzymatic activity of the breast cancer associated gene BRCA1?',\n",
              "  'BRCA1 ubiquitinates the transcriptional preinitiation complex, preventing stable association of TFIIE and TFIIH, and thus blocks the initiation of mRNA synthesis.',\n",
              "  0),\n",
              " (164,\n",
              "  8,\n",
              "  'What is the enzymatic activity of the breast cancer associated gene BRCA1?',\n",
              "  'In the structure the arrangement of the Ring-domains is similar to another H2A E3 ligase, the BRCA1/BARD1 complex, but complex formation depends on an N-terminal arm of Ring1b that embraces the Bmi1 Ring-domain.',\n",
              "  0),\n",
              " (164,\n",
              "  9,\n",
              "  'What is the enzymatic activity of the breast cancer associated gene BRCA1?',\n",
              "  'Discovering the precise function of the breast and ovarian specific tumor suppressor, BRCA1, has proven to be quite complicated.',\n",
              "  1),\n",
              " (164,\n",
              "  10,\n",
              "  'What is the enzymatic activity of the breast cancer associated gene BRCA1?',\n",
              "  'It has been determined that BRCA1, together with BARD1, comprise an E3 ubiquitin ligase.',\n",
              "  0),\n",
              " (164,\n",
              "  11,\n",
              "  'What is the enzymatic activity of the breast cancer associated gene BRCA1?',\n",
              "  'Since it is now known that BRCA1 is an enzyme, the challenge for BRCA1 research is to learn how this enzymatic activity functions in normal breast and ovarian cells in order to suppress cancerous transformation.',\n",
              "  0),\n",
              " (164,\n",
              "  12,\n",
              "  'What is the enzymatic activity of the breast cancer associated gene BRCA1?',\n",
              "  'The BRCA1 tumor suppressor gene is expressed in all mammalian cells.',\n",
              "  0),\n",
              " (164,\n",
              "  13,\n",
              "  'What is the enzymatic activity of the breast cancer associated gene BRCA1?',\n",
              "  'Within these cells, the BRCA1 protein product interacts with several seemingly distinct nuclear complexes.',\n",
              "  0),\n",
              " (164,\n",
              "  14,\n",
              "  'What is the enzymatic activity of the breast cancer associated gene BRCA1?',\n",
              "  'Proteins within these complexes are potential targets for the E3-ubiquitin ligase activity associated with BRCA1:BARD1 complexes.',\n",
              "  0),\n",
              " (164,\n",
              "  15,\n",
              "  'What is the enzymatic activity of the breast cancer associated gene BRCA1?',\n",
              "  'During both DNA replication and DNA repair, BRCA1 appears to serve both adaptor and enzymatic functions.',\n",
              "  0),\n",
              " (164,\n",
              "  16,\n",
              "  'What is the enzymatic activity of the breast cancer associated gene BRCA1?',\n",
              "  'Roles include transient physical recruitment of NBS1, gammaH2AX, FANCD2 and other proteins in specific repair associated complexes, and enzymatic activity as an E3-ubiquitin ligase against a subset of these proteins.',\n",
              "  0),\n",
              " (164,\n",
              "  17,\n",
              "  'What is the enzymatic activity of the breast cancer associated gene BRCA1?',\n",
              "  'The BRCA1 protein displays E3 ubiquitin ligase activity, and this enzymatic function is thought to be required for tumor suppression.',\n",
              "  0),\n",
              " (1397,\n",
              "  0,\n",
              "  'Which drugs are included in TAS-102?',\n",
              "  'PURPOSE: Trifluridine (TFT) is an antitumor component of a novel nucleoside antitumor agent, TAS-102, which consists of TFT and tipiracil hydrochloride (thymidine phosphorylase inhibitor).',\n",
              "  1),\n",
              " (1397,\n",
              "  1,\n",
              "  'Which drugs are included in TAS-102?',\n",
              "  'TAS-102, a combination of trifluorothymidine and the thymidine phosphorylase inhibitor TPI in a 1:0.5 ratio, is a novel oral formulation, which is active in 5FU-resistant models, both in vitro and in xenograft models.',\n",
              "  1),\n",
              " (1397,\n",
              "  2,\n",
              "  'Which drugs are included in TAS-102?',\n",
              "  'TAS-102 is a novel oral nucleoside antitumor agent consisting of trifluridine and tipiracil hydrochloride at a molar ratio of 1:0.5.',\n",
              "  1),\n",
              " (1397,\n",
              "  3,\n",
              "  'Which drugs are included in TAS-102?',\n",
              "  'PURPOSE: TAS-102 is an orally administered anticancer agent composed of α,α,α-trifluorothymidine (FTD) and thymidine phosphorylase inhibitor (TPI).',\n",
              "  1),\n",
              " (1397,\n",
              "  4,\n",
              "  'Which drugs are included in TAS-102?',\n",
              "  'Novel DNA synthesis inhibitors for the treatment of gastrointestinal malignancies include a combination of the antimetabolite TAS-102, which consists of trifluorothymidine with a thymidine phosphorylase inhibitor, and a novel micellar formulation of cisplatin NC-6004 that uses a nanotechnology-based drug delivery system.',\n",
              "  0),\n",
              " (1397,\n",
              "  5,\n",
              "  'Which drugs are included in TAS-102?',\n",
              "  'Repeated oral dosing of TAS-102 confers high trifluridine incorporation into DNA and sustained antitumor activity in mouse models.',\n",
              "  0),\n",
              " (1397,\n",
              "  6,\n",
              "  'Which drugs are included in TAS-102?',\n",
              "  'TAS-102 is a novel oral nucleoside antitumor agent containing trifluridine (FTD) and tipiracil hydrochloride (TPI).',\n",
              "  1),\n",
              " (3083,\n",
              "  0,\n",
              "  'Describe CapSim',\n",
              "  'Targeted sequencing using capture probes has become increasingly popular in clinical applications due to its scalability and cost-effectiveness.',\n",
              "  0),\n",
              " (3083,\n",
              "  1,\n",
              "  'Describe CapSim',\n",
              "  'The approach also allows for higher sequencing coverage of the targeted regions resulting in better analysis statistical power.',\n",
              "  0),\n",
              " (3083,\n",
              "  2,\n",
              "  'Describe CapSim',\n",
              "  'However, because of the dynamics of the hybridization process, it is difficult to evaluate the efficiency of the probe design prior to the experiments which are time consuming and costly.Results: We developed CapSim, a software package for simulation of targeted sequencing.',\n",
              "  0),\n",
              " (3083,\n",
              "  3,\n",
              "  'Describe CapSim',\n",
              "  'Given a genome sequence and a set of probes, CapSim simulates the fragmentation, the dynamics of probe hybridization and the sequencing of the captured fragments on Illumina and PacBio sequencing platforms.',\n",
              "  1),\n",
              " (3083,\n",
              "  4,\n",
              "  'Describe CapSim',\n",
              "  'The simulated data can be used for evaluating the performance of the analysis pipeline, as well as the efficiency of the probe design.',\n",
              "  0),\n",
              " (3083,\n",
              "  5,\n",
              "  'Describe CapSim',\n",
              "  'Parameters of the various stages in the sequencing process can also be evaluated in order to optimize the experiments.Availability and implementation: CapSim is publicly available under BSD license at https://github.com/Devika1/capsim.',\n",
              "  1),\n",
              " (3083,\n",
              "  6,\n",
              "  'Describe CapSim',\n",
              "  'Simulating the dynamics of targeted capture sequencing with CapSim.',\n",
              "  0),\n",
              " (3083,\n",
              "  7,\n",
              "  'Describe CapSim',\n",
              "  'Given a genome sequence and a set of probes, CapSim simulates the fragmentation, the dynamics of probe hybridization and the sequencing of the captured fragments on Illumina and PacBio sequencing platforms.',\n",
              "  1),\n",
              " (3083,\n",
              "  8,\n",
              "  'Describe CapSim',\n",
              "  'Results\\nWe developed CapSim, a software package for simulation of targeted sequencing.',\n",
              "  0),\n",
              " (3083,\n",
              "  9,\n",
              "  'Describe CapSim',\n",
              "  'Given a genome sequence and a set of probes, CapSim simulates the fragmentation, the dynamics of probe hybridization and the sequencing of the captured fragments on Illumina and PacBio sequencing platforms.',\n",
              "  1),\n",
              " (3083,\n",
              "  10,\n",
              "  'Describe CapSim',\n",
              "  'Parameters of the various stages in the sequencing process can also be evaluated in order to optimize the experiments.<br><b>Availability and implementation</b>: CapSim is publicly available under BSD license at https://github.com/Devika1/capsim.<br><b>Contact</b>: l.coin@imb.uq.edu.au.<br><b>Supplementary information</b>: Supplementary data are available at Bioinformatics online.<br>',\n",
              "  0),\n",
              " (3083,\n",
              "  11,\n",
              "  'Describe CapSim',\n",
              "  'However, because of the dynamics of the hybridization process, it is difficult to evaluate the efficiency of the probe design prior to the experiments which are time consuming and costly.<br><b>Results</b>: We developed CapSim, a software package for simulation of targeted sequencing.',\n",
              "  0),\n",
              " (3083,\n",
              "  12,\n",
              "  'Describe CapSim',\n",
              "  'We developed CapSim, a software package for simulation of targeted sequencing.',\n",
              "  0),\n",
              " (3083,\n",
              "  13,\n",
              "  'Describe CapSim',\n",
              "  'Given a genome sequence and a set of probes, CapSim simulates the fragmentation, the dynamics of probe hybridization and the sequencing of the captured fragments on Illumina and PacBio sequencing platforms.',\n",
              "  1),\n",
              " (2831,\n",
              "  0,\n",
              "  'Which cells produce Interleukin 17A?',\n",
              "  'Several studies have shown an increased expression/release of Th17 related cytokine, IL-17A in ASD.',\n",
              "  1),\n",
              " (2831,\n",
              "  1,\n",
              "  'Which cells produce Interleukin 17A?',\n",
              "  ' interleukin (IL)-17A, secreted from T helper type 17 (TH17) cells,',\n",
              "  1),\n",
              " (2831,\n",
              "  2,\n",
              "  'Which cells produce Interleukin 17A?',\n",
              "  ' IL-17A, markedly produced by TH17 cells',\n",
              "  1),\n",
              " (2831,\n",
              "  3,\n",
              "  'Which cells produce Interleukin 17A?',\n",
              "  'IL-22 and IL-17A derive from both independent and overlapping lineages, defined as T helper (Th)22 and IL-22+ Th17 cells.',\n",
              "  1),\n",
              " (3517,\n",
              "  0,\n",
              "  'Which are the main G1/S transcription factors in yeast?',\n",
              "  'To understand how commitment to cell division in late G1 phase (Start) is controlled by growth and nutrients in budding yeast, we determined the absolute concentrations of the G1/S transcription factors SBF (composed of Swi4 and Swi6) and MBF (composed of Mbp1 and Swi6), the transcriptional repressor Whi5, and the G1 cyclins, Cln1 and Cln2, in single live yeast cells using scanning number and brightness (sN&B) microscopy.',\n",
              "  1),\n",
              " (3517,\n",
              "  1,\n",
              "  'Which are the main G1/S transcription factors in yeast?',\n",
              "  'We previously isolated the SKN7 gene in a screen designed to isolate new components of the G1-S cell cycle transcription machinery in budding yeast.',\n",
              "  1),\n",
              " (3517,\n",
              "  2,\n",
              "  'Which are the main G1/S transcription factors in yeast?',\n",
              "  'We have now found that Skn7 associates with Mbp1, the DNA-binding component of the G1-S transcription factor DSC1/MBF',\n",
              "  0),\n",
              " (3517,\n",
              "  3,\n",
              "  'Which are the main G1/S transcription factors in yeast?',\n",
              "  'To gain more insight into the G(1)/S circuitry, we characterized Swi6p, Swi4p and Mbp1p, the closest orthologues of SBF (Swi6p and Swi4p) and MBF (Swi6p and Mbp1p) components in S. cerevisiae',\n",
              "  0),\n",
              " (3517,\n",
              "  4,\n",
              "  'Which are the main G1/S transcription factors in yeast?',\n",
              "  'The G(1)/S transition is a critical control point for cell proliferation and involves essential transcription complexes termed SBF and MBF in Saccharomyces cerevisiae or MBF in Schizosaccharomyces pombe',\n",
              "  1),\n",
              " (3517,\n",
              "  5,\n",
              "  'Which are the main G1/S transcription factors in yeast?',\n",
              "  'In Saccharomyces cerevisiae, G1/S transcription factors MBF and SBF regulate a large family of genes important for entry to the cell cycle and DNA replication and repair.',\n",
              "  1),\n",
              " (3517,\n",
              "  6,\n",
              "  'Which are the main G1/S transcription factors in yeast?',\n",
              "  ' MBF and SBF consist of a common component, Swi6, and a DNA-specific binding protein, Mbp1 and Swi4, respectively.',\n",
              "  0),\n",
              " (3517,\n",
              "  7,\n",
              "  'Which are the main G1/S transcription factors in yeast?',\n",
              "  'Over 200 G1/S genes are regulated by either one of the two TF complexes, SBF and MBF, which bind to specific DNA binding sequences, SCB and MCB, respectively.',\n",
              "  1),\n",
              " (4202,\n",
              "  0,\n",
              "  'Which protein is encoded by the protein APOBEC3C?',\n",
              "  'apolipoprotein B mRNA-editing enzyme catalytic polypeptide-like 3 (APOBEC3; shortened here to A3), act by deaminating cytidines to uridines during the reverse transcription reaction of HIV-1.',\n",
              "  1),\n",
              " (4202,\n",
              "  1,\n",
              "  'Which protein is encoded by the protein APOBEC3C?',\n",
              "  'apolipoprotein B mRNA editing enzyme catalytic polypeptide-like 3C (APOBEC3C)',\n",
              "  1),\n",
              " (4202,\n",
              "  2,\n",
              "  'Which protein is encoded by the protein APOBEC3C?',\n",
              "  'apolipoprotein B mRNA editing enzyme catalytic subunit (APOBEC)3A,',\n",
              "  1),\n",
              " (4202,\n",
              "  3,\n",
              "  'Which protein is encoded by the protein APOBEC3C?',\n",
              "  'apolipoprotein B mRNA-editing enzyme catalytic subunit 3C (APOBEC3C, a nucleic acid-editing deaminase)',\n",
              "  1),\n",
              " (4202,\n",
              "  4,\n",
              "  'Which protein is encoded by the protein APOBEC3C?',\n",
              "  ' APOBEC3C (apolipoprotein-B mRNA-editing catalytic polypeptide-like 3C (A3C))',\n",
              "  1),\n",
              " (1376,\n",
              "  0,\n",
              "  'Which are the enzymes involved in the control of tubulin acetylation?',\n",
              "  'Inhibition of HDAC6 deacetylase activity increases its binding with microtubules and suppresses microtubule dynamic instability in MCF-7 cells',\n",
              "  0),\n",
              " (1376,\n",
              "  1,\n",
              "  'Which are the enzymes involved in the control of tubulin acetylation?',\n",
              "  'HDAC6, the tubulin deacetylase, plays a key role in maintaining typical distribution of acetylated microtubules in cells.',\n",
              "  0),\n",
              " (1376,\n",
              "  2,\n",
              "  'Which are the enzymes involved in the control of tubulin acetylation?',\n",
              "  'We found that whereas both pharmacological inhibition of HDAC6 as well as its depletion enhance microtubule acetylation, only pharmacological inhibition of HDAC6 activity leads to an increase in microtubule stability against cold and nocodazole-induced depolymerizing conditions.',\n",
              "  0),\n",
              " (1376,\n",
              "  3,\n",
              "  'Which are the enzymes involved in the control of tubulin acetylation?',\n",
              "  'The evidence presented in this study indicated that the increased binding of HDAC6, rather than the acetylation per se, causes microtubule stability.',\n",
              "  0),\n",
              " (1376,\n",
              "  4,\n",
              "  'Which are the enzymes involved in the control of tubulin acetylation?',\n",
              "  'The results are in support of a hypothesis that in addition to its deacetylase function, HDAC6 might function as a MAP that regulates microtubule dynamics under certain conditions.',\n",
              "  0),\n",
              " (1376,\n",
              "  5,\n",
              "  'Which are the enzymes involved in the control of tubulin acetylation?',\n",
              "  'Moreover, acetylation of α-tubulin is under the control of the acetyltransferase MEC-17 and deacetylases SIRT2 (Sirtuin 2) and HDAC6 (histone deacetylase 6).',\n",
              "  1),\n",
              " (1376,\n",
              "  6,\n",
              "  'Which are the enzymes involved in the control of tubulin acetylation?',\n",
              "  'MEC-17 deficiency leads to reduced α-tubulin acetylation and impaired migration of cortical neurons',\n",
              "  1),\n",
              " (1376,\n",
              "  7,\n",
              "  'Which are the enzymes involved in the control of tubulin acetylation?',\n",
              "  'MEC-17 is a newly discovered α-tubulin acetyltransferase that has been found to play a major role in the acetylation of α-tubulin in different species in vivo.',\n",
              "  0),\n",
              " (1376,\n",
              "  8,\n",
              "  'Which are the enzymes involved in the control of tubulin acetylation?',\n",
              "  'Thus, MEC-17, which regulates the acetylation of α-tubulin, appears to control the migration and morphological transition of cortical neurons.',\n",
              "  1),\n",
              " (1376,\n",
              "  9,\n",
              "  'Which are the enzymes involved in the control of tubulin acetylation?',\n",
              "  'This finding reveals the importance of MEC-17 and α-tubulin acetylation in cortical development.',\n",
              "  1),\n",
              " (1376,\n",
              "  10,\n",
              "  'Which are the enzymes involved in the control of tubulin acetylation?',\n",
              "  'NT secretion is prevented by overexpression of HDAC6, an α-tubulin deacetylase.',\n",
              "  0),\n",
              " (1376,\n",
              "  11,\n",
              "  'Which are the enzymes involved in the control of tubulin acetylation?',\n",
              "  'Moreover, α-tubulin acetylation levels of microtubules specifically extending into the terminal synaptic boutons are reduced in response to dTip60 HAT reduction.',\n",
              "  0),\n",
              " (1376,\n",
              "  12,\n",
              "  'Which are the enzymes involved in the control of tubulin acetylation?',\n",
              "  'In this study, angiotensin II induced disassembly and deacetylation of α-tubulin, which were blocked by pretreatment with an angiotensin II type 1 receptor blocker losartan and a sirtuin class deacetylase inhibitor sirtinol, and by depletion of a deacetylase SIRT2 using RNA interference.',\n",
              "  0),\n",
              " (1376,\n",
              "  13,\n",
              "  'Which are the enzymes involved in the control of tubulin acetylation?',\n",
              "  'These data show that angiotensin II and mechanical stretch stimulate microtubule redistribution and deacetylation via SIRT2 in endothelial cells, suggesting the emerging role of SIRT2 in hypertension-induced vascular remodeling.',\n",
              "  0),\n",
              " (1376,\n",
              "  14,\n",
              "  'Which are the enzymes involved in the control of tubulin acetylation?',\n",
              "  'Tubulin is a major substrate of the cytoplasmic class II histone deacetylase HDAC6.',\n",
              "  0),\n",
              " (1376,\n",
              "  15,\n",
              "  'Which are the enzymes involved in the control of tubulin acetylation?',\n",
              "  'Inhibition of HDAC6 results in higher levels of acetylated tubulin and enhanced binding of the motor protein kinesin-1 to tubulin, which promotes transport of cargoes along microtubules.',\n",
              "  0),\n",
              " (1376,\n",
              "  16,\n",
              "  'Which are the enzymes involved in the control of tubulin acetylation?',\n",
              "  'Alcohol-induced alterations in hepatic microtubule dynamics can be explained by impaired histone deacetylase 6 function',\n",
              "  0),\n",
              " (1376,\n",
              "  17,\n",
              "  'Which are the enzymes involved in the control of tubulin acetylation?',\n",
              "  'Addition of trichostatin A (TSA), an HDAC6 inhibitor, induced microtubule acetylation to the same extent as in ethanol-treated cells (approximately threefold).',\n",
              "  0),\n",
              " (1376,\n",
              "  18,\n",
              "  'Which are the enzymes involved in the control of tubulin acetylation?',\n",
              "  'Interestingly, HDAC6 from ethanol-treated cells was able to bind and deacetylate exogenous tubulin to the same extent as control, suggesting that ethanol-induced tubulin modifications prevented HDAC6 binding to endogenous microtubules.',\n",
              "  0),\n",
              " (1376,\n",
              "  19,\n",
              "  'Which are the enzymes involved in the control of tubulin acetylation?',\n",
              "  'We propose that lower HDAC6 levels combined with decreased microtubule binding lead to increased tubulin acetylation in ethanol-treated cells.',\n",
              "  0),\n",
              " (1376,\n",
              "  20,\n",
              "  'Which are the enzymes involved in the control of tubulin acetylation?',\n",
              "  'The results also showed that both pan-HDAC and class-I-specific inhibitor treatment resulted in increased acetylation of histones, but only pan-HDAC inhibitor treatment resulted in increased tubulin acetylation, which is in agreement with their activity towards the HDAC6 isoform.',\n",
              "  0),\n",
              " (1376,\n",
              "  21,\n",
              "  'Which are the enzymes involved in the control of tubulin acetylation?',\n",
              "  'Nicotinamide but not 3-aminobenzamide, an inhibitor for poly(ADP)ribose polymerase, enhanced tubulin acetylation and resistance to axonal degeneration in cultured cerebellar granule cells from wild-type (WT) mice, suggesting that mammalian Sir2-related protein (SIRT) 2, a nicotinamide adenine dinucleotide (NAD)--dependent tubulin deacetylase, could modulate resistance to axonal degeneration.',\n",
              "  0),\n",
              " (1376,\n",
              "  22,\n",
              "  'Which are the enzymes involved in the control of tubulin acetylation?',\n",
              "  'Moreover, SIRT2 overexpression abrogated microtubule hyperacetylation and resistance to axonal degeneration in these cells.',\n",
              "  0),\n",
              " (1376,\n",
              "  23,\n",
              "  'Which are the enzymes involved in the control of tubulin acetylation?',\n",
              "  'Conversely, SIRT2 knockdown by using a lentiviral vector expressing small interfering RNA, enhanced microtubule acetylation and resistance to axonal degeneration in WT granule cells.',\n",
              "  0),\n",
              " (1376,\n",
              "  24,\n",
              "  'Which are the enzymes involved in the control of tubulin acetylation?',\n",
              "  'Taken together, these results suggest that SIRT2-mediated tubulin deacetylation is involved in both microtubule hyperacetylation and resistance to axonal degeneration in Wld(S) granule cells.',\n",
              "  0),\n",
              " (1376,\n",
              "  25,\n",
              "  'Which are the enzymes involved in the control of tubulin acetylation?',\n",
              "  'Compared to control cultures, higher levels of acetylated tubulin were found in neurons treated with tubacin',\n",
              "  1),\n",
              " (648,\n",
              "  0,\n",
              "  'Which transcription factors (TFs) participate in the formation of the interferon-beta (IFN-b) enhanceosome?',\n",
              "  'The dimer formed by the ATF-2 and c-Jun transcription factors is one of the main components of the human interferon-beta enhanceosome.',\n",
              "  0),\n",
              " (648,\n",
              "  1,\n",
              "  'Which transcription factors (TFs) participate in the formation of the interferon-beta (IFN-b) enhanceosome?',\n",
              "  'The induction of IFN transcription resulted from the activation of the components of the IFN-beta enhanceosome, i.e.',\n",
              "  0),\n",
              " (648,\n",
              "  2,\n",
              "  'Which transcription factors (TFs) participate in the formation of the interferon-beta (IFN-b) enhanceosome?',\n",
              "  'IFN regulatory factor (IRF) 3, nuclear factor (NF)-kappaB, activating transcription factor (ATF)-2 and c-Jun.',\n",
              "  0),\n",
              " (648,\n",
              "  3,\n",
              "  'Which transcription factors (TFs) participate in the formation of the interferon-beta (IFN-b) enhanceosome?',\n",
              "  'Transcriptional activation of the interferon-beta (IFN-beta) gene requires assembly of an enhanceosome containing the transcription factors ATF-2/c-Jun, IRF-3/IRF-7, NF-kappaB and HMGI(Y).',\n",
              "  0),\n",
              " (648,\n",
              "  4,\n",
              "  'Which transcription factors (TFs) participate in the formation of the interferon-beta (IFN-b) enhanceosome?',\n",
              "  'These factors cooperatively bind a composite DNA site and activate expression of the IFN-beta gene.',\n",
              "  0),\n",
              " (648,\n",
              "  5,\n",
              "  'Which transcription factors (TFs) participate in the formation of the interferon-beta (IFN-b) enhanceosome?',\n",
              "  'Here we report that within the IFN-beta enhanceosome the ATF-2-c-jun heterodimer binds in a specific orientation, which is required for assembly of a complex between ATF-2-c-jun and interferon regulatory factor 3 (IRF-3).',\n",
              "  0),\n",
              " (648,\n",
              "  6,\n",
              "  'Which transcription factors (TFs) participate in the formation of the interferon-beta (IFN-b) enhanceosome?',\n",
              "  ' Here, we identified a small molecule that induces the assembly of the interferon-beta (IFN-beta) enhanceosome by stimulating all the enhancer-binding activator proteins: ATF2/c-JUN, IRF3, and p50/p65 of NF-kappaB.',\n",
              "  0),\n",
              " (648,\n",
              "  7,\n",
              "  'Which transcription factors (TFs) participate in the formation of the interferon-beta (IFN-b) enhanceosome?',\n",
              "  'Transcriptional activation of the virus inducible enhancer of the human interferon-beta (IFN-beta) gene in response to virus infection requires the assembly of an enhanceosome, consisting of the transcriptional activators NF-kappaB, ATF-2/c-Jun, IRFs and the architectural protein of the mammalian high mobility group I(Y) [HMG I(Y)].',\n",
              "  1),\n",
              " (648,\n",
              "  8,\n",
              "  'Which transcription factors (TFs) participate in the formation of the interferon-beta (IFN-b) enhanceosome?',\n",
              "  'Here, we demonstrate that the first step in enhanceosome assembly, i.e.',\n",
              "  0),\n",
              " (648,\n",
              "  9,\n",
              "  'Which transcription factors (TFs) participate in the formation of the interferon-beta (IFN-b) enhanceosome?',\n",
              "  'HMG I(Y)-dependent recruitment of NF-kappaB and ATF-2/c-Jun to the enhancer, is facilitated by discrete regions of HMG I and is mediated by allosteric changes induced in the DNA by HMG I(Y) and not by protein-protein interactions between HMG I(Y) and these proteins.',\n",
              "  0),\n",
              " (648,\n",
              "  10,\n",
              "  'Which transcription factors (TFs) participate in the formation of the interferon-beta (IFN-b) enhanceosome?',\n",
              "  'However, we show that completion of the enhanceosome assembly process requires protein-protein interactions between HMG I(Y) and the activators.',\n",
              "  0),\n",
              " (648,\n",
              "  11,\n",
              "  'Which transcription factors (TFs) participate in the formation of the interferon-beta (IFN-b) enhanceosome?',\n",
              "  'Transcriptional activation of the human interferon-beta (IFN-beta) gene by virus infection requires the assembly of a higher order nucleoprotein complex, the enhanceosome, which consists of the transcriptional activators NF-kappa B (p50/p65), ATF-2/c-jun, IRF-3 and IRF-7, architectural protein HMGI(Y), and the coactivators p300 and CBP.',\n",
              "  1),\n",
              " (648,\n",
              "  12,\n",
              "  'Which transcription factors (TFs) participate in the formation of the interferon-beta (IFN-b) enhanceosome?',\n",
              "  'The transcriptional coactivators CBP and P/CAF are required for activation of transcription from the IFN beta enhanceosome.',\n",
              "  0),\n",
              " (648,\n",
              "  13,\n",
              "  'Which transcription factors (TFs) participate in the formation of the interferon-beta (IFN-b) enhanceosome?',\n",
              "  'We show that CBP and P/CAF acetylate HMG I(Y), the essential architectural component required for enhanceosome assembly, at distinct lysine residues, causing distinct effects on transcription.',\n",
              "  0),\n",
              " (648,\n",
              "  14,\n",
              "  'Which transcription factors (TFs) participate in the formation of the interferon-beta (IFN-b) enhanceosome?',\n",
              "  'The transcriptional activity of an in vitro assembled human interferon-beta gene enhanceosome is highly synergistic.',\n",
              "  0),\n",
              " (648,\n",
              "  15,\n",
              "  'Which transcription factors (TFs) participate in the formation of the interferon-beta (IFN-b) enhanceosome?',\n",
              "  'This synergy requires five distinct transcriptional activator proteins (ATF2/c-JUN, interferon regulatory factor 1, and p50/p65 of NF-kappaB), the high mobility group protein HMG I(Y), and the correct alignment of protein-binding sites on the face of the DNA double helix.',\n",
              "  0),\n",
              " (648,\n",
              "  16,\n",
              "  'Which transcription factors (TFs) participate in the formation of the interferon-beta (IFN-b) enhanceosome?',\n",
              "  'In addition, we provide evidence that recruitment of the holoenzyme by the enhanceosome is due, at least in part, to interactions between the enhanceosome and the transcriptional coactivator CREB, cAMP responsive element binding protein (CBP).',\n",
              "  0),\n",
              " (648,\n",
              "  17,\n",
              "  'Which transcription factors (TFs) participate in the formation of the interferon-beta (IFN-b) enhanceosome?',\n",
              "  'Transcriptional activation of the IFN beta gene in response to virus infection requires the assembly of an enhanceosome, consisting of the transcriptional activators NF-kappa B, IRF1, ATF2/c-Jun, and the architectural protein HMG I(Y).',\n",
              "  1),\n",
              " (648,\n",
              "  18,\n",
              "  'Which transcription factors (TFs) participate in the formation of the interferon-beta (IFN-b) enhanceosome?',\n",
              "  'Transcriptional synergy requires recruitment of the CBP/p300 coactivator to the enhanceosome, via a new activating surface assembled from the novel p65 domain and the activation domains of all of the activators.',\n",
              "  0),\n",
              " (648,\n",
              "  19,\n",
              "  'Which transcription factors (TFs) participate in the formation of the interferon-beta (IFN-b) enhanceosome?',\n",
              "  'A functional interferon-beta gene enhanceosome was assembled in vitro using the purified recombinant transcriptional activator proteins ATF2/c-JUN, IRF1, and p50/p65 of NF-kappa B. Maximal levels of transcriptional synergy between these activators required the specific interactions with the architectural protein HMG I(Y) and the correct helical phasing of the binding sites of these proteins on the DNA helix.',\n",
              "  0),\n",
              " (648,\n",
              "  20,\n",
              "  'Which transcription factors (TFs) participate in the formation of the interferon-beta (IFN-b) enhanceosome?',\n",
              "  'We present evidence that transcriptional activation of the human interferon-beta (IFN beta) gene requires the assembly of a higher order transcription enhancer complex (enhanceosome).',\n",
              "  0),\n",
              " (648,\n",
              "  21,\n",
              "  'Which transcription factors (TFs) participate in the formation of the interferon-beta (IFN-b) enhanceosome?',\n",
              "  'This multicomponent complex includes at least three distinct transcription factors and the high mobility group protein HMG I(Y).',\n",
              "  0),\n",
              " (648,\n",
              "  22,\n",
              "  'Which transcription factors (TFs) participate in the formation of the interferon-beta (IFN-b) enhanceosome?',\n",
              "  'Thus, HMG I(Y) plays an essential role in the assembly and function of the IFN beta gene enhanceosome.',\n",
              "  0),\n",
              " (648,\n",
              "  23,\n",
              "  'Which transcription factors (TFs) participate in the formation of the interferon-beta (IFN-b) enhanceosome?',\n",
              "  'Transcriptional activation of the interferon-beta (IFN-beta) gene requires assembly of an enhanceosome containing the transcription factors ATF-2/c-Jun, IRF-3/IRF-7, NF-kappaB and HMGI(Y).',\n",
              "  0),\n",
              " (648,\n",
              "  24,\n",
              "  'Which transcription factors (TFs) participate in the formation of the interferon-beta (IFN-b) enhanceosome?',\n",
              "  'A functional interferon-beta gene enhanceosome was assembled in vitro using the purified recombinant transcriptional activator proteins ATF2/c-JUN, IRF1, and p50/p65 of NF-kappa B',\n",
              "  0),\n",
              " (648,\n",
              "  25,\n",
              "  'Which transcription factors (TFs) participate in the formation of the interferon-beta (IFN-b) enhanceosome?',\n",
              "  'Transcriptional activation of the human interferon-beta (IFN-beta) gene by virus infection requires the assembly of a higher order nucleoprotein complex, the enhanceosome, which consists of the transcriptional activators NF-kappa B (p50/p65), ATF-2/c-jun, IRF-3 and IRF-7, architectural protein HMGI(Y), and the coactivators p300 and CBP',\n",
              "  1),\n",
              " (648,\n",
              "  26,\n",
              "  'Which transcription factors (TFs) participate in the formation of the interferon-beta (IFN-b) enhanceosome?',\n",
              "  'Transcriptional activation of the IFN beta gene in response to virus infection requires the assembly of an enhanceosome, consisting of the transcriptional activators NF-kappa B, IRF1, ATF2/c-Jun, and the architectural protein HMG I(Y)',\n",
              "  1),\n",
              " (380,\n",
              "  0,\n",
              "  'What memory problems are reported in the \" Gulf war syndrome\"?',\n",
              "  'The strongest associations were for mood swings (OR 20.9, 95%CI 16.2-27.0), memory loss/lack of concentration (OR 19.6, 95% CI 15.5-24.8), night sweats (OR 9.9, 95% CI 6.5-15.2), general fatigue (OR 9.6, 95% CI 8.3-11.1) and sexual dysfunction (OR 4.6, 95%CI 3.2-6.6).',\n",
              "  0),\n",
              " (380,\n",
              "  1,\n",
              "  'What memory problems are reported in the \" Gulf war syndrome\"?',\n",
              "  'The symptoms include incapacitating fatigue, musculoskeletel and joint pains, headaches, neuropsychiatric disorders, affect changes, confusion, visual problems, changes of gait, loss of memory, lymphadenopathies, respiratory impairment, impotence, and urinary tract morphological and functional alterations.',\n",
              "  1),\n",
              " (380,\n",
              "  2,\n",
              "  'What memory problems are reported in the \" Gulf war syndrome\"?',\n",
              "  'In early 1992, U.S. troops returning from the Gulf War began reporting a variety of nonspecific symptoms such as fatigue, skin rash, headache, muscle and joint pain, and loss of memory.',\n",
              "  1),\n",
              " (380,\n",
              "  3,\n",
              "  'What memory problems are reported in the \" Gulf war syndrome\"?',\n",
              "  'These reports marked the beginning of what was to be identified as the Gulf War Syndrome (GWS).',\n",
              "  1),\n",
              " (380,\n",
              "  4,\n",
              "  'What memory problems are reported in the \" Gulf war syndrome\"?',\n",
              "  'We report on a 29-year-old man who suffered from dysmnesia, disturbance of orientation, cognitive impairment, and double vision.',\n",
              "  0),\n",
              " (380,\n",
              "  5,\n",
              "  'What memory problems are reported in the \" Gulf war syndrome\"?',\n",
              "  'His history revealed several front-line operations in 1990 and 1991 during the Gulf War.',\n",
              "  1),\n",
              " (380,\n",
              "  6,\n",
              "  'What memory problems are reported in the \" Gulf war syndrome\"?',\n",
              "  'Neuropsychological tests disclosed severe cognitive impairment especially concerning memory.',\n",
              "  0),\n",
              " (380,\n",
              "  7,\n",
              "  'What memory problems are reported in the \" Gulf war syndrome\"?',\n",
              "  'A cluster of common health problems included: skin rash, cough, depression, unintentional weight loss, insomnia, and memory problems.',\n",
              "  1),\n",
              " (380,\n",
              "  8,\n",
              "  'What memory problems are reported in the \" Gulf war syndrome\"?',\n",
              "  'Syndromes 1 (\"impaired cognition,\" characterized by problems with attention, memory, and reasoning, as well as insomnia, depression, daytime sleepiness, and headaches), 2 (\"confusion-ataxia,\" characterized by problems with thinking, disorientation, balance disturbances, vertigo, and impotence), and 3 (\"arthro-myo-neuropathy,\" characterized by joint and muscle pains, muscle fatigue, difficulty lifting, and extremity paresthesias) represented strongly clustered symptoms; whereas, syndromes 4 (\"phobia-apraxia\"), 5 (\"fever-adenopathy\"), and 6 (\"weakness-incontinence\") involved weaker clustering and mostly overlapped syndromes 2 and 3.',\n",
              "  0),\n",
              " (3161,\n",
              "  0,\n",
              "  'List targets of classical analgesics.',\n",
              "  'Patient phenotypes in pharmacological pain treatment varies between individuals, which could be partly assigned to their genotypes regarding the targets of classical analgesics (OPRM1, PTGS2)',\n",
              "  1),\n",
              " (3161,\n",
              "  1,\n",
              "  'List targets of classical analgesics.',\n",
              "  'Morphine is the current gold standard analgesic acting at MOP receptors but produces a range of variably troublesome side-effects, in particular tolerance.',\n",
              "  1),\n",
              " (3560,\n",
              "  0,\n",
              "  'What is the cyberknife used for?',\n",
              "  'Stereotactic radiosurgery using CyberKnife® seems to be an efficient and safe therapeutic option for malignant melanomas affecting the choroid and ciliary body.',\n",
              "  1),\n",
              " (3560,\n",
              "  1,\n",
              "  'What is the cyberknife used for?',\n",
              "  'Stereotactic radiosurgery (SRS) has emerged as a principal alternative to microresection for small- and medium-sized CSHs.',\n",
              "  1),\n",
              " (3560,\n",
              "  2,\n",
              "  'What is the cyberknife used for?',\n",
              "  'CyberKnife® is a robotic stereotactic radiotherapy system.',\n",
              "  1),\n",
              " ...]"
            ]
          },
          "metadata": {},
          "execution_count": 14
        }
      ],
      "source": [
        "test_set = []\n",
        "for d in q_test:\n",
        "    document_data = dataset[dataset['question'] == d]\n",
        "    test_set += list(document_data.itertuples(index=False, name=None))\n",
        "test_set"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "xbUJWlD_0dCv"
      },
      "source": [
        "# Task 2 (5 marks): Cosine similarity\n",
        "\n",
        "Use the files `training.csv`, `dev_test.csv`, and `test.csv` provided by us in the file `data.zip` (so that any possible errors that you may have introduced in task 1 do not propagate to this task and following tasks).\n",
        "\n",
        "Implement a simple text summariser that is based on the cosine similarity between the question and the text. Use the following function signature.\n",
        "\n",
        "```{python}\n",
        "def cosine_summariser(csvfile, questionids, n=5):\n",
        "   \"\"\"Return the IDs of the n sentences that have the highest cosine similarity\n",
        "    with the question. The input questionids is a list of question ids. The \n",
        "    output is a list of lists of sentence ids\n",
        "    >>> cosine_summariser('test.csv', [3, 11], 3)\n",
        "    [[3, 1, 4], [12, 4, 13]]\"\"\"\n",
        "\n",
        "```\n",
        "\n",
        "To obtain the text vectors, use sklearn's tf.idf libraries this way:\n",
        "\n",
        "* Use all the defaults from the TfidfVectorizer instance, except for `stop_words=\"english\"` and `max_features=10000`. The latter option will restrict the vocabulary size to 10,000. This will speed up the computations and reduce the memory footprint in the subsequent tasks.\n",
        "* Use the `fit` method on the text of `training.csv`. In your documentation, please explain and justify what decision choices you made to select the correct text: would you use the question text only, the sentence text, or both?\n",
        "\n",
        "Evaluate the summariser by reporting the mean F1 score on each of the three CSV files `training.csv`, `devtest.csv`, and `test.csv`, for $n=5$. To calculate the mean F1 score, do this:\n",
        "\n",
        "1. For each question ID in the file, calculate the F1 score by comparing the result of your cosine summariser and the given labels. Feel free to use sklearn's functions to compute the F1 score, or implement your own version of the F1 scoring function if you prefer.\n",
        "2. Calculate the mean of the F1 scores calculated in step 1.\n",
        "\n",
        "Find the value of $n$ that returns the highest mean F1 score on the dev_test data.\n",
        "\n",
        "The breakdown of marks is as follows:\n",
        "\n",
        "* **1 mark** if the code generates the tf.idf vectors correctly. The explanations that justify the decisions made are reasonable. In particular, explain and justify what information you used to fit tf.idf.\n",
        "* **1 mark** if the code calculates cosine similarity correctly.\n",
        "* **1 mark** if the code returns the IDs of the $n$ sentences that have the highest cosine similarity with the question.\n",
        "* **1 mark** if the notebook reports the F1 scores of the dev_test file and identifies the value of $n$ that gives the highest score on the dev_test file.\n",
        "* **1 mark** for good coding and documentation in this task. In particular, comment on the reason why you think the value of $n$ that gives highest F1 has that value."
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 15,
      "metadata": {
        "id": "4X79S7QMOC5k"
      },
      "outputs": [],
      "source": [
        "from sklearn.feature_extraction.text import TfidfVectorizer\n",
        "tfidf = TfidfVectorizer(input='contents', max_features=10000, stop_words = \"english\")\n",
        "train_tfidf = tfidf.fit_transform([r[2] for r in training_set])"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "UAr6x5Zc0dCw"
      },
      "outputs": [],
      "source": [
        "def cosine_summariser(csvfile, questionids, n=5):\n",
        "   \"\"\"Return the IDs of the n sentences that have the highest cosine similarity\n",
        "    with the question. The input questionids is a list of question ids. The \n",
        "    output is a list of lists of sentence ids\n",
        "    >>> cosine_summariser('test.csv', [3, 11], 3)\n",
        "    [[3, 1, 4], [12, 4, 13]]\"\"\""
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "9VTTgRnN0dC4"
      },
      "source": [
        "# Task 3 (5 marks): Simple NN\n",
        "\n",
        "Use the files `training.csv`, `dev_test.csv`, and `test.csv` provided by us.\n",
        "\n",
        "Implement a simple TensorFlow-Keras neural model that has the following sequence of layers:\n",
        "\n",
        "1. An input layer that will accept the tf.idf of the sentence text (we will ignore the question text in this task). Use the TfidfVectorizer instance that you have fitted in task 2.\n",
        "2. A hidden layer and a relu activation function. You need to determine the size of the hidden layer.\n",
        "3. An output layer with one cell. The output layer will classify the input text (binary classification).\n",
        "\n",
        "Train the model with the training data and use the dev_test set to determine a good size of the hidden layer. \n",
        "\n",
        "With the model that you have trained, and implement a summariser that returns the $n$ sentences with highest predicted score. Use the following function signature:\n",
        "\n",
        "```{python}\n",
        "def nn_summariser(csvfile, questionids, n=5):\n",
        "   \"\"\"Return the IDs of the n sentences that have the highest predicted score. The input questionids is a list of question ids. The \n",
        "    output is a list of lists of sentence ids\n",
        "    >>> cosine_summariser('test.csv', [3, 11], 3)\n",
        "    [[2, 1, 3], [7, 14, 10]]\"\"\"\n",
        "\n",
        "```\n",
        "\n",
        "Report the final results using the test set. Remember: use the test set to report the final results of the best system only.\n",
        "\n",
        "Based on your experiments, comment on whether this system is better than the system developed in task 2. To make this task less time-consuming, focus only on $n=5$.\n",
        "\n",
        "The breakdown of marks is as follows:\n",
        "\n",
        "* **1 mark** if the NN model has the correct layers, the correct activation functions, and the correct loss function.\n",
        "* **1 mark** if the code passes the tf.idf information of the text to the model correctly.\n",
        "* **1 mark** if the code returns the IDs of the $n$ sentences that have the highest prediction score in the given question.\n",
        "* **1 mark** if the notebook reports the F1 scores of the test sets and comments on the results.\n",
        "* **1 mark** for good coding and documentation in this task. In particular, the code and results must include evidence that shows your choice of best size of the hidden layer. The explanations must be clear and concise. To make this task less time-consuming, use $n=5$."
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 3,
      "metadata": {
        "id": "uY6sDbUn0dC6"
      },
      "outputs": [],
      "source": [
        "# Write your code and answers here. Feel free to add more code and markdown cells.\n",
        "from tensorflow.keras import models, layers"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 16,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "8RoOchKEOC5m",
        "outputId": "eedd94cd-5de7-4ef3-c120-8748163ca2f3"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.7/dist-packages/sklearn/utils/deprecation.py:87: FutureWarning: Function get_feature_names is deprecated; get_feature_names is deprecated in 1.0 and will be removed in 1.2. Please use get_feature_names_out instead.\n",
            "  warnings.warn(msg, category=FutureWarning)\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Model: \"sequential\"\n",
            "_________________________________________________________________\n",
            " Layer (type)                Output Shape              Param #   \n",
            "=================================================================\n",
            " dense (Dense)               (None, 363)               1732236   \n",
            "                                                                 \n",
            "=================================================================\n",
            "Total params: 1,732,236\n",
            "Trainable params: 1,732,236\n",
            "Non-trainable params: 0\n",
            "_________________________________________________________________\n"
          ]
        }
      ],
      "source": [
        "\n",
        "simple_nn = models.Sequential()\n",
        "simple_nn.add(layers.Dense(363, activation='sigmoid', input_shape=(len(tfidf.get_feature_names()),)))\n",
        "simple_nn.summary()"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "QNDZMO6yOC5m"
      },
      "outputs": [],
      "source": [
        ""
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "k0NeK3gM0dC9"
      },
      "source": [
        "# Task 4 (5 marks): Recurrent NN\n",
        "\n",
        "Use the files `training.csv`, `dev_test.csv`, and `test.csv` provided by us.\n",
        "\n",
        "Implement a more complex neural network that is composed of the following layers:\n",
        "\n",
        "* An embedding layer that generates embedding vectors of the sentence text with 35 dimensions.\n",
        "* A LSTM layer. You need to determine the size of this LSTM layer, and the text length limit (if needed).\n",
        "* The final output layer with one cell for binary classification, as in task 3.\n",
        "\n",
        "Train the model with the training data, use the dev_test set to determine a good size of the LSTM layer and an appropriate length limit (if needed), and report the final results using the test set. Again, remember to use the test set only after you have determined the optimal parameters of the LSTM layer.\n",
        "\n",
        "Based on your experiments, comment on whether this system is better than the systems developed in the previous tasks.\n",
        "\n",
        "The breakdown of marks is as follows:\n",
        "\n",
        "* **1 mark** if the NN model has the correct layers, the correct activation functions, and the correct loss function.\n",
        "* **1 mark** if the code passes the sentence text to the model correctly. The documentation needs to explain what decisions had to be made to process long sentences. In particular, did you need to truncate the input text, and how did you determine the length limit?\n",
        "* **1 mark** if the code returns the IDs of the *n* sentences that have the highest prediction score in the given question.\n",
        "* **1 mark** if the notebook reports the F1 scores of the test sets and comments on the results.\n",
        "* **1 mark** for good coding and documentation in this task. In particular, the code and results must include evidence that shows your choice of best size of the LSTM layer. The explanations must be clear and concise. To make this task less time-consuming, use $n=5$."
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "c8RRCWeQTrPl"
      },
      "outputs": [],
      "source": [
        "# Write your code and answers here. Feel free to add more code and markdown cells."
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "ppkBsuB_0dC9"
      },
      "source": [
        "# Submission of results\n",
        "\n",
        "Your submission should consist of this Jupyter notebook with all your code and explanations inserted into the notebook as text cells. **The notebook should contain the output of the runs. All code should run. Code with syntax errors or code without output will not be assessed.**\n",
        "\n",
        "**Do not submit multiple files. If you feel you need to submit multiple files, please contact Diego.Molla-Aliod@mq.edu.au first.**\n",
        "\n",
        "Examine the text cells of this notebook so that you can have an idea of how to format text for good visual impact. You can also read this useful [guide to the MarkDown notation](https://daringfireball.net/projects/markdown/syntax),  which explains the format of the text cells.\n",
        "\n",
        "Each task specifies a number of marks. The final mark of the assignment is the sum of all the marks of each individual task.\n",
        "\n",
        "By submitting this assignment you are acknowledging that this is your own work. Any submissions that break the code of academic honesty will be penalised as per [the academic integrity policy](https://policies.mq.edu.au/document/view.php?id=3).\n",
        "\n",
        "Late submissions **will not be accepted** without an approved [Special Consideration](http://from.mq.edu.au/MT0X0E0FUrrU200rm0JB0U0) request.  Assessments submitted after the due date will receive a mark of **zero**."
      ]
    }
  ],
  "metadata": {
    "accelerator": "GPU",
    "colab": {
      "name": "A2.ipynb",
      "provenance": [],
      "collapsed_sections": [],
      "include_colab_link": true
    },
    "interpreter": {
      "hash": "a7b63e7410c98f344f02082f10d790581d1dba1eeb1c8fe30f342f6109f0429e"
    },
    "kernelspec": {
      "display_name": "Python 3",
      "language": "python",
      "name": "python3"
    },
    "language_info": {
      "codemirror_mode": {
        "name": "ipython",
        "version": 3
      },
      "file_extension": ".py",
      "mimetype": "text/x-python",
      "name": "python",
      "nbconvert_exporter": "python",
      "pygments_lexer": "ipython3",
      "version": "3.8.8"
    }
  },
  "nbformat": 4,
  "nbformat_minor": 0
}